{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/var/folders/92/gfh58db97nv7kkklt8263w9w0000gp/T/ipykernel_22512/3824191928.py:5: DtypeWarning: Columns (15) have mixed types. Specify dtype option on import or set low_memory=False.\n",
      "  df = pd.read_csv(\"../data/rummagenexrummageo.csv\")\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>rummagene</th>\n",
       "      <th>rummageo</th>\n",
       "      <th>rummagene-desc</th>\n",
       "      <th>p-value</th>\n",
       "      <th>sidak</th>\n",
       "      <th>fdr</th>\n",
       "      <th>odds</th>\n",
       "      <th>n-overlap</th>\n",
       "      <th>rummagene-size</th>\n",
       "      <th>rummageo-size</th>\n",
       "      <th>overlaps</th>\n",
       "      <th>species</th>\n",
       "      <th>geo_gse</th>\n",
       "      <th>pmc_id</th>\n",
       "      <th>pmid</th>\n",
       "      <th>geo_pmid</th>\n",
       "      <th>geo_gsei</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>PMC7185143-aging-12-102991-s001..docx-0-Gene_name</td>\n",
       "      <td>GSE159694-3-vs-0-human up</td>\n",
       "      <td>co-expression analysis of all ArrayExpress dat...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>64.730851</td>\n",
       "      <td>183</td>\n",
       "      <td>219.0</td>\n",
       "      <td>284</td>\n",
       "      <td>STIL;POLE2;TTK;CDCA2;RAD51;NCAPD2;DEPDC1B;PKMY...</td>\n",
       "      <td>human</td>\n",
       "      <td>GSE159694</td>\n",
       "      <td>PMC7185143</td>\n",
       "      <td>32275642</td>\n",
       "      <td>33264627</td>\n",
       "      <td>GSE159694</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>PMC9762333-ccr-21-3329_supplementary_tables_1-...</td>\n",
       "      <td>GSE159694-3-vs-0-human up</td>\n",
       "      <td>Click here for additional data file</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>63.068678</td>\n",
       "      <td>184</td>\n",
       "      <td>226.0</td>\n",
       "      <td>284</td>\n",
       "      <td>XRCC2;MCM3;SKA3;NCAPG;MELK;PCNA;NUF2;ZNF367;FA...</td>\n",
       "      <td>human</td>\n",
       "      <td>GSE159694</td>\n",
       "      <td>PMC9762333</td>\n",
       "      <td>34965943</td>\n",
       "      <td>33264627</td>\n",
       "      <td>GSE159694</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>PMC7185143-aging-12-102991-s001..docx-0-Gene_name</td>\n",
       "      <td>GSE184536-7-vs-11-human dn</td>\n",
       "      <td>co-expression analysis of all ArrayExpress dat...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>62.962248</td>\n",
       "      <td>178</td>\n",
       "      <td>219.0</td>\n",
       "      <td>284</td>\n",
       "      <td>STIL;TTK;CDCA2;RAD51;NCAPD3;NCAPD2;DEPDC1B;PKM...</td>\n",
       "      <td>human</td>\n",
       "      <td>GSE184536</td>\n",
       "      <td>PMC7185143</td>\n",
       "      <td>32275642</td>\n",
       "      <td>34523966</td>\n",
       "      <td>GSE184536</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>PMC9762333-ccr-21-3329_supplementary_tables_1-...</td>\n",
       "      <td>GSE184536-7-vs-11-human dn</td>\n",
       "      <td>Click here for additional data file</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>62.040384</td>\n",
       "      <td>181</td>\n",
       "      <td>226.0</td>\n",
       "      <td>284</td>\n",
       "      <td>XRCC2;MCM3;SKA3;NCAPG;MELK;GINS4;PCNA;NUF2;ZNF...</td>\n",
       "      <td>human</td>\n",
       "      <td>GSE184536</td>\n",
       "      <td>PMC9762333</td>\n",
       "      <td>34965943</td>\n",
       "      <td>34523966</td>\n",
       "      <td>GSE184536</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>PMC7185143-aging-12-102991-s001..docx-0-Gene_name</td>\n",
       "      <td>GSE159694-3-vs-2-human up</td>\n",
       "      <td>co-expression analysis of all ArrayExpress dat...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>51.160932</td>\n",
       "      <td>192</td>\n",
       "      <td>219.0</td>\n",
       "      <td>377</td>\n",
       "      <td>STIL;POLE2;TTK;CDCA2;RAD51;NCAPD3;NCAPD2;DEPDC...</td>\n",
       "      <td>human</td>\n",
       "      <td>GSE159694</td>\n",
       "      <td>PMC7185143</td>\n",
       "      <td>32275642</td>\n",
       "      <td>33264627</td>\n",
       "      <td>GSE159694</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>995</th>\n",
       "      <td>995</td>\n",
       "      <td>PMC5794756-41598_2018_20217_MOESM4_ESM.xlsx-Do...</td>\n",
       "      <td>GSE119336,GSE119337-0-vs-3-human up</td>\n",
       "      <td>were upregulated and 835 were downregulated in...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>13.020687</td>\n",
       "      <td>378</td>\n",
       "      <td>628.0</td>\n",
       "      <td>1017</td>\n",
       "      <td>CNIH1;F9;NR1I3;C6;SFXN1;TLCD4;CD36;KLKB1;IGFBP...</td>\n",
       "      <td>human</td>\n",
       "      <td>GSE119336</td>\n",
       "      <td>PMC5794756</td>\n",
       "      <td>29391527</td>\n",
       "      <td>36476255</td>\n",
       "      <td>GSE119336,GSE119337</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>996</th>\n",
       "      <td>996</td>\n",
       "      <td>PMC4951018-pone.0159329.s004.xlsx-Cluster_2_67...</td>\n",
       "      <td>GSE166238-2-vs-6-human dn</td>\n",
       "      <td>Expression values of all upregulated genes. (c...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>13.018896</td>\n",
       "      <td>397</td>\n",
       "      <td>589.0</td>\n",
       "      <td>1139</td>\n",
       "      <td>BMAL2;NBN;ICAM1;IFIT2;PARP9;BATF2;HIVEP2;PELI1...</td>\n",
       "      <td>human</td>\n",
       "      <td>GSE166238</td>\n",
       "      <td>PMC4951018</td>\n",
       "      <td>27434537</td>\n",
       "      <td>34267957,35930640</td>\n",
       "      <td>GSE166238</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>997</th>\n",
       "      <td>997</td>\n",
       "      <td>PMC9362915-pone.0272017.s004.xlsx-Day0_0_0_Top...</td>\n",
       "      <td>GSE210352-5-vs-6-mouse dn</td>\n",
       "      <td>them. A full list of genes for each path is pr...</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>NaN</td>\n",
       "      <td>13.015083</td>\n",
       "      <td>479</td>\n",
       "      <td>714.0</td>\n",
       "      <td>1134</td>\n",
       "      <td>PALM;PPM1J;E2F3;NCAPH;EGFL7;MAP3K8;CCNE2;S1PR5...</td>\n",
       "      <td>mouse</td>\n",
       "      <td>GSE210352</td>\n",
       "      <td>PMC9362915</td>\n",
       "      <td>35944008</td>\n",
       "      <td>Empty</td>\n",
       "      <td>GSE210352</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>998</th>\n",
       "      <td>998</td>\n",
       "      <td>PMC11188533-12993_2024_240_MOESM2_ESM.xlsx-Fig...</td>\n",
       "      <td>GSE75246-4-vs-6-mouse up</td>\n",
       "      <td>None here</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>13.011917</td>\n",
       "      <td>351</td>\n",
       "      <td>464.0</td>\n",
       "      <td>1279</td>\n",
       "      <td>SORBS1;P2RY14;MAP2;ASTN2;PDZRN4;GRIN2A;CHSY3;E...</td>\n",
       "      <td>mouse</td>\n",
       "      <td>GSE75246</td>\n",
       "      <td>PMC11188533</td>\n",
       "      <td>38898502</td>\n",
       "      <td>27097852</td>\n",
       "      <td>GSE75246</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>999</th>\n",
       "      <td>999</td>\n",
       "      <td>PMC11188533-12993_2024_240_MOESM2_ESM.xlsx-Sup...</td>\n",
       "      <td>GSE75246-4-vs-6-mouse up</td>\n",
       "      <td>None here</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>13.011917</td>\n",
       "      <td>351</td>\n",
       "      <td>464.0</td>\n",
       "      <td>1279</td>\n",
       "      <td>SORBS1;P2RY14;MAP2;ASTN2;PDZRN4;GRIN2A;CHSY3;E...</td>\n",
       "      <td>mouse</td>\n",
       "      <td>GSE75246</td>\n",
       "      <td>PMC11188533</td>\n",
       "      <td>38898502</td>\n",
       "      <td>27097852</td>\n",
       "      <td>GSE75246</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>1000 rows × 18 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     Unnamed: 0                                          rummagene  \\\n",
       "0             0  PMC7185143-aging-12-102991-s001..docx-0-Gene_name   \n",
       "1             1  PMC9762333-ccr-21-3329_supplementary_tables_1-...   \n",
       "2             2  PMC7185143-aging-12-102991-s001..docx-0-Gene_name   \n",
       "3             3  PMC9762333-ccr-21-3329_supplementary_tables_1-...   \n",
       "4             4  PMC7185143-aging-12-102991-s001..docx-0-Gene_name   \n",
       "..          ...                                                ...   \n",
       "995         995  PMC5794756-41598_2018_20217_MOESM4_ESM.xlsx-Do...   \n",
       "996         996  PMC4951018-pone.0159329.s004.xlsx-Cluster_2_67...   \n",
       "997         997  PMC9362915-pone.0272017.s004.xlsx-Day0_0_0_Top...   \n",
       "998         998  PMC11188533-12993_2024_240_MOESM2_ESM.xlsx-Fig...   \n",
       "999         999  PMC11188533-12993_2024_240_MOESM2_ESM.xlsx-Sup...   \n",
       "\n",
       "                                rummageo  \\\n",
       "0              GSE159694-3-vs-0-human up   \n",
       "1              GSE159694-3-vs-0-human up   \n",
       "2             GSE184536-7-vs-11-human dn   \n",
       "3             GSE184536-7-vs-11-human dn   \n",
       "4              GSE159694-3-vs-2-human up   \n",
       "..                                   ...   \n",
       "995  GSE119336,GSE119337-0-vs-3-human up   \n",
       "996            GSE166238-2-vs-6-human dn   \n",
       "997            GSE210352-5-vs-6-mouse dn   \n",
       "998             GSE75246-4-vs-6-mouse up   \n",
       "999             GSE75246-4-vs-6-mouse up   \n",
       "\n",
       "                                        rummagene-desc  p-value  sidak  fdr  \\\n",
       "0    co-expression analysis of all ArrayExpress dat...      0.0    0.0  0.0   \n",
       "1                  Click here for additional data file      0.0    0.0  0.0   \n",
       "2    co-expression analysis of all ArrayExpress dat...      0.0    0.0  0.0   \n",
       "3                  Click here for additional data file      0.0    0.0  0.0   \n",
       "4    co-expression analysis of all ArrayExpress dat...      0.0    0.0  0.0   \n",
       "..                                                 ...      ...    ...  ...   \n",
       "995  were upregulated and 835 were downregulated in...      0.0    0.0  0.0   \n",
       "996  Expression values of all upregulated genes. (c...      0.0    0.0  0.0   \n",
       "997  them. A full list of genes for each path is pr...      0.0    0.0  NaN   \n",
       "998                                          None here      0.0    0.0  0.0   \n",
       "999                                          None here      0.0    0.0  0.0   \n",
       "\n",
       "          odds  n-overlap  rummagene-size  rummageo-size  \\\n",
       "0    64.730851        183           219.0            284   \n",
       "1    63.068678        184           226.0            284   \n",
       "2    62.962248        178           219.0            284   \n",
       "3    62.040384        181           226.0            284   \n",
       "4    51.160932        192           219.0            377   \n",
       "..         ...        ...             ...            ...   \n",
       "995  13.020687        378           628.0           1017   \n",
       "996  13.018896        397           589.0           1139   \n",
       "997  13.015083        479           714.0           1134   \n",
       "998  13.011917        351           464.0           1279   \n",
       "999  13.011917        351           464.0           1279   \n",
       "\n",
       "                                              overlaps species    geo_gse  \\\n",
       "0    STIL;POLE2;TTK;CDCA2;RAD51;NCAPD2;DEPDC1B;PKMY...   human  GSE159694   \n",
       "1    XRCC2;MCM3;SKA3;NCAPG;MELK;PCNA;NUF2;ZNF367;FA...   human  GSE159694   \n",
       "2    STIL;TTK;CDCA2;RAD51;NCAPD3;NCAPD2;DEPDC1B;PKM...   human  GSE184536   \n",
       "3    XRCC2;MCM3;SKA3;NCAPG;MELK;GINS4;PCNA;NUF2;ZNF...   human  GSE184536   \n",
       "4    STIL;POLE2;TTK;CDCA2;RAD51;NCAPD3;NCAPD2;DEPDC...   human  GSE159694   \n",
       "..                                                 ...     ...        ...   \n",
       "995  CNIH1;F9;NR1I3;C6;SFXN1;TLCD4;CD36;KLKB1;IGFBP...   human  GSE119336   \n",
       "996  BMAL2;NBN;ICAM1;IFIT2;PARP9;BATF2;HIVEP2;PELI1...   human  GSE166238   \n",
       "997  PALM;PPM1J;E2F3;NCAPH;EGFL7;MAP3K8;CCNE2;S1PR5...   mouse  GSE210352   \n",
       "998  SORBS1;P2RY14;MAP2;ASTN2;PDZRN4;GRIN2A;CHSY3;E...   mouse   GSE75246   \n",
       "999  SORBS1;P2RY14;MAP2;ASTN2;PDZRN4;GRIN2A;CHSY3;E...   mouse   GSE75246   \n",
       "\n",
       "          pmc_id      pmid           geo_pmid             geo_gsei  \n",
       "0     PMC7185143  32275642           33264627            GSE159694  \n",
       "1     PMC9762333  34965943           33264627            GSE159694  \n",
       "2     PMC7185143  32275642           34523966            GSE184536  \n",
       "3     PMC9762333  34965943           34523966            GSE184536  \n",
       "4     PMC7185143  32275642           33264627            GSE159694  \n",
       "..           ...       ...                ...                  ...  \n",
       "995   PMC5794756  29391527           36476255  GSE119336,GSE119337  \n",
       "996   PMC4951018  27434537  34267957,35930640            GSE166238  \n",
       "997   PMC9362915  35944008              Empty            GSE210352  \n",
       "998  PMC11188533  38898502           27097852             GSE75246  \n",
       "999  PMC11188533  38898502           27097852             GSE75246  \n",
       "\n",
       "[1000 rows x 18 columns]"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "import json\n",
    "df = pd.read_csv(\"../data/rummagenexrummageo.csv\")\n",
    "df = df.drop_duplicates(subset='pmc_id', keep='first')\n",
    "df = df.sort_values(by=[\"cosine_similarity\", \"p-value\", \"odds\"], ascending=[True, True, False])\n",
    "df.index = np.arange(1, len(df)+1)\n",
    "df = df.head(1000)\n",
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "df_rummagene_map = {row[\"rummagene\"]: [row[\"rummagene-desc\"], row[\"pmc_id\"]] for _, row in df.iterrows()}\n",
    "df_rummageo_map = {row[\"rummageo\"]: [row[\"geo_gse\"], row[\"geo_gsei\"], row[\"species\"]] for _, row in df.iterrows()}\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open(\"../data/gse_results.json\" ,\"r\") as f:\n",
    "        gse_info = json.load(f)\n",
    "with open(\"../data/title_abs.json\" ,\"r\") as f:\n",
    "        pmc_sum = json.load(f)\n",
    "with open(\"../data/human-gse-processed-meta.json\") as f:\n",
    "        gse_human_info = json.load(f)\n",
    "with open(\"../data/mouse-gse-processed-meta.json\") as f:\n",
    "        gse_mouse_info = json.load(f)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "\n",
    "def extract_after_extension(termid):\n",
    "    pattern = r'^(.+?\\.\\w+)-+(.*)$'\n",
    "    \n",
    "    match = re.match(pattern, termid)\n",
    "    \n",
    "    if match:\n",
    "        return match.group(2)\n",
    "    else:\n",
    "        return None\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'PMC7185143-aging-12-102991-s001..docx-0-Gene_name': ('PCNA-associated factor KIAA0101 transcriptionally induced by ELK1 controls cell proliferation and apoptosis in nasopharyngeal carcinoma: an integrated bioinformatics and experimental study',\n",
       "  'KIAA0101, previously identified as PCNA-associated factor, is overexpressed among almost majority of human cancers and has emerged as an important regulator of cancer progression; however, its function in human nasopharyngeal carcinoma (NPC) remain unknown. Integrated bioinformatics approaches were employed to determine the KIAA0101 expressions in the NPC samples. Lentiviral vectors carrying KIAA0101 shRNA were constructed and stable transfected cells were validated by qRT-PCR and western blot. Cellular functions were then evaluated by MTT, colony formation, Brdu staining, and flow cytometry. Mechanistic studies were systematically investigated by UCSC Genome Browser, GEO, UALCAN, QIAGEN, PROMO and JASPAR, ChIP, and the cBioPortal, et al. The results showed that KIAA0101 ranked top overexpressed gene lists in GSE6631 dataset. KIAA0101 was highly expressed in NPC tissues and cell lines. Furthermore, knockdown of KIAA0101 significantly inhibited cell proliferation and DNA replication, promoted apoptosis and cell cycle arrest in vitro. Meanwhile, the mechanistic study revealed that MAP kinase phosphorylation-dependent activation of ELK1 may enhance neighbor gene expressions of KIAA0101 and TRIP4 by binding both promotor regions in the NPC cells. Taken together, our findings indicate that overexpression of KIAA0101 activated by MAP kinase phosphorylation-dependent activation of ELK1 may play an important role in NPC progression.',\n",
       "  '0-Gene_name',\n",
       "  'co-expression analysis of all ArrayExpress datasets and tumor only datasets with two KIAA0101 probes 202**5**'),\n",
       " 'PMC9762333-ccr-21-3329_supplementary_tables_1-3_supp1-3.docx-1-Proliferation': ('Transcriptomic Determinants of Response to Pembrolizumab Monotherapy across Solid Tumor Types',\n",
       "  'Purpose: To explore relationships between biological gene expression signatures and pembrolizumab response. Experimental design: RNA-sequencing data on baseline tumor tissue from 1,188 patients across seven tumor types treated with pembrolizumab monotherapy in nine clinical trials were used. A total of 11 prespecified gene expression signatures [18-gene T-cell-inflamed gene expression profile (TcellinfGEP), angiogenesis, hypoxia, glycolysis, proliferation, MYC, RAS, granulocytic myeloid-derived suppressor cell (gMDSC), monocytic myeloid-derived suppressor cell (mMDSC), stroma/epithelial-to-mesenchymal transition (EMT)/TGFβ, and WNT] were evaluated for their relationship to objective response rate (per RECIST, version 1.1). Logistic regression analysis of response for consensus signatures was adjusted for tumor type, Eastern Cooperative Oncology Group performance status, and TcellinfGEP, an approach equivalent to evaluating the association between response and the residuals of consensus signatures after detrending them for their relationship with the TcellinfGEP (previously identified as a determinant of pembrolizumab response) and tumor type. Testing of the 10 prespecified non-TcellinfGEP consensus signatures for negative association [except proliferation (hypothesized positive association)] with response was adjusted for multiplicity. Results: Covariance patterns of the 11 signatures (including TcellinfGEP) identified in Merck-Moffitt and The Cancer Genome Atlas datasets showed highly concordant coexpression patterns in the RNA-sequencing data from pembrolizumab trials. TcellinfGEP was positively associated with response; signatures for angiogenesis, mMDSC, and stroma/EMT/TGFβ were negatively associated with response to pembrolizumab monotherapy. Conclusions: These findings suggest that features beyond IFNγ-related T-cell inflammation may be relevant to anti-programmed death 1 monotherapy response and may define other axes of tumor biology as candidates for pembrolizumab combinations. See related commentary by Cho et al., p. 1479.',\n",
       "  '1-Proliferation',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC11025768-pone.0298004.s003.xlsx-Sheet1-Unnamed_0': ('Integration of single-cell RNA-seq and bulk RNA-seq to construct liver hepatocellular carcinoma stem cell signatures to explore their impact on patient prognosis and treatment',\n",
       "  'Background: Liver hepatocellular carcinoma (LIHC) is a prevalent form of primary liver cancer. Research has demonstrated the contribution of tumor stem cells in facilitating tumor recurrence, metastasis, and treatment resistance. Despite this, there remains a lack of established cancer stem cells (CSCs)-associated genes signatures for effectively predicting the prognosis and guiding the treatment strategies for patients diagnosed with LIHC. Methods: The single-cell RNA sequencing (scRNA-seq) and bulk RNA transcriptome data were obtained based on public datasets and computerized firstly using CytoTRACE package and One Class Linear Regression (OCLR) algorithm to evaluate stemness level, respectively. Then, we explored the association of stemness indicators (CytoTRACE score and stemness index, mRNAsi) with survival outcomes and clinical characteristics by combining clinical information and survival analyses. Subsequently, weighted co-expression network analysis (WGCNA) and Cox were applied to assess mRNAsi-related genes in bulk LIHC data and construct a prognostic model for LIHC patients. Single-sample gene-set enrichment analysis (ssGSEA), Cell-type Identification By Estimating Relative Subsets Of RNA Transcripts (CIBERSORT) and Tumor Immune Estimation Resource (TIMER) analysis were employed for immune infiltration assessment. Finally, the potential immunotherapeutic response was predicted by the Tumor Immune Dysfunction and Exclusion (TIDE), and the tumor mutation burden (TMB). Additionally, pRRophetic package was applied to evaluate the sensitivity of high and low-risk groups to common chemotherapeutic drugs. Results: A total of four genes (including STIP1, H2AFZ, BRIX1, and TUBB) associated with stemness score (CytoTRACE score and mRNAsi) were identified and constructed a risk model that could predict prognosis in LIHC patients. It was observed that high stemness cells occurred predominantly in the late stages of LIHC and that poor overall survival in LIHC patients was also associated with high mRNAsi scores. In addition, pathway analysis confirmed the biological uniqueness of the two risk groups. Personalized treatment predictions suggest that patients with a low risk benefited more from immunotherapy, while those with a high risk group may be conducive to chemotherapeutic drugs. Conclusion: The current study developed a novel prognostic risk signature with genes related to CSCs, which provides novel ideas for the diagnosis, prognosis and treatment of LIHC.',\n",
       "  'Sheet1-Unnamed_0',\n",
       "  'None here'),\n",
       " 'PMC10634886-media-5.xlsx-LIN37_Uxa_et_al_2019-genename': ('HDAC activity is dispensable for repression of cell-cycle genes by DREAM and E2F:RB complexes',\n",
       "  'Histone deacetylases (HDACs) are pivotal in transcriptional regulation, and their dysregulation has been associated with various diseases including cancer. One of the critical roles of HDAC-containing complexes is the deacetylation of histone tails, which is canonically linked to transcriptional repression. Previous research has indicated that HDACs are recruited to cell-cycle gene promoters through the RB protein or the DREAM complex via SIN3B and that HDAC activity is essential for repressing G1/S and G2/M cell-cycle genes during cell-cycle arrest and exit. In this study, we sought to explore the interdependence of DREAM, RB, SIN3 proteins, and HDACs in the context of cell-cycle gene repression. We found that genetic knockout of SIN3B did not lead to derepression of cell-cycle genes in non-proliferating HCT116 and C2C12 cells. A combined loss of SIN3A and SIN3B resulted in a moderate upregulation in mRNA expression of several cell-cycle genes in arrested HCT116 cells, however, these effects appeared to be independent of DREAM or RB. Furthermore, HDAC inhibition did not induce a general upregulation of RB and DREAM target gene expression in arrested transformed or non-transformed cells. Our findings provide evidence that E2F:RB and DREAM complexes can repress cell-cycle genes without reliance on HDAC activity.',\n",
       "  'LIN37_Uxa_et_al_2019-genename',\n",
       "  'and downregulated (≥1.5fold) within all three conditions when compared to wild-type cells (Fig. 5A, **Suppl. Tab. S4**)  Suppl. Table S4: Differentially expressed genes in SIN3B−/−, SIN3A knockdown, and SIN3B−/−;SIN3A knockdown HCT116 cells'),\n",
       " 'PMC11126580-41467_2024_48724_MOESM7_ESM.xlsx-LIN37_Uxa_et_al_2019-genename': ('HDAC activity is dispensable for repression of cell-cycle genes by DREAM and E2F:RB complexes',\n",
       "  'Histone deacetylases (HDACs) play a crucial role in transcriptional regulation and are implicated in various diseases, including cancer. They are involved in histone tail deacetylation and canonically linked to transcriptional repression. Previous studies suggested that HDAC recruitment to cell-cycle gene promoters via the retinoblastoma (RB) protein or the DREAM complex through SIN3B is essential for G1/S and G2/M gene repression during cell-cycle arrest and exit. Here we investigate the interplay among DREAM, RB, SIN3 proteins, and HDACs in the context of cell-cycle gene repression. Knockout of SIN3B does not globally derepress cell-cycle genes in non-proliferating HCT116 and C2C12 cells. Loss of SIN3A/B moderately upregulates several cell-cycle genes in HCT116 cells but does so independently of DREAM/RB. HDAC inhibition does not induce general upregulation of RB/DREAM target genes in arrested transformed or non-transformed cells. Our findings suggest that E2F:RB and DREAM complexes can repress cell-cycle genes without relying on HDAC activity.',\n",
       "  'LIN37_Uxa_et_al_2019-genename',\n",
       "  'None here'),\n",
       " 'PMC10634886-media-5.xlsx-LIN37_Uxa_et_al_2019-ENSG': ('HDAC activity is dispensable for repression of cell-cycle genes by DREAM and E2F:RB complexes',\n",
       "  'Histone deacetylases (HDACs) are pivotal in transcriptional regulation, and their dysregulation has been associated with various diseases including cancer. One of the critical roles of HDAC-containing complexes is the deacetylation of histone tails, which is canonically linked to transcriptional repression. Previous research has indicated that HDACs are recruited to cell-cycle gene promoters through the RB protein or the DREAM complex via SIN3B and that HDAC activity is essential for repressing G1/S and G2/M cell-cycle genes during cell-cycle arrest and exit. In this study, we sought to explore the interdependence of DREAM, RB, SIN3 proteins, and HDACs in the context of cell-cycle gene repression. We found that genetic knockout of SIN3B did not lead to derepression of cell-cycle genes in non-proliferating HCT116 and C2C12 cells. A combined loss of SIN3A and SIN3B resulted in a moderate upregulation in mRNA expression of several cell-cycle genes in arrested HCT116 cells, however, these effects appeared to be independent of DREAM or RB. Furthermore, HDAC inhibition did not induce a general upregulation of RB and DREAM target gene expression in arrested transformed or non-transformed cells. Our findings provide evidence that E2F:RB and DREAM complexes can repress cell-cycle genes without reliance on HDAC activity.',\n",
       "  'LIN37_Uxa_et_al_2019-ENSG',\n",
       "  'and downregulated (≥1.5fold) within all three conditions when compared to wild-type cells (Fig. 5A, **Suppl. Tab. S4**)  Suppl. Table S4: Differentially expressed genes in SIN3B−/−, SIN3A knockdown, and SIN3B−/−;SIN3A knockdown HCT116 cells'),\n",
       " 'PMC11126580-41467_2024_48724_MOESM7_ESM.xlsx-LIN37_Uxa_et_al_2019-ENSG': ('HDAC activity is dispensable for repression of cell-cycle genes by DREAM and E2F:RB complexes',\n",
       "  'Histone deacetylases (HDACs) play a crucial role in transcriptional regulation and are implicated in various diseases, including cancer. They are involved in histone tail deacetylation and canonically linked to transcriptional repression. Previous studies suggested that HDAC recruitment to cell-cycle gene promoters via the retinoblastoma (RB) protein or the DREAM complex through SIN3B is essential for G1/S and G2/M gene repression during cell-cycle arrest and exit. Here we investigate the interplay among DREAM, RB, SIN3 proteins, and HDACs in the context of cell-cycle gene repression. Knockout of SIN3B does not globally derepress cell-cycle genes in non-proliferating HCT116 and C2C12 cells. Loss of SIN3A/B moderately upregulates several cell-cycle genes in HCT116 cells but does so independently of DREAM/RB. HDAC inhibition does not induce general upregulation of RB/DREAM target genes in arrested transformed or non-transformed cells. Our findings suggest that E2F:RB and DREAM complexes can repress cell-cycle genes without relying on HDAC activity.',\n",
       "  'LIN37_Uxa_et_al_2019-ENSG',\n",
       "  'None here'),\n",
       " 'PMC10121712-41467_2023_37437_MOESM21_ESM.xlsx-LDAM-Up': ('Single-nucleus RNA-sequencing of autosomal dominant Alzheimer disease and risk variant carriers',\n",
       "  'Genetic studies of Alzheimer disease (AD) have prioritized variants in genes related to the amyloid cascade, lipid metabolism, and neuroimmune modulation. However, the cell-specific effect of variants in these genes is not fully understood. Here, we perform single-nucleus RNA-sequencing (snRNA-seq) on nearly 300,000 nuclei from the parietal cortex of AD autosomal dominant (APP and PSEN1) and risk-modifying variant (APOE, TREM2 and MS4A) carriers. Within individual cell types, we capture genes commonly dysregulated across variant groups. However, specific transcriptional states are more prevalent within variant carriers. TREM2 oligodendrocytes show a dysregulated autophagy-lysosomal pathway, MS4A microglia have dysregulated complement cascade genes, and APOEε4 inhibitory neurons display signs of ferroptosis. All cell types have enriched states in autosomal dominant carriers. We leverage differential expression and single-nucleus ATAC-seq to map GWAS signals to effector cell types including the NCK2 signal to neurons in addition to the initially proposed microglia. Overall, our results provide insights into the transcriptional diversity resulting from AD genetic architecture and cellular heterogeneity. The data can be explored on the online browser ( http://web.hararilab.org/SNARE/ ).',\n",
       "  'LDAM-Up',\n",
       "  'including “disease-associated microglia” (DAM)38, “microglial neurodegenerative” (MGnD)39, and “human AD microglia” (HAM)40 (Supplementary Dataset **18**)  Supplementary Dataset 18'),\n",
       " 'PMC9827063-mmc2.xlsx-UP-aKGvsGln2_DOWN': ('Epigenetic regulation of IPF fibroblast phenotype by glutaminolysis',\n",
       "  'Objective: Excessive extra-cellular-matrix production and uncontrolled proliferation of the fibroblasts are characteristics of many fibrotic diseases, including idiopathic pulmonary fibrosis (IPF). The fibroblasts have enhanced glutaminolysis with up-regulated glutaminase, GLS1, which converts glutamine to glutamate. Here, we investigated the role of glutaminolysis and glutaminolysis-derived metabolite α-ketoglutarate (α-KG) on IPF fibroblast phenotype and gene expression. Methods: Reduced glutamine conditions were carried out either using glutamine-free culture medium or silencing the expression of GLS1 with siRNA, with or without α-KG compensation. Cell phenotype has been characterized under these different conditions, and gene expression profile was examined by RNA-Seq. Specific profibrotic genes (Col3A1 and PLK1) expression were examined by real-time PCR and western blots. The levels of repressive histone H3K27me3, which demethylase activity is affected by glutaminolysis, were examined and H3K27me3 association with promoter region of Col3A1 and PLK1 were checked by ChIP assays. Effects of reduced glutaminolysis on fibrosis markers were checked in an animal model of lung fibrosis. Results: The lack of glutamine in the culture medium alters the profibrotic phenotype of activated fibroblasts. The addition of exogenous and glutaminolysis-derived metabolite α-KG to glutamine-free media barely restores the pro-fibrotic phenotype of activated fibroblasts. Many genes are down-regulated in glutamine-free medium, α-KG supplementation only rescues a limited number of genes. As α-KG is a cofactor for histone demethylases of H3K27me3, the reduced glutaminolysis alters H3K27me3 levels, and enriches H3K27me3 association with Col3A1 and PLK1 promoter region. Adding α-KG in glutamine-free medium depleted H3K27me3 association with Col3A1 promoter region but not that of PLK1. In a murine model of lung fibrosis, mice with reduced glutaminolysis showed markedly reduced fibrotic markers. Conclusions: This study indicates that glutamine is critical for supporting pro-fibrotic fibroblast phenotype in lung fibrosis, partially through α-KG-dependent and -independent mechanisms, and supports targeting fibroblast metabolism as a therapeutic method for fibrotic diseases.',\n",
       "  'UP-aKGvsGln2_DOWN',\n",
       "  'Multimedia component 2'),\n",
       " 'PMC8789288-elife-73137-supp2.xlsx-IFNg_vs_Control-Gene_symbol': ('Overexpression screen of interferon-stimulated genes identifies RARRES3 as a restrictor of Toxoplasma gondii infection',\n",
       "  'Toxoplasma gondii is an important human pathogen infecting an estimated one in three people worldwide. The cytokine interferon gamma (IFNγ) is induced during infection and is critical for restricting T. gondii growth in human cells. Growth restriction is presumed to be due to the induction of interferon-stimulated genes (ISGs) that are upregulated to protect the host from infection. Although there are hundreds of ISGs induced by IFNγ, their individual roles in restricting parasite growth in human cells remain somewhat elusive. To address this deficiency, we screened a library of 414 IFNγ induced ISGs to identify factors that impact T. gondii infection in human cells. In addition to IRF1, which likely acts through the induction of numerous downstream genes, we identified RARRES3 as a single factor that restricts T. gondii infection by inducing premature egress of the parasite in multiple human cell lines. Overall, while we successfully identified a novel IFNγ induced factor restricting T. gondii infection, the limited number of ISGs capable of restricting T. gondii infection when individually expressed suggests that IFNγ-mediated immunity to T. gondii infection is a complex, multifactorial process.',\n",
       "  'IFNg_vs_Control-Gene_symbol',\n",
       "  'upregulated by IRF1 and 380 genes upregulated by IFNγ in A549 cells (FDR≤0.05, twofold) (**Supplementary file 2**)  List of genes induced by IRF1 ectopic expression or IFNγ treatment in A549 cells'),\n",
       " 'PMC9399107-41467_2022_32627_MOESM6_ESM.xlsx-12_main_cell-types-gene_name_Erythrocytes': ('scRNA-seq of gastric tumor shows complex intercellular interaction with an alternative T cell exhaustion trajectory',\n",
       "  'The tumor microenvironment (TME) in gastric cancer (GC) has been shown to be important for tumor control but the specific characteristics for GC are not fully appreciated. We generated an atlas of 166,533 cells from 10 GC patients with matched paratumor tissues and blood. Our results show tumor-associated stromal cells (TASCs) have upregulated activity of Wnt signaling and angiogenesis, and are negatively correlated with survival. Tumor-associated macrophages and LAMP3 + DCs are involved in mediating T cell activity and form intercellular interaction hubs with TASCs. Clonotype and trajectory analysis demonstrates that Tc17 (IL-17 + CD8 + T cells) originate from tissue-resident memory T cells and can subsequently differentiate into exhausted T cells, suggesting an alternative pathway for T cell exhaustion. Our results indicate that IL17 + cells may promote tumor progression through IL17, IL22, and IL26 signaling, highlighting the possibility of targeting IL17 + cells and associated signaling pathways as a therapeutic strategy to treat GC.',\n",
       "  '12_main_cell-types-gene_name_Erythrocytes',\n",
       "  'PCA, UMAP, and Leiden clustering independently for further analysis. For these analyses, the first **3**  Supplementary Data 3'),\n",
       " 'PMC9626989-Table_2.DOCX-0-Gene': ('CENPA acts as a prognostic factor that relates to immune infiltrates in gliomas',\n",
       "  'Background: Glioma is the most common primary tumor of the central nervous system (CNS). Centromere protein A (CENPA) plays an essential role in ensuring that mitosis proceeds normally. The effect of CENPA on glioma is rarely reported. However, the current study aims to explore whether aberrant CENPA expression promotes glioma progression and the potential mechanisms involved. Methods: The GEPIA website, The Cancer Genome Atlas, and the Gene Expression Omnibus (GEO) were used to assess the expression of CENPA in glioma. The results were validated by real-time quantitative polymerase chain reaction and immunohistochemical staining of clinical samples. The relationship between the expression and prognostic value of the CENPA gene in glioma was investigated by Kaplan-Meier (KM) survival analysis with RNA-seq and clinical profiles downloaded from the Chinese Glioma Genome Atlas (CGGA) and UCSC Xena. The association between CENPA and clinical characteristics was also evaluated. Cell Counting Kit-8 (CCK8) assay, wound healing assay using two glioma cell lines, gene set enrichment analysis (GSEA), KEGG and gene ontology (GO) enrichment analysis, immune infiltration analysis, temozolomide (TMZ) sensitivity analysis, and single-cell sequence analysis were performed to explore the underlying mechanisms of high CENPA expression and its effect on glioma development. Finally, we performed a Cox analysis based on the expression of CENPA to predict patient prognosis. Results: CENPA was significantly upregulated in glioma tissue samples and correlated with patient prognosis. Moreover, the downregulation of CENPA inhibited the migration and proliferation of glioma cells. In addition, the expression level of CENPA was significantly correlated with the grade, primary-recurrent-secondary (PRS) type, IDH mutation status, and 1p19q codeletion status. Furthermore, CENPA could serve as an independent prognostic factor for glioma that mainly interferes with the normal progression of mitosis and regulates the tumor immune microenvironment favoring glioma development. Conclusion: CENPA may act as a prognostic factor in patients with glioma and provide a novel target for the treatment of gliomas.',\n",
       "  '0-Gene',\n",
       "  '< 0.005. A total of 418 genes co-expressed with CENPA were finally screened out (**Supplementary Table 2**)  Click here for additional data file'),\n",
       " 'PMC5043525-13059_2016_1062_MOESM7_ESM.xlsx-Oligodendrocytes-Gene': ('CellMapper: rapid and accurate inference of gene expression in difficult-to-isolate cell types',\n",
       "  'We present a sensitive approach to predict genes expressed selectively in specific cell types, by searching publicly available expression data for genes with a similar expression profile to known cell-specific markers. Our method, CellMapper, strongly outperforms previous computational algorithms to predict cell type-specific expression, especially for rare and difficult-to-isolate cell types. Furthermore, CellMapper makes accurate predictions for human brain cell types that have never been isolated, and can be rapidly applied to diverse cell types from many tissues. We demonstrate a clinically relevant application to prioritize candidate genes in disease susceptibility loci identified by GWAS.',\n",
       "  'Oligodendrocytes-Gene',\n",
       "  'None here'),\n",
       " 'PMC6092349-41389_2018_72_MOESM11_ESM.xlsx-Sheet1-Supplementary_Table_S1_List_of_down-regulated_genes_common_to_Noc_8h_vs_Ctrl_and_Noc_16h_vs_Ctrl': ('Chromosomal instability-induced senescence potentiates cell non-autonomous tumourigenic effects',\n",
       "  'Chromosomal instability (CIN), a high rate of chromosome loss or gain, is often associated with poor prognosis and drug resistance in cancers. Aneuploid, including near-polyploid, cells contain an abnormal number of chromosomes and exhibit CIN. The post-mitotic cell fates following generation of different degrees of chromosome mis-segregation and aneuploidy are unclear. Here we used aneuploidy inducers, nocodazole and reversine, to create different levels of aneuploidy. A higher extent of aneuploid and near-polyploid cells in a given population led to senescence. This was in contrast to cells with relatively lower levels of abnormal ploidy that continued to proliferate. Our findings revealed that senescence was accompanied by DNA damage and robust p53 activation. These senescent cells acquired the senescence-associated secretory phenotype (SASP). Depletion of p53 reduced the number of senescent cells with concomitant increase in cells undergoing DNA replication. Characterisation of these SASP factors demonstrated that they conferred paracrine pro-tumourigenic effects such as invasion, migration and angiogenesis both in vitro and in vivo. Finally, a correlation between increased aneuploidy and senescence was observed at the invasive front in breast carcinomas. Our findings demonstrate functional non-equivalence of discernable aneuploidies on tumourigenesis and suggest a cell non-autonomous mechanism by which aneuploidy-induced senescent cells and SASP can affect the tumour microenvironment to promote tumour progression.',\n",
       "  'Sheet1-Supplementary_Table_S1_List_of_down-regulated_genes_common_to_Noc_8h_vs_Ctrl_and_Noc_16h_vs_Ctrl',\n",
       "  'of upregulated and downregulated genes from RNA-seq data can be found in Supplementary Tables **S1**  Supplementary Table S1'),\n",
       " 'PMC6092349-41389_2018_72_MOESM11_ESM.xlsx-Sheet1-Unnamed_1': ('Chromosomal instability-induced senescence potentiates cell non-autonomous tumourigenic effects',\n",
       "  'Chromosomal instability (CIN), a high rate of chromosome loss or gain, is often associated with poor prognosis and drug resistance in cancers. Aneuploid, including near-polyploid, cells contain an abnormal number of chromosomes and exhibit CIN. The post-mitotic cell fates following generation of different degrees of chromosome mis-segregation and aneuploidy are unclear. Here we used aneuploidy inducers, nocodazole and reversine, to create different levels of aneuploidy. A higher extent of aneuploid and near-polyploid cells in a given population led to senescence. This was in contrast to cells with relatively lower levels of abnormal ploidy that continued to proliferate. Our findings revealed that senescence was accompanied by DNA damage and robust p53 activation. These senescent cells acquired the senescence-associated secretory phenotype (SASP). Depletion of p53 reduced the number of senescent cells with concomitant increase in cells undergoing DNA replication. Characterisation of these SASP factors demonstrated that they conferred paracrine pro-tumourigenic effects such as invasion, migration and angiogenesis both in vitro and in vivo. Finally, a correlation between increased aneuploidy and senescence was observed at the invasive front in breast carcinomas. Our findings demonstrate functional non-equivalence of discernable aneuploidies on tumourigenesis and suggest a cell non-autonomous mechanism by which aneuploidy-induced senescent cells and SASP can affect the tumour microenvironment to promote tumour progression.',\n",
       "  'Sheet1-Unnamed_1',\n",
       "  'of upregulated and downregulated genes from RNA-seq data can be found in Supplementary Tables **S1**  Supplementary Table S1'),\n",
       " 'PMC10956518-peerj-12-17130-s005.xlsx-Stomach_Adenocarcinoma_TCGA_P-rrelated_Gene': ('Validation of CDC45 as a novel biomarker for diagnosis and prognosis of gastric cancer',\n",
       "  'Background: Cell division cycle protein 45 (CDC45) has been demonstrated to play vital roles in the progression of various malignancies. However, the clinical significance of CDC45 in gastric cancer (GC) remains unreported. Method: In this study, we employed the TCGA database and the TCGA & GTEx dataset to compare the mRNA expression levels of CDC45 between gastric cancer tissues and adjacent or normal tissues ( p < 0.05 was considered statistically significant), which was further validated in multiple datasets including GSE13911, GSE29272, GSE118916, GSE66229, as well as RT-qPCR. Furthermore, we harnessed the Human Protein Atlas (HPA) to evaluate the protein expression of CDC45, which was subsequently verified through immunohistochemistry (IHC). To ascertain the diagnostic utility of CDC45, receiver operating characteristic (ROC) curves and the area under the ROC curve (AUC) were calculated in TCGA database, and further validated it in TCGA & GTEx and GSE66229 datasets. The Kaplan-Meier method was used to reveal the prognostic importance of CDC45 in The Cancer Genome Atlas (TCGA) database and authenticated through the GSE66229, GSE84433, and GSE84437 datasets. Through cBioPortal, we identified co-expressed genes of CDC45, and pursued enrichment analysis. Additionally, we availed gene set enrichment analysis (GSEA) to annotate the biological functions of CDC45. Results: Differential expression analysis revealed that CDC45 was significantly upregulated at both the mRNA and protein levels in GC (all p < 0.05 ). Remarkably, CDC45 emerged as a promising prognostic indicator and a novel diagnostic biomarker for GC. In a comprehensive the drug susceptibility analysis, we found that patients with high expression of CDC45 had high sensitivity to various chemotherapeutic agents, among which 5-fluorouracil, docetaxel, cisplatin, and elesclomol were most evident. Furthermore, our findings suggested a plausible association between CDC45 and immune cell infiltration. Enrichment analysis revealed that CDC45 and its associated genes may play crucial roles in muscle biofunction, whereas GSEA demonstrated significant enrichment of gene sets pertaining to G protein-coupled receptor ligand binding and G alpha (i) signaling events. Conclusion: Our study elucidates that upregulation of CDC45 is intricately associated with immune cell infiltration and holds promising potential as a favorable prognostic marker and a novel diagnostic biomarker for GC.',\n",
       "  'Stomach_Adenocarcinoma_TCGA_P-rrelated_Gene',\n",
       "  'None here'),\n",
       " 'PMC10956518-peerj-12-17130-s005.xlsx-Stomach_Adenocarcinoma_TCGA_F-Correlated_Gene': ('Validation of CDC45 as a novel biomarker for diagnosis and prognosis of gastric cancer',\n",
       "  'Background: Cell division cycle protein 45 (CDC45) has been demonstrated to play vital roles in the progression of various malignancies. However, the clinical significance of CDC45 in gastric cancer (GC) remains unreported. Method: In this study, we employed the TCGA database and the TCGA & GTEx dataset to compare the mRNA expression levels of CDC45 between gastric cancer tissues and adjacent or normal tissues ( p < 0.05 was considered statistically significant), which was further validated in multiple datasets including GSE13911, GSE29272, GSE118916, GSE66229, as well as RT-qPCR. Furthermore, we harnessed the Human Protein Atlas (HPA) to evaluate the protein expression of CDC45, which was subsequently verified through immunohistochemistry (IHC). To ascertain the diagnostic utility of CDC45, receiver operating characteristic (ROC) curves and the area under the ROC curve (AUC) were calculated in TCGA database, and further validated it in TCGA & GTEx and GSE66229 datasets. The Kaplan-Meier method was used to reveal the prognostic importance of CDC45 in The Cancer Genome Atlas (TCGA) database and authenticated through the GSE66229, GSE84433, and GSE84437 datasets. Through cBioPortal, we identified co-expressed genes of CDC45, and pursued enrichment analysis. Additionally, we availed gene set enrichment analysis (GSEA) to annotate the biological functions of CDC45. Results: Differential expression analysis revealed that CDC45 was significantly upregulated at both the mRNA and protein levels in GC (all p < 0.05 ). Remarkably, CDC45 emerged as a promising prognostic indicator and a novel diagnostic biomarker for GC. In a comprehensive the drug susceptibility analysis, we found that patients with high expression of CDC45 had high sensitivity to various chemotherapeutic agents, among which 5-fluorouracil, docetaxel, cisplatin, and elesclomol were most evident. Furthermore, our findings suggested a plausible association between CDC45 and immune cell infiltration. Enrichment analysis revealed that CDC45 and its associated genes may play crucial roles in muscle biofunction, whereas GSEA demonstrated significant enrichment of gene sets pertaining to G protein-coupled receptor ligand binding and G alpha (i) signaling events. Conclusion: Our study elucidates that upregulation of CDC45 is intricately associated with immune cell infiltration and holds promising potential as a favorable prognostic marker and a novel diagnostic biomarker for GC.',\n",
       "  'Stomach_Adenocarcinoma_TCGA_F-Correlated_Gene',\n",
       "  'None here'),\n",
       " 'PMC8433181-41598_2021_97258_MOESM4_ESM.csv--GeneName': ('Novel interferon-sensitive genes unveiled by correlation-driven gene selection and systems biology',\n",
       "  \"Interferons (IFNs) are key cytokines involved in alerting the immune system to viral infection. After IFN stimulation, cellular transcriptional profile critically changes, leading to the expression of several IFN stimulated genes (ISGs) that exert a wide variety of antiviral activities. Despite many ISGs have been already identified, a comprehensive network of coding and non-coding genes with a central role in IFN-response still needs to be elucidated. We performed a global RNA-Seq transcriptome profile of the HCV permissive human hepatoma cell line Huh7.5 and its parental cell line Huh7, upon IFN treatment, to define a network of genes whose coordinated modulation plays a central role in IFN-response. Our study adds molecular actors, coding and non-coding genes, to the complex molecular network underlying IFN-response and shows how systems biology approaches, such as correlation networks, network's topology and gene ontology analyses can be leveraged to this aim.\",\n",
       "  'GeneName',\n",
       "  '(50 protein-coding and 4 non-coding) and 601 edges from the Tan module (Supplementary File **3**)  Supplementary Information 4'),\n",
       " 'PMC8433181-41598_2021_97258_MOESM4_ESM.csv--EnsembleGeneID': ('Novel interferon-sensitive genes unveiled by correlation-driven gene selection and systems biology',\n",
       "  \"Interferons (IFNs) are key cytokines involved in alerting the immune system to viral infection. After IFN stimulation, cellular transcriptional profile critically changes, leading to the expression of several IFN stimulated genes (ISGs) that exert a wide variety of antiviral activities. Despite many ISGs have been already identified, a comprehensive network of coding and non-coding genes with a central role in IFN-response still needs to be elucidated. We performed a global RNA-Seq transcriptome profile of the HCV permissive human hepatoma cell line Huh7.5 and its parental cell line Huh7, upon IFN treatment, to define a network of genes whose coordinated modulation plays a central role in IFN-response. Our study adds molecular actors, coding and non-coding genes, to the complex molecular network underlying IFN-response and shows how systems biology approaches, such as correlation networks, network's topology and gene ontology analyses can be leveraged to this aim.\",\n",
       "  'EnsembleGeneID',\n",
       "  '(50 protein-coding and 4 non-coding) and 601 edges from the Tan module (Supplementary File **3**)  Supplementary Information 4'),\n",
       " 'PMC9903462-12920_2023_1451_MOESM1_ESM.xlsx-genes_of_module_darkorange-CMKLR1': ('CXCR3 predicts the prognosis of endometrial adenocarcinoma',\n",
       "  \"Objectives: Currently, endometrial adenocarcinoma lacks an effective prognostic indicator. This study was to develop and validate a gene biomarker and a nomogram to predict the survival of endometrial adenocarcinoma, explore potential mechanisms and select sensitive drugs. Methods: 425 endometrial adenocarcinoma cases with RNA sequencing data from TCGA were used to identify the most immune-related module by WGCNA. As an external test set, 103 cases from GSE17025 were used. Immune-related genes were downloaded from Innate DB. The three sets of data were used to identify the prognostic genes. Based on 397 cases with complete clinical data from TCGA, randomly divided into the training set (n = 199) and test set (n = 198), we identified CXCR3 as the prognostic gene biomarker. Age, grade, FIGO stage, and risk were used to develop and validate a predictive nomogram. AUC, C-index, calibration curve and K-M estimate evaluated the model's predictive performance. KEGG enrichment analysis, immune functions, TMB, the effectiveness of immunotherapy, and drug sensitivity between the high-risk and low-risk groups. Results: CXCR3 was identified as a prognostic biomarker. We calculated the risk score and divided the cases into the high-risk and low-risk groups by the median value of the risk score. The OS of the high-risk group was better than the low-risk group. The risk was the prognostic indicator independent of age, grade, and FIGO stage. We constructed the nomogram including age, grade, FIGO stage, and risk to predict the prognosis of endometrial adenocarcinoma. The top five KEGG pathways enriched by the DEGs between the high- and low-risk groups were viral protein interaction with cytokine and cytokine receptors, cytokine-cytokine receptor interaction, chemokine signaling pathway, natural killer cell-mediated cytotoxicity, and cell adhesion molecules. We analyzed the difference in immune cells and found that CD8+ T cells, activated CD4+ T cells, T helper cells, monocytes, and M1 macrophages were infiltrated more in the low-risk group. However, M0 macrophages and activated dendritic cells were more in the high-risk group. The immune function including APC coinhibition, APC costimulation, CCR, checkpoint, cytolytic activity, HLA, inflammation-promoting, MHC-I, parainflammation, T cell coinhibition, T cell costimulation, type I-IFN-response, and type II-IFN-response were better in the low-risk group. TMB and TIDE scores were both better in the low-risk group. By 'the pRRophetic' package, we found 56 sensitive drugs for different risk groups. Conclusion: We identified CXCR3 as the prognostic biomarker. We also developed and validated a predictive nomogram model combining CXCR3, age, histological grade, and FIGO stage for endometrial adenocarcinoma, which could help explore the precise treatment.\",\n",
       "  'genes_of_module_darkorange-CMKLR1',\n",
       "  'from TCGA were used to identify the most immune-related module by WGCNA. (Additional file **1**)  Additional file 1: Table S1. Identification of overlapping genes and estimate scores'),\n",
       " 'PMC7136968-EMMM-12-e11466-s004.xlsx-GeneSets-Tumor_Down_-_Liver_Cancer_Stroma_-_GSE45001': ('A paracrine activin A-mDia2 axis promotes squamous carcinogenesis via fibroblast reprogramming',\n",
       "  'Cancer-associated fibroblasts (CAFs) are key regulators of tumorigenesis and promising targets for next-generation therapies. We discovered that cancer cell-derived activin A reprograms fibroblasts into pro-tumorigenic CAFs. Mechanistically, this occurs via Smad2-mediated transcriptional regulation of the formin mDia2, which directly promotes filopodia formation and cell migration. mDia2 also induces expression of CAF marker genes through prevention of p53 nuclear accumulation, resulting in the production of a pro-tumorigenic matrisome and secretome. The translational relevance of this finding is reflected by activin A overexpression in tumor cells and of mDia2 in the stroma of skin cancer and other malignancies and the correlation of high activin A/mDia2 levels with poor patient survival. Blockade of this signaling axis using inhibitors of activin, activin receptors, or mDia2 suppressed cancer cell malignancy and squamous carcinogenesis in 3D organotypic cultures, ex vivo, and in vivo, providing a rationale for pharmacological inhibition of activin A-mDia2 signaling in stratified cancer patients.',\n",
       "  'GeneSets-Tumor_Down_-_Liver_Cancer_Stroma_-_GSE45001',\n",
       "  'microarray data of CAFs or total stroma of various types of human cancer (see **Dataset EV2**)  Dataset EV2  Click here for additional data file'),\n",
       " 'PMC9153011-advancesADV2021005360-suppl1.xlsx-FANC-FDR_0_05_UP_n_710': ('Genome-wide whole-blood transcriptome profiling across inherited bone marrow failure subtypes',\n",
       "  'Gene expression profiling has long been used in understanding the contribution of genes and related pathways in disease pathogenesis and susceptibility. We have performed whole-blood transcriptomic profiling in a subset of patients with inherited bone marrow failure (IBMF) whose diseases are clinically and genetically characterized as Fanconi anemia (FA), Shwachman-Diamond syndrome (SDS), and dyskeratosis congenita (DC). We hypothesized that annotating whole-blood transcripts genome wide will aid in understanding the complexity of gene regulation across these IBMF subtypes. Initial analysis of these blood-derived transcriptomes revealed significant skewing toward upregulated genes in patients with FA when compared with controls. Patients with SDS or DC also showed similar skewing profiles in their transcriptional status revealing a common pattern across these different IBMF subtypes. Gene set enrichment analysis revealed shared pathways involved in protein translation and elongation (ribosome constituents), RNA metabolism (nonsense-mediated decay), and mitochondrial function (electron transport chain). We further identified a discovery set of 26 upregulated genes at stringent cutoff (false discovery rate < 0.05) that appeared as a unified signature across the IBMF subtypes. Subsequent transcriptomic analysis on genetically uncharacterized patients with BMF revealed a striking overlap of genes, including 22 from the discovery set, which indicates a unified transcriptional drive across the classic (FA, SDS, and DC) and uncharacterized BMF subtypes. This study has relevance in disease pathogenesis, for example, in explaining the features (including the BMF) common to all patients with IBMF and suggests harnessing this transcriptional signature for patient benefit.',\n",
       "  'FANC-FDR_0_05_UP_n_710',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC9780220-41598_2022_26217_MOESM2_ESM.xlsx-DEgenes_CTL_FGF2-Differentially_expressed_genes_when_comparing_FGF2_to_untrerated_CTL': ('Fibroblast growth factor-2 bound to specific dermal fibroblast-derived extracellular vesicles is protected from degradation',\n",
       "  'Fibroblast growth factor-2 (FGF2) has multiple roles in cutaneous wound healing but its natural low stability prevents the development of its use in skin repair therapies. Here we show that FGF2 binds the outer surface of dermal fibroblast (DF)-derived extracellular vesicles (EVs) and this association protects FGF2 from fast degradation. EVs isolated from DF cultured in the presence of FGF2 harbor FGF2 on their surface and FGF2 can bind purified EVs in absence of cells. Remarkably, FGF2 binding to EVs is restricted to a specific subpopulation of EVs, which do not express CD63 and CD81 markers. Treatment of DF with FGF2-EVs activated ERK and STAT signaling pathways and increased cell proliferation and migration. Local injection of FGF2-EVs improved wound healing in mice. We further demonstrated that binding to EVs protects FGF2 from both thermal and proteolytic degradation, thus maintaining FGF2 function. This suggests that EVs protect soluble factors from degradation and increase their stability and half-life. These results reveal a novel aspect of EV function and suggest EVs as a potential tool for delivering FGF2 in skin healing therapies.',\n",
       "  'DEgenes_CTL_FGF2-Differentially_expressed_genes_when_comparing_FGF2_to_untrerated_CTL',\n",
       "  'To investigate the effect of FGF**2**  Supplementary Information 2'),\n",
       " 'PMC6624841-NIHMS1518742-supplement-Supp_Tables_1-23.xlsx-ST10_B-genes-Supplementary_Table_10_β-cell_enriched_genes': ('Diabetes relief in mice by glucose-sensing insulin-secreting human α-cells',\n",
       "  'Cell-identity switches, in which terminally differentiated cells are converted into different cell types when stressed, represent a widespread regenerative strategy in animals, yet they are poorly documented in mammals. In mice, some glucagon-producing pancreatic α-cells and somatostatin-producing δ-cells become insulin-expressing cells after the ablation of insulin-secreting β-cells, thus promoting diabetes recovery. Whether human islets also display this plasticity, especially in diabetic conditions, remains unknown. Here we show that islet non-β-cells, namely α-cells and pancreatic polypeptide (PPY)-producing γ-cells, obtained from deceased non-diabetic or diabetic human donors, can be lineage-traced and reprogrammed by the transcription factors PDX1 and MAFA to produce and secrete insulin in response to glucose. When transplanted into diabetic mice, converted human α-cells reverse diabetes and continue to produce insulin even after six months. Notably, insulin-producing α-cells maintain expression of α-cell markers, as seen by deep transcriptomic and proteomic characterization. These observations provide conceptual evidence and a molecular framework for a mechanistic understanding of in situ cell plasticity as a treatment for diabetes and other degenerative diseases.',\n",
       "  'ST10_B-genes-Supplementary_Table_10_β-cell_enriched_genes',\n",
       "  'Data Fig. 1a-c, while δ-cell purification was unreliable with available antibodies (Extended Data Fig.1; **Supplementary Tables 1**)'),\n",
       " 'PMC11042362-media-7.xlsx-Olig2vNeuN_gene_expr_DESeq2-Unnamed_8': ('DNA methylation differences between the female and male X chromosomes in human brain',\n",
       "  'The mechanisms of X chromosome inactivation suggest fundamental epigenetic differences between the female and male X chromosomes. However, DNA methylation studies often exclude the X chromosomes. In addition, many previous studies relied on techniques that examine non-randomly selected subsets of positions such as array-based methods, rather than assessing the whole X chromosome. Consequently, our understanding of X chromosome DNA methylation lags behind that of autosomes. Here we addressed this gap of knowledge by studying X chromosome DNA methylation using 89 whole genome bisulfite sequencing (WGBS) maps from neurons and oligodendrocytes. Using this unbiased and comprehensive data, we show that DNA methylation of the female X chromosomes is globally reduced (hypomethylated) across the entire chromosome compared to the male X chromosomes and autosomes. On the other hand, the majority of X-linked promoters were more highly methylated (hypermethylated) in females compared to males, consistent with the role of DNA methylation in X chromosome inactivation and dosage compensation. Remarkably, hypermethylation of female X promoters was limited to a group of previously lowly methylated promoters. The other group of highly methylated promoters were both hyper- and hypo-methylated in females with no obvious association with gene expression. Therefore, X chromosome inactivation by DNA methylation was exclusive to a subset of promoters with distinctive epigenetic feature. Apart from this group of promoters, differentially methylated regions in the female and male X chromosomes were dominated by female hypomethylation. Our study furthers the understanding of X-chromosome dosage regulation by DNA methylation on the chromosomal level as well as on individual gene level.',\n",
       "  'Olig2vNeuN_gene_expr_DESeq2-Unnamed_8',\n",
       "  'None here'),\n",
       " 'PMC11042362-media-7.xlsx-Olig2vNeuN_gene_expr_DESeq2-Unnamed_9': ('DNA methylation differences between the female and male X chromosomes in human brain',\n",
       "  'The mechanisms of X chromosome inactivation suggest fundamental epigenetic differences between the female and male X chromosomes. However, DNA methylation studies often exclude the X chromosomes. In addition, many previous studies relied on techniques that examine non-randomly selected subsets of positions such as array-based methods, rather than assessing the whole X chromosome. Consequently, our understanding of X chromosome DNA methylation lags behind that of autosomes. Here we addressed this gap of knowledge by studying X chromosome DNA methylation using 89 whole genome bisulfite sequencing (WGBS) maps from neurons and oligodendrocytes. Using this unbiased and comprehensive data, we show that DNA methylation of the female X chromosomes is globally reduced (hypomethylated) across the entire chromosome compared to the male X chromosomes and autosomes. On the other hand, the majority of X-linked promoters were more highly methylated (hypermethylated) in females compared to males, consistent with the role of DNA methylation in X chromosome inactivation and dosage compensation. Remarkably, hypermethylation of female X promoters was limited to a group of previously lowly methylated promoters. The other group of highly methylated promoters were both hyper- and hypo-methylated in females with no obvious association with gene expression. Therefore, X chromosome inactivation by DNA methylation was exclusive to a subset of promoters with distinctive epigenetic feature. Apart from this group of promoters, differentially methylated regions in the female and male X chromosomes were dominated by female hypomethylation. Our study furthers the understanding of X-chromosome dosage regulation by DNA methylation on the chromosomal level as well as on individual gene level.',\n",
       "  'Olig2vNeuN_gene_expr_DESeq2-Unnamed_9',\n",
       "  'None here'),\n",
       " 'PMC8093261-41423_2020_574_MOESM9_ESM.xlsx-Table_1B-Unnamed_0': ('Single-cell profiling reveals the trajectories of natural killer cell differentiation in bone marrow and a stress signature induced by acute myeloid leukemia',\n",
       "  'Natural killer (NK) cells are innate cytotoxic lymphoid cells (ILCs) involved in the killing of infected and tumor cells. Among human and mouse NK cells from the spleen and blood, we previously identified by single-cell RNA sequencing (scRNAseq) two similar major subsets, NK1 and NK2. Using the same technology, we report here the identification, by single-cell RNA sequencing (scRNAseq), of three NK cell subpopulations in human bone marrow. Pseudotime analysis identified a subset of resident CD56 bright NK cells, NK0 cells, as the precursor of both circulating CD56 dim NK1-like NK cells and CD56 bright NK2-like NK cells in human bone marrow and spleen under physiological conditions. Transcriptomic profiles of bone marrow NK cells from patients with acute myeloid leukemia (AML) exhibited stress-induced repression of NK cell effector functions, highlighting the profound impact of this disease on NK cell heterogeneity. Bone marrow NK cells from AML patients exhibited reduced levels of CD160, but the CD160 high group had a significantly higher survival rate.',\n",
       "  'Table_1B-Unnamed_0',\n",
       "  'Table S1'),\n",
       " 'PMC10579699-JEM_20230707_TableS5.xlsx-Proliferating_T-Proliferating_T': ('CD8+ lymphocytes are critical for early control of tuberculosis in macaques',\n",
       "  'The functional role of CD8+ lymphocytes in tuberculosis remains poorly understood. We depleted innate and/or adaptive CD8+ lymphocytes in macaques and showed that loss of all CD8α+ cells (using anti-CD8α antibody) significantly impaired early control of Mycobacterium tuberculosis (Mtb) infection, leading to increased granulomas, lung inflammation, and bacterial burden. Analysis of barcoded Mtb from infected macaques demonstrated that depletion of all CD8+ lymphocytes allowed increased establishment of Mtb in lungs and dissemination within lungs and to lymph nodes, while depletion of only adaptive CD8+ T cells (with anti-CD8β antibody) worsened bacterial control in lymph nodes. Flow cytometry and single-cell RNA sequencing revealed polyfunctional cytotoxic CD8+ lymphocytes in control granulomas, while CD8-depleted animals were unexpectedly enriched in CD4 and γδ T cells adopting incomplete cytotoxic signatures. Ligand-receptor analyses identified IL-15 signaling in granulomas as a driver of cytotoxic T cells. These data support that CD8+ lymphocytes are required for early protection against Mtb and suggest polyfunctional cytotoxic responses as a vaccine target.',\n",
       "  'Proliferating_T-Proliferating_T',\n",
       "  'None here'),\n",
       " 'PMC8433181-41598_2021_97258_MOESM3_ESM.csv--ensemble_gene_id': ('Novel interferon-sensitive genes unveiled by correlation-driven gene selection and systems biology',\n",
       "  \"Interferons (IFNs) are key cytokines involved in alerting the immune system to viral infection. After IFN stimulation, cellular transcriptional profile critically changes, leading to the expression of several IFN stimulated genes (ISGs) that exert a wide variety of antiviral activities. Despite many ISGs have been already identified, a comprehensive network of coding and non-coding genes with a central role in IFN-response still needs to be elucidated. We performed a global RNA-Seq transcriptome profile of the HCV permissive human hepatoma cell line Huh7.5 and its parental cell line Huh7, upon IFN treatment, to define a network of genes whose coordinated modulation plays a central role in IFN-response. Our study adds molecular actors, coding and non-coding genes, to the complex molecular network underlying IFN-response and shows how systems biology approaches, such as correlation networks, network's topology and gene ontology analyses can be leveraged to this aim.\",\n",
       "  'ensemble_gene_id',\n",
       "  'baseline levels in Huh7 compared to Huh7.5 (basal higher expression in the latter) (Fig. **2**)  Supplementary Information 3'),\n",
       " 'PMC9630267-41467_2022_34264_MOESM4_ESM.xlsx-Supplementary_Data_2-Unnamed_11': ('Systematic analysis and prediction of genes associated with monogenic disorders on human chromosome X',\n",
       "  'Disease gene discovery on chromosome (chr) X is challenging owing to its unique modes of inheritance. We undertook a systematic analysis of human chrX genes. We observe a higher proportion of disorder-associated genes and an enrichment of genes involved in cognition, language, and seizures on chrX compared to autosomes. We analyze gene constraints, exon and promoter conservation, expression, and paralogues, and report 127 genes sharing one or more attributes with known chrX disorder genes. Using machine learning classifiers trained to distinguish disease-associated from dispensable genes, we classify 247 genes, including 115 of the 127, as having high probability of being disease-associated. We provide evidence of an excess of variants in predicted genes in existing databases. Finally, we report damaging variants in CDK16 and TRPC5 in patients with intellectual disability or autism spectrum disorders. This study predicts large-scale gene-disease associations that could be used for prioritization of X-linked pathogenic variants.',\n",
       "  'Supplementary_Data_2-Unnamed_11',\n",
       "  'with at least one monogenic disorder (referred to as ‘disorder genes’) in OMIM (Fig. **1**)  Supplementary Data 1-11'),\n",
       " 'PMC9630267-41467_2022_34264_MOESM4_ESM.xlsx-Supplementary_Data_2-Unnamed_9': ('Systematic analysis and prediction of genes associated with monogenic disorders on human chromosome X',\n",
       "  'Disease gene discovery on chromosome (chr) X is challenging owing to its unique modes of inheritance. We undertook a systematic analysis of human chrX genes. We observe a higher proportion of disorder-associated genes and an enrichment of genes involved in cognition, language, and seizures on chrX compared to autosomes. We analyze gene constraints, exon and promoter conservation, expression, and paralogues, and report 127 genes sharing one or more attributes with known chrX disorder genes. Using machine learning classifiers trained to distinguish disease-associated from dispensable genes, we classify 247 genes, including 115 of the 127, as having high probability of being disease-associated. We provide evidence of an excess of variants in predicted genes in existing databases. Finally, we report damaging variants in CDK16 and TRPC5 in patients with intellectual disability or autism spectrum disorders. This study predicts large-scale gene-disease associations that could be used for prioritization of X-linked pathogenic variants.',\n",
       "  'Supplementary_Data_2-Unnamed_9',\n",
       "  'with at least one monogenic disorder (referred to as ‘disorder genes’) in OMIM (Fig. **1**)  Supplementary Data 1-11'),\n",
       " 'PMC9630267-41467_2022_34264_MOESM6_ESM.xlsx-S2b-Supplementary_Figure_2b': ('Systematic analysis and prediction of genes associated with monogenic disorders on human chromosome X',\n",
       "  'Disease gene discovery on chromosome (chr) X is challenging owing to its unique modes of inheritance. We undertook a systematic analysis of human chrX genes. We observe a higher proportion of disorder-associated genes and an enrichment of genes involved in cognition, language, and seizures on chrX compared to autosomes. We analyze gene constraints, exon and promoter conservation, expression, and paralogues, and report 127 genes sharing one or more attributes with known chrX disorder genes. Using machine learning classifiers trained to distinguish disease-associated from dispensable genes, we classify 247 genes, including 115 of the 127, as having high probability of being disease-associated. We provide evidence of an excess of variants in predicted genes in existing databases. Finally, we report damaging variants in CDK16 and TRPC5 in patients with intellectual disability or autism spectrum disorders. This study predicts large-scale gene-disease associations that could be used for prioritization of X-linked pathogenic variants.',\n",
       "  'S2b-Supplementary_Figure_2b',\n",
       "  'None here'),\n",
       " 'PMC9630267-41467_2022_34264_MOESM6_ESM.xlsx-S7c-Supplementary_Figure_7c': ('Systematic analysis and prediction of genes associated with monogenic disorders on human chromosome X',\n",
       "  'Disease gene discovery on chromosome (chr) X is challenging owing to its unique modes of inheritance. We undertook a systematic analysis of human chrX genes. We observe a higher proportion of disorder-associated genes and an enrichment of genes involved in cognition, language, and seizures on chrX compared to autosomes. We analyze gene constraints, exon and promoter conservation, expression, and paralogues, and report 127 genes sharing one or more attributes with known chrX disorder genes. Using machine learning classifiers trained to distinguish disease-associated from dispensable genes, we classify 247 genes, including 115 of the 127, as having high probability of being disease-associated. We provide evidence of an excess of variants in predicted genes in existing databases. Finally, we report damaging variants in CDK16 and TRPC5 in patients with intellectual disability or autism spectrum disorders. This study predicts large-scale gene-disease associations that could be used for prioritization of X-linked pathogenic variants.',\n",
       "  'S7c-Supplementary_Figure_7c',\n",
       "  'None here'),\n",
       " 'PMC9630267-41467_2022_34264_MOESM6_ESM.xlsx-3e-Figure_3e': ('Systematic analysis and prediction of genes associated with monogenic disorders on human chromosome X',\n",
       "  'Disease gene discovery on chromosome (chr) X is challenging owing to its unique modes of inheritance. We undertook a systematic analysis of human chrX genes. We observe a higher proportion of disorder-associated genes and an enrichment of genes involved in cognition, language, and seizures on chrX compared to autosomes. We analyze gene constraints, exon and promoter conservation, expression, and paralogues, and report 127 genes sharing one or more attributes with known chrX disorder genes. Using machine learning classifiers trained to distinguish disease-associated from dispensable genes, we classify 247 genes, including 115 of the 127, as having high probability of being disease-associated. We provide evidence of an excess of variants in predicted genes in existing databases. Finally, we report damaging variants in CDK16 and TRPC5 in patients with intellectual disability or autism spectrum disorders. This study predicts large-scale gene-disease associations that could be used for prioritization of X-linked pathogenic variants.',\n",
       "  '3e-Figure_3e',\n",
       "  'None here'),\n",
       " 'PMC9630267-41467_2022_34264_MOESM6_ESM.xlsx-4d-Figure_4d': ('Systematic analysis and prediction of genes associated with monogenic disorders on human chromosome X',\n",
       "  'Disease gene discovery on chromosome (chr) X is challenging owing to its unique modes of inheritance. We undertook a systematic analysis of human chrX genes. We observe a higher proportion of disorder-associated genes and an enrichment of genes involved in cognition, language, and seizures on chrX compared to autosomes. We analyze gene constraints, exon and promoter conservation, expression, and paralogues, and report 127 genes sharing one or more attributes with known chrX disorder genes. Using machine learning classifiers trained to distinguish disease-associated from dispensable genes, we classify 247 genes, including 115 of the 127, as having high probability of being disease-associated. We provide evidence of an excess of variants in predicted genes in existing databases. Finally, we report damaging variants in CDK16 and TRPC5 in patients with intellectual disability or autism spectrum disorders. This study predicts large-scale gene-disease associations that could be used for prioritization of X-linked pathogenic variants.',\n",
       "  '4d-Figure_4d',\n",
       "  'None here'),\n",
       " 'PMC9630267-41467_2022_34264_MOESM4_ESM.xlsx-Supplementary_Data_2-Unnamed_1': ('Systematic analysis and prediction of genes associated with monogenic disorders on human chromosome X',\n",
       "  'Disease gene discovery on chromosome (chr) X is challenging owing to its unique modes of inheritance. We undertook a systematic analysis of human chrX genes. We observe a higher proportion of disorder-associated genes and an enrichment of genes involved in cognition, language, and seizures on chrX compared to autosomes. We analyze gene constraints, exon and promoter conservation, expression, and paralogues, and report 127 genes sharing one or more attributes with known chrX disorder genes. Using machine learning classifiers trained to distinguish disease-associated from dispensable genes, we classify 247 genes, including 115 of the 127, as having high probability of being disease-associated. We provide evidence of an excess of variants in predicted genes in existing databases. Finally, we report damaging variants in CDK16 and TRPC5 in patients with intellectual disability or autism spectrum disorders. This study predicts large-scale gene-disease associations that could be used for prioritization of X-linked pathogenic variants.',\n",
       "  'Supplementary_Data_2-Unnamed_1',\n",
       "  'with at least one monogenic disorder (referred to as ‘disorder genes’) in OMIM (Fig. **1**)  Supplementary Data 1-11'),\n",
       " 'PMC9630267-41467_2022_34264_MOESM6_ESM.xlsx-3b-d-Figure_3b-d': ('Systematic analysis and prediction of genes associated with monogenic disorders on human chromosome X',\n",
       "  'Disease gene discovery on chromosome (chr) X is challenging owing to its unique modes of inheritance. We undertook a systematic analysis of human chrX genes. We observe a higher proportion of disorder-associated genes and an enrichment of genes involved in cognition, language, and seizures on chrX compared to autosomes. We analyze gene constraints, exon and promoter conservation, expression, and paralogues, and report 127 genes sharing one or more attributes with known chrX disorder genes. Using machine learning classifiers trained to distinguish disease-associated from dispensable genes, we classify 247 genes, including 115 of the 127, as having high probability of being disease-associated. We provide evidence of an excess of variants in predicted genes in existing databases. Finally, we report damaging variants in CDK16 and TRPC5 in patients with intellectual disability or autism spectrum disorders. This study predicts large-scale gene-disease associations that could be used for prioritization of X-linked pathogenic variants.',\n",
       "  '3b-d-Figure_3b-d',\n",
       "  'None here'),\n",
       " 'PMC9630267-41467_2022_34264_MOESM4_ESM.xlsx-Supplementary_Data_2-Supplementary_Data_2_Annotations_of_protein-coding_genes_on_chrX': ('Systematic analysis and prediction of genes associated with monogenic disorders on human chromosome X',\n",
       "  'Disease gene discovery on chromosome (chr) X is challenging owing to its unique modes of inheritance. We undertook a systematic analysis of human chrX genes. We observe a higher proportion of disorder-associated genes and an enrichment of genes involved in cognition, language, and seizures on chrX compared to autosomes. We analyze gene constraints, exon and promoter conservation, expression, and paralogues, and report 127 genes sharing one or more attributes with known chrX disorder genes. Using machine learning classifiers trained to distinguish disease-associated from dispensable genes, we classify 247 genes, including 115 of the 127, as having high probability of being disease-associated. We provide evidence of an excess of variants in predicted genes in existing databases. Finally, we report damaging variants in CDK16 and TRPC5 in patients with intellectual disability or autism spectrum disorders. This study predicts large-scale gene-disease associations that could be used for prioritization of X-linked pathogenic variants.',\n",
       "  'Supplementary_Data_2-Supplementary_Data_2_Annotations_of_protein-coding_genes_on_chrX',\n",
       "  'with at least one monogenic disorder (referred to as ‘disorder genes’) in OMIM (Fig. **1**)  Supplementary Data 1-11'),\n",
       " 'PMC6958619-13293_2019_278_MOESM1_ESM.xlsx-Xgenes-DEF_data-Gene_name': ('SeXY chromosomes and the immune system: reflections after a comparative study',\n",
       "  'Background: Sex bias in immune function has been contributed in part to a preponderance of immune system-related genes (ISRG) on the X-chromosome. We verified whether ISRG are more abundant on the X chromosome as compared to autosomal chromosomes and reflected on the impact of our findings. Methods: Consulting freely accessible databases, we performed a comparative study consisting of three complementary strategies. First, among coding X/Y-linked genes, the abundance of ISRG was compared to the abundance of genes dedicated to other systems. Genes were assigned considering three criteria: disease, tissue expression, and function (DEF approach). In addition, we carried out two genome-wide approaches to compare the contribution of sex and autosomal chromosomes to immune genes defined by an elevated expression in lymphatic tissues (LTEEG approach) or annotation to an immune system process, GO:0002376 (GO approach). Results: The X chromosome had less immune genes than the median of the autosomal chromosomes. Among X-linked genes, ISRG ranked fourth after the reproductive and nervous systems and genes dedicated to development, proliferation and apoptosis. On the Y chromosome, ISRG ranked second, and at the pseudoautosomal region (PAR) first. According to studies on the expression of X-linked genes in a variety of (mostly non-lymphatic) tissues, almost two-thirds of ISRG are expressed without sex bias, and the remaining ISRG presented female and male bias with similar frequency. Various epigenetic controllers, X-linked MSL3 and Y-linked KDM5D and UTY, were preferentially expressed in leukocytes and deserve further attention for a possible role in sex biased expression or its neutralisation. Conclusions: The X chromosome is not enriched for ISRG, though particular X-linked genes may be responsible for sex differences in certain immune responses. So far, there is insufficient information on sex-biased expression of X/Y-linked ISRG in leukocytes to draw general conclusions on the impact of X/Y-linked ISRG in immune function. More research on the regulation of the expression X-linked genes is required with attention to 1) female and male mechanisms that may either augment or diminish sex biased expression and 2) tissue-specific expression studies.',\n",
       "  'Xgenes-DEF_data-Gene_name',\n",
       "  'and pseudoautosomal regions (PAR) were downloaded from the HUGO Gene Nomenclature Committee (HGNC) database [**1**]  Additional file 1. XY genes, DEF data and manual system annotation. This Excel file contains 4 sheets: 1 for codes and numerical analysis, and 3 sheets for the DEF data collection on X-, Y-, and PAR-linked genes. It formed the basis for Table 1'),\n",
       " 'PMC11042362-media-5.xlsx-FvM_gene_expr_DESeq2-Unnamed_8': ('DNA methylation differences between the female and male X chromosomes in human brain',\n",
       "  'The mechanisms of X chromosome inactivation suggest fundamental epigenetic differences between the female and male X chromosomes. However, DNA methylation studies often exclude the X chromosomes. In addition, many previous studies relied on techniques that examine non-randomly selected subsets of positions such as array-based methods, rather than assessing the whole X chromosome. Consequently, our understanding of X chromosome DNA methylation lags behind that of autosomes. Here we addressed this gap of knowledge by studying X chromosome DNA methylation using 89 whole genome bisulfite sequencing (WGBS) maps from neurons and oligodendrocytes. Using this unbiased and comprehensive data, we show that DNA methylation of the female X chromosomes is globally reduced (hypomethylated) across the entire chromosome compared to the male X chromosomes and autosomes. On the other hand, the majority of X-linked promoters were more highly methylated (hypermethylated) in females compared to males, consistent with the role of DNA methylation in X chromosome inactivation and dosage compensation. Remarkably, hypermethylation of female X promoters was limited to a group of previously lowly methylated promoters. The other group of highly methylated promoters were both hyper- and hypo-methylated in females with no obvious association with gene expression. Therefore, X chromosome inactivation by DNA methylation was exclusive to a subset of promoters with distinctive epigenetic feature. Apart from this group of promoters, differentially methylated regions in the female and male X chromosomes were dominated by female hypomethylation. Our study furthers the understanding of X-chromosome dosage regulation by DNA methylation on the chromosomal level as well as on individual gene level.',\n",
       "  'FvM_gene_expr_DESeq2-Unnamed_8',\n",
       "  'None here'),\n",
       " 'PMC11042362-media-4.xlsx-FvM_genebody_methylation-Unnamed_8': ('DNA methylation differences between the female and male X chromosomes in human brain',\n",
       "  'The mechanisms of X chromosome inactivation suggest fundamental epigenetic differences between the female and male X chromosomes. However, DNA methylation studies often exclude the X chromosomes. In addition, many previous studies relied on techniques that examine non-randomly selected subsets of positions such as array-based methods, rather than assessing the whole X chromosome. Consequently, our understanding of X chromosome DNA methylation lags behind that of autosomes. Here we addressed this gap of knowledge by studying X chromosome DNA methylation using 89 whole genome bisulfite sequencing (WGBS) maps from neurons and oligodendrocytes. Using this unbiased and comprehensive data, we show that DNA methylation of the female X chromosomes is globally reduced (hypomethylated) across the entire chromosome compared to the male X chromosomes and autosomes. On the other hand, the majority of X-linked promoters were more highly methylated (hypermethylated) in females compared to males, consistent with the role of DNA methylation in X chromosome inactivation and dosage compensation. Remarkably, hypermethylation of female X promoters was limited to a group of previously lowly methylated promoters. The other group of highly methylated promoters were both hyper- and hypo-methylated in females with no obvious association with gene expression. Therefore, X chromosome inactivation by DNA methylation was exclusive to a subset of promoters with distinctive epigenetic feature. Apart from this group of promoters, differentially methylated regions in the female and male X chromosomes were dominated by female hypomethylation. Our study furthers the understanding of X-chromosome dosage regulation by DNA methylation on the chromosomal level as well as on individual gene level.',\n",
       "  'FvM_genebody_methylation-Unnamed_8',\n",
       "  'None here'),\n",
       " 'PMC11042362-media-3.xlsx-FvM_promoter_methylation-Unnamed_8': ('DNA methylation differences between the female and male X chromosomes in human brain',\n",
       "  'The mechanisms of X chromosome inactivation suggest fundamental epigenetic differences between the female and male X chromosomes. However, DNA methylation studies often exclude the X chromosomes. In addition, many previous studies relied on techniques that examine non-randomly selected subsets of positions such as array-based methods, rather than assessing the whole X chromosome. Consequently, our understanding of X chromosome DNA methylation lags behind that of autosomes. Here we addressed this gap of knowledge by studying X chromosome DNA methylation using 89 whole genome bisulfite sequencing (WGBS) maps from neurons and oligodendrocytes. Using this unbiased and comprehensive data, we show that DNA methylation of the female X chromosomes is globally reduced (hypomethylated) across the entire chromosome compared to the male X chromosomes and autosomes. On the other hand, the majority of X-linked promoters were more highly methylated (hypermethylated) in females compared to males, consistent with the role of DNA methylation in X chromosome inactivation and dosage compensation. Remarkably, hypermethylation of female X promoters was limited to a group of previously lowly methylated promoters. The other group of highly methylated promoters were both hyper- and hypo-methylated in females with no obvious association with gene expression. Therefore, X chromosome inactivation by DNA methylation was exclusive to a subset of promoters with distinctive epigenetic feature. Apart from this group of promoters, differentially methylated regions in the female and male X chromosomes were dominated by female hypomethylation. Our study furthers the understanding of X-chromosome dosage regulation by DNA methylation on the chromosomal level as well as on individual gene level.',\n",
       "  'FvM_promoter_methylation-Unnamed_8',\n",
       "  'None here'),\n",
       " 'PMC11042362-media-5.xlsx-FvM_gene_expr_DESeq2-Unnamed_9': ('DNA methylation differences between the female and male X chromosomes in human brain',\n",
       "  'The mechanisms of X chromosome inactivation suggest fundamental epigenetic differences between the female and male X chromosomes. However, DNA methylation studies often exclude the X chromosomes. In addition, many previous studies relied on techniques that examine non-randomly selected subsets of positions such as array-based methods, rather than assessing the whole X chromosome. Consequently, our understanding of X chromosome DNA methylation lags behind that of autosomes. Here we addressed this gap of knowledge by studying X chromosome DNA methylation using 89 whole genome bisulfite sequencing (WGBS) maps from neurons and oligodendrocytes. Using this unbiased and comprehensive data, we show that DNA methylation of the female X chromosomes is globally reduced (hypomethylated) across the entire chromosome compared to the male X chromosomes and autosomes. On the other hand, the majority of X-linked promoters were more highly methylated (hypermethylated) in females compared to males, consistent with the role of DNA methylation in X chromosome inactivation and dosage compensation. Remarkably, hypermethylation of female X promoters was limited to a group of previously lowly methylated promoters. The other group of highly methylated promoters were both hyper- and hypo-methylated in females with no obvious association with gene expression. Therefore, X chromosome inactivation by DNA methylation was exclusive to a subset of promoters with distinctive epigenetic feature. Apart from this group of promoters, differentially methylated regions in the female and male X chromosomes were dominated by female hypomethylation. Our study furthers the understanding of X-chromosome dosage regulation by DNA methylation on the chromosomal level as well as on individual gene level.',\n",
       "  'FvM_gene_expr_DESeq2-Unnamed_9',\n",
       "  'None here'),\n",
       " 'PMC11042362-media-4.xlsx-FvM_genebody_methylation-Unnamed_9': ('DNA methylation differences between the female and male X chromosomes in human brain',\n",
       "  'The mechanisms of X chromosome inactivation suggest fundamental epigenetic differences between the female and male X chromosomes. However, DNA methylation studies often exclude the X chromosomes. In addition, many previous studies relied on techniques that examine non-randomly selected subsets of positions such as array-based methods, rather than assessing the whole X chromosome. Consequently, our understanding of X chromosome DNA methylation lags behind that of autosomes. Here we addressed this gap of knowledge by studying X chromosome DNA methylation using 89 whole genome bisulfite sequencing (WGBS) maps from neurons and oligodendrocytes. Using this unbiased and comprehensive data, we show that DNA methylation of the female X chromosomes is globally reduced (hypomethylated) across the entire chromosome compared to the male X chromosomes and autosomes. On the other hand, the majority of X-linked promoters were more highly methylated (hypermethylated) in females compared to males, consistent with the role of DNA methylation in X chromosome inactivation and dosage compensation. Remarkably, hypermethylation of female X promoters was limited to a group of previously lowly methylated promoters. The other group of highly methylated promoters were both hyper- and hypo-methylated in females with no obvious association with gene expression. Therefore, X chromosome inactivation by DNA methylation was exclusive to a subset of promoters with distinctive epigenetic feature. Apart from this group of promoters, differentially methylated regions in the female and male X chromosomes were dominated by female hypomethylation. Our study furthers the understanding of X-chromosome dosage regulation by DNA methylation on the chromosomal level as well as on individual gene level.',\n",
       "  'FvM_genebody_methylation-Unnamed_9',\n",
       "  'None here'),\n",
       " 'PMC11042362-media-3.xlsx-FvM_promoter_methylation-Unnamed_9': ('DNA methylation differences between the female and male X chromosomes in human brain',\n",
       "  'The mechanisms of X chromosome inactivation suggest fundamental epigenetic differences between the female and male X chromosomes. However, DNA methylation studies often exclude the X chromosomes. In addition, many previous studies relied on techniques that examine non-randomly selected subsets of positions such as array-based methods, rather than assessing the whole X chromosome. Consequently, our understanding of X chromosome DNA methylation lags behind that of autosomes. Here we addressed this gap of knowledge by studying X chromosome DNA methylation using 89 whole genome bisulfite sequencing (WGBS) maps from neurons and oligodendrocytes. Using this unbiased and comprehensive data, we show that DNA methylation of the female X chromosomes is globally reduced (hypomethylated) across the entire chromosome compared to the male X chromosomes and autosomes. On the other hand, the majority of X-linked promoters were more highly methylated (hypermethylated) in females compared to males, consistent with the role of DNA methylation in X chromosome inactivation and dosage compensation. Remarkably, hypermethylation of female X promoters was limited to a group of previously lowly methylated promoters. The other group of highly methylated promoters were both hyper- and hypo-methylated in females with no obvious association with gene expression. Therefore, X chromosome inactivation by DNA methylation was exclusive to a subset of promoters with distinctive epigenetic feature. Apart from this group of promoters, differentially methylated regions in the female and male X chromosomes were dominated by female hypomethylation. Our study furthers the understanding of X-chromosome dosage regulation by DNA methylation on the chromosomal level as well as on individual gene level.',\n",
       "  'FvM_promoter_methylation-Unnamed_9',\n",
       "  'None here'),\n",
       " 'PMC9054707-mmc3.xlsx-Sheet1-Unnamed_59': ('Mapping molecular gene signatures mediated by SARS-COV-2 and large-scale and genome-wide transcriptomics comparative analysis among respiratory viruses of medical importance',\n",
       "  'Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) is an emerging RNA virus causing COVID-19 disease, across the globe. SARS-CoV-2 infected patients may exhibit acute respiratory distress syndrome which can be compounded by endemic respiratory viruses and thus highlighting the need to understand the genetic bases of clinical outcome under multiple respiratory infections. In this study, 42 individual datasets and a multi-parametric based selected list of over 12,000 genes against five medically important respiratory viruses (SARS-CoV-2, SARS-CoV-1, influenza A, respiratory syncytial virus (RSV) and rhinovirus were collected and analysed in an attempt to understand differentially regulated gene patterns and to cast genetic markers of individual and multiple co-infections. While a certain cohort of virus-specific genes were regulated (negatively and positively), notably results revealed a greatest correlation among genes regulation by SARS-CoV-2 and RSV. Furthermore, out of analysed genes, the MAP2K5 and NFKBIL1 were specifically and highly upregulated in SARS-CoV-2 infection both in vivo or in vitro. The most conserved genetic signature was JAK2 gene as well as the constitutively downregulated ZNF219 gene. In contrast, several genes including GPBAR1 and SC5DL were specifically downregulated in SARS-CoV-2 datasets. Finally, we catalogued a set of genes that were conserved or differentially regulated across studied respiratory viruses. These finding provide foundational and genome-wide data to gauge the markers of respiratory viral infections individually and under co-infection. This work compares the virogenomic signatures among human respiratory viruses and provides valid targets for potential antiviral therapy.',\n",
       "  'Sheet1-Unnamed_59',\n",
       "  'Multimedia component 3'),\n",
       " 'PMC10274745-media-5.xlsx-S4A-Unnamed_1': ('The human Y and inactive X chromosomes similarly modulate autosomal gene expression',\n",
       "  'Somatic cells of human males and females have 45 chromosomes in common, including the \"active\" X chromosome. In males the 46 th chromosome is a Y; in females it is an \"inactive\" X (Xi). Through linear modeling of autosomal gene expression in cells from individuals with zero to three Xi and zero to four Y chromosomes, we found that Xi and Y impact autosomal expression broadly and with remarkably similar effects. Studying sex-chromosome structural anomalies, promoters of Xi- and Y-responsive genes, and CRISPR inhibition, we traced part of this shared effect to homologous transcription factors - ZFX and ZFY - encoded by Chr X and Y. This demonstrates sex-shared mechanisms by which Xi and Y modulate autosomal expression. Combined with earlier analyses of sex-linked gene expression, our studies show that 21% of all genes expressed in lymphoblastoid cells or fibroblasts change expression significantly in response to Xi or Y chromosomes.',\n",
       "  'S4A-Unnamed_1',\n",
       "  'to identify strictly “Xa-expressed” genes that are consistently “silenced” on Xi across studies (Methods; **Table S4**)'),\n",
       " 'PMC8244145-13072_2021_404_MOESM4_ESM.xlsx-DNAme-gene': ('Contribution of genetic and epigenetic changes to escape from X-chromosome inactivation',\n",
       "  'Background: X-chromosome inactivation (XCI) is the epigenetic inactivation of one of two X chromosomes in XX eutherian mammals. The inactive X chromosome is the result of multiple silencing pathways that act in concert to deposit chromatin changes, including DNA methylation and histone modifications. Yet over 15% of genes escape or variably escape from inactivation and continue to be expressed from the otherwise inactive X chromosome. To the extent that they have been studied, epigenetic marks correlate with this expression. Results: Using publicly available data, we compared XCI status calls with DNA methylation, H3K4me1, H3K4me3, H3K9me3, H3K27ac, H3K27me3 and H3K36me3. At genes subject to XCI we found heterochromatic marks enriched, and euchromatic marks depleted on the inactive X when compared to the active X. Genes escaping XCI were more similar between the active and inactive X. Using sample-specific XCI status calls, we found some marks differed significantly with variable XCI status, but which marks were significant was not consistent between genes. A model trained to predict XCI status from these epigenetic marks obtained over 75% accuracy for genes escaping and over 90% for genes subject to XCI. This model made novel XCI status calls for genes without allelic differences or CpG islands required for other methods. Examining these calls across a domain of variably escaping genes, we saw XCI status vary across individual genes rather than at the domain level. Lastly, we compared XCI status calls to genetic polymorphisms, finding multiple loci associated with XCI status changes at variably escaping genes, but none individually sufficient to induce an XCI status change. Conclusion: The control of expression from the inactive X chromosome is multifaceted, but ultimately regulated at the individual gene level with detectable but limited impact of distant polymorphisms. On the inactive X, at silenced genes euchromatic marks are depleted while heterochromatic marks are enriched. Genes escaping inactivation show a less significant enrichment of heterochromatic marks and depletion of H3K27ac. Combining all examined marks improved XCI status prediction, particularly for genes without CpG islands or polymorphisms, as no single feature is a consistent feature of silenced or expressed genes.',\n",
       "  'DNAme-gene',\n",
       "  'two samples with each XCI status to be called variable (Fig. 2a, Additional file **4**)  Additional file4: Table S4. All XCI status calls made here. See additional files. The first sheet contains a single XCI status call per gene per method. Published calls are from Balaton, et al. 2015. Other sheets contain all calls per sample for each method. Each row is one entry into the model, so Xi/Xa is per gene and the others are for unique transcripts. For DNAme, the samples on the far right in shades of grey are males while the samples on the left in color are females. For the epigenetic predictor, separate low confidence categories were made for when transcripts have only 12-14 of the 20 models per sample predicted a certain XCI status. Start and stop locations are from hg38'),\n",
       " 'PMC8244145-13072_2021_404_MOESM4_ESM.xlsx-allMethods-gene': ('Contribution of genetic and epigenetic changes to escape from X-chromosome inactivation',\n",
       "  'Background: X-chromosome inactivation (XCI) is the epigenetic inactivation of one of two X chromosomes in XX eutherian mammals. The inactive X chromosome is the result of multiple silencing pathways that act in concert to deposit chromatin changes, including DNA methylation and histone modifications. Yet over 15% of genes escape or variably escape from inactivation and continue to be expressed from the otherwise inactive X chromosome. To the extent that they have been studied, epigenetic marks correlate with this expression. Results: Using publicly available data, we compared XCI status calls with DNA methylation, H3K4me1, H3K4me3, H3K9me3, H3K27ac, H3K27me3 and H3K36me3. At genes subject to XCI we found heterochromatic marks enriched, and euchromatic marks depleted on the inactive X when compared to the active X. Genes escaping XCI were more similar between the active and inactive X. Using sample-specific XCI status calls, we found some marks differed significantly with variable XCI status, but which marks were significant was not consistent between genes. A model trained to predict XCI status from these epigenetic marks obtained over 75% accuracy for genes escaping and over 90% for genes subject to XCI. This model made novel XCI status calls for genes without allelic differences or CpG islands required for other methods. Examining these calls across a domain of variably escaping genes, we saw XCI status vary across individual genes rather than at the domain level. Lastly, we compared XCI status calls to genetic polymorphisms, finding multiple loci associated with XCI status changes at variably escaping genes, but none individually sufficient to induce an XCI status change. Conclusion: The control of expression from the inactive X chromosome is multifaceted, but ultimately regulated at the individual gene level with detectable but limited impact of distant polymorphisms. On the inactive X, at silenced genes euchromatic marks are depleted while heterochromatic marks are enriched. Genes escaping inactivation show a less significant enrichment of heterochromatic marks and depletion of H3K27ac. Combining all examined marks improved XCI status prediction, particularly for genes without CpG islands or polymorphisms, as no single feature is a consistent feature of silenced or expressed genes.',\n",
       "  'allMethods-gene',\n",
       "  'two samples with each XCI status to be called variable (Fig. 2a, Additional file **4**)  Additional file4: Table S4. All XCI status calls made here. See additional files. The first sheet contains a single XCI status call per gene per method. Published calls are from Balaton, et al. 2015. Other sheets contain all calls per sample for each method. Each row is one entry into the model, so Xi/Xa is per gene and the others are for unique transcripts. For DNAme, the samples on the far right in shades of grey are males while the samples on the left in color are females. For the epigenetic predictor, separate low confidence categories were made for when transcripts have only 12-14 of the 20 models per sample predicted a certain XCI status. Start and stop locations are from hg38'),\n",
       " 'PMC8244145-13072_2021_404_MOESM4_ESM.xlsx-epigeneticPredictorCREST-gene': ('Contribution of genetic and epigenetic changes to escape from X-chromosome inactivation',\n",
       "  'Background: X-chromosome inactivation (XCI) is the epigenetic inactivation of one of two X chromosomes in XX eutherian mammals. The inactive X chromosome is the result of multiple silencing pathways that act in concert to deposit chromatin changes, including DNA methylation and histone modifications. Yet over 15% of genes escape or variably escape from inactivation and continue to be expressed from the otherwise inactive X chromosome. To the extent that they have been studied, epigenetic marks correlate with this expression. Results: Using publicly available data, we compared XCI status calls with DNA methylation, H3K4me1, H3K4me3, H3K9me3, H3K27ac, H3K27me3 and H3K36me3. At genes subject to XCI we found heterochromatic marks enriched, and euchromatic marks depleted on the inactive X when compared to the active X. Genes escaping XCI were more similar between the active and inactive X. Using sample-specific XCI status calls, we found some marks differed significantly with variable XCI status, but which marks were significant was not consistent between genes. A model trained to predict XCI status from these epigenetic marks obtained over 75% accuracy for genes escaping and over 90% for genes subject to XCI. This model made novel XCI status calls for genes without allelic differences or CpG islands required for other methods. Examining these calls across a domain of variably escaping genes, we saw XCI status vary across individual genes rather than at the domain level. Lastly, we compared XCI status calls to genetic polymorphisms, finding multiple loci associated with XCI status changes at variably escaping genes, but none individually sufficient to induce an XCI status change. Conclusion: The control of expression from the inactive X chromosome is multifaceted, but ultimately regulated at the individual gene level with detectable but limited impact of distant polymorphisms. On the inactive X, at silenced genes euchromatic marks are depleted while heterochromatic marks are enriched. Genes escaping inactivation show a less significant enrichment of heterochromatic marks and depletion of H3K27ac. Combining all examined marks improved XCI status prediction, particularly for genes without CpG islands or polymorphisms, as no single feature is a consistent feature of silenced or expressed genes.',\n",
       "  'epigeneticPredictorCREST-gene',\n",
       "  'two samples with each XCI status to be called variable (Fig. 2a, Additional file **4**)  Additional file4: Table S4. All XCI status calls made here. See additional files. The first sheet contains a single XCI status call per gene per method. Published calls are from Balaton, et al. 2015. Other sheets contain all calls per sample for each method. Each row is one entry into the model, so Xi/Xa is per gene and the others are for unique transcripts. For DNAme, the samples on the far right in shades of grey are males while the samples on the left in color are females. For the epigenetic predictor, separate low confidence categories were made for when transcripts have only 12-14 of the 20 models per sample predicted a certain XCI status. Start and stop locations are from hg38'),\n",
       " 'PMC8244145-13072_2021_404_MOESM4_ESM.xlsx-epigeneticPredictorCEMT-gene': ('Contribution of genetic and epigenetic changes to escape from X-chromosome inactivation',\n",
       "  'Background: X-chromosome inactivation (XCI) is the epigenetic inactivation of one of two X chromosomes in XX eutherian mammals. The inactive X chromosome is the result of multiple silencing pathways that act in concert to deposit chromatin changes, including DNA methylation and histone modifications. Yet over 15% of genes escape or variably escape from inactivation and continue to be expressed from the otherwise inactive X chromosome. To the extent that they have been studied, epigenetic marks correlate with this expression. Results: Using publicly available data, we compared XCI status calls with DNA methylation, H3K4me1, H3K4me3, H3K9me3, H3K27ac, H3K27me3 and H3K36me3. At genes subject to XCI we found heterochromatic marks enriched, and euchromatic marks depleted on the inactive X when compared to the active X. Genes escaping XCI were more similar between the active and inactive X. Using sample-specific XCI status calls, we found some marks differed significantly with variable XCI status, but which marks were significant was not consistent between genes. A model trained to predict XCI status from these epigenetic marks obtained over 75% accuracy for genes escaping and over 90% for genes subject to XCI. This model made novel XCI status calls for genes without allelic differences or CpG islands required for other methods. Examining these calls across a domain of variably escaping genes, we saw XCI status vary across individual genes rather than at the domain level. Lastly, we compared XCI status calls to genetic polymorphisms, finding multiple loci associated with XCI status changes at variably escaping genes, but none individually sufficient to induce an XCI status change. Conclusion: The control of expression from the inactive X chromosome is multifaceted, but ultimately regulated at the individual gene level with detectable but limited impact of distant polymorphisms. On the inactive X, at silenced genes euchromatic marks are depleted while heterochromatic marks are enriched. Genes escaping inactivation show a less significant enrichment of heterochromatic marks and depletion of H3K27ac. Combining all examined marks improved XCI status prediction, particularly for genes without CpG islands or polymorphisms, as no single feature is a consistent feature of silenced or expressed genes.',\n",
       "  'epigeneticPredictorCEMT-gene',\n",
       "  'two samples with each XCI status to be called variable (Fig. 2a, Additional file **4**)  Additional file4: Table S4. All XCI status calls made here. See additional files. The first sheet contains a single XCI status call per gene per method. Published calls are from Balaton, et al. 2015. Other sheets contain all calls per sample for each method. Each row is one entry into the model, so Xi/Xa is per gene and the others are for unique transcripts. For DNAme, the samples on the far right in shades of grey are males while the samples on the left in color are females. For the epigenetic predictor, separate low confidence categories were made for when transcripts have only 12-14 of the 20 models per sample predicted a certain XCI status. Start and stop locations are from hg38'),\n",
       " 'PMC8244145-13072_2021_404_MOESM4_ESM.xlsx-DNAme_variable_escape-gene': ('Contribution of genetic and epigenetic changes to escape from X-chromosome inactivation',\n",
       "  'Background: X-chromosome inactivation (XCI) is the epigenetic inactivation of one of two X chromosomes in XX eutherian mammals. The inactive X chromosome is the result of multiple silencing pathways that act in concert to deposit chromatin changes, including DNA methylation and histone modifications. Yet over 15% of genes escape or variably escape from inactivation and continue to be expressed from the otherwise inactive X chromosome. To the extent that they have been studied, epigenetic marks correlate with this expression. Results: Using publicly available data, we compared XCI status calls with DNA methylation, H3K4me1, H3K4me3, H3K9me3, H3K27ac, H3K27me3 and H3K36me3. At genes subject to XCI we found heterochromatic marks enriched, and euchromatic marks depleted on the inactive X when compared to the active X. Genes escaping XCI were more similar between the active and inactive X. Using sample-specific XCI status calls, we found some marks differed significantly with variable XCI status, but which marks were significant was not consistent between genes. A model trained to predict XCI status from these epigenetic marks obtained over 75% accuracy for genes escaping and over 90% for genes subject to XCI. This model made novel XCI status calls for genes without allelic differences or CpG islands required for other methods. Examining these calls across a domain of variably escaping genes, we saw XCI status vary across individual genes rather than at the domain level. Lastly, we compared XCI status calls to genetic polymorphisms, finding multiple loci associated with XCI status changes at variably escaping genes, but none individually sufficient to induce an XCI status change. Conclusion: The control of expression from the inactive X chromosome is multifaceted, but ultimately regulated at the individual gene level with detectable but limited impact of distant polymorphisms. On the inactive X, at silenced genes euchromatic marks are depleted while heterochromatic marks are enriched. Genes escaping inactivation show a less significant enrichment of heterochromatic marks and depletion of H3K27ac. Combining all examined marks improved XCI status prediction, particularly for genes without CpG islands or polymorphisms, as no single feature is a consistent feature of silenced or expressed genes.',\n",
       "  'DNAme_variable_escape-gene',\n",
       "  'two samples with each XCI status to be called variable (Fig. 2a, Additional file **4**)  Additional file4: Table S4. All XCI status calls made here. See additional files. The first sheet contains a single XCI status call per gene per method. Published calls are from Balaton, et al. 2015. Other sheets contain all calls per sample for each method. Each row is one entry into the model, so Xi/Xa is per gene and the others are for unique transcripts. For DNAme, the samples on the far right in shades of grey are males while the samples on the left in color are females. For the epigenetic predictor, separate low confidence categories were made for when transcripts have only 12-14 of the 20 models per sample predicted a certain XCI status. Start and stop locations are from hg38'),\n",
       " 'PMC5990591-table_2.xlsx-overlap-VM_VM_n_396-Gene_Symbol': ('Influenza A Virus Facilitates Its Infectivity by Activating p53 to Inhibit the Expression of Interferon-Induced Transmembrane Proteins',\n",
       "  'Human influenza virus (IAV) are among the most common pathogens to cause human respiratory infections. A better understanding on interplay between IAV and host factors may provide clues for disease prevention and control. While many viruses are known to downregulate p53 upon entering the cell to reduce the innate host antiviral response, IAV infection is unusual in that it activates p53. However, it has not been clear whether this process has proviral or antiviral effects. In this study, using human isogenic p53 wild-type and p53null A549 cells generated from the CRISPR/Cas9 technology, we observed that p53null cells exhibit significantly reduced viral propagation when infected with influenza A virus (strain A/Puerto Rico/8/1934 H1N1). Genome-wide microarray analysis revealed that p53 regulates the expression of a large set of interferon-inducible genes, among which the interferon-induced transmembrane family members IFITM1, IFITM2, and IFITM3 were most significantly downregulated by the expression of p53. Knockdown of interferon-induced transmembrane proteins (IFITMs) by short interfering RNAs enhanced influenza virus infectivity in p53null A549 cells, while overexpressed IFITMs in A549 cells blocked virus entry. Intriguingly, regulation of IFITMs by p53 is independent of its transcriptional activity, as the p53 short isoform Δ40p53 recapitulates IFITM regulation. Taken together, these data reveal that p53 activation by IAV is an essential step in maintaining its infectivity. This novel association between human p53 and the broad spectrum antiviral proteins, the IFITMs, demonstrates a previous mechanism employed by influenza virus to enhance its propagation via p53 inhibition of IFITMs.',\n",
       "  'overlap-VM_VM_n_396-Gene_Symbol',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC7509539-Table_1.XLSX-Upregulated_in_CD8conv-Gene_Symbol': ('MAIT Cells Display a Specific Response to Type 1 IFN Underlying the Adjuvant Effect of TLR7/8 Ligands',\n",
       "  'Mucosal-associated invariant T (MAIT) cells constitute a highly conserved subset of effector T cells with innate-like recognition of a wide array of bacteria and fungi in humans. Harnessing the potential of these cells could represent a major advance as a new immunotherapy approach to fight difficult-to-treat bacterial infections. However, despite recent advances in the design of potent agonistic ligands for MAIT cells, it has become increasingly evident that adjuvants are required to elicit potent antimicrobial effector functions by these cells, such as IFNγ production and cytotoxicity. Indeed, TCR triggering alone elicits mostly barrier repair functions in MAIT cells, whereas an inflammatory milieu is required to drive the antibacterial functions. Cytokines such as IL-7, IL-12 and IL-18, IL-15 or more recently type 1 IFN all display an apparently similar ability to synergize with TCR stimulation to induce IFNγ production and/or cytotoxic functions in vitro , but their mechanisms of action are not well established. Herein, we show that MAIT cells feature a build-in mechanism to respond to IFNα. We confirm that IFNα acts directly and specifically on MAIT cells and synergizes with TCR/CD3 triggering to induce maximum cytokine production and cytotoxic functions. We provide evidences suggesting that the preferential activation of the Stat4 pathway is involved in the high sensitivity of MAIT cells to IFNα stimulation. Finally, gene expression data confirm the specific responsiveness of MAIT cells to IFNα and pinpoints specific pathways that could be the target of this cytokine. Altogether, these data highlight the potential of IFNα-inducing adjuvants to maximize MAIT cells responsiveness to purified ligands in order to induce potent anti-infectious responses.',\n",
       "  'Upregulated_in_CD8conv-Gene_Symbol',\n",
       "  'found in common between both cell types, including multiple interferon-specific genes (ISG) as expected (**Supplementary Table S1**)  Click here for additional data file'),\n",
       " 'PMC6092349-41389_2018_72_MOESM12_ESM.xlsx-Sheet1-Supplementary_Table_S2_List_of_down-regulated_genes_in_Noc_16h_vs_Ctrl': ('Chromosomal instability-induced senescence potentiates cell non-autonomous tumourigenic effects',\n",
       "  'Chromosomal instability (CIN), a high rate of chromosome loss or gain, is often associated with poor prognosis and drug resistance in cancers. Aneuploid, including near-polyploid, cells contain an abnormal number of chromosomes and exhibit CIN. The post-mitotic cell fates following generation of different degrees of chromosome mis-segregation and aneuploidy are unclear. Here we used aneuploidy inducers, nocodazole and reversine, to create different levels of aneuploidy. A higher extent of aneuploid and near-polyploid cells in a given population led to senescence. This was in contrast to cells with relatively lower levels of abnormal ploidy that continued to proliferate. Our findings revealed that senescence was accompanied by DNA damage and robust p53 activation. These senescent cells acquired the senescence-associated secretory phenotype (SASP). Depletion of p53 reduced the number of senescent cells with concomitant increase in cells undergoing DNA replication. Characterisation of these SASP factors demonstrated that they conferred paracrine pro-tumourigenic effects such as invasion, migration and angiogenesis both in vitro and in vivo. Finally, a correlation between increased aneuploidy and senescence was observed at the invasive front in breast carcinomas. Our findings demonstrate functional non-equivalence of discernable aneuploidies on tumourigenesis and suggest a cell non-autonomous mechanism by which aneuploidy-induced senescent cells and SASP can affect the tumour microenvironment to promote tumour progression.',\n",
       "  'Sheet1-Supplementary_Table_S2_List_of_down-regulated_genes_in_Noc_16h_vs_Ctrl',\n",
       "  'Supplementary Table S2'),\n",
       " 'PMC6092349-41389_2018_72_MOESM12_ESM.xlsx-Sheet1-Unnamed_1': ('Chromosomal instability-induced senescence potentiates cell non-autonomous tumourigenic effects',\n",
       "  'Chromosomal instability (CIN), a high rate of chromosome loss or gain, is often associated with poor prognosis and drug resistance in cancers. Aneuploid, including near-polyploid, cells contain an abnormal number of chromosomes and exhibit CIN. The post-mitotic cell fates following generation of different degrees of chromosome mis-segregation and aneuploidy are unclear. Here we used aneuploidy inducers, nocodazole and reversine, to create different levels of aneuploidy. A higher extent of aneuploid and near-polyploid cells in a given population led to senescence. This was in contrast to cells with relatively lower levels of abnormal ploidy that continued to proliferate. Our findings revealed that senescence was accompanied by DNA damage and robust p53 activation. These senescent cells acquired the senescence-associated secretory phenotype (SASP). Depletion of p53 reduced the number of senescent cells with concomitant increase in cells undergoing DNA replication. Characterisation of these SASP factors demonstrated that they conferred paracrine pro-tumourigenic effects such as invasion, migration and angiogenesis both in vitro and in vivo. Finally, a correlation between increased aneuploidy and senescence was observed at the invasive front in breast carcinomas. Our findings demonstrate functional non-equivalence of discernable aneuploidies on tumourigenesis and suggest a cell non-autonomous mechanism by which aneuploidy-induced senescent cells and SASP can affect the tumour microenvironment to promote tumour progression.',\n",
       "  'Sheet1-Unnamed_1',\n",
       "  'Supplementary Table S2'),\n",
       " 'PMC10956518-peerj-12-17130-s005.xlsx-Stomach_Adenocarcinoma_TCGA_N-Correlated_Gene': ('Validation of CDC45 as a novel biomarker for diagnosis and prognosis of gastric cancer',\n",
       "  'Background: Cell division cycle protein 45 (CDC45) has been demonstrated to play vital roles in the progression of various malignancies. However, the clinical significance of CDC45 in gastric cancer (GC) remains unreported. Method: In this study, we employed the TCGA database and the TCGA & GTEx dataset to compare the mRNA expression levels of CDC45 between gastric cancer tissues and adjacent or normal tissues ( p < 0.05 was considered statistically significant), which was further validated in multiple datasets including GSE13911, GSE29272, GSE118916, GSE66229, as well as RT-qPCR. Furthermore, we harnessed the Human Protein Atlas (HPA) to evaluate the protein expression of CDC45, which was subsequently verified through immunohistochemistry (IHC). To ascertain the diagnostic utility of CDC45, receiver operating characteristic (ROC) curves and the area under the ROC curve (AUC) were calculated in TCGA database, and further validated it in TCGA & GTEx and GSE66229 datasets. The Kaplan-Meier method was used to reveal the prognostic importance of CDC45 in The Cancer Genome Atlas (TCGA) database and authenticated through the GSE66229, GSE84433, and GSE84437 datasets. Through cBioPortal, we identified co-expressed genes of CDC45, and pursued enrichment analysis. Additionally, we availed gene set enrichment analysis (GSEA) to annotate the biological functions of CDC45. Results: Differential expression analysis revealed that CDC45 was significantly upregulated at both the mRNA and protein levels in GC (all p < 0.05 ). Remarkably, CDC45 emerged as a promising prognostic indicator and a novel diagnostic biomarker for GC. In a comprehensive the drug susceptibility analysis, we found that patients with high expression of CDC45 had high sensitivity to various chemotherapeutic agents, among which 5-fluorouracil, docetaxel, cisplatin, and elesclomol were most evident. Furthermore, our findings suggested a plausible association between CDC45 and immune cell infiltration. Enrichment analysis revealed that CDC45 and its associated genes may play crucial roles in muscle biofunction, whereas GSEA demonstrated significant enrichment of gene sets pertaining to G protein-coupled receptor ligand binding and G alpha (i) signaling events. Conclusion: Our study elucidates that upregulation of CDC45 is intricately associated with immune cell infiltration and holds promising potential as a favorable prognostic marker and a novel diagnostic biomarker for GC.',\n",
       "  'Stomach_Adenocarcinoma_TCGA_N-Correlated_Gene',\n",
       "  'None here'),\n",
       " 'PMC9795333-mmc4.xlsx-SE1-res-5a-X-linked_genes': ('De-erosion of X chromosome dosage compensation by the editing of XIST regulatory regions restores the differentiation potential in hPSCs',\n",
       "  'Human pluripotent stem cells (hPSCs) regularly and irreversibly show the erosion of X chromosome inactivation (XCI) by long non-coding RNA (lncRNA) XIST silencing, causing challenges in various applications of female hPSCs. Here, we report reliable methods to reactivate XIST with monoallelic expression in female hPSCs. Surprisingly, we find that the editing of XIST regulatory regions by Cas9-mediated non-homologous end joining is sufficient for the reactivation of XIST by endogenous systems. Proliferated hPSCs with XIST reactivation show XCI from an eroded X chromosome, suggesting that hPSCs with normal dosage compensation might lead to a growth advantage. Furthermore, the use of targeting vectors, including the XIST regulatory region sequences and selection cassette, enables XIST reactivation in hPSCs with high efficiency. XIST -reactivated hPSCs can show the restoration of differentiation potential. Thus, our findings demonstrate that XIST re-expression is a beneficial method to maximize the use of female hPSCs in various applications, such as proper disease modeling.',\n",
       "  'SE1-res-5a-X-linked_genes',\n",
       "  'is listed in the key resources table. The related statistical data is provided in **Table S3**  Table S3. Statistical data in scRNA-seq, related to Figures 1, 3, S2, and S5'),\n",
       " 'PMC8906867-pbio.3001569.s011.xlsx-HEK_and_SupT1_up_XBP1s_ATF6-Unnamed_1': ('The endoplasmic reticulum proteostasis network profoundly shapes the protein sequence space accessible to HIV envelope',\n",
       "  'The sequence space accessible to evolving proteins can be enhanced by cellular chaperones that assist biophysically defective clients in navigating complex folding landscapes. It is also possible, at least in theory, for proteostasis mechanisms that promote strict quality control to greatly constrain accessible protein sequence space. Unfortunately, most efforts to understand how proteostasis mechanisms influence evolution rely on artificial inhibition or genetic knockdown of specific chaperones. The few experiments that perturb quality control pathways also generally modulate the levels of only individual quality control factors. Here, we use chemical genetic strategies to tune proteostasis networks via natural stress response pathways that regulate the levels of entire suites of chaperones and quality control mechanisms. Specifically, we upregulate the unfolded protein response (UPR) to test the hypothesis that the host endoplasmic reticulum (ER) proteostasis network shapes the sequence space accessible to human immunodeficiency virus-1 (HIV-1) envelope (Env) protein. Elucidating factors that enhance or constrain Env sequence space is critical because Env evolves extremely rapidly, yielding HIV strains with antibody- and drug-escape mutations. We find that UPR-mediated upregulation of ER proteostasis factors, particularly those controlled by the IRE1-XBP1s UPR arm, globally reduces Env mutational tolerance. Conserved, functionally important Env regions exhibit the largest decreases in mutational tolerance upon XBP1s induction. Our data indicate that this phenomenon likely reflects strict quality control endowed by XBP1s-mediated remodeling of the ER proteostasis environment. Intriguingly, and in contrast, specific regions of Env, including regions targeted by broadly neutralizing antibodies, display enhanced mutational tolerance when XBP1s is induced, hinting at a role for host proteostasis network hijacking in potentiating antibody escape. These observations reveal a key function for proteostasis networks in decreasing instead of expanding the sequence space accessible to client proteins, while also demonstrating that the host ER proteostasis network profoundly shapes the mutational tolerance of Env in ways that could have important consequences for HIV adaptation.',\n",
       "  'HEK_and_SupT1_up_XBP1s_ATF6-Unnamed_1',\n",
       "  'HEK293DAX cells (note that the +ATF6 condition was not tested in the previous study; **S11 Data**)  Transcriptome comparison of HEK293DAX cells and SupT1DAX cells.(XLSX)  Click here for additional data file'),\n",
       " 'PMC8906867-pbio.3001569.s011.xlsx-HEK_and_SupT1_up_XBP1s_ATF6-S7_Data_List_of_genes_upregulated_in_both_HEK293DAX_and_SupT1DAX_cells_for_XBP1s_ATF6_vs_Basal': ('The endoplasmic reticulum proteostasis network profoundly shapes the protein sequence space accessible to HIV envelope',\n",
       "  'The sequence space accessible to evolving proteins can be enhanced by cellular chaperones that assist biophysically defective clients in navigating complex folding landscapes. It is also possible, at least in theory, for proteostasis mechanisms that promote strict quality control to greatly constrain accessible protein sequence space. Unfortunately, most efforts to understand how proteostasis mechanisms influence evolution rely on artificial inhibition or genetic knockdown of specific chaperones. The few experiments that perturb quality control pathways also generally modulate the levels of only individual quality control factors. Here, we use chemical genetic strategies to tune proteostasis networks via natural stress response pathways that regulate the levels of entire suites of chaperones and quality control mechanisms. Specifically, we upregulate the unfolded protein response (UPR) to test the hypothesis that the host endoplasmic reticulum (ER) proteostasis network shapes the sequence space accessible to human immunodeficiency virus-1 (HIV-1) envelope (Env) protein. Elucidating factors that enhance or constrain Env sequence space is critical because Env evolves extremely rapidly, yielding HIV strains with antibody- and drug-escape mutations. We find that UPR-mediated upregulation of ER proteostasis factors, particularly those controlled by the IRE1-XBP1s UPR arm, globally reduces Env mutational tolerance. Conserved, functionally important Env regions exhibit the largest decreases in mutational tolerance upon XBP1s induction. Our data indicate that this phenomenon likely reflects strict quality control endowed by XBP1s-mediated remodeling of the ER proteostasis environment. Intriguingly, and in contrast, specific regions of Env, including regions targeted by broadly neutralizing antibodies, display enhanced mutational tolerance when XBP1s is induced, hinting at a role for host proteostasis network hijacking in potentiating antibody escape. These observations reveal a key function for proteostasis networks in decreasing instead of expanding the sequence space accessible to client proteins, while also demonstrating that the host ER proteostasis network profoundly shapes the mutational tolerance of Env in ways that could have important consequences for HIV adaptation.',\n",
       "  'HEK_and_SupT1_up_XBP1s_ATF6-S7_Data_List_of_genes_upregulated_in_both_HEK293DAX_and_SupT1DAX_cells_for_XBP1s_ATF6_vs_Basal',\n",
       "  'HEK293DAX cells (note that the +ATF6 condition was not tested in the previous study; **S11 Data**)  Transcriptome comparison of HEK293DAX cells and SupT1DAX cells.(XLSX)  Click here for additional data file'),\n",
       " 'PMC9795333-mmc4.xlsx-SEES4-X-linked_genes': ('De-erosion of X chromosome dosage compensation by the editing of XIST regulatory regions restores the differentiation potential in hPSCs',\n",
       "  'Human pluripotent stem cells (hPSCs) regularly and irreversibly show the erosion of X chromosome inactivation (XCI) by long non-coding RNA (lncRNA) XIST silencing, causing challenges in various applications of female hPSCs. Here, we report reliable methods to reactivate XIST with monoallelic expression in female hPSCs. Surprisingly, we find that the editing of XIST regulatory regions by Cas9-mediated non-homologous end joining is sufficient for the reactivation of XIST by endogenous systems. Proliferated hPSCs with XIST reactivation show XCI from an eroded X chromosome, suggesting that hPSCs with normal dosage compensation might lead to a growth advantage. Furthermore, the use of targeting vectors, including the XIST regulatory region sequences and selection cassette, enables XIST reactivation in hPSCs with high efficiency. XIST -reactivated hPSCs can show the restoration of differentiation potential. Thus, our findings demonstrate that XIST re-expression is a beneficial method to maximize the use of female hPSCs in various applications, such as proper disease modeling.',\n",
       "  'SEES4-X-linked_genes',\n",
       "  'is listed in the key resources table. The related statistical data is provided in **Table S3**  Table S3. Statistical data in scRNA-seq, related to Figures 1, 3, S2, and S5'),\n",
       " 'PMC8109120-aging-13-202765-s002.xlsx-Sheet1-Supplementary_Data_1_Genes_that_were_significantly_co-expressed_with_ITIH1_across_pan-cancers': ('Pan-cancer analysis identifies ITIH1 as a novel prognostic indicator for hepatocellular carcinoma',\n",
       "  'Although a previous pan-cancer study has reported the expression patterns of ITIHs in various tumors, their analyses have been restricted to limited cancer types. We thus comprehensively analyzed the expression profiles and clinical significances of ITIHs in a broader spectrum of cancers from TCGA. Our results showed that ITIHs were primarily down-regulated in tested cancers. The ITIH members were associated with either survival advantage or disadvantage, depending on the cancer type tested and the genes queried. Importantly, we for the first time demonstrated that ITIH1 had substantially decreased expression in liver hepatocellular carcinoma (LIHC) compared with corresponding normal tissue, and its down-regulation adversely impacted patient outcome. Moreover, ITIH1 expression was consistently declining during the progression of LIHC. Further analysis revealed that ITIH1 may be involved in cellular metabolic processes. Our findings established ITIH1 as a potential diagnostic and prognostic biomarker for LIHC, which awaits future experimental validation.',\n",
       "  'Sheet1-Supplementary_Data_1_Genes_that_were_significantly_co-expressed_with_ITIH1_across_pan-cancers',\n",
       "  'we derived genes that were significantly co-expressed with it across pan-cancers (r > 0.4, **Supplementary data 1**)'),\n",
       " 'PMC9780220-41598_2022_26217_MOESM2_ESM.xlsx-DEgenes_CTL_FGF2-Unnamed_1': ('Fibroblast growth factor-2 bound to specific dermal fibroblast-derived extracellular vesicles is protected from degradation',\n",
       "  'Fibroblast growth factor-2 (FGF2) has multiple roles in cutaneous wound healing but its natural low stability prevents the development of its use in skin repair therapies. Here we show that FGF2 binds the outer surface of dermal fibroblast (DF)-derived extracellular vesicles (EVs) and this association protects FGF2 from fast degradation. EVs isolated from DF cultured in the presence of FGF2 harbor FGF2 on their surface and FGF2 can bind purified EVs in absence of cells. Remarkably, FGF2 binding to EVs is restricted to a specific subpopulation of EVs, which do not express CD63 and CD81 markers. Treatment of DF with FGF2-EVs activated ERK and STAT signaling pathways and increased cell proliferation and migration. Local injection of FGF2-EVs improved wound healing in mice. We further demonstrated that binding to EVs protects FGF2 from both thermal and proteolytic degradation, thus maintaining FGF2 function. This suggests that EVs protect soluble factors from degradation and increase their stability and half-life. These results reveal a novel aspect of EV function and suggest EVs as a potential tool for delivering FGF2 in skin healing therapies.',\n",
       "  'DEgenes_CTL_FGF2-Unnamed_1',\n",
       "  'To investigate the effect of FGF**2**  Supplementary Information 2'),\n",
       " 'PMC8305866-13041_2021_832_MOESM4_ESM.xlsx-Maximum_vs_Minimum-Gene_Name': ('Transcriptomic profiling of high- and low-spiking regions reveals novel epileptogenic mechanisms in focal cortical dysplasia type II patients',\n",
       "  'Focal cortical dysplasia (FCD) is a malformation of the cerebral cortex with poorly-defined epileptogenic zones (EZs), and poor surgical outcome in FCD is associated with inaccurate localization of the EZ. Hence, identifying novel epileptogenic markers to aid in the localization of EZ in patients with FCD is very much needed. High-throughput gene expression studies of FCD samples have the potential to uncover molecular changes underlying the epileptogenic process and identify novel markers for delineating the EZ. For this purpose, we, for the first time performed RNA sequencing of surgically resected paired tissue samples obtained from electrocorticographically graded high (MAX) and low spiking (MIN) regions of FCD type II patients and autopsy controls. We identified significant changes in the MAX samples of the FCD type II patients when compared to non-epileptic controls, but not in the case of MIN samples. We found significant enrichment for myelination, oligodendrocyte development and differentiation, neuronal and axon ensheathment, phospholipid metabolism, cell adhesion and cytoskeleton, semaphorins, and ion channels in the MAX region. Through the integration of both MAX vs non-epileptic control and MAX vs MIN RNA sequencing (RNA Seq) data, PLP1, PLLP, UGT8, KLK6, SOX10, MOG, MAG, MOBP, ANLN, ERMN, SPP1, CLDN11, TNC, GPR37, SLC12A2, ABCA2, ABCA8, ASPA, P2RX7, CERS2, MAP4K4, TF, CTGF, Semaphorins, Opalin, FGFs, CALB2, and TNC were identified as potential key regulators of multiple pathways related to FCD type II pathology. We have identified novel epileptogenic marker elements that may contribute to epileptogenicity in patients with FCD and could be possible markers for the localization of EZ.',\n",
       "  'Maximum_vs_Minimum-Gene_Name',\n",
       "  'differentially expressed genes (16 up-regulated and 22 down-regulated) in MIN vs autopsy; 325 DEGs (22**4**)  Additional file 4. List of significantly altered genes analyzed by DESeq 2'),\n",
       " 'PMC8305866-13041_2021_832_MOESM4_ESM.xlsx-Maximum_vs_Minimum-Gene_ID': ('Transcriptomic profiling of high- and low-spiking regions reveals novel epileptogenic mechanisms in focal cortical dysplasia type II patients',\n",
       "  'Focal cortical dysplasia (FCD) is a malformation of the cerebral cortex with poorly-defined epileptogenic zones (EZs), and poor surgical outcome in FCD is associated with inaccurate localization of the EZ. Hence, identifying novel epileptogenic markers to aid in the localization of EZ in patients with FCD is very much needed. High-throughput gene expression studies of FCD samples have the potential to uncover molecular changes underlying the epileptogenic process and identify novel markers for delineating the EZ. For this purpose, we, for the first time performed RNA sequencing of surgically resected paired tissue samples obtained from electrocorticographically graded high (MAX) and low spiking (MIN) regions of FCD type II patients and autopsy controls. We identified significant changes in the MAX samples of the FCD type II patients when compared to non-epileptic controls, but not in the case of MIN samples. We found significant enrichment for myelination, oligodendrocyte development and differentiation, neuronal and axon ensheathment, phospholipid metabolism, cell adhesion and cytoskeleton, semaphorins, and ion channels in the MAX region. Through the integration of both MAX vs non-epileptic control and MAX vs MIN RNA sequencing (RNA Seq) data, PLP1, PLLP, UGT8, KLK6, SOX10, MOG, MAG, MOBP, ANLN, ERMN, SPP1, CLDN11, TNC, GPR37, SLC12A2, ABCA2, ABCA8, ASPA, P2RX7, CERS2, MAP4K4, TF, CTGF, Semaphorins, Opalin, FGFs, CALB2, and TNC were identified as potential key regulators of multiple pathways related to FCD type II pathology. We have identified novel epileptogenic marker elements that may contribute to epileptogenicity in patients with FCD and could be possible markers for the localization of EZ.',\n",
       "  'Maximum_vs_Minimum-Gene_ID',\n",
       "  'differentially expressed genes (16 up-regulated and 22 down-regulated) in MIN vs autopsy; 325 DEGs (22**4**)  Additional file 4. List of significantly altered genes analyzed by DESeq 2'),\n",
       " 'PMC9574363-Table_3.doc-0-Gene_Symbol': ('Tryptophan 2,3-dioxygenase may be a potential prognostic biomarker and immunotherapy target in cancer: A meta-analysis and bioinformatics analysis',\n",
       "  'Background: Tryptophan 2,3-dioxygenase (TDO2) is one of the emerging immune checkpoints. Meanwhile, TDO2 is also a key enzyme in the tryptophan (Trp)-kynurenine (Kyn) signaling pathway. Many studies have evaluated that TDO2 is highly expressed in various malignant tumor patients and plays a prognostic role. However, the sample size of a single prognostic study was small, and the results were still controversial. Methods: We used Stata software and referenced the Preferred Reporting Items for Systematic Reviews and Meta-analysis (PRISMA) statement to conduct a meta-analysis on TDO2 and its clinical features and prognosis. We searched the PubMed, Cochrane Library, and Web of Science databases to find publications concerning TDO2 expression in malignant tumor patients up to June 2021. We used the Newcastle-Ottawa Scale (NOS) to evaluate the bias risk of the included literature. Risk ratios (RRs) and hazard ratios (HRs) were used for clinical outcomes, specifically overall survival (OS) and progression-free survival (PFS). In addition, we used data from The Cancer Genome Atlas (TCGA) to verify our conclusions. Results: Nine studies including 667 patients with malignant tumors were identified. Our results suggested that overexpression of TDO2 was statistically correlated with poor OS and poor PFS (HR = 2.58, 95% CI = 1.52-4.40, p = 0.0005; HR = 2.38, 95% CI = 0.99-5.73, p = 0.05). In terms of clinicopathological characteristics, the overexpression level of TDO2 was statistically correlated with TNM (tumor-node-metastasis) stage (RR = 0.65, 95% CI = 0.48-0.89, p = 0.002) and regional lymph node metastasis (RR = 0.76, 95% CI = 0.59-0.99, p = 0.04). Subgroup analysis revealed the potential sources of heterogeneity. In addition, bioinformatics studies suggested that the level of TDO2 was high in malignant tumors and higher in cancer tissue than in matched paracarcinoma tissue. Gene enrichment analysis showed that TDO2 was closely related to immune response. Conclusion: Overall, TDO2 may be a biomarker for the survival and prognosis of patients with malignant tumors and a potential therapeutic target in the future. Systematic review registration: https://www.crd.york.ac.uk/prospero/display_record.php?RecordID=260442, identifier (CRD42021260442).',\n",
       "  '0-Gene_Symbol',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC9586873-41388_2022_2465_MOESM3_ESM.xlsx-H-Unnamed_0': ('Regulatory chromatin rewiring promotes metabolic switching during adaptation to oncogenic receptor tyrosine kinase inhibition',\n",
       "  'Oesophageal adenocarcinoma (OAC) patients show poor survival rates and there are few targeted molecular therapies available. However, components of the receptor tyrosine kinase (RTK) driven pathways are commonly mutated in OAC, typified by high frequency amplifications of the RTK ERBB2. ERBB2 can be therapeutically targeted, but this has limited clinical benefit due to the acquisition of drug resistance. Here we examined how OAC cells adapt to ERBB2 inhibition as they transition to a drug resistant state. ERBB2 inhibition triggers widespread remodelling of the accessible chromatin landscape and the underlying gene regulatory networks. The transcriptional regulators HNF4A and PPARGC1A play a key role in this network rewiring. Initially, inhibition of cell cycle associated gene expression programmes is observed, with compensatory increases in the programmes driving changes in metabolic activity. Both PPARGC1A and HNF4A are required for the acquisition of resistance to ERBB2 inhibition and PPARGC1A is instrumental in promoting a switch to dependency on oxidative phosphorylation. Our work therefore reveals the molecular pathways that support the acquisition of a resistant state and points to potential new therapeutic strategies to combat cellular adaptation and ensuing drug resistance.',\n",
       "  'H-Unnamed_0',\n",
       "  'how ERRB2 inhibition rewires OAC cells along the path to resistance, we focussed on OE**1**  Supplementary Table 1'),\n",
       " 'PMC4479346-12864_2015_1686_MOESM3_ESM.docx-0-Gene': ('The human cardiac and skeletal muscle proteomes defined by transcriptomics and antibody-based profiling',\n",
       "  'Background: To understand cardiac and skeletal muscle function, it is important to define and explore their molecular constituents and also to identify similarities and differences in the gene expression in these two different striated muscle tissues. Here, we have investigated the genes and proteins with elevated expression in cardiac and skeletal muscle in relation to all other major human tissues and organs using a global transcriptomics analysis complemented with antibody-based profiling to localize the corresponding proteins on a single cell level. Results: Our study identified a comprehensive list of genes expressed in cardiac and skeletal muscle. The genes with elevated expression were further stratified according to their global expression pattern across the human body as well as their precise localization in the muscle tissues. The functions of the proteins encoded by the elevated genes are well in line with the physiological functions of cardiac and skeletal muscle, such as contraction, ion transport, regulation of membrane potential and actomyosin structure organization. A large fraction of the transcripts in both cardiac and skeletal muscle correspond to mitochondrial proteins involved in energy metabolism, which demonstrates the extreme specialization of these muscle tissues to provide energy for contraction. Conclusions: Our results provide a comprehensive list of genes and proteins elevated in striated muscles. A number of proteins not previously characterized in cardiac and skeletal muscle were identified and localized to specific cellular subcompartments. These proteins represent an interesting starting point for further functional analysis of their role in muscle biology and disease.',\n",
       "  '0-Gene',\n",
       "  'None here'),\n",
       " 'PMC9153011-advancesADV2021005360-suppl1.xlsx-FANC-FDR_0_05_ALL_n_875': ('Genome-wide whole-blood transcriptome profiling across inherited bone marrow failure subtypes',\n",
       "  'Gene expression profiling has long been used in understanding the contribution of genes and related pathways in disease pathogenesis and susceptibility. We have performed whole-blood transcriptomic profiling in a subset of patients with inherited bone marrow failure (IBMF) whose diseases are clinically and genetically characterized as Fanconi anemia (FA), Shwachman-Diamond syndrome (SDS), and dyskeratosis congenita (DC). We hypothesized that annotating whole-blood transcripts genome wide will aid in understanding the complexity of gene regulation across these IBMF subtypes. Initial analysis of these blood-derived transcriptomes revealed significant skewing toward upregulated genes in patients with FA when compared with controls. Patients with SDS or DC also showed similar skewing profiles in their transcriptional status revealing a common pattern across these different IBMF subtypes. Gene set enrichment analysis revealed shared pathways involved in protein translation and elongation (ribosome constituents), RNA metabolism (nonsense-mediated decay), and mitochondrial function (electron transport chain). We further identified a discovery set of 26 upregulated genes at stringent cutoff (false discovery rate < 0.05) that appeared as a unified signature across the IBMF subtypes. Subsequent transcriptomic analysis on genetically uncharacterized patients with BMF revealed a striking overlap of genes, including 22 from the discovery set, which indicates a unified transcriptional drive across the classic (FA, SDS, and DC) and uncharacterized BMF subtypes. This study has relevance in disease pathogenesis, for example, in explaining the features (including the BMF) common to all patients with IBMF and suggests harnessing this transcriptional signature for patient benefit.',\n",
       "  'FANC-FDR_0_05_ALL_n_875',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC6363966-mmc4.xlsx-Sheet1-Unnamed_5': ('Circulating Tumor Cell Clustering Shapes DNA Methylation to Enable Metastasis Seeding',\n",
       "  'The ability of circulating tumor cells (CTCs) to form clusters has been linked to increased metastatic potential. Yet biological features and vulnerabilities of CTC clusters remain largely unknown. Here, we profile the DNA methylation landscape of single CTCs and CTC clusters from breast cancer patients and mouse models on a genome-wide scale. We find that binding sites for stemness- and proliferation-associated transcription factors are specifically hypomethylated in CTC clusters, including binding sites for OCT4, NANOG, SOX2, and SIN3A, paralleling embryonic stem cell biology. Among 2,486 FDA-approved compounds, we identify Na + /K + ATPase inhibitors that enable the dissociation of CTC clusters into single cells, leading to DNA methylation remodeling at critical sites and metastasis suppression. Thus, our results link CTC clustering to specific changes in DNA methylation that promote stemness and metastasis and point to cluster-targeting compounds to suppress the spread of cancer.',\n",
       "  'Sheet1-Unnamed_5',\n",
       "  'no co-expression modules were found to be significantly enriched in single CTCs (Figure 3B; **Table S4**)  Table S4. WGCNA of Genes (≥3 TPM Counts) Significantly Enriched (p < 0.05) in Patient- and Xenograft-Derived CTC Clusters, Xenograft-Derived Single CTCs, as well as Stemness-Related Genes in Patient- and Xenograft-Derived CTCs, Related to Figures 3 and 4'),\n",
       " 'PMC10302540-mmc9.xlsx-G1_Positive-Names_of_the_transcripts_with_Gi1_0_02': ('Overexpressed kinetochore genes are used by cancer cells as genome destabilizers and transformation catalysts',\n",
       "  'Cancer cells have an altered transcriptome, which contributes to their abnormal behavior. Many tumors have high levels of kinetochore genes, which play important roles in genome stability. This overexpression could be utilized to destabilize cancer cell genomes, however this has not been proven specifically. We investigated the link between kinetochore gene overexpression, chromosomal number variations (CNVs) and genomic instability. Data on RNA expression and CNV from 12 different cancer types were evaluated using information theory. In all cancer types, we looked at the relationship between RNA expression and CNVs. Kinetochore gene expression was found to be substantially linked with CNV levels. In all cancer types, with the exception of thyroid cancer, highly expressed kinetochore genes were enriched in the most dominant cancer-specific co-expression subnetworks characterizing the largest patient subgroups. Except for thyroid cancer, kinetochore inner protein CENPA was among the transcripts most strongly associated with CNV values in all cancer types studied, with significantly higher expression levels in patients with high CNVs than in patients with low CNVs. CENPA function was investigated further in cell models by transfecting genomically stable (HCT116) and unstable (MCF7 and HT29) cancer cell lines using CENPA overexpression vectors. This overexpression increased the number of abnormal cell divisions in the stable cancer cell line HCT116 and, to a lesser extent, in the unstable cell lines MCF7 and HT29. Overexpression improved anchorage-independent growth properties of all cell lines. Our findings suggest that overexpression of kinetochore genes in general, and CENPA in particular, can cause genomic instability and cancer progression.',\n",
       "  'G1_Positive-Names_of_the_transcripts_with_Gi1_0_02',\n",
       "  'Table S9'),\n",
       " 'PMC6335716-12864_2018_5404_MOESM2_ESM.xlsx-List_16-Gene_name': ('RiboTag translatomic profiling of Drosophila oenocytes under aging and induced oxidative stress',\n",
       "  'Background: Aging is accompanied with loss of tissue homeostasis and accumulation of cellular damages. As one of the important metabolic centers, liver shows age-related dysregulation of lipid metabolism, impaired detoxification pathway, increased inflammation and oxidative stress response. However, the mechanisms for these age-related changes still remain unclear. In the fruit fly, Drosophila melanogaster, liver-like functions are controlled by two distinct tissues, fat body and oenocytes. Compared to fat body, little is known about how oenocytes age and what are their roles in aging regulation. To characterize age- and stress-regulated gene expression in oenocytes, we performed cell-type-specific ribosome profiling (RiboTag) to examine the impacts of aging and oxidative stress on oenocyte translatome in Drosophila. Results: We show that aging and oxidant paraquat significantly increased the levels of reactive oxygen species (ROS) in adult oenocytes of Drosophila, and aged oenocytes exhibited reduced sensitivity to paraquat treatment. Through RiboTag sequencing, we identified 3324 and 949 differentially expressed genes in oenocytes under aging and paraquat treatment, respectively. Aging and paraquat exhibit both shared and distinct regulations on oenocyte translatome. Among all age-regulated genes, oxidative phosphorylation, ribosome, proteasome, fatty acid metabolism, and cytochrome P450 pathways were down-regulated, whereas DNA replication and immune response pathways were up-regulated. In addition, most of the peroxisomal genes were down-regulated in aged oenocytes, including genes involved in peroxisomal biogenesis factors and fatty acid beta-oxidation. Many age-related mRNA translational changes in oenocytes are similar to aged mammalian liver, such as up-regulation of innate immune response and Ras/MAPK signaling pathway and down-regulation of peroxisome and fatty acid metabolism. Furthermore, oenocytes highly expressed genes involving in liver-like processes (e.g., ketogenesis). Conclusions: Our oenocyte-specific translatome analysis identified many genes and pathways that are shared between Drosophila oenocytes and mammalian liver, highlighting the molecular and functional similarities between the two tissues. Many of these genes were altered in both oenocytes and liver during aging. Thus, our translatome analysis provide important genomic resource for future dissection of oenocyte function and its role in lipid metabolism, stress response and aging regulation.',\n",
       "  'List_16-Gene_name',\n",
       "  'DGE analysis identified 33**2**  All gene lists. (XLSX 3055 kb)'),\n",
       " 'PMC10956518-peerj-12-17130-s005.xlsx-255_intersection_genes-GENE': ('Validation of CDC45 as a novel biomarker for diagnosis and prognosis of gastric cancer',\n",
       "  'Background: Cell division cycle protein 45 (CDC45) has been demonstrated to play vital roles in the progression of various malignancies. However, the clinical significance of CDC45 in gastric cancer (GC) remains unreported. Method: In this study, we employed the TCGA database and the TCGA & GTEx dataset to compare the mRNA expression levels of CDC45 between gastric cancer tissues and adjacent or normal tissues ( p < 0.05 was considered statistically significant), which was further validated in multiple datasets including GSE13911, GSE29272, GSE118916, GSE66229, as well as RT-qPCR. Furthermore, we harnessed the Human Protein Atlas (HPA) to evaluate the protein expression of CDC45, which was subsequently verified through immunohistochemistry (IHC). To ascertain the diagnostic utility of CDC45, receiver operating characteristic (ROC) curves and the area under the ROC curve (AUC) were calculated in TCGA database, and further validated it in TCGA & GTEx and GSE66229 datasets. The Kaplan-Meier method was used to reveal the prognostic importance of CDC45 in The Cancer Genome Atlas (TCGA) database and authenticated through the GSE66229, GSE84433, and GSE84437 datasets. Through cBioPortal, we identified co-expressed genes of CDC45, and pursued enrichment analysis. Additionally, we availed gene set enrichment analysis (GSEA) to annotate the biological functions of CDC45. Results: Differential expression analysis revealed that CDC45 was significantly upregulated at both the mRNA and protein levels in GC (all p < 0.05 ). Remarkably, CDC45 emerged as a promising prognostic indicator and a novel diagnostic biomarker for GC. In a comprehensive the drug susceptibility analysis, we found that patients with high expression of CDC45 had high sensitivity to various chemotherapeutic agents, among which 5-fluorouracil, docetaxel, cisplatin, and elesclomol were most evident. Furthermore, our findings suggested a plausible association between CDC45 and immune cell infiltration. Enrichment analysis revealed that CDC45 and its associated genes may play crucial roles in muscle biofunction, whereas GSEA demonstrated significant enrichment of gene sets pertaining to G protein-coupled receptor ligand binding and G alpha (i) signaling events. Conclusion: Our study elucidates that upregulation of CDC45 is intricately associated with immune cell infiltration and holds promising potential as a favorable prognostic marker and a novel diagnostic biomarker for GC.',\n",
       "  '255_intersection_genes-GENE',\n",
       "  'None here'),\n",
       " 'PMC10956518-peerj-12-17130-s006.xlsx-heatmap_of_co-expression_genes-CDCA5': ('Validation of CDC45 as a novel biomarker for diagnosis and prognosis of gastric cancer',\n",
       "  'Background: Cell division cycle protein 45 (CDC45) has been demonstrated to play vital roles in the progression of various malignancies. However, the clinical significance of CDC45 in gastric cancer (GC) remains unreported. Method: In this study, we employed the TCGA database and the TCGA & GTEx dataset to compare the mRNA expression levels of CDC45 between gastric cancer tissues and adjacent or normal tissues ( p < 0.05 was considered statistically significant), which was further validated in multiple datasets including GSE13911, GSE29272, GSE118916, GSE66229, as well as RT-qPCR. Furthermore, we harnessed the Human Protein Atlas (HPA) to evaluate the protein expression of CDC45, which was subsequently verified through immunohistochemistry (IHC). To ascertain the diagnostic utility of CDC45, receiver operating characteristic (ROC) curves and the area under the ROC curve (AUC) were calculated in TCGA database, and further validated it in TCGA & GTEx and GSE66229 datasets. The Kaplan-Meier method was used to reveal the prognostic importance of CDC45 in The Cancer Genome Atlas (TCGA) database and authenticated through the GSE66229, GSE84433, and GSE84437 datasets. Through cBioPortal, we identified co-expressed genes of CDC45, and pursued enrichment analysis. Additionally, we availed gene set enrichment analysis (GSEA) to annotate the biological functions of CDC45. Results: Differential expression analysis revealed that CDC45 was significantly upregulated at both the mRNA and protein levels in GC (all p < 0.05 ). Remarkably, CDC45 emerged as a promising prognostic indicator and a novel diagnostic biomarker for GC. In a comprehensive the drug susceptibility analysis, we found that patients with high expression of CDC45 had high sensitivity to various chemotherapeutic agents, among which 5-fluorouracil, docetaxel, cisplatin, and elesclomol were most evident. Furthermore, our findings suggested a plausible association between CDC45 and immune cell infiltration. Enrichment analysis revealed that CDC45 and its associated genes may play crucial roles in muscle biofunction, whereas GSEA demonstrated significant enrichment of gene sets pertaining to G protein-coupled receptor ligand binding and G alpha (i) signaling events. Conclusion: Our study elucidates that upregulation of CDC45 is intricately associated with immune cell infiltration and holds promising potential as a favorable prognostic marker and a novel diagnostic biomarker for GC.',\n",
       "  'heatmap_of_co-expression_genes-CDCA5',\n",
       "  'None here'),\n",
       " 'PMC11101176-Table1.XLSX-S_Table_3_Y_X_Auto-Unnamed_8': ('Sex-biased gene expression during neural differentiation of human embryonic stem cells',\n",
       "  'Sex differences in the developing human brain are primarily attributed to hormonal influence. Recently however, genetic differences and their impact on the developing nervous system have attracted increased attention. To understand genetically driven sexual dimorphisms in neurodevelopment, we investigated genome-wide gene expression in an in vitro differentiation model of male and female human embryonic stem cell lines (hESC), independent of the effects of human sex hormones. Four male and four female-derived hESC lines were differentiated into a population of mixed neurons over 37 days. Differential gene expression and gene set enrichment analyses were conducted on bulk RNA sequencing data. While similar differentiation tendencies in all cell lines demonstrated the robustness and reproducibility of our differentiation protocol, we found sex-biased gene expression already in undifferentiated ESCs at day 0, but most profoundly after 37 days of differentiation. Male and female cell lines exhibited sex-biased expression of genes involved in neurodevelopment, suggesting that sex influences the differentiation trajectory. Interestingly, the highest contribution to sex differences was found to arise from the male transcriptome, involving both Y chromosome and autosomal genes. We propose 13 sex-biased candidate genes (10 upregulated in male cell lines and 3 in female lines) that are likely to affect neuronal development. Additionally, we confirmed gene dosage compensation of X/Y homologs escaping X chromosome inactivation through their Y homologs and identified a significant overexpression of the Y-linked demethylase UTY and KDM5D in male hESC during neuron development, confirming previous results in neural stem cells. Our results suggest that genetic sex differences affect neuronal differentiation trajectories, which could ultimately contribute to sex biases during human brain development.',\n",
       "  'S_Table_3_Y_X_Auto-Unnamed_8',\n",
       "  'None here'),\n",
       " 'PMC9795333-mmc4.xlsx-SE5-res-4C-X-linked_genes': ('De-erosion of X chromosome dosage compensation by the editing of XIST regulatory regions restores the differentiation potential in hPSCs',\n",
       "  'Human pluripotent stem cells (hPSCs) regularly and irreversibly show the erosion of X chromosome inactivation (XCI) by long non-coding RNA (lncRNA) XIST silencing, causing challenges in various applications of female hPSCs. Here, we report reliable methods to reactivate XIST with monoallelic expression in female hPSCs. Surprisingly, we find that the editing of XIST regulatory regions by Cas9-mediated non-homologous end joining is sufficient for the reactivation of XIST by endogenous systems. Proliferated hPSCs with XIST reactivation show XCI from an eroded X chromosome, suggesting that hPSCs with normal dosage compensation might lead to a growth advantage. Furthermore, the use of targeting vectors, including the XIST regulatory region sequences and selection cassette, enables XIST reactivation in hPSCs with high efficiency. XIST -reactivated hPSCs can show the restoration of differentiation potential. Thus, our findings demonstrate that XIST re-expression is a beneficial method to maximize the use of female hPSCs in various applications, such as proper disease modeling.',\n",
       "  'SE5-res-4C-X-linked_genes',\n",
       "  'is listed in the key resources table. The related statistical data is provided in **Table S3**  Table S3. Statistical data in scRNA-seq, related to Figures 1, 3, S2, and S5'),\n",
       " 'PMC6335716-12864_2018_5404_MOESM2_ESM.xlsx-List_16-Gene': ('RiboTag translatomic profiling of Drosophila oenocytes under aging and induced oxidative stress',\n",
       "  'Background: Aging is accompanied with loss of tissue homeostasis and accumulation of cellular damages. As one of the important metabolic centers, liver shows age-related dysregulation of lipid metabolism, impaired detoxification pathway, increased inflammation and oxidative stress response. However, the mechanisms for these age-related changes still remain unclear. In the fruit fly, Drosophila melanogaster, liver-like functions are controlled by two distinct tissues, fat body and oenocytes. Compared to fat body, little is known about how oenocytes age and what are their roles in aging regulation. To characterize age- and stress-regulated gene expression in oenocytes, we performed cell-type-specific ribosome profiling (RiboTag) to examine the impacts of aging and oxidative stress on oenocyte translatome in Drosophila. Results: We show that aging and oxidant paraquat significantly increased the levels of reactive oxygen species (ROS) in adult oenocytes of Drosophila, and aged oenocytes exhibited reduced sensitivity to paraquat treatment. Through RiboTag sequencing, we identified 3324 and 949 differentially expressed genes in oenocytes under aging and paraquat treatment, respectively. Aging and paraquat exhibit both shared and distinct regulations on oenocyte translatome. Among all age-regulated genes, oxidative phosphorylation, ribosome, proteasome, fatty acid metabolism, and cytochrome P450 pathways were down-regulated, whereas DNA replication and immune response pathways were up-regulated. In addition, most of the peroxisomal genes were down-regulated in aged oenocytes, including genes involved in peroxisomal biogenesis factors and fatty acid beta-oxidation. Many age-related mRNA translational changes in oenocytes are similar to aged mammalian liver, such as up-regulation of innate immune response and Ras/MAPK signaling pathway and down-regulation of peroxisome and fatty acid metabolism. Furthermore, oenocytes highly expressed genes involving in liver-like processes (e.g., ketogenesis). Conclusions: Our oenocyte-specific translatome analysis identified many genes and pathways that are shared between Drosophila oenocytes and mammalian liver, highlighting the molecular and functional similarities between the two tissues. Many of these genes were altered in both oenocytes and liver during aging. Thus, our translatome analysis provide important genomic resource for future dissection of oenocyte function and its role in lipid metabolism, stress response and aging regulation.',\n",
       "  'List_16-Gene',\n",
       "  'DGE analysis identified 33**2**  All gene lists. (XLSX 3055 kb)'),\n",
       " 'PMC5323210-oncotarget-07-66003-s003.xlsx-Down_5FUVSCtrl_48hr-Gene_ID': ('Identification of candidate anti-cancer molecular mechanisms of Compound Kushen Injection using functional genomics',\n",
       "  'Compound Kushen Injection (CKI) has been clinically used in China for over 15 years to treat various types of solid tumours. However, because such Traditional Chinese Medicine (TCM) preparations are complex mixtures of plant secondary metabolites, it is essential to explore their underlying molecular mechanisms in a systematic fashion. We have used the MCF-7 human breast cancer cell line as an initial in vitro model to identify CKI induced changes in gene expression. Cells were treated with CKI for 24 and 48 hours at two concentrations (1 and 2 mg/mL total alkaloids), and the effect of CKI on cell proliferation and apoptosis were measured using XTT and Annexin V/Propidium Iodide staining assays respectively. Transcriptome data of cells treated with CKI or 5-Fluorouracil (5-FU) for 24 and 48 hours were subsequently acquired using high-throughput Illumina RNA-seq technology. In this report we show that CKI inhibited MCF-7 cell proliferation and induced apoptosis in a dose-dependent fashion. We integrated and applied a series of transcriptome analysis methods, including gene differential expression analysis, pathway over-representation analysis, de novo identification of long non-coding RNAs (lncRNA) as well as co-expression network reconstruction, to identify candidate anti-cancer molecular mechanisms of CKI. Multiple pathways were perturbed and the cell cycle was identified as the potential primary target pathway of CKI in MCF-7 cells. CKI may also induce apoptosis in MCF-7 cells via a p53 independent mechanism. In addition, we identified novel lncRNAs and showed that many of them might be expressed as a response to CKI treatment.',\n",
       "  'Down_5FUVSCtrl_48hr-Gene_ID',\n",
       "  '2 mg/mL CKI and 5-FU for 24 or 48 hours respectively (Figure 2 and **Supplementary Table 2**)'),\n",
       " 'PMC9795333-mmc4.xlsx-SEES1-X-linked_genes': ('De-erosion of X chromosome dosage compensation by the editing of XIST regulatory regions restores the differentiation potential in hPSCs',\n",
       "  'Human pluripotent stem cells (hPSCs) regularly and irreversibly show the erosion of X chromosome inactivation (XCI) by long non-coding RNA (lncRNA) XIST silencing, causing challenges in various applications of female hPSCs. Here, we report reliable methods to reactivate XIST with monoallelic expression in female hPSCs. Surprisingly, we find that the editing of XIST regulatory regions by Cas9-mediated non-homologous end joining is sufficient for the reactivation of XIST by endogenous systems. Proliferated hPSCs with XIST reactivation show XCI from an eroded X chromosome, suggesting that hPSCs with normal dosage compensation might lead to a growth advantage. Furthermore, the use of targeting vectors, including the XIST regulatory region sequences and selection cassette, enables XIST reactivation in hPSCs with high efficiency. XIST -reactivated hPSCs can show the restoration of differentiation potential. Thus, our findings demonstrate that XIST re-expression is a beneficial method to maximize the use of female hPSCs in various applications, such as proper disease modeling.',\n",
       "  'SEES1-X-linked_genes',\n",
       "  'is listed in the key resources table. The related statistical data is provided in **Table S3**  Table S3. Statistical data in scRNA-seq, related to Figures 1, 3, S2, and S5'),\n",
       " 'PMC9795333-mmc4.xlsx-EDOM-X-linked_genes': ('De-erosion of X chromosome dosage compensation by the editing of XIST regulatory regions restores the differentiation potential in hPSCs',\n",
       "  'Human pluripotent stem cells (hPSCs) regularly and irreversibly show the erosion of X chromosome inactivation (XCI) by long non-coding RNA (lncRNA) XIST silencing, causing challenges in various applications of female hPSCs. Here, we report reliable methods to reactivate XIST with monoallelic expression in female hPSCs. Surprisingly, we find that the editing of XIST regulatory regions by Cas9-mediated non-homologous end joining is sufficient for the reactivation of XIST by endogenous systems. Proliferated hPSCs with XIST reactivation show XCI from an eroded X chromosome, suggesting that hPSCs with normal dosage compensation might lead to a growth advantage. Furthermore, the use of targeting vectors, including the XIST regulatory region sequences and selection cassette, enables XIST reactivation in hPSCs with high efficiency. XIST -reactivated hPSCs can show the restoration of differentiation potential. Thus, our findings demonstrate that XIST re-expression is a beneficial method to maximize the use of female hPSCs in various applications, such as proper disease modeling.',\n",
       "  'EDOM-X-linked_genes',\n",
       "  'is listed in the key resources table. The related statistical data is provided in **Table S3**  Table S3. Statistical data in scRNA-seq, related to Figures 1, 3, S2, and S5'),\n",
       " 'PMC7005888-41598_2020_58845_MOESM2_ESM.xlsx-Liver_expression-APOA2': ('Exome sequencing revealed DNA variants in NCOR1, IGF2BP1, SGLT2 and NEK11 as potential novel causes of ketotic hypoglycemia in children',\n",
       "  'Unexplained or idiopathic ketotic hypoglycemia (KH) is the most common type of hypoglycemia in children. The diagnosis is based on the exclusion of routine hormonal and metabolic causes of hypoglycemia. We aimed to identify novel genes that cause KH, as this may lead to a more targeted treatment. Deep phenotyping of ten preschool age at onset KH patients (boys, n = 5; girls, n = 5) was performed followed by trio exome sequencing and comprehensive bioinformatics analysis. Data analysis revealed four novel candidate genes: (1) NCOR1 in a patient with KH, iron deficiency and loose stools; (2) IGF2BP1 in a proband with KH, short stature and delayed bone age; (3) SLC5A2 in a proband with KH, intermittent glucosuria and extremely elevated p-GLP-1; and (4) NEK11 in a proband with ketotic hypoglycemia and liver affliction. These genes are associated with different metabolic processes, such as gluconeogenesis, translational regulation, and glucose transport. In conclusion, WES identified DNA variants in four different genes as potential novel causes of IKH, suggesting that IKH is a heterogeneous disorder that can be split into several novel diseases: NCOR1-KH, IGF2BP1-KH, SGLT2-KH or familial renal glucosuria KH, and NEK11-KH. Precision medicine treatment based on exome sequencing may lead to advances in the management of IKH.',\n",
       "  'Liver_expression-APOA2',\n",
       "  'Supplementary Dataset 1'),\n",
       " 'PMC9681209-elife-77335-supp4.xlsx-Supplementary_File_4i-OPC-Enriched': ('Global hypo-methylation in a proportion of glioblastoma enriched for an astrocytic signature is associated with increased invasion and altered immune landscape',\n",
       "  'We describe a subset of glioblastoma, the most prevalent malignant adult brain tumour, harbouring a bias towards hypomethylation at defined differentially methylated regions. This epigenetic signature correlates with an enrichment for an astrocytic gene signature, which together with the identification of enriched predicted binding sites of transcription factors known to cause demethylation and to be involved in astrocytic/glial lineage specification, point to a shared ontogeny between these glioblastomas and astroglial progenitors. At functional level, increased invasiveness, at least in part mediated by SRPX2, and macrophage infiltration characterise this subset of glioblastoma.',\n",
       "  'Supplementary_File_4i-OPC-Enriched',\n",
       "  'further differentiated into iNSCs, iAPCs, and iOPCs. All media recipes can be found in **Supplementary file 4a-e**  Media recipes, primer sequences, antibody details, shRNA plasmid details and gene signature lists used in this study'),\n",
       " 'PMC9795333-mmc4.xlsx-SE5-res-2C-X-linked_genes': ('De-erosion of X chromosome dosage compensation by the editing of XIST regulatory regions restores the differentiation potential in hPSCs',\n",
       "  'Human pluripotent stem cells (hPSCs) regularly and irreversibly show the erosion of X chromosome inactivation (XCI) by long non-coding RNA (lncRNA) XIST silencing, causing challenges in various applications of female hPSCs. Here, we report reliable methods to reactivate XIST with monoallelic expression in female hPSCs. Surprisingly, we find that the editing of XIST regulatory regions by Cas9-mediated non-homologous end joining is sufficient for the reactivation of XIST by endogenous systems. Proliferated hPSCs with XIST reactivation show XCI from an eroded X chromosome, suggesting that hPSCs with normal dosage compensation might lead to a growth advantage. Furthermore, the use of targeting vectors, including the XIST regulatory region sequences and selection cassette, enables XIST reactivation in hPSCs with high efficiency. XIST -reactivated hPSCs can show the restoration of differentiation potential. Thus, our findings demonstrate that XIST re-expression is a beneficial method to maximize the use of female hPSCs in various applications, such as proper disease modeling.',\n",
       "  'SE5-res-2C-X-linked_genes',\n",
       "  'is listed in the key resources table. The related statistical data is provided in **Table S3**  Table S3. Statistical data in scRNA-seq, related to Figures 1, 3, S2, and S5'),\n",
       " 'PMC9795333-mmc4.xlsx-SEES2-X-linked_genes': ('De-erosion of X chromosome dosage compensation by the editing of XIST regulatory regions restores the differentiation potential in hPSCs',\n",
       "  'Human pluripotent stem cells (hPSCs) regularly and irreversibly show the erosion of X chromosome inactivation (XCI) by long non-coding RNA (lncRNA) XIST silencing, causing challenges in various applications of female hPSCs. Here, we report reliable methods to reactivate XIST with monoallelic expression in female hPSCs. Surprisingly, we find that the editing of XIST regulatory regions by Cas9-mediated non-homologous end joining is sufficient for the reactivation of XIST by endogenous systems. Proliferated hPSCs with XIST reactivation show XCI from an eroded X chromosome, suggesting that hPSCs with normal dosage compensation might lead to a growth advantage. Furthermore, the use of targeting vectors, including the XIST regulatory region sequences and selection cassette, enables XIST reactivation in hPSCs with high efficiency. XIST -reactivated hPSCs can show the restoration of differentiation potential. Thus, our findings demonstrate that XIST re-expression is a beneficial method to maximize the use of female hPSCs in various applications, such as proper disease modeling.',\n",
       "  'SEES2-X-linked_genes',\n",
       "  'is listed in the key resources table. The related statistical data is provided in **Table S3**  Table S3. Statistical data in scRNA-seq, related to Figures 1, 3, S2, and S5'),\n",
       " 'PMC9795333-mmc4.xlsx-SEES7-X-linked_genes': ('De-erosion of X chromosome dosage compensation by the editing of XIST regulatory regions restores the differentiation potential in hPSCs',\n",
       "  'Human pluripotent stem cells (hPSCs) regularly and irreversibly show the erosion of X chromosome inactivation (XCI) by long non-coding RNA (lncRNA) XIST silencing, causing challenges in various applications of female hPSCs. Here, we report reliable methods to reactivate XIST with monoallelic expression in female hPSCs. Surprisingly, we find that the editing of XIST regulatory regions by Cas9-mediated non-homologous end joining is sufficient for the reactivation of XIST by endogenous systems. Proliferated hPSCs with XIST reactivation show XCI from an eroded X chromosome, suggesting that hPSCs with normal dosage compensation might lead to a growth advantage. Furthermore, the use of targeting vectors, including the XIST regulatory region sequences and selection cassette, enables XIST reactivation in hPSCs with high efficiency. XIST -reactivated hPSCs can show the restoration of differentiation potential. Thus, our findings demonstrate that XIST re-expression is a beneficial method to maximize the use of female hPSCs in various applications, such as proper disease modeling.',\n",
       "  'SEES7-X-linked_genes',\n",
       "  'is listed in the key resources table. The related statistical data is provided in **Table S3**  Table S3. Statistical data in scRNA-seq, related to Figures 1, 3, S2, and S5'),\n",
       " 'PMC9795333-mmc4.xlsx-SEES5-X-linked_genes': ('De-erosion of X chromosome dosage compensation by the editing of XIST regulatory regions restores the differentiation potential in hPSCs',\n",
       "  'Human pluripotent stem cells (hPSCs) regularly and irreversibly show the erosion of X chromosome inactivation (XCI) by long non-coding RNA (lncRNA) XIST silencing, causing challenges in various applications of female hPSCs. Here, we report reliable methods to reactivate XIST with monoallelic expression in female hPSCs. Surprisingly, we find that the editing of XIST regulatory regions by Cas9-mediated non-homologous end joining is sufficient for the reactivation of XIST by endogenous systems. Proliferated hPSCs with XIST reactivation show XCI from an eroded X chromosome, suggesting that hPSCs with normal dosage compensation might lead to a growth advantage. Furthermore, the use of targeting vectors, including the XIST regulatory region sequences and selection cassette, enables XIST reactivation in hPSCs with high efficiency. XIST -reactivated hPSCs can show the restoration of differentiation potential. Thus, our findings demonstrate that XIST re-expression is a beneficial method to maximize the use of female hPSCs in various applications, such as proper disease modeling.',\n",
       "  'SEES5-X-linked_genes',\n",
       "  'is listed in the key resources table. The related statistical data is provided in **Table S3**  Table S3. Statistical data in scRNA-seq, related to Figures 1, 3, S2, and S5'),\n",
       " 'PMC9795333-mmc4.xlsx-ADSC-X-linked_genes': ('De-erosion of X chromosome dosage compensation by the editing of XIST regulatory regions restores the differentiation potential in hPSCs',\n",
       "  'Human pluripotent stem cells (hPSCs) regularly and irreversibly show the erosion of X chromosome inactivation (XCI) by long non-coding RNA (lncRNA) XIST silencing, causing challenges in various applications of female hPSCs. Here, we report reliable methods to reactivate XIST with monoallelic expression in female hPSCs. Surprisingly, we find that the editing of XIST regulatory regions by Cas9-mediated non-homologous end joining is sufficient for the reactivation of XIST by endogenous systems. Proliferated hPSCs with XIST reactivation show XCI from an eroded X chromosome, suggesting that hPSCs with normal dosage compensation might lead to a growth advantage. Furthermore, the use of targeting vectors, including the XIST regulatory region sequences and selection cassette, enables XIST reactivation in hPSCs with high efficiency. XIST -reactivated hPSCs can show the restoration of differentiation potential. Thus, our findings demonstrate that XIST re-expression is a beneficial method to maximize the use of female hPSCs in various applications, such as proper disease modeling.',\n",
       "  'ADSC-X-linked_genes',\n",
       "  'is listed in the key resources table. The related statistical data is provided in **Table S3**  Table S3. Statistical data in scRNA-seq, related to Figures 1, 3, S2, and S5'),\n",
       " 'PMC6042247-NIHMS966841-supplement-Sup_Table_2.xlsx-ENTEROCYTE_xls-PROBE': ('Parasitic helminths induce fetal-like reversion in the intestinal stem cell niche',\n",
       "  'Epithelial surfaces form critical barriers to the outside world and are continuously renewed by adult stem cells 1 . Whereas dynamics of epithelial stem cells during homeostasis are increasingly well understood, how stem cells are redirected from a tissue-maintenance program to initiate repair after injury remains unclear. Here we examined infection by Heligmosomoides polygyrus, a co-evolved pathosymbiont of mice, to assess the epithelial response to disruption of the mucosal barrier. H. polygyrus disrupts tissue integrity by penetrating the duodenal mucosa, where it develops while surrounded by a multicellular granulomatous infiltrate 2 . Crypts overlying larvae-associated granulomas did not express intestinal stem cell markers, including Lgr5 3 , in spite of continued epithelial proliferation. Granuloma-associated Lgr5 - crypt epithelium activated an interferon-gamma (IFN-γ)-dependent transcriptional program, highlighted by Sca-1 expression, and IFN-γ-producing immune cells were found in granulomas. A similar epithelial response accompanied systemic activation of immune cells, intestinal irradiation, or ablation of Lgr5 + intestinal stem cells. When cultured in vitro, granuloma-associated crypt cells formed spheroids similar to those formed by fetal epithelium, and a sub-population of H. polygyrus-induced cells activated a fetal-like transcriptional program, demonstrating that adult intestinal tissues can repurpose aspects of fetal development. Therefore, re-initiation of the developmental program represents a fundamental mechanism by which the intestinal crypt can remodel itself to sustain function after injury.',\n",
       "  'ENTEROCYTE_xls-PROBE',\n",
       "  'suite of ISC signature genes9 was down-regulated in GCs (Extended Data Fig. 2c–d and **Supplementary Table 2**)'),\n",
       " 'PMC9220854-jciinsight-7-153045-s290.xlsx-Differentiated_epithelial_cells-Absorptive': ('MTG16 regulates colonic epithelial differentiation, colitis, and tumorigenesis by repressing E protein transcription factors',\n",
       "  'Aberrant epithelial differentiation and regeneration contribute to colon pathologies, including inflammatory bowel disease (IBD) and colitis-associated cancer (CAC). Myeloid translocation gene 16 (MTG16, also known as CBFA2T3) is a transcriptional corepressor expressed in the colonic epithelium. MTG16 deficiency in mice exacerbates colitis and increases tumor burden in CAC, though the underlying mechanisms remain unclear. Here, we identified MTG16 as a central mediator of epithelial differentiation, promoting goblet and restraining enteroendocrine cell development in homeostasis and enabling regeneration following dextran sulfate sodium-induced (DSS-induced) colitis. Transcriptomic analyses implicated increased Ephrussi box-binding transcription factor (E protein) activity in MTG16-deficient colon crypts. Using a mouse model with a point mutation that attenuates MTG16:E protein interactions (Mtg16P209T), we showed that MTG16 exerts control over colonic epithelial differentiation and regeneration by repressing E protein-mediated transcription. Mimicking murine colitis, MTG16 expression was increased in biopsies from patients with active IBD compared with unaffected controls. Finally, uncoupling MTG16:E protein interactions partially phenocopied the enhanced tumorigenicity of Mtg16-/- colon in the azoxymethane/DSS-induced model of CAC, indicating that MTG16 protects from tumorigenesis through additional mechanisms. Collectively, our results demonstrate that MTG16, via its repression of E protein targets, is a key regulator of cell fate decisions during colon homeostasis, colitis, and cancer.',\n",
       "  'Differentiated_epithelial_cells-Absorptive',\n",
       "  'Supplemental table 4'),\n",
       " 'PMC9095838-41467_2022_30352_MOESM3_ESM.xlsx-Supplementary_Data_3-Supplementary_Data_3_TPM_values_of_X-linked_genes_in_Human': ('Activation of Xist by an evolutionarily conserved function of KDM5C demethylase',\n",
       "  'XX female and XY male therian mammals equalize X-linked gene expression through the mitotically-stable transcriptional inactivation of one of the two X chromosomes in female somatic cells. Here, we describe an essential function of the X-linked homolog of an ancestral X-Y gene pair, Kdm5c-Kdm5d, in the expression of Xist lncRNA, which is required for stable X-inactivation. Ablation of Kdm5c function in females results in a significant reduction in Xist RNA expression. Kdm5c encodes a demethylase that enhances Xist expression by converting histone H3K4me2/3 modifications into H3K4me1. Ectopic expression of mouse and human KDM5C, but not the Y-linked homolog KDM5D, induces Xist in male mouse embryonic stem cells (mESCs). Similarly, marsupial (opossum) Kdm5c but not Kdm5d also upregulates Xist in male mESCs, despite marsupials lacking Xist, suggesting that the KDM5C function that activates Xist in eutherians is strongly conserved and predates the divergence of eutherian and metatherian mammals. In support, prototherian (platypus) Kdm5c also induces Xist in male mESCs. Together, our data suggest that eutherian mammals co-opted the ancestral demethylase KDM5C during sex chromosome evolution to upregulate Xist for the female-specific induction of X-inactivation.',\n",
       "  'Supplementary_Data_3-Supplementary_Data_3_TPM_values_of_X-linked_genes_in_Human',\n",
       "  '[Monodelphis domestica], using prototherian (monotreme) species (platypus [Ornithorhynchus anatinus] and echidna [Tachyglossus aculeatus]) as outgroups7,7**1**  Supplementary Data 1-12'),\n",
       " 'PMC8906867-pbio.3001569.s011.xlsx-All_genes_up_in_HEK_XBP1s-Unnamed_1': ('The endoplasmic reticulum proteostasis network profoundly shapes the protein sequence space accessible to HIV envelope',\n",
       "  'The sequence space accessible to evolving proteins can be enhanced by cellular chaperones that assist biophysically defective clients in navigating complex folding landscapes. It is also possible, at least in theory, for proteostasis mechanisms that promote strict quality control to greatly constrain accessible protein sequence space. Unfortunately, most efforts to understand how proteostasis mechanisms influence evolution rely on artificial inhibition or genetic knockdown of specific chaperones. The few experiments that perturb quality control pathways also generally modulate the levels of only individual quality control factors. Here, we use chemical genetic strategies to tune proteostasis networks via natural stress response pathways that regulate the levels of entire suites of chaperones and quality control mechanisms. Specifically, we upregulate the unfolded protein response (UPR) to test the hypothesis that the host endoplasmic reticulum (ER) proteostasis network shapes the sequence space accessible to human immunodeficiency virus-1 (HIV-1) envelope (Env) protein. Elucidating factors that enhance or constrain Env sequence space is critical because Env evolves extremely rapidly, yielding HIV strains with antibody- and drug-escape mutations. We find that UPR-mediated upregulation of ER proteostasis factors, particularly those controlled by the IRE1-XBP1s UPR arm, globally reduces Env mutational tolerance. Conserved, functionally important Env regions exhibit the largest decreases in mutational tolerance upon XBP1s induction. Our data indicate that this phenomenon likely reflects strict quality control endowed by XBP1s-mediated remodeling of the ER proteostasis environment. Intriguingly, and in contrast, specific regions of Env, including regions targeted by broadly neutralizing antibodies, display enhanced mutational tolerance when XBP1s is induced, hinting at a role for host proteostasis network hijacking in potentiating antibody escape. These observations reveal a key function for proteostasis networks in decreasing instead of expanding the sequence space accessible to client proteins, while also demonstrating that the host ER proteostasis network profoundly shapes the mutational tolerance of Env in ways that could have important consequences for HIV adaptation.',\n",
       "  'All_genes_up_in_HEK_XBP1s-Unnamed_1',\n",
       "  'HEK293DAX cells (note that the +ATF6 condition was not tested in the previous study; **S11 Data**)  Transcriptome comparison of HEK293DAX cells and SupT1DAX cells.(XLSX)  Click here for additional data file'),\n",
       " 'PMC8906867-pbio.3001569.s011.xlsx-All_genes_up_in_HEK_XBP1s-S11_Data_List_of_all_genes_that_were_upregulated_in_HEK293DAX_cells_for_XBP1s_vs_Basal': ('The endoplasmic reticulum proteostasis network profoundly shapes the protein sequence space accessible to HIV envelope',\n",
       "  'The sequence space accessible to evolving proteins can be enhanced by cellular chaperones that assist biophysically defective clients in navigating complex folding landscapes. It is also possible, at least in theory, for proteostasis mechanisms that promote strict quality control to greatly constrain accessible protein sequence space. Unfortunately, most efforts to understand how proteostasis mechanisms influence evolution rely on artificial inhibition or genetic knockdown of specific chaperones. The few experiments that perturb quality control pathways also generally modulate the levels of only individual quality control factors. Here, we use chemical genetic strategies to tune proteostasis networks via natural stress response pathways that regulate the levels of entire suites of chaperones and quality control mechanisms. Specifically, we upregulate the unfolded protein response (UPR) to test the hypothesis that the host endoplasmic reticulum (ER) proteostasis network shapes the sequence space accessible to human immunodeficiency virus-1 (HIV-1) envelope (Env) protein. Elucidating factors that enhance or constrain Env sequence space is critical because Env evolves extremely rapidly, yielding HIV strains with antibody- and drug-escape mutations. We find that UPR-mediated upregulation of ER proteostasis factors, particularly those controlled by the IRE1-XBP1s UPR arm, globally reduces Env mutational tolerance. Conserved, functionally important Env regions exhibit the largest decreases in mutational tolerance upon XBP1s induction. Our data indicate that this phenomenon likely reflects strict quality control endowed by XBP1s-mediated remodeling of the ER proteostasis environment. Intriguingly, and in contrast, specific regions of Env, including regions targeted by broadly neutralizing antibodies, display enhanced mutational tolerance when XBP1s is induced, hinting at a role for host proteostasis network hijacking in potentiating antibody escape. These observations reveal a key function for proteostasis networks in decreasing instead of expanding the sequence space accessible to client proteins, while also demonstrating that the host ER proteostasis network profoundly shapes the mutational tolerance of Env in ways that could have important consequences for HIV adaptation.',\n",
       "  'All_genes_up_in_HEK_XBP1s-S11_Data_List_of_all_genes_that_were_upregulated_in_HEK293DAX_cells_for_XBP1s_vs_Basal',\n",
       "  'HEK293DAX cells (note that the +ATF6 condition was not tested in the previous study; **S11 Data**)  Transcriptome comparison of HEK293DAX cells and SupT1DAX cells.(XLSX)  Click here for additional data file'),\n",
       " 'PMC6887585-MOL2-13-2588-s003.xlsx-Sheet1-Table_S2_Significantly_upregulated_genes_in_each_cluster_identified_by_SCDE': ('Single-cell RNA-seq reveals the invasive trajectory and molecular cascades underlying glioblastoma progression',\n",
       "  \"Glioblastoma (GBM) is the most common and aggressive primary brain tumor, in which GBM stem cells (GSCs) were identified to contribute to aggressive phenotypes and poor prognosis. Yet, how GSCs progress to invasive cells remains largely unexplored. Here, we revealed the cell subpopulations with distinct functional status and the existence of cells with high invasive potential within heterogeneous primary GBM tumors. We reconstructed a branched trajectory by pseudotemporal ordering of single tumor cells, in which the root showed GSC-like phenotype while the end displayed high invasive activity. Thus, we further determined a path along which GSCs gradually transformed to invasive cells, called the 'stem-to-invasion path'. Along this path, cells showed incremental expression of GBM invasion-associated signatures and diminishing expression of GBM stem cell markers. These findings were validated in an independent single-cell data set of GBM. Through analyzing the molecular cascades underlying the path, we identify crucial factors controlling the attainment of invasive potential of tumor cells, including transcription factors and long noncoding RNAs. Our work provides novel insights into GBM progression, especially the attainment of invasive potential in primary tumor cells, and supports the cancer stem cell model, with valuable implications for GBM therapy.\",\n",
       "  'Sheet1-Table_S2_Significantly_upregulated_genes_in_each_cluster_identified_by_SCDE',\n",
       "  'different numbers of significantly upregulated genes in clusters ranging from 4 to 336 (Table **S2**)  Table S2 . Significantly upregulated genes in each cluster identified by SCDE  Click here for additional data file'),\n",
       " 'PMC3419070-1471-2121-13-15-S5.xls-Sheet1-Unnamed_4': ('Identification of novel mitosis regulators through data mining with human centromere/kinetochore proteins as group queries',\n",
       "  'Background: Proteins functioning in the same biological pathway tend to be transcriptionally co-regulated or form protein-protein interactions (PPI). Multiple spatially and temporally regulated events are coordinated during mitosis to achieve faithful chromosome segregation. The molecular players participating in mitosis regulation are still being unravelled experimentally or using in silico methods. Results: An extensive literature review has led to a compilation of 196 human centromere/kinetochore proteins, all with experimental evidence supporting the subcellular localization. Sixty-four were designated as \"core\" centromere/kinetochore components based on peak expression and/or well-characterized functions during mitosis. By interrogating and integrating online resources, we have mined for genes/proteins that display transcriptional co-expression or PPI with the core centromere/kinetochore components. Top-ranked hubs in either co-expression or PPI network are not only enriched with known mitosis regulators, but also contain candidates whose mitotic functions are not yet established. Experimental validation found that KIAA1377 is a novel centrosomal protein that also associates with microtubules and midbody; while TRIP13 is a novel kinetochore protein and directly interacts with mitotic checkpoint silencing protein p31(comet). Conclusions: Transcriptional co-expression and PPI network analyses with known human centromere/kinetochore proteins as a query group help identify novel potential mitosis regulators.',\n",
       "  'Sheet1-Unnamed_4',\n",
       "  'rank common co-expressing genes [48,49] (see Methods for details). As seen in Additional file **5**  Additional file 5Table S5. Top-ranking co-expressing genes retrieved by CoExSearch program  Click here for file'),\n",
       " 'PMC8010670-Table_5.XLSX-Bcell_AntiCD3-Unnamed_0': ('Single Cell Analysis of Blood Mononuclear Cells Stimulated Through Either LPS or Anti-CD3 and Anti-CD28',\n",
       "  'Immune cell activation assays have been widely used for immune monitoring and for understanding disease mechanisms. However, these assays are typically limited in scope. A holistic study of circulating immune cell responses to different activators is lacking. Here we developed a cost-effective high-throughput multiplexed single-cell RNA-seq combined with epitope tagging (CITE-seq) to determine how classic activators of T cells (anti-CD3 coupled with anti-CD28) or monocytes (LPS) alter the cell composition and transcriptional profiles of peripheral blood mononuclear cells (PBMCs) from healthy human donors. Anti-CD3/CD28 treatment activated all classes of lymphocytes either directly (T cells) or indirectly (B and NK cells) but reduced monocyte numbers. Activated T and NK cells expressed senescence and effector molecules, whereas activated B cells transcriptionally resembled autoimmune disease- or age-associated B cells (e.g., CD11c, T-bet). In contrast, LPS specifically targeted monocytes and induced two main states: early activation characterized by the expression of chemoattractants and a later pro-inflammatory state characterized by expression of effector molecules. These data provide a foundation for future immune activation studies with single cell technologies (https://czi-pbmc-cite-seq.jax.org/).',\n",
       "  'Bcell_AntiCD3-Unnamed_0',\n",
       "  'Anti-CD3/CD28 treatment induced CD69 protein expression on NK cells, and transcription of 74 genes (**Supplementary Table 5**)  Click here for additional data file'),\n",
       " 'PMC3338421-pone.0035618.s007.xls-Novelty_categories-Ensembl_ID': ('Exploring the transcriptome of ciliated cells using in silico dissection of human tissues',\n",
       "  \"Cilia are cell organelles that play important roles in cell motility, sensory and developmental functions and are involved in a range of human diseases, known as ciliopathies. Here, we search for novel human genes related to cilia using a strategy that exploits the previously reported tendency of cell type-specific genes to be coexpressed in the transcriptome of complex tissues. Gene coexpression networks were constructed using the noise-resistant WGCNA algorithm in 12 publicly available microarray datasets from human tissues rich in motile cilia: airways, fallopian tubes and brain. A cilia-related coexpression module was detected in 10 out of the 12 datasets. A consensus analysis of this module's gene composition recapitulated 297 known and predicted 74 novel cilia-related genes. 82% of the novel candidates were supported by tissue-specificity expression data from GEO and/or proteomic data from the Human Protein Atlas. The novel findings included a set of genes (DCDC2, DYX1C1, KIAA0319) related to a neurological disease dyslexia suggesting their potential involvement in ciliary functions. Furthermore, we searched for differences in gene composition of the ciliary module between the tissues. A multidrug-and-toxin extrusion transporter MATE2 (SLC47A2) was found as a brain-specific central gene in the ciliary module. We confirm the localization of MATE2 in cilia by immunofluorescence staining using MDCK cells as a model. While MATE2 has previously gained attention as a pharmacologically relevant transporter, its potential relation to cilia is suggested for the first time. Taken together, our large-scale analysis of gene coexpression networks identifies novel genes related to human cell cilia.\",\n",
       "  'Novelty_categories-Ensembl_ID',\n",
       "  'evidence from the previous studies (“novel candidates”). The gene-to-category mapping is provided in the **Table S7**  Stratification of the signature genes into novelty categories. For each gene, the table specifies number of ciliome studies in which the gene was detected with a strong/medium/weak evidence (according to the CilDB evidence codes). The “MEDLINE” column specifies whether the gene is described as potentially related to cilia in MEDLINE abstracts. The “Category” column provides the resulting assignment of the gene to a novelty category: strong evidence from the previous studies, weak evidence from the previous studies, no evidence from the previous studies.(XLS)  Click here for additional data file'),\n",
       " 'PMC3338421-pone.0035618.s010.xls-Summary-EnsemblID': ('Exploring the transcriptome of ciliated cells using in silico dissection of human tissues',\n",
       "  \"Cilia are cell organelles that play important roles in cell motility, sensory and developmental functions and are involved in a range of human diseases, known as ciliopathies. Here, we search for novel human genes related to cilia using a strategy that exploits the previously reported tendency of cell type-specific genes to be coexpressed in the transcriptome of complex tissues. Gene coexpression networks were constructed using the noise-resistant WGCNA algorithm in 12 publicly available microarray datasets from human tissues rich in motile cilia: airways, fallopian tubes and brain. A cilia-related coexpression module was detected in 10 out of the 12 datasets. A consensus analysis of this module's gene composition recapitulated 297 known and predicted 74 novel cilia-related genes. 82% of the novel candidates were supported by tissue-specificity expression data from GEO and/or proteomic data from the Human Protein Atlas. The novel findings included a set of genes (DCDC2, DYX1C1, KIAA0319) related to a neurological disease dyslexia suggesting their potential involvement in ciliary functions. Furthermore, we searched for differences in gene composition of the ciliary module between the tissues. A multidrug-and-toxin extrusion transporter MATE2 (SLC47A2) was found as a brain-specific central gene in the ciliary module. We confirm the localization of MATE2 in cilia by immunofluorescence staining using MDCK cells as a model. While MATE2 has previously gained attention as a pharmacologically relevant transporter, its potential relation to cilia is suggested for the first time. Taken together, our large-scale analysis of gene coexpression networks identifies novel genes related to human cell cilia.\",\n",
       "  'Summary-EnsemblID',\n",
       "  'ciliary candidates. The complete summary (for all the signature proteins) is provided in the **Table S10**  Data summary for the signature genes. For each signature gene, this table summarizes data from the Tables S5, S7, S8 and S9.(XLS)  Click here for additional data file'),\n",
       " 'PMC3338421-pone.0035618.s006.xls-Orthology_information-EnsemblID': ('Exploring the transcriptome of ciliated cells using in silico dissection of human tissues',\n",
       "  \"Cilia are cell organelles that play important roles in cell motility, sensory and developmental functions and are involved in a range of human diseases, known as ciliopathies. Here, we search for novel human genes related to cilia using a strategy that exploits the previously reported tendency of cell type-specific genes to be coexpressed in the transcriptome of complex tissues. Gene coexpression networks were constructed using the noise-resistant WGCNA algorithm in 12 publicly available microarray datasets from human tissues rich in motile cilia: airways, fallopian tubes and brain. A cilia-related coexpression module was detected in 10 out of the 12 datasets. A consensus analysis of this module's gene composition recapitulated 297 known and predicted 74 novel cilia-related genes. 82% of the novel candidates were supported by tissue-specificity expression data from GEO and/or proteomic data from the Human Protein Atlas. The novel findings included a set of genes (DCDC2, DYX1C1, KIAA0319) related to a neurological disease dyslexia suggesting their potential involvement in ciliary functions. Furthermore, we searched for differences in gene composition of the ciliary module between the tissues. A multidrug-and-toxin extrusion transporter MATE2 (SLC47A2) was found as a brain-specific central gene in the ciliary module. We confirm the localization of MATE2 in cilia by immunofluorescence staining using MDCK cells as a model. While MATE2 has previously gained attention as a pharmacologically relevant transporter, its potential relation to cilia is suggested for the first time. Taken together, our large-scale analysis of gene coexpression networks identifies novel genes related to human cell cilia.\",\n",
       "  'Orthology_information-EnsemblID',\n",
       "  'each gene from the signature we determined its orthologs in the model organisms (see **Table S6**)  Orthology information and overlap of the consensus signature with the previous studies. Spreadsheet “Orthology information”: for each gene, orthologs in the model organisms are specified. The orthology information was extracted from the CilDB database. Spreadsheet “Comparison with prev. studies”: overlap of the consensus signature with ciliary genes lists from the previous studies. Columns in this table correspond to those in the Table 3 of the main text (see Results).(XLS)  Click here for additional data file'),\n",
       " 'PMC9935276-mmc3.xlsx-V2A_BNT162b2-EdgeR_analysis_comparing_V0_vs_V2A_for_BNT162b2': ('A systems immunology study comparing innate and adaptive immune responses in adults to COVID-19 mRNA and adenovirus vectored vaccines',\n",
       "  'Identifying the molecular mechanisms that promote optimal immune responses to coronavirus disease 2019 (COVID-19) vaccination is critical for future rational vaccine design. Here, we longitudinally profile innate and adaptive immune responses in 102 adults after the first, second, and third doses of mRNA or adenovirus-vectored COVID-19 vaccines. Using a multi-omics approach, we identify key differences in the immune responses induced by ChAdOx1-S and BNT162b2 that correlate with antigen-specific antibody and T cell responses or vaccine reactogenicity. Unexpectedly, we observe that vaccination with ChAdOx1-S, but not BNT162b2, induces an adenoviral vector-specific memory response after the first dose, which correlates with the expression of proteins with roles in thrombosis with potential implications for thrombosis with thrombocytopenia syndrome (TTS), a rare but serious adverse event linked to adenovirus-vectored vaccines. The COVID-19 Vaccine Immune Responses Study thus represents a major resource that can be used to understand the immunogenicity and reactogenicity of these COVID-19 vaccines.',\n",
       "  'V2A_BNT162b2-EdgeR_analysis_comparing_V0_vs_V2A_for_BNT162b2',\n",
       "  'third (V3A) doses, generating more than 12 billion reads across 263 samples (Figure S3A, **Table S2**)  Table S2. RNA-seq data, differentially expressed genes at each timepoint, pathway, and blood transcriptional module (BTM) analysis, related to Figures 2 and 3'),\n",
       " 'PMC9709075-41598_2022_24792_MOESM10_ESM.xlsx-Fig1A_468_genes-AL645608_8': ('Transcriptomic comparison of primary human lung cells with lung tissue samples and the human A549 lung cell line highlights cell type specific responses during infections with influenza A virus',\n",
       "  'Influenza A virus (IAV) causes pandemics and annual epidemics of severe respiratory infections. A better understanding of the molecular regulation in tissue and cells upon IAV infection is needed to thoroughly understand pathogenesis. We analyzed IAV replication and gene expression induced by IAV strain H3N2 Panama in isolated primary human alveolar epithelial type II cells (AECIIs), the permanent A549 adenocarcinoma cell line, alveolar macrophages (AMs) and explanted human lung tissue by bulk RNA sequencing. Primary AECII exhibit in comparison to AM a broad set of strongly induced genes related to RIG-I and interferon (IFN) signaling. The response of AECII was partly mirrored in A549 cells. In human lung tissue, we observed induction of genes unlike in isolated cells. Viral RNA was used to correlate host cell gene expression changes with viral burden. While relative induction of key genes was similar, gene abundance was highest in AECII cells and AM, while weaker in the human lung (due to less IAV replication) and A549 cells (pointing to their limited suitability as a model). Correlation of host gene induction with viral burden allows a better understanding of the cell-type specific induction of pathways and a possible role of cellular crosstalk requiring intact tissue.',\n",
       "  'Fig1A_468_genes-AL645608_8',\n",
       "  'GEO under the accession number GSE206606. All gene groups utilized are listed in table **S1**  Supplementary Table S1'),\n",
       " 'PMC9935276-mmc3.xlsx-V2A_BNT162b2-Unnamed_6': ('A systems immunology study comparing innate and adaptive immune responses in adults to COVID-19 mRNA and adenovirus vectored vaccines',\n",
       "  'Identifying the molecular mechanisms that promote optimal immune responses to coronavirus disease 2019 (COVID-19) vaccination is critical for future rational vaccine design. Here, we longitudinally profile innate and adaptive immune responses in 102 adults after the first, second, and third doses of mRNA or adenovirus-vectored COVID-19 vaccines. Using a multi-omics approach, we identify key differences in the immune responses induced by ChAdOx1-S and BNT162b2 that correlate with antigen-specific antibody and T cell responses or vaccine reactogenicity. Unexpectedly, we observe that vaccination with ChAdOx1-S, but not BNT162b2, induces an adenoviral vector-specific memory response after the first dose, which correlates with the expression of proteins with roles in thrombosis with potential implications for thrombosis with thrombocytopenia syndrome (TTS), a rare but serious adverse event linked to adenovirus-vectored vaccines. The COVID-19 Vaccine Immune Responses Study thus represents a major resource that can be used to understand the immunogenicity and reactogenicity of these COVID-19 vaccines.',\n",
       "  'V2A_BNT162b2-Unnamed_6',\n",
       "  'third (V3A) doses, generating more than 12 billion reads across 263 samples (Figure S3A, **Table S2**)  Table S2. RNA-seq data, differentially expressed genes at each timepoint, pathway, and blood transcriptional module (BTM) analysis, related to Figures 2 and 3'),\n",
       " 'PMC8748974-41389_2021_377_MOESM2_ESM.xlsx-shCCND3-DEG_1_5_fold-Gene_Symbol': ('CCND3 is indispensable for the maintenance of B-cell acute lymphoblastic leukemia',\n",
       "  'The D-type cyclins (CCND1, CCND2, and CCND3) in association with CDK4/6 are known drivers of cell cycle progression. We reported previously that inactivation of FOXO1 confers growth arrest and apoptosis in B-ALL, partially mediated by subsequent depletion of CCND3. Given that previously the canonical MYC target CCND2 has been considered to play the major role in B-ALL proliferation, further investigation of the role of FOXO1 in CCND3 transcription and the role of CCND3 in B-ALL is warranted. In this study, we demonstrated that CCND3 is essential for the proliferation and survival of B-ALL, independent of the mutational background. Respectively, its expression at mRNA level exceeds that of CCND1 and CCND2. Furthermore, we identified FOXO1 as a CCND3-activating transcription factor in B-ALL. By comparing the effects of CCND3 depletion and CDK4/6 inhibition by palbociclib on B-ALL cells harboring different driver mutations, we found that the anti-apoptotic effect of CCND3 is independent of the kinase activity of the CCND3-CDK4/6 complex. Moreover, we found that CCND3 contributes to CDK8 transcription, which in part might explain the anti-apoptotic effect of CCND3. Finally, we found that increased CCND3 expression is associated with the development of resistance to palbociclib. We conclude that CCND3 plays an essential role in the maintenance of B-ALL, regardless of the underlying driver mutation. Moreover, downregulation of CCND3 expression might be superior to inhibition of CDK4/6 kinase activity in terms of B-ALL treatment.',\n",
       "  'shCCND3-DEG_1_5_fold-Gene_Symbol',\n",
       "  'transcriptome level in three B-ALL cell lines of different subtypes using RNA-sequencing (Supplementary Table **S1**)  Supplementary Table S1'),\n",
       " 'PMC9547573-41467_2022_33623_MOESM4_ESM.xlsx-Cluster_10-Gene': ('Enhanced metanephric specification to functional proximal tubule enables toxicity screening and infectious disease modelling in kidney organoids',\n",
       "  'While pluripotent stem cell-derived kidney organoids are now being used to model renal disease, the proximal nephron remains immature with limited evidence for key functional solute channels. This may reflect early mispatterning of the nephrogenic mesenchyme and/or insufficient maturation. Here we show that enhanced specification to metanephric nephron progenitors results in elongated and radially aligned proximalised nephrons with distinct S1 - S3 proximal tubule cell types. Such PT-enhanced organoids possess improved albumin and organic cation uptake, appropriate KIM-1 upregulation in response to cisplatin, and improved expression of SARS-CoV-2 entry factors resulting in increased viral replication. The striking proximo-distal orientation of nephrons resulted from localized WNT antagonism originating from the organoid stromal core. PT-enhanced organoids represent an improved model to study inherited and acquired proximal tubular disease as well as drug and viral responses.',\n",
       "  'Cluster_10-Gene',\n",
       "  'The resulting D13 and D13 + 14 pooled replicate libraries resolved 19,956 and 15,85**2**  Supplementary Data 2'),\n",
       " 'PMC11218163-13014_2024_2480_MOESM1_ESM.xlsx-DEG-Figure_5_A': ('Transcriptomic response of prostate cancer cells to carbon ion and photon irradiation with focus on androgen receptor and TP53 signaling',\n",
       "  'Background: Radiotherapy is essential in the treatment of prostate cancer. An alternative to conventional photon radiotherapy is the application of carbon ions, which provide a superior intratumoral dose distribution and less induced damage to adjacent healthy tissue. A common characteristic of prostate cancer cells is their dependence on androgens which is exploited therapeutically by androgen deprivation therapy in the advanced prostate cancer stage. Here, we aimed to analyze the transcriptomic response of prostate cancer cells to irradiation by photons in comparison to carbon ions, focusing on DNA damage, DNA repair and androgen receptor signaling. Methods: Prostate cancer cell lines LNCaP (functional TP53 and androgen receptor signaling) and DU145 (dysfunctional TP53 and androgen receptor signaling) were irradiated by photons or carbon ions and the subsequent DNA damage was assessed by immuno-cytofluorescence. Furthermore, the cells were treated with an androgen-receptor agonist. The effects of irradiation and androgen treatment on the gene regulation and the transcriptome were investigated by RT-qPCR and RNA sequencing, followed by bioinformatic analysis. Results: Following photon or carbon ion irradiation, both LNCaP and DU145 cells showed a dose-dependent amount of visible DNA damage that decreased over time, indicating occurring DNA repair. In terms of gene regulation, mRNAs involved in the TP53-dependent DNA damage response were significantly upregulated by photons and carbon ions in LNCaP but not in DU145 cells, which generally showed low levels of gene regulation after irradiation. Both LNCaP and DU145 cells responded to photons and carbon ions by downregulation of genes involved in DNA repair and cell cycle, partially resembling the transcriptome response to the applied androgen receptor agonist. Neither photons nor carbon ions significantly affected canonical androgen receptor-dependent gene regulation. Furthermore, certain genes that were specifically regulated by either photon or carbon ion irradiation were identified. Conclusion: Photon and carbon ion irradiation showed a significant congruence in terms of induced signaling pathways and transcriptomic responses. These responses were strongly impacted by the TP53 status. Nevertheless, irradiation mode-dependent distinct gene regulations with undefined implication for radiotherapy outcome were revealed. Androgen receptor signaling and irradiations shared regulation of certain genes with respect to DNA-repair and cell-cycle.',\n",
       "  'DEG-Figure_5_A',\n",
       "  'None here'),\n",
       " 'PMC5642107-NIHMS907763-supplement-7.xlsx-Sheet1-Genes_up-regulated_in_Cluster_6': ('Temporal Tracking of Microglia Activation in Neurodegeneration at Single-Cell Resolution',\n",
       "  \"Microglia, the tissue-resident macrophages in the brain, are damage sensors that react to nearly any perturbation, including neurodegenerative diseases such as Alzheimer's disease (AD). Here, using single-cell RNA sequencing, we determined the transcriptome of more than 1,600 individual microglia cells isolated from the hippocampus of a mouse model of severe neurodegeneration with AD-like phenotypes and of control mice at multiple time points during progression of neurodegeneration. In this neurodegeneration model, we discovered two molecularly distinct reactive microglia phenotypes that are typified by modules of co-regulated type I and type II interferon response genes, respectively. Furthermore, our work identified previously unobserved heterogeneity in the response of microglia to neurodegeneration, discovered disease stage-specific microglia cell states, revealed the trajectory of cellular reprogramming of microglia in response to neurodegeneration, and uncovered the underlying transcriptional programs.\",\n",
       "  'Sheet1-Genes_up-regulated_in_Cluster_6',\n",
       "  'Ifi30 were found among the genes most significantly correlated with age in human microglia (**Table S6**)'),\n",
       " 'PMC10014581-41556_2022_1075_MOESM2_ESM.xlsx-1f-Unnamed_1': ('Expansion of ventral foregut is linked to changes in the enhancer landscape for organ-specific differentiation',\n",
       "  'Cell proliferation is fundamental for almost all stages of development and differentiation that require an increase in cell number. Although cell cycle phase has been associated with differentiation, the actual process of proliferation has not been considered as having a specific role. Here we exploit human embryonic stem cell-derived endodermal progenitors that we find are an in vitro model for the ventral foregut. These cells exhibit expansion-dependent increases in differentiation efficiency to pancreatic progenitors that are linked to organ-specific enhancer priming at the level of chromatin accessibility and the decommissioning of lineage-inappropriate enhancers. Our findings suggest that cell proliferation in embryonic development is about more than tissue expansion; it is required to ensure equilibration of gene regulatory networks allowing cells to become primed for future differentiation. Expansion of lineage-specific intermediates may therefore be an important step in achieving high-fidelity in vitro differentiation.',\n",
       "  '1f-Unnamed_1',\n",
       "  'downregulated primitive-streak markers (GSC, CER1 and LEFTY1) (Extended Data Fig. 4a and Supplementary Table **1a**)  Summary of dynamic ATAC peaks and their annotated genes classified by differential expression between VFGs and PE samples'),\n",
       " 'PMC10834515-41467_2024_45128_MOESM13_ESM.xlsx-Suppl_Fig_3b-AIZARANI_LIVER_C11_HEPATOCYTES_1': ('Nuclear to cytoplasmic transport is a druggable dependency in MYC-driven hepatocellular carcinoma',\n",
       "  'The MYC oncogene is often dysregulated in human cancer, including hepatocellular carcinoma (HCC). MYC is considered undruggable to date. Here, we comprehensively identify genes essential for survival of MYC high but not MYC low cells by a CRISPR/Cas9 genome-wide screen in a MYC-conditional HCC model. Our screen uncovers novel MYC synthetic lethal (MYC-SL) interactions and identifies most MYC-SL genes described previously. In particular, the screen reveals nucleocytoplasmic transport to be a MYC-SL interaction. We show that the majority of MYC-SL nucleocytoplasmic transport genes are upregulated in MYC high murine HCC and are associated with poor survival in HCC patients. Inhibiting Exportin-1 (XPO1) in vivo induces marked tumor regression in an autochthonous MYC-transgenic HCC model and inhibits tumor growth in HCC patient-derived xenografts. XPO1 expression is associated with poor prognosis only in HCC patients with high MYC activity. We infer that MYC may generally regulate and require altered expression of nucleocytoplasmic transport genes for tumorigenesis.',\n",
       "  'Suppl_Fig_3b-AIZARANI_LIVER_C11_HEPATOCYTES_1',\n",
       "  'None here'),\n",
       " 'PMC10156813-41467_2023_37951_MOESM5_ESM.xlsx-Gene_sets_hep_and_duc_identity-Hepatocyte_identity_C11_Aizarani_et_al_Nature_2019': ('Organoid models of fibrolamellar carcinoma mutations reveal hepatocyte transdifferentiation through cooperative BAP1 and PRKAR2A loss',\n",
       "  'Fibrolamellar carcinoma (FLC) is a lethal primary liver cancer, affecting young patients in absence of chronic liver disease. Molecular understanding of FLC tumorigenesis is limited, partly due to the scarcity of experimental models. Here, we CRISPR-engineer human hepatocyte organoids to recreate different FLC backgrounds, including the predominant genetic alteration, the DNAJB1-PRKACA fusion, as well as a recently reported background of FLC-like tumors, encompassing inactivating mutations of BAP1 and PRKAR2A. Phenotypic characterizations and comparisons with primary FLC tumor samples revealed mutant organoid-tumor similarities. All FLC mutations caused hepatocyte dedifferentiation, yet only combined loss of BAP1 and PRKAR2A resulted in hepatocyte transdifferentiation into liver ductal/progenitor-like cells that could exclusively grow in a ductal cell environment. BAP1-mutant hepatocytes represent primed cells attempting to proliferate in this cAMP-stimulating environment, but require concomitant PRKAR2A loss to overcome cell cycle arrest. In all analyses, DNAJB1-PRKACA fus organoids presented with milder phenotypes, suggesting differences between FLC genetic backgrounds, or for example the need for additional mutations, interactions with niche cells, or a different cell-of-origin. These engineered human organoid models facilitate the study of FLC.',\n",
       "  'Gene_sets_hep_and_duc_identity-Hepatocyte_identity_C11_Aizarani_et_al_Nature_2019',\n",
       "  'means. First, we assessed similarities by performing correlation analysis based on whole transcriptomes (Fig. **3**)  Supplementary Data 3'),\n",
       " 'PMC9643508-41467_2022_34413_MOESM5_ESM.xlsx-Supp_Data_11-Unnamed_26': ('Spatially resolved gene regulatory and disease-related vulnerability map of the adult Macaque cortex',\n",
       "  'Single cell approaches have increased our knowledge about the cell type composition of the non-human primate (NHP), but a detailed characterization of area-specific regulatory features remains outstanding. We generated single-cell transcriptomic and chromatin accessibility (single-cell ATAC) data of 358,237 cells from prefrontal cortex (PFC), primary motor cortex (M1) and primary visual cortex (V1) of adult female cynomolgus monkey brain, and integrated this dataset with Stereo-seq (spatial enhanced resolution omics-sequencing) of the corresponding cortical areas to assign topographic information to molecular states. We identified area-specific chromatin accessible sites and their targeted genes, including the cell type-specific transcriptional regulatory network associated with excitatory neurons heterogeneity. We reveal calcium ion transport and axon guidance genes related to specialized functions of PFC and M1, identified the similarities and differences between adult macaque and human oligodendrocyte trajectories, and mapped the genetic variants and gene perturbations of human diseases to NHP cortical cells. This resource establishes a transcriptomic and chromatin accessibility combinatory regulatory landscape at a single-cell and spatially resolved resolution in NHP cortex.',\n",
       "  'Supp_Data_11-Unnamed_26',\n",
       "  'also exhibit area-specificity in Stereo-seq data (42 genes in PFC, P = 5.13 × **10**)  Supplementary Data 10 - 12'),\n",
       " 'PMC7887593-ADVS-8-2003526-s004.xlsx-mature_enterocytes-Mature_enterocyte_gene_name_n_497': ('In Well-Treated Celiac Patients Low-Level Mucosal Inflammation Predicts Response to 14-day Gluten Challenge',\n",
       "  'In celiac disease (CeD), gluten activates adaptive immune cells that cause damage to the small intestinal mucosa. Histological evaluation of intestinal biopsies allows for grading of disease severity. CeD can effectively be treated with a life-long gluten-free diet. Gluten challenge of treated CeD patients is used to confirm diagnosis and to test drug efficacy in clinical trials, but patients respond with different magnitudes to the same gluten challenge. In this study of 19 well-treated CeD patients, proteome analysis of total tissue or isolated epithelial cell compartment from formalin-fixed paraffin embedded biopsies collected before and after 14-day gluten challenge demonstrates that patients with strong mucosal response to challenge have signs of ongoing tissue inflammation already before challenge. This low-level tissue inflammation at baseline is paralleled by increased gluten specific CD4+ T-cell frequencies in the gut and presence of a low-level blood inflammatory profile. Thus, apparently well-treated CeD is frequently not entirely quiescent, with presence of low-grade inflammation and antigluten immunity in the gut mucosa. Histology assessment alone appears insufficient to judge full recovery and gut mucosal healing of CeD patients. The findings raise a concern whether a seemingly proper gluten-free diet is able to curb gut inflammation in all CeD patients.',\n",
       "  'mature_enterocytes-Mature_enterocyte_gene_name_n_497',\n",
       "  'Supplemental Table 4  Click here for additional data file'),\n",
       " 'PMC3758364-NIHMS498302-supplement-2.xls-Sheet1-Unnamed_1': ('Independent specialization of the human and mouse X chromosomes for the male germ line',\n",
       "  \"We compared the human and mouse X chromosomes to systematically test Ohno's law, which states that the gene content of X chromosomes is conserved across placental mammals. First, we improved the accuracy of the human X-chromosome reference sequence through single-haplotype sequencing of ampliconic regions. The new sequence closed gaps in the reference sequence, corrected previously misassembled regions and identified new palindromic amplicons. Our subsequent analysis led us to conclude that the evolution of human and mouse X chromosomes was bimodal. In accord with Ohno's law, 94-95% of X-linked single-copy genes are shared by humans and mice; most are expressed in both sexes. Notably, most X-ampliconic genes are exceptions to Ohno's law: only 31% of human and 22% of mouse X-ampliconic genes had orthologs in the other species. X-ampliconic genes are expressed predominantly in testicular germ cells, and many were independently acquired since divergence from the common ancestor of humans and mice, specializing portions of their X chromosomes for sperm production.\",\n",
       "  'Sheet1-Unnamed_1',\n",
       "  'gene contents of the human and mouse X chromosomes. Contrary to Ohno’s Law, 18% (1**4**)'),\n",
       " 'PMC9553210-elife-76781-supp1.xlsx-cluster_10-Unnamed_0': ('A single cell transcriptional roadmap of human pacemaker cell differentiation',\n",
       "  \"Each heartbeat is triggered by the sinoatrial node (SAN), the primary pacemaker of the heart. Studies in animal models have revealed that pacemaker cells share a common progenitor with the (pro)epicardium, and that the pacemaker cardiomyocytes further diversify into 'transitional', 'tail', and 'head' subtypes. However, the underlying molecular mechanisms, especially of human pacemaker cell development, are poorly understood. Here, we performed single cell RNA sequencing (scRNA-seq) and trajectory inference on human induced pluripotent stem cells (hiPSCs) differentiating to SAN-like cardiomyocytes (SANCMs) to construct a roadmap of transcriptional changes and lineage decisions. In differentiated SANCM, we identified distinct clusters that closely resemble different subpopulations of the in vivo SAN. Moreover, the presence of a side population of proepicardial cells suggested their shared ontogeny with SANCM, as also reported in vivo. Our results demonstrate that the divergence of SANCM and proepicardial lineages is determined by WNT signaling. Furthermore, we uncovered roles for TGFβ and WNT signaling in the branching of transitional and head SANCM subtypes, respectively. These findings provide new insights into the molecular processes involved in human pacemaker cell differentiation, opening new avenues for complex disease modeling in vitro and inform approaches for cell therapy-based regeneration of the SAN.\",\n",
       "  'cluster_10-Unnamed_0',\n",
       "  '3—figure supplement 1A. List of genes differentially expressed in each cluster are provided in **Supplementary file 1**  Differentially expressed genes in sinoatrial node-like cardiomyocyte (SANCM) and ventricular-like cardiomyocyte (VCM) clusters identified at day 19.Related to Figure 3 and Figure 3—figure supplement 1'),\n",
       " 'PMC9840153-mmc3.xlsx-Sheet1-Microglia': ('Cell type characterization of spatiotemporal gene co-expression modules in Down syndrome brain',\n",
       "  \"Down syndrome (DS) is the most common genetic cause of intellectual disability and increases the risk of other brain-related dysfunctions, like seizures, early-onset Alzheimer's disease, and autism. To reveal the molecular profiles of DS-associated brain phenotypes, we performed a meta-data analysis of the developmental DS brain transcriptome at cell type and co-expression module levels. In the DS brain, astrocyte-, microglia-, and endothelial cell-associated genes show upregulated patterns, whereas neuron- and oligodendrocyte-associated genes show downregulated patterns. Weighted gene co-expression network analysis identified cell type-enriched co-expressed gene modules. We present eight representative cell-type modules for neurons, astrocytes, oligodendrocytes, and microglia. We classified the neuron modules into glutamatergic and GABAergic neurons and associated them with detailed subtypes. Cell type modules were interpreted by analyzing spatiotemporal expression patterns, functional annotations, and co-expression networks of the modules. This study provides insight into the mechanisms underlying brain abnormalities in DS and related disorders.\",\n",
       "  'Sheet1-Microglia',\n",
       "  'We divided neuron-enriched genes into glutamatergic neuron- and GABAergic neuron-enriched genes and their subtypes (**Table S2**)  Table S2. Cell type-enriched gene list, related to Figures 1 and 2'),\n",
       " 'PMC9780220-41598_2022_26217_MOESM2_ESM.xlsx-DEgenes_CTL-EV_FGF2-EV-Differentially_expressed_genes_when_comparing_FGF2-EV_to_CTL-EV': ('Fibroblast growth factor-2 bound to specific dermal fibroblast-derived extracellular vesicles is protected from degradation',\n",
       "  'Fibroblast growth factor-2 (FGF2) has multiple roles in cutaneous wound healing but its natural low stability prevents the development of its use in skin repair therapies. Here we show that FGF2 binds the outer surface of dermal fibroblast (DF)-derived extracellular vesicles (EVs) and this association protects FGF2 from fast degradation. EVs isolated from DF cultured in the presence of FGF2 harbor FGF2 on their surface and FGF2 can bind purified EVs in absence of cells. Remarkably, FGF2 binding to EVs is restricted to a specific subpopulation of EVs, which do not express CD63 and CD81 markers. Treatment of DF with FGF2-EVs activated ERK and STAT signaling pathways and increased cell proliferation and migration. Local injection of FGF2-EVs improved wound healing in mice. We further demonstrated that binding to EVs protects FGF2 from both thermal and proteolytic degradation, thus maintaining FGF2 function. This suggests that EVs protect soluble factors from degradation and increase their stability and half-life. These results reveal a novel aspect of EV function and suggest EVs as a potential tool for delivering FGF2 in skin healing therapies.',\n",
       "  'DEgenes_CTL-EV_FGF2-EV-Differentially_expressed_genes_when_comparing_FGF2-EV_to_CTL-EV',\n",
       "  'To investigate the effect of FGF**2**  Supplementary Information 2'),\n",
       " 'PMC5602299-elife-26876-supp2.xlsx-upregulated_in_Lin37---Unnamed_4': ('The DREAM complex through its subunit Lin37 cooperates with Rb to initiate quiescence',\n",
       "  'The retinoblastoma Rb protein is an important factor controlling the cell cycle. Yet, mammalian cells carrying Rb deletions are still able to arrest under growth-limiting conditions. The Rb-related proteins p107 and p130, which are components of the DREAM complex, had been suggested to be responsible for a continued ability to arrest by inhibiting E2f activity and by recruiting chromatin-modifying enzymes. Here, we show that p130 and p107 are not sufficient for DREAM-dependent repression. We identify the MuvB protein Lin37 as an essential factor for DREAM function. Cells not expressing Lin37 proliferate normally, but DREAM completely loses its ability to repress genes in G 0 /G 1 while all remaining subunits, including p130/p107, still bind to target gene promoters. Furthermore, cells lacking both Rb and Lin37 are incapable of exiting the cell cycle. Thus, Lin37 is an essential component of DREAM that cooperates with Rb to induce quiescence.',\n",
       "  'upregulated_in_Lin37---Unnamed_4',\n",
       "  'which led to the identification of 387 significantly up- and 76 significantly down-regulated genes (**Supplementary file 2**)  Transcriptome analysis of quiescent Lin37+/+ vs. Lin37−/− cells revel differentially expressed genes'),\n",
       " 'PMC6624841-NIHMS1518742-supplement-Supp_Tables_1-23.xlsx-ST_14_A-genes-Supplementary_Table_14_α-cell_enriched_genes': ('Diabetes relief in mice by glucose-sensing insulin-secreting human α-cells',\n",
       "  'Cell-identity switches, in which terminally differentiated cells are converted into different cell types when stressed, represent a widespread regenerative strategy in animals, yet they are poorly documented in mammals. In mice, some glucagon-producing pancreatic α-cells and somatostatin-producing δ-cells become insulin-expressing cells after the ablation of insulin-secreting β-cells, thus promoting diabetes recovery. Whether human islets also display this plasticity, especially in diabetic conditions, remains unknown. Here we show that islet non-β-cells, namely α-cells and pancreatic polypeptide (PPY)-producing γ-cells, obtained from deceased non-diabetic or diabetic human donors, can be lineage-traced and reprogrammed by the transcription factors PDX1 and MAFA to produce and secrete insulin in response to glucose. When transplanted into diabetic mice, converted human α-cells reverse diabetes and continue to produce insulin even after six months. Notably, insulin-producing α-cells maintain expression of α-cell markers, as seen by deep transcriptomic and proteomic characterization. These observations provide conceptual evidence and a molecular framework for a mechanistic understanding of in situ cell plasticity as a treatment for diabetes and other degenerative diseases.',\n",
       "  'ST_14_A-genes-Supplementary_Table_14_α-cell_enriched_genes',\n",
       "  'Data Fig. 1a-c, while δ-cell purification was unreliable with available antibodies (Extended Data Fig.1; **Supplementary Tables 1**)'),\n",
       " 'PMC7898412-13059_2021_2283_MOESM6_ESM.xlsx-Genic_XCI_gene_list-gene_ID_genecodeV34': ('Megabase-scale methylation phasing using nanopore long reads and NanoMethPhase',\n",
       "  'The ability of nanopore sequencing to simultaneously detect modified nucleotides while producing long reads makes it ideal for detecting and phasing allele-specific methylation. However, there is currently no complete software for detecting SNPs, phasing haplotypes, and mapping methylation to these from nanopore sequence data. Here, we present NanoMethPhase, a software tool to phase 5-methylcytosine from nanopore sequencing. We also present SNVoter, which can post-process nanopore SNV calls to improve accuracy in low coverage regions. Together, these tools can accurately detect allele-specific methylation genome-wide using nanopore sequence data with low coverage of about ten-fold redundancy.',\n",
       "  'Genic_XCI_gene_list-gene_ID_genecodeV34',\n",
       "  '371, 204, 75, and 71 inactive, variable, escapee, and unknown genes, respectively (Additional file **6**)  Additional file 6. List of genes studied by Carrel and Willard, Cotton et al., and Tukiainen et al. for genic XCI'),\n",
       " 'PMC8160578-JCB_202009147_TableS3.xlsx-CDK4_6-dependent_genes-Gene': ('LRR1-mediated replisome disassembly promotes DNA replication by recycling replisome components',\n",
       "  'After two converging DNA replication forks meet, active replisomes are disassembled and unloaded from chromatin. A key process in replisome disassembly is the unloading of CMG helicases (CDC45-MCM-GINS), which is initiated in Caenorhabditis elegans and Xenopus laevis by the E3 ubiquitin ligase CRL2LRR1. Here, we show that human cells lacking LRR1 fail to unload CMG helicases and accumulate increasing amounts of chromatin-bound replisome components as cells progress through S phase. Markedly, we demonstrate that the failure to disassemble replisomes reduces the rate of DNA replication increasingly throughout S phase by sequestering rate-limiting replisome components on chromatin and blocking their recycling. Continued binding of CMG helicases to chromatin during G2 phase blocks mitosis by activating an ATR-mediated G2/M checkpoint. Finally, we provide evidence that LRR1 is an essential gene for human cell division, suggesting that CRL2LRR1 enzyme activity is required for the proliferation of cancer cells and is thus a potential target for cancer therapy.',\n",
       "  'CDK4_6-dependent_genes-Gene',\n",
       "  'led to the identification of 459 CDK4/6–dependent genes (Fig. S1, A and B; see **Table S3**)  lists CDK4/6 activity-dependent genes and their manually annotated function  Click here for additional data file'),\n",
       " 'PMC5077964-oncotarget-07-31623-s004.xlsx-Cluster3_DW_Nutlin_Nutlin_Wip1i-Gene_Symbol': ('Cooperation of Nutlin-3a and a Wip1 inhibitor to induce p53 activity',\n",
       "  'Targeting the Mdm2 oncoprotein by drugs has the potential of re-establishing p53 function and tumor suppression. However, Mdm2-antagonizing drug candidates, e. g. Nutlin-3a, often fail to abolish cancer cell growth sustainably. To overcome these limitations, we inhibited Mdm2 and simultaneously a second negative regulator of p53, the phosphatase Wip1/PPM1D. When combining Nutlin-3a with the Wip1 inhibitor GSK2830371 in the treatment of p53-proficient but not p53-deficient cells, we observed enhanced phosphorylation (Ser 15) and acetylation (Lys 382) of p53, increased expression of p53 target gene products, and synergistic inhibition of cell proliferation. Surprisingly, when testing the two compounds individually, largely distinct sets of genes were induced, as revealed by deep sequencing analysis of RNA. In contrast, the combination of both drugs led to an expression signature that largely comprised that of Nutlin-3a alone. Moreover, the combination of drugs, or the combination of Nutlin-3a with Wip1-depletion by siRNA, activated p53-responsive genes to a greater extent than either of the compounds alone. Simultaneous inhibition of Mdm2 and Wip1 enhanced cell senescence and G2/M accumulation. Taken together, the inhibition of Wip1 might fortify p53-mediated tumor suppression by Mdm2 antagonists.',\n",
       "  'Cluster3_DW_Nutlin_Nutlin_Wip1i-Gene_Symbol',\n",
       "  'None here'),\n",
       " 'PMC6707279-41598_2019_48243_MOESM2_ESM.xls-coexpression_AK8_mRNA_expressio-Gene_Symbol': ('A co-expressed gene status of adenylate kinase 1/4 reveals prognostic gene signature associated with prognosis and sensitivity to EGFR targeted therapy in lung adenocarcinoma',\n",
       "  'Cancer cells utilize altered bioenergetics to fuel uncontrolled proliferation and progression. At the core of bioenergetics, adenine nucleotides are the building blocks for nucleotide synthesis, energy transfer and diverse metabolic processes. Adenylate kinases (AK) are ubiquitous phosphotransferases that catalyze the conversion of adenine nucleotides and regulate the homeostasis of nucleotide ratios within cellular compartments. Recently, different isoforms of AK have been shown to induce metabolic reprograming in cancer and were identified as biomarkers for predicting disease progression. Here we aim to systemically analyze the impact of all AK-associated gene signatures on lung adenocarcinoma patient survival and decipher the value for therapeutic interventions. By analyzing TCGA Lung Adenocarcinoma (LUAD) RNA Seq data, we found gene signatures from AK4 and AK1 have higher percentage of prognostic genes compared to other AK-gene signatures. A 118-gene signature was identified from consensus gene expression in AK1 and AK4 prognostic gene signatures. Immunohistochemistry (IHC) analyses in 140 lung adenocarcinoma patients showed overexpression of AK4 significantly correlated with worse overall survival (P = 0.001) whereas overexpression of AK1 significantly associated with good prognosis (P = 0.009). Furthermore, reduced AK4 expression by shRNA reduced the EGFR protein expression in EGFR mutation cells. The inhibition of AK4-AK1 signal might provide a potential target for synergistic effect in target therapy in lung cancer patients.',\n",
       "  'coexpression_AK8_mRNA_expressio-Gene_Symbol',\n",
       "  '(LUAD) dataset and identified unique signatures for each AK (Fig. 1a and Supplementary Table **S1**)  Table S1'),\n",
       " 'PMC4859332-mmc1.xlsx-murine_SIG-SP_RPM': ('Comparative genomics analysis of mononuclear phagocyte subsets confirms homology between lymphoid tissue-resident and dermal XCR1(+) DCs in mouse and human and distinguishes them from Langerhans cells',\n",
       "  'Dendritic cells (DC) are mononuclear phagocytes which exhibit a branching (dendritic) morphology and excel at naïve T cell activation. DC encompass several subsets initially identified by their expression of cell surface molecules and later shown to possess distinct functions. DC subset differentiation is orchestrated by transcription factors, growth factors and cytokines. Identifying DC subsets is challenging as very few cell surface molecules are uniquely expressed on any one of these cell populations. There is no standard consensus to identify mononuclear phagocyte subsets; varying antigens are employed depending on the tissue and animal species studied and between laboratories. This has led to confusion in how to accurately define and classify DCs across tissues and between species. Here we report a comparative genomics strategy that enables universal definition of DC and other mononuclear phagocyte subsets across species. We performed a meta-analysis of several public datasets of human and mouse mononuclear phagocyte subsets isolated from blood, spleen, skin or cutaneous lymph nodes, including by using a novel and user friendly software, BubbleGUM, which generates and integrates gene signatures for high throughput gene set enrichment analysis. This analysis demonstrates the equivalence between human and mouse skin XCR1(+) DCs, and between mouse and human Langerhans cells.',\n",
       "  'murine_SIG-SP_RPM',\n",
       "  'Supplementary material'),\n",
       " 'PMC4586137-pone.0138913.s002.xlsx-Sheet1-Unnamed_6': ('Identification of Circulating Biomarker Candidates for Hepatocellular Carcinoma (HCC): An Integrated Prioritization Approach',\n",
       "  \"Hepatocellular carcinoma (HCC) is the world's third most widespread cancer. Currently available circulating biomarkers for this silently progressing malignancy are not sufficiently specific and sensitive to meet all clinical needs. There is an imminent and pressing need for the identification of novel circulating biomarkers to increase disease-free survival rate. In order to facilitate the selection of the most promising circulating protein biomarkers, we attempted to define an objective method likely to have a significant impact on the analysis of vast data generated from cutting-edge technologies. Current study exploits data available in seven publicly accessible gene and protein databases, unveiling 731 liver-specific proteins through initial enrichment analysis. Verification of expression profiles followed by integration of proteomic datasets, enriched for the cancer secretome, filtered out 20 proteins including 6 previously characterized circulating HCC biomarkers. Finally, interactome analysis of these proteins with midkine (MDK), dickkopf-1 (DKK-1), current standard HCC biomarker alpha-fetoprotein (AFP), its interacting partners in conjunction with HCC-specific circulating and liver deregulated miRNAs target filtration highlighted seven novel statistically significant putative biomarkers including complement component 8, alpha (C8A), mannose binding lectin (MBL2), antithrombin III (SERPINC1), 11β-hydroxysteroid dehydrogenase type 1 (HSD11B1), alcohol dehydrogenase 6 (ADH6), beta-ureidopropionase (UPB1) and cytochrome P450, family 2, subfamily A, polypeptide 6 (CYP2A6). Our proposed methodology provides a swift assortment process for biomarker prioritization that eventually reduces the economic burden of experimental evaluation. Further dedicated validation studies of potential putative biomarkers on HCC patient blood samples are warranted. We hope that the use of such integrative secretome, interactome and miRNAs target filtration approach will accelerate the selection of high-priority biomarkers for other diseases as well, that are more amenable to downstream clinical validation experiments.\",\n",
       "  'Sheet1-Unnamed_6',\n",
       "  '(Table 1). A complete list of proteins identified by each database is presented in **S2 Table**  List of liver-specific proteins identified by seven renowned databases.Results of each database have been represented in separate columns. Proteins declared to be liver-specific in more than 2 databases were selected for further analysis.(XLSX)  Click here for additional data file'),\n",
       " 'PMC10796008-pone.0291995.s003.xlsx-common_up_genes_AD_v_models-M19A_apoe3_w_TBI': (\"Large-scale gene expression changes in APP/PSEN1 and GFAP mutation models exhibit high congruence with Alzheimer's disease\",\n",
       "  \"Alzheimer's disease (AD) is a complex neurodegenerative disorder with both genetic and non-genetic causes. Animal research models are available for a multitude of diseases and conditions affecting the central nervous system (CNS), and large-scale CNS gene expression data exist for many of these. Although there are several models specifically for AD, each recapitulates different aspects of the human disease. In this study we evaluate over 500 animal models to identify those with CNS gene expression patterns matching human AD datasets. Approaches included a hypergeometric based scoring system that rewards congruent gene expression patterns but penalizes discordant gene expression patterns. The top two models identified were APP/PS1 transgenic mice expressing mutant APP and PSEN1, and mice carrying a GFAP mutation that is causative of Alexander disease, a primary disorder of astrocytes in the CNS. The APP/PS1 and GFAP models both matched over 500 genes moving in the same direction as in human AD, and both had elevated GFAP expression and were highly congruent with one another. Also scoring highly were the 5XFAD model (with five mutations in APP and PSEN1) and mice carrying CK-p25, APP, and MAPT mutations. Animals with the APOE3 and 4 mutations combined with traumatic brain injury ranked highly. Bulbectomized rats scored high, suggesting anosmia could be causative of AD-like gene expression. Other matching models included the SOD1G93A strain and knockouts for SNORD116 (Prader-Willi mutation), GRID2, INSM1, XBP1, and CSTB. Many top models demonstrated increased expression of GFAP, and results were similar across multiple human AD datasets. Heatmap and Uniform Manifold Approximation Plot results were consistent with hypergeometric ranking. Finally, some gene manipulation models, including for TYROBP and ATG7, were identified with reversed AD patterns, suggesting possible neuroprotective effects. This study provides insight for the pathobiology of AD and the potential utility of available animal models.\",\n",
       "  'common_up_genes_AD_v_models-M19A_apoe3_w_TBI',\n",
       "  'None here'),\n",
       " 'PMC5063615-supp_E16-05-0268v1_mc-E16-05-0268-s10.xlsx-Depleted_transcripts-Unnamed_12': ('Transcriptome analysis of tetraploid cells identifies cyclin D2 as a facilitator of adaptation to genome doubling in the presence of p53',\n",
       "  'Tetraploidization, or genome doubling, is a prominent event in tumorigenesis, primarily because cell division in polyploid cells is error-prone and produces aneuploid cells. This study investigates changes in gene expression evoked in acute and adapted tetraploid cells and their effect on cell-cycle progression. Acute polyploidy was generated by knockdown of the essential regulator of cytokinesis anillin, which resulted in cytokinesis failure and formation of binucleate cells, or by chemical inhibition of Aurora kinases, causing abnormal mitotic exit with formation of single cells with aberrant nuclear morphology. Transcriptome analysis of these acute tetraploid cells revealed common signatures of activation of the tumor-suppressor protein p53. Suppression of proliferation in these cells was dependent on p53 and its transcriptional target, CDK inhibitor p21. Rare proliferating tetraploid cells can emerge from acute polyploid populations. Gene expression analysis of single cell-derived, adapted tetraploid clones showed up-regulation of several p53 target genes and cyclin D2, the activator of CDK4/6/2. Overexpression of cyclin D2 in diploid cells strongly potentiated the ability to proliferate with increased DNA content despite the presence of functional p53. These results indicate that p53-mediated suppression of proliferation of polyploid cells can be averted by increased levels of oncogenes such as cyclin D2, elucidating a possible route for tetraploidy-mediated genomic instability in carcinogenesis.',\n",
       "  'Depleted_transcripts-Unnamed_12',\n",
       "  'Supplemental Materials'),\n",
       " 'PMC10756545-pone.0296454.s002.xlsx-5_Tumour_modules-T2': ('Gene expression signatures of stepwise progression of Hepatocellular Carcinoma',\n",
       "  'The molecular pathogenesis of Hepatocellular Carcinoma (HCC) is a complex process progressing from premalignant stages to cancer in a stepwise manner. Mostly, HCC is detected at advanced stages, leading to high mortality rates. Hence, characterising the molecular underpinnings of HCC from normal to cancer state through precancerous state may help in early detection and improve its prognosis and treatment. In this work, we analysed the transcriptomic profile of tumour and premalignant samples from HCC or chronic liver disease patients, who had undergone either total or partial hepatectomy. The normal samples from patients with metastatic cancer/polycystic liver disease/ cholangiocarcinoma were also included. A gene co-expression network approach was applied to identify hierarchical changes: modules, pathways, and genes related to different trajectories of HCC and patient survival. Our analysis shows that the progression from premalignant conditions to tumour is accompanied by differences in the downregulation of genes associated with HNF4A activity and the immune system and upregulation of cell cycle genes, bringing about variability in patient outcomes. However, an increase in immune and cell cycle activity is observed in premalignant samples. Interestingly, co-expression modules and genes from premalignant stages are associated with survival. THBD, a classical marker for dendritic cells, is a predictor of survival at the premalignant stage. Further, genes linked to microtubules, kinetochores, and centromere are altered in both premalignant and tumour conditions and are associated with survival. Our analysis revealed a three-way molecular axis of liver function, immune pathways, and cell cycle driving HCC pathogenesis.',\n",
       "  '5_Tumour_modules-T2',\n",
       "  '**S1 Dataset**  List of genes.Survival genes based on modules from tumour samples, and normal and premalignant samples; List of genes in each module; HIR signature; Cell cycle genes.(XLSX)  Click here for additional data file'),\n",
       " 'PMC7066904-aging-12-102817-s005..xlsx-StromalDiff-gene': ('Identification of prognostic immune-related genes in the tumor microenvironment of endometrial cancer',\n",
       "  'Endometrial cancer (EC) is one of the most common gynecologic malignancies. To identify potential prognostic biomarkers for EC, we analyzed the relationship between the EC tumor microenvironment and gene expression profiles. Using the ESTIMATE R tool, we found that immune and stromal scores correlated with clinical data and the prognosis of EC patients. Based on the immune and stromal scores, 387 intersection differentially expressed genes were identified. Eight immune-related genes were then identified using two machine learning algorithms. Functional enrichment analysis revealed that these genes were mainly associated with T cell activation and response. Kaplan-Meier survival analysis showed that expression of TMEM150B, CACNA2D2, TRPM5, NOL4, CTSW, and SIGLEC1 significantly correlated with overall survival times of EC patients. In addition, using the TIMER algorithm, we found that expression of TMEM150B, SIGLEC1, and CTSW correlated positively with the tumor infiltration levels of B cells, CD8+ T cells, CD4+ T cells, macrophages, and dendritic cells. These findings indicate that the composition of the tumor microenvironment affects the clinical outcomes of EC patients, and suggests that it may provide a basis for development of novel prognostic biomarkers and immunotherapies for EC patients.',\n",
       "  'StromalDiff-gene',\n",
       "  'identify the immune-related and stromal-related genes, differential analysis by using “limma” package was performed (**Supplementary Table 3**)'),\n",
       " 'PMC9171790-pnas.2115709119.sd02.xlsx-Sheet1-Dataset_S2_833_EVT-upregulated_gene_set': ('Induction of human trophoblast stem-like cells from primed pluripotent stem cells',\n",
       "  'The placenta is a transient but important multifunctional organ crucial for healthy pregnancy for both mother and fetus. Nevertheless, limited access to human placenta samples and the paucity of a proper in vitro model system have hampered our understanding of the mechanisms underlying early human placental development and placenta-associated pregnancy complications. To overcome these constraints, we established a simple procedure with a short-term treatment of bone morphogenetic protein 4 (BMP4) in trophoblast stem cell culture medium (TSCM) to convert human primed pluripotent stem cells (PSCs) to trophoblast stem-like cells (TSLCs). These TSLCs show not only morphology and global gene expression profiles comparable to bona fide human trophoblast stem cells (TSCs) but also long-term self-renewal capacity with bipotency that allows the cells to differentiate into functional extravillous trophoblasts (EVT) and syncytiotrophoblasts (ST). These indicate that TSLCs are equivalent to genuine human TSCs. Our data suggest a straightforward approach to make human TSCs directly from preexisting primed PSCs and provide a valuable opportunity to study human placenta development and pathology from patients with placenta-related diseases.',\n",
       "  'Sheet1-Dataset_S2_833_EVT-upregulated_gene_set',\n",
       "  'None here'),\n",
       " 'PMC9533278-41380_2022_1786_MOESM11_ESM.xlsx-Gene_Signatures-Celltype_Signatures_used_to_score_individual_cells_as_seen_in_Figure_5': ('SARS-CoV-2 promotes microglial synapse elimination in human brain organoids',\n",
       "  'Neuropsychiatric manifestations are common in both the acute and post-acute phase of SARS-CoV-2 infection, but the mechanisms of these effects are unknown. In a newly established brain organoid model with innately developing microglia, we demonstrate that SARS-CoV-2 infection initiate neuronal cell death and cause a loss of post-synaptic termini. Despite limited neurotropism and a decelerating viral replication, we observe a threefold increase in microglial engulfment of postsynaptic termini after SARS-CoV-2 exposure. We define the microglial responses to SARS-CoV-2 infection by single cell transcriptomic profiling and observe an upregulation of interferon-responsive genes as well as genes promoting migration and synapse engulfment. To a large extent, SARS-CoV-2 exposed microglia adopt a transcriptomic profile overlapping with neurodegenerative disorders that display an early synapse loss as well as an increased incident risk after a SARS-CoV-2 infection. Our results reveal that brain organoids infected with SARS-CoV-2 display disruption in circuit integrity via microglia-mediated synapse elimination and identifies a potential novel mechanism contributing to cognitive impairments in patients recovering from COVID-19.',\n",
       "  'Gene_Signatures-Celltype_Signatures_used_to_score_individual_cells_as_seen_in_Figure_5',\n",
       "  'organoids exposed to an estimated MOI of 0.3, we then decreased the MOI to 0.**1**  Supplementary Table 1'),\n",
       " 'PMC9296597-41421_2022_415_MOESM4_ESM.xlsx-Endothelium_VCAM1--gene': ('Single-cell multiomics analysis reveals regulatory programs in clear cell renal cell carcinoma',\n",
       "  'The clear cell renal cell carcinoma (ccRCC) microenvironment consists of many different cell types and structural components that play critical roles in cancer progression and drug resistance, but the cellular architecture and underlying gene regulatory features of ccRCC have not been fully characterized. Here, we applied single-cell RNA sequencing (scRNA-seq) and single-cell assay for transposase-accessible chromatin sequencing (scATAC-seq) to generate transcriptional and epigenomic landscapes of ccRCC. We identified tumor cell-specific regulatory programs mediated by four key transcription factors (TFs) (HOXC5, VENTX, ISL1, and OTP), and these TFs have prognostic significance in The Cancer Genome Atlas (TCGA) database. Targeting these TFs via short hairpin RNAs (shRNAs) or small molecule inhibitors decreased tumor cell proliferation. We next performed an integrative analysis of chromatin accessibility and gene expression for CD8 + T cells and macrophages to reveal the different regulatory elements in their subgroups. Furthermore, we delineated the intercellular communications mediated by ligand-receptor interactions within the tumor microenvironment. Taken together, our multiomics approach further clarifies the cellular heterogeneity of ccRCC and identifies potential therapeutic targets.',\n",
       "  'Endothelium_VCAM1--gene',\n",
       "  '15 cell types in the scRNA-seq dataset (Fig. 1b, c and Supplementary Tables S2, **S3**)  Supplementary Table S3'),\n",
       " 'PMC9157497-Table_2.xlsx-ADA2_related_genes-Unnamed_0': ('Distinct Roles of Adenosine Deaminase Isoenzymes ADA1 and ADA2: A Pan-Cancer Analysis',\n",
       "  'Objective: Adenosine deaminase (ADA) plays an important role in immune response, which includes two isoenzymes: ADA1 and ADA2. This study aims to explore the roles of ADA1 and ADA2 in cancers. Methods: Human Protein Atlas (HPA) and Gene Expression Profiling Interactive Analysis (GEPIA2) databases were used to analyze the mRNA expression of ADA1 and ADA2 in human normal cells and tumor tissues. The enzyme assay was used to detect the ADA1 and ADA2 activities in serum from cancer patients. The Kaplan-Meier (KM) plotter was used to analyze the prognostic value of ADA1 and ADA2. TIMER2.0 was used to explore how ADA1 and ADA2 correlate with immune infiltration and immune checkpoints. cBioPortal database was used to investigate the mutations of ADA1 and ADA2. LinkedOmics was used to screen the ADA1 and ADA2 expression-related genes. Results: ADA1 was significantly increased in several tumor tissues, including cholangiocarcinoma (CHOL), lymphoid neoplasm diffuse large B-cell lymphoma (DLBC), head and neck squamous cell carcinoma (HNSC), kidney renal clear cell carcinoma (KIRC), ovarian serous cystadenocarcinoma (OV), pancreatic adenocarcinoma (PAAD), thymoma (THYM), and uterine carcinosarcoma (UCS). ADA2 expression was significantly increased in esophageal carcinoma (ESCA), glioblastoma multiforme (GBM), acute myeloid leukemia (LAML), OV, PAAD, skin cutaneous melanoma (SKCM), and stomach adenocarcinoma (STAD). There were no significant changes in serum ADA1 activities in most cancers, while serum ADA2 activities were increased in most cancers. For prognosis, high ADA1 expression was associated with the poor survival in several cancers, including esophageal squamous cell carcinoma (ESCC), HNSC, KIRC, kidney renal papillary cell carcinoma (KIRP), liver hepatocellular carcinoma (LIHC), lung adenocarcinoma (LUAD), and uterine corpus endometrial carcinoma (UCEC). However, high ADA2 expression showed a favorable prognosis in breast invasive carcinoma (BRCA), cervical squamous cell carcinoma and endocervical adenocarcinoma (CESC), HNSC, KIRC, KIRP, LUAD, OV, PAAD, sarcoma, and THYM. ADA1 showed a moderate positive correlation with multiple infiltrating immune cells in most cancers. ADA2 was positively correlated with B cells, CD8 T cells, monocytes/macrophages, and dendritic cells (DCs) and was strongly negatively correlated with myeloid-derived suppressor cells. Function analysis showed that ADA1 expression-related genes were mainly enriched in cell division biological progression. However, ADA2-related genes were mainly associated with immune response. Conclusion: As isoenzymes, ADA1 and ADA2 showed opposite prognostic values and different correlative patterns with immune infiltrating. These data demonstrated the distinct roles of ADA1 and ADA2 in cancer. ADA2 might act as a protective factor in cancer.',\n",
       "  'ADA2_related_genes-Unnamed_0',\n",
       "  'The detailed information of the clinical sample is listed in **Supplementary Table 2**  The list of ADA1 and ADA2 expression related genes and function enrichment of these genes  Click here for additional data file'),\n",
       " 'PMC5990591-table_1.xlsx-de_KO3V_vs_KO3M_n_506-Gene_Symbol': ('Influenza A Virus Facilitates Its Infectivity by Activating p53 to Inhibit the Expression of Interferon-Induced Transmembrane Proteins',\n",
       "  'Human influenza virus (IAV) are among the most common pathogens to cause human respiratory infections. A better understanding on interplay between IAV and host factors may provide clues for disease prevention and control. While many viruses are known to downregulate p53 upon entering the cell to reduce the innate host antiviral response, IAV infection is unusual in that it activates p53. However, it has not been clear whether this process has proviral or antiviral effects. In this study, using human isogenic p53 wild-type and p53null A549 cells generated from the CRISPR/Cas9 technology, we observed that p53null cells exhibit significantly reduced viral propagation when infected with influenza A virus (strain A/Puerto Rico/8/1934 H1N1). Genome-wide microarray analysis revealed that p53 regulates the expression of a large set of interferon-inducible genes, among which the interferon-induced transmembrane family members IFITM1, IFITM2, and IFITM3 were most significantly downregulated by the expression of p53. Knockdown of interferon-induced transmembrane proteins (IFITMs) by short interfering RNAs enhanced influenza virus infectivity in p53null A549 cells, while overexpressed IFITMs in A549 cells blocked virus entry. Intriguingly, regulation of IFITMs by p53 is independent of its transcriptional activity, as the p53 short isoform Δ40p53 recapitulates IFITM regulation. Taken together, these data reveal that p53 activation by IAV is an essential step in maintaining its infectivity. This novel association between human p53 and the broad spectrum antiviral proteins, the IFITMs, demonstrates a previous mechanism employed by influenza virus to enhance its propagation via p53 inhibition of IFITMs.',\n",
       "  'de_KO3V_vs_KO3M_n_506-Gene_Symbol',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC8902307-DataSheet3.CSV--m': ('Development and Validation of a Novel Stemness-Index-Related Long Noncoding RNA Signature for Breast Cancer Based on Weighted Gene Co-Expression Network Analysis',\n",
       "  'Background: Breast cancer (BC) is a major leading cause of woman deaths worldwide. Increasing evidence has revealed that stemness features are related to the prognosis and progression of tumors. Nevertheless, the roles of stemness-index-related long noncoding RNAs (lncRNAs) in BC remain unclear. Methods: Differentially expressed stemness-index-related lncRNAs between BC and normal samples in The Cancer Genome Atlas database were screened based on weighted gene co-expression network analysis and differential analysis. Univariate Cox and least absolute shrinkage and selection operator regression analyses were performed to identify prognostic lncRNAs and construct a stemness-index-related lncRNA signature. Time-dependent receiver operating characteristic curves were plotted to evaluate the predictive capability of the stemness-index-related lncRNA signature. Moreover, correlation analysis and functional enrichment analyses were conducted to investigate the stemness-index-related lncRNA signature-related biological function. Finally, a quantitative real-time polymerase chain reaction was used to detect the expression levels of lncRNAs. Results: A total of 73 differentially expressed stemness-index-related lncRNAs were identified. Next, FAM83H-AS1, HID1-AS1, HOXB-AS1, RP11-1070N10.3, RP11-1100L3.8, and RP11-696F12.1 were used to construct a stemness-index-related lncRNA signature, and receiver operating characteristic curves indicated that stemness-index-related lncRNA signature could predict the prognosis of BC well. Moreover, functional enrichment analysis suggested that differentially expressed genes between the high-risk group and low-risk group were mainly involved in immune-related biological processes and pathways. Furthermore, functional enrichment analysis of lncRNA-related protein-coding genes revealed that FAM83H-AS1, HID1-AS1, HOXB-AS1, RP11-1070N10.3, RP11-1100L3.8, and RP11-696F12.1 were associated with neuroactive ligand-receptor interaction, AMPK signaling pathway, PPAR signaling pathway, and cGMP-PKG signaling pathway. Finally, quantitative real-time polymerase chain reaction revealed that FAM83H-AS1, HID1-AS1, RP11-1100L3.8, and RP11-696F12.1 might be used as the potential diagnostic biomarkers of BC. Conclusion: The stemness-index-related lncRNA signature based on FAM83H-AS1, HID1-AS1, HOXB-AS1, RP11-1070N10.3, RP11-1100L3.8, and RP11-696F12.1 could be used as an independent predictor for the survival of BC, and FAM83H-AS1, HID1-AS1, RP11-1100L3.8, and RP11-696F12.1 might be used as the diagnostic markers of BC.',\n",
       "  'm',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC10652867-NIHMS1928228-supplement-2.xlsx-DEG_MEF2AKD_Fig1-Gene_ID': ('MEF2A suppresses stress responses that trigger DDX41-dependent IFN production',\n",
       "  'Cellular stress in the form of disrupted transcription, loss of organelle integrity, or damage to nucleic acids can elicit inflammatory responses by activating signaling cascades canonically tasked with controlling pathogen infections. These stressors must be kept in check to prevent unscheduled activation of interferon, which contributes to autoinflammation. This study examines the role of the transcription factor myocyte enhancing factor 2A (MEF2A) in setting the threshold of transcriptional stress responses to prevent R-loop accumulation. Increases in R-loops lead to the induction of interferon and inflammatory responses in a DEAD-box helicase 41 (DDX41)-, cyclic GMP-AMP synthase (cGAS)-, and stimulator of interferon genes (STING)-dependent manner. The loss of MEF2A results in the activation of ATM and RAD3-related (ATR) kinase, which is also necessary for the activation of STING. This study identifies the role of MEF2A in sustaining transcriptional homeostasis and highlights the role of ATR in positively regulating R-loop-associated inflammatory responses.',\n",
       "  'DEG_MEF2AKD_Fig1-Gene_ID',\n",
       "  'in MEF2A-targeted cells compared with cells transfected with non-targeting control (NC) dsiRNA (Figure 1D; **Data S1**)'),\n",
       " 'PMC9310525-jciinsight-7-159832-s066.xlsx-cl_9_proliferating_basal-Unnamed_0': ('IL-13-programmed airway tuft cells produce PGE2, which promotes CFTR-dependent mucociliary function',\n",
       "  'Chronic type 2 (T2) inflammatory diseases of the respiratory tract are characterized by mucus overproduction and disordered mucociliary function, which are largely attributed to the effects of IL-13 on common epithelial cell types (mucus secretory and ciliated cells). The role of rare cells in airway T2 inflammation is less clear, though tuft cells have been shown to be critical in the initiation of T2 immunity in the intestine. Using bulk and single-cell RNA sequencing of airway epithelium and mouse modeling, we found that IL-13 expanded and programmed airway tuft cells toward eicosanoid metabolism and that tuft cell deficiency led to a reduction in airway prostaglandin E2 (PGE2) concentration. Allergic airway epithelia bore a signature of PGE2 activation, and PGE2 activation led to cystic fibrosis transmembrane receptor-dependent ion and fluid secretion and accelerated mucociliary transport. These data reveal a role for tuft cells in regulating epithelial mucociliary function in the allergic airway.',\n",
       "  'cl_9_proliferating_basal-Unnamed_0',\n",
       "  'Supplemental table 3'),\n",
       " 'PMC10238473-41598_2023_36187_MOESM2_ESM.xlsx-Table_S2-Table_S2_Top_500_genes_with_enriched_expression_in_the_hepatocyte_fraction': ('Sample size calculation for a NanoString GeoMx spatial transcriptomics experiment to study predictors of fibrosis progression in non-alcoholic fatty liver disease',\n",
       "  'Sample size calculation for spatial transcriptomics is a novel and understudied research topic. Prior publications focused on powering spatial transcriptomics studies to detect specific cell populations or spatially variable expression patterns on tissue slides. However, power calculations for translational or clinical studies often relate to the difference between patient groups, and this is poorly described in the literature. Here, we present a stepwise process for sample size calculation to identify predictors of fibrosis progression in non-alcoholic fatty liver disease as a case study. We illustrate how to infer study hypothesis from prior bulk RNA-sequencing data, gather input requirements and perform a simulation study to estimate required sample size to evaluate gene expression differences between patients with stable fibrosis and fibrosis progressors with NanoString GeoMx Whole Transcriptome Atlas assay.',\n",
       "  'Table_S2-Table_S2_Top_500_genes_with_enriched_expression_in_the_hepatocyte_fraction',\n",
       "  'cell types (mesenchymal, immune etc.) and hepatocyte fraction was enriched in hepatocyte markers (Tables **S1**)  Supplementary Tables'),\n",
       " 'PMC10962129-JCMM-28-e18261-s002.xlsx-Sheet3-RP11-40C6_2': ('Depletion of CPNE7 sensitizes colorectal cancer to 5-fluorouracil by downregulating ATG9B expression',\n",
       "  'We aimed to explore the biological function of CPNE7 and determine the impact of CPNE7 on chemotherapy resistance in colorectal cancer (CRC) patients. According to the Gene Expression Profiling Interactive Analysis database and previously published data, CPNE7 was identified as a potential oncogene in CRC. RT-qPCR and Western blotting were performed to verify the expression of CPNE7. Chi-square test was used to evaluate the associations between CPNE7 and clinical features. Cell proliferation, colony formation, cell migration and invasion, cell cycle and apoptosis were assessed to determine the effects of CPNE7. Transcriptome sequencing was used to identify potential downstream regulatory genes, and gene set enrichment analysis was performed to investigate downstream pathways. The effect of CPNE7 on 5-fluorouracil chemosensitivity was verified by half maximal inhibitory concentration (IC50). Subcutaneous tumorigenesis assay was used to examine the role of CPNE7 in sensitivity of CRC to chemotherapy in vivo. Transmission electron microscopy was used to detect autophagosomes. CPNE7 was highly expressed in CRC tissues, and its expression was correlated with T stage and tumour site. Knockdown of CPNE7 inhibited the proliferation and colony formation of CRC cells and promoted apoptosis. Knockdown of CPNE7 suppressed the expression of ATG9B and enhanced the sensitivity of CRC cells to 5-fluorouracil in vitro and in vivo. Knockdown of CPNE7 reversed the induction of the autophagy pathway by rapamycin and reduced the number of autophagosomes. Depletion of CPNE7 attenuated the malignant proliferation of CRC cells and enhanced the chemosensitivity of CRC cells to 5-fluorouracil.',\n",
       "  'Sheet3-RP11-40C6_2',\n",
       "  'None here'),\n",
       " 'PMC7331728-41598_2020_66453_MOESM2_ESM.xlsx-All_tissue_gene_clusters-Lung': ('The transcriptional signature associated with human motile cilia',\n",
       "  'Cilia are complex microtubule-based organelles essential to a range of processes associated with embryogenesis and tissue homeostasis. Mutations in components of these organelles or those involved in their assembly may result in a diverse set of diseases collectively known as ciliopathies. Accordingly, many cilia-associated proteins have been described, while those distinguishing cilia subtypes are poorly defined. Here we set out to define genes associated with motile cilia in humans based on their transcriptional signature. To define the signature, we performed network deconvolution of transcriptomics data derived from tissues possessing motile ciliated cell populations. For each tissue, genes coexpressed with the motile cilia-associated transcriptional factor, FOXJ1, were identified. The consensus across tissues provided a transcriptional signature of 248 genes. To validate these, we examined the literature, databases (CilDB, CentrosomeDB, CiliaCarta and SysCilia), single cell RNA-Seq data, and the localisation of mRNA and proteins in motile ciliated cells. In the case of six poorly characterised signature genes, we performed new localisation experiments on ARMC3, EFCAB6, FAM183A, MYCBPAP, RIBC2 and VWA3A. In summary, we report a set of motile cilia-associated genes that helps shape our understanding of these complex cellular organelles.',\n",
       "  'All_tissue_gene_clusters-Lung',\n",
       "  'Supplementary Information2'),\n",
       " 'PMC5374077-jciinsight-2-91868-s002.xlsx-All_summary-Unnamed_9': ('M1-like monocytes are a major immunological determinant of severity in previously healthy adults with life-threatening influenza',\n",
       "  'In each influenza season, a distinct group of young, otherwise healthy individuals with no risk factors succumbs to life-threatening infection. To better understand the cause for this, we analyzed a broad range of immune responses in blood from a unique cohort of patients, comprising previously healthy individuals hospitalized with and without respiratory failure during one influenza season, and infected with one specific influenza A strain. This analysis was compared with similarly hospitalized influenza patients with known risk factors (total of n = 60 patients recruited). We found a sustained increase in a specific subset of proinflammatory monocytes, with high TNF-α expression and an M1-like phenotype (independent of viral titers), in these previously healthy patients with severe disease. The relationship between M1-like monocytes and immunopathology was strengthened using murine models of influenza, in which severe infection generated using different models (including the high-pathogenicity H5N1 strain) was also accompanied by high levels of circulating M1-like monocytes. Additionally, a raised M1/M2 macrophage ratio in the lungs was observed. These studies identify a specific subtype of monocytes as a modifiable immunological determinant of disease severity in this subgroup of severely ill, previously healthy patients, offering potential novel therapeutic avenues.',\n",
       "  'All_summary-Unnamed_9',\n",
       "  'Supplemental Table 7'),\n",
       " 'PMC9691389-1805216.f2.txt--gene_id': ('Integrated Analysis of mRNA and lncRNA Expression Profiles Reveals Regulatory Networks Associated with Decompensated Cirrhosis',\n",
       "  'The stage of decompensation is termed end-stage liver cirrhosis. Patients with decompensated cirrhosis (DCC) often have a variety of comorbidities that contribute to exacerbation of the disease and its high mortality rate. By comparing differential gene expression, transcriptomic analysis is useful for exploring relevant functional changes during disease progression. The purpose of this study was to identify differentially expressed long noncoding RNAs (lncRNAs) and mRNAs in patients with decompensated cirrhosis and to further explore the functions as well as interactions between lncRNAs and mRNAs. Four patients with decompensated cirrhosis and four controls with liver cirrhosis were recruited in this study. RNA was isolated from peripheral blood mononuclear cells, and RNA-seq was used for transcriptome analysis. The functions of differentially expressed mRNAs were revealed by Gene Ontology (GO) functional annotation and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analyses, and a regulatory network was also constructed. A total of 1046 differentially expressed mRNAs and 1175 lncRNAs were identified between the decompensated cirrhosis patients and cirrhosis controls. Functional enrichment analyses indicated enrichment of genes involved in pathways related to inflammation and cellular metabolic activities. In addition, the findings suggested that the phagosome/endosome/autophagy-lysosome pathway might play an important role in cirrhotic decompensation. In summary, this study identified differentially expressed mRNAs (DE-mRNAs) and DE-lncRNAs and predicted the biological processes and signaling pathways involved in cirrhotic decompensation, which might provide new ideas for further revealing the molecular mechanism of DCC pathogenesis.',\n",
       "  'gene_id',\n",
       "  'Table S2: the list of DE-mRNAs in PBMCs between DCC patients and LC controls  Click here for additional data file'),\n",
       " 'PMC9691389-1805216.f2.txt--gene_name': ('Integrated Analysis of mRNA and lncRNA Expression Profiles Reveals Regulatory Networks Associated with Decompensated Cirrhosis',\n",
       "  'The stage of decompensation is termed end-stage liver cirrhosis. Patients with decompensated cirrhosis (DCC) often have a variety of comorbidities that contribute to exacerbation of the disease and its high mortality rate. By comparing differential gene expression, transcriptomic analysis is useful for exploring relevant functional changes during disease progression. The purpose of this study was to identify differentially expressed long noncoding RNAs (lncRNAs) and mRNAs in patients with decompensated cirrhosis and to further explore the functions as well as interactions between lncRNAs and mRNAs. Four patients with decompensated cirrhosis and four controls with liver cirrhosis were recruited in this study. RNA was isolated from peripheral blood mononuclear cells, and RNA-seq was used for transcriptome analysis. The functions of differentially expressed mRNAs were revealed by Gene Ontology (GO) functional annotation and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analyses, and a regulatory network was also constructed. A total of 1046 differentially expressed mRNAs and 1175 lncRNAs were identified between the decompensated cirrhosis patients and cirrhosis controls. Functional enrichment analyses indicated enrichment of genes involved in pathways related to inflammation and cellular metabolic activities. In addition, the findings suggested that the phagosome/endosome/autophagy-lysosome pathway might play an important role in cirrhotic decompensation. In summary, this study identified differentially expressed mRNAs (DE-mRNAs) and DE-lncRNAs and predicted the biological processes and signaling pathways involved in cirrhotic decompensation, which might provide new ideas for further revealing the molecular mechanism of DCC pathogenesis.',\n",
       "  'gene_name',\n",
       "  'Table S2: the list of DE-mRNAs in PBMCs between DCC patients and LC controls  Click here for additional data file'),\n",
       " 'PMC7547155-pnas.2008004117.sd01.xls-S1J_lake_cell_genes-Oli': ('Convergent molecular, cellular, and cortical neuroimaging signatures of major depressive disorder',\n",
       "  'Major depressive disorder emerges from the complex interactions of biological systems that span genes and molecules through cells, networks, and behavior. Establishing how neurobiological processes coalesce to contribute to depression requires a multiscale approach, encompassing measures of brain structure and function as well as genetic and cell-specific transcriptional data. Here, we examine anatomical (cortical thickness) and functional (functional variability, global brain connectivity) correlates of depression and negative affect across three population-imaging datasets: UK Biobank, Brain Genomics Superstruct Project, and Enhancing NeuroImaging through Meta Analysis (ENIGMA; combined n ≥ 23,723). Integrative analyses incorporate measures of cortical gene expression, postmortem patient transcriptional data, depression genome-wide association study (GWAS), and single-cell gene transcription. Neuroimaging correlates of depression and negative affect were consistent across three independent datasets. Linking ex vivo gene down-regulation with in vivo neuroimaging, we find that transcriptional correlates of depression imaging phenotypes track gene down-regulation in postmortem cortical samples of patients with depression. Integrated analysis of single-cell and Allen Human Brain Atlas expression data reveal somatostatin interneurons and astrocytes to be consistent cell associates of depression, through both in vivo imaging and ex vivo cortical gene dysregulation. Providing converging evidence for these observations, GWAS-derived polygenic risk for depression was enriched for genes expressed in interneurons, but not glia. Underscoring the translational potential of multiscale approaches, the transcriptional correlates of depression-linked brain function and structure were enriched for disorder-relevant molecular pathways. These findings bridge levels to connect specific genes, cell classes, and biological pathways to in vivo imaging correlates of depression.',\n",
       "  'S1J_lake_cell_genes-Oli',\n",
       "  'None here'),\n",
       " 'PMC9399107-41467_2022_32627_MOESM6_ESM.xlsx-12_main_cell-types-gene_name_T_cells_NK_cells': ('scRNA-seq of gastric tumor shows complex intercellular interaction with an alternative T cell exhaustion trajectory',\n",
       "  'The tumor microenvironment (TME) in gastric cancer (GC) has been shown to be important for tumor control but the specific characteristics for GC are not fully appreciated. We generated an atlas of 166,533 cells from 10 GC patients with matched paratumor tissues and blood. Our results show tumor-associated stromal cells (TASCs) have upregulated activity of Wnt signaling and angiogenesis, and are negatively correlated with survival. Tumor-associated macrophages and LAMP3 + DCs are involved in mediating T cell activity and form intercellular interaction hubs with TASCs. Clonotype and trajectory analysis demonstrates that Tc17 (IL-17 + CD8 + T cells) originate from tissue-resident memory T cells and can subsequently differentiate into exhausted T cells, suggesting an alternative pathway for T cell exhaustion. Our results indicate that IL17 + cells may promote tumor progression through IL17, IL22, and IL26 signaling, highlighting the possibility of targeting IL17 + cells and associated signaling pathways as a therapeutic strategy to treat GC.',\n",
       "  '12_main_cell-types-gene_name_T_cells_NK_cells',\n",
       "  'PCA, UMAP, and Leiden clustering independently for further analysis. For these analyses, the first **3**  Supplementary Data 3'),\n",
       " 'PMC9124307-ACEL-21-e13606-s002.xlsx-Table_S3-Unnamed_2': ('Transcriptional landscape of human microglia implicates age, sex, and APOE-related immunometabolic pathway perturbations',\n",
       "  'Microglia have fundamental roles in health and disease; however, effects of age, sex, and genetic factors on human microglia have not been fully explored. We applied bulk and single-cell approaches to comprehensively characterize human microglia transcriptomes and their associations with age, sex, and APOE. We identified a novel microglial signature, characterized its expression in bulk tissue and single-cell microglia transcriptomes. We discovered microglial co-expression network modules associated with age, sex, and APOE-ε4 that are enriched for lipid and carbohydrate metabolism genes. Integrated analyses of modules with single-cell transcriptomes revealed significant overlap between age-associated module genes and both pro-inflammatory and disease-associated microglial clusters. These modules and clusters harbor known neurodegenerative disease genes including APOE, PLCG2, and BIN1. Meta-analyses with published bulk and single-cell microglial datasets further supported our findings. Thus, these data represent a well-characterized human microglial transcriptome resource and highlight age, sex, and APOE-related microglial immunometabolism perturbations with potential relevance in neurodegeneration.',\n",
       "  'Table_S3-Unnamed_2',\n",
       "  'our bulk microglia and a q‐value threshold of 0.05, we identified 1971 genes (Table **S1–S6**)  Table S1‐S20  Click here for additional data file'),\n",
       " 'PMC9018830-41467_2022_29675_MOESM7_ESM.xlsx-Gene_Modules_Oetjen-EARLY_ERYTROCYTE': ('Single cell analyses identify a highly regenerative and homogenous human CD34+ hematopoietic stem cell population',\n",
       "  'The heterogeneous nature of human CD34 + hematopoietic stem cells (HSCs) has hampered our understanding of the cellular and molecular trajectories that HSCs navigate during lineage commitment. Using various platforms including single cell RNA-sequencing and extensive xenotransplantation, we have uncovered an uncharacterized human CD34 + HSC population. These CD34 + EPCR + (CD38/CD45RA) - (simply as EPCR + ) HSCs have a high repopulating and self-renewal abilities, reaching a stem cell frequency of ~1 in 3 cells, the highest described to date. Their unique transcriptomic wiring in which many gene modules associated with differentiated cell lineages confers their multilineage lineage output both in vivo and in vitro. At the single cell level, EPCR + HSCs are the most transcriptomically and functionally homogenous human HSC population defined to date and can also be easily identified in post-natal tissues. Therefore, this EPCR + population not only offers a high human HSC resolution but also a well-structured human hematopoietic hierarchical organization at the most primitive level.',\n",
       "  'Gene_Modules_Oetjen-EARLY_ERYTROCYTE',\n",
       "  'EPCR+ cells and the immediate downstream CD90+EPCR- and MPP progeny fractions. This analysis revealed 5**4**  Supplementary Data 4'),\n",
       " 'PMC8957021-mmc9.xlsx-PC2-Unnamed_0': ('Single-cell temporal analysis of natural dengue infection reveals skin-homing lymphocyte expansion one day before defervescence',\n",
       "  'Effective clinical management of acute dengue virus (DENV) infection relies on the timing of suitable treatments during the disease progression. We analyzed single-cell transcriptomic profiles of the peripheral blood mononuclear cell samples from two DENV patients, collected daily during acute phase and also at convalescence. Key immune cell types demonstrated different dynamic responses over the course of the infection. On the day before defervescence (Day -1), we observed the peak expression of several prominent genes in the adaptive immunological pathways. We also characterized unique effector T cell clusters that expressed skin-homing signature genes at Day -1, whereas upregulation of skin and gut homing genes was also observed in plasma cells and plasmablasts during the febrile period. This work provides an overview of unique molecular dynamics that signify the entry of the critical phase, and the findings could improve the patient management of DENV infection.',\n",
       "  'PC2-Unnamed_0',\n",
       "  'cells/PCs/PBs (Figures 2C and S3, see full lists of HVGs in Tables S6, S7, **S8**)  Table S8. Average expression of the top 500 genes in Principal Component (PC) 1 and PC2 in T cells, related to Figure 2C, S3 STAR Methods'),\n",
       " 'PMC9124307-ACEL-21-e13606-s002.xlsx-Table_S3-Unnamed_3': ('Transcriptional landscape of human microglia implicates age, sex, and APOE-related immunometabolic pathway perturbations',\n",
       "  'Microglia have fundamental roles in health and disease; however, effects of age, sex, and genetic factors on human microglia have not been fully explored. We applied bulk and single-cell approaches to comprehensively characterize human microglia transcriptomes and their associations with age, sex, and APOE. We identified a novel microglial signature, characterized its expression in bulk tissue and single-cell microglia transcriptomes. We discovered microglial co-expression network modules associated with age, sex, and APOE-ε4 that are enriched for lipid and carbohydrate metabolism genes. Integrated analyses of modules with single-cell transcriptomes revealed significant overlap between age-associated module genes and both pro-inflammatory and disease-associated microglial clusters. These modules and clusters harbor known neurodegenerative disease genes including APOE, PLCG2, and BIN1. Meta-analyses with published bulk and single-cell microglial datasets further supported our findings. Thus, these data represent a well-characterized human microglial transcriptome resource and highlight age, sex, and APOE-related microglial immunometabolism perturbations with potential relevance in neurodegeneration.',\n",
       "  'Table_S3-Unnamed_3',\n",
       "  'our bulk microglia and a q‐value threshold of 0.05, we identified 1971 genes (Table **S1–S6**)  Table S1‐S20  Click here for additional data file'),\n",
       " 'PMC8815209-12885_2022_9251_MOESM4_ESM.xls-Table_S3-Unnamed_16': ('FCER1G positively relates to macrophage infiltration in clear cell renal cell carcinoma and contributes to unfavorable prognosis by regulating tumor immunity',\n",
       "  \"Background: Tumor-associated macrophages (TAMs) are closely related to unfavorable prognosis of patients with clear cell renal cell carcinoma (ccRCC). However, the important molecules in the interaction between ccRCC and TAMs are unclear. Methods: TCGA-KIRC gene expression data of tumor tissues and normal tissues adjacent to tumor were compared to identify differentially expressed genes in ccRCC. TAMs related genes were discovered by analyzing the correlation between these differentially expressed genes and common macrophage biomarkers. Gene set enrichment analysis was performed to predict functions of TAMs related gene. The findings were further validated using RNA sequencing data obtained from the CheckMate 025 study and immunohistochemical analysis of samples from 350 patients with ccRCC. Kaplan-Meier survival curve, Cox regression analysis and Harrell's concordance index analysis were used to determine the prognostic significance. Results: In this study, we applied bioinformatic analysis to explore TAMs related differentially expressed genes in ccRCC and identified 5 genes strongly correlated with all selected macrophage biomarkers: STAC3, LGALS9, TREM2, FCER1G, and PILRA. Among them, FCER1G was abundantly expressed in tumor tissues and showed prognostic importance in patients with ccRCC who received treatment with Nivolumab; however, it did not exhibit prognostic value in those treated with Everolimus. We also discovered that high expression levels of FCER1G are related to T cell suppression. Moreover, combination of FCER1G and macrophage biomarker CD68 can improve the prognostic stratification of patients with ccRCC from TCGA-KIRC. Based on the immunohistochemical analysis of samples from patients with ccRCC, we further validated that FCER1G and CD68 are both highly expressed in tumor tissue and correlate with each other. Higher expression of CD68 or FCER1G in ccRCC tissue indicates shorter overall survival and progression-free survival; patients with high expression of both CD68 and FCER1G have the worst outcome. Combining CD68 and FCER1G facilitates the screening of patients with a worse prognosis from the same TNM stage group. Conclusions: High expression of FCER1G in ccRCC is closely related to TAMs infiltration and suppression of T cell activation and proliferation. Combining the expression levels of FCER1G and macrophage biomarker CD68 may be a promising postoperative prognostic index for patients with ccRCC.\",\n",
       "  'Table_S3-Unnamed_16',\n",
       "  'Additional file 4. Gene sets enriched in TCGA-KIRC with high FCER1G expression'),\n",
       " 'PMC10891104-41598_2024_54924_MOESM1_ESM.xlsx-Na-vs-Nb_filter_annot-Identification_of_differentially_expressed_genes_in_CPAM': ('The underlying molecular mechanism of ciliated epithelium dysfunction and TGF-β signaling in children with congenital pulmonary airway malformations',\n",
       "  'The aim of this study was to investigate the variation in gene expression in the complete transcripts of Congenitalpulmonary airwaymalformation (CPAM) of the lung using Next Generation Sequencing (NGS) technology. There were 20 cases involving children with CPAM were used for selection of study sample. NGS was used to establish RNA-Seq libraries for the two groups of samples separately, and both groups were conducted to differential expression analysis and Gene Ontology (GO) functional enrichment analysis. The pathways of the differential genes were analyzed to find the enriched target pathways. A total of 592 genes were expressed with significant differences (CPAM vs. normal tissue, P < 0.05). GO functional analysis of DEGs indicated that abnormal ciliary function played a role in the development of CPAM. Subsequently, analysis of these genes pathways showed the TGF-β signaling pathway was significantly enriched. Finally, the results of immunohistochemical analysis of some DEGs showed that a significant reduction in the expression of SMAD6, a gene related to the TGF-β signaling pathway, led to abnormal activation of the pathway. TGF-β signaling pathway involved in the evolution of the disease obtained by DEGs enrichment pathway analysis. SMAD6, a gene involved in this pathway, might be a potential biomarker for the diagnosis and treatment of CPAM.',\n",
       "  'Na-vs-Nb_filter_annot-Identification_of_differentially_expressed_genes_in_CPAM',\n",
       "  'None here'),\n",
       " 'PMC10891104-41598_2024_54924_MOESM1_ESM.xlsx-Na-vs-Nb_filter_annot-Unnamed_26': ('The underlying molecular mechanism of ciliated epithelium dysfunction and TGF-β signaling in children with congenital pulmonary airway malformations',\n",
       "  'The aim of this study was to investigate the variation in gene expression in the complete transcripts of Congenitalpulmonary airwaymalformation (CPAM) of the lung using Next Generation Sequencing (NGS) technology. There were 20 cases involving children with CPAM were used for selection of study sample. NGS was used to establish RNA-Seq libraries for the two groups of samples separately, and both groups were conducted to differential expression analysis and Gene Ontology (GO) functional enrichment analysis. The pathways of the differential genes were analyzed to find the enriched target pathways. A total of 592 genes were expressed with significant differences (CPAM vs. normal tissue, P < 0.05). GO functional analysis of DEGs indicated that abnormal ciliary function played a role in the development of CPAM. Subsequently, analysis of these genes pathways showed the TGF-β signaling pathway was significantly enriched. Finally, the results of immunohistochemical analysis of some DEGs showed that a significant reduction in the expression of SMAD6, a gene related to the TGF-β signaling pathway, led to abnormal activation of the pathway. TGF-β signaling pathway involved in the evolution of the disease obtained by DEGs enrichment pathway analysis. SMAD6, a gene involved in this pathway, might be a potential biomarker for the diagnosis and treatment of CPAM.',\n",
       "  'Na-vs-Nb_filter_annot-Unnamed_26',\n",
       "  'None here'),\n",
       " 'PMC9500203-Table5.xlsx-Sheet1-Gene': ('Subtyping children with asthma by clustering analysis of mRNA expression data',\n",
       "  \"Background: Asthma is a heterogeneous disease. There are several phenotypic classifications for childhood asthma. Methods: Unsupervised consensus cluster analysis was used to classify 36 children with persistent asthma from the GSE65204 dataset. The differentially expressed genes (DEGs) between different asthma subtypes were identified, and weighted gene co-expression network analysis (WGCNA) was carried out. Gene Ontology and Kyoto Encyclopedia of Genes and Genomes enrichment analysis was performed for DEGs and critical gene modules. Protein-protein interactions (PPI) were constructed to obtain the hub genes. Finally, differences in the immune microenvironment were analyzed between different subtypes. Results: Two subtypes (C1, C2) were identified using unsupervised consensus clustering. The DEGs between different asthma subtypes were mainly enriched in immune regulation and the release of inflammatory mediators. The important modular genes screened by WGCNA were mainly enriched in aspects of inflammatory mediator regulation. PPI analysis found 10 hub genes (DRC1, TTC25, DNALI1, DNAI1, DNAI2, PIH1D3, ARMC4, RSPH1, DNAAF3, and DNAH5), and ROC analysis demonstrated that 10 hub genes had a reliably ability to distinguish C1 from C2. And we observed differences between C1 and C2 in their immune microenvironment. Conclusion: Using the gene expression profiles of children's nasal epithelium, we identified two asthma subtypes that have different gene expression patterns, biological characteristics, and immune microenvironments. This will provide a reference point for future childhood asthma typing and personalized therapy.\",\n",
       "  'Sheet1-Gene',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC11188533-12993_2024_240_MOESM2_ESM.xlsx-Figure_3A_VolcanoPlots-Unnamed_50': ('Transcriptome analysis identifies an ASD-Like phenotype in oligodendrocytes and microglia from C58/J amygdala that is dependent on sex and sociability',\n",
       "  \"Background: Autism Spectrum Disorder (ASD) is a group of neurodevelopmental disorders with higher incidence in males and is characterized by atypical verbal/nonverbal communication, restricted interests that can be accompanied by repetitive behavior, and disturbances in social behavior. This study investigated brain mechanisms that contribute to sociability deficits and sex differences in an ASD animal model. Methods: Sociability was measured in C58/J and C57BL/6J mice using the 3-chamber social choice test. Bulk RNA-Seq and snRNA-Seq identified transcriptional changes in C58/J and C57BL/6J amygdala within which DMRseq was used to measure differentially methylated regions in amygdala. Results: C58/J mice displayed divergent social strata in the 3-chamber test. Transcriptional and pathway signatures revealed immune-related biological processes differ between C58/J and C57BL/6J amygdala. Hypermethylated and hypomethylated genes were identified in C58/J versus C57BL/6J amygdala. snRNA-Seq data in C58/J amygdala identified differential transcriptional signatures within oligodendrocytes and microglia characterized by increased ASD risk gene expression and predicted impaired myelination that was dependent on sex and sociability. RNA velocity, gene regulatory network, and cell communication analysis showed diminished oligodendrocyte/microglia differentiation. Findings were verified using Bulk RNA-Seq and demonstrated oxytocin's beneficial effects on myelin gene expression. Limitations: Our findings are significant. However, limitations can be noted. The cellular mechanisms linking reduced oligodendrocyte differentiation and reduced myelination to an ASD phenotype in C58/J mice need further investigation. Additional snRNA-Seq and spatial studies would determine if effects in oligodendrocytes/microglia are unique to amygdala or if this occurs in other brain regions. Oxytocin's effects need further examination to understand its' potential as an ASD therapeutic. Conclusions: Our work demonstrates the C58/J mouse model's utility in evaluating the influence of sex and sociability on the transcriptome in concomitant brain regions involved in ASD. Our single-nucleus transcriptome analysis elucidates potential pathological roles of oligodendrocytes and microglia in ASD. This investigation provides details regarding regulatory features disrupted in these cell types, including transcriptional gene dysregulation, aberrant cell differentiation, altered gene regulatory networks, and changes to key pathways that promote microglia/oligodendrocyte differentiation. Our studies provide insight into interactions between genetic risk and epigenetic processes associated with divergent affiliative behavior and lack of positive sociability.\",\n",
       "  'Figure_3A_VolcanoPlots-Unnamed_50',\n",
       "  'None here'),\n",
       " 'PMC11188533-12993_2024_240_MOESM2_ESM.xlsx-Supp_Figure_7B-Unnamed_98': ('Transcriptome analysis identifies an ASD-Like phenotype in oligodendrocytes and microglia from C58/J amygdala that is dependent on sex and sociability',\n",
       "  \"Background: Autism Spectrum Disorder (ASD) is a group of neurodevelopmental disorders with higher incidence in males and is characterized by atypical verbal/nonverbal communication, restricted interests that can be accompanied by repetitive behavior, and disturbances in social behavior. This study investigated brain mechanisms that contribute to sociability deficits and sex differences in an ASD animal model. Methods: Sociability was measured in C58/J and C57BL/6J mice using the 3-chamber social choice test. Bulk RNA-Seq and snRNA-Seq identified transcriptional changes in C58/J and C57BL/6J amygdala within which DMRseq was used to measure differentially methylated regions in amygdala. Results: C58/J mice displayed divergent social strata in the 3-chamber test. Transcriptional and pathway signatures revealed immune-related biological processes differ between C58/J and C57BL/6J amygdala. Hypermethylated and hypomethylated genes were identified in C58/J versus C57BL/6J amygdala. snRNA-Seq data in C58/J amygdala identified differential transcriptional signatures within oligodendrocytes and microglia characterized by increased ASD risk gene expression and predicted impaired myelination that was dependent on sex and sociability. RNA velocity, gene regulatory network, and cell communication analysis showed diminished oligodendrocyte/microglia differentiation. Findings were verified using Bulk RNA-Seq and demonstrated oxytocin's beneficial effects on myelin gene expression. Limitations: Our findings are significant. However, limitations can be noted. The cellular mechanisms linking reduced oligodendrocyte differentiation and reduced myelination to an ASD phenotype in C58/J mice need further investigation. Additional snRNA-Seq and spatial studies would determine if effects in oligodendrocytes/microglia are unique to amygdala or if this occurs in other brain regions. Oxytocin's effects need further examination to understand its' potential as an ASD therapeutic. Conclusions: Our work demonstrates the C58/J mouse model's utility in evaluating the influence of sex and sociability on the transcriptome in concomitant brain regions involved in ASD. Our single-nucleus transcriptome analysis elucidates potential pathological roles of oligodendrocytes and microglia in ASD. This investigation provides details regarding regulatory features disrupted in these cell types, including transcriptional gene dysregulation, aberrant cell differentiation, altered gene regulatory networks, and changes to key pathways that promote microglia/oligodendrocyte differentiation. Our studies provide insight into interactions between genetic risk and epigenetic processes associated with divergent affiliative behavior and lack of positive sociability.\",\n",
       "  'Supp_Figure_7B-Unnamed_98',\n",
       "  'None here'),\n",
       " 'PMC8233431-41598_2021_92725_MOESM1_ESM.xlsx-Table_S1-Unnamed_1': ('Lung adenocarcinoma and lung squamous cell carcinoma cancer classification, biomarker identification, and gene expression analysis using overlapping feature selection methods',\n",
       "  'Lung cancer is one of the deadliest cancers in the world. Two of the most common subtypes, lung adenocarcinoma (LUAD) and lung squamous cell carcinoma (LUSC), have drastically different biological signatures, yet they are often treated similarly and classified together as non-small cell lung cancer (NSCLC). LUAD and LUSC biomarkers are scarce, and their distinct biological mechanisms have yet to be elucidated. To detect biologically relevant markers, many studies have attempted to improve traditional machine learning algorithms or develop novel algorithms for biomarker discovery. However, few have used overlapping machine learning or feature selection methods for cancer classification, biomarker identification, or gene expression analysis. This study proposes to use overlapping traditional feature selection or feature reduction techniques for cancer classification and biomarker discovery. The genes selected by the overlapping method were then verified using random forest. The classification statistics of the overlapping method were compared to those of the traditional feature selection methods. The identified biomarkers were validated in an external dataset using AUC and ROC analysis. Gene expression analysis was then performed to further investigate biological differences between LUAD and LUSC. Overall, our method achieved classification results comparable to, if not better than, the traditional algorithms. It also identified multiple known biomarkers, and five potentially novel biomarkers with high discriminating values between LUAD and LUSC. Many of the biomarkers also exhibit significant prognostic potential, particularly in LUAD. Our study also unraveled distinct biological pathways between LUAD and LUSC.',\n",
       "  'Table_S1-Unnamed_1',\n",
       "  'Top 500 genes from DGE (Table **S1**)  Supplementary Tables'),\n",
       " 'PMC7641354-pgen.1009090.s008.xlsx-Feuil1-S3_Table': ('A genetic variant controls interferon-β gene expression in human myeloid cells by preventing C/EBP-β binding on a conserved enhancer',\n",
       "  'Interferon β (IFN-β) is a cytokine that induces a global antiviral proteome, and regulates the adaptive immune response to infections and tumors. Its effects strongly depend on its level and timing of expression. Therefore, the transcription of its coding gene IFNB1 is strictly controlled. We have previously shown that in mice, the TRIM33 protein restrains Ifnb1 transcription in activated myeloid cells through an upstream inhibitory sequence called ICE. Here, we show that the deregulation of Ifnb1 expression observed in murine Trim33-/- macrophages correlates with abnormal looping of both ICE and the Ifnb1 gene to a 100 kb downstream region overlapping the Ptplad2/Hacd4 gene. This region is a predicted myeloid super-enhancer in which we could characterize 3 myeloid-specific active enhancers, one of which (E5) increases the response of the Ifnb1 promoter to activation. In humans, the orthologous region contains several single nucleotide polymorphisms (SNPs) known to be associated with decreased expression of IFNB1 in activated monocytes, and loops to the IFNB1 gene. The strongest association is found for the rs12553564 SNP, located in the E5 orthologous region. The minor allele of rs12553564 disrupts a conserved C/EBP-β binding motif, prevents binding of C/EBP-β, and abolishes the activation-induced enhancer activity of E5. Altogether, these results establish a link between a genetic variant preventing binding of a transcription factor and a higher order phenotype, and suggest that the frequent minor allele (around 30% worldwide) might be associated with phenotypes regulated by IFN-β expression in myeloid cells.',\n",
       "  'Feuil1-S3_Table',\n",
       "  'with a total of 433 genes (FDR < 0.01, |βeQTL| > 0.2, Fig 2D, **S3 Table**)  List of the 433 genes regulated in trans by rs12553564.(XLSX)  Click here for additional data file'),\n",
       " 'PMC1636363-nar_gkl590_Supplementary_table_1_revised.xls-Supplementary_Table_1-Unnamed_1': ('Identification of miRNA targets with stable isotope labeling by amino acids in cell culture',\n",
       "  \"miRNAs are small noncoding RNAs that regulate gene expression. We have used stable isotope labeling by amino acids in cell culture (SILAC) to investigate the effect of miRNA-1 on the HeLa cell proteome. Expression of 12 out of 504 investigated proteins was repressed by miRNA-1 transfection. This repressed set of genes significantly overlaps with miRNA-1 regulated genes that have been identified with DNA array technology and are predicted by computational methods. Moreover, we find that the 3'-untranslated region for the repressed set are enriched in miRNA-1 complementary sites. Our findings demonstrate that SILAC can be used for miRNA target identification and that one highly expressed miRNA can regulate the levels of many different proteins.\",\n",
       "  'Supplementary_Table_1-Unnamed_1',\n",
       "  '[Supplementary Data]'),\n",
       " 'PMC6998390-MOL2-14-407-s003.xlsx-Sheet1-Negative_Genes': ('TGFβ1-induced beta-site APP-cleaving enzyme 2 upregulation promotes tumorigenesis through the NF-κB signalling pathway in human gliomas',\n",
       "  'Gliomas are the most common primary malignant tumours of the central nervous system, and new molecular biomarkers are urgently needed for diagnosis and targeted therapy. Here, we report that increased beta-site APP-cleaving enzyme 2 (BACE2) expression is associated with increases in the grade of human glioma, the incidence of the mesenchymal molecular glioblastoma multiforme subtype and the likelihood of poor prognoses for patients. BACE2 knockdown suppressed cell invasion, cell migration and tumour growth both in vitro and in vivo, while BACE2 overexpression promoted the mesenchymal transition and cell proliferation. Furthermore, TGFβ1 stimulated BACE2 expression through Smad-dependent signalling, which modulated TNF-α-induced NF-κB activity through the PP1A/IKK pathway to promote tumorigenesis in both U87MG and U251 cells. Our study indicated that BACE2 plays a significant role in glioma development. Therefore, BACE2 is a potential therapeutic target for human gliomas due to its function and ability to be regulated.',\n",
       "  'Sheet1-Negative_Genes',\n",
       "  'database, 1247 and 649 genes were positively and negatively related to BACE2, respectively (Table **S2**)  Table S2. Positively related genes and negatively related genes of BACE2 in TCGA  Click here for additional data file'),\n",
       " 'PMC11056698-CPR-57-e13591-s007.xlsx-Neutrophil_S100-marker_genes_of_Neutrophil_S100': ('Single-cell RNA sequencing reveals immune cell dysfunction in the peripheral blood of patients with highly aggressive gastric cancer',\n",
       "  'Highly aggressive gastric cancer (HAGC) is a gastric cancer characterized by bone marrow metastasis and disseminated intravascular coagulation (DIC). Information about the disease is limited. Here we employed single-cell RNA sequencing to investigate peripheral blood mononuclear cells (PBMCs), aiming to unravel the immune response of patients toward HAGC. PBMCs from seven HAGC patients, six normal advanced gastric cancer (NAGC) patients, and five healthy individuals were analysed by single-cell RNA sequencing. The expression of genes of interest was validated by bulk RNA-sequencing and ELISA. We found a massive expansion of neutrophils in PBMCs of HAGC. These neutrophils are activated, but immature. Besides, mononuclear phagocytes exhibited an M2-like signature and T cells were suppressed and reduced in number. Analysis of cell-cell crosstalk revealed that several signalling pathways involved in neutrophil to T-cell suppression including APP-CD74, MIF-(CD74+CXCR2), and MIF-(CD74+CD44) pathways were increased in HAGC. NETosis-associated genes S100A8 and S100A9 as well as VEGF, PDGF, FGF, and NOTCH signalling that contribute to DIC development were upregulated in HAGC too. This study reveals significant changes in the distribution and interactions of the PBMC subsets and provides valuable insight into the immune response in patients with HAGC. S100A8 and S100A9 are highly expressed in HAGC neutrophils, suggesting their potential to be used as novel diagnostic and therapeutic targets for HAGC.',\n",
       "  'Neutrophil_S100-marker_genes_of_Neutrophil_S100',\n",
       "  'None here'),\n",
       " 'PMC7641354-pgen.1009090.s008.xlsx-Feuil1-Unnamed_1': ('A genetic variant controls interferon-β gene expression in human myeloid cells by preventing C/EBP-β binding on a conserved enhancer',\n",
       "  'Interferon β (IFN-β) is a cytokine that induces a global antiviral proteome, and regulates the adaptive immune response to infections and tumors. Its effects strongly depend on its level and timing of expression. Therefore, the transcription of its coding gene IFNB1 is strictly controlled. We have previously shown that in mice, the TRIM33 protein restrains Ifnb1 transcription in activated myeloid cells through an upstream inhibitory sequence called ICE. Here, we show that the deregulation of Ifnb1 expression observed in murine Trim33-/- macrophages correlates with abnormal looping of both ICE and the Ifnb1 gene to a 100 kb downstream region overlapping the Ptplad2/Hacd4 gene. This region is a predicted myeloid super-enhancer in which we could characterize 3 myeloid-specific active enhancers, one of which (E5) increases the response of the Ifnb1 promoter to activation. In humans, the orthologous region contains several single nucleotide polymorphisms (SNPs) known to be associated with decreased expression of IFNB1 in activated monocytes, and loops to the IFNB1 gene. The strongest association is found for the rs12553564 SNP, located in the E5 orthologous region. The minor allele of rs12553564 disrupts a conserved C/EBP-β binding motif, prevents binding of C/EBP-β, and abolishes the activation-induced enhancer activity of E5. Altogether, these results establish a link between a genetic variant preventing binding of a transcription factor and a higher order phenotype, and suggest that the frequent minor allele (around 30% worldwide) might be associated with phenotypes regulated by IFN-β expression in myeloid cells.',\n",
       "  'Feuil1-Unnamed_1',\n",
       "  'with a total of 433 genes (FDR < 0.01, |βeQTL| > 0.2, Fig 2D, **S3 Table**)  List of the 433 genes regulated in trans by rs12553564.(XLSX)  Click here for additional data file'),\n",
       " 'PMC1636363-nar_gkl590_Supplementary_table_1_revised.xls-Supplementary_Table_1-Unnamed_4': ('Identification of miRNA targets with stable isotope labeling by amino acids in cell culture',\n",
       "  \"miRNAs are small noncoding RNAs that regulate gene expression. We have used stable isotope labeling by amino acids in cell culture (SILAC) to investigate the effect of miRNA-1 on the HeLa cell proteome. Expression of 12 out of 504 investigated proteins was repressed by miRNA-1 transfection. This repressed set of genes significantly overlaps with miRNA-1 regulated genes that have been identified with DNA array technology and are predicted by computational methods. Moreover, we find that the 3'-untranslated region for the repressed set are enriched in miRNA-1 complementary sites. Our findings demonstrate that SILAC can be used for miRNA target identification and that one highly expressed miRNA can regulate the levels of many different proteins.\",\n",
       "  'Supplementary_Table_1-Unnamed_4',\n",
       "  '[Supplementary Data]'),\n",
       " 'PMC9982587-ADVS-10-2205486-s002.xlsx-Table_S4-Unnamed_4': ('Gene Expression has Distinct Associations with Brain Structure and Function in Major Depressive Disorder',\n",
       "  'Major depressive disorder (MDD) is associated with structural and functional brain abnormalities. MDD as well as brain anatomy and function are influenced by genetic factors, but the role of gene expression remains unclear. Here, this work investigates how cortical gene expression contributes to structural and functional brain abnormalities in MDD. This work compares the gray matter volume and resting-state functional measures in a Chinese sample of 848 MDD patients and 749 healthy controls, and these case-control differences are then associated with cortical variation of gene expression. While whole gene expression is positively associated with structural abnormalities, it is negatively associated with functional abnormalities. This work observes the relationships of expression levels with brain abnormalities for individual genes, and found that transcriptional correlates of brain structure and function show opposite relations with gene dysregulation in postmortem cortical tissue from MDD patients. This work further identifies genes that are positively or negatively related to structural abnormalities as well as functional abnormalities. The MDD-related genes are enriched for brain tissue, cortical cells, and biological pathways. These findings suggest that distinct genetic mechanisms underlie structural and functional brain abnormalities in MDD, and highlight the importance of cortical gene expression for the development of cortical abnormalities.',\n",
       "  'Table_S4-Unnamed_4',\n",
       "  'regions of each gene were related to the funcPC1 and GMV differences, respectively (Tables **S1**)  Supplemental Table  Click here for additional data file'),\n",
       " 'PMC10756545-pone.0296454.s002.xlsx-6_TH_modules-TH1': ('Gene expression signatures of stepwise progression of Hepatocellular Carcinoma',\n",
       "  'The molecular pathogenesis of Hepatocellular Carcinoma (HCC) is a complex process progressing from premalignant stages to cancer in a stepwise manner. Mostly, HCC is detected at advanced stages, leading to high mortality rates. Hence, characterising the molecular underpinnings of HCC from normal to cancer state through precancerous state may help in early detection and improve its prognosis and treatment. In this work, we analysed the transcriptomic profile of tumour and premalignant samples from HCC or chronic liver disease patients, who had undergone either total or partial hepatectomy. The normal samples from patients with metastatic cancer/polycystic liver disease/ cholangiocarcinoma were also included. A gene co-expression network approach was applied to identify hierarchical changes: modules, pathways, and genes related to different trajectories of HCC and patient survival. Our analysis shows that the progression from premalignant conditions to tumour is accompanied by differences in the downregulation of genes associated with HNF4A activity and the immune system and upregulation of cell cycle genes, bringing about variability in patient outcomes. However, an increase in immune and cell cycle activity is observed in premalignant samples. Interestingly, co-expression modules and genes from premalignant stages are associated with survival. THBD, a classical marker for dendritic cells, is a predictor of survival at the premalignant stage. Further, genes linked to microtubules, kinetochores, and centromere are altered in both premalignant and tumour conditions and are associated with survival. Our analysis revealed a three-way molecular axis of liver function, immune pathways, and cell cycle driving HCC pathogenesis.',\n",
       "  '6_TH_modules-TH1',\n",
       "  '**S1 Dataset**  List of genes.Survival genes based on modules from tumour samples, and normal and premalignant samples; List of genes in each module; HIR signature; Cell cycle genes.(XLSX)  Click here for additional data file'),\n",
       " 'PMC9915673-media-2.xlsx-ribo_24h_1277_true-index': ('CDK4/6 inhibition enhances SHP2 inhibitor efficacy and is dependent upon restoration of RB function in malignant peripheral nerve sheath tumors',\n",
       "  'Malignant peripheral nerve sheath tumors (MPNST) are highly aggressive soft tissue sarcomas with limited treatment options, and novel effective therapeutic strategies are desperately needed. We observe anti-proliferative efficacy of genetic depletion or pharmacological inhibition using the clinically available SHP2 inhibitor (SHP2i) TNO155. Our studies into the signaling response to SHP2i reveal that resistance to TNO155 is partially mediated by reduced RB function, and we therefore test the addition of a CDK4/6 inhibitor (CDK4/6i) to enhance RB activity and improve TNO155 efficacy. In combination, TNO155 attenuates the adaptive response to CDK4/6i, potentiates its anti-proliferative effects, and converges on enhancement of RB activity, with greater suppression of cell cycle and inhibitor-of-apoptosis proteins, leading to deeper and more durable anti-tumor activity in in vitro and in vivo patient-derived models of MPNST, relative to either single agent. Overall, our study provides timely evidence to support the clinical advancement of this combination strategy in patients with MPNST and other tumors driven by loss of NF1.',\n",
       "  'ribo_24h_1277_true-index',\n",
       "  'genes compared to the treatment of 20 nmol/L trametinib or shPTPN11 (Fig 2I and **Table S2**)'),\n",
       " 'PMC9915673-media-2.xlsx-ribo_24h_1277_true-identifier': ('CDK4/6 inhibition enhances SHP2 inhibitor efficacy and is dependent upon restoration of RB function in malignant peripheral nerve sheath tumors',\n",
       "  'Malignant peripheral nerve sheath tumors (MPNST) are highly aggressive soft tissue sarcomas with limited treatment options, and novel effective therapeutic strategies are desperately needed. We observe anti-proliferative efficacy of genetic depletion or pharmacological inhibition using the clinically available SHP2 inhibitor (SHP2i) TNO155. Our studies into the signaling response to SHP2i reveal that resistance to TNO155 is partially mediated by reduced RB function, and we therefore test the addition of a CDK4/6 inhibitor (CDK4/6i) to enhance RB activity and improve TNO155 efficacy. In combination, TNO155 attenuates the adaptive response to CDK4/6i, potentiates its anti-proliferative effects, and converges on enhancement of RB activity, with greater suppression of cell cycle and inhibitor-of-apoptosis proteins, leading to deeper and more durable anti-tumor activity in in vitro and in vivo patient-derived models of MPNST, relative to either single agent. Overall, our study provides timely evidence to support the clinical advancement of this combination strategy in patients with MPNST and other tumors driven by loss of NF1.',\n",
       "  'ribo_24h_1277_true-identifier',\n",
       "  'genes compared to the treatment of 20 nmol/L trametinib or shPTPN11 (Fig 2I and **Table S2**)'),\n",
       " 'PMC6707279-41598_2019_48243_MOESM2_ESM.xls-coexpression_AK9_mRNA_expressio-Gene_Symbol': ('A co-expressed gene status of adenylate kinase 1/4 reveals prognostic gene signature associated with prognosis and sensitivity to EGFR targeted therapy in lung adenocarcinoma',\n",
       "  'Cancer cells utilize altered bioenergetics to fuel uncontrolled proliferation and progression. At the core of bioenergetics, adenine nucleotides are the building blocks for nucleotide synthesis, energy transfer and diverse metabolic processes. Adenylate kinases (AK) are ubiquitous phosphotransferases that catalyze the conversion of adenine nucleotides and regulate the homeostasis of nucleotide ratios within cellular compartments. Recently, different isoforms of AK have been shown to induce metabolic reprograming in cancer and were identified as biomarkers for predicting disease progression. Here we aim to systemically analyze the impact of all AK-associated gene signatures on lung adenocarcinoma patient survival and decipher the value for therapeutic interventions. By analyzing TCGA Lung Adenocarcinoma (LUAD) RNA Seq data, we found gene signatures from AK4 and AK1 have higher percentage of prognostic genes compared to other AK-gene signatures. A 118-gene signature was identified from consensus gene expression in AK1 and AK4 prognostic gene signatures. Immunohistochemistry (IHC) analyses in 140 lung adenocarcinoma patients showed overexpression of AK4 significantly correlated with worse overall survival (P = 0.001) whereas overexpression of AK1 significantly associated with good prognosis (P = 0.009). Furthermore, reduced AK4 expression by shRNA reduced the EGFR protein expression in EGFR mutation cells. The inhibition of AK4-AK1 signal might provide a potential target for synergistic effect in target therapy in lung cancer patients.',\n",
       "  'coexpression_AK9_mRNA_expressio-Gene_Symbol',\n",
       "  '(LUAD) dataset and identified unique signatures for each AK (Fig. 1a and Supplementary Table **S1**)  Table S1'),\n",
       " 'PMC10972588-jciinsight-9-172942-s150.xlsx-Differential_expression_RNA-seq-Unnamed_1': ('Tregs from human blood differentiate into nonlymphoid tissue-resident effector cells upon TNFR2 costimulation',\n",
       "  'Tregs can facilitate transplant tolerance and attenuate autoimmune and inflammatory diseases. Therefore, it is clinically relevant to stimulate Treg expansion and function in vivo and to create therapeutic Treg products in vitro. We report that TNF receptor 2 (TNFR2) is a unique costimulus for naive, thymus-derived Tregs (tTregs) from human blood that promotes their differentiation into nonlymphoid tissue-resident (NLT-resident) effector Tregs, without Th-like polarization. In contrast, CD28 costimulation maintains a lymphoid tissue-resident (LT-resident) Treg phenotype. We base this conclusion on transcriptome and proteome analysis of TNFR2- and CD28-costimulated CD4+ tTregs and conventional T cells (Tconvs), followed by bioinformatic comparison with published transcriptomic Treg signatures from NLT and LT in health and disease, including autoimmunity and cancer. These analyses illuminate that TNFR2 costimulation promoted tTreg capacity for survival, migration, immunosuppression, and tissue regeneration. Functional studies confirmed improved migratory ability of TNFR2-costimulated tTregs. Flow cytometry validated the presence of the TNFR2-driven tTreg signature in effector/memory Tregs from the human placenta, as opposed to blood. Thus, TNFR2 can be exploited as a driver of NLT-resident tTreg differentiation for adoptive cell therapy or antibody-based immunomodulation in human disease.',\n",
       "  'Differential_expression_RNA-seq-Unnamed_1',\n",
       "  'None here'),\n",
       " 'PMC9220854-jciinsight-7-153045-s290.xlsx-Differentiated_epithelial_cells-Tuft': ('MTG16 regulates colonic epithelial differentiation, colitis, and tumorigenesis by repressing E protein transcription factors',\n",
       "  'Aberrant epithelial differentiation and regeneration contribute to colon pathologies, including inflammatory bowel disease (IBD) and colitis-associated cancer (CAC). Myeloid translocation gene 16 (MTG16, also known as CBFA2T3) is a transcriptional corepressor expressed in the colonic epithelium. MTG16 deficiency in mice exacerbates colitis and increases tumor burden in CAC, though the underlying mechanisms remain unclear. Here, we identified MTG16 as a central mediator of epithelial differentiation, promoting goblet and restraining enteroendocrine cell development in homeostasis and enabling regeneration following dextran sulfate sodium-induced (DSS-induced) colitis. Transcriptomic analyses implicated increased Ephrussi box-binding transcription factor (E protein) activity in MTG16-deficient colon crypts. Using a mouse model with a point mutation that attenuates MTG16:E protein interactions (Mtg16P209T), we showed that MTG16 exerts control over colonic epithelial differentiation and regeneration by repressing E protein-mediated transcription. Mimicking murine colitis, MTG16 expression was increased in biopsies from patients with active IBD compared with unaffected controls. Finally, uncoupling MTG16:E protein interactions partially phenocopied the enhanced tumorigenicity of Mtg16-/- colon in the azoxymethane/DSS-induced model of CAC, indicating that MTG16 protects from tumorigenesis through additional mechanisms. Collectively, our results demonstrate that MTG16, via its repression of E protein targets, is a key regulator of cell fate decisions during colon homeostasis, colitis, and cancer.',\n",
       "  'Differentiated_epithelial_cells-Tuft',\n",
       "  'Supplemental table 4'),\n",
       " 'PMC7455079-jciinsight-5-140380-s227.xlsx-Supp_Table_8-ILL2_Erythrocytes': ('Patient ancestry significantly contributes to molecular heterogeneity of systemic lupus erythematosus',\n",
       "  'Gene expression signatures can stratify patients with heterogeneous diseases, such as systemic lupus erythematosus (SLE), yet understanding the contributions of ancestral background to this heterogeneity is not well understood. We hypothesized that ancestry would significantly influence gene expression signatures and measured 34 gene modules in 1566 SLE patients of African ancestry (AA), European ancestry (EA), or Native American ancestry (NAA). Healthy subject ancestry-specific gene expression provided the transcriptomic background upon which the SLE patient signatures were built. Although standard therapy affected every gene signature and significantly increased myeloid cell signatures, logistic regression analysis determined that ancestral background significantly changed 23 of 34 gene signatures. Additionally, the strongest association to gene expression changes was found with autoantibodies, and this also had etiology in ancestry: the AA predisposition to have both RNP and dsDNA autoantibodies compared with EA predisposition to have only anti-dsDNA. A machine learning approach was used to determine a gene signature characteristic to distinguish AA SLE and was most influenced by genes characteristic of the perturbed B cell axis in AA SLE patients.',\n",
       "  'Supp_Table_8-ILL2_Erythrocytes',\n",
       "  'Supplemental Tables 1-22'),\n",
       " 'PMC8973469-peerj-10-13201-s004.xlsx-positive_genes-Query': ('Comprehensive analysis of the correlations of S100B with hypoxia response and immune infiltration in hepatocellular carcinoma',\n",
       "  'S100B has been found to be dysregulated in many cancers including hepatocellular carcinoma (HCC). However, the functions of S100B and its underlying mechanisms in HCC remain poorly understood, especially in the tumor microenvironment. In this study, functions enrichment analysis indicated that S100B expression was correlated with hypoxia and immune responses. We found that hypoxia could induce S100B expression in an HIF-1α-dependent manner in HepG2 cells. Luciferase reporter and ChIP-qRCR assays demonstrated that HIF-1α regulates S100B transcription by directly binding to hypoxia-response elements (HREs) of the S100B promoter. Functionally, knockdown of S100B reduces hypoxia-induced HepG2 cell invasion and migration. Furthermore, GSVA enrichment results displayed that S100B and its co-expressed genes were positively correlated with EMT pathway in HCC. Additionally, GO/KEGG cluster analysis results indicated that co-expressed genes of S100B were involved in biological processes of immune response and multiple tumor immune-related signaling pathways in HCC. S100B expression was positively correlated with multiple immune cells tumor infiltration and associated with chemokines/chemokine receptors and immune checkpoint genes. Moreover, S100B is predominantly expressed in immune cells, especially NK (Natural Killer) cell. In addition, the hub genes of S100B co-expression and hypoxia response in HepG2 cell were also associated with immune cells infiltration in HCC. Taken together, these findings provide a new insight into the complex networks between hypoxia response and immune cells infiltration in tumor microenvironment of liver cancer. S100B maybe serve as a novel target for future HCC therapies.',\n",
       "  'positive_genes-Query',\n",
       "  'no negatively correlated genes (R < −0.5 P < 0.05) were found in LIHC (**Table S1**)  S100B co-expressed genes in LIHC  Click here for additional data file'),\n",
       " 'PMC8097075-41467_2021_22779_MOESM2_ESM.xlsx-Sheet1-Supplementary_Data_1_Genes_comprising_the_Holoclone_signature': ('Single-keratinocyte transcriptomic analyses identify different clonal types and proliferative potential mediated by FOXM1 in human epidermal stem cells',\n",
       "  'Autologous epidermal cultures restore a functional epidermis on burned patients. Transgenic epidermal grafts do so also in genetic skin diseases such as Junctional Epidermolysis Bullosa. Clinical success strictly requires an adequate number of epidermal stem cells, detected as holoclone-forming cells, which can be only partially distinguished from the other clonogenic keratinocytes and cannot be prospectively isolated. Here we report that single-cell transcriptome analysis of primary human epidermal cultures identifies categories of genes clearly distinguishing the different keratinocyte clonal types, which are hierarchically organized along a continuous, mainly linear trajectory showing that stem cells sequentially generate progenitors producing terminally differentiated cells. Holoclone-forming cells display stem cell hallmarks as genes regulating DNA repair, chromosome segregation, spindle organization and telomerase activity. Finally, we identify FOXM1 as a YAP-dependent key regulator of epidermal stem cells. These findings improve criteria for measuring stem cells in epidermal cultures, which is an essential feature of the graft.',\n",
       "  'Sheet1-Supplementary_Data_1_Genes_comprising_the_Holoclone_signature',\n",
       "  'Differential expression analysis unveiled 55**1**  Supplementary Data 1'),\n",
       " 'PMC11239819-41467_2024_49400_MOESM4_ESM.xlsx-Holoclone_genes-Holoclone_signature': ('A scalable and cGMP-compatible autologous organotypic cell therapy for Dystrophic Epidermolysis Bullosa',\n",
       "  'We present Dystrophic Epidermolysis Bullosa Cell Therapy (DEBCT), a scalable platform producing autologous organotypic iPS cell-derived induced skin composite (iSC) grafts for definitive treatment. Clinical-grade manufacturing integrates CRISPR-mediated genetic correction with reprogramming into one step, accelerating derivation of COL7A1-edited iPS cells from patients. Differentiation into epidermal, dermal and melanocyte progenitors is followed by CD49f-enrichment, minimizing maturation heterogeneity. Mouse xenografting of iSCs from four patients with different mutations demonstrates disease modifying activity at 1 month. Next-generation sequencing, biodistribution and tumorigenicity assays establish a favorable safety profile at 1-9 months. Single cell transcriptomics reveals that iSCs are composed of the major skin cell lineages and include prominent holoclone stem cell-like signatures of keratinocytes, and the recently described Gibbin-dependent signature of fibroblasts. The latter correlates with enhanced graftability of iSCs. In conclusion, DEBCT overcomes manufacturing and safety roadblocks and establishes a reproducible, safe, and cGMP-compatible therapeutic approach to heal lesions of DEB patients.',\n",
       "  'Holoclone_genes-Holoclone_signature',\n",
       "  'None here'),\n",
       " 'PMC7066904-aging-12-102817-s005..xlsx-ImmuneDiff-Supplementary_Table_3_Differentially_expressed_genes_selected_based_on_immune_and_stromal_scores': ('Identification of prognostic immune-related genes in the tumor microenvironment of endometrial cancer',\n",
       "  'Endometrial cancer (EC) is one of the most common gynecologic malignancies. To identify potential prognostic biomarkers for EC, we analyzed the relationship between the EC tumor microenvironment and gene expression profiles. Using the ESTIMATE R tool, we found that immune and stromal scores correlated with clinical data and the prognosis of EC patients. Based on the immune and stromal scores, 387 intersection differentially expressed genes were identified. Eight immune-related genes were then identified using two machine learning algorithms. Functional enrichment analysis revealed that these genes were mainly associated with T cell activation and response. Kaplan-Meier survival analysis showed that expression of TMEM150B, CACNA2D2, TRPM5, NOL4, CTSW, and SIGLEC1 significantly correlated with overall survival times of EC patients. In addition, using the TIMER algorithm, we found that expression of TMEM150B, SIGLEC1, and CTSW correlated positively with the tumor infiltration levels of B cells, CD8+ T cells, CD4+ T cells, macrophages, and dendritic cells. These findings indicate that the composition of the tumor microenvironment affects the clinical outcomes of EC patients, and suggests that it may provide a basis for development of novel prognostic biomarkers and immunotherapies for EC patients.',\n",
       "  'ImmuneDiff-Supplementary_Table_3_Differentially_expressed_genes_selected_based_on_immune_and_stromal_scores',\n",
       "  'identify the immune-related and stromal-related genes, differential analysis by using “limma” package was performed (**Supplementary Table 3**)'),\n",
       " 'PMC9935276-mmc3.xlsx-V3A_BNT162b2-Unnamed_6': ('A systems immunology study comparing innate and adaptive immune responses in adults to COVID-19 mRNA and adenovirus vectored vaccines',\n",
       "  'Identifying the molecular mechanisms that promote optimal immune responses to coronavirus disease 2019 (COVID-19) vaccination is critical for future rational vaccine design. Here, we longitudinally profile innate and adaptive immune responses in 102 adults after the first, second, and third doses of mRNA or adenovirus-vectored COVID-19 vaccines. Using a multi-omics approach, we identify key differences in the immune responses induced by ChAdOx1-S and BNT162b2 that correlate with antigen-specific antibody and T cell responses or vaccine reactogenicity. Unexpectedly, we observe that vaccination with ChAdOx1-S, but not BNT162b2, induces an adenoviral vector-specific memory response after the first dose, which correlates with the expression of proteins with roles in thrombosis with potential implications for thrombosis with thrombocytopenia syndrome (TTS), a rare but serious adverse event linked to adenovirus-vectored vaccines. The COVID-19 Vaccine Immune Responses Study thus represents a major resource that can be used to understand the immunogenicity and reactogenicity of these COVID-19 vaccines.',\n",
       "  'V3A_BNT162b2-Unnamed_6',\n",
       "  'third (V3A) doses, generating more than 12 billion reads across 263 samples (Figure S3A, **Table S2**)  Table S2. RNA-seq data, differentially expressed genes at each timepoint, pathway, and blood transcriptional module (BTM) analysis, related to Figures 2 and 3'),\n",
       " 'PMC5390115-mmc5.xlsx-HUV1-Unnamed_1': ('A Genome-wide Analysis of Human Pluripotent Stem Cell-Derived Endothelial Cells in 2D or 3D Culture',\n",
       "  'A defined protocol for efficiently deriving endothelial cells from human pluripotent stem cells was established and vascular morphogenesis was used as a model system to understand how synthetic hydrogels influence global biological function compared with common 2D and 3D culture platforms. RNA sequencing demonstrated that gene expression profiles were similar for endothelial cells and pericytes cocultured in polyethylene glycol (PEG) hydrogels or Matrigel, while monoculture comparisons identified distinct vascular signatures for each cell type. Endothelial cells cultured on tissue-culture polystyrene adopted a proliferative phenotype compared with cells cultured on or encapsulated in PEG hydrogels. The proliferative phenotype correlated to increased FAK-ERK activity, and knockdown or inhibition of ERK signaling reduced proliferation and expression for cell-cycle genes while increasing expression for \"3D-like\" vasculature development genes. Our results provide insight into the influence of 2D and 3D culture formats on global biological processes that regulate cell function.',\n",
       "  'HUV1-Unnamed_1',\n",
       "  'upregulated genes for pericytes that overlapped on days 1–3 of culture in PEG hydrogels (**Table S4**)  Table S4. Differential Gene Expression for Endothelial Cells Compared with Pericytes, Related to Figure 3'),\n",
       " 'PMC9935276-mmc3.xlsx-V3A_BNT162b2-EdgeR_analysis_comparing_V0_vs_V3A_for_BNT162b2': ('A systems immunology study comparing innate and adaptive immune responses in adults to COVID-19 mRNA and adenovirus vectored vaccines',\n",
       "  'Identifying the molecular mechanisms that promote optimal immune responses to coronavirus disease 2019 (COVID-19) vaccination is critical for future rational vaccine design. Here, we longitudinally profile innate and adaptive immune responses in 102 adults after the first, second, and third doses of mRNA or adenovirus-vectored COVID-19 vaccines. Using a multi-omics approach, we identify key differences in the immune responses induced by ChAdOx1-S and BNT162b2 that correlate with antigen-specific antibody and T cell responses or vaccine reactogenicity. Unexpectedly, we observe that vaccination with ChAdOx1-S, but not BNT162b2, induces an adenoviral vector-specific memory response after the first dose, which correlates with the expression of proteins with roles in thrombosis with potential implications for thrombosis with thrombocytopenia syndrome (TTS), a rare but serious adverse event linked to adenovirus-vectored vaccines. The COVID-19 Vaccine Immune Responses Study thus represents a major resource that can be used to understand the immunogenicity and reactogenicity of these COVID-19 vaccines.',\n",
       "  'V3A_BNT162b2-EdgeR_analysis_comparing_V0_vs_V3A_for_BNT162b2',\n",
       "  'third (V3A) doses, generating more than 12 billion reads across 263 samples (Figure S3A, **Table S2**)  Table S2. RNA-seq data, differentially expressed genes at each timepoint, pathway, and blood transcriptional module (BTM) analysis, related to Figures 2 and 3'),\n",
       " 'PMC9945105-Table4.XLSX-up-Unnamed_0': ('Integrated bioinformatic analyses investigate macrophage-M1-related biomarkers and tuberculosis therapeutic drugs',\n",
       "  \"Tuberculosis (TB) is a common infectious disease linked to host genetics and the innate immune response. It is vital to investigate new molecular mechanisms and efficient biomarkers for Tuberculosis because the pathophysiology of the disease is still unclear, and there aren't any precise diagnostic tools. This study downloaded three blood datasets from the GEO database, two of which (GSE19435 and 83456) were used to build a weighted gene co-expression network for searching hub genes associated with macrophage M1 by the CIBERSORT and WGCNA algorithms. Furthermore, 994 differentially expressed genes (DEGs) were extracted from healthy and TB samples, four of which were associated with macrophage M1, naming RTP4, CXCL10, CD38, and IFI44. They were confirmed as upregulation in TB samples by external dataset validation (GSE34608) and quantitative real-time PCR analysis (qRT-PCR). CMap was used to predict potential therapeutic compounds for tuberculosis using 300 differentially expressed genes (150 downregulated and 150 upregulated genes), and six small molecules (RWJ-21757, phenamil, benzanthrone, TG-101348, metyrapone, and WT-161) with a higher confidence value were extracted. We used in-depth bioinformatics analysis to investigate significant macrophage M1-related genes and promising anti-Tuberculosis therapeutic compounds. However, more clinical trials were necessary to determine their effect on Tuberculosis.\",\n",
       "  'up-Unnamed_0',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC6544546-EMS81327-supplement-Supplementary_Table_3.xlsx-Neuron2-gene': ('Altered human oligodendrocyte heterogeneity in multiple sclerosis',\n",
       "  'Oligodendrocyte pathology is increasingly implicated in neurodegenerative diseases as oligodendrocytes both myelinate and provide metabolic support to axons. In multiple sclerosis (MS), demyelination in the central nervous system thus leads to neurodegeneration, but the severity of MS between patients is very variable. Disability does not correlate well with the extent of demyelination 1 , which suggests that other factors contribute to this variability. One such factor may be oligodendrocyte heterogeneity. Not all oligodendrocytes are the same-those from the mouse spinal cord inherently produce longer myelin sheaths than those from the cortex 2 , and single-cell analysis of the mouse central nervous system identified further differences 3,4 . However, the extent of human oligodendrocyte heterogeneity and its possible contribution to MS pathology remain unknown. Here we performed single-nucleus RNA sequencing from white matter areas of post-mortem human brain from patients with MS and from unaffected controls. We identified subclusters of oligodendroglia in control human white matter, some with similarities to mouse, and defined new markers for these cell states. Notably, some subclusters were underrepresented in MS tissue, whereas others were more prevalent. These differences in mature oligodendrocyte subclusters may indicate different functional states of oligodendrocytes in MS lesions. We found similar changes in normal-appearing white matter, showing that MS is a more diffuse disease than its focal demyelination suggests. Our findings of an altered oligodendroglial heterogeneity in MS may be important for understanding disease progression and developing therapeutic approaches.',\n",
       "  'Neuron2-gene',\n",
       "  'smooth muscle cells (VSM), pericytes, endothelial cells, and immune cells (Fig.1a, Extended Fig.1g and **Supplementary Table3**)'),\n",
       " 'PMC7592753-pone.0240647.s008.xlsx-ACE2_kidney_-_cortex_correlatio-correlated_gene': ('Bioinformatic characterization of angiotensin-converting enzyme 2, the entry receptor for SARS-CoV-2',\n",
       "  'The World Health Organization declared the COVID-19 epidemic a public health emergency of international concern on March 11th, 2020, and the pandemic is rapidly spreading worldwide. COVID-19 is caused by a novel coronavirus SARS-CoV-2, which enters human target cells via angiotensin converting enzyme 2 (ACE2). We used a number of bioinformatics tools to computationally characterize ACE2 by determining its cell-specific expression in trachea, lung, and small intestine, derive its putative functions, and predict transcriptional regulation. The small intestine expressed higher levels of ACE2 mRNA than any other organ. By immunohistochemistry, duodenum, kidney and testis showed strong signals, whereas the signal was weak in the respiratory tract. Single cell RNA-Seq data from trachea indicated positive signals along the respiratory tract in key protective cell types including club, goblet, proliferating, and ciliary epithelial cells; while in lung the ratio of ACE2-expressing cells was low in all cell types (<2.6%), but was highest in vascular endothelial and goblet cells. Gene ontology analysis suggested that, besides its classical role in the renin-angiotensin system, ACE2 may be functionally associated with angiogenesis/blood vessel morphogenesis. Using a novel tool for the prediction of transcription factor binding sites we identified several putative binding sites within two tissue-specific promoters of the ACE2 gene as well as a new putative short form of ACE2. These include several interferon-stimulated response elements sites for STAT1, IRF8, and IRF9. Our results also confirmed that age and gender play no significant role in the regulation of ACE2 mRNA expression in the lung.',\n",
       "  'ACE2_kidney_-_cortex_correlatio-correlated_gene',\n",
       "  'testis, colon, and kidney which are highly correlated (>0.8) with ACE2 (Tables 1 and **S7**)  (XLSX)  Click here for additional data file'),\n",
       " 'PMC8479788-mmc2.xlsx-Table_S1-purple': ('Systems-level network modeling deciphers the master regulators of phenotypic plasticity and heterogeneity in melanoma',\n",
       "  'Phenotypic (i.e. non-genetic) heterogeneity in melanoma drives dedifferentiation, recalcitrance to targeted therapy and immunotherapy, and consequent tumor relapse and metastasis. Various markers or regulators associated with distinct phenotypes in melanoma have been identified, but, how does a network of interactions among these regulators give rise to multiple \"attractor\" states and phenotypic switching remains elusive. Here, we inferred a network of transcription factors (TFs) that act as master regulators for gene signatures of diverse cell-states in melanoma. Dynamical simulations of this network predicted how this network can settle into different \"attractors\" (TF expression patterns), suggesting that TF network dynamics drives the emergence of phenotypic heterogeneity. These simulations can recapitulate major phenotypes observed in melanoma and explain de-differentiation trajectory observed upon BRAF inhibition. Our systems-level modeling framework offers a platform to understand trajectories of phenotypic transitions in the landscape of a regulatory TF network and identify novel therapeutic strategies targeting melanoma plasticity.',\n",
       "  'Table_S1-purple',\n",
       "  'the above-mentioned regulators by using public databases (ChEA, ENCODE) and examining relevant experimental literature (**Table S1**)  Table S1. List of genes belonging to different modules identified using WGCNA, related to STAR Methods (Identification of gene modules for phenotypes)'),\n",
       " 'PMC9122025-NIHMS1780000-supplement-Data_S3.xlsx-Data_S3-_CSF_Clonal_CD4_T_Cell-Gene': ('CD4 + T cells contribute to neurodegeneration in Lewy body dementia',\n",
       "  'Recent studies indicate that the adaptive immune system plays a role in Lewy body dementia (LBD). However, the mechanism regulating T cell brain homing in LBD is unknown. Here, we observed T cells adjacent to Lewy bodies and dopaminergic neurons in postmortem LBD brains. Single-cell RNA sequencing of cerebrospinal fluid (CSF) identified up-regulated expression of C-X-C motif chemokine receptor 4 ( CXCR4 ) in CD4 + T cells in LBD. CSF protein levels of the CXCR4 ligand, C-X-C motif chemokine ligand 12 (CXCL12), were associated with neuroaxonal damage in LBD. Furthermore, we observed clonal expansion and up-regulated interleukin 17A expression by CD4 + T cells stimulated with a phosphorylated α-synuclein epitope. Thus, CXCR4-CXCL12 signaling may represent a mechanistic target for inhibiting pathological interleukin-17–producing T cell trafficking in LBD.',\n",
       "  'Data_S3-_CSF_Clonal_CD4_T_Cell-Gene',\n",
       "  'showed increased expression of CD69 and CXCR4 in PD-DLB (Fig. 2F and G and **Data S3**)'),\n",
       " 'PMC8421417-41467_2021_25624_MOESM6_ESM.xlsx-Gene_signatures-Unnamed_15': ('Single-cell ATAC and RNA sequencing reveal pre-existing and persistent cells associated with prostate cancer relapse',\n",
       "  'Prostate cancer is heterogeneous and patients would benefit from methods that stratify those who are likely to respond to systemic therapy. Here, we employ single-cell assays for transposase-accessible chromatin (ATAC) and RNA sequencing in models of early treatment response and resistance to enzalutamide. In doing so, we identify pre-existing and treatment-persistent cell subpopulations that possess regenerative potential when subjected to treatment. We find distinct chromatin landscapes associated with enzalutamide treatment and resistance that are linked to alternative transcriptional programs. Transcriptional profiles characteristic of persistent cells are able to stratify the treatment response of patients. Ultimately, we show that defining changes in chromatin and gene expression in single-cell populations from pre-clinical models can reveal as yet unrecognized molecular predictors of treatment response. This suggests that the application of single-cell methods with high analytical resolution in pre-clinical models may powerfully inform clinical decision-making.',\n",
       "  'Gene_signatures-Unnamed_15',\n",
       "  'and PROSGenesis, we included (1) NEPC markers (Supplementary Fig. 1g), (2) a BRCAness gene signature**3**  Supplementary Data 3'),\n",
       " 'PMC8815209-12885_2022_9251_MOESM4_ESM.xls-Table_S3-Unnamed_8': ('FCER1G positively relates to macrophage infiltration in clear cell renal cell carcinoma and contributes to unfavorable prognosis by regulating tumor immunity',\n",
       "  \"Background: Tumor-associated macrophages (TAMs) are closely related to unfavorable prognosis of patients with clear cell renal cell carcinoma (ccRCC). However, the important molecules in the interaction between ccRCC and TAMs are unclear. Methods: TCGA-KIRC gene expression data of tumor tissues and normal tissues adjacent to tumor were compared to identify differentially expressed genes in ccRCC. TAMs related genes were discovered by analyzing the correlation between these differentially expressed genes and common macrophage biomarkers. Gene set enrichment analysis was performed to predict functions of TAMs related gene. The findings were further validated using RNA sequencing data obtained from the CheckMate 025 study and immunohistochemical analysis of samples from 350 patients with ccRCC. Kaplan-Meier survival curve, Cox regression analysis and Harrell's concordance index analysis were used to determine the prognostic significance. Results: In this study, we applied bioinformatic analysis to explore TAMs related differentially expressed genes in ccRCC and identified 5 genes strongly correlated with all selected macrophage biomarkers: STAC3, LGALS9, TREM2, FCER1G, and PILRA. Among them, FCER1G was abundantly expressed in tumor tissues and showed prognostic importance in patients with ccRCC who received treatment with Nivolumab; however, it did not exhibit prognostic value in those treated with Everolimus. We also discovered that high expression levels of FCER1G are related to T cell suppression. Moreover, combination of FCER1G and macrophage biomarker CD68 can improve the prognostic stratification of patients with ccRCC from TCGA-KIRC. Based on the immunohistochemical analysis of samples from patients with ccRCC, we further validated that FCER1G and CD68 are both highly expressed in tumor tissue and correlate with each other. Higher expression of CD68 or FCER1G in ccRCC tissue indicates shorter overall survival and progression-free survival; patients with high expression of both CD68 and FCER1G have the worst outcome. Combining CD68 and FCER1G facilitates the screening of patients with a worse prognosis from the same TNM stage group. Conclusions: High expression of FCER1G in ccRCC is closely related to TAMs infiltration and suppression of T cell activation and proliferation. Combining the expression levels of FCER1G and macrophage biomarker CD68 may be a promising postoperative prognostic index for patients with ccRCC.\",\n",
       "  'Table_S3-Unnamed_8',\n",
       "  'Additional file 4. Gene sets enriched in TCGA-KIRC with high FCER1G expression'),\n",
       " 'PMC9281170-12859_2022_4817_MOESM2_ESM.xlsx-Comet_B-gene_1': ('Detection of cell markers from single cell RNA-seq with sc2marker',\n",
       "  'Background: Single-cell RNA sequencing (scRNA-seq) allows the detection of rare cell types in complex tissues. The detection of markers for rare cell types is useful for further biological analysis of, for example, flow cytometry and imaging data sets for either physical isolation or spatial characterization of these cells. However, only a few computational approaches consider the problem of selecting specific marker genes from scRNA-seq data. Results: Here, we propose sc2marker, which is based on the maximum margin index and a database of proteins with antibodies, to select markers for flow cytometry or imaging. We evaluated the performances of sc2marker and competing methods in ranking known markers in scRNA-seq data of immune and stromal cells. The results showed that sc2marker performed better than the competing methods in accuracy, while having a competitive running time.',\n",
       "  'Comet_B-gene_1',\n",
       "  'method should have low rank scores; i.e., true marker genes are ranked as 1, **2**  Additional file 2: Results of markers detection for MCA-Lung dataset with sc2marker and other competing methods'),\n",
       " 'PMC4617899-12864_2015_2012_MOESM2_ESM.xlsx-CD8a_DC_SIG_minmax1_5x-Unnamed_1': ('BubbleGUM: automatic extraction of phenotype molecular signatures and comprehensive visualization of multiple Gene Set Enrichment Analyses',\n",
       "  \"Background: Recent advances in the analysis of high-throughput expression data have led to the development of tools that scaled-up their focus from single-gene to gene set level. For example, the popular Gene Set Enrichment Analysis (GSEA) algorithm can detect moderate but coordinated expression changes of groups of presumably related genes between pairs of experimental conditions. This considerably improves extraction of information from high-throughput gene expression data. However, although many gene sets covering a large panel of biological fields are available in public databases, the ability to generate home-made gene sets relevant to one's biological question is crucial but remains a substantial challenge to most biologists lacking statistic or bioinformatic expertise. This is all the more the case when attempting to define a gene set specific of one condition compared to many other ones. Thus, there is a crucial need for an easy-to-use software for generation of relevant home-made gene sets from complex datasets, their use in GSEA, and the correction of the results when applied to multiple comparisons of many experimental conditions. Result: We developed BubbleGUM (GSEA Unlimited Map), a tool that allows to automatically extract molecular signatures from transcriptomic data and perform exhaustive GSEA with multiple testing correction. One original feature of BubbleGUM notably resides in its capacity to integrate and compare numerous GSEA results into an easy-to-grasp graphical representation. We applied our method to generate transcriptomic fingerprints for murine cell types and to assess their enrichments in human cell types. This analysis allowed us to confirm homologies between mouse and human immunocytes. Conclusions: BubbleGUM is an open-source software that allows to automatically generate molecular signatures out of complex expression datasets and to assess directly their enrichment by GSEA on independent datasets. Enrichments are displayed in a graphical output that helps interpreting the results. This innovative methodology has recently been used to answer important questions in functional genomics, such as the degree of similarities between microarray datasets from different laboratories or with different experimental models or clinical cohorts. BubbleGUM is executable through an intuitive interface so that both bioinformaticians and biologists can use it. It is available at http://www.ciml.univ-mrs.fr/applications/BubbleGUM/index.html .\",\n",
       "  'CD8a_DC_SIG_minmax1_5x-Unnamed_1',\n",
       "  'None here'),\n",
       " 'PMC9669397-mmc4.xlsx-TSC_up_DEGs-Name': ('Human primed and naïve PSCs are both able to differentiate into trophoblast stem cells',\n",
       "  'The recent derivation of human trophoblast stem cells (TSCs) from placental cytotrophoblasts and blastocysts opened opportunities for studying the development and function of the human placenta. Recent reports have suggested that human naïve, but not primed, pluripotent stem cells (PSCs) retain an exclusive potential to generate TSCs. Here we report that, in the absence of WNT stimulation, transforming growth factor β (TGF-β) pathway inhibition leads to direct and robust conversion of primed human PSCs into TSCs. The resulting primed PSC-derived TSC lines exhibit self-renewal, can differentiate into the main trophoblast lineages, and present RNA and epigenetic profiles that are indistinguishable from recently established TSC lines derived from human placenta, blastocysts, or isogenic human naïve PSCs expanded under human enhanced naïve stem cell medium (HENSM) conditions. Activation of nuclear Yes-associated protein (YAP) signaling is sufficient for this conversion and necessary for human TSC maintenance. Our findings underscore a residual plasticity in primed human PSCs that allows their in vitro conversion into extra-embryonic trophoblast lineages.',\n",
       "  'TSC_up_DEGs-Name',\n",
       "  'signatures. Selected genes are highlighted. The complete DEG lists and enrichment are available in **Table S3**  Table S3. Differential gene expression between TSCs and PSCs and their annotation analysis, related to Figure 3'),\n",
       " 'PMC6787028-41467_2019_12518_MOESM10_ESM.xlsx-Erythroid_Gene_Sets-BFU-CFU_DOWN': ('N 6 -methyladenosine mRNA marking promotes selective translation of regulons required for human erythropoiesis',\n",
       "  'Many of the regulatory features governing erythrocyte specification, maturation, and associated disorders remain enigmatic. To identify new regulators of erythropoiesis, we utilize a functional genomic screen for genes affecting expression of the erythroid marker CD235a/GYPA. Among validating hits are genes coding for the N 6 -methyladenosine (m 6 A) mRNA methyltransferase (MTase) complex, including, METTL14, METTL3, and WTAP. We demonstrate that m 6 A MTase activity promotes erythroid gene expression programs through selective translation of ~300 m 6 A marked mRNAs, including those coding for SETD histone methyltransferases, ribosomal components, and polyA RNA binding proteins. Remarkably, loss of m 6 A marks results in dramatic loss of H3K4me3 marks across key erythroid-specific KLF1 transcriptional targets (e.g., Heme biosynthesis genes). Further, each m 6 A MTase subunit and a subset of their mRNAs targets are required for human erythroid specification in primary bone-marrow derived progenitors. Thus, m 6 A mRNA marks promote the translation of a network of genes required for human erythropoiesis.',\n",
       "  'Erythroid_Gene_Sets-BFU-CFU_DOWN',\n",
       "  '(Fig. 3c, d), similar to LMO2-KO (Fig. 3d, Supplementary Fig. 1b, c, Supplementary Data **7**)  Supplementary Data 7'),\n",
       " 'PMC10720203-13046_2023_2922_MOESM2_ESM.xlsx-Suppl_Table3-Unnamed_2': ('Nutlin-3a induces KRAS mutant/p53 wild type lung cancer specific methuosis-like cell death that is dependent on GFPT2',\n",
       "  'Background: Oncogenic KRAS mutation, the most frequent mutation in non-small cell lung cancer (NSCLC), is an aggressiveness risk factor and leads to the metabolic reprogramming of cancer cells by promoting glucose, glutamine, and fatty acid absorption and glycolysis. Lately, sotorasib was approved by the FDA as a first-in-class KRAS-G12C inhibitor. However, sotorasib still has a derivative barrier, which is not effective for other KRAS mutation types, except for G12C. Additionally, resistance to sotorasib is likely to develop, demanding the need for alternative therapeutic strategies. Methods: KRAS mutant, and wildtype NSCLC cells were used in vitro cell analyses. Cell viability, proliferation, and death were measured by MTT, cell counting, colony analyses, and annexin V staining for FACS. Cell tracker dyes were used to investigate cell morphology, which was examined by holotomograpy, and confocal microscopes. RNA sequencing was performed to identify key target molecule or pathway, which was confirmed by qRT-PCR, western blotting, and metabolite analyses by UHPLC-MS/MS. Zebrafish and mouse xenograft model were used for in vivo analysis. Results: In this study, we found that nutlin-3a, an MDM2 antagonist, inhibited the KRAS-PI3K/Akt-mTOR pathway and disrupted the fusion of both autophagosomes and macropinosomes with lysosomes. This further elucidated non-apoptotic and catastrophic macropinocytosis associated methuosis-like cell death, which was found to be dependent on GFPT2 of the hexosamine biosynthetic pathway, specifically in KRAS mutant /p53 wild type NSCLC cells. Conclusion: These results indicate the potential of nutlin-3a as an alternative agent for treating KRAS mutant/p53 wild type NSCLC cells.',\n",
       "  'Suppl_Table3-Unnamed_2',\n",
       "  'KRAS WT nor p53 MT NSCLC cells, showed a significant decrease in viability (Fig. **1**)  Additional file 2'),\n",
       " 'PMC5316832-ncomms14421-s10.xlsx-Cluster1-5-Unnamed_25': ('Recurrently deregulated lncRNAs in hepatocellular carcinoma',\n",
       "  \"Hepatocellular carcinoma (HCC) cells often invade the portal venous system and subsequently develop into portal vein tumour thrombosis (PVTT). Long noncoding RNAs (lncRNAs) have been associated with HCC, but a comprehensive analysis of their specific association with HCC metastasis has not been conducted. Here, by analysing 60 clinical samples' RNA-seq data from 20 HCC patients, we have identified and characterized 8,603 candidate lncRNAs. The expression patterns of 917 recurrently deregulated lncRNAs are correlated with clinical data in a TCGA cohort and published liver cancer data. Matched array data from the 60 samples show that copy number variations (CNVs) and alterations in DNA methylation contribute to the observed recurrent deregulation of 235 lncRNAs. Many recurrently deregulated lncRNAs are enriched in co-expressed clusters of genes related to cell adhesion, immune response and metabolic processes. Candidate lncRNAs related to metastasis, such as HAND2-AS1, were further validated using RNAi-based loss-of-function assays. Thus, we provide a valuable resource of functional lncRNAs and biomarkers associated with HCC tumorigenesis and metastasis.\",\n",
       "  'Cluster1-5-Unnamed_25',\n",
       "  'Supplementary Data 9Gene and lncRNA lists and clusters in co-expression network'),\n",
       " 'PMC8382756-41467_2021_25377_MOESM3_ESM.xlsx-Supplementary_Data_4-Supplementary_Data_4_Gene_list_that_are_correlated_with_CHMP1A_and_DPEP1_in_human_kidney_samples': ('A single genetic locus controls both expression of DPEP1/CHMP1A and kidney disease development via ferroptosis',\n",
       "  'Genome-wide association studies (GWAS) have identified loci for kidney disease, but the causal variants, genes, and pathways remain unknown. Here we identify two kidney disease genes Dipeptidase 1 (DPEP1) and Charged Multivesicular Body Protein 1 A (CHMP1A) via the triangulation of kidney function GWAS, human kidney expression, and methylation quantitative trait loci. Using single-cell chromatin accessibility and genome editing, we fine map the region that controls the expression of both genes. Mouse genetic models demonstrate the causal roles of both genes in kidney disease. Cellular studies indicate that both Dpep1 and Chmp1a are important regulators of a single pathway, ferroptosis and lead to kidney disease development via altering cellular iron trafficking.',\n",
       "  'Supplementary_Data_4-Supplementary_Data_4_Gene_list_that_are_correlated_with_CHMP1A_and_DPEP1_in_human_kidney_samples',\n",
       "  'As both mQTL and eQTLs appeared to be tagged by rs**1**  Supplementary Data 1-6'),\n",
       " 'PMC9357053-NIHMS1810045-supplement-07.xlsx-Ciliated-gene': ('Cell function and identity revealed by comparative scRNA-seq analysis in human nasal, bronchial and epididymis epithelia',\n",
       "  'The evolutionary relationship of cells within tissues having a similar function but located in different anatomical sites is of considerable biological interest. The development of single-cell RNA sequencing (scRNA-seq) protocols has greatly enhanced opportunities to address this topic. Here we focus on cells in the epithelium which lines two regions of the human respiratory tract and the male genital ducts to delineate the shared, differentiated functions of the different cell populations. Transcriptomic data were used to assess the gene expression profiles of human bronchial, nasal, and epididymal epithelium (HBE, HNE, and HEE). Bulk RNA-seq showed many shared genes expressed in cells from the nasal and bronchial epithelium and highlighted their divergence from the epididymal epithelium. ScRNA-seq in HBE and HNE cells demonstrated overlapping gene expression patterns within basal and secretory cell populations. Moreover, the distribution of cell types was altered in HNE cells derived from donors with cystic fibrosis (CF) when compared to cells from healthy donors. Next, the HBE and HNE datasets were merged and confirmed intersection of cell type gene expression profiles from the two sites. However, secretory and ciliated cells were the most abundant types in the HBE samples, while more basal cells were seen in the HNE populations. We then merged single-cell data from the epididymis to determine if overlapping functions of these cells corresponded to those in the airway. Of note, only the pulmonary ionocytes/epididymis clear cells showed a strongly conserved identity, which was confirmed by imputation in bulk RNA-seq datasets from the same cells.',\n",
       "  'Ciliated-gene',\n",
       "  'both cell types. In both HBE and HNE datasets (Fig. S2, Table S3 and **S4**)'),\n",
       " 'PMC9669397-mmc4.xlsx-TSC_up_DEGs-ensembl_ID': ('Human primed and naïve PSCs are both able to differentiate into trophoblast stem cells',\n",
       "  'The recent derivation of human trophoblast stem cells (TSCs) from placental cytotrophoblasts and blastocysts opened opportunities for studying the development and function of the human placenta. Recent reports have suggested that human naïve, but not primed, pluripotent stem cells (PSCs) retain an exclusive potential to generate TSCs. Here we report that, in the absence of WNT stimulation, transforming growth factor β (TGF-β) pathway inhibition leads to direct and robust conversion of primed human PSCs into TSCs. The resulting primed PSC-derived TSC lines exhibit self-renewal, can differentiate into the main trophoblast lineages, and present RNA and epigenetic profiles that are indistinguishable from recently established TSC lines derived from human placenta, blastocysts, or isogenic human naïve PSCs expanded under human enhanced naïve stem cell medium (HENSM) conditions. Activation of nuclear Yes-associated protein (YAP) signaling is sufficient for this conversion and necessary for human TSC maintenance. Our findings underscore a residual plasticity in primed human PSCs that allows their in vitro conversion into extra-embryonic trophoblast lineages.',\n",
       "  'TSC_up_DEGs-ensembl_ID',\n",
       "  'signatures. Selected genes are highlighted. The complete DEG lists and enrichment are available in **Table S3**  Table S3. Differential gene expression between TSCs and PSCs and their annotation analysis, related to Figure 3'),\n",
       " 'PMC4859332-mmc1.xlsx-Murine_tissue_spec_SIG-SP_RPM': ('Comparative genomics analysis of mononuclear phagocyte subsets confirms homology between lymphoid tissue-resident and dermal XCR1(+) DCs in mouse and human and distinguishes them from Langerhans cells',\n",
       "  'Dendritic cells (DC) are mononuclear phagocytes which exhibit a branching (dendritic) morphology and excel at naïve T cell activation. DC encompass several subsets initially identified by their expression of cell surface molecules and later shown to possess distinct functions. DC subset differentiation is orchestrated by transcription factors, growth factors and cytokines. Identifying DC subsets is challenging as very few cell surface molecules are uniquely expressed on any one of these cell populations. There is no standard consensus to identify mononuclear phagocyte subsets; varying antigens are employed depending on the tissue and animal species studied and between laboratories. This has led to confusion in how to accurately define and classify DCs across tissues and between species. Here we report a comparative genomics strategy that enables universal definition of DC and other mononuclear phagocyte subsets across species. We performed a meta-analysis of several public datasets of human and mouse mononuclear phagocyte subsets isolated from blood, spleen, skin or cutaneous lymph nodes, including by using a novel and user friendly software, BubbleGUM, which generates and integrates gene signatures for high throughput gene set enrichment analysis. This analysis demonstrates the equivalence between human and mouse skin XCR1(+) DCs, and between mouse and human Langerhans cells.',\n",
       "  'Murine_tissue_spec_SIG-SP_RPM',\n",
       "  'Supplementary material'),\n",
       " 'PMC10861553-42003_2024_5840_MOESM9_ESM.xlsx-C5_cilium-Gene_Symbol': ('Genomic attributes of airway commensal bacteria and mucosa',\n",
       "  'Microbial communities at the airway mucosal barrier are conserved and highly ordered, in likelihood reflecting co-evolution with human host factors. Freed of selection to digest nutrients, the airway microbiome underpins cognate management of mucosal immunity and pathogen resistance. We show here the initial results of systematic culture and whole-genome sequencing of the thoracic airway bacteria, identifying 52 novel species amongst 126 organisms that constitute 75% of commensals typically present in heathy individuals. Clinically relevant genes encode antimicrobial synthesis, adhesion and biofilm formation, immune modulation, iron utilisation, nitrous oxide (NO) metabolism and sphingolipid signalling. Using whole-genome content we identify dysbiotic features that may influence asthma and chronic obstructive pulmonary disease. We match isolate gene content to transcripts and metabolites expressed late in airway epithelial differentiation, identifying pathways to sustain host interactions with microbiota. Our results provide a systematic basis for decrypting interactions between commensals, pathogens, and mucosa in lung diseases of global significance.',\n",
       "  'C5_cilium-Gene_Symbol',\n",
       "  'None here'),\n",
       " 'PMC9039265-DataSheet_1.docx-1-Genes': ('Identification of a Tumor Immunological Phenotype-Related Gene Signature for Predicting Prognosis, Immunotherapy Efficacy, and Drug Candidates in Hepatocellular Carcinoma',\n",
       "  'Hepatocellular carcinoma (HCC) is the predominant subtype of primary liver cancer and represents a highly heterogeneous disease, making it hard to predict the prognosis and therapy efficacy. Here, we established a novel tumor immunological phenotype-related gene index (TIPRGPI) consisting of 11 genes by Univariate Cox regression and the least absolute shrinkage and selection operator (LASSO) algorithm to predict HCC prognosis and immunotherapy response. TIPRGPI was validated in multiple datasets and exhibited outstanding performance in predicting the overall survival of HCC. Multivariate analysis verified it as an independent predictor and a TIPRGPI-integrated nomogram was constructed to provide a quantitative tool for clinical practice. Distinct mutation profiles, hallmark pathways, and infiltration of immune cells in tumor microenvironment were shown between the TIPRGPI high and low-risk groups. Notably, significant differences in tumor immunogenicity and tumor immune dysfunction and exclusion (TIDE) were observed between the two risk groups, suggesting a better response to immune checkpoint blockade (ICB) therapy of the low-risk group. Besides, six potential drugs binding to the core target of the TIPRGPI signature were predicted via molecular docking. Taken together, our study shows that the proposed TIPRGPI was a reliable signature to predict the risk classification, immunotherapy response, and drugs candidate with potential application in the clinical decision and treatment of HCC. The novel \"TIP genes\"-guided strategy for predicting the survival and immunotherapy efficacy, we reported here, might be also applied to more cancers other than HCC.',\n",
       "  '1-Genes',\n",
       "  'from the GEO database (https://www.ncbi.nlm.nih.gov/geo/). The summary of the demographic information was listed in **Supplementary Table 1**  Click here for additional data file'),\n",
       " 'PMC7347797-Table_1.XLSX-Supplementary_Table_S3-Network_analysis_of_curated_cell_cycle_genes': ('The Transcriptional Network That Controls Growth Arrest and Macrophage Differentiation in the Human Myeloid Leukemia Cell Line THP-1',\n",
       "  'The response of the human acute myeloid leukemia cell line THP-1 to phorbol esters has been widely studied to test candidate leukemia therapies and as a model of cell cycle arrest and monocyte-macrophage differentiation. Here we have employed Cap Analysis of Gene Expression (CAGE) to analyze a dense time course of transcriptional regulation in THP-1 cells treated with phorbol myristate acetate (PMA) over 96 h. PMA treatment greatly reduced the numbers of cells entering S phase and also blocked cells exiting G2/M. The PMA-treated cells became adherent and expression of mature macrophage-specific genes increased progressively over the duration of the time course. Within 1-2 h PMA induced known targets of tumor protein p53 (TP53), notably CDKN1A , followed by gradual down-regulation of cell-cycle associated genes. Also within the first 2 h, PMA induced immediate early genes including transcription factor genes encoding proteins implicated in macrophage differentiation ( EGR2, JUN, MAFB ) and down-regulated genes for transcription factors involved in immature myeloid cell proliferation ( MYB, IRF8, GFI1 ). The dense time course revealed that the response to PMA was not linear and progressive. Rather, network-based clustering of the time course data highlighted a sequential cascade of transient up- and down-regulated expression of genes encoding feedback regulators, as well as transcription factors associated with macrophage differentiation and their inferred target genes. CAGE also identified known and candidate novel enhancers expressed in THP-1 cells and many novel inducible genes that currently lack functional annotation and/or had no previously known function in macrophages. The time course is available on the ZENBU platform allowing comparison to FANTOM4 and FANTOM5 data.',\n",
       "  'Supplementary_Table_S3-Network_analysis_of_curated_cell_cycle_genes',\n",
       "  'were confirmed in the more stringent time course set. The results are available in **Supplementary Tables S7A,B**  Comparison between enhancers in the THP-1 time course and enhancers in the main FANTOM5 set  Click here for additional data file'),\n",
       " 'PMC10585762-12885_2023_11315_MOESM1_ESM.docx-0-Gene2': ('Overexpression of SH2D1A promotes cancer progression and is associated with immune cell infiltration in hepatocellular carcinoma via bioinformatics and in vitro study',\n",
       "  'Background: SH2 domain containing 1A (SH2D1A) expression has been linked to cancer progression. However, the functions of SH2D1A in hepatocellular carcinoma (HCC) have not been reported. Methods: The effects of SH2D1A on the proliferation, migration, and invasion of HCC cells and the related pathways were re-explored in cell models with SH2D1A overexpression using the CCK-8, migration and invasion assays and western blotting. The functions and mechanisms of genes co-expressed with SH2D1A were analyzed using gene ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analysis. The relationship between SH2D1A expression and immune microenvironment features in HCC was explored. Results: Elevated SH2D1A expression promoted cell proliferation, migration, and invasion, which was related to the overexpression of p-Nf-κB and BCL2A1 protein levels in HCC. SH2D1A expression was related to the immune, stromal, and ESTIMATE scores, and the abundance of immune cells, such as B cells, CD8 + T cells, and T cells. SH2D1A expression was significantly related to the expression of immune cell markers, such as PDCD1, CD8A, and CTLA4 in HCC. Conclusion: SH2D1A overexpression was found to promote cell growth and metastasis via the Nf-κB signaling pathway and may be related to the immune microenvironment in HCC. The findings indicate that SH2D1A can function as a biomarker in HCC.',\n",
       "  '0-Gene2',\n",
       "  'Five hundred and sixty-seven genes co-expressed with SH2D**1**  Additional file 1: Figure S1. SH2D1A expression is significantly correlated with immune cell abundance according to data from the TCGA database. Table S1. Five hundred and sixty-seven SH2D1A co-expressed genes. Table S2. The functions of SH2D1A co-expressed genes'),\n",
       " 'PMC5470989-oncotarget-08-34534-s009.xlsx-LIHC-Symbol': ('Transcriptional landscape of human cancers',\n",
       "  'The homogeneity and heterogeneity in somatic mutations, copy number alterations and methylation across different cancer types have been extensively explored. However, the related exploration based on transcriptome data is lacking. In this study we explored gene expression profiles across 33 human cancer types using The Cancer Genome Atlas (TCGA) data. We identified consistently upregulated genes (such as E2F1, EZH2, FOXM1, MYBL2, PLK1, TTK, AURKA/B and BUB1) and consistently downregulated genes (such as SCARA5, MYOM1, NKAPL, PEG3, USP2, SLC5A7 and HMGCLL1) across various cancers. The dysregulation of these genes is likely to be associated with poor clinical outcomes in cancer. The dysregulated pathways commonly in cancers include cell cycle, DNA replication, repair, and recombination, Notch signaling, p53 signaling, Wnt signaling, TGFβ signaling, immune response etc. We also identified genes consistently upregulated or downregulated in highly-advanced cancers compared to lowly-advanced cancers. The highly (low) expressed genes in highly-advanced cancers are likely to have higher (lower) expression levels in cancers than in normal tissue, indicating that common gene expression perturbations drive cancer initiation and cancer progression. In addition, we identified a substantial number of genes exclusively dysregulated in a single cancer type or inconsistently dysregulated in different cancer types, demonstrating the intertumor heterogeneity. More importantly, we found a number of genes commonly dysregulated in various cancers such as PLP1, MYOM1, NKAPL and USP2 which were investigated in few cancer related studies, and thus represent our novel findings. Our study provides comprehensive portraits of transcriptional landscape of human cancers.',\n",
       "  'LIHC-Symbol',\n",
       "  'Supplementary Tables 10 and **11**'),\n",
       " 'PMC6787028-41467_2019_12518_MOESM10_ESM.xlsx-Erythroid_Gene_Sets-CD34-BFU_DOWN': ('N 6 -methyladenosine mRNA marking promotes selective translation of regulons required for human erythropoiesis',\n",
       "  'Many of the regulatory features governing erythrocyte specification, maturation, and associated disorders remain enigmatic. To identify new regulators of erythropoiesis, we utilize a functional genomic screen for genes affecting expression of the erythroid marker CD235a/GYPA. Among validating hits are genes coding for the N 6 -methyladenosine (m 6 A) mRNA methyltransferase (MTase) complex, including, METTL14, METTL3, and WTAP. We demonstrate that m 6 A MTase activity promotes erythroid gene expression programs through selective translation of ~300 m 6 A marked mRNAs, including those coding for SETD histone methyltransferases, ribosomal components, and polyA RNA binding proteins. Remarkably, loss of m 6 A marks results in dramatic loss of H3K4me3 marks across key erythroid-specific KLF1 transcriptional targets (e.g., Heme biosynthesis genes). Further, each m 6 A MTase subunit and a subset of their mRNAs targets are required for human erythroid specification in primary bone-marrow derived progenitors. Thus, m 6 A mRNA marks promote the translation of a network of genes required for human erythropoiesis.',\n",
       "  'Erythroid_Gene_Sets-CD34-BFU_DOWN',\n",
       "  '(Fig. 3c, d), similar to LMO2-KO (Fig. 3d, Supplementary Fig. 1b, c, Supplementary Data **7**)  Supplementary Data 7'),\n",
       " 'PMC9538974-JCB_202111137_TableS3.xlsx-Lists_of_DE_genes-Unnamed_4': ('Coordinated metabolic transitions and gene expression by NAD+ during adipogenesis',\n",
       "  'Adipocytes are the main cell type in adipose tissue, which is a critical regulator of metabolism, highly specialized in storing energy as fat. Adipocytes differentiate from multipotent mesenchymal stromal cells (hMSCs) through adipogenesis, a tightly controlled differentiation process involving close interplay between metabolic transitions and sequential programs of gene expression. However, the specific gears driving this interplay remain largely obscure. Additionally, the metabolite nicotinamide adenine dinucleotide (NAD+) is becoming increasingly recognized as a regulator of lipid metabolism, and a promising therapeutic target for dyslipidemia and obesity. Here, we explored how NAD+ bioavailability controls adipogenic differentiation from hMSC. We found a previously unappreciated repressive role for NAD+ on adipocyte commitment, while a functional NAD+-dependent deacetylase SIRT1 appeared crucial for terminal differentiation of pre-adipocytes. Repressing NAD+ biosynthesis during adipogenesis promoted the adipogenic transcriptional program, while two-photon microscopy and extracellular flux analyses suggest that SIRT1 activity mostly relies on the metabolic switch. Interestingly, SIRT1 controls subcellular compartmentalization of redox metabolism during adipogenesis.',\n",
       "  'Lists_of_DE_genes-Unnamed_4',\n",
       "  '57 DE genes compared with the untreated cells (Fig. S2 B, AD8_EX vs. AD8; **Table S3**)  lists DE genes in from EX527-treated cells at terminal differentiation (day 16) and their functional analyses  Click here for additional data file'),\n",
       " 'PMC10637813-aging-15-205126-s002.xlsx-TableS2-Supplementary_Table_2_Results_of_up-regulated_DEGs_between_TAAD_and_HC': ('Integrated bulk and scRNA sequence identified anoikis-related diagnostic biomarkers and potential association with immune infiltration in type A aortic dissection',\n",
       "  'Type-A aortic dissection (TAAD) is common life-threatening cardiovascular diseases with high-morbidity and mortality but the concrete etiology of disease remains unclear, which might disturb or delay the early diagnosis for TAAD. Anoikis is a special form of programmed cell-death (PCD) induced by detachment of anchorage-dependent cells from the extracellular matrix (ECM) or neighboring cells, and has been widely applied to identify anoikis-related biomarkers for the prediction and prognosis in oncological fields. However, the specific roles of anoikis-related genes (ARGs) in TAAD remain unclear. In this study, we first identified and validated eight diagnostic ARGs for TAAD based on multiple RNA-sequence datasets, including CHEK2, HIF1A, HK2, HMGA1, SERPINA1, PTPN1, SLC2A1 and VEGFA. The comprehensive functional annotation was evaluated by the integrated functional enrichments analysis. We identified the activation of inflammatory-related pathways, metabolic reprogramming and angiogenesis, and the inhibition of cardiovascular development pathways in TAAD. Immune cell infiltration (ICI) analysis further demonstrated that innate immune-cells were more dominant than adaptive immune-cells in TAAD tissues, especially in macrophages, monocytes, activated-DC, NKT cells and CD56+dim NK cells. The cellular landscape was further validated by single-cell RNA sequence technology with significant associations with anoikis in TAAD patients. Four vital ARGs ( HIF1A, HMGA1, SERPINA1 and VEGFA ) were ultimately identified along with the changes of differentiation trajectory, and major expressions were conformably concentrated on Macro1-3, Mono1-2 and Mono4 subtypes. These findings provide a promising diagnostic biomarker for the accurately diagnosing the disease and would be helpful to further explore the potential pathogenesis with anoikis process for TAAD.',\n",
       "  'TableS2-Supplementary_Table_2_Results_of_up-regulated_DEGs_between_TAAD_and_HC',\n",
       "  'for TAAD and exhibited in the volcano plots with top genes (Figure 2B and **Supplementary Tables 2**)'),\n",
       " 'PMC8323949-pcbi.1009228.s009.xlsx-X12-Unnamed_0': ('Single-cell analysis reveals the pan-cancer invasiveness-associated transition of adipose-derived stromal cells into COL11A1-expressing cancer-associated fibroblasts',\n",
       "  'During the last ten years, many research results have been referring to a particular type of cancer-associated fibroblasts associated with poor prognosis, invasiveness, metastasis and resistance to therapy in multiple cancer types, characterized by a gene expression signature with prominent presence of genes COL11A1, THBS2 and INHBA. Identifying the underlying biological mechanisms responsible for their creation may facilitate the discovery of targets for potential pan-cancer therapeutics. Using a novel computational approach for single-cell gene expression data analysis identifying the dominant cell populations in a sequence of samples from patients at various stages, we conclude that these fibroblasts are produced by a pan-cancer cellular transition originating from a particular type of adipose-derived stromal cells naturally present in the stromal vascular fraction of normal adipose tissue, having a characteristic gene expression signature. Focusing on a rich pancreatic cancer dataset, we provide a detailed description of the continuous modification of the gene expression profiles of cells as they transition from APOD-expressing adipose-derived stromal cells to COL11A1-expressing cancer-associated fibroblasts, identifying the key genes that participate in this transition. These results also provide an explanation to the well-known fact that the adipose microenvironment contributes to cancer progression.',\n",
       "  'X12-Unnamed_0',\n",
       "  'two clusters (X6 and X9) expressing COL11A1-associated genes and ASC-associated genes, respectively (S3B Fig, **S6 Table**)  Differentially expressed genes among different clusters of ovarian cancer dataset.(XLSX)  Click here for additional data file'),\n",
       " 'PMC8323949-pcbi.1009228.s009.xlsx-OV_ALL_3066-X12': ('Single-cell analysis reveals the pan-cancer invasiveness-associated transition of adipose-derived stromal cells into COL11A1-expressing cancer-associated fibroblasts',\n",
       "  'During the last ten years, many research results have been referring to a particular type of cancer-associated fibroblasts associated with poor prognosis, invasiveness, metastasis and resistance to therapy in multiple cancer types, characterized by a gene expression signature with prominent presence of genes COL11A1, THBS2 and INHBA. Identifying the underlying biological mechanisms responsible for their creation may facilitate the discovery of targets for potential pan-cancer therapeutics. Using a novel computational approach for single-cell gene expression data analysis identifying the dominant cell populations in a sequence of samples from patients at various stages, we conclude that these fibroblasts are produced by a pan-cancer cellular transition originating from a particular type of adipose-derived stromal cells naturally present in the stromal vascular fraction of normal adipose tissue, having a characteristic gene expression signature. Focusing on a rich pancreatic cancer dataset, we provide a detailed description of the continuous modification of the gene expression profiles of cells as they transition from APOD-expressing adipose-derived stromal cells to COL11A1-expressing cancer-associated fibroblasts, identifying the key genes that participate in this transition. These results also provide an explanation to the well-known fact that the adipose microenvironment contributes to cancer progression.',\n",
       "  'OV_ALL_3066-X12',\n",
       "  'two clusters (X6 and X9) expressing COL11A1-associated genes and ASC-associated genes, respectively (S3B Fig, **S6 Table**)  Differentially expressed genes among different clusters of ovarian cancer dataset.(XLSX)  Click here for additional data file'),\n",
       " 'PMC8542044-41419_2021_4290_MOESM3_ESM.xlsx-Table_S5-Unnamed_2': ('Quantitative analysis of phosphoproteome in necroptosis reveals a role of TRIM28 phosphorylation in promoting necroptosis-induced cytokine production',\n",
       "  'Necroptosis is a form of regulated necrotic cell death that promotes inflammation. In cells undergoing necroptosis, activated RIPK1 kinase mediates the formation of RIPK1/RIPK3/MLKL complex to promote MLKL oligomerization and execution of necroptosis. RIPK1 kinase activity also promotes cell-autonomous activation of proinflammatory cytokine production in necroptosis. However, the signaling pathways downstream of RIPK1 kinase in necroptosis and how RIPK1 kinase activation controls inflammatory response induced by necroptosis are still largely unknown. Here, we quantitatively measured the temporal dynamics of over 7000 confident phosphorylation-sites during necroptosis using mass spectrometry. Our study defined a RIPK1-dependent phosphorylation pattern in late necroptosis that is associated with a proinflammatory component marked by p-S473 TRIM28. We show that the activation of p38 MAPK mediated by oligomerized MLKL promotes the phosphorylation of S473 TRIM28, which in turn mediates inflammation during late necroptosis. Taken together, our study illustrates a mechanism by which p38 MAPK may be activated by oligomerized MLKL to promote inflammation in necroptosis.',\n",
       "  'Table_S5-Unnamed_2',\n",
       "  'Quantitative mass spectrometry analysis was performed at indicated time points (Fig. S**1**)  Supplementary tables'),\n",
       " 'PMC8133874-NIHMS1693697-supplement-4.xlsx-Gene_Module_9-AADACP1': ('High-dimensional profiling clusters asthma severity by lymphoid and non-lymphoid status',\n",
       "  'Clinical definitions of asthma fail to capture the heterogeneity of immune dysfunction in severe, treatment-refractory disease. Applying mass cytometry and machine learning to bronchoalveolar lavage (BAL) cells, we find that corticosteroid-resistant asthma patients cluster largely into two groups: one enriched in interleukin (IL)-4 + innate immune cells and another dominated by interferon (IFN)-γ + T cells, including tissue-resident memory cells. In contrast, BAL cells of a healthier population are enriched in IL-10 + macrophages. To better understand cellular mediators of severe asthma, we developed the Immune Cell Linkage through Exploratory Matrices (ICLite) algorithm to perform deconvolution of bulk RNA sequencing of mixed-cell populations. Signatures of mitosis and IL-7 signaling in CD206 - FcεRI + CD127 + IL-4 + innate cells in one patient group, contrasting with adaptive immune response in T cells in the other, are preserved across technologies. Transcriptional signatures uncovered by ICLite identify T-cell-high and T-cell-poor severe asthma patients in an independent cohort, suggesting broad applicability of our findings.',\n",
       "  'Gene_Module_9-AADACP1',\n",
       "  'and 40 negative correlations among 24 cell populations and 26 gene modules (Figure 5A; **Table S5**)'),\n",
       " 'PMC6707279-41598_2019_48243_MOESM2_ESM.xls-coexpression_AK7_mRNA_expressio-Gene_Symbol': ('A co-expressed gene status of adenylate kinase 1/4 reveals prognostic gene signature associated with prognosis and sensitivity to EGFR targeted therapy in lung adenocarcinoma',\n",
       "  'Cancer cells utilize altered bioenergetics to fuel uncontrolled proliferation and progression. At the core of bioenergetics, adenine nucleotides are the building blocks for nucleotide synthesis, energy transfer and diverse metabolic processes. Adenylate kinases (AK) are ubiquitous phosphotransferases that catalyze the conversion of adenine nucleotides and regulate the homeostasis of nucleotide ratios within cellular compartments. Recently, different isoforms of AK have been shown to induce metabolic reprograming in cancer and were identified as biomarkers for predicting disease progression. Here we aim to systemically analyze the impact of all AK-associated gene signatures on lung adenocarcinoma patient survival and decipher the value for therapeutic interventions. By analyzing TCGA Lung Adenocarcinoma (LUAD) RNA Seq data, we found gene signatures from AK4 and AK1 have higher percentage of prognostic genes compared to other AK-gene signatures. A 118-gene signature was identified from consensus gene expression in AK1 and AK4 prognostic gene signatures. Immunohistochemistry (IHC) analyses in 140 lung adenocarcinoma patients showed overexpression of AK4 significantly correlated with worse overall survival (P = 0.001) whereas overexpression of AK1 significantly associated with good prognosis (P = 0.009). Furthermore, reduced AK4 expression by shRNA reduced the EGFR protein expression in EGFR mutation cells. The inhibition of AK4-AK1 signal might provide a potential target for synergistic effect in target therapy in lung cancer patients.',\n",
       "  'coexpression_AK7_mRNA_expressio-Gene_Symbol',\n",
       "  '(LUAD) dataset and identified unique signatures for each AK (Fig. 1a and Supplementary Table **S1**)  Table S1'),\n",
       " 'PMC7743079-ccb0fab282708b596561a2f0.xlsx-upregulated_genes-647_genes_are_up-regulated_by_IFNb_adjusted_p-value_0_05_log2FoldChange_1_and_normalized_number_5': ('Activation of Interferon-Stimulated Transcriptomes and ACE2 Isoforms in Human Airway Epithelium Is Curbed by Janus Kinase Inhibitors',\n",
       "  'SARS-CoV-2 infection of human airway epithelium activates genetic programs that lead to progressive hyperinflammation in COVID-19 patients. Here we report on genetic programs activated by interferons and the suppression by Janus kinase (JAK) inhibitors. The angiotensin-converting enzyme 2 (ACE2) is the receptor for SARS-CoV-2 and deciphering its regulation is paramount for understanding the cell tropism of SARS-CoV-2 infection. We identified candidate regulatory elements in the ACE2 locus in human primary airway cells and lung tissue. Activating histone and promoter marks and Pol II loading characterize the intronic dACE2 and define novel candidate enhancers distal to the genuine ACE2 promoter and within additional introns. dACE2, and to a lesser extent ACE2, RNA levels increased in primary cells treated with interferons and this induction was mitigated by JAK inhibitors that are used therapeutically in COVID-19 patients. Our analyses provide insight into ACE2 regulatory elements and highlight JAK inhibitors as suitable tools to suppress interferon-activated genetic programs in bronchial cells.',\n",
       "  'upregulated_genes-647_genes_are_up-regulated_by_IFNb_adjusted_p-value_0_05_log2FoldChange_1_and_normalized_number_5',\n",
       "  'None here'),\n",
       " 'PMC7743079-e53838f794689a761e804083.xlsx-upregulated_genes-647_genes_are_up-regulated_by_IFNb_adjusted_p-value_0_05_log2FoldChange_1_and_normalized_number_5': ('Activation of Interferon-Stimulated Transcriptomes and ACE2 Isoforms in Human Airway Epithelium Is Curbed by Janus Kinase Inhibitors',\n",
       "  'SARS-CoV-2 infection of human airway epithelium activates genetic programs that lead to progressive hyperinflammation in COVID-19 patients. Here we report on genetic programs activated by interferons and the suppression by Janus kinase (JAK) inhibitors. The angiotensin-converting enzyme 2 (ACE2) is the receptor for SARS-CoV-2 and deciphering its regulation is paramount for understanding the cell tropism of SARS-CoV-2 infection. We identified candidate regulatory elements in the ACE2 locus in human primary airway cells and lung tissue. Activating histone and promoter marks and Pol II loading characterize the intronic dACE2 and define novel candidate enhancers distal to the genuine ACE2 promoter and within additional introns. dACE2, and to a lesser extent ACE2, RNA levels increased in primary cells treated with interferons and this induction was mitigated by JAK inhibitors that are used therapeutically in COVID-19 patients. Our analyses provide insight into ACE2 regulatory elements and highlight JAK inhibitors as suitable tools to suppress interferon-activated genetic programs in bronchial cells.',\n",
       "  'upregulated_genes-647_genes_are_up-regulated_by_IFNb_adjusted_p-value_0_05_log2FoldChange_1_and_normalized_number_5',\n",
       "  'None here'),\n",
       " 'PMC7743079-b1ba84e30bc6ca807fdf0775.xlsx-upregulated_genes-647_genes_are_up-regulated_by_IFNb_adjusted_p-value_0_05_log2FoldChange_1_and_normalized_number_5': ('Activation of Interferon-Stimulated Transcriptomes and ACE2 Isoforms in Human Airway Epithelium Is Curbed by Janus Kinase Inhibitors',\n",
       "  'SARS-CoV-2 infection of human airway epithelium activates genetic programs that lead to progressive hyperinflammation in COVID-19 patients. Here we report on genetic programs activated by interferons and the suppression by Janus kinase (JAK) inhibitors. The angiotensin-converting enzyme 2 (ACE2) is the receptor for SARS-CoV-2 and deciphering its regulation is paramount for understanding the cell tropism of SARS-CoV-2 infection. We identified candidate regulatory elements in the ACE2 locus in human primary airway cells and lung tissue. Activating histone and promoter marks and Pol II loading characterize the intronic dACE2 and define novel candidate enhancers distal to the genuine ACE2 promoter and within additional introns. dACE2, and to a lesser extent ACE2, RNA levels increased in primary cells treated with interferons and this induction was mitigated by JAK inhibitors that are used therapeutically in COVID-19 patients. Our analyses provide insight into ACE2 regulatory elements and highlight JAK inhibitors as suitable tools to suppress interferon-activated genetic programs in bronchial cells.',\n",
       "  'upregulated_genes-647_genes_are_up-regulated_by_IFNb_adjusted_p-value_0_05_log2FoldChange_1_and_normalized_number_5',\n",
       "  'None here'),\n",
       " 'PMC7743079-7f67d3c21f11f4a286c4f73c.xlsx-upregulated_genes-647_genes_are_up-regulated_by_IFNb_adjusted_p-value_0_05_log2FoldChange_1_and_normalized_number_5': ('Activation of Interferon-Stimulated Transcriptomes and ACE2 Isoforms in Human Airway Epithelium Is Curbed by Janus Kinase Inhibitors',\n",
       "  'SARS-CoV-2 infection of human airway epithelium activates genetic programs that lead to progressive hyperinflammation in COVID-19 patients. Here we report on genetic programs activated by interferons and the suppression by Janus kinase (JAK) inhibitors. The angiotensin-converting enzyme 2 (ACE2) is the receptor for SARS-CoV-2 and deciphering its regulation is paramount for understanding the cell tropism of SARS-CoV-2 infection. We identified candidate regulatory elements in the ACE2 locus in human primary airway cells and lung tissue. Activating histone and promoter marks and Pol II loading characterize the intronic dACE2 and define novel candidate enhancers distal to the genuine ACE2 promoter and within additional introns. dACE2, and to a lesser extent ACE2, RNA levels increased in primary cells treated with interferons and this induction was mitigated by JAK inhibitors that are used therapeutically in COVID-19 patients. Our analyses provide insight into ACE2 regulatory elements and highlight JAK inhibitors as suitable tools to suppress interferon-activated genetic programs in bronchial cells.',\n",
       "  'upregulated_genes-647_genes_are_up-regulated_by_IFNb_adjusted_p-value_0_05_log2FoldChange_1_and_normalized_number_5',\n",
       "  'with growth hormone (GH), Interleukin 6 (IL6) and IL7, followed by RNA-seq transcriptome analyses (**Supplementary Tables 2**)  Supplementary Table 2. List of all genes with normalized read counts in each replicate at Control and IFNa treated SAEC, log2 (fold change), p-value and adjusted p-value as well as upregulated gene list and GSEA analysis'),\n",
       " 'PMC6364032-4861e687947d7dd010482926cdff0def_JVI.01885-18-sd002.xlsx-Contorl_IFN_vs_Control-Unnamed_2': ('Cell Cycle Arrest in G 2 /M Phase Enhances Replication of Interferon-Sensitive Cytoplasmic RNA Viruses via Inhibition of Antiviral Gene Expression',\n",
       "  'Vesicular stomatitis virus (VSV) (a rhabdovirus) and its variant VSV-ΔM51 are widely used model systems to study mechanisms of virus-host interactions. Here, we investigated how the cell cycle affects replication of these viruses using an array of cell lines with different levels of impairment of antiviral signaling and a panel of chemical compounds arresting the cell cycle at different phases. We observed that all compounds inducing cell cycle arrest in G 2 /M phase strongly enhanced the replication of VSV-ΔM51 in cells with functional antiviral signaling. G 2 /M arrest strongly inhibited type I and type III interferon (IFN) production as well as expression of IFN-stimulated genes in response to exogenously added IFN. Moreover, G 2 /M arrest enhanced the replication of Sendai virus (a paramyxovirus), which is also highly sensitive to the type I IFN response but did not stimulate the replication of a wild-type VSV that is more effective at evading antiviral responses. In contrast, the positive effect of G 2 /M arrest on virus replication was not observed in cells defective in IFN signaling. Altogether, our data show that replication of IFN-sensitive cytoplasmic viruses can be strongly stimulated during G 2 /M phase as a result of inhibition of antiviral gene expression, likely due to mitotic inhibition of transcription, a global repression of cellular transcription during G 2 /M phase. The G 2 /M phase thus could represent an \"Achilles\\' heel\" of the infected cell, a phase when the cell is inadequately protected. This model could explain at least one of the reasons why many viruses have been shown to induce G 2 /M arrest. IMPORTANCE Vesicular stomatitis virus (VSV) (a rhabdovirus) and its variant VSV-ΔM51 are widely used model systems to study mechanisms of virus-host interactions. Here, we investigated how the cell cycle affects replication of VSV and VSV-ΔM51. We show that G 2 /M cell cycle arrest strongly enhances the replication of VSV-ΔM51 (but not of wild-type VSV) and Sendai virus (a paramyxovirus) via inhibition of antiviral gene expression, likely due to mitotic inhibition of transcription, a global repression of cellular transcription during G 2 /M phase. Our data suggest that the G 2 /M phase could represent an \"Achilles\\' heel\" of the infected cell, a phase when the cell is inadequately protected. This model could explain at least one of the reasons why many viruses have been shown to induce G 2 /M arrest, and it has important implications for oncolytic virotherapy, suggesting that frequent cell cycle progression in cancer cells could make them more permissive to viruses.',\n",
       "  '18-sd002.xlsx-Contorl_IFN_vs_Control-Unnamed_2',\n",
       "  'Supplemental file 2'),\n",
       " 'PMC9153011-advancesADV2021005360-suppl3.xlsx-Sheet1-Unnamed_2': ('Genome-wide whole-blood transcriptome profiling across inherited bone marrow failure subtypes',\n",
       "  'Gene expression profiling has long been used in understanding the contribution of genes and related pathways in disease pathogenesis and susceptibility. We have performed whole-blood transcriptomic profiling in a subset of patients with inherited bone marrow failure (IBMF) whose diseases are clinically and genetically characterized as Fanconi anemia (FA), Shwachman-Diamond syndrome (SDS), and dyskeratosis congenita (DC). We hypothesized that annotating whole-blood transcripts genome wide will aid in understanding the complexity of gene regulation across these IBMF subtypes. Initial analysis of these blood-derived transcriptomes revealed significant skewing toward upregulated genes in patients with FA when compared with controls. Patients with SDS or DC also showed similar skewing profiles in their transcriptional status revealing a common pattern across these different IBMF subtypes. Gene set enrichment analysis revealed shared pathways involved in protein translation and elongation (ribosome constituents), RNA metabolism (nonsense-mediated decay), and mitochondrial function (electron transport chain). We further identified a discovery set of 26 upregulated genes at stringent cutoff (false discovery rate < 0.05) that appeared as a unified signature across the IBMF subtypes. Subsequent transcriptomic analysis on genetically uncharacterized patients with BMF revealed a striking overlap of genes, including 22 from the discovery set, which indicates a unified transcriptional drive across the classic (FA, SDS, and DC) and uncharacterized BMF subtypes. This study has relevance in disease pathogenesis, for example, in explaining the features (including the BMF) common to all patients with IBMF and suggests harnessing this transcriptional signature for patient benefit.',\n",
       "  'Sheet1-Unnamed_2',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC9949049-media-12.xlsx-Liver-Liver-specific': ('Gene body DNA hydroxymethylation restricts the magnitude of transcriptional changes during aging',\n",
       "  'DNA hydroxymethylation (5hmC), the most abundant oxidative derivative of DNA methylation, is typically enriched at enhancers and gene bodies of transcriptionally active and tissue-specific genes. Although aberrant genomic 5hmC has been implicated in age-related diseases, its functional role in aging remains unknown. Here, using mouse liver and cerebellum as model organs, we show that 5hmC accumulates in gene bodies associated with tissue-specific function and restricts the magnitude of gene expression changes with age. Mechanistically, 5hmC decreases the binding of splicing associated factors and correlates with age-related alternative splicing events. We found that various age-related contexts, such as prolonged quiescence and senescence, drive the accumulation of 5hmC with age. We provide evidence that this age-related transcriptionally restrictive function is conserved in mouse and human tissues. Our findings reveal that 5hmC regulates tissue-specific function and may play a role in longevity.',\n",
       "  'Liver-Liver-specific',\n",
       "  'and one gene that was transiently upregulated with age (Supp. Fig. 9B, left panel, **Supp. Table 11**)'),\n",
       " 'PMC8800370-NIHMS1769771-supplement-Supplemental_Table_2.xlsx-YFV_RNA_sig_gmx-YFV_T_Mem': ('CD19-CAR T cells undergo exhaustion DNA methylation programming in patients with acute lymphoblastic leukemia',\n",
       "  'CD19-CAR T cell therapy has evolved into the standard of care for relapsed/refractory B cell acute lymphoblastic leukemia (ALL); however, limited persistence of the CAR T cells enables tumor relapse for many patients. To gain a deeper understanding of the molecular characteristics associated with CAR T cell differentiation, we performed longitudinal genome-wide DNA methylation profiling of CD8 + CD19-CAR T cells post-infusion in ALL patients. We report that CAR T cells undergo a rapid and broad erasure of repressive DNA methylation reprograms at effector-associated genes. The CAR T cell post-infusion changes are further characterized by repression of genes (e.g., TCF7 and LEF1) associated with memory potential and a DNA methylation signature (e.g., demethylation at CX3CR1, BATF, and TOX) demarcating a transition toward exhaustion-progenitor T cells. Thus, CD19-CAR T cells undergo exhaustion-associated DNA methylation programming, indicating that efforts to prevent this process may be an attractive approach to improve CAR T cell efficacy.',\n",
       "  'YFV_RNA_sig_gmx-YFV_T_Mem',\n",
       "  'None here'),\n",
       " 'PMC8172581-42003_2021_2167_MOESM6_ESM.xlsx-upregulated_genes-647_genes_are_up-regulated_by_IFNb_adjusted_p-value_0_05_log2FoldChange_1_and_normalized_number_5': ('JAK inhibitors dampen activation of interferon-stimulated transcription of ACE2 isoforms in human airway epithelial cells',\n",
       "  'SARS-CoV-2 infection of human airway epithelium activates genetic programs leading to progressive hyperinflammation in COVID-19 patients. Here, we report on transcriptomes activated in primary airway cells by interferons and their suppression by Janus kinase (JAK) inhibitors. Deciphering the regulation of the angiotensin-converting enzyme 2 (ACE2), the receptor for SARS-CoV-2, is paramount for understanding the cell tropism of SARS-CoV-2 infection. ChIP-seq for activating histone marks and Pol II loading identified candidate enhancer elements controlling the ACE2 locus, including the intronic dACE2 promoter. Employing RNA-seq, we demonstrate that interferons activate expression of dACE2 and, to a lesser extent, the genuine ACE2 gene. Interferon-induced gene expression was mitigated by the JAK inhibitors baricitinib and ruxolitinib, used therapeutically in COVID-19 patients. Through integrating RNA-seq and ChIP-seq data we provide an in-depth understanding of genetic programs activated by interferons, and our study highlights JAK inhibitors as suitable tools to suppress these in bronchial cells.',\n",
       "  'upregulated_genes-647_genes_are_up-regulated_by_IFNb_adjusted_p-value_0_05_log2FoldChange_1_and_normalized_number_5',\n",
       "  'coinciding with activate chromatin marks. The DHS site in intron 9 overlaps with strong H**3**  Supplementary Data 3'),\n",
       " 'PMC10575775-41380_2023_1959_MOESM26_ESM.xlsx-human_cell_id-B_cell_Markers_TM': ('MYT1L haploinsufficiency in human neurons and mice causes autism-associated phenotypes that can be reversed by genetic and pharmacologic intervention',\n",
       "  'MYT1L is an autism spectrum disorder (ASD)-associated transcription factor that is expressed in virtually all neurons throughout life. How MYT1L mutations cause neurological phenotypes and whether they can be targeted remains enigmatic. Here, we examine the effects of MYT1L deficiency in human neurons and mice. Mutant mice exhibit neurodevelopmental delays with thinner cortices, behavioural phenotypes, and gene expression changes that resemble those of ASD patients. MYT1L target genes, including WNT and NOTCH, are activated upon MYT1L depletion and their chemical inhibition can rescue delayed neurogenesis in vitro. MYT1L deficiency also causes upregulation of the main cardiac sodium channel, SCN5A, and neuronal hyperactivity, which could be restored by shRNA-mediated knockdown of SCN5A or MYT1L overexpression in postmitotic neurons. Acute application of the sodium channel blocker, lamotrigine, also rescued electrophysiological defects in vitro and behaviour phenotypes in vivo. Hence, MYT1L mutation causes both developmental and postmitotic neurological defects. However, acute intervention can normalise resulting electrophysiological and behavioural phenotypes in adulthood.',\n",
       "  'human_cell_id-B_cell_Markers_TM',\n",
       "  'Supplementary Table 12'),\n",
       " 'PMC7244579-41467_2020_16327_MOESM6_ESM.xlsx-Sheet1-Unnamed_14': ('An integrated multi-omics approach identifies the landscape of interferon-α-mediated responses of human pancreatic beta cells',\n",
       "  'Interferon-α (IFNα), a type I interferon, is expressed in the islets of type 1 diabetic individuals, and its expression and signaling are regulated by T1D genetic risk variants and viral infections associated with T1D. We presently characterize human beta cell responses to IFNα by combining ATAC-seq, RNA-seq and proteomics assays. The initial response to IFNα is characterized by chromatin remodeling, followed by changes in transcriptional and translational regulation. IFNα induces changes in alternative splicing (AS) and first exon usage, increasing the diversity of transcripts expressed by the beta cells. This, combined with changes observed on protein modification/degradation, ER stress and MHC class I, may expand antigens presented by beta cells to the immune system. Beta cells also up-regulate the checkpoint proteins PDL1 and HLA-E that may exert a protective role against the autoimmune assault. Data mining of the present multi-omics analysis identifies two compound classes that antagonize IFNα effects on human beta cells.',\n",
       "  'Sheet1-Unnamed_14',\n",
       "  'the expression of differentially expressed genes (DEG) from all RNA-seq time points (Supplementary Data **3**)  Supplementary Data 3'),\n",
       " 'PMC10791629-41598_2024_51620_MOESM3_ESM.csv--geneID': ('Dectin-1 ligands produce distinct training phenotypes in human monocytes through differential activation of signaling networks',\n",
       "  'Cells of the innate immune system retain memory of prior exposures through a process known as innate immune training. β-glucan, a Dectin-1 ligand purified from the Candida albicans cell wall, has been one of the most widely utilized ligands for inducing innate immune training. However, many Dectin-1 ligands exist, and it is not known whether these all produce the same phenotype. Using a well-established in vitro model of innate immune training, we compared two commercially available Dectin-1 agonists, zymosan and depleted zymosan, with the gold standard β-glucan in the literature. We found that depleted zymosan, a β-glucan purified from Saccharomyces cerevisiae cell wall through alkali treatment, produced near identical effects as C. albicans β-glucan. However, untreated zymosan produced a distinct training effect from β-glucans at both the transcript and cytokine level. Training with zymosan diminished, rather than potentiated, induction of cytokines such as TNF and IL-6. Zymosan activated NFκB and AP-1 transcription factors more strongly than β-glucans. The addition of the toll-like receptor (TLR) ligand Pam3CSK4 was sufficient to convert the training effect of β-glucans to a phenotype resembling zymosan. We conclude that differential activation of TLR signaling pathways determines the phenotype of innate immune training induced by Dectin-1 ligands.',\n",
       "  'geneID',\n",
       "  'None here'),\n",
       " 'PMC10791629-41598_2024_51620_MOESM3_ESM.csv--Unnamed_0': ('Dectin-1 ligands produce distinct training phenotypes in human monocytes through differential activation of signaling networks',\n",
       "  'Cells of the innate immune system retain memory of prior exposures through a process known as innate immune training. β-glucan, a Dectin-1 ligand purified from the Candida albicans cell wall, has been one of the most widely utilized ligands for inducing innate immune training. However, many Dectin-1 ligands exist, and it is not known whether these all produce the same phenotype. Using a well-established in vitro model of innate immune training, we compared two commercially available Dectin-1 agonists, zymosan and depleted zymosan, with the gold standard β-glucan in the literature. We found that depleted zymosan, a β-glucan purified from Saccharomyces cerevisiae cell wall through alkali treatment, produced near identical effects as C. albicans β-glucan. However, untreated zymosan produced a distinct training effect from β-glucans at both the transcript and cytokine level. Training with zymosan diminished, rather than potentiated, induction of cytokines such as TNF and IL-6. Zymosan activated NFκB and AP-1 transcription factors more strongly than β-glucans. The addition of the toll-like receptor (TLR) ligand Pam3CSK4 was sufficient to convert the training effect of β-glucans to a phenotype resembling zymosan. We conclude that differential activation of TLR signaling pathways determines the phenotype of innate immune training induced by Dectin-1 ligands.',\n",
       "  'Unnamed_0',\n",
       "  'None here'),\n",
       " 'PMC8281254-Table_5.XLSX-ALL_Genes-Genes_mRNA_miRNA_lncRNA_selected_for_model_construction': ('Development of a Novel Immune Infiltration-Related ceRNA Network and Prognostic Model for Sarcoma',\n",
       "  'Due to the rarity and heterogeneity, it is challenging to explore and develop new therapeutic targets for patients with sarcoma. Recently, immune cell infiltration in the tumor microenvironment (TME) was widely studied, which provided a novel potential approach for cancer treatment. The competing endogenous RNA (ceRNA) regulatory network has been reported as a critical molecular mechanism of tumor development. However, the role of the ceRNA regulatory network in the TME of sarcoma remains unclear. In this study, gene expression data and clinical information were obtained from The Cancer Genome Atlas (TCGA) sarcoma datasets, and an immune infiltration-related ceRNA network was constructed, which comprised 14 lncRNAs, 13 miRNAs, and 23 mRNAs. Afterward, we constructed an immune infiltration-related risk score model based on the expression of IRF1, MFNG, hsa-miR-940, and hsa-miR-378a-5p, presenting a promising performance in predicting the prognosis of patients with sarcoma.',\n",
       "  'ALL_Genes-Genes_mRNA_miRNA_lncRNA_selected_for_model_construction',\n",
       "  'and two miRNAs were identified and further analyzed with a stepwise multivariate Cox regression (**Supplementary Table 5**)  Click here for additional data file'),\n",
       " 'PMC7244579-41467_2020_16327_MOESM6_ESM.xlsx-Sheet1-Unnamed_15': ('An integrated multi-omics approach identifies the landscape of interferon-α-mediated responses of human pancreatic beta cells',\n",
       "  'Interferon-α (IFNα), a type I interferon, is expressed in the islets of type 1 diabetic individuals, and its expression and signaling are regulated by T1D genetic risk variants and viral infections associated with T1D. We presently characterize human beta cell responses to IFNα by combining ATAC-seq, RNA-seq and proteomics assays. The initial response to IFNα is characterized by chromatin remodeling, followed by changes in transcriptional and translational regulation. IFNα induces changes in alternative splicing (AS) and first exon usage, increasing the diversity of transcripts expressed by the beta cells. This, combined with changes observed on protein modification/degradation, ER stress and MHC class I, may expand antigens presented by beta cells to the immune system. Beta cells also up-regulate the checkpoint proteins PDL1 and HLA-E that may exert a protective role against the autoimmune assault. Data mining of the present multi-omics analysis identifies two compound classes that antagonize IFNα effects on human beta cells.',\n",
       "  'Sheet1-Unnamed_15',\n",
       "  'the expression of differentially expressed genes (DEG) from all RNA-seq time points (Supplementary Data **3**)  Supplementary Data 3'),\n",
       " 'PMC7509539-Table_1.XLSX-Upregulated_in_MAIT-Gene_symbol': ('MAIT Cells Display a Specific Response to Type 1 IFN Underlying the Adjuvant Effect of TLR7/8 Ligands',\n",
       "  'Mucosal-associated invariant T (MAIT) cells constitute a highly conserved subset of effector T cells with innate-like recognition of a wide array of bacteria and fungi in humans. Harnessing the potential of these cells could represent a major advance as a new immunotherapy approach to fight difficult-to-treat bacterial infections. However, despite recent advances in the design of potent agonistic ligands for MAIT cells, it has become increasingly evident that adjuvants are required to elicit potent antimicrobial effector functions by these cells, such as IFNγ production and cytotoxicity. Indeed, TCR triggering alone elicits mostly barrier repair functions in MAIT cells, whereas an inflammatory milieu is required to drive the antibacterial functions. Cytokines such as IL-7, IL-12 and IL-18, IL-15 or more recently type 1 IFN all display an apparently similar ability to synergize with TCR stimulation to induce IFNγ production and/or cytotoxic functions in vitro , but their mechanisms of action are not well established. Herein, we show that MAIT cells feature a build-in mechanism to respond to IFNα. We confirm that IFNα acts directly and specifically on MAIT cells and synergizes with TCR/CD3 triggering to induce maximum cytokine production and cytotoxic functions. We provide evidences suggesting that the preferential activation of the Stat4 pathway is involved in the high sensitivity of MAIT cells to IFNα stimulation. Finally, gene expression data confirm the specific responsiveness of MAIT cells to IFNα and pinpoints specific pathways that could be the target of this cytokine. Altogether, these data highlight the potential of IFNα-inducing adjuvants to maximize MAIT cells responsiveness to purified ligands in order to induce potent anti-infectious responses.',\n",
       "  'Upregulated_in_MAIT-Gene_symbol',\n",
       "  'found in common between both cell types, including multiple interferon-specific genes (ISG) as expected (**Supplementary Table S1**)  Click here for additional data file'),\n",
       " 'PMC6411624-mmc1.xlsx-DEG-Unnamed_5': ('Integrated transcriptome interactome study of oncogenes and tumor suppressor genes in breast cancer',\n",
       "  'Breast cancer is the leading cause for mortality among women worldwide. Dysregulation of oncogenes and tumor suppressor genes is the major reason for the cause of cancer. Understanding these genes will provide clues and insights about their regulatory mechanism and their interplay in cancer. In the present study, an attempt is made to compare the functional characteristics and interactions of oncogenes and tumor suppressor genes to understand their biological role. 431 breast cancer samples from seven publicly available microarray datasets were collected and analysed using GEO2R tool. The identified 416 differentially expressed genes were classified into five gene sets as oncogenes (OG), tumor suppressor genes (TSG), druggable genes, essential genes and other genes. The gene sets were subjected to various analysis such as enrichment analysis ( viz ., GO, Pathways, Diseases and Drugs), network analysis, calculation of mutation frequencies and Guanine-Cytosine (GC) content. From the results, it was observed that the OG were having high GC content as well as high interactions than TSG. Moreover, the OG are found to have frequent mutations than TSG. The enrichment analysis results suggest that the oncogenes are involved in positive regulation of cellular protein metabolic process, macromolecule biosynthetic process and majorly in cell cycle and focal adhesion pathway in cancer. It was also found that these oncogenes are involved in other diseases such as skin diseases and viral infections. Collagenase, paclitaxel and docetaxel are some of the drugs found to be enriched for oncogenes.',\n",
       "  'DEG-Unnamed_5',\n",
       "  'None here'),\n",
       " 'PMC9003654-sj-xlsx-1-bbi-10.1177_11779322221088796.xlsx-Common_Down_regulated_DEGs-GSE18842': ('Integrative Analysis for Identification of Therapeutic Targets and Prognostic Signatures in Non-Small Cell Lung Cancer',\n",
       "  'Differential expressions of certain genes during tumorigenesis may serve to identify novel manageable targets in the clinic. In this work with an integrated bioinformatics approach, we analyzed public microarray datasets from Gene Expression Omnibus (GEO) to explore the key differentially expressed genes (DEGs) in non-small cell lung cancer (NSCLC). We identified a total of 984 common DEGs in 252 healthy and 254 NSCLC gene expression samples. The top 10 DEGs as a result of pathway enrichment and protein-protein interaction analysis were further investigated for their prognostic performances. Among these, we identified high expressions of CDC20 , AURKA , CDK1 , EZH2 , and CDKN2A genes that were associated with significantly poorer overall survival in NSCLC patients. On the contrary, high mRNA expressions of CBL , FYN , LRKK2 , and SOCS2 were associated with a significantly better prognosis. Furthermore, our drug target analysis for these hub genes suggests a potential use of Trichostatin A, Pracinostat, TGX-221, PHA-793887, AG-879, and IMD0354 antineoplastic agents to reverse the expression of these DEGs in NSCLC patients.',\n",
       "  'Common_Down_regulated_DEGs-GSE18842',\n",
       "  'sj-xlsx-1-bbi-10.1177_11779322221088796 – Supplemental material for Integrative Analysis for Identification of Therapeutic Targets and Prognostic Signatures in Non-Small Cell Lung Cancer  Click here for additional data file'),\n",
       " 'PMC9003654-sj-xlsx-1-bbi-10.1177_11779322221088796.xlsx-common_DEGs-Common_Down_regulated_DEGs': ('Integrative Analysis for Identification of Therapeutic Targets and Prognostic Signatures in Non-Small Cell Lung Cancer',\n",
       "  'Differential expressions of certain genes during tumorigenesis may serve to identify novel manageable targets in the clinic. In this work with an integrated bioinformatics approach, we analyzed public microarray datasets from Gene Expression Omnibus (GEO) to explore the key differentially expressed genes (DEGs) in non-small cell lung cancer (NSCLC). We identified a total of 984 common DEGs in 252 healthy and 254 NSCLC gene expression samples. The top 10 DEGs as a result of pathway enrichment and protein-protein interaction analysis were further investigated for their prognostic performances. Among these, we identified high expressions of CDC20 , AURKA , CDK1 , EZH2 , and CDKN2A genes that were associated with significantly poorer overall survival in NSCLC patients. On the contrary, high mRNA expressions of CBL , FYN , LRKK2 , and SOCS2 were associated with a significantly better prognosis. Furthermore, our drug target analysis for these hub genes suggests a potential use of Trichostatin A, Pracinostat, TGX-221, PHA-793887, AG-879, and IMD0354 antineoplastic agents to reverse the expression of these DEGs in NSCLC patients.',\n",
       "  'common_DEGs-Common_Down_regulated_DEGs',\n",
       "  'sj-xlsx-1-bbi-10.1177_11779322221088796 – Supplemental material for Integrative Analysis for Identification of Therapeutic Targets and Prognostic Signatures in Non-Small Cell Lung Cancer  Click here for additional data file'),\n",
       " 'PMC9003654-sj-xlsx-1-bbi-10.1177_11779322221088796.xlsx-Common_Down_regulated_DEGs-GSE19804': ('Integrative Analysis for Identification of Therapeutic Targets and Prognostic Signatures in Non-Small Cell Lung Cancer',\n",
       "  'Differential expressions of certain genes during tumorigenesis may serve to identify novel manageable targets in the clinic. In this work with an integrated bioinformatics approach, we analyzed public microarray datasets from Gene Expression Omnibus (GEO) to explore the key differentially expressed genes (DEGs) in non-small cell lung cancer (NSCLC). We identified a total of 984 common DEGs in 252 healthy and 254 NSCLC gene expression samples. The top 10 DEGs as a result of pathway enrichment and protein-protein interaction analysis were further investigated for their prognostic performances. Among these, we identified high expressions of CDC20 , AURKA , CDK1 , EZH2 , and CDKN2A genes that were associated with significantly poorer overall survival in NSCLC patients. On the contrary, high mRNA expressions of CBL , FYN , LRKK2 , and SOCS2 were associated with a significantly better prognosis. Furthermore, our drug target analysis for these hub genes suggests a potential use of Trichostatin A, Pracinostat, TGX-221, PHA-793887, AG-879, and IMD0354 antineoplastic agents to reverse the expression of these DEGs in NSCLC patients.',\n",
       "  'Common_Down_regulated_DEGs-GSE19804',\n",
       "  'sj-xlsx-1-bbi-10.1177_11779322221088796 – Supplemental material for Integrative Analysis for Identification of Therapeutic Targets and Prognostic Signatures in Non-Small Cell Lung Cancer  Click here for additional data file'),\n",
       " 'PMC9003654-sj-xlsx-1-bbi-10.1177_11779322221088796.xlsx-Common_Down_regulated_DEGs-GSE102287': ('Integrative Analysis for Identification of Therapeutic Targets and Prognostic Signatures in Non-Small Cell Lung Cancer',\n",
       "  'Differential expressions of certain genes during tumorigenesis may serve to identify novel manageable targets in the clinic. In this work with an integrated bioinformatics approach, we analyzed public microarray datasets from Gene Expression Omnibus (GEO) to explore the key differentially expressed genes (DEGs) in non-small cell lung cancer (NSCLC). We identified a total of 984 common DEGs in 252 healthy and 254 NSCLC gene expression samples. The top 10 DEGs as a result of pathway enrichment and protein-protein interaction analysis were further investigated for their prognostic performances. Among these, we identified high expressions of CDC20 , AURKA , CDK1 , EZH2 , and CDKN2A genes that were associated with significantly poorer overall survival in NSCLC patients. On the contrary, high mRNA expressions of CBL , FYN , LRKK2 , and SOCS2 were associated with a significantly better prognosis. Furthermore, our drug target analysis for these hub genes suggests a potential use of Trichostatin A, Pracinostat, TGX-221, PHA-793887, AG-879, and IMD0354 antineoplastic agents to reverse the expression of these DEGs in NSCLC patients.',\n",
       "  'Common_Down_regulated_DEGs-GSE102287',\n",
       "  'sj-xlsx-1-bbi-10.1177_11779322221088796 – Supplemental material for Integrative Analysis for Identification of Therapeutic Targets and Prognostic Signatures in Non-Small Cell Lung Cancer  Click here for additional data file'),\n",
       " 'PMC9003654-sj-xlsx-1-bbi-10.1177_11779322221088796.xlsx-Common_Down_regulated_DEGs-GSE27262': ('Integrative Analysis for Identification of Therapeutic Targets and Prognostic Signatures in Non-Small Cell Lung Cancer',\n",
       "  'Differential expressions of certain genes during tumorigenesis may serve to identify novel manageable targets in the clinic. In this work with an integrated bioinformatics approach, we analyzed public microarray datasets from Gene Expression Omnibus (GEO) to explore the key differentially expressed genes (DEGs) in non-small cell lung cancer (NSCLC). We identified a total of 984 common DEGs in 252 healthy and 254 NSCLC gene expression samples. The top 10 DEGs as a result of pathway enrichment and protein-protein interaction analysis were further investigated for their prognostic performances. Among these, we identified high expressions of CDC20 , AURKA , CDK1 , EZH2 , and CDKN2A genes that were associated with significantly poorer overall survival in NSCLC patients. On the contrary, high mRNA expressions of CBL , FYN , LRKK2 , and SOCS2 were associated with a significantly better prognosis. Furthermore, our drug target analysis for these hub genes suggests a potential use of Trichostatin A, Pracinostat, TGX-221, PHA-793887, AG-879, and IMD0354 antineoplastic agents to reverse the expression of these DEGs in NSCLC patients.',\n",
       "  'Common_Down_regulated_DEGs-GSE27262',\n",
       "  'sj-xlsx-1-bbi-10.1177_11779322221088796 – Supplemental material for Integrative Analysis for Identification of Therapeutic Targets and Prognostic Signatures in Non-Small Cell Lung Cancer  Click here for additional data file'),\n",
       " 'PMC5043525-13059_2016_1062_MOESM7_ESM.xlsx-Skeletal_Muscle_Cells-Gene': ('CellMapper: rapid and accurate inference of gene expression in difficult-to-isolate cell types',\n",
       "  'We present a sensitive approach to predict genes expressed selectively in specific cell types, by searching publicly available expression data for genes with a similar expression profile to known cell-specific markers. Our method, CellMapper, strongly outperforms previous computational algorithms to predict cell type-specific expression, especially for rare and difficult-to-isolate cell types. Furthermore, CellMapper makes accurate predictions for human brain cell types that have never been isolated, and can be rapidly applied to diverse cell types from many tissues. We demonstrate a clinically relevant application to prioritize candidate genes in disease susceptibility loci identified by GWAS.',\n",
       "  'Skeletal_Muscle_Cells-Gene',\n",
       "  'None here'),\n",
       " 'PMC9538974-JCB_202111137_TableS2.xlsx-Lists_of_DE_genes-Unnamed_7': ('Coordinated metabolic transitions and gene expression by NAD+ during adipogenesis',\n",
       "  'Adipocytes are the main cell type in adipose tissue, which is a critical regulator of metabolism, highly specialized in storing energy as fat. Adipocytes differentiate from multipotent mesenchymal stromal cells (hMSCs) through adipogenesis, a tightly controlled differentiation process involving close interplay between metabolic transitions and sequential programs of gene expression. However, the specific gears driving this interplay remain largely obscure. Additionally, the metabolite nicotinamide adenine dinucleotide (NAD+) is becoming increasingly recognized as a regulator of lipid metabolism, and a promising therapeutic target for dyslipidemia and obesity. Here, we explored how NAD+ bioavailability controls adipogenic differentiation from hMSC. We found a previously unappreciated repressive role for NAD+ on adipocyte commitment, while a functional NAD+-dependent deacetylase SIRT1 appeared crucial for terminal differentiation of pre-adipocytes. Repressing NAD+ biosynthesis during adipogenesis promoted the adipogenic transcriptional program, while two-photon microscopy and extracellular flux analyses suggest that SIRT1 activity mostly relies on the metabolic switch. Interestingly, SIRT1 controls subcellular compartmentalization of redox metabolism during adipogenesis.',\n",
       "  'Lists_of_DE_genes-Unnamed_7',\n",
       "  'consistently upregulated, while 407 were always silenced by the treatment (Fig. 3 B and **Table S2**)  lists DE genes from NAD+-treated cells genes and their functional analyses  Click here for additional data file'),\n",
       " 'PMC11089003-41593_2024_1600_MOESM4_ESM.xlsx-overlap_mouse-SalaFrigerio': (\"Xenografted human microglia display diverse transcriptomic states in response to Alzheimer's disease-related amyloid-β pathology\",\n",
       "  \"Microglia are central players in Alzheimer's disease pathology but analyzing microglial states in human brain samples is challenging due to genetic diversity, postmortem delay and admixture of pathologies. To circumvent these issues, here we generated 138,577 single-cell expression profiles of human stem cell-derived microglia xenotransplanted in the brain of the App NL-G-F model of amyloid pathology and wild-type controls. Xenografted human microglia adopt a disease-associated profile similar to that seen in mouse microglia, but display a more pronounced human leukocyte antigen or HLA state, likely related to antigen presentation in response to amyloid plaques. The human microglial response also involves a pro-inflammatory cytokine/chemokine cytokine response microglia or CRM response to oligomeric Aβ oligomers. Genetic deletion of TREM2 or APOE as well as APOE polymorphisms and TREM2 R47H expression in the transplanted microglia modulate these responses differentially. The expression of other Alzheimer's disease risk genes is differentially regulated across the distinct cell states elicited in response to amyloid pathology. Thus, we have identified multiple transcriptomic cell states adopted by human microglia in a multipronged response to Alzheimer's disease-related pathology, which should be taken into account in translational studies.\",\n",
       "  'overlap_mouse-SalaFrigerio',\n",
       "  'None here'),\n",
       " 'PMC3510497-supp_gks888_nar-02158-v-2012-File012.xlsx-C42B_DHT-upregulated-ENSG': ('Persistent androgen receptor-mediated transcription in castration-resistant prostate cancer under androgen-deprived conditions',\n",
       "  'The androgen receptor (AR) is a ligand-inducible transcription factor that mediates androgen action in target tissues. Upon ligand binding, the AR binds to thousands of genomic loci and activates a cell-type specific gene program. Prostate cancer growth and progression depend on androgen-induced AR signaling. Treatment of advanced prostate cancer through medical or surgical castration leads to initial response and durable remission, but resistance inevitably develops. In castration-resistant prostate cancer (CRPC), AR activity remains critical for tumor growth despite androgen deprivation. Although previous studies have focused on ligand-dependent AR signaling, in this study we explore AR function under the androgen-deprived conditions characteristic of CRPC. Our data demonstrate that AR persistently occupies a distinct set of genomic loci after androgen deprivation in CRPC. These androgen-independent AR occupied regions have constitutively open chromatin structures that lack the canonical androgen response element and are independent of FoxA1, a transcription factor involved in ligand-dependent AR targeting. Many AR binding events occur at proximal promoters, which can act as enhancers to augment transcriptional activities of other promoters through DNA looping. We further show that androgen-independent AR binding directs a gene expression program in CRPC, which is necessary for the growth of CRPC after androgen withdrawal.',\n",
       "  'C42B_DHT-upregulated-ENSG',\n",
       "  'Supplementary Data'),\n",
       " 'PMC6449245-TCA-10-715-s003.xlsx-sheet1-Table_S1_Finally_identified_DEGs_of_E-GEOD-19804_dataset_by_R_software': ('Metabolic gene NR4A1 as a potential therapeutic target for non-smoking female non-small cell lung cancer patients',\n",
       "  'Background: Although cigarette smoking is considered one of the key risk factors for lung cancer, 15% of male patients and 53% of female patients with lung cancer are non-smokers. Metabolic changes are critical features of cancer. Therapeutic target identification from a metabolic perspective in non-small cell lung cancer (NSCLC) tissue of female non-smokers has long been ignored. Results: Based on microarray data retrieved from Affymetrix expression arrays E-GEOD-19804, we found that the downregulated genes in non-smoking female NSCLC patients tended to participate in protein/amino acid and lipid metabolism, while upregulated genes were more involved in protein/amino acid and carbohydrate metabolism. Combining nutrient metabolic co-expression, protein-protein interaction network construction and overall survival assessment, we identified NR4A1 and TIE1 as potential therapeutic targets for NSCLC in female non-smokers. To accelerate the drug development for non-smoking female NSCLC patients, we identified nilotinib as a potential agonist targeting NR4A1 encoded protein by molecular docking and molecular dynamic stimulation. We also show that nilotinib inhibited proliferation and induced senescence of cells in non-smoking female NSCLC patients in vitro. Conclusions: These results not only uncover nutrient metabolic characteristics in non-smoking female NSCLC patients, but also provide a new paradigm for identifying new targets and drugs for novel therapy for such patients.',\n",
       "  'sheet1-Table_S1_Finally_identified_DEGs_of_E-GEOD-19804_dataset_by_R_software',\n",
       "  'were identified after analyses of E‐GEOD‐19804, including 976 downregulated and 478 upregulated genes (Table **S1**)  Table S1. Finally identified differentially expressed genes (DEGs) of E‐GEOD‐19804 dataset by R software  Click here for additional data file'),\n",
       " 'PMC7331728-41598_2020_66453_MOESM2_ESM.xlsx-All_tissue_gene_clusters-Brain': ('The transcriptional signature associated with human motile cilia',\n",
       "  'Cilia are complex microtubule-based organelles essential to a range of processes associated with embryogenesis and tissue homeostasis. Mutations in components of these organelles or those involved in their assembly may result in a diverse set of diseases collectively known as ciliopathies. Accordingly, many cilia-associated proteins have been described, while those distinguishing cilia subtypes are poorly defined. Here we set out to define genes associated with motile cilia in humans based on their transcriptional signature. To define the signature, we performed network deconvolution of transcriptomics data derived from tissues possessing motile ciliated cell populations. For each tissue, genes coexpressed with the motile cilia-associated transcriptional factor, FOXJ1, were identified. The consensus across tissues provided a transcriptional signature of 248 genes. To validate these, we examined the literature, databases (CilDB, CentrosomeDB, CiliaCarta and SysCilia), single cell RNA-Seq data, and the localisation of mRNA and proteins in motile ciliated cells. In the case of six poorly characterised signature genes, we performed new localisation experiments on ARMC3, EFCAB6, FAM183A, MYCBPAP, RIBC2 and VWA3A. In summary, we report a set of motile cilia-associated genes that helps shape our understanding of these complex cellular organelles.',\n",
       "  'All_tissue_gene_clusters-Brain',\n",
       "  'Supplementary Information2'),\n",
       " 'PMC3510497-supp_gks888_nar-02158-v-2012-File012.xlsx-C42B_DHT-upregulated-NAME': ('Persistent androgen receptor-mediated transcription in castration-resistant prostate cancer under androgen-deprived conditions',\n",
       "  'The androgen receptor (AR) is a ligand-inducible transcription factor that mediates androgen action in target tissues. Upon ligand binding, the AR binds to thousands of genomic loci and activates a cell-type specific gene program. Prostate cancer growth and progression depend on androgen-induced AR signaling. Treatment of advanced prostate cancer through medical or surgical castration leads to initial response and durable remission, but resistance inevitably develops. In castration-resistant prostate cancer (CRPC), AR activity remains critical for tumor growth despite androgen deprivation. Although previous studies have focused on ligand-dependent AR signaling, in this study we explore AR function under the androgen-deprived conditions characteristic of CRPC. Our data demonstrate that AR persistently occupies a distinct set of genomic loci after androgen deprivation in CRPC. These androgen-independent AR occupied regions have constitutively open chromatin structures that lack the canonical androgen response element and are independent of FoxA1, a transcription factor involved in ligand-dependent AR targeting. Many AR binding events occur at proximal promoters, which can act as enhancers to augment transcriptional activities of other promoters through DNA looping. We further show that androgen-independent AR binding directs a gene expression program in CRPC, which is necessary for the growth of CRPC after androgen withdrawal.',\n",
       "  'C42B_DHT-upregulated-NAME',\n",
       "  'Supplementary Data'),\n",
       " 'PMC9535846-12974_2022_2604_MOESM2_ESM.xlsx-Table_S2-Unnamed_1': (\"Alzheimer's disease-related transcriptional sex differences in myeloid cells\",\n",
       "  \"Sex differences have been identified in many diseases associated with dysregulated immune responses, including Alzheimer's disease (AD), for which approximately two-thirds of patients are women. An accumulating body of research indicates that microglia may play a causal role in the pathogenesis of this disease. We hypothesised that sex differences in the transcriptome of human myeloid cells may contribute to the sex difference observed in AD prevalence. To explore this, we assessed bulk and single-nuclear RNA sequencing data sets generated from four human derived myeloid cell populations: post-mortem microglial nuclei, peripheral monocytes, monocyte-derived macrophages (MDMs) and induced pluripotent stem cell derived microglial-like cells (MGLs). We found that expression of AD risk genes, gene signatures associated with the inflammatory response in AD, and genes related to proinflammatory immune responses were enriched in microglial nuclei isolated from aged female donors without ante-mortem neurological disease, relative to those from males. In addition, these inflammation-associated gene sets were found to be enriched in peripheral monocytes isolated from postmenopausal women and in MDMs obtained from premenopausal individuals relative to age-matched males. Expression of these gene sets did not differ in MDMs derived from women whose blood was sampled across the menstrual cycle or in MGLs cultured with 17β-oestradiol. This suggests that the observed gene set enrichments in myeloid cells from women were not being driven by acute hormonal influences. Together, these data support the hypothesis that the increased prevalence of AD in women may be partly explained by a myeloid cell phenotype biased towards expression of biological processes relevant to AD.\",\n",
       "  'Table_S2-Unnamed_1',\n",
       "  'genes by sex (sDEG) in our control discovery microglia snRNA-seq data set using MAST [4**2**]  Additional file 2: Table S1. Sexually differentially expressed genes in the microglial nuclei. Table S2. Genes comprising each of the IR-AD gene sets. Table S3. Sexually differentially expressed genes between the F- and M-MCs. Table S4. GSVA of IR-AD gene sets between Flo- and Fhi-MDMs results. Table S5. Sexually differentially expressed genes between the Flo- and M-MDMs. Table S6. Sexually differentially expressed reactome pathways between the Flo- and M-MDMs. Table S7. Differentially expressed reactome pathways between the Flo- and Fhi-MDMs. Table S8. Smith et al. [39], sample specific metadata. Table S9. Reactome gene sets used for the analysis of the MDMs'),\n",
       " 'PMC9709075-41598_2022_24792_MOESM10_ESM.xlsx-Fig2A_620_genes-AL645608_8': ('Transcriptomic comparison of primary human lung cells with lung tissue samples and the human A549 lung cell line highlights cell type specific responses during infections with influenza A virus',\n",
       "  'Influenza A virus (IAV) causes pandemics and annual epidemics of severe respiratory infections. A better understanding of the molecular regulation in tissue and cells upon IAV infection is needed to thoroughly understand pathogenesis. We analyzed IAV replication and gene expression induced by IAV strain H3N2 Panama in isolated primary human alveolar epithelial type II cells (AECIIs), the permanent A549 adenocarcinoma cell line, alveolar macrophages (AMs) and explanted human lung tissue by bulk RNA sequencing. Primary AECII exhibit in comparison to AM a broad set of strongly induced genes related to RIG-I and interferon (IFN) signaling. The response of AECII was partly mirrored in A549 cells. In human lung tissue, we observed induction of genes unlike in isolated cells. Viral RNA was used to correlate host cell gene expression changes with viral burden. While relative induction of key genes was similar, gene abundance was highest in AECII cells and AM, while weaker in the human lung (due to less IAV replication) and A549 cells (pointing to their limited suitability as a model). Correlation of host gene induction with viral burden allows a better understanding of the cell-type specific induction of pathways and a possible role of cellular crosstalk requiring intact tissue.',\n",
       "  'Fig2A_620_genes-AL645608_8',\n",
       "  'GEO under the accession number GSE206606. All gene groups utilized are listed in table **S1**  Supplementary Table S1'),\n",
       " 'PMC4617899-12864_2015_2012_MOESM2_ESM.xlsx-pDC_SIG_minPair_2x-Unnamed_1': ('BubbleGUM: automatic extraction of phenotype molecular signatures and comprehensive visualization of multiple Gene Set Enrichment Analyses',\n",
       "  \"Background: Recent advances in the analysis of high-throughput expression data have led to the development of tools that scaled-up their focus from single-gene to gene set level. For example, the popular Gene Set Enrichment Analysis (GSEA) algorithm can detect moderate but coordinated expression changes of groups of presumably related genes between pairs of experimental conditions. This considerably improves extraction of information from high-throughput gene expression data. However, although many gene sets covering a large panel of biological fields are available in public databases, the ability to generate home-made gene sets relevant to one's biological question is crucial but remains a substantial challenge to most biologists lacking statistic or bioinformatic expertise. This is all the more the case when attempting to define a gene set specific of one condition compared to many other ones. Thus, there is a crucial need for an easy-to-use software for generation of relevant home-made gene sets from complex datasets, their use in GSEA, and the correction of the results when applied to multiple comparisons of many experimental conditions. Result: We developed BubbleGUM (GSEA Unlimited Map), a tool that allows to automatically extract molecular signatures from transcriptomic data and perform exhaustive GSEA with multiple testing correction. One original feature of BubbleGUM notably resides in its capacity to integrate and compare numerous GSEA results into an easy-to-grasp graphical representation. We applied our method to generate transcriptomic fingerprints for murine cell types and to assess their enrichments in human cell types. This analysis allowed us to confirm homologies between mouse and human immunocytes. Conclusions: BubbleGUM is an open-source software that allows to automatically generate molecular signatures out of complex expression datasets and to assess directly their enrichment by GSEA on independent datasets. Enrichments are displayed in a graphical output that helps interpreting the results. This innovative methodology has recently been used to answer important questions in functional genomics, such as the degree of similarities between microarray datasets from different laboratories or with different experimental models or clinical cohorts. BubbleGUM is executable through an intuitive interface so that both bioinformaticians and biologists can use it. It is available at http://www.ciml.univ-mrs.fr/applications/BubbleGUM/index.html .\",\n",
       "  'pDC_SIG_minPair_2x-Unnamed_1',\n",
       "  'None here'),\n",
       " 'PMC10849790-CPR-57-e13545-s011.xlsx-muscle_vs_endothelial-gene_short_name': ('Bulk and single-cell alternative splicing analyses reveal roles of TRA2B in myogenic differentiation',\n",
       "  'Alternative splicing (AS) disruption has been linked to disorders of muscle development, as well as muscular atrophy. However, the precise changes in AS patterns that occur during myogenesis are not well understood. Here, we employed isoform long-reads RNA-seq (Iso-seq) and single-cell RNA-seq (scRNA-seq) to investigate the AS landscape during myogenesis. Our Iso-seq data identified 61,146 full-length isoforms representing 11,682 expressed genes, of which over 52% were novel. We identified 38,022 AS events, with most of these events altering coding sequences and exhibiting stage-specific splicing patterns. We identified AS dynamics in different types of muscle cells through scRNA-seq analysis, revealing genes essential for the contractile muscle system and cytoskeleton that undergo differential splicing across cell types. Gene-splicing analysis demonstrated that AS acts as a regulator, independent of changes in overall gene expression. Two isoforms of splicing factor TRA2B play distinct roles in myogenic differentiation by triggering AS of TGFBR2 to regulate canonical TGF-β signalling cascades differently. Our study provides a valuable transcriptome resource for myogenesis and reveals the complexity of AS and its regulation during myogenesis.',\n",
       "  'muscle_vs_endothelial-gene_short_name',\n",
       "  'None here'),\n",
       " 'PMC9715646-41467_2022_35089_MOESM4_ESM.xlsx-D_K1-DMSOvsNutlin-Unnamed_17': ('PPM1D suppresses p53-dependent transactivation and cell death by inhibiting the Integrated Stress Response',\n",
       "  'The p53 transcription factor is a master regulator of cellular stress responses inhibited by repressors such as MDM2 and the phosphatase PPM1D. Activation of p53 with pharmacological inhibitors of its repressors is being tested in clinical trials for cancer therapy, but efficacy has been limited by poor induction of tumor cell death. We demonstrate that dual inhibition of MDM2 and PPM1D induces apoptosis in multiple cancer cell types via amplification of the p53 transcriptional program through the eIF2α-ATF4 pathway. PPM1D inhibition induces phosphorylation of eIF2α, ATF4 accumulation, and ATF4-dependent enhancement of p53-dependent transactivation upon MDM2 inhibition. Dual inhibition of p53 repressors depletes heme and induces HRI-dependent eIF2α phosphorylation. Pharmacological induction of eIF2α phosphorylation synergizes with MDM2 inhibition to induce cell death and halt tumor growth in mice. These results demonstrate that PPM1D inhibits both the p53 network and the integrated stress response controlled by eIF2α-ATF4, with clear therapeutic implications.',\n",
       "  'D_K1-DMSOvsNutlin-Unnamed_17',\n",
       "  'It has been previously demonstrated that small molecule inhibitors of MDM2 and PPM**1**  Supplementary Data 1'),\n",
       " 'PMC3156788-1476-4598-10-86-S10.XLS-Table_S8A-Unnamed_1': ('Kinome expression profiling and prognosis of basal breast cancers',\n",
       "  'Background: Basal breast cancers (BCs) represent ~15% of BCs. Although overall poor, prognosis is heterogeneous. Identification of good- versus poor-prognosis patients is difficult or impossible using the standard histoclinical features and the recently defined prognostic gene expression signatures (GES). Kinases are often activated or overexpressed in cancers, and constitute targets for successful therapies. We sought to define a prognostic model of basal BCs based on kinome expression profiling. Methods: DNA microarray-based gene expression and histoclinical data of 2515 early BCs from thirteen datasets were collected. We searched for a kinome-based GES associated with disease-free survival (DFS) in basal BCs of the learning set using a metagene-based approach. The signature was then tested in basal tumors of the independent validation set. Results: A total of 591 samples were basal. We identified a 28-kinase metagene associated with DFS in the learning set (N = 73). This metagene was associated with immune response and particularly cytotoxic T-cell response. On multivariate analysis, a metagene-based predictor outperformed the classical prognostic factors, both in the learning and the validation (N = 518) sets, independently of the lymphocyte infiltrate. In the validation set, patients whose tumors overexpressed the metagene had a 78% 5-year DFS versus 54% for other patients (p = 1.62E-4, log-rank test). Conclusions: Based on kinome expression, we identified a predictor that separated basal BCs into two subgroups of different prognosis. Tumors associated with higher activation of cytotoxic tumor-infiltrative lymphocytes harbored a better prognosis. Such classification should help tailor the treatment and develop new therapies based on immune response manipulation.',\n",
       "  'Table_S8A-Unnamed_1',\n",
       "  'genes. Most of them (n = 506) were overexpressed in \"Immune-High\" samples (Additional file **10**)  Additional file 10Table S8: Genes differentially expressed between the \"Immune-High\" and \"Immune-Low\" basal tumor subgroups in the IPC set. (A) Summary of the 532 genes differentially expressed (Student\\'s t-test). (B) Canonical pathways associated with the genes overexpressed in \"Immune-High\" tumors in the IPC set  Click here for file'),\n",
       " 'PMC10799059-41598_2024_51681_MOESM3_ESM.xlsx-Supplementary_Table_2-Unnamed_52': ('Development of an invasion score based on metastasis-related pathway activity profiles for identifying invasive molecular subtypes of lung adenocarcinoma',\n",
       "  'The invasive capacity of lung adenocarcinoma (LUAD) is an important factor influencing patients\\' metastatic status and survival outcomes. However, there is still a lack of suitable biomarkers to evaluate tumor invasiveness. LUAD molecular subtypes were identified by unsupervised consistent clustering of LUAD. The differences in prognosis, tumor microenvironment (TME), and mutation were assessed among different subtypes. After that, the invasion-related gene score (IRGS) was constructed by genetic differential analysis, WGCNA analysis, and LASSO analysis, then we evaluated the relationship between IRGS and invasive characteristics, TME, and prognosis. The predictive ability of the IRGS was verified by in vitro experiments. Next, the \"oncoPredict\" R package and CMap were used to assess the potential value of IRGS in drug therapy. The results showed that LUAD was clustered into two molecular subtypes. And the C1 subtype exhibited a worse prognosis, higher stemness enrichment activity, less immune infiltration, and higher mutation frequency. Subsequently, IRGS developed based on molecular subtypes demonstrated a strong association with malignant characteristics such as invasive features, higher stemness scores, less immune infiltration, and worse survival. In vitro experiments showed that the higher IRGS LUAD cell had a stronger invasive capacity than the lower IRGS LUAD cell. Predictive analysis based on the \"oncoPredict\" R package showed that the high IRGS group was more sensitive to docetaxel, erlotinib, paclitaxel, and gefitinib. Among them, in vitro experiments verified the greater killing effect of paclitaxel on high IRGS cell lines. In addition, CMap showed that purvalanol-a, angiogenesis-inhibitor, and masitinib have potential therapeutic effects in the high IRGS group. In summary we identified and analyzed the molecular subtypes associated with the invasiveness of LUAD and developed IRGS that can efficiently predict the prognosis and invasive ability of the tumor. IRGS may be able to facilitate the precision treatment of LUAD to some extent.',\n",
       "  'Supplementary_Table_2-Unnamed_52',\n",
       "  'None here'),\n",
       " 'PMC4996651-elife-15657-supp1.xlsx-1E-PC6_high': ('An integrative transcriptomic atlas of organogenesis in human embryos',\n",
       "  'Human organogenesis is when severe developmental abnormalities commonly originate. However, understanding this critical embryonic phase has relied upon inference from patient phenotypes and assumptions from in vitro stem cell models and non-human vertebrates. We report an integrated transcriptomic atlas of human organogenesis. By lineage-guided principal components analysis, we uncover novel relatedness of particular developmental genes across different organs and tissues and identified unique transcriptional codes which correctly predicted the cause of many congenital disorders. By inference, our model pinpoints co-enriched genes as new causes of developmental disorders such as cleft palate and congenital heart disease. The data revealed more than 6000 novel transcripts, over 90% of which fulfil criteria as long non-coding RNAs correlated with the protein-coding genome over megabase distances. Taken together, we have uncovered cryptic transcriptional programs used by the human embryo and established a new resource for the molecular understanding of human organogenesis and its associated disorders.',\n",
       "  '1E-PC6_high',\n",
       "  \"al., 2010 and a false discovery rate (FDR) of 0.05 (see Materials and methods, **Supplementary file 1B**)  Supplementary tables.(A) Samples used in this study. Details on the material derived from individual human embryos (each listed according to the Carnegie Stage (CS)) used in the biological replicates and the sequencing statistics for each sample. A conversion of Carnegie Stage to an approximate days post-conception is available in Jennings et al. (2015) (open access). Gene level read counts are available for download as a TSV file in Supplementary file 2. (B) Differential gene expression between paired embryonic and fetal RNA-seq data. The R package edgeR (Robinson et al., 2010) was used to test for differential gene expression between embryonic and fetal (Roadmap Epigenomics Consortium, 2015) datasets. Shared tissues were adrenal gland, heart, lung, stomach, kidney, upper limb, lower limb and testis. The table is sorted by FDR (column H) and can be filtered by log fold change (column E) to give embryo-enriched genes (negative values) or fetal-enriched genes (positive values). (C) Gene Ontology (GO) terms and the genes underlying them for embryonic vs.fetal (Roadmap) up-regulated genes. Genes up-regulated in embryonic tissues versus fetal tissues (edgeR, FDR < 0.05, see Supplementary file 1B) were tested for GO term enrichment using Fisher’s exact test and the elimination algorithm implemented in the R package topGO (Alexa and Rahnenfuhrer, 2010). Separate tests were run for embryo up-regulated and fetal up-regulated genes. The table is sorted by enrichment in embryonic genes. (D) Tissue-specific genes contributing to metagenes. All genes with relative basis contribution (across metagenes) greater than 0.8 are listed. (E) The most extreme 1000 genes (high and low) for all principal components (PC1-31) of the LgPCA. The dataset is derived from genes annotated in GENCODE18. Raw gene-level loadings for each principal component are available for download as a TSV file in Supplementary file 3. (F) Gene Ontology (GO) terms and the genes underlying them for organ and tissue-specific transcriptomic signatures from the extremes of the LgPCA. GO terms were identified as enriched in extreme scoring genes (annotated in GENCODE 18) in the principal components (PCs) of the LgPCA. Due to the very large number of terms returned at p<0.0001 by Wilcoxon test (the topGO 'elim' method, see Materials and methods) an illustrative selection are listed with raw gene-level loadings available for download in Supplementary file 3. (G) Transcription factors in the extremes of the LgPCA and their links to developmental morbidity. The most extreme 1000 annotated genes (GENCODE 18) of the LgPCA dataset were filtered for transcription factors based on KEGG and PHANTOM5 annotations and for read counts >500. To identify disease associations each gene was entered as a search term in OMIM (www.ncbi.nlm.nih.gov/omim) and in PubMed. Batch queries were undertaken at Mouse Genome Informatics (MGI, www.informatics.jax.org) with 'Mammalian phenotype' as the output. (H) LgPCA predictions of causal genes for critical regions in either solved or unsolved developmental disorders. Fifty-three developmental disorders (Column A, 'solved') with causally associated transcription factors identified in the appropriate transcriptomic signature of Supplementary file 1G were originally defined by critical regions (Column C with hyperlink). These critical regions were identified by searching OMIM and usually derived from mapping data on affected families or chromosomal deletions in affected patients. Larger critical regions were preferentially selected to test more meaningfully whether the LgPCA model could have pinpointed the causal gene based solely on transcriptomic signatures that involved an affected organ(s) or tissue(s) (Column B). The average critical region was 13.7 Mb (Column D) and contained an average of 111 protein-coding genes (Column E; identified from searching BIOMART on ENSEMBL). In 48/53 instances (91%), LgPCA narrowed the field down to three or fewer transcription factors and in 37 instances (73%) excluded all except the correct transcription factor. Therefore, the same approach was applied to 13 unsolved developmental disorders (mostly deletion syndromes) with predictions made in each case for any type of protein-coding gene (Column H) and transcription factor(s) (Column I). In many instances the transcription factor in Column I possesses an appropriate mutant mouse phenotype. (I) 6251 unannotated transcripts identified during human organogenesis. These are the 6251 novel and distinct transcripts underlying Figure 4 of the main text, which also describes the transcript classification: Anti-sense (AS), Overlapping (OT), Bidirectional (BI), Long-intergenic non-coding (LINC) and / or Transcripts of uncertain coding potential (TUCP) (based on Mattick and Rinn, 2015). Intergenic transcripts are numbered sequentially within each chromosome. Exon lengths and starts (blocks) are recorded here in UCSC BED12 format. Correlations in expression profile were calculated for annotated genes with transcript transcriptional start sites situated within 1 Mb of the novel transcript TSS; the total number of genes in this window is listed. Columns AF-AT (organs and tissues) represent mean, quantile-normalised read counts across tissue replicates. Correlations (and distance) are shown for the closest, best correlated or best anti-correlated genes and were generated using only embryonic RNA-seq data. The pipeline to generate transcripts, distinguish them from previous annotations, name, characterise and filter is described in the Materials and methods. (J) NIH roadmap samples (Kundaje et al., 2015) used in this study.DOI: http://dx.doi.org/10.7554/eLife.15657.014\"),\n",
       " 'PMC8052288-mmc4.xlsx-PAF_DREAM_targets_H1792-Unnamed_1': ('PAF remodels the DREAM complex to bypass cell quiescence and promote lung tumorigenesis',\n",
       "  'The DREAM complex orchestrates cell quiescence and the cell cycle. However, how the DREAM complex is deregulated in cancer remains elusive. Here, we report that PAF (PCLAF/KIAA0101) drives cell quiescence exit to promote lung tumorigenesis by remodeling the DREAM complex. PAF is highly expressed in lung adenocarcinoma (LUAD) and is associated with poor prognosis. Importantly, Paf knockout markedly suppressed LUAD development in mouse models. PAF depletion induced LUAD cell quiescence and growth arrest. PAF is required for the global expression of cell-cycle genes controlled by the repressive DREAM complex. Mechanistically, PAF inhibits DREAM complex formation by binding to RBBP4, a core DREAM subunit, leading to transactivation of DREAM target genes. Furthermore, pharmacological mimicking of PAF-depleted transcriptomes inhibited LUAD tumor growth. Our results unveil how the PAF-remodeled DREAM complex bypasses cell quiescence to promote lung tumorigenesis and suggest that the PAF-DREAM axis may be a therapeutic vulnerability in lung cancer.',\n",
       "  'PAF_DREAM_targets_H1792-Unnamed_1',\n",
       "  'enriched both in mouse and human PAF-activated gene sets (Figures 4B–4D; Tables S2 and **S3**)  Table S3. PAF-DREAM targets and their correlation with poor clinical outcomes in LUAD, related to Figure 4'),\n",
       " 'PMC8052288-mmc4.xlsx-PAF-DREAM-Lung_Poor_Survival-Unnamed_2': ('PAF remodels the DREAM complex to bypass cell quiescence and promote lung tumorigenesis',\n",
       "  'The DREAM complex orchestrates cell quiescence and the cell cycle. However, how the DREAM complex is deregulated in cancer remains elusive. Here, we report that PAF (PCLAF/KIAA0101) drives cell quiescence exit to promote lung tumorigenesis by remodeling the DREAM complex. PAF is highly expressed in lung adenocarcinoma (LUAD) and is associated with poor prognosis. Importantly, Paf knockout markedly suppressed LUAD development in mouse models. PAF depletion induced LUAD cell quiescence and growth arrest. PAF is required for the global expression of cell-cycle genes controlled by the repressive DREAM complex. Mechanistically, PAF inhibits DREAM complex formation by binding to RBBP4, a core DREAM subunit, leading to transactivation of DREAM target genes. Furthermore, pharmacological mimicking of PAF-depleted transcriptomes inhibited LUAD tumor growth. Our results unveil how the PAF-remodeled DREAM complex bypasses cell quiescence to promote lung tumorigenesis and suggest that the PAF-DREAM axis may be a therapeutic vulnerability in lung cancer.',\n",
       "  'PAF-DREAM-Lung_Poor_Survival-Unnamed_2',\n",
       "  'enriched both in mouse and human PAF-activated gene sets (Figures 4B–4D; Tables S2 and **S3**)  Table S3. PAF-DREAM targets and their correlation with poor clinical outcomes in LUAD, related to Figure 4'),\n",
       " 'PMC10644098-BLOOD_BLD-2023-019721-mmc14.xlsx-all_clusterpreBII-small_pat_ctr-Unnamed_0': ('IL-7 receptor signaling drives human B-cell progenitor differentiation and expansion',\n",
       "  'Although absence of interleukin-7 (IL-7) signaling completely abrogates T and B lymphopoiesis in mice, patients with severe combined immunodeficiency caused by mutations in the IL-7 receptor α chain (IL-7Rα) still generate peripheral blood B cells. Consequently, human B lymphopoiesis has been thought to be independent of IL-7 signaling. Using flow cytometric analysis and single-cell RNA sequencing of bone marrow samples from healthy controls and patients who are IL-7Rα deficient, in combination with in vitro modeling of human B-cell differentiation, we demonstrate that IL-7R signaling plays a crucial role in human B lymphopoiesis. IL-7 drives proliferation and expansion of early B-cell progenitors but not of pre-BII large cells and has a limited role in the prevention of cell death. Furthermore, IL-7 guides cell fate decisions by enhancing the expression of BACH2, EBF1, and PAX5, which jointly orchestrate the specification and commitment of early B-cell progenitors. In line with this observation, early B-cell progenitors of patients with IL-7Rα deficiency still expressed myeloid-specific genes. Collectively, our results unveil a previously unknown role for IL-7 signaling in promoting the B-lymphoid fate and expanding early human B-cell progenitors while defining important differences between mice and humans. Our results have implications for hematopoietic stem cell transplantation strategies in patients with T- B+ severe combined immunodeficiency and provide insights into the role of IL-7R signaling in leukemogenesis.',\n",
       "  'all_clusterpreBII-small_pat_ctr-Unnamed_0',\n",
       "  'Supplemental Table 13'),\n",
       " 'PMC10354062-41525_2023_363_MOESM2_ESM.xlsx-Supplementary_Table_6-Unnamed_5': ('Chromatin regulators in the TBX1 network confer risk for conotruncal heart defects in 22q11.2DS',\n",
       "  'Congenital heart disease (CHD) affecting the conotruncal region of the heart, occurs in 40-50% of patients with 22q11.2 deletion syndrome (22q11.2DS). This syndrome is a rare disorder with relative genetic homogeneity that can facilitate identification of genetic modifiers. Haploinsufficiency of TBX1, encoding a T-box transcription factor, is one of the main genes responsible for the etiology of the syndrome. We suggest that genetic modifiers of conotruncal defects in patients with 22q11.2DS may be in the TBX1 gene network. To identify genetic modifiers, we analyzed rare, predicted damaging variants in whole genome sequence of 456 cases with conotruncal defects and 537 controls, with 22q11.2DS. We then performed gene set approaches and identified chromatin regulatory genes as modifiers. Chromatin genes with recurrent damaging variants include EP400, KAT6A, KMT2C, KMT2D, NSD1, CHD7 and PHF21A. In total, we identified 37 chromatin regulatory genes, that may increase risk for conotruncal heart defects in 8.5% of 22q11.2DS cases. Many of these genes were identified as risk factors for sporadic CHD in the general population. These genes are co-expressed in cardiac progenitor cells with TBX1, suggesting that they may be in the same genetic network. The genes KAT6A, KMT2C, CHD7 and EZH2, have been previously shown to genetically interact with TBX1 in mouse models. Our findings indicate that disturbance of chromatin regulatory genes impact the TBX1 gene network serving as genetic modifiers of 22q11.2DS and sporadic CHD, suggesting that there are some shared mechanisms involving the TBX1 gene network in the etiology of CHD.',\n",
       "  'Supplementary_Table_6-Unnamed_5',\n",
       "  'We obtained WGS from **1**  Supplementary Tables'),\n",
       " 'PMC10354062-41525_2023_363_MOESM2_ESM.xlsx-Supplementary_Table_6-Unnamed_4': ('Chromatin regulators in the TBX1 network confer risk for conotruncal heart defects in 22q11.2DS',\n",
       "  'Congenital heart disease (CHD) affecting the conotruncal region of the heart, occurs in 40-50% of patients with 22q11.2 deletion syndrome (22q11.2DS). This syndrome is a rare disorder with relative genetic homogeneity that can facilitate identification of genetic modifiers. Haploinsufficiency of TBX1, encoding a T-box transcription factor, is one of the main genes responsible for the etiology of the syndrome. We suggest that genetic modifiers of conotruncal defects in patients with 22q11.2DS may be in the TBX1 gene network. To identify genetic modifiers, we analyzed rare, predicted damaging variants in whole genome sequence of 456 cases with conotruncal defects and 537 controls, with 22q11.2DS. We then performed gene set approaches and identified chromatin regulatory genes as modifiers. Chromatin genes with recurrent damaging variants include EP400, KAT6A, KMT2C, KMT2D, NSD1, CHD7 and PHF21A. In total, we identified 37 chromatin regulatory genes, that may increase risk for conotruncal heart defects in 8.5% of 22q11.2DS cases. Many of these genes were identified as risk factors for sporadic CHD in the general population. These genes are co-expressed in cardiac progenitor cells with TBX1, suggesting that they may be in the same genetic network. The genes KAT6A, KMT2C, CHD7 and EZH2, have been previously shown to genetically interact with TBX1 in mouse models. Our findings indicate that disturbance of chromatin regulatory genes impact the TBX1 gene network serving as genetic modifiers of 22q11.2DS and sporadic CHD, suggesting that there are some shared mechanisms involving the TBX1 gene network in the etiology of CHD.',\n",
       "  'Supplementary_Table_6-Unnamed_4',\n",
       "  'We obtained WGS from **1**  Supplementary Tables'),\n",
       " 'PMC6797711-41467_2019_12582_MOESM2_ESM.xlsx-SuppData1_Genelists-Unnamed_10': ('De novo and recessive forms of congenital heart disease have distinct genetic and phenotypic landscapes',\n",
       "  'The genetic architecture of sporadic congenital heart disease (CHD) is characterized by enrichment in damaging de novo variants in chromatin-modifying genes. To test the hypothesis that gene pathways contributing to de novo forms of CHD are distinct from those for recessive forms, we analyze 2391 whole-exome trios from the Pediatric Cardiac Genomics Consortium. We deploy a permutation-based gene-burden analysis to identify damaging recessive and compound heterozygous genotypes and disease genes, controlling for confounding effects, such as background mutation rate and ancestry. Cilia-related genes are significantly enriched for damaging rare recessive genotypes, but comparatively depleted for de novo variants. The opposite trend is observed for chromatin-modifying genes. Other cardiac developmental gene classes have less stratification by mode of inheritance than cilia and chromatin-modifying gene classes. Our analyses reveal dominant and recessive CHD are associated with distinct gene functions, with cilia-related genes providing a reservoir of rare segregating variation leading to CHD.',\n",
       "  'SuppData1_Genelists-Unnamed_10',\n",
       "  'Supplementary Dataset 1'),\n",
       " 'PMC6876711-EMS84316-supplement-Suppl_Table_10.xlsx-Gene_per_Signature-Module_1': ('Resolving the fibrotic niche of human liver cirrhosis at single-cell level',\n",
       "  'Liver cirrhosis is a major cause of death worldwide and is characterized by extensive fibrosis. There are currently no effective antifibrotic therapies available. To obtain a better understanding of the cellular and molecular mechanisms involved in disease pathogenesis and enable the discovery of therapeutic targets, here we profile the transcriptomes of more than 100,000 single human cells, yielding molecular definitions for non-parenchymal cell types that are found in healthy and cirrhotic human liver. We identify a scar-associated TREM2 + CD9 + subpopulation of macrophages, which expands in liver fibrosis, differentiates from circulating monocytes and is pro-fibrogenic. We also define ACKR1 + and PLVAP + endothelial cells that expand in cirrhosis, are topographically restricted to the fibrotic niche and enhance the transmigration of leucocytes. Multi-lineage modelling of ligand and receptor interactions between the scar-associated macrophages, endothelial cells and PDGFRα + collagen-producing mesenchymal cells reveals intra-scar activity of several pro-fibrogenic pathways including TNFRSF12A, PDGFR and NOTCH signalling. Our work dissects unanticipated aspects of the cellular and molecular basis of human organ fibrosis at a single-cell level, and provides a conceptual framework for the discovery of rational therapeutic targets in liver cirrhosis.',\n",
       "  'Gene_per_Signature-Module_1',\n",
       "  'None here'),\n",
       " 'PMC3920930-cddis2013412x8.xls-DEGs-24h-Symbol': ('Transcriptome analysis of glioma cells for the dynamic response to γ-irradiation and dual regulation of apoptosis genes: a new insight into radiotherapy for glioblastomas',\n",
       "  'Ionizing radiation (IR) is of clinical importance for glioblastoma therapy; however, the recurrence of glioma characterized by radiation resistance remains a therapeutic challenge. Research on irradiation-induced transcription in glioblastomas can contribute to the understanding of radioresistance mechanisms. In this study, by using the total mRNA sequencing (RNA-seq) analysis, we assayed the global gene expression in a human glioma cell line U251 MG at various time points after exposure to a growth arrest dose of γ-rays. We identified 1656 genes with obvious changes at the transcriptional level in response to irradiation, and these genes were dynamically enriched in various biological processes or pathways, including cell cycle arrest, DNA replication, DNA repair and apoptosis. Interestingly, the results showed that cell death was not induced even many proapoptotic molecules, including death receptor 5 (DR5) and caspases were activated after radiation. The RNA-seq data analysis further revealed that both proapoptosis and antiapoptosis genes were affected by irradiation. Namely, most proapoptosis genes were early continually responsive, whereas antiapoptosis genes were responsive at later stages. Moreover, HMGB1, HMGB2 and TOP2A involved in the positive regulation of DNA fragmentation during apoptosis showed early continual downregulation due to irradiation. Furthermore, targeting of the TRAIL/DR5 pathway after irradiation led to significant apoptotic cell death, accompanied by the recovered gene expression of HMGB1, HMGB2 and TOP2A. Taken together, these results revealed that inactivation of proapoptotic signaling molecules in the nucleus and late activation of antiapoptotic genes may contribute to the radioresistance of gliomas. Overall, this study provided novel insights into not only the underlying mechanisms of radioresistance in glioblastomas but also the screening of multiple targets for radiotherapy.',\n",
       "  'DEGs-24h-Symbol',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC7689470-41598_2020_76347_MOESM2_ESM.xlsx-CF-WholeBlood-Symbol': ('Integrative genomic meta-analysis reveals novel molecular insights into cystic fibrosis and ΔF508-CFTR rescue',\n",
       "  'Cystic fibrosis (CF), caused by mutations to CFTR, leads to severe and progressive lung disease. The most common mutant, ΔF508-CFTR, undergoes proteasomal degradation, extinguishing its anion channel function. Numerous in vitro interventions have been identified to partially rescue ΔF508-CFTR function yet remain poorly understood. Improved understanding of both the altered state of CF cells and the mechanisms of existing rescue strategies could reveal novel therapeutic strategies. Toward this aim, we measured transcriptional profiles of established temperature, genetic, and chemical interventions that rescue ΔF508-CFTR and also re-analyzed public datasets characterizing transcription in human CF vs. non-CF samples from airway and whole blood. Meta-analysis yielded a core disease signature and two core rescue signatures. To interpret these through the lens of prior knowledge, we compiled a \"CFTR Gene Set Library\" from literature. The core disease signature revealed remarkably strong connections to genes with established effects on CFTR trafficking and function and suggested novel roles of EGR1 and SGK1 in the disease state. Our data also revealed an unexpected mechanistic link between several genetic rescue interventions and the unfolded protein response. Finally, we found that C18, an analog of the CFTR corrector compound Lumacaftor, induces almost no transcriptional perturbation despite its rescue activity.',\n",
       "  'CF-WholeBlood-Symbol',\n",
       "  'For this analysis, we used **1**  Additional File 1 - Individual Signatures'),\n",
       " 'PMC6545701-40880_2019_376_MOESM5_ESM.xlsx-Ark1-Unnamed_5': ('PD-L1 and PD-L2 expression correlated genes in non-small-cell lung cancer',\n",
       "  'Background: Programmed cell death ligand-1 (PD-L1) and ligand-2 (PD-L2) interaction with programmed cell death protein-1 (PD-1) represent an immune-inhibiting checkpoint mediating immune evasion and is, accordingly, an important target for blockade-based immunotherapy in cancer. In non-small-cell lung cancer (NSCLC), improved understanding of PD-1 checkpoint blockade-responsive biology and identification of biomarkers for prediction of a clinical response to immunotherapy is warranted. Thus, in the present study, we systematically described PD-L1 and PD-L2 expression correlated genes in NSCLC. Methods: We performed comparative retrospective analyses to identify PD-L1 and PD-L2 mRNA expression correlated genes in NSCLC. For this, we examined available datasets from the cancer cell line encyclopedia (CCLE) project lung non-small-cell (Lung_NSC) and the cancer genome atlas (TCGA) projects lung adenocarcinoma (LUAD) and squamous cell carcinoma (LUSC). Results: Analysis of the CCLE dataset Lung_NSC identified expression correlation between PD-L1 and PD-L2. Moreover, we identified expression correlation between 489 genes and PD-L1, 191 genes and PD-L2, and 111 genes for both. PD-L1 and PD-L2 also expression correlated in TCGA datasets LUAD and LUSC. In LUAD, we identified expression correlation between 257 genes and PD-L1, 914 genes and PD-L2, and 211 genes for both. In LUSC, we identified expression correlation between 26 genes and PD-L1, 326 genes and PD-L2, and 13 genes for both. Only a few genes expression correlated with PD-L1 and PD-L2 across the CCLE and TCGA datasets. Expression of Interferon signaling-involved genes converged in particular with the expression correlated genes for PD-L1 in Lung_NSC, for PD-L2 in LUSC, and for both PD-L1 and PD-L2 in LUAD. In LUSC, PD-L1, and to a lesser extent PD-L2, expression correlated with chromosome 9p24 localized genes, indicating a chromosome 9p24 topologically associated domain as an important driver of in particular LUSC PD-L1 expression. Expression correlation analyses of the PD-L1 and PD-L2 receptors programmed cell death protein-1 (PD-1), Cluster of differentiation 80 (CD80), and Repulsive guidance molecule B (RGMB) showed that PD-1 and CD80 expression correlated with both PD-L1 and PD-L2 in LUAD. CD80 expression correlated with PD-L2 in LUSC. Conclusions: We present gene signatures associated with PD-L1 and PD-L2 mRNA expression in NSCLC which could possess importance in relation to understand PD-1 checkpoint blockade-responsive biology and development of gene signature based biomarkers for predicting clinical responses to immunotherapy.',\n",
       "  'Ark1-Unnamed_5',\n",
       "  'correlation coefficient r ≥ 0.3 or ≤ − 0.3, and all P values < 0.0**5**  Additional file 5: Table S4. Expression correlated genes for IRF1 and IRF9'),\n",
       " 'PMC10014581-41556_2022_1075_MOESM2_ESM.xlsx-1d-Unnamed_1': ('Expansion of ventral foregut is linked to changes in the enhancer landscape for organ-specific differentiation',\n",
       "  'Cell proliferation is fundamental for almost all stages of development and differentiation that require an increase in cell number. Although cell cycle phase has been associated with differentiation, the actual process of proliferation has not been considered as having a specific role. Here we exploit human embryonic stem cell-derived endodermal progenitors that we find are an in vitro model for the ventral foregut. These cells exhibit expansion-dependent increases in differentiation efficiency to pancreatic progenitors that are linked to organ-specific enhancer priming at the level of chromatin accessibility and the decommissioning of lineage-inappropriate enhancers. Our findings suggest that cell proliferation in embryonic development is about more than tissue expansion; it is required to ensure equilibration of gene regulatory networks allowing cells to become primed for future differentiation. Expansion of lineage-specific intermediates may therefore be an important step in achieving high-fidelity in vitro differentiation.',\n",
       "  '1d-Unnamed_1',\n",
       "  'downregulated primitive-streak markers (GSC, CER1 and LEFTY1) (Extended Data Fig. 4a and Supplementary Table **1a**)  Summary of dynamic ATAC peaks and their annotated genes classified by differential expression between VFGs and PE samples'),\n",
       " 'PMC11003700-pbio.3002240.s013.xlsx-siWDR5_RNA_IMR32_48h_q_0_05_1_5-Gene_ID': ('MYCN drives oncogenesis by cooperating with the histone methyltransferase G9a and the WDR5 adaptor to orchestrate global gene transcription',\n",
       "  \"MYCN activates canonical MYC targets involved in ribosome biogenesis, protein synthesis, and represses neuronal differentiation genes to drive oncogenesis in neuroblastoma (NB). How MYCN orchestrates global gene expression remains incompletely understood. Our study finds that MYCN binds promoters to up-regulate canonical MYC targets but binds to both enhancers and promoters to repress differentiation genes. MYCN binding also increases H3K4me3 and H3K27ac on canonical MYC target promoters and decreases H3K27ac on neuronal differentiation gene enhancers and promoters. WDR5 facilitates MYCN promoter binding to activate canonical MYC target genes, whereas MYCN recruits G9a to enhancers to repress neuronal differentiation genes. Targeting both MYCN's active and repressive transcriptional activities using both WDR5 and G9a inhibitors synergistically suppresses NB growth. We demonstrate that MYCN cooperates with WDR5 and G9a to orchestrate global gene transcription. The targeting of both these cofactors is a novel therapeutic strategy to indirectly target the oncogenic activity of MYCN.\",\n",
       "  'siWDR5_RNA_IMR32_48h_q_0_05_1_5-Gene_ID',\n",
       "  'None here'),\n",
       " 'PMC8957021-mmc11.xlsx-T_cells-gene': ('Single-cell temporal analysis of natural dengue infection reveals skin-homing lymphocyte expansion one day before defervescence',\n",
       "  'Effective clinical management of acute dengue virus (DENV) infection relies on the timing of suitable treatments during the disease progression. We analyzed single-cell transcriptomic profiles of the peripheral blood mononuclear cell samples from two DENV patients, collected daily during acute phase and also at convalescence. Key immune cell types demonstrated different dynamic responses over the course of the infection. On the day before defervescence (Day -1), we observed the peak expression of several prominent genes in the adaptive immunological pathways. We also characterized unique effector T cell clusters that expressed skin-homing signature genes at Day -1, whereas upregulation of skin and gut homing genes was also observed in plasma cells and plasmablasts during the febrile period. This work provides an overview of unique molecular dynamics that signify the entry of the critical phase, and the findings could improve the patient management of DENV infection.',\n",
       "  'T_cells-gene',\n",
       "  'Tables S6, S7, S8, and S9 and the percentages of cells expressing HVGs in **Table S10**  Table S10. Percentages of cells in populations of interest that expressed particular genes, related to Figures 2C, S3 and STAR Methods'),\n",
       " 'PMC9283889-mmc2.xlsx-Endothelial_cell-gene': ('Establishment of a reference single-cell RNA sequencing dataset for human pancreatic adenocarcinoma',\n",
       "  'Single-cell RNA sequencing (scRNAseq) has been used to assess the intra-tumor heterogeneity and microenvironment of pancreatic ductal adenocarcinoma (PDAC). However, previous knowledge is not fully universalized. Here, we built a single cell atlas of PDAC from six datasets containing over 70 samples and >130,000 cells, and demonstrated its application to the reanalysis of the previous bulk transcriptomic cohorts and inferring cell-cell communications. The cell decomposition of bulk transcriptomics using scRNAseq data showed the cellular heterogeneity of PDAC; moreover, high levels of tumor cells and fibroblasts were indicative of poor-prognosis. Refined tumor subtypes signature indicated the tumor cell dynamics in intra-tumor and their specific regulatory network. We further identified functionally distinct tumor clusters that had close interaction with fibroblast subtypes via different signaling pathways dependent on subtypes. Our analysis provided a reference dataset for PDAC and showed its utility in research on the microenvironment of intra-tumor heterogeneity.',\n",
       "  'Endothelial_cell-gene',\n",
       "  '2, respectively. The integrated data included 10 cell types with reasonable differentially expressed genes (**Table S1**)  Table S1. Differential expression genes in scRNAseq data of PDAC, related to Figure 1'),\n",
       " 'PMC10835465-mmc10.xlsx-Cluster_27-gene': ('Association of HBsAg levels with differential gene expression in NK, CD8 T, and memory B cells in treated patients with chronic HBV',\n",
       "  'Background & aims: HBsAg secretion may impact immune responses to chronic HBV infection. Thus, therapeutic approaches to suppress HBsAg production are being investigated. Our study aims to examine the immunomodulatory effects of high and low levels of circulating HBsAg and thereby improve our understanding of anti-HBV immunity. Methods: An optimized 10x Genomics single-cell RNA sequencing workflow was applied to blood samples and liver fine-needle aspirates from 18 patients undergoing tenofovir/entecavir (NUC) treatment for chronic HBV infection. They were categorized based on their HBsAg levels: high (920-12,447 IU/ml) or low (1-100 IU/ml). Cluster frequencies, differential gene expression, and phenotypes were analyzed. Results: In the blood of HBV-infected patients on NUC, the proportion of KLRC2 + \"adaptive\" natural killer (NK) cells was significantly lower in the HBsAg-high group and, remarkably, both KLRC2 + NK and KLRG1+ CD8 T cells display enrichment of lymphocyte activation-associated gene sets in the HBsAg-low group. High levels of HBsAg were associated with mild immune activation in the liver. However, no suppression of liver-resident CXCR6+ NCAM1+ NK or CXCR6+ CD69+ CD8 T cells was detected, while memory B cells showed signs of activation in both the blood and liver. Conclusions: Among NUC-treated patients, we observed a minimal impact of HBsAg on leukocyte populations in the blood and liver. However, for the first time, we found that HBsAg has distinct effects, restricted to NK-, CD8 T-, and memory B-cell subsets, in the blood and liver. Our findings are highly relevant for current clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Impact and implications: This study provides unique insight into the impact of HBsAg on gene expression levels of immune cell subsets in the blood and liver, particularly in the context of NUC-treated chronic HBV infection. It holds significant relevance for current and future clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Our findings raise questions about the effectiveness of such treatment strategies and challenge the previously hypothesized immunomodulatory effects of HBsAg on immune responses against HBV.',\n",
       "  'Cluster_27-gene',\n",
       "  'None here'),\n",
       " 'PMC5405195-mmc5.xlsx-qvalue_0_1-Gene_lists_containing_significantly_up-_and_downregulated_genes': ('c-Myc Antagonises the Transcriptional Activity of the Androgen Receptor in Prostate Cancer Affecting Key Gene Networks',\n",
       "  'Prostate cancer (PCa) is the most common non-cutaneous cancer in men. The androgen receptor (AR), a ligand-activated transcription factor, constitutes the main drug target for advanced cases of the disease. However, a variety of other transcription factors and signaling networks have been shown to be altered in patients and to influence AR activity. Amongst these, the oncogenic transcription factor c-Myc has been studied extensively in multiple malignancies and elevated protein levels of c-Myc are commonly observed in PCa. Its impact on AR activity, however, remains elusive. In this study, we assessed the impact of c-Myc overexpression on AR activity and transcriptional output in a PCa cell line model and validated the antagonistic effect of c-MYC on AR-targets in patient samples. We found that c-Myc overexpression partially reprogrammed AR chromatin occupancy and was associated with altered histone marks distribution, most notably H3K4me1 and H3K27me3. We found c-Myc and the AR co-occupy a substantial number of binding sites and these exhibited enhancer-like characteristics. Interestingly, c-Myc overexpression antagonised clinically relevant AR target genes. Therefore, as an example, we validated the antagonistic relationship between c-Myc and two AR target genes, KLK3 (alias PSA, prostate specific antigen), and Glycine N-Methyltransferase (GNMT), in patient samples. Our findings provide unbiased evidence that MYC overexpression deregulates the AR transcriptional program, which is thought to be a driving force in PCa.',\n",
       "  'qvalue_0_1-Gene_lists_containing_significantly_up-_and_downregulated_genes',\n",
       "  'genes were further increased in expression when MYC levels were elevated (Fig. 2c–f and **Table S4**)  Table S4Gene lists containing significantly up- and downregulated genes'),\n",
       " 'PMC8316550-41467_2021_24482_MOESM8_ESM.xlsx-Supplementary_Data5_1-Unnamed_2': ('The trans-omics landscape of COVID-19',\n",
       "  'The outbreak of coronavirus disease 2019 (COVID-19) is a global health emergency. Various omics results have been reported for COVID-19, but the molecular hallmarks of COVID-19, especially in those patients without comorbidities, have not been fully investigated. Here we collect blood samples from 231 COVID-19 patients, prefiltered to exclude those with selected comorbidities, yet with symptoms ranging from asymptomatic to critically ill. Using integrative analysis of genomic, transcriptomic, proteomic, metabolomic and lipidomic profiles, we report a trans-omics landscape for COVID-19. Our analyses find neutrophils heterogeneity between asymptomatic and critically ill patients. Meanwhile, neutrophils over-activation, arginine depletion and tryptophan metabolites accumulation correlate with T cell dysfunction in critical patients. Our multi-omics data and characterization of peripheral blood from COVID-19 patients may thus help provide clues regarding pathophysiology of and potential therapeutic strategies for COVID-19.',\n",
       "  'Supplementary_Data5_1-Unnamed_2',\n",
       "  'expression patterns were identified across patients with different disease severities (Fig. 2a, Supplementary Data **5.1**)  Supplementary Data 5'),\n",
       " 'PMC6544546-EMS81327-supplement-Supplementary_Table_3.xlsx-Neuron1-gene': ('Altered human oligodendrocyte heterogeneity in multiple sclerosis',\n",
       "  'Oligodendrocyte pathology is increasingly implicated in neurodegenerative diseases as oligodendrocytes both myelinate and provide metabolic support to axons. In multiple sclerosis (MS), demyelination in the central nervous system thus leads to neurodegeneration, but the severity of MS between patients is very variable. Disability does not correlate well with the extent of demyelination 1 , which suggests that other factors contribute to this variability. One such factor may be oligodendrocyte heterogeneity. Not all oligodendrocytes are the same-those from the mouse spinal cord inherently produce longer myelin sheaths than those from the cortex 2 , and single-cell analysis of the mouse central nervous system identified further differences 3,4 . However, the extent of human oligodendrocyte heterogeneity and its possible contribution to MS pathology remain unknown. Here we performed single-nucleus RNA sequencing from white matter areas of post-mortem human brain from patients with MS and from unaffected controls. We identified subclusters of oligodendroglia in control human white matter, some with similarities to mouse, and defined new markers for these cell states. Notably, some subclusters were underrepresented in MS tissue, whereas others were more prevalent. These differences in mature oligodendrocyte subclusters may indicate different functional states of oligodendrocytes in MS lesions. We found similar changes in normal-appearing white matter, showing that MS is a more diffuse disease than its focal demyelination suggests. Our findings of an altered oligodendroglial heterogeneity in MS may be important for understanding disease progression and developing therapeutic approaches.',\n",
       "  'Neuron1-gene',\n",
       "  'smooth muscle cells (VSM), pericytes, endothelial cells, and immune cells (Fig.1a, Extended Fig.1g and **Supplementary Table3**)'),\n",
       " 'PMC7346435-12864_2020_6860_MOESM5_ESM.xlsx-HE_H2_Genes-Gene_SymbolH': ('Comparative analysis of single-cell transcriptomics in human and Zebrafish oocytes',\n",
       "  'Background: Zebrafish is a popular model organism, which is widely used in developmental biology research. Despite its general use, the direct comparison of the zebrafish and human oocyte transcriptomes has not been well studied. It is significant to see if the similarity observed between the two organisms at the gene sequence level is also observed at the expression level in key cell types such as the oocyte. Results: We performed single-cell RNA-seq of the zebrafish oocyte and compared it with two studies that have performed single-cell RNA-seq of the human oocyte. We carried out a comparative analysis of genes expressed in the oocyte and genes highly expressed in the oocyte across the three studies. Overall, we found high consistency between the human studies and high concordance in expression for the orthologous genes in the two organisms. According to the Ensembl database, about 60% of the human protein coding genes are orthologous to the zebrafish genes. Our results showed that a higher percentage of the genes that are highly expressed in both organisms show orthology compared to the lower expressed genes. Systems biology analysis of the genes highly expressed in the three studies showed significant overlap of the enriched pathways and GO terms. Moreover, orthologous genes that are commonly overexpressed in both organisms were involved in biological mechanisms that are functionally essential to the oocyte. Conclusions: Orthologous genes are concurrently highly expressed in the oocytes of the two organisms and these genes belong to similar functional categories. Our results provide evidence that zebrafish could serve as a valid model organism to study the oocyte with direct implications in human.',\n",
       "  'HE_H2_Genes-Gene_SymbolH',\n",
       "  '1.5 in two out of the three replicates in each study. This resulted in **4**  Additional file 5: Supplementary file 4. Lists of highly expressed genes in the three data sets (H1, H2, ZF) and statistically significantly overrepresented Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways, and Gene Ontology (GO) Biological Process (BP), Molecular Function (MF), and Cellular Component (CC) categories based on Database for Annotation, Visualization and Integrated Discovery (DAVID v6.8) analysis. DAVID analysis is done separately for the three highly expressed gene lists and a comparative analysis of these individual results are presented'),\n",
       " 'PMC3589490-pone.0057886.s003.xls-Downregulated_probes-TARGET_ID': ('Integrative genome-wide gene expression profiling of clear cell renal cell carcinoma in Czech Republic and in the United States',\n",
       "  'Gene expression microarray and next generation sequencing efforts on conventional, clear cell renal cell carcinoma (ccRCC) have been mostly performed in North American and Western European populations, while the highest incidence rates are found in Central/Eastern Europe. We conducted whole-genome expression profiling on 101 pairs of ccRCC tumours and adjacent non-tumour renal tissue from Czech patients recruited within the \"K2 Study\", using the Illumina HumanHT-12 v4 Expression BeadChips to explore the molecular variations underlying the biological and clinical heterogeneity of this cancer. Differential expression analysis identified 1650 significant probes (fold change ≥2 and false discovery rate <0.05) mapping to 630 up- and 720 down-regulated unique genes. We performed similar statistical analysis on the RNA sequencing data of 65 ccRCC cases from the Cancer Genome Atlas (TCGA) project and identified 60% (402) of the downregulated and 74% (469) of the upregulated genes found in the K2 series. The biological characterization of the significantly deregulated genes demonstrated involvement of downregulated genes in metabolic and catabolic processes, excretion, oxidation reduction, ion transport and response to chemical stimulus, while simultaneously upregulated genes were associated with immune and inflammatory responses, response to hypoxia, stress, wounding, vasculature development and cell activation. Furthermore, genome-wide DNA methylation analysis of 317 TCGA ccRCC/adjacent non-tumour renal tissue pairs indicated that deregulation of approximately 7% of genes could be explained by epigenetic changes. Finally, survival analysis conducted on 89 K2 and 464 TCGA cases identified 8 genes associated with differential prognostic outcomes. In conclusion, a large proportion of ccRCC molecular characteristics were common to the two populations and several may have clinical implications when validated further through large clinical cohorts.',\n",
       "  'Downregulated_probes-TARGET_ID',\n",
       "  'Hochberg (BH) false discovery rate (FDR) adjusted p-value <0.05 for the paired t-test comparison, **Table S1**  Complete list of statistically significant differentially expressed probes (FDR-adjusted p-value (BH) <0.05, FC ≥2) in ccRCC as compared with adjacent non-tumour tissues in Czech Republic (K2 series)–whole-genome expression profiling using microarrays. (XLS)  Click here for additional data file'),\n",
       " 'PMC6501285-13075_2019_1894_MOESM3_ESM.xlsx-UPREGULATED-Input': ('RNA sequencing of mesenchymal stem cells reveals a blocking of differentiation and immunomodulatory activities under inflammatory conditions in rheumatoid arthritis patients',\n",
       "  'Introduction: Mesenchymal stem cells (MSCs) have the ability to differentiate into different types of cells of the mesenchymal lineage, such as osteocytes, chondrocytes, and adipocytes. It is also known that under inflammatory stimuli or in the appropriate experimental conditions, they can also act as regulators of inflammation. Thus, in addition to their regenerating potential, their interest has been extended to their possible use in cell therapy strategies for treatment of immune disorders. Objective: To analyze, by RNA-seq analysis, the transcriptome profiling of allogenic MSCs under RA lymphocyte activation. Methods: We identified the differentially expressed genes in bone marrow mesenchymal stem cells after exposure to an inflammatory environment. The transcriptome profiling was evaluated by means of the precise measurement of transcripts provided by the RNA-Seq technology. Results: Our results evidenced the existence of blocking of both regenerative (differentiation) and immunomodulatory phenotypes under inflammatory conditions characterized by an upregulation of genes involved in immune processes and a simultaneous downregulation of genes mainly involved in regenerative or cell differentiation functions. Conclusions: We conclude that the two main functions of MSCs (immunomodulation and differentiation) are blocked, at least while the inflammation is being resolved. Inflammation, at least partially mediated by gamma-interferon, drives MSCs to a cellular distress adopting a defensive state. This knowledge could be of particular interest in cases where the damage to be repaired has an important immune-mediated component.',\n",
       "  'UPREGULATED-Input',\n",
       "  'PBMCs and exposed to resting RA PBMCs, we observed 847 DEGs in total, with 2**3**  Table S3. Number of reads successfully mapped and assigned to the metafeature “gene” per sample. (XLSX 3343 kb)'),\n",
       " 'PMC7346435-12864_2020_6860_MOESM5_ESM.xlsx-HE_H2_Genes-GeneH': ('Comparative analysis of single-cell transcriptomics in human and Zebrafish oocytes',\n",
       "  'Background: Zebrafish is a popular model organism, which is widely used in developmental biology research. Despite its general use, the direct comparison of the zebrafish and human oocyte transcriptomes has not been well studied. It is significant to see if the similarity observed between the two organisms at the gene sequence level is also observed at the expression level in key cell types such as the oocyte. Results: We performed single-cell RNA-seq of the zebrafish oocyte and compared it with two studies that have performed single-cell RNA-seq of the human oocyte. We carried out a comparative analysis of genes expressed in the oocyte and genes highly expressed in the oocyte across the three studies. Overall, we found high consistency between the human studies and high concordance in expression for the orthologous genes in the two organisms. According to the Ensembl database, about 60% of the human protein coding genes are orthologous to the zebrafish genes. Our results showed that a higher percentage of the genes that are highly expressed in both organisms show orthology compared to the lower expressed genes. Systems biology analysis of the genes highly expressed in the three studies showed significant overlap of the enriched pathways and GO terms. Moreover, orthologous genes that are commonly overexpressed in both organisms were involved in biological mechanisms that are functionally essential to the oocyte. Conclusions: Orthologous genes are concurrently highly expressed in the oocytes of the two organisms and these genes belong to similar functional categories. Our results provide evidence that zebrafish could serve as a valid model organism to study the oocyte with direct implications in human.',\n",
       "  'HE_H2_Genes-GeneH',\n",
       "  '1.5 in two out of the three replicates in each study. This resulted in **4**  Additional file 5: Supplementary file 4. Lists of highly expressed genes in the three data sets (H1, H2, ZF) and statistically significantly overrepresented Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways, and Gene Ontology (GO) Biological Process (BP), Molecular Function (MF), and Cellular Component (CC) categories based on Database for Annotation, Visualization and Integrated Discovery (DAVID v6.8) analysis. DAVID analysis is done separately for the three highly expressed gene lists and a comparative analysis of these individual results are presented'),\n",
       " 'PMC8321697-mmc3.xlsx-cytokine-regulated_genes-Table_S2_related_to_Figures_1-4_genes_significantly_deregulated_by_cytokine_treatments_mean_count_5_p-adjusted_for_variance_0_05_log2_fold_change_1_OR_-1': ('JAK inhibitors dampen activation of interferon-activated transcriptomes and the SARS-CoV-2 receptor ACE2 in human renal proximal tubules',\n",
       "  'SARS-CoV-2 infections initiate cytokine storms and activate genetic programs leading to progressive hyperinflammation in multiple organs of patients with COVID-19. While it is known that COVID-19 impacts kidney function, leading to increased mortality, cytokine response of renal epithelium has not been studied in detail. Here, we report on the genetic programs activated in human primary proximal tubule (HPPT) cells by interferons and their suppression by ruxolitinib, a Janus kinase (JAK) inhibitor used in COVID-19 treatment. Integration of our data with those from patients with acute kidney injury and COVID-19, as well as other tissues, permitted the identification of kidney-specific interferon responses. Additionally, we investigated the regulation of the recently discovered isoform (dACE2) of the angiotensin-converting enzyme 2 (ACE2), the SARS-CoV-2 receptor. Using ChIP-seq, we identified candidate interferon-activated enhancers controlling the ACE2 locus, including the intronic dACE2 promoter. Taken together, our study provides an in-depth understanding of genetic programs activated in kidney cells.',\n",
       "  'cytokine-regulated_genes-Table_S2_related_to_Figures_1-4_genes_significantly_deregulated_by_cytokine_treatments_mean_count_5_p-adjusted_for_variance_0_05_log2_fold_change_1_OR_-1',\n",
       "  'IFNα, 1169 by IFNβ, 1280 by IFNγ, and 2190 by IL-1β (Tables S1 and **S2**)  Table S2. Genes significantly deregulated by cytokine treatments, related to Figures 1–4List of significantly upregulated and downregulated genes in HPPT cells treated with IFNα, IFNβ, IFNγ, IL-1β, ruxolitinib + IFNβ'),\n",
       " 'PMC5784090-41598_2018_19837_MOESM2_ESM.xlsx-Table_S5-Unnamed_1': ('Persistent activation of interlinked type 2 airway epithelial gene networks in sputum-derived cells from aeroallergen-sensitized symptomatic asthmatics',\n",
       "  'Atopic asthma is a persistent disease characterized by intermittent wheeze and progressive loss of lung function. The disease is thought to be driven primarily by chronic aeroallergen-induced type 2-associated inflammation. However, the vast majority of atopics do not develop asthma despite ongoing aeroallergen exposure, suggesting additional mechanisms operate in conjunction with type 2 immunity to drive asthma pathogenesis. We employed RNA-Seq profiling of sputum-derived cells to identify gene networks operative at baseline in house dust mite-sensitized (HDM S ) subjects with/without wheezing history that are characteristic of the ongoing asthmatic state. The expression of type 2 effectors (IL-5, IL-13) was equivalent in both cohorts of subjects. However, in HDM S -wheezers they were associated with upregulation of two coexpression modules comprising multiple type 2- and epithelial-associated genes. The first module was interlinked by the hubs EGFR, ERBB2, CDH1 and IL-13. The second module was associated with CDHR3 and mucociliary clearance genes. Our findings provide new insight into the molecular mechanisms operative at baseline in the airway mucosa in atopic asthmatics undergoing natural aeroallergen exposure, and suggest that susceptibility to asthma amongst these subjects involves complex interactions between type 2- and epithelial-associated gene networks, which are not operative in equivalently sensitized/exposed atopic non-asthmatics.',\n",
       "  'Table_S5-Unnamed_1',\n",
       "  'Supplementary Tables S1–11'),\n",
       " 'PMC8323949-pcbi.1009228.s007.xlsx-HNSCC_ALL_exceptLN-X12': ('Single-cell analysis reveals the pan-cancer invasiveness-associated transition of adipose-derived stromal cells into COL11A1-expressing cancer-associated fibroblasts',\n",
       "  'During the last ten years, many research results have been referring to a particular type of cancer-associated fibroblasts associated with poor prognosis, invasiveness, metastasis and resistance to therapy in multiple cancer types, characterized by a gene expression signature with prominent presence of genes COL11A1, THBS2 and INHBA. Identifying the underlying biological mechanisms responsible for their creation may facilitate the discovery of targets for potential pan-cancer therapeutics. Using a novel computational approach for single-cell gene expression data analysis identifying the dominant cell populations in a sequence of samples from patients at various stages, we conclude that these fibroblasts are produced by a pan-cancer cellular transition originating from a particular type of adipose-derived stromal cells naturally present in the stromal vascular fraction of normal adipose tissue, having a characteristic gene expression signature. Focusing on a rich pancreatic cancer dataset, we provide a detailed description of the continuous modification of the gene expression profiles of cells as they transition from APOD-expressing adipose-derived stromal cells to COL11A1-expressing cancer-associated fibroblasts, identifying the key genes that participate in this transition. These results also provide an explanation to the well-known fact that the adipose microenvironment contributes to cancer progression.',\n",
       "  'HNSCC_ALL_exceptLN-X12',\n",
       "  'MMP11, INHBA, THBS2, COL10A1, COL8A1, FN1 and ASC associated genes (APOD, C7, PTGDS), respectively (**S4 Table**)  Differentially expressed genes among different clusters of HNSCC dataset.(XLSX)  Click here for additional data file'),\n",
       " 'PMC8323949-pcbi.1009228.s007.xlsx-X12-Unnamed_0': ('Single-cell analysis reveals the pan-cancer invasiveness-associated transition of adipose-derived stromal cells into COL11A1-expressing cancer-associated fibroblasts',\n",
       "  'During the last ten years, many research results have been referring to a particular type of cancer-associated fibroblasts associated with poor prognosis, invasiveness, metastasis and resistance to therapy in multiple cancer types, characterized by a gene expression signature with prominent presence of genes COL11A1, THBS2 and INHBA. Identifying the underlying biological mechanisms responsible for their creation may facilitate the discovery of targets for potential pan-cancer therapeutics. Using a novel computational approach for single-cell gene expression data analysis identifying the dominant cell populations in a sequence of samples from patients at various stages, we conclude that these fibroblasts are produced by a pan-cancer cellular transition originating from a particular type of adipose-derived stromal cells naturally present in the stromal vascular fraction of normal adipose tissue, having a characteristic gene expression signature. Focusing on a rich pancreatic cancer dataset, we provide a detailed description of the continuous modification of the gene expression profiles of cells as they transition from APOD-expressing adipose-derived stromal cells to COL11A1-expressing cancer-associated fibroblasts, identifying the key genes that participate in this transition. These results also provide an explanation to the well-known fact that the adipose microenvironment contributes to cancer progression.',\n",
       "  'X12-Unnamed_0',\n",
       "  'MMP11, INHBA, THBS2, COL10A1, COL8A1, FN1 and ASC associated genes (APOD, C7, PTGDS), respectively (**S4 Table**)  Differentially expressed genes among different clusters of HNSCC dataset.(XLSX)  Click here for additional data file'),\n",
       " 'PMC11120958-Table1.XLSX-Table_S5-Supplementary_Table_S5_Minimal_set_of_TFs_and_disjoint_sets_of_TFs_for_each_of_the_target_gene_in_Human': ('Bipartite networks represent causality better than simple networks: evidence, algorithms, and applications',\n",
       "  'A network, whose nodes are genes and whose directed edges represent positive or negative influences of a regulatory gene and its targets, is often used as a representation of causality. To infer a network, researchers often develop a machine learning model and then evaluate the model based on its match with experimentally verified \"gold standard\" edges. The desired result of such a model is a network that may extend the gold standard edges. Since networks are a form of visual representation, one can compare their utility with architectural or machine blueprints. Blueprints are clearly useful because they provide precise guidance to builders in construction. If the primary role of gene regulatory networks is to characterize causality, then such networks should be good tools of prediction because prediction is the actionable benefit of knowing causality. But are they? In this paper, we compare prediction quality based on \"gold standard\" regulatory edges from previous experimental work with non-linear models inferred from time series data across four different species. We show that the same non-linear machine learning models have better predictive performance, with improvements from 5.3% to 25.3% in terms of the reduction in the root mean square error (RMSE) compared with the same models based on the gold standard edges. Having established that networks fail to characterize causality properly, we suggest that causality research should focus on four goals: (i) predictive accuracy; (ii) a parsimonious enumeration of predictive regulatory genes for each target gene g ; (iii) the identification of disjoint sets of predictive regulatory genes for each target g of roughly equal accuracy; and (iv) the construction of a bipartite network (whose node types are genes and models) representation of causality. We provide algorithms for all goals.',\n",
       "  'Table_S5-Supplementary_Table_S5_Minimal_set_of_TFs_and_disjoint_sets_of_TFs_for_each_of_the_target_gene_in_Human',\n",
       "  'None here'),\n",
       " 'PMC10804933-mmc1.xlsx-Table_S7-Unnamed_68': ('Single-Nuclei RNA Sequencing of 5 Regions of the Human Prenatal Brain Implicates Developing Neuron Populations in Genetic Risk for Schizophrenia',\n",
       "  'Background: While a variety of evidence supports a prenatal component in schizophrenia, there are few data regarding the cell populations involved. We sought to identify cells of the human prenatal brain mediating genetic risk for schizophrenia by integrating cell-specific gene expression measures generated through single-nuclei RNA sequencing with recent large-scale genome-wide association study (GWAS) and exome sequencing data for the condition. Methods: Single-nuclei RNA sequencing was performed on 5 brain regions (frontal cortex, ganglionic eminence, hippocampus, thalamus, and cerebellum) from 3 fetuses from the second trimester of gestation. Enrichment of schizophrenia common variant genetic liability and rare damaging coding variation was assessed in relation to gene expression specificity within each identified cell population. Results: Common risk variants were prominently enriched within genes with high expression specificity for developing neuron populations within the frontal cortex, ganglionic eminence, and hippocampus. Enrichments were largely independent of genes expressed in neuronal populations of the adult brain that have been implicated in schizophrenia through the same methods. Genes containing an excess of rare damaging variants in schizophrenia had higher expression specificity for developing glutamatergic neurons of the frontal cortex and hippocampus that were also enriched for common variant liability. Conclusions: We found evidence for a distinct contribution of prenatal neuronal development to genetic risk for schizophrenia, involving specific populations of developing neurons within the second-trimester fetal brain. Our study significantly advances the understanding of the neurodevelopmental origins of schizophrenia and provides a resource with which to investigate the prenatal antecedents of other psychiatric and neurologic disorders.',\n",
       "  'Table_S7-Unnamed_68',\n",
       "  'None here'),\n",
       " 'PMC9103331-12967_2022_3424_MOESM4_ESM.xlsx-17-gene': ('Single-cell landscape of immunocytes in patients with extrahepatic cholangiocarcinoma',\n",
       "  'Background: The intricate landscape of immunocytes in the tumor microenvironment (TME) is fundamental to immunotherapy but notably under-researched in extrahepatic cholangiocarcinoma (ECCA). Methods: Single-cell RNA sequencing technology was conducted to make an in-depth analysis of immunocytes from matched tumor tissues, paratumor tissues and peripheral blood from ECCA patients. The potential cellular interactions between two cell populations were analyzed with software CellPhoneDB (v2.1.7). Results: We obtained 13526 cells and characterized the transcriptomes and heterogeneity of different clusters and subclusters of immunocytes from ECCA, including CD4+ T cells, CD8+ T cells, B cells and myeloid immunocytes. We observed the rarely described immunocyte subclusters \"intermediate\" exhausted CD8+ T (CD8+ Tex) cells and \"nonclassic\" plasmacytes (CD27+ CD138+ CD38-). In addition, we identified potential immunotherapy targets, for example, ACP5, MAGEH1, TNFRSF9 and CCR8 for Tregs and MT1 for CD8+ Tex cells. We also found strong cellular interactions among Treg cells, M2 macrophages and CD8+ Tex cells through ligand-receptor analysis, implying that potential cellular cross-linkage promoted the immunosuppressive nature of the TME. Conclusions: In a word, our study illuminated the components of the TME and revealed potential cellular interactions at the individual cellular level in ECCA, we aimed to provide a new perspective for further immunological studies and immunotherapy of ECCA.',\n",
       "  '17-gene',\n",
       "  '(Fig. 1B). Significantly differentially expressed genes in each cluster are shown in Additional file **4**  Additional file 4. Table S2. List of signature genes expressed in cell clusters'),\n",
       " 'PMC4425567-ppat.1004897.s007.xlsx-GeneSets-IL1218SG': ('Natural Killer Cell Sensing of Infected Cells Compensates for MyD88 Deficiency but Not IFN-I Activity in Resistance to Mouse Cytomegalovirus',\n",
       "  'In mice, plasmacytoid dendritic cells (pDC) and natural killer (NK) cells both contribute to resistance to systemic infections with herpes viruses including mouse Cytomegalovirus (MCMV). pDCs are the major source of type I IFN (IFN-I) during MCMV infection. This response requires pDC-intrinsic MyD88-dependent signaling by Toll-Like Receptors 7 and 9. Provided that they express appropriate recognition receptors such as Ly49H, NK cells can directly sense and kill MCMV-infected cells. The loss of any one of these responses increases susceptibility to infection. However, the relative importance of these antiviral immune responses and how they are related remain unclear. In humans, while IFN-I responses are essential, MyD88 is dispensable for antiviral immunity. Hence, a higher redundancy has been proposed in the mechanisms promoting protective immune responses against systemic infections by herpes viruses during natural infections in humans. It has been assumed, but not proven, that mice fail to mount protective MyD88-independent IFN-I responses. In humans, the mechanism that compensates MyD88 deficiency has not been elucidated. To address these issues, we compared resistance to MCMV infection and immune responses between mouse strains deficient for MyD88, the IFN-I receptor and/or Ly49H. We show that selective depletion of pDC or genetic deficiencies for MyD88 or TLR9 drastically decreased production of IFN-I, but not the protective antiviral responses. Moreover, MyD88, but not IFN-I receptor, deficiency could largely be compensated by Ly49H-mediated antiviral NK cell responses. Thus, contrary to the current dogma but consistent with the situation in humans, we conclude that, in mice, in our experimental settings, MyD88 is redundant for IFN-I responses and overall defense against a systemic herpes virus infection. Moreover, we identified direct NK cell sensing of infected cells as one mechanism able to compensate for MyD88 deficiency in mice. Similar mechanisms likely contribute to protect MyD88- or IRAK4-deficient patients from viral infections.',\n",
       "  'GeneSets-IL1218SG',\n",
       "  'Microarray analyses were performed as previously described [23]. Gene set compositions are given in **S1 Table**  GeneSets used in bioinformatics analyses of microarray data.(XLSX)  Click here for additional data file'),\n",
       " 'PMC6121016-Table_1.XLS-TableS1_txt-Unnamed_13': ('DECtp: Calling Differential Gene Expression Between Cancer and Normal Samples by Integrating Tumor Purity Information',\n",
       "  'Identifying differentially expressed genes (DEGs) between tumor and normal samples is critical for studying tumorigenesis, and has been routinely applied to identify diagnostic, prognostic, and therapeutic biomarkers for many cancers. It is well-known that solid tumor tissue samples obtained from clinical settings are always mixtures of cancer and normal cells. However, the tumor purity information is more or less ignored in traditional differential expression analyses, which might decrease the power of differential gene identification or even bias the results. In this paper, we have developed a novel differential gene calling method called DECtp by integrating tumor purity information into a generalized least square procedure, followed by the Wald test. We compared DECtp with popular methods like t -test and limma on nine simulation datasets with different sample sizes and noise levels. DECtp achieved the highest area under curves (AUCs) for all the comparisons, suggesting that cancer purity information is critical for DEG calling between tumor and normal samples. In addition, we applied DECtp into cancer and normal samples of 14 tumor types collected from The Cancer Genome Atlas (TCGA) and compared the DEGs with those called by limma. As a result, DECtp achieved more sensitive, consistent, and biologically meaningful results and identified a few novel DEGs for further experimental validation.',\n",
       "  'TableS1_txt-Unnamed_13',\n",
       "  'genes with the largest correlations for each cancer type were shown in Supplementary Table **S1**  Click here for additional data file'),\n",
       " 'PMC4417536-12943_2015_358_MOESM2_ESM.xls-Downregulated-id': ('Next-generation sequencing reveals novel differentially regulated mRNAs, lncRNAs, miRNAs, sdRNAs and a piRNA in pancreatic cancer',\n",
       "  \"Background: Previous studies identified microRNAs (miRNAs) and messenger RNAs with significantly different expression between normal pancreas and pancreatic cancer (PDAC) tissues. Due to technological limitations of microarrays and real-time PCR systems these studies focused on a fixed set of targets. Expression of other RNA classes such as long intergenic non-coding RNAs or sno-derived RNAs has rarely been examined in pancreatic cancer. Here, we analysed the coding and non-coding transcriptome of six PDAC and five control tissues using next-generation sequencing. Results: Besides the confirmation of several deregulated mRNAs and miRNAs, miRNAs without previous implication in PDAC were detected: miR-802, miR-2114 or miR-561. SnoRNA-derived RNAs (e.g. sno-HBII-296B) and piR-017061, a piwi-interacting RNA, were found to be differentially expressed between PDAC and control tissues. In silico target analysis of miR-802 revealed potential binding sites in the 3' UTR of TCF4, encoding a transcription factor that controls Wnt signalling genes. Overexpression of miR-802 in MiaPaCa pancreatic cancer cells reduced TCF4 protein levels. Using Massive Analysis of cDNA Ends (MACE) we identified differential expression of 43 lincRNAs, long intergenic non-coding RNAs, e.g. LINC00261 and LINC00152 as well as several natural antisense transcripts like HNF1A-AS1 and AFAP1-AS1. Differential expression was confirmed by qPCR on the mRNA/miRNA/lincRNA level and by immunohistochemistry on the protein level. Conclusions: Here, we report a novel lncRNA, sncRNA and mRNA signature of PDAC. In silico prediction of ncRNA targets allowed for assigning potential functions to differentially regulated RNAs.\",\n",
       "  'Downregulated-id',\n",
       "  'None here'),\n",
       " 'PMC10644098-BLOOD_BLD-2023-019721-mmc16.xlsx-all_clusterImmature_pat_ctr_Wil-Unnamed_0': ('IL-7 receptor signaling drives human B-cell progenitor differentiation and expansion',\n",
       "  'Although absence of interleukin-7 (IL-7) signaling completely abrogates T and B lymphopoiesis in mice, patients with severe combined immunodeficiency caused by mutations in the IL-7 receptor α chain (IL-7Rα) still generate peripheral blood B cells. Consequently, human B lymphopoiesis has been thought to be independent of IL-7 signaling. Using flow cytometric analysis and single-cell RNA sequencing of bone marrow samples from healthy controls and patients who are IL-7Rα deficient, in combination with in vitro modeling of human B-cell differentiation, we demonstrate that IL-7R signaling plays a crucial role in human B lymphopoiesis. IL-7 drives proliferation and expansion of early B-cell progenitors but not of pre-BII large cells and has a limited role in the prevention of cell death. Furthermore, IL-7 guides cell fate decisions by enhancing the expression of BACH2, EBF1, and PAX5, which jointly orchestrate the specification and commitment of early B-cell progenitors. In line with this observation, early B-cell progenitors of patients with IL-7Rα deficiency still expressed myeloid-specific genes. Collectively, our results unveil a previously unknown role for IL-7 signaling in promoting the B-lymphoid fate and expanding early human B-cell progenitors while defining important differences between mice and humans. Our results have implications for hematopoietic stem cell transplantation strategies in patients with T- B+ severe combined immunodeficiency and provide insights into the role of IL-7R signaling in leukemogenesis.',\n",
       "  'all_clusterImmature_pat_ctr_Wil-Unnamed_0',\n",
       "  'Supplemental Table 15'),\n",
       " 'PMC7929494-cm9-134-584-s004.xlsx-Downregulated_DE-mRNAs_1427-Downregulated_DE-mRNAs_1427': ('Construction of potential idiopathic pulmonary fibrosis related microRNA and messenger RNA regulatory network',\n",
       "  'Abstract not found',\n",
       "  'Downregulated_DE-mRNAs_1427-Downregulated_DE-mRNAs_1427',\n",
       "  'Supplemental Digital Content'),\n",
       " 'PMC11218163-13014_2024_2480_MOESM1_ESM.xlsx-DEG-Figure_5_D': ('Transcriptomic response of prostate cancer cells to carbon ion and photon irradiation with focus on androgen receptor and TP53 signaling',\n",
       "  'Background: Radiotherapy is essential in the treatment of prostate cancer. An alternative to conventional photon radiotherapy is the application of carbon ions, which provide a superior intratumoral dose distribution and less induced damage to adjacent healthy tissue. A common characteristic of prostate cancer cells is their dependence on androgens which is exploited therapeutically by androgen deprivation therapy in the advanced prostate cancer stage. Here, we aimed to analyze the transcriptomic response of prostate cancer cells to irradiation by photons in comparison to carbon ions, focusing on DNA damage, DNA repair and androgen receptor signaling. Methods: Prostate cancer cell lines LNCaP (functional TP53 and androgen receptor signaling) and DU145 (dysfunctional TP53 and androgen receptor signaling) were irradiated by photons or carbon ions and the subsequent DNA damage was assessed by immuno-cytofluorescence. Furthermore, the cells were treated with an androgen-receptor agonist. The effects of irradiation and androgen treatment on the gene regulation and the transcriptome were investigated by RT-qPCR and RNA sequencing, followed by bioinformatic analysis. Results: Following photon or carbon ion irradiation, both LNCaP and DU145 cells showed a dose-dependent amount of visible DNA damage that decreased over time, indicating occurring DNA repair. In terms of gene regulation, mRNAs involved in the TP53-dependent DNA damage response were significantly upregulated by photons and carbon ions in LNCaP but not in DU145 cells, which generally showed low levels of gene regulation after irradiation. Both LNCaP and DU145 cells responded to photons and carbon ions by downregulation of genes involved in DNA repair and cell cycle, partially resembling the transcriptome response to the applied androgen receptor agonist. Neither photons nor carbon ions significantly affected canonical androgen receptor-dependent gene regulation. Furthermore, certain genes that were specifically regulated by either photon or carbon ion irradiation were identified. Conclusion: Photon and carbon ion irradiation showed a significant congruence in terms of induced signaling pathways and transcriptomic responses. These responses were strongly impacted by the TP53 status. Nevertheless, irradiation mode-dependent distinct gene regulations with undefined implication for radiotherapy outcome were revealed. Androgen receptor signaling and irradiations shared regulation of certain genes with respect to DNA-repair and cell-cycle.',\n",
       "  'DEG-Figure_5_D',\n",
       "  'None here'),\n",
       " 'PMC7786291-Table_1.xlsx-Table_S1-Supplementary_table_1_ETV7-related_genes_in_the_TCGA_SKCM_data_set': ('Integrated Analysis of the ETS Family in Melanoma Reveals a Regulatory Role of ETV7 in the Immune Microenvironment',\n",
       "  'The ETS family modulates immune response and drug efficiency to targeted therapies, but their role in melanoma is largely unclear. In this study, the ETS family was systematically analyzed in multiple public data sets. Bioinformatics tools were used to characterize the function of ETV7 in melanoma. A prognostic model was constructed using the LASSO Cox regression method. We found that ETV7 was the only differentially expressed gene with significant prognostic relevance in melanoma. Enrichment analysis of seven independent data sets indicated ETV7 participation in various immune-related pathways. ETV7 particularly showed a strong positive correlation with CD8+ T cell infiltration. The prognostic model based on ETV7 and its hub genes showed a relatively good predictive value in training and testing data sets. Thus, ETV7 can potentially regulate the immune microenvironment in melanoma.',\n",
       "  'Table_S1-Supplementary_table_1_ETV7-related_genes_in_the_TCGA_SKCM_data_set',\n",
       "  'melanoma. A total of 623 ETV7-related genes were found in the TCGA_SKCM data set (**Supplementary Table 1**)  Click here for additional data file'),\n",
       " 'PMC6028641-41467_2018_4985_MOESM4_ESM.xlsx-gene_lists-Unnamed_16': ('Human in vivo-generated monocyte-derived dendritic cells and macrophages cross-present antigens through a vacuolar pathway',\n",
       "  'Presentation of exogenous antigens on MHC-I molecules, termed cross-presentation, is essential for cytotoxic CD8 + T cell responses. In mice, dendritic cells (DCs) that arise from monocytes (mo-DCs) during inflammation have a key function in these responses by cross-presenting antigens locally in peripheral tissues. Whether human naturally-occurring mo-DCs can cross-present is unknown. Here, we use human mo-DCs and macrophages directly purified from ascites to address this question. Single-cell RNA-seq data show that ascites CD1c + DCs contain exclusively monocyte-derived cells. Both ascites mo-DCs and monocyte-derived macrophages cross-present efficiently, but are inefficient for transferring exogenous proteins into their cytosol. Inhibition of cysteine proteases, but not of proteasome, abolishes cross-presentation in these cells. We conclude that human monocyte-derived cells cross-present exclusively using a vacuolar pathway. Finally, only ascites mo-DCs provide co-stimulatory signals to induce effector cytotoxic CD8 + T cells. Our findings thus provide important insights on how to harness cross-presentation for therapeutic purposes.',\n",
       "  'gene_lists-Unnamed_16',\n",
       "  'the average expression of each signature, substracted by the aggregated expression of control gene sets**1**  Supplementary Data 1'),\n",
       " 'PMC4071233-gm560-S6.xlsx-Sheet1-Axis_5_456_Probes_401_Genes': ('Gene expression profiles associated with acute myocardial infarction and risk of cardiovascular death',\n",
       "  'Background: Genetic risk scores have been developed for coronary artery disease and atherosclerosis, but are not predictive of adverse cardiovascular events. We asked whether peripheral blood expression profiles may be predictive of acute myocardial infarction (AMI) and/or cardiovascular death. Methods: Peripheral blood samples from 338 subjects aged 62 ± 11 years with coronary artery disease (CAD) were analyzed in two phases (discovery N = 175, and replication N = 163), and followed for a mean 2.4 years for cardiovascular death. Gene expression was measured on Illumina HT-12 microarrays with two different normalization procedures to control technical and biological covariates. Whole genome genotyping was used to support comparative genome-wide association studies of gene expression. Analysis of variance was combined with receiver operating curve and survival analysis to define a transcriptional signature of cardiovascular death. Results: In both phases, there was significant differential expression between healthy and AMI groups with overall down-regulation of genes involved in T-lymphocyte signaling and up-regulation of inflammatory genes. Expression quantitative trait loci analysis provided evidence for altered local genetic regulation of transcript abundance in AMI samples. On follow-up there were 31 cardiovascular deaths. A principal component (PC1) score capturing covariance of 238 genes that were differentially expressed between deceased and survivors in the discovery phase significantly predicted risk of cardiovascular death in the replication and combined samples (hazard ratio = 8.5, P < 0.0001) and improved the C-statistic (area under the curve 0.82 to 0.91, P = 0.03) after adjustment for traditional covariates. Conclusions: A specific blood gene expression profile is associated with a significant risk of death in Caucasian subjects with CAD. This comprises a subset of transcripts that are also altered in expression during acute myocardial infarction.',\n",
       "  'Sheet1-Axis_5_456_Probes_401_Genes',\n",
       "  'into a linear model absorbs the association of the axes with AMI (Additional file **6**)  Additional file 6: Table S4Association of white blood cells with axes  Click here for file'),\n",
       " 'PMC9232268-DataSheet_1.docx-0-Query': ('Pan-Cancer Integrated Analysis Identification of SASH3, a Potential Biomarker That Inhibits Lung Adenocarcinoma Progression',\n",
       "  'Sterile alpha motif (SAM) and Src homology-3 (SH3) domain-containing 3 (SASH3) is an adaptor protein expressed mainly in lymphocytes, and plays significant roles in T-cell proliferation and cell survival. However, its expression level, clinical significance, and correlation with tumor-infiltrating immune cells across cancers remain unclear. In this study, we comprehensively examined the expression, dysregulation, and prognostic significance of SASH3, and the correlation with clinicopathological parameters and immune infiltration in pan-cancer. The mRNA and protein expression status of SASH3 were determined by TCGA, GTEx, and UALCAN. Kaplan-Meier analysis utilized the prognostic values of SASH3 in diverse cancers. The association between SASH3 expression and gene mutation, DNA methylation, immune cells infiltration, immune checkpoints, tumor mutation burden (TMB), and microsatellite instability (MSI) were analyzed using data from the TCGA database. High expression of SASH3 was not only linked to poor OS in ESCC, LAML, LGG, and UVM, but also associated with better OS in CESC, HNSC, LUAD, SARC, SKCM, THYM, and UCEC. As for DSS, a high level of SASH3 correlated with adverse DSS in ESCC, LGG, and UVM, and lowly expressed SASH3 was associated with shorter OS in CESC, HNSC, LUAD, SARC, SKCM, and UCEC. The results of Cox regression and nomogram analyses confirmed that SASH3 was an independent factor for LUAD prognosis. Gene Ontology (GO), the Kyoto Encyclopedia of Genes and Genomes (KEGG), and gene set enrichment analysis (GSEA) results showed that SASH3 was involved in natural killer cell-mediated cytotoxicity, Th17 cell differentiation, PD-L1 expression and PD-1 checkpoint pathway in cancer, NF-kappa B signaling pathway, B-cell receptor signaling pathway, and Toll-like receptor signaling pathway. SASH3 expression was correlated with TMB in 28 cancer types and associated with MSI in 22 cancer types, while there was a negative correlation between SASH3 expression and DNA methylation in diverse human cancer. The high DNA methylation level of SASH3 was correlated with better OS in KIRC and UVM, and associated with poor OS in SKCM. Moreover, we uncover that SASH3 expression was positively associated with the stroma score in 27 cancer types, the microenvironment score, and immune score in 32 cancer types, 38 types of immune cells in 32 cancer types, the 45 immune stimulators, 24 immune inhibitors, 41 chemokines, 18 receptors, and 21 major histocompatibility complex (MHC) molecules in 33 cancer types. Finally, forced SASH3 expression inhibited lung adenocarcinoma (LUAD) cell proliferation and cell migration. Our findings confirmed that SASH3 may be a biomarker for the prognosis and diagnosis of human cancer.',\n",
       "  '0-Query',\n",
       "  'SASH3 was correlated with better OS, and associated with poor OS in SKCM ( **Supplementary Figure 1**)  Click here for additional data file'),\n",
       " 'PMC10835465-mmc6.xlsx-Cluster12-gene': ('Association of HBsAg levels with differential gene expression in NK, CD8 T, and memory B cells in treated patients with chronic HBV',\n",
       "  'Background & aims: HBsAg secretion may impact immune responses to chronic HBV infection. Thus, therapeutic approaches to suppress HBsAg production are being investigated. Our study aims to examine the immunomodulatory effects of high and low levels of circulating HBsAg and thereby improve our understanding of anti-HBV immunity. Methods: An optimized 10x Genomics single-cell RNA sequencing workflow was applied to blood samples and liver fine-needle aspirates from 18 patients undergoing tenofovir/entecavir (NUC) treatment for chronic HBV infection. They were categorized based on their HBsAg levels: high (920-12,447 IU/ml) or low (1-100 IU/ml). Cluster frequencies, differential gene expression, and phenotypes were analyzed. Results: In the blood of HBV-infected patients on NUC, the proportion of KLRC2 + \"adaptive\" natural killer (NK) cells was significantly lower in the HBsAg-high group and, remarkably, both KLRC2 + NK and KLRG1+ CD8 T cells display enrichment of lymphocyte activation-associated gene sets in the HBsAg-low group. High levels of HBsAg were associated with mild immune activation in the liver. However, no suppression of liver-resident CXCR6+ NCAM1+ NK or CXCR6+ CD69+ CD8 T cells was detected, while memory B cells showed signs of activation in both the blood and liver. Conclusions: Among NUC-treated patients, we observed a minimal impact of HBsAg on leukocyte populations in the blood and liver. However, for the first time, we found that HBsAg has distinct effects, restricted to NK-, CD8 T-, and memory B-cell subsets, in the blood and liver. Our findings are highly relevant for current clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Impact and implications: This study provides unique insight into the impact of HBsAg on gene expression levels of immune cell subsets in the blood and liver, particularly in the context of NUC-treated chronic HBV infection. It holds significant relevance for current and future clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Our findings raise questions about the effectiveness of such treatment strategies and challenge the previously hypothesized immunomodulatory effects of HBsAg on immune responses against HBV.',\n",
       "  'Cluster12-gene',\n",
       "  'None here'),\n",
       " 'PMC6338385-ppat.1007505.s008.xlsx-Sheet1-Name': ('Human polyomavirus BKV infection of endothelial cells results in interferon pathway induction and persistence',\n",
       "  'Polyomavirus BKV is highly prevalent among humans. The virus establishes an asymptomatic persistent infection in the urinary system in healthy people, but uncontrolled productive infection of the virus in immunocompromised patients can lead to serious diseases. In spite of its high prevalence, our knowledge regarding key aspects of BKV polyomavirus infection remains incomplete. To determine tissue and cell type tropism of the virus, primary human epithelial cells, endothelial cells and fibroblasts isolated from the respiratory and urinary systems were tested. Results from this study demonstrated that all 9 different types of human cells were infectable by BKV polyomavirus but showed differential cellular responses. In microvascular endothelial cells from the lung and the bladder, BKV persistent infection led to prolonged viral protein expression, low yield of infectious progeny and delayed cell death, in contrast with infection in renal proximal tubular epithelial cells, a widely used cell culture model for studying productive infection of this virus. Transcriptomic profiling revealed the activation of interferon signaling and induction of multiple interferon stimulated genes in infected microvascular endothelial cells. Further investigation demonstrated production of IFNβ and secretion of chemokine CXCL10 by infected endothelial cells. Activation of IRF3 and STAT1 in infected endothelial cells was also confirmed. In contrast, renal proximal tubular epithelial cells failed to mount an interferon response and underwent progressive cell death. These results demonstrated that microvascular endothelial cells are able to activate interferon signaling in response to polyomavirus BKV infection. This raises the possibility that endothelial cells might provide initial immune defense against BKV infection. Our results shed light on the persistence of and immunity against infection by BKV polyomavirus.',\n",
       "  'Sheet1-Name',\n",
       "  'The complete lists of upregulated genes with corresponding log ratios can be found in **S2**  Complete list of upregulated genes in RPTE1 RNAseq with corresponding log ratios.(XLSX)  Click here for additional data file'),\n",
       " 'PMC6501285-13075_2019_1894_MOESM3_ESM.xlsx-UPREGULATED-HGNC_ID': ('RNA sequencing of mesenchymal stem cells reveals a blocking of differentiation and immunomodulatory activities under inflammatory conditions in rheumatoid arthritis patients',\n",
       "  'Introduction: Mesenchymal stem cells (MSCs) have the ability to differentiate into different types of cells of the mesenchymal lineage, such as osteocytes, chondrocytes, and adipocytes. It is also known that under inflammatory stimuli or in the appropriate experimental conditions, they can also act as regulators of inflammation. Thus, in addition to their regenerating potential, their interest has been extended to their possible use in cell therapy strategies for treatment of immune disorders. Objective: To analyze, by RNA-seq analysis, the transcriptome profiling of allogenic MSCs under RA lymphocyte activation. Methods: We identified the differentially expressed genes in bone marrow mesenchymal stem cells after exposure to an inflammatory environment. The transcriptome profiling was evaluated by means of the precise measurement of transcripts provided by the RNA-Seq technology. Results: Our results evidenced the existence of blocking of both regenerative (differentiation) and immunomodulatory phenotypes under inflammatory conditions characterized by an upregulation of genes involved in immune processes and a simultaneous downregulation of genes mainly involved in regenerative or cell differentiation functions. Conclusions: We conclude that the two main functions of MSCs (immunomodulation and differentiation) are blocked, at least while the inflammation is being resolved. Inflammation, at least partially mediated by gamma-interferon, drives MSCs to a cellular distress adopting a defensive state. This knowledge could be of particular interest in cases where the damage to be repaired has an important immune-mediated component.',\n",
       "  'UPREGULATED-HGNC_ID',\n",
       "  'PBMCs and exposed to resting RA PBMCs, we observed 847 DEGs in total, with 2**3**  Table S3. Number of reads successfully mapped and assigned to the metafeature “gene” per sample. (XLSX 3343 kb)'),\n",
       " 'PMC8440811-Table_1.XLS-downregulated_genes-Name': ('Comprehensive Analysis of a Nine-Gene Signature Related to Tumor Microenvironment in Lung Adenocarcinoma',\n",
       "  'Lung adenocarcinoma (LUAD) is the most common malignancy, leading to more than 1 million related deaths each year. Due to low long-term survival rates, the exploration of molecular mechanisms underlying LUAD progression and novel prognostic predictors is urgently needed to improve LUAD treatment. In our study, cancer-specific differentially expressed genes (DEGs) were identified using the robust rank aggregation (RRA) method between tumor and normal tissues from six Gene Expression Omnibus databases (GSE43458, GSE62949, GSE68465, GSE115002, GSE116959, and GSE118370), followed by a selection of prognostic modules using weighted gene co-expression network analysis. Univariate Cox regression, least absolute shrinkage and selection operator (LASSO), and multivariate Cox regression analyses were applied to identify nine hub genes ( CBFA2T3, CR2, SEL1L3, TM6SF1, TSPAN32, ITGA6, MAPK11, RASA3, and TLR6 ) that constructed a prognostic risk model. The RNA expressions of nine hub genes were validated in tumor and normal tissues by RNA-sequencing and single-cell RNA-sequencing, while immunohistochemistry staining from the Human Protein Atlas database showed consistent results in the protein levels. The risk model revealed that high-risk patients were associated with poor prognoses, including advanced stages and low survival rates. Furthermore, a multivariate Cox regression analysis suggested that the prognostic risk model could be an independent prognostic factor for LUAD patients. A nomogram that incorporated the signature and clinical features was additionally built for prognostic prediction. Moreover, the levels of hub genes were related to immune cell infiltration in LUAD microenvironments. A CMap analysis identified 13 small molecule drugs as potential agents based on the risk model for LUAD treatment. Thus, we identified a prognostic risk model including CBFA2T3, CR2, SEL1L3, TM6SF1, TSPAN32, ITGA6, MAPK11, RASA3, and TLR6 as novel biomarkers and validated their prognostic and predicted values for LUAD.',\n",
       "  'downregulated_genes-Name',\n",
       "  'value < 0.05, 1,219 significant DEGs, including 496 upregulated and 723 downregulated, were identified (**Supplementary File 1**)  Click here for additional data file'),\n",
       " 'PMC7396846-EMBJ-39-e103790-s003.xlsx-Table_EV1-Unnamed_1': ('Microglia promote glioblastoma via mTOR-mediated immunosuppression of the tumour microenvironment',\n",
       "  'Tumour-associated microglia/macrophages (TAM) are the most numerous non-neoplastic populations in the tumour microenvironment in glioblastoma multiforme (GBM), the most common malignant brain tumour in adulthood. The mTOR pathway, an important regulator of cell survival/proliferation, is upregulated in GBM, but little is known about the potential role of this pathway in TAM. Here, we show that GBM-initiating cells induce mTOR signalling in the microglia but not bone marrow-derived macrophages in both in vitro and in vivo GBM mouse models. mTOR-dependent regulation of STAT3 and NF-κB activity promotes an immunosuppressive microglial phenotype. This hinders effector T-cell infiltration, proliferation and immune reactivity, thereby contributing to tumour immune evasion and promoting tumour growth in mouse models. The translational value of our results is demonstrated in whole transcriptome datasets of human GBM and in a novel in vitro model, whereby expanded-potential stem cells (EPSC)-derived microglia-like cells are conditioned by syngeneic patient-derived GBM-initiating cells. These results raise the possibility that microglia could be the primary target of mTOR inhibition, rather than the intrinsic tumour cells in GBM.',\n",
       "  'Table_EV1-Unnamed_1',\n",
       "  'the enrichment score of specific immune pathways identified in bulk tumour tissue (Fig 5A, **Table EV1**)  Table EV1  Click here for additional data file'),\n",
       " 'PMC4617899-12864_2015_2012_MOESM2_ESM.xlsx-pDC_SIG_minmax1_5x-Unnamed_1': ('BubbleGUM: automatic extraction of phenotype molecular signatures and comprehensive visualization of multiple Gene Set Enrichment Analyses',\n",
       "  \"Background: Recent advances in the analysis of high-throughput expression data have led to the development of tools that scaled-up their focus from single-gene to gene set level. For example, the popular Gene Set Enrichment Analysis (GSEA) algorithm can detect moderate but coordinated expression changes of groups of presumably related genes between pairs of experimental conditions. This considerably improves extraction of information from high-throughput gene expression data. However, although many gene sets covering a large panel of biological fields are available in public databases, the ability to generate home-made gene sets relevant to one's biological question is crucial but remains a substantial challenge to most biologists lacking statistic or bioinformatic expertise. This is all the more the case when attempting to define a gene set specific of one condition compared to many other ones. Thus, there is a crucial need for an easy-to-use software for generation of relevant home-made gene sets from complex datasets, their use in GSEA, and the correction of the results when applied to multiple comparisons of many experimental conditions. Result: We developed BubbleGUM (GSEA Unlimited Map), a tool that allows to automatically extract molecular signatures from transcriptomic data and perform exhaustive GSEA with multiple testing correction. One original feature of BubbleGUM notably resides in its capacity to integrate and compare numerous GSEA results into an easy-to-grasp graphical representation. We applied our method to generate transcriptomic fingerprints for murine cell types and to assess their enrichments in human cell types. This analysis allowed us to confirm homologies between mouse and human immunocytes. Conclusions: BubbleGUM is an open-source software that allows to automatically generate molecular signatures out of complex expression datasets and to assess directly their enrichment by GSEA on independent datasets. Enrichments are displayed in a graphical output that helps interpreting the results. This innovative methodology has recently been used to answer important questions in functional genomics, such as the degree of similarities between microarray datasets from different laboratories or with different experimental models or clinical cohorts. BubbleGUM is executable through an intuitive interface so that both bioinformaticians and biologists can use it. It is available at http://www.ciml.univ-mrs.fr/applications/BubbleGUM/index.html .\",\n",
       "  'pDC_SIG_minmax1_5x-Unnamed_1',\n",
       "  'None here'),\n",
       " 'PMC6338385-ppat.1007505.s008.xlsx-Sheet1-ENSEMBL': ('Human polyomavirus BKV infection of endothelial cells results in interferon pathway induction and persistence',\n",
       "  'Polyomavirus BKV is highly prevalent among humans. The virus establishes an asymptomatic persistent infection in the urinary system in healthy people, but uncontrolled productive infection of the virus in immunocompromised patients can lead to serious diseases. In spite of its high prevalence, our knowledge regarding key aspects of BKV polyomavirus infection remains incomplete. To determine tissue and cell type tropism of the virus, primary human epithelial cells, endothelial cells and fibroblasts isolated from the respiratory and urinary systems were tested. Results from this study demonstrated that all 9 different types of human cells were infectable by BKV polyomavirus but showed differential cellular responses. In microvascular endothelial cells from the lung and the bladder, BKV persistent infection led to prolonged viral protein expression, low yield of infectious progeny and delayed cell death, in contrast with infection in renal proximal tubular epithelial cells, a widely used cell culture model for studying productive infection of this virus. Transcriptomic profiling revealed the activation of interferon signaling and induction of multiple interferon stimulated genes in infected microvascular endothelial cells. Further investigation demonstrated production of IFNβ and secretion of chemokine CXCL10 by infected endothelial cells. Activation of IRF3 and STAT1 in infected endothelial cells was also confirmed. In contrast, renal proximal tubular epithelial cells failed to mount an interferon response and underwent progressive cell death. These results demonstrated that microvascular endothelial cells are able to activate interferon signaling in response to polyomavirus BKV infection. This raises the possibility that endothelial cells might provide initial immune defense against BKV infection. Our results shed light on the persistence of and immunity against infection by BKV polyomavirus.',\n",
       "  'Sheet1-ENSEMBL',\n",
       "  'The complete lists of upregulated genes with corresponding log ratios can be found in **S2**  Complete list of upregulated genes in RPTE1 RNAseq with corresponding log ratios.(XLSX)  Click here for additional data file'),\n",
       " 'PMC6501285-13075_2019_1894_MOESM3_ESM.xlsx-UPREGULATED-Approved_symbol': ('RNA sequencing of mesenchymal stem cells reveals a blocking of differentiation and immunomodulatory activities under inflammatory conditions in rheumatoid arthritis patients',\n",
       "  'Introduction: Mesenchymal stem cells (MSCs) have the ability to differentiate into different types of cells of the mesenchymal lineage, such as osteocytes, chondrocytes, and adipocytes. It is also known that under inflammatory stimuli or in the appropriate experimental conditions, they can also act as regulators of inflammation. Thus, in addition to their regenerating potential, their interest has been extended to their possible use in cell therapy strategies for treatment of immune disorders. Objective: To analyze, by RNA-seq analysis, the transcriptome profiling of allogenic MSCs under RA lymphocyte activation. Methods: We identified the differentially expressed genes in bone marrow mesenchymal stem cells after exposure to an inflammatory environment. The transcriptome profiling was evaluated by means of the precise measurement of transcripts provided by the RNA-Seq technology. Results: Our results evidenced the existence of blocking of both regenerative (differentiation) and immunomodulatory phenotypes under inflammatory conditions characterized by an upregulation of genes involved in immune processes and a simultaneous downregulation of genes mainly involved in regenerative or cell differentiation functions. Conclusions: We conclude that the two main functions of MSCs (immunomodulation and differentiation) are blocked, at least while the inflammation is being resolved. Inflammation, at least partially mediated by gamma-interferon, drives MSCs to a cellular distress adopting a defensive state. This knowledge could be of particular interest in cases where the damage to be repaired has an important immune-mediated component.',\n",
       "  'UPREGULATED-Approved_symbol',\n",
       "  'PBMCs and exposed to resting RA PBMCs, we observed 847 DEGs in total, with 2**3**  Table S3. Number of reads successfully mapped and assigned to the metafeature “gene” per sample. (XLSX 3343 kb)'),\n",
       " 'PMC9780220-41598_2022_26217_MOESM2_ESM.xlsx-DEgenes_CTL-EV_FGF2-EV-Unnamed_1': ('Fibroblast growth factor-2 bound to specific dermal fibroblast-derived extracellular vesicles is protected from degradation',\n",
       "  'Fibroblast growth factor-2 (FGF2) has multiple roles in cutaneous wound healing but its natural low stability prevents the development of its use in skin repair therapies. Here we show that FGF2 binds the outer surface of dermal fibroblast (DF)-derived extracellular vesicles (EVs) and this association protects FGF2 from fast degradation. EVs isolated from DF cultured in the presence of FGF2 harbor FGF2 on their surface and FGF2 can bind purified EVs in absence of cells. Remarkably, FGF2 binding to EVs is restricted to a specific subpopulation of EVs, which do not express CD63 and CD81 markers. Treatment of DF with FGF2-EVs activated ERK and STAT signaling pathways and increased cell proliferation and migration. Local injection of FGF2-EVs improved wound healing in mice. We further demonstrated that binding to EVs protects FGF2 from both thermal and proteolytic degradation, thus maintaining FGF2 function. This suggests that EVs protect soluble factors from degradation and increase their stability and half-life. These results reveal a novel aspect of EV function and suggest EVs as a potential tool for delivering FGF2 in skin healing therapies.',\n",
       "  'DEgenes_CTL-EV_FGF2-EV-Unnamed_1',\n",
       "  'To investigate the effect of FGF**2**  Supplementary Information 2'),\n",
       " 'PMC5784090-41598_2018_19837_MOESM2_ESM.xlsx-Table_S4-Unnamed_1': ('Persistent activation of interlinked type 2 airway epithelial gene networks in sputum-derived cells from aeroallergen-sensitized symptomatic asthmatics',\n",
       "  'Atopic asthma is a persistent disease characterized by intermittent wheeze and progressive loss of lung function. The disease is thought to be driven primarily by chronic aeroallergen-induced type 2-associated inflammation. However, the vast majority of atopics do not develop asthma despite ongoing aeroallergen exposure, suggesting additional mechanisms operate in conjunction with type 2 immunity to drive asthma pathogenesis. We employed RNA-Seq profiling of sputum-derived cells to identify gene networks operative at baseline in house dust mite-sensitized (HDM S ) subjects with/without wheezing history that are characteristic of the ongoing asthmatic state. The expression of type 2 effectors (IL-5, IL-13) was equivalent in both cohorts of subjects. However, in HDM S -wheezers they were associated with upregulation of two coexpression modules comprising multiple type 2- and epithelial-associated genes. The first module was interlinked by the hubs EGFR, ERBB2, CDH1 and IL-13. The second module was associated with CDHR3 and mucociliary clearance genes. Our findings provide new insight into the molecular mechanisms operative at baseline in the airway mucosa in atopic asthmatics undergoing natural aeroallergen exposure, and suggest that susceptibility to asthma amongst these subjects involves complex interactions between type 2- and epithelial-associated gene networks, which are not operative in equivalently sensitized/exposed atopic non-asthmatics.',\n",
       "  'Table_S4-Unnamed_1',\n",
       "  'Supplementary Tables S1–11'),\n",
       " 'PMC9034549-13046_2022_2344_MOESM1_ESM.xlsx-sheet1-Ensemnl_ids': ('Genome-wide CRISPR screen identified Rad18 as a determinant of doxorubicin sensitivity in osteosarcoma',\n",
       "  'Background: Osteosarcoma (OS) is a malignant bone tumor mostly occurring in children and adolescents, while chemotherapy resistance often develops and the mechanisms involved remain challenging to be fully investigated. Methods: Genome-wide CRISPR screening combined with transcriptomic sequencing were used to identify the critical genes of doxorubicin resistance. Analysis of clinical samples and datasets, and in vitro and in vivo experiments (including CCK-8, apoptosis, western blot, qRT-PCR and mouse models) were applied to confirm the function of these genes. The bioinformatics and IP-MS assays were utilized to further verify the downstream pathway. RGD peptide-directed and exosome-delivered siRNA were developed for the novel therapy strategy. Results: We identified that E3 ubiquitin-protein ligase Rad18 (Rad18) contributed to doxorubicin-resistance in OS. Further exploration revealed that Rad18 interact with meiotic recombination 11 (MRE11) to promote the formation of the MRE11-RAD50-NBS1 (MRN) complex, facilitating the activation of the homologous recombination (HR) pathway, which ultimately mediated DNA damage tolerance and leaded to a poor prognosis and chemotherapy response in patients with OS. Rad18-knockout effectively restored the chemotherapy response in vitro and in vivo. Also, RGD-exosome loading chemically modified siRad18 combined with doxorubicin, where exosome and chemical modification guaranteed the stability of siRad18 and the RGD peptide provided prominent targetability, had significantly improved antitumor activity of doxorubicin. Conclusions: Collectively, our study identifies Rad18 as a driver of OS doxorubicin resistance that promotes the HR pathway and indicates that targeting Rad18 is an effective approach to overcome chemotherapy resistance in OS.',\n",
       "  'sheet1-Ensemnl_ids',\n",
       "  'by Tsingke Biotechnology Co., Ltd. (Beijing, China), and sequences of siRNAs were shown in **supplementary materials**  Additional file 1'),\n",
       " 'PMC8321697-mmc3.xlsx-cytokine-regulated_genes-Unnamed_7': ('JAK inhibitors dampen activation of interferon-activated transcriptomes and the SARS-CoV-2 receptor ACE2 in human renal proximal tubules',\n",
       "  'SARS-CoV-2 infections initiate cytokine storms and activate genetic programs leading to progressive hyperinflammation in multiple organs of patients with COVID-19. While it is known that COVID-19 impacts kidney function, leading to increased mortality, cytokine response of renal epithelium has not been studied in detail. Here, we report on the genetic programs activated in human primary proximal tubule (HPPT) cells by interferons and their suppression by ruxolitinib, a Janus kinase (JAK) inhibitor used in COVID-19 treatment. Integration of our data with those from patients with acute kidney injury and COVID-19, as well as other tissues, permitted the identification of kidney-specific interferon responses. Additionally, we investigated the regulation of the recently discovered isoform (dACE2) of the angiotensin-converting enzyme 2 (ACE2), the SARS-CoV-2 receptor. Using ChIP-seq, we identified candidate interferon-activated enhancers controlling the ACE2 locus, including the intronic dACE2 promoter. Taken together, our study provides an in-depth understanding of genetic programs activated in kidney cells.',\n",
       "  'cytokine-regulated_genes-Unnamed_7',\n",
       "  'IFNα, 1169 by IFNβ, 1280 by IFNγ, and 2190 by IL-1β (Tables S1 and **S2**)  Table S2. Genes significantly deregulated by cytokine treatments, related to Figures 1–4List of significantly upregulated and downregulated genes in HPPT cells treated with IFNα, IFNβ, IFNγ, IL-1β, ruxolitinib + IFNβ'),\n",
       " 'PMC9906673-BLOODA_ADV-2022-008445-mmc1.xlsx-105-Day_0_vs_Day7': ('Analysis of immune responses in patients with CLL after heterologous COVID-19 vaccination',\n",
       "  'Patients with chronic lymphocytic leukemia (CLL) treated with B-cell pathway inhibitors and anti-CD20 antibodies exhibit low humoral response rates following SARS-CoV-2 vaccination. To investigate this observation, a prospective single-institution study was conducted comparing peripheral blood mononuclear cell transcriptional response with antibody and T-cell response rates following heterologous BNT162b2/ChAdOx1 vaccination of 15 patients with CLL/small lymphocytic lymphoma (SLL). Two-dose antibody response rate was 40%, increasing to 53% after booster. Patients on Bruton tyrosine kinase inhibitor (BTKi) and venetoclax ± anti-CD20 antibody within 12 months of vaccination responded inferiorly compared with those under BTKi alone. The 2-dose-T-cell response rate was 80%, which increased to 93% after the booster dose. Key transcriptional findings were that interferon-mediated signaling activation including activation of the JAK-STAT pathway generally occurred within days of vaccination, but was independent from the magnitude of the antibody response. Increasing counts of IGHV genes were associated with B-cell reconstitution and improved humoral response rate in the vaccinated patients. T-cell responses in patients with CLL appeared independent of treatment status, whereas higher humoral response rate was associated with BTKi treatment and B-cell reconstitution. Boosting was particularly effective when intrinsic immune status was improved by CLL treatment. Limitations included studying a relatively small cohort, with different treatments and vaccination schedules.',\n",
       "  '105-Day_0_vs_Day7',\n",
       "  'Supplemental File 1'),\n",
       " 'PMC5844691-elife-30496-supp1.xls-All_GeneLists-Unnamed_10': ('The transcription factors Runx3 and ThPOK cross-regulate acquisition of cytotoxic function by human Th1 lymphocytes',\n",
       "  'Cytotoxic CD4 (CD4 CTX ) T cells are emerging as an important component of antiviral and antitumor immunity, but the molecular basis of their development remains poorly understood. In the context of human cytomegalovirus infection, a significant proportion of CD4 T cells displays cytotoxic functions. We observed that the transcriptional program of these cells was enriched in CD8 T cell lineage genes despite the absence of ThPOK downregulation. We further show that establishment of CD4 CTX -specific transcriptional and epigenetic programs occurred in a stepwise fashion along the Th1-differentiation pathway. In vitro, prolonged activation of naive CD4 T cells in presence of Th1 polarizing cytokines led to the acquisition of perforin-dependent cytotoxic activity. This process was dependent on the Th1 transcription factor Runx3 and was limited by the sustained expression of ThPOK. This work elucidates the molecular program of human CD4 CTX T cells and identifies potential targets for immunotherapy against viral infections and cancer.',\n",
       "  'All_GeneLists-Unnamed_10',\n",
       "  'identified and their enrichment in naive CD8 and CD4 T cell transcriptomes was assessed (**Supplementary file 1**)  Lists of genes included in the 12 GeneSets obtained from CD4 versus CD8 T cell comparison (Figure 2) and naive CD4 T cell versus CMTh1 cell versus CD4CTX T cell comparison (Figure 4).Genes expressed at higher level by the first listed subset as compared to the subset indicated between brackets were identified using the min/max method. Genes are ranked according to mean log2 fold change calculated using the Limma package in R'),\n",
       " 'PMC10835465-mmc6.xlsx-Cluster25-gene': ('Association of HBsAg levels with differential gene expression in NK, CD8 T, and memory B cells in treated patients with chronic HBV',\n",
       "  'Background & aims: HBsAg secretion may impact immune responses to chronic HBV infection. Thus, therapeutic approaches to suppress HBsAg production are being investigated. Our study aims to examine the immunomodulatory effects of high and low levels of circulating HBsAg and thereby improve our understanding of anti-HBV immunity. Methods: An optimized 10x Genomics single-cell RNA sequencing workflow was applied to blood samples and liver fine-needle aspirates from 18 patients undergoing tenofovir/entecavir (NUC) treatment for chronic HBV infection. They were categorized based on their HBsAg levels: high (920-12,447 IU/ml) or low (1-100 IU/ml). Cluster frequencies, differential gene expression, and phenotypes were analyzed. Results: In the blood of HBV-infected patients on NUC, the proportion of KLRC2 + \"adaptive\" natural killer (NK) cells was significantly lower in the HBsAg-high group and, remarkably, both KLRC2 + NK and KLRG1+ CD8 T cells display enrichment of lymphocyte activation-associated gene sets in the HBsAg-low group. High levels of HBsAg were associated with mild immune activation in the liver. However, no suppression of liver-resident CXCR6+ NCAM1+ NK or CXCR6+ CD69+ CD8 T cells was detected, while memory B cells showed signs of activation in both the blood and liver. Conclusions: Among NUC-treated patients, we observed a minimal impact of HBsAg on leukocyte populations in the blood and liver. However, for the first time, we found that HBsAg has distinct effects, restricted to NK-, CD8 T-, and memory B-cell subsets, in the blood and liver. Our findings are highly relevant for current clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Impact and implications: This study provides unique insight into the impact of HBsAg on gene expression levels of immune cell subsets in the blood and liver, particularly in the context of NUC-treated chronic HBV infection. It holds significant relevance for current and future clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Our findings raise questions about the effectiveness of such treatment strategies and challenge the previously hypothesized immunomodulatory effects of HBsAg on immune responses against HBV.',\n",
       "  'Cluster25-gene',\n",
       "  'None here'),\n",
       " 'PMC4595011-pgen.1005555.s003.xlsx-MITF_DOWN-regulated-Gene_name': ('Chromatin-Remodelling Complex NURF Is Essential for Differentiation of Adult Melanocyte Stem Cells',\n",
       "  'MIcrophthalmia-associated Transcription Factor (MITF) regulates melanocyte and melanoma physiology. We show that MITF associates the NURF chromatin-remodelling factor in melanoma cells. ShRNA-mediated silencing of the NURF subunit BPTF revealed its essential role in several melanoma cell lines and in untransformed melanocytes in vitro. Comparative RNA-seq shows that MITF and BPTF co-regulate overlapping gene expression programs in cell lines in vitro. Somatic and specific inactivation of Bptf in developing murine melanoblasts in vivo shows that Bptf regulates their proliferation, migration and morphology. Once born, Bptf-mutant mice display premature greying where the second post-natal coat is white. This second coat is normally pigmented by differentiated melanocytes derived from the adult melanocyte stem cell (MSC) population that is stimulated to proliferate and differentiate at anagen. An MSC population is established and maintained throughout the life of the Bptf-mutant mice, but these MSCs are abnormal and at anagen, give rise to reduced numbers of transient amplifying cells (TACs) that do not express melanocyte markers and fail to differentiate into mature melanin producing melanocytes. MSCs display a transcriptionally repressed chromatin state and Bptf is essential for reactivation of the melanocyte gene expression program at anagen, the subsequent normal proliferation of TACs and their differentiation into mature melanocytes.',\n",
       "  'MITF_DOWN-regulated-Gene_name',\n",
       "  'showed that BPTF silencing repressed 1356 genes and up-regulated 1139 genes (Fig 3E and **S3 Dataset**)  Excel spread sheet of genes specifically and commonly regulated by BPTF and MITF knockdown in Hermes 3A melanocytes cells along with the gene ontology for each gene set.(XLSX)  Click here for additional data file'),\n",
       " 'PMC4595011-pgen.1005555.s003.xlsx-MITF_DOWN-regulated-Ensembl_gene_id': ('Chromatin-Remodelling Complex NURF Is Essential for Differentiation of Adult Melanocyte Stem Cells',\n",
       "  'MIcrophthalmia-associated Transcription Factor (MITF) regulates melanocyte and melanoma physiology. We show that MITF associates the NURF chromatin-remodelling factor in melanoma cells. ShRNA-mediated silencing of the NURF subunit BPTF revealed its essential role in several melanoma cell lines and in untransformed melanocytes in vitro. Comparative RNA-seq shows that MITF and BPTF co-regulate overlapping gene expression programs in cell lines in vitro. Somatic and specific inactivation of Bptf in developing murine melanoblasts in vivo shows that Bptf regulates their proliferation, migration and morphology. Once born, Bptf-mutant mice display premature greying where the second post-natal coat is white. This second coat is normally pigmented by differentiated melanocytes derived from the adult melanocyte stem cell (MSC) population that is stimulated to proliferate and differentiate at anagen. An MSC population is established and maintained throughout the life of the Bptf-mutant mice, but these MSCs are abnormal and at anagen, give rise to reduced numbers of transient amplifying cells (TACs) that do not express melanocyte markers and fail to differentiate into mature melanin producing melanocytes. MSCs display a transcriptionally repressed chromatin state and Bptf is essential for reactivation of the melanocyte gene expression program at anagen, the subsequent normal proliferation of TACs and their differentiation into mature melanocytes.',\n",
       "  'MITF_DOWN-regulated-Ensembl_gene_id',\n",
       "  'showed that BPTF silencing repressed 1356 genes and up-regulated 1139 genes (Fig 3E and **S3 Dataset**)  Excel spread sheet of genes specifically and commonly regulated by BPTF and MITF knockdown in Hermes 3A melanocytes cells along with the gene ontology for each gene set.(XLSX)  Click here for additional data file'),\n",
       " 'PMC8957021-mmc11.xlsx-B_cells-gene': ('Single-cell temporal analysis of natural dengue infection reveals skin-homing lymphocyte expansion one day before defervescence',\n",
       "  'Effective clinical management of acute dengue virus (DENV) infection relies on the timing of suitable treatments during the disease progression. We analyzed single-cell transcriptomic profiles of the peripheral blood mononuclear cell samples from two DENV patients, collected daily during acute phase and also at convalescence. Key immune cell types demonstrated different dynamic responses over the course of the infection. On the day before defervescence (Day -1), we observed the peak expression of several prominent genes in the adaptive immunological pathways. We also characterized unique effector T cell clusters that expressed skin-homing signature genes at Day -1, whereas upregulation of skin and gut homing genes was also observed in plasma cells and plasmablasts during the febrile period. This work provides an overview of unique molecular dynamics that signify the entry of the critical phase, and the findings could improve the patient management of DENV infection.',\n",
       "  'B_cells-gene',\n",
       "  'Tables S6, S7, S8, and S9 and the percentages of cells expressing HVGs in **Table S10**  Table S10. Percentages of cells in populations of interest that expressed particular genes, related to Figures 2C, S3 and STAR Methods'),\n",
       " 'PMC8591264-Table_1.XLS-GSE14520_DOWN-Gene_symbol': ('Downregulated Ferroptosis-Related Gene STEAP3 as a Novel Diagnostic and Prognostic Target for Hepatocellular Carcinoma and Its Roles in Immune Regulation',\n",
       "  'Ferroptosis, a distinct type of regulated cell death, has been reported to be involved in the tumorigenesis of liver hepatocellular carcinoma (LIHC). However, the precise functions and potential mechanisms of ferroptosis in LIHC were still poorly understood. Herein, we investigated the biological roles of ferroptosis-related gene STEAP3 in LIHC. STEAP3 was previously proved to serve a key regulator in ferroptosis via mediating the iron metabolism. Comprehensive bioinformatics from several databases revealed that STEAP3 was significantly downregulated in LIHC tissues and exhibited the favorable prognostic significance in LIHC patients. The downregulated STEAP3 was further confirmed in two LIHC cells Huh7 and MHCC97H using real-time PCR and western blot. And STEAP3 overexpression significantly inhibited the cell proliferation in Huh7 and MHCC97H cells. In addition, clinical data identified the relationship between STEAP3 expression and several clinicopathological parameters of LIHC patients, including histologic grade, alpha fetal protein (AFP) concentration, etc. Receiver operation characteristic (ROC) curve revealed STEAP3 as a potential diagnostic biomarker for LIHC patients. Moreover, the co-expression network of STEAP3 was explored to gain a better insight into its underlying signaling pathways. Finally, aberrant STEAP3 might participate in varieties of immune-associated signatures in LIHC pathogenesis, including immunostimulators, immunoinhibitors, chemokines, and chemokine receptors. Taken together, these findings could enhance our knowledge regarding the inhibitory roles and underlying biological significance of STEAP3 in LIHC tumorigenesis.',\n",
       "  'GSE14520_DOWN-Gene_symbol',\n",
       "  'genes in GSE14520, and 343 up-regulated genes and 258 down-regulated genes in GSE14323, respectively (**Supplementary Table 1**)  Click here for additional data file'),\n",
       " 'PMC7935557-Table_4.xlsx-SKCM-Query': ('CD96 Correlates With Immune Infiltration and Impacts Patient Prognosis: A Pan-Cancer Analysis',\n",
       "  'Background: Immunotherapy has significantly improved patient outcomes, but encountered obstacles recently. CD96, a novel immune checkpoint expressed on T cells and natural killer (NK) cells, is essential for regulating immune functions. However, how CD96 correlating with immune infiltration and patient prognosis in pan-cancer remains unclear. Methods: HPA, TCGA, GEO, GTEx, Oncomine, TIMER2.0, PrognoScan, Linkedomics, Metascape, and GEPIA2 databases were used to analyze CD96 in cancers. Visualization of data was mostly achieved by R language, version 4.0.2. Results: In general, CD96 was differentially expressed between most cancer and adjacent normal tissues. CD96 significantly impacted the prognosis of diverse cancers. Especially, high CD96 expression was associated with poorer overall survival (OS) and disease-specific survival (DSS) in the TCGA lower grade glioma (LGG) cohort (OS, HR = 2.18, 95% CI = 1.79-2.66, P < 0.001). The opposite association was significantly observed in skin cutaneous melanoma (SKCM) cohort (OS, HR = 0.96, 95% CI = 0.94-0.98, P < 0.001). Notably, SKCM samples demonstrated the highest CD96 mutation frequency among all cancer types. Furthermore, in most cancers, CD96 expression level was significantly correlated with expression levels of recognized immune checkpoints and abundance of multiple immune infiltrates including CD8+ T cells, dendric cells (DCs), macrophages, monocytes, NK cells, neutrophils, regulatory T cells (Tregs), and follicular helper T cells (Tfh). CD96 was identified as a risk factor, protective factor, and irrelevant variable in LGG, SKCM and adrenocortical carcinoma (ACC), respectively. CD96 related genes were involved in negative regulation of leukocyte in LGG, however, involved in multiple positive immune processes in SKCM. Furthermore, CD96 was significantly associated with particular immune marker subsets. Importantly, it strongly correlated with markers of type 1 helper T cell (Th1) in SKCM, but not in LGG or ACC either. Conclusions: CD96 participates in diverse immune responses, governs immune cell infiltration, and impacts malignant properties of various cancer types, thus standing as a potential biomarker for determining patient prognosis and immune infiltration in multiple cancers, especially in glioma and melanoma.',\n",
       "  'SKCM-Query',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC8157440-12885_2021_8378_MOESM1_ESM.docx-0-Corgene': ('Evaluation of clinical value and potential mechanism of MTFR2 in lung adenocarcinoma via bioinformatics',\n",
       "  'Background: Mitochondrial fission regulator 2 (MTFR2) was involved in the progression and development of various cancers. However, the relationship between MTFR2 with lung adenocarcinoma (LUAD) had not been reported. Herein, this study analyzed the clinical significance and potential mechanisms of MTFR2 in LUAD via bioinformatics tools. Results: We found that the level of MTFR2 was increased, and correlated with sex, age, smoking history, neoplasm staging, histological subtype and TP53 mutation status in LUAD patients. Kaplan-Meier survival analysis showed LUAD patients with increased MTFR2 had a poor prognosis. In addition, univariate COX regression analysis showed neoplasm staging, T stage, distant metastasis and MTFR2 level were risk factors for the prognosis of LUAD. A total of 1127 genes were coexpressed with MTFR2, including 840 positive and 208 negative related genes. KEGG and GSEA found that MTFR2 participated in the progression of LUAD by affecting cell cycle, DNA replication, homologous recombination, p53 signaling pathway and other mechanisms. The top 10 coexpressed genes, namely CDK1, CDC20, CCNB1, PLK1, CCNA2, AURKB, CCNB2, BUB1B, MAD2L1 and BUB1 were highly expressed, and were associated with poor prognosis in LUAD. Conclusions: Consequently, we elucidated MTFR2 was a biomarker for diagnosis and poor prognosis in LUAD, and might participate in the progression of LUAD via affecting cell cycle, DNA replication, homologous recombination and p53 signaling pathway.',\n",
       "  '0-Corgene',\n",
       "  'We screened **1**  Additional file 1: Table S1. Co-expressed gene of MTFR2'),\n",
       " 'PMC9399107-41467_2022_32627_MOESM6_ESM.xlsx-20_T-cell_subsets-gene_name_Cycling_T_cells': ('scRNA-seq of gastric tumor shows complex intercellular interaction with an alternative T cell exhaustion trajectory',\n",
       "  'The tumor microenvironment (TME) in gastric cancer (GC) has been shown to be important for tumor control but the specific characteristics for GC are not fully appreciated. We generated an atlas of 166,533 cells from 10 GC patients with matched paratumor tissues and blood. Our results show tumor-associated stromal cells (TASCs) have upregulated activity of Wnt signaling and angiogenesis, and are negatively correlated with survival. Tumor-associated macrophages and LAMP3 + DCs are involved in mediating T cell activity and form intercellular interaction hubs with TASCs. Clonotype and trajectory analysis demonstrates that Tc17 (IL-17 + CD8 + T cells) originate from tissue-resident memory T cells and can subsequently differentiate into exhausted T cells, suggesting an alternative pathway for T cell exhaustion. Our results indicate that IL17 + cells may promote tumor progression through IL17, IL22, and IL26 signaling, highlighting the possibility of targeting IL17 + cells and associated signaling pathways as a therapeutic strategy to treat GC.',\n",
       "  '20_T-cell_subsets-gene_name_Cycling_T_cells',\n",
       "  'PCA, UMAP, and Leiden clustering independently for further analysis. For these analyses, the first **3**  Supplementary Data 3'),\n",
       " 'PMC6652661-JCMM-23-5600-s006.xlsx-Sheet1-Coexpressed_Gene_Symbol': ('Survival analysis and functional annotation of long non-coding RNAs in lung adenocarcinoma',\n",
       "  'Long non-coding RNAs (lncRNAs) are a subclass of non-protein coding transcripts that are involved in several regulatory processes and are considered as potential biomarkers for almost all cancer types. This study aims to investigate the prognostic value of lncRNAs for lung adenocarcinoma (LUAD), the most prevalent subtype of lung cancer. To this end, the processed data of The Cancer Genome Atlas LUAD were retrieved from GEPIA and circlncRNAnet databases, matched with each other and integrated with the analysis results of a non-small cell lung cancer plasma RNA-Seq study. Then, the data were filtered in order to separate the differentially expressed lncRNAs that have a prognostic value for LUAD. Finally, the selected lncRNAs were functionally annotated using a bioinformatic and systems biology approach. Accordingly, we identified 19 lncRNAs as the novel LUAD prognostic lncRNAs. Also, based on our results, all 19 lncRNAs might be involved in lung cancer-related biological processes. Overall, we suggested several novel biomarkers and drug targets which could help early diagnosis, prognosis and treatment of LUAD patients.',\n",
       "  'Sheet1-Coexpressed_Gene_Symbol',\n",
       "  'GEPIA CEGs ⋂ GEPIA DEGs) so as to separate differentially expressed CEGs (DECEGs) (Table **S4**)  Click here for additional data file'),\n",
       " 'PMC9579367-Table_1.xlsx-GSE14520_DOWN-Gene_symbol': ('Application of genomic selection and experimental techniques to predict cell death and immunotherapeutic efficacy of ferroptosis-related CXCL2 in hepatocellular carcinoma',\n",
       "  \"Since most hepatocellular carcinoma (HCC) patients are diagnosed at advanced stages, there is no effective treatment to improve patient survival. Ferroptosis, a regulated cell death driven by iron accumulation and lipid peroxidation, has been reported to play an important role in tumorigenesis. However, the detailed mechanism and biological function of ferroptosis are still incompletely understood in HCC patients. In this study, we analyzed genomic profiles of three HCC datasets, GSE6764, GSE14520, and GSE14323. Venn diagrams were implemented to visualize the overlapping genes between differentially expressed genes and ferroptosis-related gene set. Then, one up-regulated gene, ACSL4, and five down-regulated genes, STEAP3, MT1G, GCH1, HAMP, and CXCL2, were screened. Based on the survival analysis performed by Kaplan-Meier plotter database, ferroptosis-related gene CXCL2 was demonstrated positively-correlated with the patients' prognosis. Moreover, CXCL2 overexpression significantly inhibited cell growth and improved cellular ROS, Fe 2+ and MDA levels in HCC cells Huh7 and MHCC97H, suggesting the roles of CXCL2 in inducing ferroptotic cell death. In addition, aberrantly expressed CXCL2 was negatively associated with malignancy clinical features, such as nodal metastasis and higher grades. The ssGSEA enrichment analysis revealed that CXCL2 co-expressed molecules were mainly involved in inflammation and immune-related pathways, such as acute inflammatory response, humoral immune response, adaptive immune response. TISIDB algorithm indicated the positive correlation between CXCL2 expression and tumor-infiltrating immune cells, including neutrophils and macrophages. Additionally, we also found that CXCL2 was positively correlated with immune infiltration score, and HCC patients with higher score harbored better prognosis. Together, these findings suggested that CXCL2 may enhance ferroptosis sensitivity and regulate immune microenvironment in HCC, and serve as a promising prognosis biomarker for HCC patients.\",\n",
       "  'GSE14520_DOWN-Gene_symbol',\n",
       "  'and 583 in GSE14520 had been identified to be significantly down-regulated in HCC ( **Supplementary Table S1**)  Differentially expressed genes in GSE6764, GSE14323, and GSE14520  Click here for additional data file'),\n",
       " 'PMC10804933-mmc1.xlsx-Table_S7-Unnamed_21': ('Single-Nuclei RNA Sequencing of 5 Regions of the Human Prenatal Brain Implicates Developing Neuron Populations in Genetic Risk for Schizophrenia',\n",
       "  'Background: While a variety of evidence supports a prenatal component in schizophrenia, there are few data regarding the cell populations involved. We sought to identify cells of the human prenatal brain mediating genetic risk for schizophrenia by integrating cell-specific gene expression measures generated through single-nuclei RNA sequencing with recent large-scale genome-wide association study (GWAS) and exome sequencing data for the condition. Methods: Single-nuclei RNA sequencing was performed on 5 brain regions (frontal cortex, ganglionic eminence, hippocampus, thalamus, and cerebellum) from 3 fetuses from the second trimester of gestation. Enrichment of schizophrenia common variant genetic liability and rare damaging coding variation was assessed in relation to gene expression specificity within each identified cell population. Results: Common risk variants were prominently enriched within genes with high expression specificity for developing neuron populations within the frontal cortex, ganglionic eminence, and hippocampus. Enrichments were largely independent of genes expressed in neuronal populations of the adult brain that have been implicated in schizophrenia through the same methods. Genes containing an excess of rare damaging variants in schizophrenia had higher expression specificity for developing glutamatergic neurons of the frontal cortex and hippocampus that were also enriched for common variant liability. Conclusions: We found evidence for a distinct contribution of prenatal neuronal development to genetic risk for schizophrenia, involving specific populations of developing neurons within the second-trimester fetal brain. Our study significantly advances the understanding of the neurodevelopmental origins of schizophrenia and provides a resource with which to investigate the prenatal antecedents of other psychiatric and neurologic disorders.',\n",
       "  'Table_S7-Unnamed_21',\n",
       "  'None here'),\n",
       " 'PMC4741458-oncotarget-06-34358-s005.xlsx-Sheet1-Unnamed_3': ('The androgen receptor controls expression of the cancer-associated sTn antigen and cell adhesion through induction of ST6GalNAc1 in prostate cancer',\n",
       "  'Patterns of glycosylation are important in cancer, but the molecular mechanisms that drive changes are often poorly understood. The androgen receptor drives prostate cancer (PCa) development and progression to lethal metastatic castration-resistant disease. Here we used RNA-Seq coupled with bioinformatic analyses of androgen-receptor (AR) binding sites and clinical PCa expression array data to identify ST6GalNAc1 as a direct and rapidly activated target gene of the AR in PCa cells. ST6GalNAc1 encodes a sialytransferase that catalyses formation of the cancer-associated sialyl-Tn antigen (sTn), which we find is also induced by androgen exposure. Androgens induce expression of a novel splice variant of the ST6GalNAc1 protein in PCa cells. This splice variant encodes a shorter protein isoform that is still fully functional as a sialyltransferase and able to induce expression of the sTn-antigen. Surprisingly, given its high expression in tumours, stable expression of ST6GalNAc1 in PCa cells reduced formation of stable tumours in mice, reduced cell adhesion and induced a switch towards a more mesenchymal-like cell phenotype in vitro. ST6GalNAc1 has a dynamic expression pattern in clinical datasets, beingsignificantly up-regulated in primary prostate carcinoma but relatively down-regulated in established metastatic tissue. ST6GalNAc1 is frequently upregulated concurrently with another important glycosylation enzyme GCNT1 previously associated with prostate cancer progression and implicated in Sialyl Lewis X antigen synthesis. Together our data establishes an androgen-dependent mechanism for sTn antigen expression in PCa, and are consistent with a general role for the androgen receptor in driving important coordinate changes to the glycoproteome during PCa progression.',\n",
       "  'Sheet1-Unnamed_3',\n",
       "  'an 86% overlap, with > 4 times more differentially expressed genes identified by RNA-Seq (**Supplementary Table 4**)'),\n",
       " 'PMC8223271-13104_2021_5657_MOESM5_ESM.xlsx-Table_S2-Gene': ('Time-course transcriptome analysis of host cell response to poxvirus infection using a dual long-read sequencing approach',\n",
       "  'Objective: In this study, we applied two long-read sequencing (LRS) approaches, including single-molecule real-time and nanopore-based sequencing methods to investigate the time-lapse transcriptome patterns of host gene expression as a response to Vaccinia virus infection. Transcriptomes determined using short-read sequencing approaches are incomplete because these platforms are inefficient or fail to distinguish between polycistronic RNAs, transcript isoforms, transcriptional start sites, as well as transcriptional readthroughs and overlaps. Long-read sequencing is able to read full-length nucleic acids and can therefore be used to assemble complete transcriptome atlases. Results: In this work, we identified a number of novel transcripts and transcript isoforms of Chlorocebus sabaeus. Additionally, analysis of the most abundant 768 host transcripts revealed a significant overrepresentation of the class of genes in the \"regulation of signaling receptor activity\" Gene Ontology annotation as a result of viral infection.',\n",
       "  'Table_S2-Gene',\n",
       "  'within the range between the top score and the top score–1 SD (Additional file **5**)  Additional file 5: Table S1. Clustering of the 768 most active host genes based on their expression during VACV infection. Relative expression values at each p.i. time point are normalized to the highest value for a given gene. Genes that were most characteristic of each cluster are highlighted in gray'),\n",
       " 'PMC4741458-oncotarget-06-34358-s003.xlsx-Sheet1-Unnamed_2': ('The androgen receptor controls expression of the cancer-associated sTn antigen and cell adhesion through induction of ST6GalNAc1 in prostate cancer',\n",
       "  'Patterns of glycosylation are important in cancer, but the molecular mechanisms that drive changes are often poorly understood. The androgen receptor drives prostate cancer (PCa) development and progression to lethal metastatic castration-resistant disease. Here we used RNA-Seq coupled with bioinformatic analyses of androgen-receptor (AR) binding sites and clinical PCa expression array data to identify ST6GalNAc1 as a direct and rapidly activated target gene of the AR in PCa cells. ST6GalNAc1 encodes a sialytransferase that catalyses formation of the cancer-associated sialyl-Tn antigen (sTn), which we find is also induced by androgen exposure. Androgens induce expression of a novel splice variant of the ST6GalNAc1 protein in PCa cells. This splice variant encodes a shorter protein isoform that is still fully functional as a sialyltransferase and able to induce expression of the sTn-antigen. Surprisingly, given its high expression in tumours, stable expression of ST6GalNAc1 in PCa cells reduced formation of stable tumours in mice, reduced cell adhesion and induced a switch towards a more mesenchymal-like cell phenotype in vitro. ST6GalNAc1 has a dynamic expression pattern in clinical datasets, beingsignificantly up-regulated in primary prostate carcinoma but relatively down-regulated in established metastatic tissue. ST6GalNAc1 is frequently upregulated concurrently with another important glycosylation enzyme GCNT1 previously associated with prostate cancer progression and implicated in Sialyl Lewis X antigen synthesis. Together our data establishes an androgen-dependent mechanism for sTn antigen expression in PCa, and are consistent with a general role for the androgen receptor in driving important coordinate changes to the glycoproteome during PCa progression.',\n",
       "  'Sheet1-Unnamed_2',\n",
       "  '674 up- and 1834 down- regulated genes (p < 0.0075, Supplementary Figure 1 and **Supplementary Tables 2**)'),\n",
       " 'PMC8141567-DataSheet_1.xlsx-Table_S2-Supplementary_table_2_Correlation_analysis_between_transciptional_level_of_genes_in_the_TCGA_SKCM_dataset_and_infiltration_of_CD8_T_cells_evaluated_by_TIMER': ('Development and Validation of a CD8+ T Cell Infiltration-Related Signature for Melanoma Patients',\n",
       "  \"Aim: Immunotherapy shows efficacy in only a subset of melanoma patients. Here, we intended to construct a risk score model to predict melanoma patients' sensitivity to immunotherapy. Methods: Integration analyses were performed on melanoma patients from high-dimensional public datasets. The CD8+ T cell infiltration related genes (TIRGs) were selected via TIMER and CIBERSORT algorithm. LASSO Cox regression was performed to screen for the crucial TIRGs. Single sample gene set enrichment analysis (ssGSEA) and ESTIMATE algorithm were used to evaluate the immune activity. The prognostic value of the risk score was determined by univariate and multivariate Cox regression analysis. Results: 184 candidate TIRGs were identified in melanoma patients. Based on the candidate TIRGs, melanoma patients were classified into three clusters which were characterized by different immune activity. Six signature genes were further screened out of 184 TIRGs and a representative risk score for patient survival was constructed based on these six signature genes. The risk score served as an indicator for the level of CD8+ T cell infiltration and acted as an independent prognostic factor for the survival of melanoma patients. By using the risk score, we achieved a good predicting result for the response of cancer patients to immunotherapy. Moreover, pan-cancer analysis revealed the risk score could be used in a wide range of non-hematologic tumors. Conclusions: Our results showed the potential of using signature gene-based risk score as an indicator to predict melanoma patients' sensitivity to immunotherapy.\",\n",
       "  'Table_S2-Supplementary_table_2_Correlation_analysis_between_transciptional_level_of_genes_in_the_TCGA_SKCM_dataset_and_infiltration_of_CD8_T_cells_evaluated_by_TIMER',\n",
       "  'and CIBERSORT, and analyzed their prognostic significance by univariate Cox analysis. As shown in **Supplementary Table 1**  Click here for additional data file'),\n",
       " 'PMC11188533-12993_2024_240_MOESM2_ESM.xlsx-Figure_3A_VolcanoPlots-Unnamed_26': ('Transcriptome analysis identifies an ASD-Like phenotype in oligodendrocytes and microglia from C58/J amygdala that is dependent on sex and sociability',\n",
       "  \"Background: Autism Spectrum Disorder (ASD) is a group of neurodevelopmental disorders with higher incidence in males and is characterized by atypical verbal/nonverbal communication, restricted interests that can be accompanied by repetitive behavior, and disturbances in social behavior. This study investigated brain mechanisms that contribute to sociability deficits and sex differences in an ASD animal model. Methods: Sociability was measured in C58/J and C57BL/6J mice using the 3-chamber social choice test. Bulk RNA-Seq and snRNA-Seq identified transcriptional changes in C58/J and C57BL/6J amygdala within which DMRseq was used to measure differentially methylated regions in amygdala. Results: C58/J mice displayed divergent social strata in the 3-chamber test. Transcriptional and pathway signatures revealed immune-related biological processes differ between C58/J and C57BL/6J amygdala. Hypermethylated and hypomethylated genes were identified in C58/J versus C57BL/6J amygdala. snRNA-Seq data in C58/J amygdala identified differential transcriptional signatures within oligodendrocytes and microglia characterized by increased ASD risk gene expression and predicted impaired myelination that was dependent on sex and sociability. RNA velocity, gene regulatory network, and cell communication analysis showed diminished oligodendrocyte/microglia differentiation. Findings were verified using Bulk RNA-Seq and demonstrated oxytocin's beneficial effects on myelin gene expression. Limitations: Our findings are significant. However, limitations can be noted. The cellular mechanisms linking reduced oligodendrocyte differentiation and reduced myelination to an ASD phenotype in C58/J mice need further investigation. Additional snRNA-Seq and spatial studies would determine if effects in oligodendrocytes/microglia are unique to amygdala or if this occurs in other brain regions. Oxytocin's effects need further examination to understand its' potential as an ASD therapeutic. Conclusions: Our work demonstrates the C58/J mouse model's utility in evaluating the influence of sex and sociability on the transcriptome in concomitant brain regions involved in ASD. Our single-nucleus transcriptome analysis elucidates potential pathological roles of oligodendrocytes and microglia in ASD. This investigation provides details regarding regulatory features disrupted in these cell types, including transcriptional gene dysregulation, aberrant cell differentiation, altered gene regulatory networks, and changes to key pathways that promote microglia/oligodendrocyte differentiation. Our studies provide insight into interactions between genetic risk and epigenetic processes associated with divergent affiliative behavior and lack of positive sociability.\",\n",
       "  'Figure_3A_VolcanoPlots-Unnamed_26',\n",
       "  'None here'),\n",
       " 'PMC11188533-12993_2024_240_MOESM2_ESM.xlsx-Supp_Figure_7A-Unnamed_74': ('Transcriptome analysis identifies an ASD-Like phenotype in oligodendrocytes and microglia from C58/J amygdala that is dependent on sex and sociability',\n",
       "  \"Background: Autism Spectrum Disorder (ASD) is a group of neurodevelopmental disorders with higher incidence in males and is characterized by atypical verbal/nonverbal communication, restricted interests that can be accompanied by repetitive behavior, and disturbances in social behavior. This study investigated brain mechanisms that contribute to sociability deficits and sex differences in an ASD animal model. Methods: Sociability was measured in C58/J and C57BL/6J mice using the 3-chamber social choice test. Bulk RNA-Seq and snRNA-Seq identified transcriptional changes in C58/J and C57BL/6J amygdala within which DMRseq was used to measure differentially methylated regions in amygdala. Results: C58/J mice displayed divergent social strata in the 3-chamber test. Transcriptional and pathway signatures revealed immune-related biological processes differ between C58/J and C57BL/6J amygdala. Hypermethylated and hypomethylated genes were identified in C58/J versus C57BL/6J amygdala. snRNA-Seq data in C58/J amygdala identified differential transcriptional signatures within oligodendrocytes and microglia characterized by increased ASD risk gene expression and predicted impaired myelination that was dependent on sex and sociability. RNA velocity, gene regulatory network, and cell communication analysis showed diminished oligodendrocyte/microglia differentiation. Findings were verified using Bulk RNA-Seq and demonstrated oxytocin's beneficial effects on myelin gene expression. Limitations: Our findings are significant. However, limitations can be noted. The cellular mechanisms linking reduced oligodendrocyte differentiation and reduced myelination to an ASD phenotype in C58/J mice need further investigation. Additional snRNA-Seq and spatial studies would determine if effects in oligodendrocytes/microglia are unique to amygdala or if this occurs in other brain regions. Oxytocin's effects need further examination to understand its' potential as an ASD therapeutic. Conclusions: Our work demonstrates the C58/J mouse model's utility in evaluating the influence of sex and sociability on the transcriptome in concomitant brain regions involved in ASD. Our single-nucleus transcriptome analysis elucidates potential pathological roles of oligodendrocytes and microglia in ASD. This investigation provides details regarding regulatory features disrupted in these cell types, including transcriptional gene dysregulation, aberrant cell differentiation, altered gene regulatory networks, and changes to key pathways that promote microglia/oligodendrocyte differentiation. Our studies provide insight into interactions between genetic risk and epigenetic processes associated with divergent affiliative behavior and lack of positive sociability.\",\n",
       "  'Supp_Figure_7A-Unnamed_74',\n",
       "  'None here'),\n",
       " 'PMC10966709-mmc5.xlsx-Top_1000_regulated_genes-Unnamed_23': ('Integrated genomic and proteomic analyses identify PYGL as a novel experimental therapeutic target for clear cell renal cell carcinoma',\n",
       "  'Sunitinib, the first-line targeted therapy for metastatic clear cell renal cell carcinoma (ccRCC), faces a significant challenge as most patients develop acquired resistance. Integrated genomic and proteomic analyses identified PYGL as a novel therapeutic target for ccRCC. PYGL knockdown inhibited cell proliferation, cloning capacity, migration, invasion, and tumorigenesis in ccRCC cell lines. PYGL expression was increased in sunitinib-resistant ccRCC cell lines, and CP-91149 targeting the PYGL could restore drug sensitivity in these cell lines. Moreover, chromatin immune-precipitation assays revealed that PYGL upregulation is induced by the transcription factor, hypoxia-inducible factor 1α. Overall, PYGL was identified as a novel diagnostic biomarker by combining genomic and proteomic approaches in ccRCC, and sunitinib resistance to ccRCC may be overcome by targeting PYGL.',\n",
       "  'Top_1000_regulated_genes-Unnamed_23',\n",
       "  'None here'),\n",
       " 'PMC10579699-JEM_20230707_TableS5.xlsx-Endothelial-Endothelial': ('CD8+ lymphocytes are critical for early control of tuberculosis in macaques',\n",
       "  'The functional role of CD8+ lymphocytes in tuberculosis remains poorly understood. We depleted innate and/or adaptive CD8+ lymphocytes in macaques and showed that loss of all CD8α+ cells (using anti-CD8α antibody) significantly impaired early control of Mycobacterium tuberculosis (Mtb) infection, leading to increased granulomas, lung inflammation, and bacterial burden. Analysis of barcoded Mtb from infected macaques demonstrated that depletion of all CD8+ lymphocytes allowed increased establishment of Mtb in lungs and dissemination within lungs and to lymph nodes, while depletion of only adaptive CD8+ T cells (with anti-CD8β antibody) worsened bacterial control in lymph nodes. Flow cytometry and single-cell RNA sequencing revealed polyfunctional cytotoxic CD8+ lymphocytes in control granulomas, while CD8-depleted animals were unexpectedly enriched in CD4 and γδ T cells adopting incomplete cytotoxic signatures. Ligand-receptor analyses identified IL-15 signaling in granulomas as a driver of cytotoxic T cells. These data support that CD8+ lymphocytes are required for early protection against Mtb and suggest polyfunctional cytotoxic responses as a vaccine target.',\n",
       "  'Endothelial-Endothelial',\n",
       "  'None here'),\n",
       " 'PMC4683939-40478_2015_266_MOESM4_ESM.xlsx-PAs-Symbol_1': ('Molecular analysis of pediatric brain tumors identifies microRNAs in pilocytic astrocytomas that target the MAPK and NF-κB pathways',\n",
       "  'Introduction: Pilocytic astrocytomas are slow-growing tumors that usually occur in the cerebellum or in the midline along the hypothalamic/optic pathways. The most common genetic alterations in pilocytic astrocytomas activate the ERK/MAPK signal transduction pathway, which is a major driver of proliferation but is also believed to induce senescence in these tumors. Here, we have conducted a detailed investigation of microRNA and gene expression, together with pathway analysis, to improve our understanding of the regulatory mechanisms in pilocytic astrocytomas. Results: Pilocytic astrocytomas were found to have distinctive microRNA and gene expression profiles compared to normal brain tissue and a selection of other pediatric brain tumors. Several microRNAs found to be up-regulated in pilocytic astrocytomas are predicted to target the ERK/MAPK and NF-κB signaling pathways as well as genes involved in senescence-associated inflammation and cell cycle control. Furthermore, IGFBP7 and CEBPB, which are transcriptional inducers of the senescence-associated secretory phenotype (SASP), were also up-regulated together with the markers of senescence and inflammation, CDKN1A (p21), CDKN2A (p16) and IL1B. Conclusion: These findings provide further evidence of a senescent phenotype in pilocytic astrocytomas. In addition, they suggest that the ERK/MAPK pathway, which is considered the major driver of these tumors, is regulated not only by genetic aberrations but also by microRNAs.',\n",
       "  'PAs-Symbol_1',\n",
       "  'None here'),\n",
       " 'PMC8628461-12920_2021_1134_MOESM1_ESM.xlsx-GSE84796_human_up-DEGs_up': ('Comprehensive analysis of miRNA-mRNA regulatory network and potential drugs in chronic chagasic cardiomyopathy across human and mouse',\n",
       "  'Background: Chronic chagasic cardiomyopathy (CCC) is the leading cause of heart failure in Latin America and often causes severe inflammation and fibrosis in the heart. Studies on myocardial function and its molecular mechanisms in patients with Chronic chagasic cardiomyopathy are very limited. In order to understand the development and progression of Chronic chagasic cardiomyopathy and find targets for its diagnosis and treatment, the field needs to better understand the exact molecular mechanisms involved in these processes. Methods: The mRNA microarray datasets GSE84796 (human) and GSE24088 (mouse) were obtained from the Gene Expression Omnibus (GEO) database. Homologous genes between the two species were identified using the online database mining tool Biomart, followed by differential expression analysis, gene enrichment analysis and protein-protein interaction (PPI) network construction. Cytohubba plug-in of Cytoscape software was used to identify Hub gene, and miRNet was used to construct the corresponding miRNA-mRNA regulatory network. miRNA-related databases: miRDB, Targetscan and miRWalk were used to further evaluate miRNAs in the miRNA-mRNA network. Furthermore, Comparative Toxicogenomics Database (CTD) and L1000 Platform were used to identify hub gene-related drugs. Results: A total of 86 homologous genes were significantly differentially expressed in the two datasets, including 73 genes with high expression and 13 genes with low expression. These differentially expressed genes were mainly enriched in the terms of innate immune response, signal transduction, protein binding, Natural killer cell mediated cytotoxicity, Tuberculosis, Chemokine signaling pathway, Chagas disease and PI3K-Akt signaling pathway. The top 10 hub genes LAPTM5, LCP1, HCLS1, CORO1A, CD48, TYROBP, RAC2, ARHGDIB, FERMT3 and NCF4 were identified from the PPI network. A total of 122 miRNAs were identified to target these hub genes and 30 of them regulated two or more hub genes at the same time. miRDB, Targetscan and miRWalk were further analyzed and screened out hsa-miR-34c-5p, hsa-miR-34a-5p and hsa-miR-16-5p as miRNAs regulating these hub genes. Finally, Progesterone, Flutamide, Nimesulide, Methotrexate and Temozolomide were identified to target these hub genes and might be targeted therapies for Chronic chagasic cardiomyopathy. Conclusions: In this study, the potential genes associated with Chronic chagasic cardiomyopathy are identified and a miRNA-mRNA regulatory network is constructed. This study explores the molecular mechanisms of Chronic chagasic cardiomyopathy and provides important clues for finding new therapeutic targets.',\n",
       "  'GSE84796_human_up-DEGs_up',\n",
       "  'analysis and the screening conditions, DEGs in each GEO dataset were identified. There were **1**  Additional file 1. DEGs in GSE84796 (human) and GSE24088 (mouse)'),\n",
       " 'PMC9982587-ADVS-10-2205486-s002.xlsx-Table_S4-Unnamed_5': ('Gene Expression has Distinct Associations with Brain Structure and Function in Major Depressive Disorder',\n",
       "  'Major depressive disorder (MDD) is associated with structural and functional brain abnormalities. MDD as well as brain anatomy and function are influenced by genetic factors, but the role of gene expression remains unclear. Here, this work investigates how cortical gene expression contributes to structural and functional brain abnormalities in MDD. This work compares the gray matter volume and resting-state functional measures in a Chinese sample of 848 MDD patients and 749 healthy controls, and these case-control differences are then associated with cortical variation of gene expression. While whole gene expression is positively associated with structural abnormalities, it is negatively associated with functional abnormalities. This work observes the relationships of expression levels with brain abnormalities for individual genes, and found that transcriptional correlates of brain structure and function show opposite relations with gene dysregulation in postmortem cortical tissue from MDD patients. This work further identifies genes that are positively or negatively related to structural abnormalities as well as functional abnormalities. The MDD-related genes are enriched for brain tissue, cortical cells, and biological pathways. These findings suggest that distinct genetic mechanisms underlie structural and functional brain abnormalities in MDD, and highlight the importance of cortical gene expression for the development of cortical abnormalities.',\n",
       "  'Table_S4-Unnamed_5',\n",
       "  'regions of each gene were related to the funcPC1 and GMV differences, respectively (Tables **S1**)  Supplemental Table  Click here for additional data file'),\n",
       " 'PMC7642433-41467_2020_19408_MOESM4_ESM.xlsx-Supplementary_Data_1-Unnamed_11': ('Integrated digital pathology and transcriptome analysis identifies molecular mediators of T-cell exclusion in ovarian cancer',\n",
       "  'Close proximity between cytotoxic T lymphocytes and tumour cells is required for effective immunotherapy. However, what controls the spatial distribution of T cells in the tumour microenvironment is not well understood. Here we couple digital pathology and transcriptome analysis on a large ovarian tumour cohort and develop a machine learning approach to molecularly classify and characterize tumour-immune phenotypes. Our study identifies two important hallmarks characterizing T cell excluded tumours: 1) loss of antigen presentation on tumour cells and 2) upregulation of TGFβ and activated stroma. Furthermore, we identify TGFβ as an important mediator of T cell exclusion. TGFβ reduces MHC-I expression in ovarian cancer cells in vitro. TGFβ also activates fibroblasts and induces extracellular matrix production as a potential physical barrier to hinder T cell infiltration. Our findings indicate that targeting TGFβ might be a promising strategy to overcome T cell exclusion and improve clinical benefits of cancer immunotherapy.',\n",
       "  'Supplementary_Data_1-Unnamed_11',\n",
       "  'Supplementary Data 1'),\n",
       " 'PMC7642433-41467_2020_19408_MOESM4_ESM.xlsx-Supplementary_Data_1-Unnamed_9': ('Integrated digital pathology and transcriptome analysis identifies molecular mediators of T-cell exclusion in ovarian cancer',\n",
       "  'Close proximity between cytotoxic T lymphocytes and tumour cells is required for effective immunotherapy. However, what controls the spatial distribution of T cells in the tumour microenvironment is not well understood. Here we couple digital pathology and transcriptome analysis on a large ovarian tumour cohort and develop a machine learning approach to molecularly classify and characterize tumour-immune phenotypes. Our study identifies two important hallmarks characterizing T cell excluded tumours: 1) loss of antigen presentation on tumour cells and 2) upregulation of TGFβ and activated stroma. Furthermore, we identify TGFβ as an important mediator of T cell exclusion. TGFβ reduces MHC-I expression in ovarian cancer cells in vitro. TGFβ also activates fibroblasts and induces extracellular matrix production as a potential physical barrier to hinder T cell infiltration. Our findings indicate that targeting TGFβ might be a promising strategy to overcome T cell exclusion and improve clinical benefits of cancer immunotherapy.',\n",
       "  'Supplementary_Data_1-Unnamed_9',\n",
       "  'Supplementary Data 1'),\n",
       " 'PMC5390115-mmc6.xlsx-DE_LISTS-Unnamed_24': ('A Genome-wide Analysis of Human Pluripotent Stem Cell-Derived Endothelial Cells in 2D or 3D Culture',\n",
       "  'A defined protocol for efficiently deriving endothelial cells from human pluripotent stem cells was established and vascular morphogenesis was used as a model system to understand how synthetic hydrogels influence global biological function compared with common 2D and 3D culture platforms. RNA sequencing demonstrated that gene expression profiles were similar for endothelial cells and pericytes cocultured in polyethylene glycol (PEG) hydrogels or Matrigel, while monoculture comparisons identified distinct vascular signatures for each cell type. Endothelial cells cultured on tissue-culture polystyrene adopted a proliferative phenotype compared with cells cultured on or encapsulated in PEG hydrogels. The proliferative phenotype correlated to increased FAK-ERK activity, and knockdown or inhibition of ERK signaling reduced proliferation and expression for cell-cycle genes while increasing expression for \"3D-like\" vasculature development genes. Our results provide insight into the influence of 2D and 3D culture formats on global biological processes that regulate cell function.',\n",
       "  'DE_LISTS-Unnamed_24',\n",
       "  'genes (higher expression on TCP) were predominantly associated with the cell cycle and mitosis (**Table S5**)  Table S5. Differential Gene Expression for Cells Cultured on TCP Surfaces, in 2D, or Encapsulated in PEG Hydrogels, in 3D, Related to Figures 4 and 5'),\n",
       " 'PMC6002610-mmc5.xlsx-Signature_gene_lists-Microglial_signature_genes_present_in_pig': ('Environmentally enriched pigs have transcriptional profiles consistent with neuroprotective effects and reduced microglial activity',\n",
       "  'Environmental enrichment (EE) is widely used to study the effects of external factors on brain development, function and health in rodent models, but very little is known of the effects of EE on the brain in a large animal model such as the pig. Twenty-four young pigs (aged 5 weeks at start of study, 1:1 male: female ratio) were housed in environmentally enriched (EE) pens and provided with additional enrichment stimulation (a bag filled with straw) once daily. Litter, weight and sex matched controls n= (24) were housed in barren (B) conditions. Behaviour was recorded on alternate days from study day 10. After 21 days, RNA-sequencing of the frontal cortex of male piglets culled one hour after the enrichment stimulation, but not those at 4 h after stimulation, showed upregulation of genes involved in neuronal activity and synaptic plasticity in the EE compared to the B condition. This result is mirrored in the behavioural response to the stimulation which showed a peak in activity around the 1 h time-point. By contrast, EE piglets displayed a signature consistent with a relative decrease in microglial activity compared to those in the B condition. These results confirm those from rodents, suggesting that EE may also confer neuronal health benefits in large mammal models, through a potential relative reduction in neuroinflammatory process and increase in neuroprotection driven by an enrichment-induced increase in behavioural activity.',\n",
       "  'Signature_gene_lists-Microglial_signature_genes_present_in_pig',\n",
       "  'None here'),\n",
       " 'PMC5390115-mmc3.xlsx-DE_Lists_Exp_1-HUVECs_HUV1_vs_H1_ES_cells': ('A Genome-wide Analysis of Human Pluripotent Stem Cell-Derived Endothelial Cells in 2D or 3D Culture',\n",
       "  'A defined protocol for efficiently deriving endothelial cells from human pluripotent stem cells was established and vascular morphogenesis was used as a model system to understand how synthetic hydrogels influence global biological function compared with common 2D and 3D culture platforms. RNA sequencing demonstrated that gene expression profiles were similar for endothelial cells and pericytes cocultured in polyethylene glycol (PEG) hydrogels or Matrigel, while monoculture comparisons identified distinct vascular signatures for each cell type. Endothelial cells cultured on tissue-culture polystyrene adopted a proliferative phenotype compared with cells cultured on or encapsulated in PEG hydrogels. The proliferative phenotype correlated to increased FAK-ERK activity, and knockdown or inhibition of ERK signaling reduced proliferation and expression for cell-cycle genes while increasing expression for \"3D-like\" vasculature development genes. Our results provide insight into the influence of 2D and 3D culture formats on global biological processes that regulate cell function.',\n",
       "  'DE_Lists_Exp_1-HUVECs_HUV1_vs_H1_ES_cells',\n",
       "  'al., 2002, Huang et al., 2008 categories relevant to endothelial phenotypes (Figure 1H and **Table S2**)  Table S2. Differential Gene Expression for H1-ECs, HUVECs, and Pericytes Compared with H1 ESCs, Related to Figures 1 and 2'),\n",
       " 'PMC10777062-mmc4.xlsx-Brazil_TBDM_cor-Var2': ('The sound of silent RNA in tuberculosis and the lncRNA role on infection',\n",
       "  'Tuberculosis (TB) is one of the leading causes of death worldwide, and Diabetes Mellitus is one of the major comorbidities (TB/DM) associated with the disease. A total of 103 differentially expressed ncRNAs have been identified in the TB and TB/DM comparisons. A machine learning algorithm was employed to identify the most informative lncRNAs: ADM-DT, LINC02009, LINC02471, SOX2-OT, and GK-AS1. These lncRNAs presented substantial accuracy in classifying TB from HC (AUCs >0.85) and TB/DM from HC (AUCs >0.90) in the other three countries. Genes with significant correlations with the five lncRNAs enriched common pathways in Brazil and India for both TB and TB/DM. This suggests that lncRNAs play an important role in the regulation of genes related to the TB immune response.',\n",
       "  'Brazil_TBDM_cor-Var2',\n",
       "  'genes. The correlated genes and their respective rho and p values are available in **supplemental material S2**  Supplemental material 2. Correlated genes results.xlsx file containing detailed data regarding the correlation analysis performed between the most informative lncRNAs and all genes'),\n",
       " 'PMC10849790-CPR-57-e13545-s005.xlsx-myoblast1_vs_myocyteSJ_expressi-gene_short_name': ('Bulk and single-cell alternative splicing analyses reveal roles of TRA2B in myogenic differentiation',\n",
       "  'Alternative splicing (AS) disruption has been linked to disorders of muscle development, as well as muscular atrophy. However, the precise changes in AS patterns that occur during myogenesis are not well understood. Here, we employed isoform long-reads RNA-seq (Iso-seq) and single-cell RNA-seq (scRNA-seq) to investigate the AS landscape during myogenesis. Our Iso-seq data identified 61,146 full-length isoforms representing 11,682 expressed genes, of which over 52% were novel. We identified 38,022 AS events, with most of these events altering coding sequences and exhibiting stage-specific splicing patterns. We identified AS dynamics in different types of muscle cells through scRNA-seq analysis, revealing genes essential for the contractile muscle system and cytoskeleton that undergo differential splicing across cell types. Gene-splicing analysis demonstrated that AS acts as a regulator, independent of changes in overall gene expression. Two isoforms of splicing factor TRA2B play distinct roles in myogenic differentiation by triggering AS of TGFBR2 to regulate canonical TGF-β signalling cascades differently. Our study provides a valuable transcriptome resource for myogenesis and reveals the complexity of AS and its regulation during myogenesis.',\n",
       "  'myoblast1_vs_myocyteSJ_expressi-gene_short_name',\n",
       "  'None here'),\n",
       " 'PMC5844691-elife-30496-supp1.xls-All_GeneLists-Unnamed_6': ('The transcription factors Runx3 and ThPOK cross-regulate acquisition of cytotoxic function by human Th1 lymphocytes',\n",
       "  'Cytotoxic CD4 (CD4 CTX ) T cells are emerging as an important component of antiviral and antitumor immunity, but the molecular basis of their development remains poorly understood. In the context of human cytomegalovirus infection, a significant proportion of CD4 T cells displays cytotoxic functions. We observed that the transcriptional program of these cells was enriched in CD8 T cell lineage genes despite the absence of ThPOK downregulation. We further show that establishment of CD4 CTX -specific transcriptional and epigenetic programs occurred in a stepwise fashion along the Th1-differentiation pathway. In vitro, prolonged activation of naive CD4 T cells in presence of Th1 polarizing cytokines led to the acquisition of perforin-dependent cytotoxic activity. This process was dependent on the Th1 transcription factor Runx3 and was limited by the sustained expression of ThPOK. This work elucidates the molecular program of human CD4 CTX T cells and identifies potential targets for immunotherapy against viral infections and cancer.',\n",
       "  'All_GeneLists-Unnamed_6',\n",
       "  'identified and their enrichment in naive CD8 and CD4 T cell transcriptomes was assessed (**Supplementary file 1**)  Lists of genes included in the 12 GeneSets obtained from CD4 versus CD8 T cell comparison (Figure 2) and naive CD4 T cell versus CMTh1 cell versus CD4CTX T cell comparison (Figure 4).Genes expressed at higher level by the first listed subset as compared to the subset indicated between brackets were identified using the min/max method. Genes are ranked according to mean log2 fold change calculated using the Limma package in R'),\n",
       " 'PMC11218163-13014_2024_2480_MOESM1_ESM.xlsx-DEG-Figure_5_B': ('Transcriptomic response of prostate cancer cells to carbon ion and photon irradiation with focus on androgen receptor and TP53 signaling',\n",
       "  'Background: Radiotherapy is essential in the treatment of prostate cancer. An alternative to conventional photon radiotherapy is the application of carbon ions, which provide a superior intratumoral dose distribution and less induced damage to adjacent healthy tissue. A common characteristic of prostate cancer cells is their dependence on androgens which is exploited therapeutically by androgen deprivation therapy in the advanced prostate cancer stage. Here, we aimed to analyze the transcriptomic response of prostate cancer cells to irradiation by photons in comparison to carbon ions, focusing on DNA damage, DNA repair and androgen receptor signaling. Methods: Prostate cancer cell lines LNCaP (functional TP53 and androgen receptor signaling) and DU145 (dysfunctional TP53 and androgen receptor signaling) were irradiated by photons or carbon ions and the subsequent DNA damage was assessed by immuno-cytofluorescence. Furthermore, the cells were treated with an androgen-receptor agonist. The effects of irradiation and androgen treatment on the gene regulation and the transcriptome were investigated by RT-qPCR and RNA sequencing, followed by bioinformatic analysis. Results: Following photon or carbon ion irradiation, both LNCaP and DU145 cells showed a dose-dependent amount of visible DNA damage that decreased over time, indicating occurring DNA repair. In terms of gene regulation, mRNAs involved in the TP53-dependent DNA damage response were significantly upregulated by photons and carbon ions in LNCaP but not in DU145 cells, which generally showed low levels of gene regulation after irradiation. Both LNCaP and DU145 cells responded to photons and carbon ions by downregulation of genes involved in DNA repair and cell cycle, partially resembling the transcriptome response to the applied androgen receptor agonist. Neither photons nor carbon ions significantly affected canonical androgen receptor-dependent gene regulation. Furthermore, certain genes that were specifically regulated by either photon or carbon ion irradiation were identified. Conclusion: Photon and carbon ion irradiation showed a significant congruence in terms of induced signaling pathways and transcriptomic responses. These responses were strongly impacted by the TP53 status. Nevertheless, irradiation mode-dependent distinct gene regulations with undefined implication for radiotherapy outcome were revealed. Androgen receptor signaling and irradiations shared regulation of certain genes with respect to DNA-repair and cell-cycle.',\n",
       "  'DEG-Figure_5_B',\n",
       "  'None here'),\n",
       " 'PMC8765667-pbio.3001065.s013.xlsx-DEG_Summary-Upregulated_DEGs_Q_0_05': ('Elevated temperature inhibits SARS-CoV-2 replication in respiratory epithelium independently of IFN-mediated innate immune defenses',\n",
       "  'The pandemic spread of Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2), the etiological agent of Coronavirus Disease 2019 (COVID-19), represents an ongoing international health crisis. A key symptom of SARS-CoV-2 infection is the onset of fever, with a hyperthermic temperature range of 38 to 41°C. Fever is an evolutionarily conserved host response to microbial infection that can influence the outcome of viral pathogenicity and regulation of host innate and adaptive immune responses. However, it remains to be determined what effect elevated temperature has on SARS-CoV-2 replication. Utilizing a three-dimensional (3D) air-liquid interface (ALI) model that closely mimics the natural tissue physiology of SARS-CoV-2 infection in the respiratory airway, we identify tissue temperature to play an important role in the regulation of SARS-CoV-2 infection. Respiratory tissue incubated at 40°C remained permissive to SARS-CoV-2 entry but refractory to viral transcription, leading to significantly reduced levels of viral RNA replication and apical shedding of infectious virus. We identify tissue temperature to play an important role in the differential regulation of epithelial host responses to SARS-CoV-2 infection that impact upon multiple pathways, including intracellular immune regulation, without disruption to general transcription or epithelium integrity. We present the first evidence that febrile temperatures associated with COVID-19 inhibit SARS-CoV-2 replication in respiratory epithelia. Our data identify an important role for tissue temperature in the epithelial restriction of SARS-CoV-2 independently of canonical interferon (IFN)-mediated antiviral immune defenses.',\n",
       "  'DEG_Summary-Upregulated_DEGs_Q_0_05',\n",
       "  '< 0.05 expression for each paired experimental condition analyzed (Fig 5A and 5B and **S6 Data**)  Comparative DEG analysis of RNA-Seq data derived from Mck-treated or SCV2-infected respiratory tissues at 37 and 40°C derived from S1, S3, S4, and S5 data analysis.DEG, differentially expressed gene; Mck, mock; RNA-Seq, RNA sequencing; SCV2, SARS-CoV-2.(XLSX)  Click here for additional data file'),\n",
       " 'PMC6042247-NIHMS966841-supplement-Sup_Table_2.xlsx-GOBLET_xls-PROBE': ('Parasitic helminths induce fetal-like reversion in the intestinal stem cell niche',\n",
       "  'Epithelial surfaces form critical barriers to the outside world and are continuously renewed by adult stem cells 1 . Whereas dynamics of epithelial stem cells during homeostasis are increasingly well understood, how stem cells are redirected from a tissue-maintenance program to initiate repair after injury remains unclear. Here we examined infection by Heligmosomoides polygyrus, a co-evolved pathosymbiont of mice, to assess the epithelial response to disruption of the mucosal barrier. H. polygyrus disrupts tissue integrity by penetrating the duodenal mucosa, where it develops while surrounded by a multicellular granulomatous infiltrate 2 . Crypts overlying larvae-associated granulomas did not express intestinal stem cell markers, including Lgr5 3 , in spite of continued epithelial proliferation. Granuloma-associated Lgr5 - crypt epithelium activated an interferon-gamma (IFN-γ)-dependent transcriptional program, highlighted by Sca-1 expression, and IFN-γ-producing immune cells were found in granulomas. A similar epithelial response accompanied systemic activation of immune cells, intestinal irradiation, or ablation of Lgr5 + intestinal stem cells. When cultured in vitro, granuloma-associated crypt cells formed spheroids similar to those formed by fetal epithelium, and a sub-population of H. polygyrus-induced cells activated a fetal-like transcriptional program, demonstrating that adult intestinal tissues can repurpose aspects of fetal development. Therefore, re-initiation of the developmental program represents a fundamental mechanism by which the intestinal crypt can remodel itself to sustain function after injury.',\n",
       "  'GOBLET_xls-PROBE',\n",
       "  'suite of ISC signature genes9 was down-regulated in GCs (Extended Data Fig. 2c–d and **Supplementary Table 2**)'),\n",
       " 'PMC7649975-mmc7.xlsx-Mitochondrial-Unnamed_0': ('Defective Sec61α1 underlies a novel cause of autosomal dominant severe congenital neutropenia',\n",
       "  \"Background: The molecular cause of severe congenital neutropenia (SCN) is unknown in 30% to 50% of patients. SEC61A1 encodes the α-subunit of the Sec61 complex, which governs endoplasmic reticulum protein transport and passive calcium leakage. Recently, mutations in SEC61A1 were reported to be pathogenic in common variable immunodeficiency and glomerulocystic kidney disease. Objective: Our aim was to expand the spectrum of SEC61A1-mediated disease to include autosomal dominant SCN. Methods: Whole exome sequencing findings were validated, and reported mutations were compared by Western blotting, Ca2 + flux assays, differentiation of transduced HL-60 cells, in vitro differentiation of primary CD34 cells, quantitative PCR for unfolded protein response (UPR) genes, and single-cell RNA sequencing on whole bone marrow. Results: We identified a novel de novo missense mutation in SEC61A1 (c.A275G;p.Q92R) in a patient with SCN who was born to nonconsanguineous Belgian parents. The mutation results in diminished protein expression, disturbed protein translocation, and an increase in calcium leakage from the endoplasmic reticulum. In vitro differentiation of CD34 + cells recapitulated the patient's clinical arrest in granulopoiesis. The impact of Q92R-Sec61α1 on neutrophil maturation was validated by using HL-60 cells, in which transduction reduced differentiation into CD11b + CD16 + cells. A potential mechanism for this defect is the uncontrolled initiation of the unfolded protein stress response, with single-cell analysis of primary bone marrow revealing perturbed UPR in myeloid precursors and in vitro differentiation of primary CD34 + cells revealing upregulation of CCAAT/enhancer-binding protein homologous protein and immunoglobulin heavy chain binding protein UPR-response genes. Conclusion: Specific mutations in SEC61A1 cause SCN through dysregulation of the UPR.\",\n",
       "  'Mitochondrial-Unnamed_0',\n",
       "  'Table E6'),\n",
       " 'PMC3690947-NIHMS455856-supplement-3.xls-GN_upregulated-Unnamed_1': ('Identification of transcriptional regulators in the mouse immune system',\n",
       "  'The differentiation of hematopoietic stem cells into cells of the immune system has been studied extensively in mammals, but the transcriptional circuitry that controls it is still only partially understood. Here, the Immunological Genome Project gene-expression profiles across mouse immune lineages allowed us to systematically analyze these circuits. To analyze this data set we developed Ontogenet, an algorithm for reconstructing lineage-specific regulation from gene-expression profiles across lineages. Using Ontogenet, we found differentiation stage-specific regulators of mouse hematopoiesis and identified many known hematopoietic regulators and 175 previously unknown candidate regulators, as well as their target genes and the cell types in which they act. Among the previously unknown regulators, we emphasize the role of ETV5 in the differentiation of γδ T cells. As the transcriptional programs of human and mouse cells are highly conserved, it is likely that many lessons learned from the mouse model apply to humans.',\n",
       "  'GN_upregulated-Unnamed_1',\n",
       "  'under-expressed genes for each of the main eleven lineages, compared to all other lineages (**Supplementary Table 2**)'),\n",
       " 'PMC9220854-jciinsight-7-153045-s290.xlsx-Differentiated_epithelial_cells-Goblet': ('MTG16 regulates colonic epithelial differentiation, colitis, and tumorigenesis by repressing E protein transcription factors',\n",
       "  'Aberrant epithelial differentiation and regeneration contribute to colon pathologies, including inflammatory bowel disease (IBD) and colitis-associated cancer (CAC). Myeloid translocation gene 16 (MTG16, also known as CBFA2T3) is a transcriptional corepressor expressed in the colonic epithelium. MTG16 deficiency in mice exacerbates colitis and increases tumor burden in CAC, though the underlying mechanisms remain unclear. Here, we identified MTG16 as a central mediator of epithelial differentiation, promoting goblet and restraining enteroendocrine cell development in homeostasis and enabling regeneration following dextran sulfate sodium-induced (DSS-induced) colitis. Transcriptomic analyses implicated increased Ephrussi box-binding transcription factor (E protein) activity in MTG16-deficient colon crypts. Using a mouse model with a point mutation that attenuates MTG16:E protein interactions (Mtg16P209T), we showed that MTG16 exerts control over colonic epithelial differentiation and regeneration by repressing E protein-mediated transcription. Mimicking murine colitis, MTG16 expression was increased in biopsies from patients with active IBD compared with unaffected controls. Finally, uncoupling MTG16:E protein interactions partially phenocopied the enhanced tumorigenicity of Mtg16-/- colon in the azoxymethane/DSS-induced model of CAC, indicating that MTG16 protects from tumorigenesis through additional mechanisms. Collectively, our results demonstrate that MTG16, via its repression of E protein targets, is a key regulator of cell fate decisions during colon homeostasis, colitis, and cancer.',\n",
       "  'Differentiated_epithelial_cells-Goblet',\n",
       "  'Supplemental table 4'),\n",
       " 'PMC10835465-mmc6.xlsx-Cluster11-gene': ('Association of HBsAg levels with differential gene expression in NK, CD8 T, and memory B cells in treated patients with chronic HBV',\n",
       "  'Background & aims: HBsAg secretion may impact immune responses to chronic HBV infection. Thus, therapeutic approaches to suppress HBsAg production are being investigated. Our study aims to examine the immunomodulatory effects of high and low levels of circulating HBsAg and thereby improve our understanding of anti-HBV immunity. Methods: An optimized 10x Genomics single-cell RNA sequencing workflow was applied to blood samples and liver fine-needle aspirates from 18 patients undergoing tenofovir/entecavir (NUC) treatment for chronic HBV infection. They were categorized based on their HBsAg levels: high (920-12,447 IU/ml) or low (1-100 IU/ml). Cluster frequencies, differential gene expression, and phenotypes were analyzed. Results: In the blood of HBV-infected patients on NUC, the proportion of KLRC2 + \"adaptive\" natural killer (NK) cells was significantly lower in the HBsAg-high group and, remarkably, both KLRC2 + NK and KLRG1+ CD8 T cells display enrichment of lymphocyte activation-associated gene sets in the HBsAg-low group. High levels of HBsAg were associated with mild immune activation in the liver. However, no suppression of liver-resident CXCR6+ NCAM1+ NK or CXCR6+ CD69+ CD8 T cells was detected, while memory B cells showed signs of activation in both the blood and liver. Conclusions: Among NUC-treated patients, we observed a minimal impact of HBsAg on leukocyte populations in the blood and liver. However, for the first time, we found that HBsAg has distinct effects, restricted to NK-, CD8 T-, and memory B-cell subsets, in the blood and liver. Our findings are highly relevant for current clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Impact and implications: This study provides unique insight into the impact of HBsAg on gene expression levels of immune cell subsets in the blood and liver, particularly in the context of NUC-treated chronic HBV infection. It holds significant relevance for current and future clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Our findings raise questions about the effectiveness of such treatment strategies and challenge the previously hypothesized immunomodulatory effects of HBsAg on immune responses against HBV.',\n",
       "  'Cluster11-gene',\n",
       "  'None here'),\n",
       " 'PMC11026436-41467_2024_47647_MOESM5_ESM.xlsx-Apical_cell-gene': ('An esophagus cell atlas reveals dynamic rewiring during active eosinophilic esophagitis and remission',\n",
       "  'Coordinated cell interactions within the esophagus maintain homeostasis, and disruption can lead to eosinophilic esophagitis (EoE), a chronic inflammatory disease with poorly understood pathogenesis. We profile 421,312 individual cells from the esophageal mucosa of 7 healthy and 15 EoE participants, revealing 60 cell subsets and functional alterations in cell states, compositions, and interactions that highlight previously unclear features of EoE. Active disease displays enrichment of ALOX15 + macrophages, PRDM16 + dendritic cells expressing the EoE risk gene ATP10A, and cycling mast cells, with concomitant reduction of T H 17 cells. Ligand-receptor expression uncovers eosinophil recruitment programs, increased fibroblast interactions in disease, and IL-9 + IL-4 + IL-13 + T H 2 and endothelial cells as potential mast cell interactors. Resolution of inflammation-associated signatures includes mast and CD4 + T RM cell contraction and cell type-specific downregulation of eosinophil chemoattractant, growth, and survival factors. These cellular alterations in EoE and remission advance our understanding of eosinophilic inflammation and opportunities for therapeutic intervention.',\n",
       "  'Apical_cell-gene',\n",
       "  'None here'),\n",
       " 'PMC8020120-mmc15.xlsx-DEGs-Conserved_Up_699_genes': ('Secondary analysis of transcriptomes of SARS-CoV-2 infection models to characterize COVID-19',\n",
       "  'Standard transcriptomic analyses alone have limited power in capturing the molecular mechanisms driving disease pathophysiology and outcomes. To overcome this, unsupervised network analyses are used to identify clusters of genes that can be associated with distinct molecular mechanisms and outcomes for a disease. In this study, we developed an integrated network analysis framework that integrates transcriptional signatures from multiple model systems with protein-protein interaction data to find gene modules. Through a meta-analysis of different enriched features from these gene modules, we extract communities of highly interconnected features. These clusters of higher-order features, working as a multifeatured machine, enable collective assessment of their contribution for disease or phenotype characterization. We show the utility of this workflow using transcriptomics data from three different models of SARS-CoV-2 infection and identify several pathways and biological processes that could enable understanding or hypothesizing molecular signatures inducing pathophysiological changes, risks, or sequelae of COVID-19.',\n",
       "  'DEGs-Conserved_Up_699_genes',\n",
       "  'described previously for SARS-CoV-2. We identified 68 modules that had at least five genes (**Table S14**)  Table S14. SARS-CoV-1 consensus DEG and human interaction mapSARS-CoV-1 human interaction map constructed from a joint interactome of SARS-CoV-1 consensus signature and SARS-CoV-1-human interaction map'),\n",
       " 'PMC9023741-Table4.xlsx-DEG_noninf_vs_con-Unnamed_0': ('Digital Spatial Profiling Reveals Functional Shift of Enterochromaffin Cell in Patients With Ulcerative Colitis',\n",
       "  \"As a major component of the enteroendocrine system, enterochromaffin (EC) cells play a key role in ulcerative colitis (UC). However, the scarcity of EC cells has limited the investigation of their function. In this study, we applied digital spatial profiling to acquire transcriptomic data for EC cells and other epithelial cells from colonoscopic biopsy samples from eight patients with UC and seven healthy controls. Differential expression analysis, gene set enrichment analysis, and weighted gene coexpression network analysis were performed to identify differentially expressed genes and pathways and coexpression networks. Results were validated using an online dataset obtained by single-cell RNA sequencing, along with immunofluorescence staining and quantitative real-time PCR. In healthy participants, 10 genes were significantly enriched in EC cells, functionally concentrated in protein and bioamine synthesis. A coexpression network containing 17 hub genes, including TPH1 , CHGA , and GCLC , was identified in EC cells. In patients with UC, EC cells gained increased capacity for protein synthesis, along with novel immunological functions such as antigen processing and presentation, whereas chemical sensation was downregulated. The specific expression of CHGB and RGS2 in EC cells was confirmed by immunofluorescence staining. Our results illuminate the transcriptional signatures of EC cells in the human colon. EC cells' newly observed functional shift from sensation to secretion and immunity indicates their pivotal role in UC.\",\n",
       "  'DEG_noninf_vs_con-Unnamed_0',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC10644098-BLOOD_BLD-2023-019721-mmc15.xlsx-all_clusterlate_stage_pat_ctr_W-Unnamed_0': ('IL-7 receptor signaling drives human B-cell progenitor differentiation and expansion',\n",
       "  'Although absence of interleukin-7 (IL-7) signaling completely abrogates T and B lymphopoiesis in mice, patients with severe combined immunodeficiency caused by mutations in the IL-7 receptor α chain (IL-7Rα) still generate peripheral blood B cells. Consequently, human B lymphopoiesis has been thought to be independent of IL-7 signaling. Using flow cytometric analysis and single-cell RNA sequencing of bone marrow samples from healthy controls and patients who are IL-7Rα deficient, in combination with in vitro modeling of human B-cell differentiation, we demonstrate that IL-7R signaling plays a crucial role in human B lymphopoiesis. IL-7 drives proliferation and expansion of early B-cell progenitors but not of pre-BII large cells and has a limited role in the prevention of cell death. Furthermore, IL-7 guides cell fate decisions by enhancing the expression of BACH2, EBF1, and PAX5, which jointly orchestrate the specification and commitment of early B-cell progenitors. In line with this observation, early B-cell progenitors of patients with IL-7Rα deficiency still expressed myeloid-specific genes. Collectively, our results unveil a previously unknown role for IL-7 signaling in promoting the B-lymphoid fate and expanding early human B-cell progenitors while defining important differences between mice and humans. Our results have implications for hematopoietic stem cell transplantation strategies in patients with T- B+ severe combined immunodeficiency and provide insights into the role of IL-7R signaling in leukemogenesis.',\n",
       "  'all_clusterlate_stage_pat_ctr_W-Unnamed_0',\n",
       "  'Supplemental Table 14'),\n",
       " 'PMC7396846-EMBJ-39-e103790-s003.xlsx-Table_EV1-Table_EV1_-_Gene_signatures_used_to_calculate_ssGSEA_enrichment_scores': ('Microglia promote glioblastoma via mTOR-mediated immunosuppression of the tumour microenvironment',\n",
       "  'Tumour-associated microglia/macrophages (TAM) are the most numerous non-neoplastic populations in the tumour microenvironment in glioblastoma multiforme (GBM), the most common malignant brain tumour in adulthood. The mTOR pathway, an important regulator of cell survival/proliferation, is upregulated in GBM, but little is known about the potential role of this pathway in TAM. Here, we show that GBM-initiating cells induce mTOR signalling in the microglia but not bone marrow-derived macrophages in both in vitro and in vivo GBM mouse models. mTOR-dependent regulation of STAT3 and NF-κB activity promotes an immunosuppressive microglial phenotype. This hinders effector T-cell infiltration, proliferation and immune reactivity, thereby contributing to tumour immune evasion and promoting tumour growth in mouse models. The translational value of our results is demonstrated in whole transcriptome datasets of human GBM and in a novel in vitro model, whereby expanded-potential stem cells (EPSC)-derived microglia-like cells are conditioned by syngeneic patient-derived GBM-initiating cells. These results raise the possibility that microglia could be the primary target of mTOR inhibition, rather than the intrinsic tumour cells in GBM.',\n",
       "  'Table_EV1-Table_EV1_-_Gene_signatures_used_to_calculate_ssGSEA_enrichment_scores',\n",
       "  'the enrichment score of specific immune pathways identified in bulk tumour tissue (Fig 5A, **Table EV1**)  Table EV1  Click here for additional data file'),\n",
       " 'PMC9338292-41467_2022_32098_MOESM13_ESM.xlsx-Fig3k-Overlap': ('CDK9 activity switch associated with AFF1 and HEXIM1 controls differentiation initiation from epidermal progenitors',\n",
       "  'Progenitors in epithelial tissues, such as human skin epidermis, continuously make fate decisions between self-renewal and differentiation. Here we show that the Super Elongation Complex (SEC) controls progenitor fate decisions by directly suppressing a group of \"rapid response\" genes, which feature high enrichment of paused Pol II in the progenitor state and robust Pol II elongation in differentiation. SEC\\'s repressive role is dependent on the AFF1 scaffold, but not AFF4. In the progenitor state, AFF1-SEC associates with the HEXIM1-containing inactive CDK9 to suppress these rapid-response genes. A key rapid-response SEC target is ATF3, which promotes the upregulation of differentiation-activating transcription factors (GRHL3, OVOL1, PRDM1, ZNF750) to advance terminal differentiation. SEC peptidomimetic inhibitors or PKC signaling activates CDK9 and rapidly induces these transcription factors within hours in keratinocytes. Thus, our data suggest that the activity switch of SEC-associated CDK9 underlies the initial processes bifurcating progenitor fates between self-renewal and differentiation.',\n",
       "  'Fig3k-Overlap',\n",
       "  'None here'),\n",
       " 'PMC9023741-Table4.xlsx-DEG_inf_vs_noninf-Unnamed_0': ('Digital Spatial Profiling Reveals Functional Shift of Enterochromaffin Cell in Patients With Ulcerative Colitis',\n",
       "  \"As a major component of the enteroendocrine system, enterochromaffin (EC) cells play a key role in ulcerative colitis (UC). However, the scarcity of EC cells has limited the investigation of their function. In this study, we applied digital spatial profiling to acquire transcriptomic data for EC cells and other epithelial cells from colonoscopic biopsy samples from eight patients with UC and seven healthy controls. Differential expression analysis, gene set enrichment analysis, and weighted gene coexpression network analysis were performed to identify differentially expressed genes and pathways and coexpression networks. Results were validated using an online dataset obtained by single-cell RNA sequencing, along with immunofluorescence staining and quantitative real-time PCR. In healthy participants, 10 genes were significantly enriched in EC cells, functionally concentrated in protein and bioamine synthesis. A coexpression network containing 17 hub genes, including TPH1 , CHGA , and GCLC , was identified in EC cells. In patients with UC, EC cells gained increased capacity for protein synthesis, along with novel immunological functions such as antigen processing and presentation, whereas chemical sensation was downregulated. The specific expression of CHGB and RGS2 in EC cells was confirmed by immunofluorescence staining. Our results illuminate the transcriptional signatures of EC cells in the human colon. EC cells' newly observed functional shift from sensation to secretion and immunity indicates their pivotal role in UC.\",\n",
       "  'DEG_inf_vs_noninf-Unnamed_0',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC9939840-Table_3.docx-0-Table_S3_Identification_of_DEGs_in_two_ICGs_clusters_using_limma_algorithm': ('Immune checkpoints expression patterns in early-stage triple-negative breast cancer predict prognosis and remodel the tumor immune microenvironment',\n",
       "  'Background: Currently, targeting immune checkpoint molecules holds great promise for triple-negative breast cancer (TNBC). However, the expression landscape of immune checkpoint genes (ICGs) in TNBC remains largely unknown. Method: Herein, we systematically investigated the ICGs expression patterns in 422 TNBC samples. We evaluated the ICGs molecular typing based on the ICGs expression profile and explored the associations between ICGs molecular subtypes and tumor immune characteristics, clinical significance, and response to immune checkpoint inhibitors (ICIs). Results: Two ICGs clusters and two ICGs-related gene clusters were determined, which were involved in different survival outcomes, biological roles and infiltration levels of immune cells. We established a quantification system ICGs riskscore (named IRS) to assess the ICGs expression patterns for individuals. TNBC patients with lower IRS were characterized by increased immune cell infiltration, favorable clinical outcomes and high sensitivity to ICIs therapy. We also developed a nomogram model combining clinicopathological variables to predict overall survival in TNBC. Genomic feature analysis revealed that high IRS group presented an increased tumor mutation burden compared with the low IRS group. Conclusion: Collectively, dissecting the ICGs expression patterns not only provides a new insight into TNBC subtypes but also deepens the understanding of ICGs in the tumor immune microenvironment.',\n",
       "  '0-Table_S3_Identification_of_DEGs_in_two_ICGs_clusters_using_limma_algorithm',\n",
       "  'using the “limma” algorithm and 570 DEGs were recognized between two ICGs clusters ( **Supplementary Table S3**)  Click here for additional data file'),\n",
       " 'PMC10835465-mmc10.xlsx-Cluster_19-Unnamed_0': ('Association of HBsAg levels with differential gene expression in NK, CD8 T, and memory B cells in treated patients with chronic HBV',\n",
       "  'Background & aims: HBsAg secretion may impact immune responses to chronic HBV infection. Thus, therapeutic approaches to suppress HBsAg production are being investigated. Our study aims to examine the immunomodulatory effects of high and low levels of circulating HBsAg and thereby improve our understanding of anti-HBV immunity. Methods: An optimized 10x Genomics single-cell RNA sequencing workflow was applied to blood samples and liver fine-needle aspirates from 18 patients undergoing tenofovir/entecavir (NUC) treatment for chronic HBV infection. They were categorized based on their HBsAg levels: high (920-12,447 IU/ml) or low (1-100 IU/ml). Cluster frequencies, differential gene expression, and phenotypes were analyzed. Results: In the blood of HBV-infected patients on NUC, the proportion of KLRC2 + \"adaptive\" natural killer (NK) cells was significantly lower in the HBsAg-high group and, remarkably, both KLRC2 + NK and KLRG1+ CD8 T cells display enrichment of lymphocyte activation-associated gene sets in the HBsAg-low group. High levels of HBsAg were associated with mild immune activation in the liver. However, no suppression of liver-resident CXCR6+ NCAM1+ NK or CXCR6+ CD69+ CD8 T cells was detected, while memory B cells showed signs of activation in both the blood and liver. Conclusions: Among NUC-treated patients, we observed a minimal impact of HBsAg on leukocyte populations in the blood and liver. However, for the first time, we found that HBsAg has distinct effects, restricted to NK-, CD8 T-, and memory B-cell subsets, in the blood and liver. Our findings are highly relevant for current clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Impact and implications: This study provides unique insight into the impact of HBsAg on gene expression levels of immune cell subsets in the blood and liver, particularly in the context of NUC-treated chronic HBV infection. It holds significant relevance for current and future clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Our findings raise questions about the effectiveness of such treatment strategies and challenge the previously hypothesized immunomodulatory effects of HBsAg on immune responses against HBV.',\n",
       "  'Cluster_19-Unnamed_0',\n",
       "  'None here'),\n",
       " 'PMC3921226-pone.0088889.s005.xls-Table_S3-Unnamed_8': ('Functionally enigmatic genes: a case study of the brain ignorome',\n",
       "  'What proportion of genes with intense and selective expression in specific tissues, cells, or systems are still almost completely uncharacterized with respect to biological function? In what ways do these functionally enigmatic genes differ from well-studied genes? To address these two questions, we devised a computational approach that defines so-called ignoromes. As proof of principle, we extracted and analyzed a large subset of genes with intense and selective expression in brain. We find that publications associated with this set are highly skewed--the top 5% of genes absorb 70% of the relevant literature. In contrast, approximately 20% of genes have essentially no neuroscience literature. Analysis of the ignorome over the past decade demonstrates that it is stubbornly persistent, and the rapid expansion of the neuroscience literature has not had the expected effect on numbers of these genes. Surprisingly, ignorome genes do not differ from well-studied genes in terms of connectivity in coexpression networks. Nor do they differ with respect to numbers of orthologs, paralogs, or protein domains. The major distinguishing characteristic between these sets of genes is date of discovery, early discovery being associated with greater research momentum--a genomic bandwagon effect. Finally we ask to what extent massive genomic, imaging, and phenotype data sets can be used to provide high-throughput functional annotation for an entire ignorome. In a majority of cases we have been able to extract and add significant information for these neglected genes. In several cases--ELMOD1, TMEM88B, and DZANK1--we have exploited sequence polymorphisms, large phenome data sets, and reverse genetic methods to evaluate the function of ignorome genes.',\n",
       "  'Table_S3-Unnamed_8',\n",
       "  'and expression selectivity and ranges from 0 (relatively well known) to 1 (relatively unknown, **Table S4**)  (XLS)  Click here for additional data file'),\n",
       " 'PMC8323949-pcbi.1009228.s007.xlsx-HNSCC_ALL_exceptLN-X5': ('Single-cell analysis reveals the pan-cancer invasiveness-associated transition of adipose-derived stromal cells into COL11A1-expressing cancer-associated fibroblasts',\n",
       "  'During the last ten years, many research results have been referring to a particular type of cancer-associated fibroblasts associated with poor prognosis, invasiveness, metastasis and resistance to therapy in multiple cancer types, characterized by a gene expression signature with prominent presence of genes COL11A1, THBS2 and INHBA. Identifying the underlying biological mechanisms responsible for their creation may facilitate the discovery of targets for potential pan-cancer therapeutics. Using a novel computational approach for single-cell gene expression data analysis identifying the dominant cell populations in a sequence of samples from patients at various stages, we conclude that these fibroblasts are produced by a pan-cancer cellular transition originating from a particular type of adipose-derived stromal cells naturally present in the stromal vascular fraction of normal adipose tissue, having a characteristic gene expression signature. Focusing on a rich pancreatic cancer dataset, we provide a detailed description of the continuous modification of the gene expression profiles of cells as they transition from APOD-expressing adipose-derived stromal cells to COL11A1-expressing cancer-associated fibroblasts, identifying the key genes that participate in this transition. These results also provide an explanation to the well-known fact that the adipose microenvironment contributes to cancer progression.',\n",
       "  'HNSCC_ALL_exceptLN-X5',\n",
       "  'MMP11, INHBA, THBS2, COL10A1, COL8A1, FN1 and ASC associated genes (APOD, C7, PTGDS), respectively (**S4 Table**)  Differentially expressed genes among different clusters of HNSCC dataset.(XLSX)  Click here for additional data file'),\n",
       " 'PMC8323949-pcbi.1009228.s007.xlsx-X5-Unnamed_0': ('Single-cell analysis reveals the pan-cancer invasiveness-associated transition of adipose-derived stromal cells into COL11A1-expressing cancer-associated fibroblasts',\n",
       "  'During the last ten years, many research results have been referring to a particular type of cancer-associated fibroblasts associated with poor prognosis, invasiveness, metastasis and resistance to therapy in multiple cancer types, characterized by a gene expression signature with prominent presence of genes COL11A1, THBS2 and INHBA. Identifying the underlying biological mechanisms responsible for their creation may facilitate the discovery of targets for potential pan-cancer therapeutics. Using a novel computational approach for single-cell gene expression data analysis identifying the dominant cell populations in a sequence of samples from patients at various stages, we conclude that these fibroblasts are produced by a pan-cancer cellular transition originating from a particular type of adipose-derived stromal cells naturally present in the stromal vascular fraction of normal adipose tissue, having a characteristic gene expression signature. Focusing on a rich pancreatic cancer dataset, we provide a detailed description of the continuous modification of the gene expression profiles of cells as they transition from APOD-expressing adipose-derived stromal cells to COL11A1-expressing cancer-associated fibroblasts, identifying the key genes that participate in this transition. These results also provide an explanation to the well-known fact that the adipose microenvironment contributes to cancer progression.',\n",
       "  'X5-Unnamed_0',\n",
       "  'MMP11, INHBA, THBS2, COL10A1, COL8A1, FN1 and ASC associated genes (APOD, C7, PTGDS), respectively (**S4 Table**)  Differentially expressed genes among different clusters of HNSCC dataset.(XLSX)  Click here for additional data file'),\n",
       " 'PMC7362484-13073_2020_760_MOESM7_ESM.xlsx-Genes_regulated_by_BAP1-DNAJA1': ('ASXL3 bridges BRD4 to BAP1 complex and governs enhancer activity in small cell lung cancer',\n",
       "  \"Background: Small cell lung cancer (SCLC) is a more aggressive subtype of lung cancer that often results in rapid tumor growth, early metastasis, and acquired therapeutic resistance. Consequently, such phenotypical characteristics of SCLC set limitations on viable procedural options, making it difficult to develop both screenings and effective treatments. In this study, we examine a novel mechanistic insight in SCLC cells that could potentially provide a more sensitive therapeutic alternative for SCLC patients. Methods: Biochemistry studies, including size exclusion chromatography, mass spectrometry, and western blot analysis, were conducted to determine the protein-protein interaction between additional sex combs-like protein 3 (ASXL3) and bromodomain-containing protein 4 (BRD4). Genomic studies, including chromatin immunoprecipitation sequencing (ChIP-seq), RNA sequencing, and genome-wide analysis, were performed in both human and mouse SCLC cells to determine the dynamic relationship between BRD4/ASXL3/BAP1 epigenetic axis in chromatin binding and its effects on transcriptional activity. Results: We report a critical link between BAP1 complex and BRD4, which is bridged by the physical interaction between ASXL3 and BRD4 in an SCLC subtype (SCLC-A), which expresses a high level of ASCL1. We further showed that ASXL3 functions as an adaptor protein, which directly interacts with BRD4's extra-terminal (ET) domain via a novel BRD4 binding motif (BBM), and maintains chromatin occupancy of BRD4 to active enhancers. Genetic depletion of ASXL3 results in a genome-wide reduction of histone H3K27Ac levels and BRD4-dependent gene expression in SCLC. Pharmacologically induced inhibition with BET-specific chemical degrader (dBET6) selectively inhibits cell proliferation of a subtype of SCLC that is characterized with high expression of ASXL3. Conclusions: Collectively, this study provides a mechanistic insight into the oncogenic function of BRD4/ASXL3/BAP1 epigenetic axis at active chromatin enhancers in SCLC-A subtype, as well as a potential new therapeutic option that could become more effective in treating SCLC patients with a biomarker of ASXL3-highly expressed SCLC cells.\",\n",
       "  'Genes_regulated_by_BAP1-DNAJA1',\n",
       "  'found a striking overlap between BAP1 and ASXL3 target genes (Fig. 6b, Additional file **7**)  Additional file 7: Table S6. Genes that are co-regulated by BAP1 and ASXL3'),\n",
       " 'PMC11089003-41593_2024_1600_MOESM4_ESM.xlsx-original_lists-Olah2020': (\"Xenografted human microglia display diverse transcriptomic states in response to Alzheimer's disease-related amyloid-β pathology\",\n",
       "  \"Microglia are central players in Alzheimer's disease pathology but analyzing microglial states in human brain samples is challenging due to genetic diversity, postmortem delay and admixture of pathologies. To circumvent these issues, here we generated 138,577 single-cell expression profiles of human stem cell-derived microglia xenotransplanted in the brain of the App NL-G-F model of amyloid pathology and wild-type controls. Xenografted human microglia adopt a disease-associated profile similar to that seen in mouse microglia, but display a more pronounced human leukocyte antigen or HLA state, likely related to antigen presentation in response to amyloid plaques. The human microglial response also involves a pro-inflammatory cytokine/chemokine cytokine response microglia or CRM response to oligomeric Aβ oligomers. Genetic deletion of TREM2 or APOE as well as APOE polymorphisms and TREM2 R47H expression in the transplanted microglia modulate these responses differentially. The expression of other Alzheimer's disease risk genes is differentially regulated across the distinct cell states elicited in response to amyloid pathology. Thus, we have identified multiple transcriptomic cell states adopted by human microglia in a multipronged response to Alzheimer's disease-related pathology, which should be taken into account in translational studies.\",\n",
       "  'original_lists-Olah2020',\n",
       "  'None here'),\n",
       " 'PMC11089003-41593_2024_1600_MOESM4_ESM.xlsx-overlap_human-Olah': (\"Xenografted human microglia display diverse transcriptomic states in response to Alzheimer's disease-related amyloid-β pathology\",\n",
       "  \"Microglia are central players in Alzheimer's disease pathology but analyzing microglial states in human brain samples is challenging due to genetic diversity, postmortem delay and admixture of pathologies. To circumvent these issues, here we generated 138,577 single-cell expression profiles of human stem cell-derived microglia xenotransplanted in the brain of the App NL-G-F model of amyloid pathology and wild-type controls. Xenografted human microglia adopt a disease-associated profile similar to that seen in mouse microglia, but display a more pronounced human leukocyte antigen or HLA state, likely related to antigen presentation in response to amyloid plaques. The human microglial response also involves a pro-inflammatory cytokine/chemokine cytokine response microglia or CRM response to oligomeric Aβ oligomers. Genetic deletion of TREM2 or APOE as well as APOE polymorphisms and TREM2 R47H expression in the transplanted microglia modulate these responses differentially. The expression of other Alzheimer's disease risk genes is differentially regulated across the distinct cell states elicited in response to amyloid pathology. Thus, we have identified multiple transcriptomic cell states adopted by human microglia in a multipronged response to Alzheimer's disease-related pathology, which should be taken into account in translational studies.\",\n",
       "  'overlap_human-Olah',\n",
       "  'None here'),\n",
       " 'PMC5622284-DataSheet2.XLSX-Gene_lists_X-rays-X-rays_1': ('Differential Impact of Single-Dose Fe Ion and X-Ray Irradiation on Endothelial Cell Transcriptomic and Proteomic Responses',\n",
       "  'Background and Purpose: Radiotherapy is an essential tool for cancer treatment. In order to spare normal tissues and to reduce the risk of normal tissue complications, particle therapy is a method of choice. Although a large part of healthy tissues can be spared due to improved depth dose characteristics, little is known about the biological and molecular mechanisms altered after particle irradiation in healthy tissues. Elucidation of these effects is also required in the context of long term space flights, as particle radiation is the main contributor to the radiation effects observed in space. Endothelial cells (EC), forming the inner layer of all vascular structures, are especially sensitive to irradiation and, if damaged, contribute to radiation-induced cardiovascular disease. Materials and Methods: Transcriptomics, proteomics and cytokine analyses were used to compare the response of ECs irradiated or not with a single 2 Gy dose of X-rays or Fe ions measured one and 7 days post-irradiation. To support the observed inflammatory effects, monocyte adhesion on ECs was also assessed. Results: Experimental data indicate time- and radiation quality-dependent changes of the EC response to irradiation. The irradiation impact was more pronounced and longer lasting for Fe ions than for X-rays. Both radiation qualities decreased the expression of genes involved in cell-cell adhesion and enhanced the expression of proteins involved in caveolar mediated endocytosis signaling. Endothelial inflammation and adhesiveness were increased with X-rays, but decreased after Fe ion exposure. Conclusions: Fe ions induce pro-atherosclerotic processes in ECs that are different in nature and kinetics than those induced by X-rays, highlighting radiation quality-dependent differences which can be linked to the induction and progression of cardiovascular diseases (CVD). Our findings give a better understanding of the underlying processes triggered by particle irradiation in ECs, a crucial aspect for the development of protective measures for cancer patients undergoing particle therapy and for astronauts in space.',\n",
       "  'Gene_lists_X-rays-X-rays_1',\n",
       "  'enrichment analysis revealed that, 1 day after exposure to a single X-ray dose of **2**  Click here for additional data file'),\n",
       " 'PMC10835465-mmc10.xlsx-Cluster_19-gene': ('Association of HBsAg levels with differential gene expression in NK, CD8 T, and memory B cells in treated patients with chronic HBV',\n",
       "  'Background & aims: HBsAg secretion may impact immune responses to chronic HBV infection. Thus, therapeutic approaches to suppress HBsAg production are being investigated. Our study aims to examine the immunomodulatory effects of high and low levels of circulating HBsAg and thereby improve our understanding of anti-HBV immunity. Methods: An optimized 10x Genomics single-cell RNA sequencing workflow was applied to blood samples and liver fine-needle aspirates from 18 patients undergoing tenofovir/entecavir (NUC) treatment for chronic HBV infection. They were categorized based on their HBsAg levels: high (920-12,447 IU/ml) or low (1-100 IU/ml). Cluster frequencies, differential gene expression, and phenotypes were analyzed. Results: In the blood of HBV-infected patients on NUC, the proportion of KLRC2 + \"adaptive\" natural killer (NK) cells was significantly lower in the HBsAg-high group and, remarkably, both KLRC2 + NK and KLRG1+ CD8 T cells display enrichment of lymphocyte activation-associated gene sets in the HBsAg-low group. High levels of HBsAg were associated with mild immune activation in the liver. However, no suppression of liver-resident CXCR6+ NCAM1+ NK or CXCR6+ CD69+ CD8 T cells was detected, while memory B cells showed signs of activation in both the blood and liver. Conclusions: Among NUC-treated patients, we observed a minimal impact of HBsAg on leukocyte populations in the blood and liver. However, for the first time, we found that HBsAg has distinct effects, restricted to NK-, CD8 T-, and memory B-cell subsets, in the blood and liver. Our findings are highly relevant for current clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Impact and implications: This study provides unique insight into the impact of HBsAg on gene expression levels of immune cell subsets in the blood and liver, particularly in the context of NUC-treated chronic HBV infection. It holds significant relevance for current and future clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Our findings raise questions about the effectiveness of such treatment strategies and challenge the previously hypothesized immunomodulatory effects of HBsAg on immune responses against HBV.',\n",
       "  'Cluster_19-gene',\n",
       "  'None here'),\n",
       " 'PMC10835465-mmc10.xlsx-Cluster_25-gene': ('Association of HBsAg levels with differential gene expression in NK, CD8 T, and memory B cells in treated patients with chronic HBV',\n",
       "  'Background & aims: HBsAg secretion may impact immune responses to chronic HBV infection. Thus, therapeutic approaches to suppress HBsAg production are being investigated. Our study aims to examine the immunomodulatory effects of high and low levels of circulating HBsAg and thereby improve our understanding of anti-HBV immunity. Methods: An optimized 10x Genomics single-cell RNA sequencing workflow was applied to blood samples and liver fine-needle aspirates from 18 patients undergoing tenofovir/entecavir (NUC) treatment for chronic HBV infection. They were categorized based on their HBsAg levels: high (920-12,447 IU/ml) or low (1-100 IU/ml). Cluster frequencies, differential gene expression, and phenotypes were analyzed. Results: In the blood of HBV-infected patients on NUC, the proportion of KLRC2 + \"adaptive\" natural killer (NK) cells was significantly lower in the HBsAg-high group and, remarkably, both KLRC2 + NK and KLRG1+ CD8 T cells display enrichment of lymphocyte activation-associated gene sets in the HBsAg-low group. High levels of HBsAg were associated with mild immune activation in the liver. However, no suppression of liver-resident CXCR6+ NCAM1+ NK or CXCR6+ CD69+ CD8 T cells was detected, while memory B cells showed signs of activation in both the blood and liver. Conclusions: Among NUC-treated patients, we observed a minimal impact of HBsAg on leukocyte populations in the blood and liver. However, for the first time, we found that HBsAg has distinct effects, restricted to NK-, CD8 T-, and memory B-cell subsets, in the blood and liver. Our findings are highly relevant for current clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Impact and implications: This study provides unique insight into the impact of HBsAg on gene expression levels of immune cell subsets in the blood and liver, particularly in the context of NUC-treated chronic HBV infection. It holds significant relevance for current and future clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Our findings raise questions about the effectiveness of such treatment strategies and challenge the previously hypothesized immunomodulatory effects of HBsAg on immune responses against HBV.',\n",
       "  'Cluster_25-gene',\n",
       "  'None here'),\n",
       " 'PMC9842760-41467_2022_35348_MOESM4_ESM.xlsx-Fig_4f-Unnamed_0': ('Expansion of interferon inducible gene pool via USP18 inhibition promotes cancer cell pyroptosis',\n",
       "  'While immunotherapy has emerged as a breakthrough cancer therapy, it is only effective in some patients, indicating the need of alternative therapeutic strategies. Induction of cancer immunogenic cell death (ICD) is one promising way to elicit potent adaptive immune responses against tumor-associated antigens. Type I interferon (IFN) is well known to play important roles in different aspects of immune responses, including modulating ICD in anti-tumor action. However, how to expand IFN effect in promoting ICD responses has not been addressed. Here we show that depletion of ubiquitin specific protease 18 (USP18), a negative regulator of IFN signaling, selectively induces cancer cell ICD. Lower USP18 expression correlates with better survival across human selected cancer types and delays cancer progression in mouse models. Mechanistically, nuclear USP18 controls the enhancer landscape of cancer cells and diminishes STAT2-mediated transcription complex binding to IFN-responsive elements. Consequently, USP18 suppression not only enhances expression of canonical IFN-stimulated genes (ISGs), but also activates the expression of a set of atypical ISGs and NF-κB target genes, including genes such as Polo like kinase 2 (PLK2), that induce cancer pyroptosis. These findings may support the use of targeting USP18 as a potential cancer immunotherapy.',\n",
       "  'Fig_4f-Unnamed_0',\n",
       "  'None here'),\n",
       " 'PMC9780220-41598_2022_26217_MOESM2_ESM.xlsx-DEgenes_CTL-EV_FGF2-EV-Unnamed_9': ('Fibroblast growth factor-2 bound to specific dermal fibroblast-derived extracellular vesicles is protected from degradation',\n",
       "  'Fibroblast growth factor-2 (FGF2) has multiple roles in cutaneous wound healing but its natural low stability prevents the development of its use in skin repair therapies. Here we show that FGF2 binds the outer surface of dermal fibroblast (DF)-derived extracellular vesicles (EVs) and this association protects FGF2 from fast degradation. EVs isolated from DF cultured in the presence of FGF2 harbor FGF2 on their surface and FGF2 can bind purified EVs in absence of cells. Remarkably, FGF2 binding to EVs is restricted to a specific subpopulation of EVs, which do not express CD63 and CD81 markers. Treatment of DF with FGF2-EVs activated ERK and STAT signaling pathways and increased cell proliferation and migration. Local injection of FGF2-EVs improved wound healing in mice. We further demonstrated that binding to EVs protects FGF2 from both thermal and proteolytic degradation, thus maintaining FGF2 function. This suggests that EVs protect soluble factors from degradation and increase their stability and half-life. These results reveal a novel aspect of EV function and suggest EVs as a potential tool for delivering FGF2 in skin healing therapies.',\n",
       "  'DEgenes_CTL-EV_FGF2-EV-Unnamed_9',\n",
       "  'To investigate the effect of FGF**2**  Supplementary Information 2'),\n",
       " 'PMC6545701-40880_2019_376_MOESM5_ESM.xlsx-Ark1-Unnamed_10': ('PD-L1 and PD-L2 expression correlated genes in non-small-cell lung cancer',\n",
       "  'Background: Programmed cell death ligand-1 (PD-L1) and ligand-2 (PD-L2) interaction with programmed cell death protein-1 (PD-1) represent an immune-inhibiting checkpoint mediating immune evasion and is, accordingly, an important target for blockade-based immunotherapy in cancer. In non-small-cell lung cancer (NSCLC), improved understanding of PD-1 checkpoint blockade-responsive biology and identification of biomarkers for prediction of a clinical response to immunotherapy is warranted. Thus, in the present study, we systematically described PD-L1 and PD-L2 expression correlated genes in NSCLC. Methods: We performed comparative retrospective analyses to identify PD-L1 and PD-L2 mRNA expression correlated genes in NSCLC. For this, we examined available datasets from the cancer cell line encyclopedia (CCLE) project lung non-small-cell (Lung_NSC) and the cancer genome atlas (TCGA) projects lung adenocarcinoma (LUAD) and squamous cell carcinoma (LUSC). Results: Analysis of the CCLE dataset Lung_NSC identified expression correlation between PD-L1 and PD-L2. Moreover, we identified expression correlation between 489 genes and PD-L1, 191 genes and PD-L2, and 111 genes for both. PD-L1 and PD-L2 also expression correlated in TCGA datasets LUAD and LUSC. In LUAD, we identified expression correlation between 257 genes and PD-L1, 914 genes and PD-L2, and 211 genes for both. In LUSC, we identified expression correlation between 26 genes and PD-L1, 326 genes and PD-L2, and 13 genes for both. Only a few genes expression correlated with PD-L1 and PD-L2 across the CCLE and TCGA datasets. Expression of Interferon signaling-involved genes converged in particular with the expression correlated genes for PD-L1 in Lung_NSC, for PD-L2 in LUSC, and for both PD-L1 and PD-L2 in LUAD. In LUSC, PD-L1, and to a lesser extent PD-L2, expression correlated with chromosome 9p24 localized genes, indicating a chromosome 9p24 topologically associated domain as an important driver of in particular LUSC PD-L1 expression. Expression correlation analyses of the PD-L1 and PD-L2 receptors programmed cell death protein-1 (PD-1), Cluster of differentiation 80 (CD80), and Repulsive guidance molecule B (RGMB) showed that PD-1 and CD80 expression correlated with both PD-L1 and PD-L2 in LUAD. CD80 expression correlated with PD-L2 in LUSC. Conclusions: We present gene signatures associated with PD-L1 and PD-L2 mRNA expression in NSCLC which could possess importance in relation to understand PD-1 checkpoint blockade-responsive biology and development of gene signature based biomarkers for predicting clinical responses to immunotherapy.',\n",
       "  'Ark1-Unnamed_10',\n",
       "  'correlation coefficient r ≥ 0.3 or ≤ − 0.3, and all P values < 0.0**5**  Additional file 5: Table S4. Expression correlated genes for IRF1 and IRF9'),\n",
       " 'PMC6544546-EMS81327-supplement-Supplementary_Table_3.xlsx-Neuron4-gene': ('Altered human oligodendrocyte heterogeneity in multiple sclerosis',\n",
       "  'Oligodendrocyte pathology is increasingly implicated in neurodegenerative diseases as oligodendrocytes both myelinate and provide metabolic support to axons. In multiple sclerosis (MS), demyelination in the central nervous system thus leads to neurodegeneration, but the severity of MS between patients is very variable. Disability does not correlate well with the extent of demyelination 1 , which suggests that other factors contribute to this variability. One such factor may be oligodendrocyte heterogeneity. Not all oligodendrocytes are the same-those from the mouse spinal cord inherently produce longer myelin sheaths than those from the cortex 2 , and single-cell analysis of the mouse central nervous system identified further differences 3,4 . However, the extent of human oligodendrocyte heterogeneity and its possible contribution to MS pathology remain unknown. Here we performed single-nucleus RNA sequencing from white matter areas of post-mortem human brain from patients with MS and from unaffected controls. We identified subclusters of oligodendroglia in control human white matter, some with similarities to mouse, and defined new markers for these cell states. Notably, some subclusters were underrepresented in MS tissue, whereas others were more prevalent. These differences in mature oligodendrocyte subclusters may indicate different functional states of oligodendrocytes in MS lesions. We found similar changes in normal-appearing white matter, showing that MS is a more diffuse disease than its focal demyelination suggests. Our findings of an altered oligodendroglial heterogeneity in MS may be important for understanding disease progression and developing therapeutic approaches.',\n",
       "  'Neuron4-gene',\n",
       "  'smooth muscle cells (VSM), pericytes, endothelial cells, and immune cells (Fig.1a, Extended Fig.1g and **Supplementary Table3**)'),\n",
       " 'PMC9362915-pone.0272017.s003.xlsx-Day4_1_3_Bottom_B-EPSTI1': ('Adrenergic signaling controls early transcriptional programs during CD8+ T cell responses to viral infection',\n",
       "  'Norepinephrine is a key sympathetic neurotransmitter, which acts to suppress CD8 + T cell cytokine secretion and lytic activity by signaling through the β2-adrenergic receptor (ADRB2). Although ADRB2 signaling is considered generally immunosuppressive, its role in regulating the differentiation of effector T cells in response to infection has not been investigated. Using an adoptive transfer approach, we compared the expansion and differentiation of wild type (WT) to Adrb2-/- CD8 + T cells throughout the primary response to vesicular stomatitis virus (VSV) infection in vivo. We measured the dynamic changes in transcriptome profiles of antigen-specific CD8 + T cells as they responded to VSV. Within the first 7 days of infection, WT cells out-paced the expansion of Adrb2-/- cells, which correlated with reduced expression of IL-2 and the IL-2Rα in the absence of ADRB2. RNASeq analysis identified over 300 differentially expressed genes that were both temporally regulated following infection and selectively regulated in WT vs Adrb2-/- cells. These genes contributed to major transcriptional pathways including cytokine receptor activation, signaling in cancer, immune deficiency, and neurotransmitter pathways. By parsing genes within groups that were either induced or repressed over time in response to infection, we identified three main branches of genes that were differentially regulated by the ADRB2. These gene sets were predicted to be regulated by specific transcription factors involved in effector T cell development, such as Tbx21 and Eomes. Collectively, these data demonstrate a significant role for ADRB2 signaling in regulating key transcriptional pathways during CD8 + T cells responses to infection that may dramatically impact their functional capabilities and downstream memory cell development.',\n",
       "  'Day4_1_3_Bottom_B-EPSTI1',\n",
       "  'to regulate them. A full list of genes for each path is provided in **S3**  Tables of genes in each path of the Adrb2+/+ SMARTS model.(XLSX)  Click here for additional data file'),\n",
       " 'PMC10274303-jjad006_suppl_supplementary_dataset_1.xlsx-Sheet1-Gene': ('GATA6-AS1 Regulates Intestinal Epithelial Mitochondrial Functions, and its Reduced Expression is Linked to Intestinal Inflammation and Less Favourable Disease Course in Ulcerative Colitis',\n",
       "  \"Background and aims: Widespread dysregulation of long non-coding RNAs [lncRNAs] including a reduction in GATA6-AS1 was noted in inflammatory bowel disease [IBD]. We previously reported a prominent inhibition of epithelial mitochondrial functions in ulcerative colitis [UC]. However, the connection between reduction of GATA6-AS1 expression and attenuated epithelial mitochondrial functions was not defined. Methods: Mucosal transcriptomics was used to conform GATA6-AS1 reduction in several treatment-naïve independent human cohorts [n=673]. RNA pull-down followed by mass spectrometry was used to determine the GATA6-AS1 interactome. Metabolomics and mitochondrial respiration following GATA6-AS1 silencing in Caco-2 cells were used to elaborate on GATA6-AS1 functions. Results: GATA6-AS1 showed predominant expression in gut epithelia using single cell datasets. GATA6-AS1 levels were reduced in Crohn's disease [CD] ileum and UC rectum in independent cohorts. Reduced GATA6-AS1 lncRNA was further linked to a more severe UC form, and to a less favourable UC course. The GATA6-AS1 interactome showed robust enrichment for mitochondrial proteins, and included TGM2, an autoantigen in coeliac disease that is induced in UC, CD and coeliac disease, in contrast to GATA6-AS1 reduction in these cohorts. GATA6-AS1 silencing resulted in induction of TGM2, and this was coupled with a reduction in mitochondrial membrane potential and mitochondrial respiration, as well as in a reduction of metabolites linked to aerobic respiration relevant to mucosal inflammation. TGM2 knockdown in GATA6-AS1-deficient cells rescued mitochondrial respiration. Conclusions: GATA6-AS1 levels are reduced in UC, CD and coeliac disease, and in more severe UC forms. We highlight GATA6-AS1 as a target regulating epithelial mitochondrial functions, potentially through controlling TGM2 levels.\",\n",
       "  'Sheet1-Gene',\n",
       "  'network in PROTECT using Euclidean distance [full list and FDR values are given in **Supplementary Dataset S1**]  Click here for additional data file'),\n",
       " 'PMC7520256-JCMM-24-9798-s006.csv--symbol': ('Systematic analysis on multiple Gene Expression Omnibus data sets reveals fierce immune response in hepatitis B virus-related acute liver failure',\n",
       "  'Acute liver failure (ALF) caused by hepatitis B virus (HBV) is common type of liver failure in the world, with high morbidity and mortality rates. However, the prevalence, genetic background and factors determining the development of HBV-related ALF are rarely studied. In this study, we examined three Gene Expression Omnibus (GEO) data sets by bioinformatics analysis to identify differentially expressed genes (DEGs), key biological processes and pathways. Immune infiltration analysis showed high immune cells infiltration in HBV-related ALF tissue. We then confirmed natural killer cells and macrophages infiltration in clinical samples by immunohistochemistry assay, implying these cells play a significant role in HBV-ALF. We found 1277 genes were co-up-regulated and that 1082 genes were co-down-regulated in the 3 data sets. Inflammation-related pathways were enriched in the co-up-regulated genes and synthetic metabolic pathways were enriched in the co-down-regulated genes. WGCNA also revealed a key module enriching in immune inflammation response and identified 10 hub genes, differentially expressed in an independent data set. In conclusion, we identified fierce immune inflammatory response to elucidate the immune-driven mechanism of HBV-ALF and 10 hub genes based on gene expression profiles.',\n",
       "  'symbol',\n",
       "  'ascertained 1277 co‐up‐regulated and 1082 co‐down‐regulated genes in three data sets (Tables S4 and **S5**)  Table S5  Click here for additional data file'),\n",
       " 'PMC3338421-pone.0035618.s003.xls-7_GSE19188_Airways-Symbol': ('Exploring the transcriptome of ciliated cells using in silico dissection of human tissues',\n",
       "  \"Cilia are cell organelles that play important roles in cell motility, sensory and developmental functions and are involved in a range of human diseases, known as ciliopathies. Here, we search for novel human genes related to cilia using a strategy that exploits the previously reported tendency of cell type-specific genes to be coexpressed in the transcriptome of complex tissues. Gene coexpression networks were constructed using the noise-resistant WGCNA algorithm in 12 publicly available microarray datasets from human tissues rich in motile cilia: airways, fallopian tubes and brain. A cilia-related coexpression module was detected in 10 out of the 12 datasets. A consensus analysis of this module's gene composition recapitulated 297 known and predicted 74 novel cilia-related genes. 82% of the novel candidates were supported by tissue-specificity expression data from GEO and/or proteomic data from the Human Protein Atlas. The novel findings included a set of genes (DCDC2, DYX1C1, KIAA0319) related to a neurological disease dyslexia suggesting their potential involvement in ciliary functions. Furthermore, we searched for differences in gene composition of the ciliary module between the tissues. A multidrug-and-toxin extrusion transporter MATE2 (SLC47A2) was found as a brain-specific central gene in the ciliary module. We confirm the localization of MATE2 in cilia by immunofluorescence staining using MDCK cells as a model. While MATE2 has previously gained attention as a pharmacologically relevant transporter, its potential relation to cilia is suggested for the first time. Taken together, our large-scale analysis of gene coexpression networks identifies novel genes related to human cell cilia.\",\n",
       "  '7_GSE19188_Airways-Symbol',\n",
       "  'in brain, airways and reproductive tracts is provided in a dataset-by-dataset form in the **Table S3**  Gene composition of the ciliary modules. For each network a list of genes in the ciliary module is provided (genome scale analysis). The “Module membership” column provides Pearson correlations between expression profile of a given gene and integrated eigengene of the ciliary module (see Methods). This measure ranks genes based on their proximity to the center of the ciliary module (hub position).(XLS)  Click here for additional data file'),\n",
       " 'PMC5794756-41598_2018_20217_MOESM4_ESM.xlsx-Downregulated-Unnamed_1': ('High expression of SMARCA4 or SMARCA2 is frequently associated with an opposite prognosis in cancer',\n",
       "  'The gene encoding the ATPase of the chromatin remodeling SWI/SNF complexes SMARCA4 (BRG1) is often mutated or silenced in tumors, suggesting a role as tumor suppressor. Nonetheless, recent reports show requirement of SMARCA4 for tumor cells growth. Here, we performed a computational meta-analysis using gene expression, prognosis, and clinicopathological data to clarify the role of SMARCA4 and the alternative SWI/SNF ATPase SMARCA2 (BRM) in cancer. We show that while the SMARCA4 gene is mostly overexpressed in tumors, SMARCA2 is almost invariably downexpressed in tumors. High SMARCA4 expression was associated with poor prognosis in many types of tumors, including liver hepatocellular carcinoma (LIHC), and kidney renal clear cell carcinoma (KIRC). In contrast, high SMARCA2 expression was associated with good prognosis. We compared tumors with high versus low expression of SMARCA4 or SMARCA2 in LIHC and KIRC cohorts from The Cancer Genome Atlas. While a high expression of SMARCA4 is associated with aggressive tumors, a high expression of SMARCA2 is associated with benign differentiated tumors, suggesting that SMARCA4 and SMARCA2 play opposite roles in cancer. Our results demonstrate that expression of SMARCA4 and SMARCA2 have a high prognostic value and challenge the broadly accepted general role of SMARCA4 as a tumor suppressor.',\n",
       "  'Downregulated-Unnamed_1',\n",
       "  'were upregulated and 835 were downregulated in SMARCA4-high tumors (Supplementary Fig. S2; Supplementary Table **S5**)  Supplementary Table S5'),\n",
       " 'PMC4859332-mmc1.xlsx-Human_tissue_spec_SIG-BD_pDC': ('Comparative genomics analysis of mononuclear phagocyte subsets confirms homology between lymphoid tissue-resident and dermal XCR1(+) DCs in mouse and human and distinguishes them from Langerhans cells',\n",
       "  'Dendritic cells (DC) are mononuclear phagocytes which exhibit a branching (dendritic) morphology and excel at naïve T cell activation. DC encompass several subsets initially identified by their expression of cell surface molecules and later shown to possess distinct functions. DC subset differentiation is orchestrated by transcription factors, growth factors and cytokines. Identifying DC subsets is challenging as very few cell surface molecules are uniquely expressed on any one of these cell populations. There is no standard consensus to identify mononuclear phagocyte subsets; varying antigens are employed depending on the tissue and animal species studied and between laboratories. This has led to confusion in how to accurately define and classify DCs across tissues and between species. Here we report a comparative genomics strategy that enables universal definition of DC and other mononuclear phagocyte subsets across species. We performed a meta-analysis of several public datasets of human and mouse mononuclear phagocyte subsets isolated from blood, spleen, skin or cutaneous lymph nodes, including by using a novel and user friendly software, BubbleGUM, which generates and integrates gene signatures for high throughput gene set enrichment analysis. This analysis demonstrates the equivalence between human and mouse skin XCR1(+) DCs, and between mouse and human Langerhans cells.',\n",
       "  'Human_tissue_spec_SIG-BD_pDC',\n",
       "  'Supplementary material'),\n",
       " 'PMC7333371-13059_2020_1990_MOESM1_ESM.xlsx-Table_S36-Unnamed_1': ('A multidimensional systems biology analysis of cellular senescence in aging and disease',\n",
       "  'Background: Cellular senescence, a permanent state of replicative arrest in otherwise proliferating cells, is a hallmark of aging and has been linked to aging-related diseases. Many genes play a role in cellular senescence, yet a comprehensive understanding of its pathways is still lacking. Results: We develop CellAge (http://genomics.senescence.info/cells), a manually curated database of 279 human genes driving cellular senescence, and perform various integrative analyses. Genes inducing cellular senescence tend to be overexpressed with age in human tissues and are significantly overrepresented in anti-longevity and tumor-suppressor genes, while genes inhibiting cellular senescence overlap with pro-longevity and oncogenes. Furthermore, cellular senescence genes are strongly conserved in mammals but not in invertebrates. We also build cellular senescence protein-protein interaction and co-expression networks. Clusters in the networks are enriched for cell cycle and immunological processes. Network topological parameters also reveal novel potential cellular senescence regulators. Using siRNAs, we observe that all 26 candidates tested induce at least one marker of senescence with 13 genes (C9orf40, CDC25A, CDCA4, CKAP2, GTF3C4, HAUS4, IMMT, MCM7, MTHFD2, MYBL2, NEK2, NIPA2, and TCEB3) decreasing cell number, activating p16/p21, and undergoing morphological changes that resemble cellular senescence. Conclusions: Overall, our work provides a benchmark resource for researchers to study cellular senescence, and our systems biology analyses reveal new insights and gene regulators of cellular senescence.',\n",
       "  'Table_S36-Unnamed_1',\n",
       "  'the CellAge genes (p < 0.05, Fisher’s exact test with BH correction) (Additional file **1**)  Excel file containing Supplementary Tables S1-S48'),\n",
       " 'PMC8259211-40364_2021_304_MOESM3_ESM.xlsx-Supplementary_Table_2-Unnamed_6': ('Inflammation-related genes S100s, RNASE3, and CYBB and risk of leukemic transformation in patients with myelodysplastic syndrome with myelofibrosis',\n",
       "  'Myelodysplastic syndrome with myelofibrosis (MDS-MF) has been associated with an inferior prognosis compared with MDS without MF. However, MDS-MF is not listed independently as a subtype of MDS, and its clinical and genetic characteristics remain poorly understood. We retrospectively compared 53 patients with MDS-MF (44 MF grade 1/MF 1 ; 9 MF grade 2-3/MF 2 - 3 ) and 31 with de novo MDS without MF (MDS). The leukemic transformation risks of both MDS-MF 2 - 3 and MDS-MF 1 were increased compared with the MDS group. To identify the potential mechanisms responsible for the leukemic transformation of MDS-MF, we performed single-cell sequencing for one MDS-MF 2 - 3 patient before and after leukemic transformation to explore the variations in gene expression levels. In addition to upgraded expression levels of acute myeloid leukemia-related genes during leukemic transformation, expression levels of some inflammation-related genes (such as S100s, RNASE3, and CYBB) were also increased, and inflammation-related pathways were up-regulated. These results suggest that inflammation-related genes and pathways may play an important role in the leukemic transformation of MDS-MF.',\n",
       "  'Supplementary_Table_2-Unnamed_6',\n",
       "  'MDS-MF. We conducted single-cell sequencing of BM mononuclear cells (BMMCs) from a patient with MDS-MF**2**  Supplementary Table 2. Marker genes of overall t-stochastic neighbor embedding map'),\n",
       " 'PMC10835465-mmc6.xlsx-Cluster10-gene': ('Association of HBsAg levels with differential gene expression in NK, CD8 T, and memory B cells in treated patients with chronic HBV',\n",
       "  'Background & aims: HBsAg secretion may impact immune responses to chronic HBV infection. Thus, therapeutic approaches to suppress HBsAg production are being investigated. Our study aims to examine the immunomodulatory effects of high and low levels of circulating HBsAg and thereby improve our understanding of anti-HBV immunity. Methods: An optimized 10x Genomics single-cell RNA sequencing workflow was applied to blood samples and liver fine-needle aspirates from 18 patients undergoing tenofovir/entecavir (NUC) treatment for chronic HBV infection. They were categorized based on their HBsAg levels: high (920-12,447 IU/ml) or low (1-100 IU/ml). Cluster frequencies, differential gene expression, and phenotypes were analyzed. Results: In the blood of HBV-infected patients on NUC, the proportion of KLRC2 + \"adaptive\" natural killer (NK) cells was significantly lower in the HBsAg-high group and, remarkably, both KLRC2 + NK and KLRG1+ CD8 T cells display enrichment of lymphocyte activation-associated gene sets in the HBsAg-low group. High levels of HBsAg were associated with mild immune activation in the liver. However, no suppression of liver-resident CXCR6+ NCAM1+ NK or CXCR6+ CD69+ CD8 T cells was detected, while memory B cells showed signs of activation in both the blood and liver. Conclusions: Among NUC-treated patients, we observed a minimal impact of HBsAg on leukocyte populations in the blood and liver. However, for the first time, we found that HBsAg has distinct effects, restricted to NK-, CD8 T-, and memory B-cell subsets, in the blood and liver. Our findings are highly relevant for current clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Impact and implications: This study provides unique insight into the impact of HBsAg on gene expression levels of immune cell subsets in the blood and liver, particularly in the context of NUC-treated chronic HBV infection. It holds significant relevance for current and future clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Our findings raise questions about the effectiveness of such treatment strategies and challenge the previously hypothesized immunomodulatory effects of HBsAg on immune responses against HBV.',\n",
       "  'Cluster10-gene',\n",
       "  'None here'),\n",
       " 'PMC5794756-41598_2018_20217_MOESM4_ESM.xlsx-Downregulated-Supplementary_Table_S5_List_of_genes_downregulated_in_SMARCA4-high_versus_the_SMARCA4-low_Liver_hepatocellular_Carcinoma_LIHC_tumors': ('High expression of SMARCA4 or SMARCA2 is frequently associated with an opposite prognosis in cancer',\n",
       "  'The gene encoding the ATPase of the chromatin remodeling SWI/SNF complexes SMARCA4 (BRG1) is often mutated or silenced in tumors, suggesting a role as tumor suppressor. Nonetheless, recent reports show requirement of SMARCA4 for tumor cells growth. Here, we performed a computational meta-analysis using gene expression, prognosis, and clinicopathological data to clarify the role of SMARCA4 and the alternative SWI/SNF ATPase SMARCA2 (BRM) in cancer. We show that while the SMARCA4 gene is mostly overexpressed in tumors, SMARCA2 is almost invariably downexpressed in tumors. High SMARCA4 expression was associated with poor prognosis in many types of tumors, including liver hepatocellular carcinoma (LIHC), and kidney renal clear cell carcinoma (KIRC). In contrast, high SMARCA2 expression was associated with good prognosis. We compared tumors with high versus low expression of SMARCA4 or SMARCA2 in LIHC and KIRC cohorts from The Cancer Genome Atlas. While a high expression of SMARCA4 is associated with aggressive tumors, a high expression of SMARCA2 is associated with benign differentiated tumors, suggesting that SMARCA4 and SMARCA2 play opposite roles in cancer. Our results demonstrate that expression of SMARCA4 and SMARCA2 have a high prognostic value and challenge the broadly accepted general role of SMARCA4 as a tumor suppressor.',\n",
       "  'Downregulated-Supplementary_Table_S5_List_of_genes_downregulated_in_SMARCA4-high_versus_the_SMARCA4-low_Liver_hepatocellular_Carcinoma_LIHC_tumors',\n",
       "  'were upregulated and 835 were downregulated in SMARCA4-high tumors (Supplementary Fig. S2; Supplementary Table **S5**)  Supplementary Table S5'),\n",
       " 'PMC11093611-jci-134-178392-s187.xlsx-summary-Unnamed_2': ('The secreted micropeptide C4orf48 enhances renal fibrosis via an RNA-binding mechanism',\n",
       "  \"Renal interstitial fibrosis is an important mechanism in the progression of chronic kidney disease (CKD) to end-stage kidney disease. However, we lack specific treatments to slow or halt renal fibrosis. Ribosome profiling identified upregulation of a secreted micropeptide, C4orf48 (Cf48), in mouse diabetic nephropathy. Cf48 RNA and protein levels were upregulated in tubular epithelial cells in human and experimental CKD. Serum Cf48 levels were increased in human CKD and correlated with loss of kidney function, increasing CKD stage, and the degree of active interstitial fibrosis. Cf48 overexpression in mice accelerated renal fibrosis, while Cf48 gene deletion or knockdown by antisense oligonucleotides significantly reduced renal fibrosis in CKD models. In vitro, recombinant Cf48 (rCf48) enhanced TGF-β1-induced fibrotic responses in renal fibroblasts and epithelial cells independently of Smad3 phosphorylation. Cellular uptake of Cf48 and its profibrotic response in fibroblasts operated via the transferrin receptor. RNA immunoprecipitation-sequencing identified Cf48 binding to mRNA of genes involved in the fibrotic response, including Serpine1, Acta2, Ccn2, and Col4a1. rCf48 binds to the 3'UTR of Serpine1 and increases mRNA half-life. We identify the secreted Cf48 micropeptide as a potential enhancer of renal fibrosis that operates as an RNA-binding peptide to promote the production of extracellular matrix.\",\n",
       "  'summary-Unnamed_2',\n",
       "  'None here'),\n",
       " 'PMC10097911-Table1.XLSX-DGE_Late-RoundStds-Genes': ('Spermiogenesis alterations in the absence of CTCF revealed by single cell RNA sequencing',\n",
       "  'CTCF is an architectonic protein that organizes the genome inside the nucleus in almost all eukaryotic cells. There is evidence that CTCF plays a critical role during spermatogenesis as its depletion produces abnormal sperm and infertility. However, defects produced by its depletion throughout spermatogenesis have not been fully characterized. In this work, we performed single cell RNA sequencing in spermatogenic cells with and without CTCF. We uncovered defects in transcriptional programs that explain the severity of the damage in the produced sperm. In the early stages of spermatogenesis, transcriptional alterations are mild. As germ cells go through the specialization stage or spermiogenesis, transcriptional profiles become more altered. We found morphology defects in spermatids that support the alterations in their transcriptional profiles. Altogether, our study sheds light on the contribution of CTCF to the phenotype of male gametes and provides a fundamental description of its role at different stages of spermiogenesis.',\n",
       "  'DGE_Late-RoundStds-Genes',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC8671437-41598_2021_3393_MOESM4_ESM.xlsx-HPA_skeletal_muscle-specific-Ensembl_ID': ('Transcriptomic adaptation during skeletal muscle habituation to eccentric or concentric exercise training',\n",
       "  \"Eccentric (ECC) and concentric (CON) contractions induce distinct muscle remodelling patterns that manifest early during exercise training, the causes of which remain unclear. We examined molecular signatures of early contraction mode-specific muscle adaptation via transcriptome-wide network and secretome analyses during 2 weeks of ECC- versus CON-specific (downhill versus uphill running) exercise training (exercise 'habituation'). Despite habituation attenuating total numbers of exercise-induced genes, functional gene-level profiles of untrained ECC or CON were largely unaltered post-habituation. Network analysis revealed 11 ECC-specific modules, including upregulated extracellular matrix and immune profiles plus downregulated mitochondrial pathways following untrained ECC. Of 3 CON-unique modules, 2 were ribosome-related and downregulated post-habituation. Across training, 376 ECC-specific and 110 CON-specific hub genes were identified, plus 45 predicted transcription factors. Secreted factors were enriched in 3 ECC- and/or CON-responsive modules, with all 3 also being under the predicted transcriptional control of SP1 and KLF4. Of 34 candidate myokine hubs, 1 was also predicted to have elevated expression in skeletal muscle versus other tissues: THBS4, of a secretome-enriched module upregulated after untrained ECC. In conclusion, distinct untrained ECC and CON transcriptional responses are dampened after habituation without substantially shifting molecular functional profiles, providing new mechanistic candidates into contraction-mode specific muscle regulation.\",\n",
       "  'HPA_skeletal_muscle-specific-Ensembl_ID',\n",
       "  'and may thus potentially represent prominent myokine-related muscle regulatory pathways (Fig. 6, Supplementary File **3**)  Supplementary Information 4'),\n",
       " 'PMC10849790-CPR-57-e13545-s005.xlsx-muscle_progenitor_vs_myoblast1S-gene_short_name': ('Bulk and single-cell alternative splicing analyses reveal roles of TRA2B in myogenic differentiation',\n",
       "  'Alternative splicing (AS) disruption has been linked to disorders of muscle development, as well as muscular atrophy. However, the precise changes in AS patterns that occur during myogenesis are not well understood. Here, we employed isoform long-reads RNA-seq (Iso-seq) and single-cell RNA-seq (scRNA-seq) to investigate the AS landscape during myogenesis. Our Iso-seq data identified 61,146 full-length isoforms representing 11,682 expressed genes, of which over 52% were novel. We identified 38,022 AS events, with most of these events altering coding sequences and exhibiting stage-specific splicing patterns. We identified AS dynamics in different types of muscle cells through scRNA-seq analysis, revealing genes essential for the contractile muscle system and cytoskeleton that undergo differential splicing across cell types. Gene-splicing analysis demonstrated that AS acts as a regulator, independent of changes in overall gene expression. Two isoforms of splicing factor TRA2B play distinct roles in myogenic differentiation by triggering AS of TGFBR2 to regulate canonical TGF-β signalling cascades differently. Our study provides a valuable transcriptome resource for myogenesis and reveals the complexity of AS and its regulation during myogenesis.',\n",
       "  'muscle_progenitor_vs_myoblast1S-gene_short_name',\n",
       "  'None here'),\n",
       " 'PMC8100333-Table_2.xlsx-Sheet1-Gene_symbol': ('Zika Virus Infection of Human Iris Pigment Epithelial Cells',\n",
       "  'During recent Zika epidemics, adults infected with Zika virus (ZIKV) have developed organ-specific inflammatory complications. The most serious Zika-associated inflammatory eye disease is uveitis, which is commonly anterior in type, affecting both eyes and responding to corticosteroid eye drops. Mechanisms of Zika-associated anterior uveitis are unknown, but ZIKV has been identified in the aqueous humor of affected individuals. The iris pigment epithelium is a target cell population in viral anterior uveitis, and it acts to maintain immune privilege within the anterior eye. Interactions between ZIKV and human iris pigment epithelial cells were investigated with infectivity assays and RNA-sequencing. Primary cell isolates were prepared from eyes of 20 cadaveric donors, and infected for 24 hours with PRVABC59 strain ZIKV or incubated uninfected as control. Cytoimmunofluorescence, RT-qPCR on total cellular RNA, and focus-forming assays of culture supernatant showed cell isolates were permissive to infection, and supported replication and release of infectious ZIKV. To explore molecular responses of cell isolates to ZIKV infection at the whole transcriptome level, RNA was sequenced on the Illumina NextSeq 500 platform, and results were aligned to the human GRCh38 genome. Multidimensional scaling showed clear separation between transcriptomes of infected and uninfected cell isolates. Differential expression analysis indicated a vigorous molecular response of the cell to ZIKV: 7,935 genes were differentially expressed between ZIKV-infected and uninfected cells (FDR < 0.05), and 99% of 613 genes that changed at least two-fold were up-regulated. Reactome and KEGG pathway and Gene Ontology enrichment analyses indicated strong activation of viral recognition and defense, in addition to biosynthesis processes. A CHAT network included 6275 molecular nodes and 24 contextual hubs in the cell response to ZIKV infection. Receptor-interacting serine/threonine kinase 1 (RIPK1) was the most significantly connected contextual hub. Correlation of gene expression with read counts assigned to the ZIKV genome identified a negative correlation between interferon signaling and viral load across isolates. This work represents the first investigation of mechanisms of Zika-associated anterior uveitis using an in vitro human cell model. The results suggest the iris pigment epithelium mounts a molecular response that limits intraocular pathology in most individuals.',\n",
       "  'Sheet1-Gene_symbol',\n",
       "  'that were decreased at least two-fold in the infected cells ( Figure 2B , **Supplementary Table 2**)  Differential Expression Analysis  Click here for additional data file'),\n",
       " 'PMC8100333-Table_2.xlsx-Sheet1-Ensembl_gene_ID': ('Zika Virus Infection of Human Iris Pigment Epithelial Cells',\n",
       "  'During recent Zika epidemics, adults infected with Zika virus (ZIKV) have developed organ-specific inflammatory complications. The most serious Zika-associated inflammatory eye disease is uveitis, which is commonly anterior in type, affecting both eyes and responding to corticosteroid eye drops. Mechanisms of Zika-associated anterior uveitis are unknown, but ZIKV has been identified in the aqueous humor of affected individuals. The iris pigment epithelium is a target cell population in viral anterior uveitis, and it acts to maintain immune privilege within the anterior eye. Interactions between ZIKV and human iris pigment epithelial cells were investigated with infectivity assays and RNA-sequencing. Primary cell isolates were prepared from eyes of 20 cadaveric donors, and infected for 24 hours with PRVABC59 strain ZIKV or incubated uninfected as control. Cytoimmunofluorescence, RT-qPCR on total cellular RNA, and focus-forming assays of culture supernatant showed cell isolates were permissive to infection, and supported replication and release of infectious ZIKV. To explore molecular responses of cell isolates to ZIKV infection at the whole transcriptome level, RNA was sequenced on the Illumina NextSeq 500 platform, and results were aligned to the human GRCh38 genome. Multidimensional scaling showed clear separation between transcriptomes of infected and uninfected cell isolates. Differential expression analysis indicated a vigorous molecular response of the cell to ZIKV: 7,935 genes were differentially expressed between ZIKV-infected and uninfected cells (FDR < 0.05), and 99% of 613 genes that changed at least two-fold were up-regulated. Reactome and KEGG pathway and Gene Ontology enrichment analyses indicated strong activation of viral recognition and defense, in addition to biosynthesis processes. A CHAT network included 6275 molecular nodes and 24 contextual hubs in the cell response to ZIKV infection. Receptor-interacting serine/threonine kinase 1 (RIPK1) was the most significantly connected contextual hub. Correlation of gene expression with read counts assigned to the ZIKV genome identified a negative correlation between interferon signaling and viral load across isolates. This work represents the first investigation of mechanisms of Zika-associated anterior uveitis using an in vitro human cell model. The results suggest the iris pigment epithelium mounts a molecular response that limits intraocular pathology in most individuals.',\n",
       "  'Sheet1-Ensembl_gene_ID',\n",
       "  'that were decreased at least two-fold in the infected cells ( Figure 2B , **Supplementary Table 2**)  Differential Expression Analysis  Click here for additional data file'),\n",
       " 'PMC9296597-41421_2022_415_MOESM4_ESM.xlsx-Endothelium_VCAM1-gene': ('Single-cell multiomics analysis reveals regulatory programs in clear cell renal cell carcinoma',\n",
       "  'The clear cell renal cell carcinoma (ccRCC) microenvironment consists of many different cell types and structural components that play critical roles in cancer progression and drug resistance, but the cellular architecture and underlying gene regulatory features of ccRCC have not been fully characterized. Here, we applied single-cell RNA sequencing (scRNA-seq) and single-cell assay for transposase-accessible chromatin sequencing (scATAC-seq) to generate transcriptional and epigenomic landscapes of ccRCC. We identified tumor cell-specific regulatory programs mediated by four key transcription factors (TFs) (HOXC5, VENTX, ISL1, and OTP), and these TFs have prognostic significance in The Cancer Genome Atlas (TCGA) database. Targeting these TFs via short hairpin RNAs (shRNAs) or small molecule inhibitors decreased tumor cell proliferation. We next performed an integrative analysis of chromatin accessibility and gene expression for CD8 + T cells and macrophages to reveal the different regulatory elements in their subgroups. Furthermore, we delineated the intercellular communications mediated by ligand-receptor interactions within the tumor microenvironment. Taken together, our multiomics approach further clarifies the cellular heterogeneity of ccRCC and identifies potential therapeutic targets.',\n",
       "  'Endothelium_VCAM1-gene',\n",
       "  '15 cell types in the scRNA-seq dataset (Fig. 1b, c and Supplementary Tables S2, **S3**)  Supplementary Table S3'),\n",
       " 'PMC5390115-mmc5.xlsx-HUV1-Unnamed_15': ('A Genome-wide Analysis of Human Pluripotent Stem Cell-Derived Endothelial Cells in 2D or 3D Culture',\n",
       "  'A defined protocol for efficiently deriving endothelial cells from human pluripotent stem cells was established and vascular morphogenesis was used as a model system to understand how synthetic hydrogels influence global biological function compared with common 2D and 3D culture platforms. RNA sequencing demonstrated that gene expression profiles were similar for endothelial cells and pericytes cocultured in polyethylene glycol (PEG) hydrogels or Matrigel, while monoculture comparisons identified distinct vascular signatures for each cell type. Endothelial cells cultured on tissue-culture polystyrene adopted a proliferative phenotype compared with cells cultured on or encapsulated in PEG hydrogels. The proliferative phenotype correlated to increased FAK-ERK activity, and knockdown or inhibition of ERK signaling reduced proliferation and expression for cell-cycle genes while increasing expression for \"3D-like\" vasculature development genes. Our results provide insight into the influence of 2D and 3D culture formats on global biological processes that regulate cell function.',\n",
       "  'HUV1-Unnamed_15',\n",
       "  'upregulated genes for pericytes that overlapped on days 1–3 of culture in PEG hydrogels (**Table S4**)  Table S4. Differential Gene Expression for Endothelial Cells Compared with Pericytes, Related to Figure 3'),\n",
       " 'PMC5352202-oncotarget-08-846-s002.docx-0-Genes_of_spot_A': ('Mapping heterogeneity in patient-derived melanoma cultures by single-cell RNA-seq',\n",
       "  'Recent technological advances in single-cell genomics make it possible to analyze cellular heterogeneity of tumor samples. Here, we applied single-cell RNA-seq to measure the transcriptomes of 307 single cells cultured from three biopsies of three different patients with a BRAF/NRAS wild type, BRAF mutant/NRAS wild type and BRAF wild type/NRAS mutant melanoma metastasis, respectively. Analysis based on self-organizing maps identified sub-populations defined by multiple gene expression modules involved in proliferation, oxidative phosphorylation, pigmentation and cellular stroma. Gene expression modules had prognostic relevance when compared with gene expression data from published melanoma samples and patient survival data. We surveyed kinome expression patterns across sub-populations of the BRAF/NRAS wild type sample and found that CDK4 and CDK2 were consistently highly expressed in the majority of cells, suggesting that these kinases might be involved in melanoma progression. Treatment of cells with the CDK4 inhibitor palbociclib restricted cell proliferation to a similar, and in some cases greater, extent than MAPK inhibitors. Finally, we identified a low abundant sub-population in this sample that highly expressed a module containing ABC transporter ABCB5, surface markers CD271 and CD133, and multiple aldehyde dehydrogenases (ALDHs). Patient-derived cultures of the BRAF mutant/NRAS wild type and BRAF wild type/NRAS mutant metastases showed more homogeneous single-cell gene expression patterns with gene expression modules for proliferation and ABC transporters. Taken together, our results describe an intertumor and intratumor heterogeneity in melanoma short-term cultures which might be relevant for patient survival, and suggest promising targets for new treatment approaches in melanoma therapy.',\n",
       "  '0-Genes_of_spot_A',\n",
       "  'clusters of co-expressed genes which were called spot-modules A–D (Figure 1C, 1D; Table 1; **Supplementary Table S1**)'),\n",
       " 'PMC11188533-12993_2024_240_MOESM2_ESM.xlsx-Figure_3A_VolcanoPlots-Unnamed_82': ('Transcriptome analysis identifies an ASD-Like phenotype in oligodendrocytes and microglia from C58/J amygdala that is dependent on sex and sociability',\n",
       "  \"Background: Autism Spectrum Disorder (ASD) is a group of neurodevelopmental disorders with higher incidence in males and is characterized by atypical verbal/nonverbal communication, restricted interests that can be accompanied by repetitive behavior, and disturbances in social behavior. This study investigated brain mechanisms that contribute to sociability deficits and sex differences in an ASD animal model. Methods: Sociability was measured in C58/J and C57BL/6J mice using the 3-chamber social choice test. Bulk RNA-Seq and snRNA-Seq identified transcriptional changes in C58/J and C57BL/6J amygdala within which DMRseq was used to measure differentially methylated regions in amygdala. Results: C58/J mice displayed divergent social strata in the 3-chamber test. Transcriptional and pathway signatures revealed immune-related biological processes differ between C58/J and C57BL/6J amygdala. Hypermethylated and hypomethylated genes were identified in C58/J versus C57BL/6J amygdala. snRNA-Seq data in C58/J amygdala identified differential transcriptional signatures within oligodendrocytes and microglia characterized by increased ASD risk gene expression and predicted impaired myelination that was dependent on sex and sociability. RNA velocity, gene regulatory network, and cell communication analysis showed diminished oligodendrocyte/microglia differentiation. Findings were verified using Bulk RNA-Seq and demonstrated oxytocin's beneficial effects on myelin gene expression. Limitations: Our findings are significant. However, limitations can be noted. The cellular mechanisms linking reduced oligodendrocyte differentiation and reduced myelination to an ASD phenotype in C58/J mice need further investigation. Additional snRNA-Seq and spatial studies would determine if effects in oligodendrocytes/microglia are unique to amygdala or if this occurs in other brain regions. Oxytocin's effects need further examination to understand its' potential as an ASD therapeutic. Conclusions: Our work demonstrates the C58/J mouse model's utility in evaluating the influence of sex and sociability on the transcriptome in concomitant brain regions involved in ASD. Our single-nucleus transcriptome analysis elucidates potential pathological roles of oligodendrocytes and microglia in ASD. This investigation provides details regarding regulatory features disrupted in these cell types, including transcriptional gene dysregulation, aberrant cell differentiation, altered gene regulatory networks, and changes to key pathways that promote microglia/oligodendrocyte differentiation. Our studies provide insight into interactions between genetic risk and epigenetic processes associated with divergent affiliative behavior and lack of positive sociability.\",\n",
       "  'Figure_3A_VolcanoPlots-Unnamed_82',\n",
       "  'None here'),\n",
       " 'PMC11188533-12993_2024_240_MOESM2_ESM.xlsx-Supp_Figure_8A-Unnamed_98': ('Transcriptome analysis identifies an ASD-Like phenotype in oligodendrocytes and microglia from C58/J amygdala that is dependent on sex and sociability',\n",
       "  \"Background: Autism Spectrum Disorder (ASD) is a group of neurodevelopmental disorders with higher incidence in males and is characterized by atypical verbal/nonverbal communication, restricted interests that can be accompanied by repetitive behavior, and disturbances in social behavior. This study investigated brain mechanisms that contribute to sociability deficits and sex differences in an ASD animal model. Methods: Sociability was measured in C58/J and C57BL/6J mice using the 3-chamber social choice test. Bulk RNA-Seq and snRNA-Seq identified transcriptional changes in C58/J and C57BL/6J amygdala within which DMRseq was used to measure differentially methylated regions in amygdala. Results: C58/J mice displayed divergent social strata in the 3-chamber test. Transcriptional and pathway signatures revealed immune-related biological processes differ between C58/J and C57BL/6J amygdala. Hypermethylated and hypomethylated genes were identified in C58/J versus C57BL/6J amygdala. snRNA-Seq data in C58/J amygdala identified differential transcriptional signatures within oligodendrocytes and microglia characterized by increased ASD risk gene expression and predicted impaired myelination that was dependent on sex and sociability. RNA velocity, gene regulatory network, and cell communication analysis showed diminished oligodendrocyte/microglia differentiation. Findings were verified using Bulk RNA-Seq and demonstrated oxytocin's beneficial effects on myelin gene expression. Limitations: Our findings are significant. However, limitations can be noted. The cellular mechanisms linking reduced oligodendrocyte differentiation and reduced myelination to an ASD phenotype in C58/J mice need further investigation. Additional snRNA-Seq and spatial studies would determine if effects in oligodendrocytes/microglia are unique to amygdala or if this occurs in other brain regions. Oxytocin's effects need further examination to understand its' potential as an ASD therapeutic. Conclusions: Our work demonstrates the C58/J mouse model's utility in evaluating the influence of sex and sociability on the transcriptome in concomitant brain regions involved in ASD. Our single-nucleus transcriptome analysis elucidates potential pathological roles of oligodendrocytes and microglia in ASD. This investigation provides details regarding regulatory features disrupted in these cell types, including transcriptional gene dysregulation, aberrant cell differentiation, altered gene regulatory networks, and changes to key pathways that promote microglia/oligodendrocyte differentiation. Our studies provide insight into interactions between genetic risk and epigenetic processes associated with divergent affiliative behavior and lack of positive sociability.\",\n",
       "  'Supp_Figure_8A-Unnamed_98',\n",
       "  'None here'),\n",
       " 'PMC2680378-blood-2008-06-162958_TableS5.xls-specific-CD66b-Ensembl_Gene_ID': ('A HaemAtlas: characterizing gene expression in differentiated human blood cells',\n",
       "  'Hematopoiesis is a carefully controlled process that is regulated by complex networks of transcription factors that are, in part, controlled by signals resulting from ligand binding to cell-surface receptors. To further understand hematopoiesis, we have compared gene expression profiles of human erythroblasts, megakaryocytes, B cells, cytotoxic and helper T cells, natural killer cells, granulocytes, and monocytes using whole genome microarrays. A bioinformatics analysis of these data was performed focusing on transcription factors, immunoglobulin superfamily members, and lineage-specific transcripts. We observed that the numbers of lineage-specific genes varies by 2 orders of magnitude, ranging from 5 for cytotoxic T cells to 878 for granulocytes. In addition, we have identified novel coexpression patterns for key transcription factors involved in hematopoiesis (eg, GATA3-GFI1 and GATA2-KLF1). This study represents the most comprehensive analysis of gene expression in hematopoietic cells to date and has identified genes that play key roles in lineage commitment and cell function. The data, which are freely accessible, will be invaluable for future studies on hematopoiesis and the role of specific genes and will also aid the understanding of the recent genome-wide association studies.',\n",
       "  'specific-CD66b-Ensembl_Gene_ID',\n",
       "  '[Supplemental Appendix, Tables, and Figure]'),\n",
       " 'PMC5952342-Supp_Tables1-5.xlsx-Supplemental_Table_4-Supplemental_Table_4_LT-related_genes_in_three_modules': ('Transcriptome Network Analysis Reveals Aging-Related Mitochondrial and Proteasomal Dysfunction and Immune Activation in Human Thyroid',\n",
       "  'Background: Elucidating aging-related transcriptomic changes in human organs is necessary to understand the aging physiology and mechanisms, but little is known regarding the thyroid gland. We investigated aging-related transcriptomic alterations in the human thyroid gland and characterized the related molecular functions. Methods: Publicly available RNA sequencing data of 322 thyroid tissue samples from the Genotype-Tissue Expression project were analyzed. In addition, our own 64 RNA sequencing data of normal thyroid tissue samples were used as a validation set. To comprehensively evaluate the associations between aging and transcriptomic changes, we performed a weighted gene coexpression network analysis and pathway enrichment analysis. The thyroid differentiation score was then used for further analysis, defining the correlations between thyroid differentiation and aging. Results: The most significant aging-related transcriptomic change in thyroid was the downregulation of genes related to the mitochondrial and proteasomal functions (p = 3 × 10 -6 ). Moreover, genes that are associated with immune processes were significantly upregulated with age (p = 3 × 10 -4 ), and all of them overlapped with the upregulated genes in the thyroid glands affected by lymphocytic thyroiditis. Furthermore, these aging-related changes were not significantly different according to sex, but in terms of the thyroid differentiation, females were more susceptible to aging-related changes (p for trend = 0.03). Conclusions: Aging-related transcriptomic changes in the thyroid gland were associated with mitochondrial and proteasomal dysfunction, loss of differentiation, and activation of autoimmune processes. Our results provide clues to better understanding the age-related decline in thyroid function and higher susceptibility to autoimmune thyroid disease.',\n",
       "  'Supplemental_Table_4-Supplemental_Table_4_LT-related_genes_in_three_modules',\n",
       "  '(9,10). The age distribution and the frequency of lymphocytic thyroiditis (LT) are presented in **Supplementary Table S1**  Supplemental data'),\n",
       " 'PMC6753140-41467_2019_12159_MOESM7_ESM.xlsx-Sheet1-Supplementary_Data_4': ('DNA methylation loss promotes immune evasion of tumours with high mutation and copy number load',\n",
       "  'Mitotic cell division increases tumour mutation burden and copy number load, predictive markers of the clinical benefit of immunotherapy. Cell division correlates also with genomic demethylation involving methylation loss in late-replicating partial methylation domains. Here we find that immunomodulatory pathway genes are concentrated in these domains and transcriptionally repressed in demethylated tumours with CpG island promoter hypermethylation. Global methylation loss correlated with immune evasion signatures independently of mutation burden and aneuploidy. Methylome data of our cohort (n = 60) and a published cohort (n = 81) in lung cancer and a melanoma cohort (n = 40) consistently demonstrated that genomic methylation alterations counteract the contribution of high mutation burden and increase immunotherapeutic resistance. Higher predictive power was observed for methylation loss than mutation burden. We also found that genomic hypomethylation correlates with the immune escape signatures of aneuploid tumours. Hence, DNA methylation alterations implicate epigenetic modulation in precision immunotherapy.',\n",
       "  'Sheet1-Supplementary_Data_4',\n",
       "  'and lymph nodes was five-fold higher than that in the remaining tissues (n = 1**4**)  Supplementary Data 4'),\n",
       " 'PMC11258657-mmc2.xlsx-S11-TableS11_Liver_tissue_special_genes_by_comparing_the_normal_tissue_and_other_normal_tissue': ('Liver metastases across cancer types sharing tumor environment immunotolerance can impede immune response therapy and immune monitoring',\n",
       "  'Background: Hepatic immune tolerance might contribute to the development of therapeutic resistance to immunotherapy. However, addressing this issue is challenging since the efficacy of immunotherapy in the context of liver metastasis (LM) remains poorly studied. Here, we aimed to establish an LM common immune feature (LMCIF) score to quantify the characteristics of LM immunotolerance across cancer types for assisting clinical disease management. Methods: Large-scale clinical data were collected to identify the prognosis of LM. Multi-omics datasets of metastatic cancers with LM special immune-related pathways (LMSIPs) from the Molecular Signatures Database (MSigDB)were used to obtain an LMCIF cluster. Based on differential expression genes (DEGs), a novel LMCIF score for the LMCIF cluster was constructed. In addition, multi-omics, and immunohistochemistry (IHC) data from the public and in-house cohorts were used to explore the features of LM, and LMCIF score. Results: Patients with LM had a worse prognosis and significantly lower infiltration of immune cells than patients with metastasis to other organs when analyzed with combined clinical and RNA sequencing data. After extracting the LMCIF cluster from 373 samples by utilizing 29 LMSIPs and validating them in a microarray cohort, an LMCIF score was established to confirm the role of the immunosuppressive environment as a contributor to the poor prognosis of LM across cancer types. Moreover, this LMCIF score could be used to predict the immune response of cancer patients undergoing immunotherapy. Finally, we identified that the majority of the 31 LMCIF genes exhibited a negative correlation with TME cells in LM patients, one of them, KRT19, which possessed the strongest positive correlation with LMCIF score, was confirmed to have an immunosuppressive effect through IHC analysis. Conclusions: Our results suggest that LM across cancer types share similar immunological profiles that induce immunotolerance and escape from immune monitoring. The novel LMCIF score represents a common liver metastasis immune cluster for predicting immunotherapy response, the results of which might benefit clinical disease management.',\n",
       "  'S11-TableS11_Liver_tissue_special_genes_by_comparing_the_normal_tissue_and_other_normal_tissue',\n",
       "  'None here'),\n",
       " 'PMC8671437-41598_2021_3393_MOESM4_ESM.xlsx-HPA_skeletal_muscle-specific-Symbol': ('Transcriptomic adaptation during skeletal muscle habituation to eccentric or concentric exercise training',\n",
       "  \"Eccentric (ECC) and concentric (CON) contractions induce distinct muscle remodelling patterns that manifest early during exercise training, the causes of which remain unclear. We examined molecular signatures of early contraction mode-specific muscle adaptation via transcriptome-wide network and secretome analyses during 2 weeks of ECC- versus CON-specific (downhill versus uphill running) exercise training (exercise 'habituation'). Despite habituation attenuating total numbers of exercise-induced genes, functional gene-level profiles of untrained ECC or CON were largely unaltered post-habituation. Network analysis revealed 11 ECC-specific modules, including upregulated extracellular matrix and immune profiles plus downregulated mitochondrial pathways following untrained ECC. Of 3 CON-unique modules, 2 were ribosome-related and downregulated post-habituation. Across training, 376 ECC-specific and 110 CON-specific hub genes were identified, plus 45 predicted transcription factors. Secreted factors were enriched in 3 ECC- and/or CON-responsive modules, with all 3 also being under the predicted transcriptional control of SP1 and KLF4. Of 34 candidate myokine hubs, 1 was also predicted to have elevated expression in skeletal muscle versus other tissues: THBS4, of a secretome-enriched module upregulated after untrained ECC. In conclusion, distinct untrained ECC and CON transcriptional responses are dampened after habituation without substantially shifting molecular functional profiles, providing new mechanistic candidates into contraction-mode specific muscle regulation.\",\n",
       "  'HPA_skeletal_muscle-specific-Symbol',\n",
       "  'and may thus potentially represent prominent myokine-related muscle regulatory pathways (Fig. 6, Supplementary File **3**)  Supplementary Information 4'),\n",
       " 'PMC10604446-12864_2023_9756_MOESM10_ESM.xlsx-S60_BRCA_Young-GENEID': ('Transcriptomic analysis reveals a tissue-specific loss of identity during ageing and cancer',\n",
       "  'Introduction: Understanding changes in cell identity in cancer and ageing is of great importance. In this work, we analyzed how gene expression changes in human tissues are associated with tissue specificity during cancer and ageing using transcriptome data from TCGA and GTEx. Results: We found significant downregulation of tissue-specific genes during ageing in 40% of the tissues analyzed, which suggests loss of tissue identity with age. For most cancer types, we have noted a consistent pattern of downregulation in genes that are specific to the tissue from which the tumor originated. Moreover, we observed in cancer an activation of genes not usually expressed in the tissue of origin as well as an upregulation of genes specific to other tissues. These patterns in cancer were associated with patient survival. The age of the patient, however, did not influence these patterns. Conclusion: We identified loss of cellular identity in 40% of the tissues analysed during human ageing, and a clear pattern in cancer, where during tumorigenesis cells express genes specific to other organs while suppressing the expression of genes from their original tissue. The loss of cellular identity observed in cancer is associated with prognosis and is not influenced by age, suggesting that it is a crucial stage in carcinogenesis.',\n",
       "  'S60_BRCA_Young-GENEID',\n",
       "  'Supplementary Material 10'),\n",
       " 'PMC8967925-42003_2022_3241_MOESM5_ESM.xlsx-Sheet2-Fig_4a': ('Resistance to chemical carcinogenesis induction via a dampened inflammatory response in naked mole-rats',\n",
       "  'Naked mole-rats (NMRs) have a very low spontaneous carcinogenesis rate, which has prompted studies on the responsible mechanisms to provide clues for human cancer prevention. However, it remains unknown whether and how NMR tissues respond to experimental carcinogenesis induction. Here, we show that NMRs exhibit extraordinary resistance against potent chemical carcinogenesis induction through a dampened inflammatory response. Although carcinogenic insults damaged skin cells of both NMRs and mice, NMR skin showed markedly lower immune cell infiltration. NMRs harbour loss-of-function mutations in RIPK3 and MLKL genes, which are essential for necroptosis, a type of necrotic cell death that activates strong inflammation. In mice, disruption of Ripk3 reduced immune cell infiltration and delayed carcinogenesis. Therefore, necroptosis deficiency may serve as a cancer resistance mechanism via attenuating the inflammatory response in NMRs. Our study sheds light on the importance of a dampened inflammatory response as a non-cell-autonomous cancer resistance mechanism in NMRs.',\n",
       "  'Sheet2-Fig_4a',\n",
       "  'underlying the graphs and charts in the main manuscript are provided as Supplementary Data **3**  Supplementary Data 3'),\n",
       " 'PMC8481972-mmc2.xlsx-Sheet1-Unnamed_5': (\"SPT6 loss permits the transdifferentiation of keratinocytes into an intestinal fate that resembles Barrett's metaplasia\",\n",
       "  \"Transient depletion of the transcription elongation factor SPT6 in the keratinocyte has been recently shown to inhibit epidermal differentiation and stratification; instead, they transdifferentiate into a gut-like lineage. We show here that this phenomenon of transdifferentiation recapitulates Barrett's metaplasia, the only human pathophysiologic condition in which a stratified squamous epithelium that is injured due to chronic acid reflux is trans-committed into an intestinal fate. The evidence we present here not only lend support to the notion that the keratinocytes are potentially the cell of origin of Barrett's metaplasia but also provide mechanistic insights linking transient acid exposure, downregulation of SPT6, stalled transcription of the master regulator of epidermal fate TP63, loss of epidermal fate, and metaplastic progression. Because Barrett's metaplasia in the esophagus is a pre-neoplastic condition with no preclinical human models, these findings have a profound impact on the modeling Barrett's metaplasia-in-a-dish.\",\n",
       "  'Sheet1-Unnamed_5',\n",
       "  '(H–J) Differentially expressed genes (DEGs; see **Data S1**)  Data S1. Genes that are differentially expressed between SPT6-depleted vs control human epidermal keratinocytes'),\n",
       " 'PMC8410838-41598_2021_96875_MOESM2_ESM.xls-Induced_Transcripts-Table_S1_Differentially_expressed_transcripts_in_High_ACE2_samples_from_the_CCLE_dataset': ('A virus-free cellular model recapitulates several features of severe COVID-19',\n",
       "  'As for all newly-emergent pathogens, SARS-CoV-2 presents with a relative paucity of clinical information and experimental models, a situation hampering both the development of new effective treatments and the prediction of future outbreaks. Here, we find that a simple virus-free model, based on publicly available transcriptional data from human cell lines, is surprisingly able to recapitulate several features of the clinically relevant infections. By segregating cell lines (n = 1305) from the CCLE project on the base of their sole angiotensin-converting enzyme 2 (ACE2) mRNA content, we found that overexpressing cells present with molecular features resembling those of at-risk patients, including senescence, impairment of antibody production, epigenetic regulation, DNA repair and apoptosis, neutralization of the interferon response, proneness to an overemphasized innate immune activity, hyperinflammation by IL-1, diabetes, hypercoagulation and hypogonadism. Likewise, several pathways were found to display a differential expression between sexes, with males being in the least advantageous position, thus suggesting that the model could reproduce even the sex-related disparities observed in the clinical outcome of patients with COVID-19. Overall, besides validating a new disease model, our data suggest that, in patients with severe COVID-19, a baseline ground could be already present and, as a consequence, the viral infection might simply exacerbate a variety of latent (or inherent) pre-existing conditions, representing therefore a tipping point at which they become clinically significant.',\n",
       "  'Induced_Transcripts-Table_S1_Differentially_expressed_transcripts_in_High_ACE2_samples_from_the_CCLE_dataset',\n",
       "  'RNA-seq data from **1**  Supplementary Table S1'),\n",
       " 'PMC6231036-mmc2.xlsx-DIB2_01_SuperSILAC_Proteome-Unnamed_2': ('Deep proteome of human nNOS/NOS1-positive versus MOCK SH-SY5Y neuroblastoma cells under full nutrition, serum free starvation and rapamycin treatment',\n",
       "  'Upregulations of neuronal nitric oxide synthase (nNOS/NOS1) in the mouse brain upon aging suggest a role in age-associated changes of protein homeostasis. We generated a cell model, in which constitutive expression of nNOS in SH-SY5Y cells at a level comparable to mouse brain replicates the aging phenotype i.e. slowing of cell proliferation, cell enlargement and expression of senescence markers. nNOS+ and MOCK cells were exposed to proteostasis stress by treatment with rapamycin or serum-free starvation. The proteomes were analyzed per SILAC or label-free using hybrid liquid chromatography/mass spectrometry (LC/MS). Full scan MS-data were acquired using Xcalibur, and raw mass spectra were analyzed using the proteomics software MaxQuant. The human reference proteome from uniprot was used as template to identify peptides and proteins and quantify protein expression. The DiB data file contains essential MaxQuant output tables and includes peptide and protein identification, accession numbers, protein and gene names, sequence coverage and quantification values of each sample. Differences in protein expression in MOCK versus nNOS+ SH-SY5Y cells and interpretation of results are presented in Valek et al. (2018). Raw mass spectra and MaxQuant output files have been deposited to the ProteomeXchange Consortium (Vizcaino et al., 2014) via the PRIDE partner repository with the dataset identifier PRIDE: PXD010538.',\n",
       "  'DIB2_01_SuperSILAC_Proteome-Unnamed_2',\n",
       "  'Supplementary material'),\n",
       " 'PMC10752590-elife-87340-data10.xlsx-Proteomics_Fig_6_Sup_Fig_7-Gene_names': ('Mitochondrial electron transport chain, ceramide, and coenzyme Q are linked in a pathway that drives insulin resistance in skeletal muscle',\n",
       "  'Insulin resistance (IR) is a complex metabolic disorder that underlies several human diseases, including type 2 diabetes and cardiovascular disease. Despite extensive research, the precise mechanisms underlying IR development remain poorly understood. Previously we showed that deficiency of coenzyme Q (CoQ) is necessary and sufficient for IR in adipocytes and skeletal muscle (Fazakerley et al., 2018). Here, we provide new insights into the mechanistic connections between cellular alterations associated with IR, including increased ceramides, CoQ deficiency, mitochondrial dysfunction, and oxidative stress. We demonstrate that elevated levels of ceramide in the mitochondria of skeletal muscle cells result in CoQ depletion and loss of mitochondrial respiratory chain components, leading to mitochondrial dysfunction and IR. Further, decreasing mitochondrial ceramide levels in vitro and in animal models (mice, C57BL/6J) (under chow and high-fat diet) increased CoQ levels and was protective against IR. CoQ supplementation also rescued ceramide-associated IR. Examination of the mitochondrial proteome from human muscle biopsies revealed a strong correlation between the respirasome system and mitochondrial ceramide as key determinants of insulin sensitivity. Our findings highlight the mitochondrial ceramide-CoQ-respiratory chain nexus as a potential foundation of an IR pathway that may also play a critical role in other conditions associated with ceramide accumulation and mitochondrial dysfunction, such as heart failure, cancer, and aging. These insights may have important clinical implications for the development of novel therapeutic strategies for the treatment of IR and related metabolic disorders.',\n",
       "  'Proteomics_Fig_6_Sup_Fig_7-Gene_names',\n",
       "  'complex I biogenesis were enriched as follows: athletes > lean > obese > T2D (**Source data 10**)  Human proteomics PairWiseComp'),\n",
       " 'PMC10752590-elife-87340-data7.xlsx-Proteomics_Fig_6_Sup_Fig_7-Gene_names': ('Mitochondrial electron transport chain, ceramide, and coenzyme Q are linked in a pathway that drives insulin resistance in skeletal muscle',\n",
       "  'Insulin resistance (IR) is a complex metabolic disorder that underlies several human diseases, including type 2 diabetes and cardiovascular disease. Despite extensive research, the precise mechanisms underlying IR development remain poorly understood. Previously we showed that deficiency of coenzyme Q (CoQ) is necessary and sufficient for IR in adipocytes and skeletal muscle (Fazakerley et al., 2018). Here, we provide new insights into the mechanistic connections between cellular alterations associated with IR, including increased ceramides, CoQ deficiency, mitochondrial dysfunction, and oxidative stress. We demonstrate that elevated levels of ceramide in the mitochondria of skeletal muscle cells result in CoQ depletion and loss of mitochondrial respiratory chain components, leading to mitochondrial dysfunction and IR. Further, decreasing mitochondrial ceramide levels in vitro and in animal models (mice, C57BL/6J) (under chow and high-fat diet) increased CoQ levels and was protective against IR. CoQ supplementation also rescued ceramide-associated IR. Examination of the mitochondrial proteome from human muscle biopsies revealed a strong correlation between the respirasome system and mitochondrial ceramide as key determinants of insulin sensitivity. Our findings highlight the mitochondrial ceramide-CoQ-respiratory chain nexus as a potential foundation of an IR pathway that may also play a critical role in other conditions associated with ceramide accumulation and mitochondrial dysfunction, such as heart failure, cancer, and aging. These insights may have important clinical implications for the development of novel therapeutic strategies for the treatment of IR and related metabolic disorders.',\n",
       "  'Proteomics_Fig_6_Sup_Fig_7-Gene_names',\n",
       "  'Correlation analysis of lipidomics and proteomics in human samples'),\n",
       " 'PMC8047639-Table_2.xlsx-Sheet1-Module_Black': ('A Nomogram Combining a Four-Gene Biomarker and Clinical Factors for Predicting Survival of Melanoma',\n",
       "  'Background: Currently there is no effective prognostic indicator for melanoma, the deadliest skin cancer. Thus, we aimed to develop and validate a nomogram predictive model for predicting survival of melanoma. Methods: Four hundred forty-nine melanoma cases with RNA sequencing (RNA-seq) data from TCGA were randomly divided into the training set I (n = 224) and validation set I (n = 225), 210 melanoma cases with RNA-seq data from Lund cohort of Lund University (available in GSE65904) were used as an external test set. The prognostic gene biomarker was developed and validated based on the above three sets. The developed gene biomarker combined with clinical characteristics was used as variables to develop and validate a nomogram predictive model based on 379 patients with complete clinical data from TCGA (Among 470 cases, 91 cases with missing clinical data were excluded from the study), which were randomly divided into the training set II (n = 189) and validation set II (n = 190). Area under the curve (AUC), concordance index (C-index), calibration curve, and Kaplan-Meier estimate were used to assess predictive performance of the nomogram model. Results: Four genes, i.e., CLEC7A , CLEC10A , HAPLN3 , and HCP5 comprise an immune-related prognostic biomarker. The predictive performance of the biomarker was validated using tROC and log-rank test in the training set I (n = 224, 5-year AUC of 0.683), validation set I (n = 225, 5-year AUC of 0.644), and test set I (n = 210, 5-year AUC of 0.645). The biomarker was also significantly associated with improved survival in the training set ( P < 0.01), validation set ( P < 0.05), and test set ( P < 0.001), respectively. In addition, a nomogram combing the four-gene biomarker and six clinical factors for predicting survival in melanoma was developed in the training set II (n = 189), and validated in the validation set II (n = 190), with a concordance index of 0.736 ± 0.041 and an AUC of 0.832 ± 0.071. Conclusion: We developed and validated a nomogram predictive model combining a four-gene biomarker and six clinical factors for melanoma patients, which could facilitate risk stratification and treatment planning.',\n",
       "  'Sheet1-Module_Black',\n",
       "  'training set II (n = 189) and validation set II (n = 190) ( **Table S2**)  Click here for additional data file'),\n",
       " 'PMC10752590-elife-87340-data9.xlsx-Proteomics_Fig_6_Sup_Fig_7-Gene_names': ('Mitochondrial electron transport chain, ceramide, and coenzyme Q are linked in a pathway that drives insulin resistance in skeletal muscle',\n",
       "  'Insulin resistance (IR) is a complex metabolic disorder that underlies several human diseases, including type 2 diabetes and cardiovascular disease. Despite extensive research, the precise mechanisms underlying IR development remain poorly understood. Previously we showed that deficiency of coenzyme Q (CoQ) is necessary and sufficient for IR in adipocytes and skeletal muscle (Fazakerley et al., 2018). Here, we provide new insights into the mechanistic connections between cellular alterations associated with IR, including increased ceramides, CoQ deficiency, mitochondrial dysfunction, and oxidative stress. We demonstrate that elevated levels of ceramide in the mitochondria of skeletal muscle cells result in CoQ depletion and loss of mitochondrial respiratory chain components, leading to mitochondrial dysfunction and IR. Further, decreasing mitochondrial ceramide levels in vitro and in animal models (mice, C57BL/6J) (under chow and high-fat diet) increased CoQ levels and was protective against IR. CoQ supplementation also rescued ceramide-associated IR. Examination of the mitochondrial proteome from human muscle biopsies revealed a strong correlation between the respirasome system and mitochondrial ceramide as key determinants of insulin sensitivity. Our findings highlight the mitochondrial ceramide-CoQ-respiratory chain nexus as a potential foundation of an IR pathway that may also play a critical role in other conditions associated with ceramide accumulation and mitochondrial dysfunction, such as heart failure, cancer, and aging. These insights may have important clinical implications for the development of novel therapeutic strategies for the treatment of IR and related metabolic disorders.',\n",
       "  'Proteomics_Fig_6_Sup_Fig_7-Gene_names',\n",
       "  'Correlation analysis between C18 ceramides and human muscle proteomics'),\n",
       " 'PMC4109854-pcbi.1003731.s010.xlsx-Nutlin_vs_Ctrl-downregulated': ('iRegulon: from a gene list to a gene regulatory network using large motif and track collections',\n",
       "  'Identifying master regulators of biological processes and mapping their downstream gene networks are key challenges in systems biology. We developed a computational method, called iRegulon, to reverse-engineer the transcriptional regulatory network underlying a co-expressed gene set using cis-regulatory sequence analysis. iRegulon implements a genome-wide ranking-and-recovery approach to detect enriched transcription factor motifs and their optimal sets of direct targets. We increase the accuracy of network inference by using very large motif collections of up to ten thousand position weight matrices collected from various species, and linking these to candidate human TFs via a motif2TF procedure. We validate iRegulon on gene sets derived from ENCODE ChIP-seq data with increasing levels of noise, and we compare iRegulon with existing motif discovery methods. Next, we use iRegulon on more challenging types of gene lists, including microRNA target sets, protein-protein interaction networks, and genetic perturbation data. In particular, we over-activate p53 in breast cancer cells, followed by RNA-seq and ChIP-seq, and could identify an extensive up-regulated network controlled directly by p53. Similarly we map a repressive network with no indication of direct p53 regulation but rather an indirect effect via E2F and NFY. Finally, we generalize our computational framework to include regulatory tracks such as ChIP-seq data and show how motif and track discovery can be combined to map functional regulatory interactions among co-expressed genes. iRegulon is available as a Cytoscape plugin from http://iregulon.aertslab.org.',\n",
       "  'Nutlin_vs_Ctrl-downregulated',\n",
       "  'the p53 regulon, with 307 genes predicted to be direct targets (Fig. 3A and **Table S2**)  Up- and down-regulated genes between Nutlin stimulated (S) vs non stimulated (NS) in MCF-7, with log fold changes and adjusted p-values. (XLSX)  Click here for additional data file'),\n",
       " 'PMC6961210-mmc4.xlsx-Multitissue_UP-Ensembl_Gene_ID': ('Ageing-associated changes in the expression of lncRNAs in human tissues reflect a transcriptional modulation in ageing pathways',\n",
       "  'Ageing-associated changes in the protein coding transcriptome have been extensively characterised, but less attention has been paid to the non-coding portion of the human genome, especially to long non-coding RNAs (lncRNAs). Only a minority of known lncRNAs have been functionally characterised; however, a handful of these lncRNAs have already been linked to ageing-associated processes. To gain more information on the effects of ageing on lncRNAs, we identified from GTEx data lncRNAs that show ageing-associated expression patterns (age-lncRNAs) in 29 human tissues in 20-79-year-old individuals. The age-lncRNAs identified were highly tissue-specific, but the protein coding genes co-expressed with the age-lncRNAs and the functional categories associated with the age-lncRNAs showed significant overlap across tissues. Functions associated with the age-lncRNAs, including immune system processes and transcription, were similar to what has previously been reported for protein coding genes with ageing-associated expression pattern. As the tissue-specific age-lncRNAs were associated with shared functions across tissues, they may reflect the tissue-specific fine-tuning of the common ageing-associated processes. The present study can be utilised as a resource when selecting and prioritising lncRNAs for further functional analyses.',\n",
       "  'Multitissue_UP-Ensembl_Gene_ID',\n",
       "  'None here'),\n",
       " 'PMC8987207-DataSheet_1.xls-Fig_1G-Beta-selection_UP': ('Monoallelic Heb/Tcf12 Deletion Reduces the Requirement for NOTCH1 Hyperactivation in T-Cell Acute Lymphoblastic Leukemia',\n",
       "  'Early T-cell development is precisely controlled by E proteins, that indistinguishably include HEB/TCF12 and E2A/TCF3 transcription factors, together with NOTCH1 and pre-T cell receptor (TCR) signalling. Importantly, perturbations of early T-cell regulatory networks are implicated in leukemogenesis. NOTCH1 gain of function mutations invariably lead to T-cell acute lymphoblastic leukemia (T-ALL), whereas inhibition of E proteins accelerates leukemogenesis. Thus, NOTCH1, pre-TCR, E2A and HEB functions are intertwined, but how these pathways contribute individually or synergistically to leukemogenesis remain to be documented. To directly address these questions, we leveraged Cd3e -deficient mice in which pre-TCR signaling and progression through β-selection is abrogated to dissect and decouple the roles of pre-TCR, NOTCH1, E2A and HEB in SCL/TAL1-induced T-ALL, via the use of Notch1 gain of function transgenic ( Notch1IC tg ) and Tcf12 +/- or Tcf3 +/- heterozygote mice. As a result, we now provide evidence that both HEB and E2A restrain cell proliferation at the β-selection checkpoint while the clonal expansion of SCL-LMO1-induced pre-leukemic stem cells in T-ALL is uniquely dependent on Tcf12 gene dosage. At the molecular level, HEB protein levels are decreased via proteasomal degradation at the leukemic stage, pointing to a reversible loss of function mechanism. Moreover, in SCL-LMO1 -induced T-ALL, loss of one Tcf12 allele is sufficient to bypass pre-TCR signaling which is required for Notch1 gain of function mutations and for progression to T-ALL. In contrast, Tcf12 monoallelic deletion does not accelerate Notch1IC -induced T-ALL, indicating that Tcf12 and Notch1 operate in the same pathway. Finally, we identify a tumor suppressor gene set downstream of HEB, exhibiting significantly lower expression levels in pediatric T-ALL compared to B-ALL and brain cancer samples, the three most frequent pediatric cancers. In summary, our results indicate a tumor suppressor function of HEB/TCF12 in T-ALL to mitigate cell proliferation controlled by NOTCH1 in pre-leukemic stem cells and prevent NOTCH1-driven progression to T-ALL.',\n",
       "  'Fig_1G-Beta-selection_UP',\n",
       "  'Advanced qPCR mix (Wisent). Primer sequences used for specific mRNA amplification are listed in **Supplemental Table S1**  Click here for additional data file'),\n",
       " 'PMC10752590-elife-87340-data8.xlsx-Proteomics_Fig_6_Sup_Fig_7-Gene_names': ('Mitochondrial electron transport chain, ceramide, and coenzyme Q are linked in a pathway that drives insulin resistance in skeletal muscle',\n",
       "  'Insulin resistance (IR) is a complex metabolic disorder that underlies several human diseases, including type 2 diabetes and cardiovascular disease. Despite extensive research, the precise mechanisms underlying IR development remain poorly understood. Previously we showed that deficiency of coenzyme Q (CoQ) is necessary and sufficient for IR in adipocytes and skeletal muscle (Fazakerley et al., 2018). Here, we provide new insights into the mechanistic connections between cellular alterations associated with IR, including increased ceramides, CoQ deficiency, mitochondrial dysfunction, and oxidative stress. We demonstrate that elevated levels of ceramide in the mitochondria of skeletal muscle cells result in CoQ depletion and loss of mitochondrial respiratory chain components, leading to mitochondrial dysfunction and IR. Further, decreasing mitochondrial ceramide levels in vitro and in animal models (mice, C57BL/6J) (under chow and high-fat diet) increased CoQ levels and was protective against IR. CoQ supplementation also rescued ceramide-associated IR. Examination of the mitochondrial proteome from human muscle biopsies revealed a strong correlation between the respirasome system and mitochondrial ceramide as key determinants of insulin sensitivity. Our findings highlight the mitochondrial ceramide-CoQ-respiratory chain nexus as a potential foundation of an IR pathway that may also play a critical role in other conditions associated with ceramide accumulation and mitochondrial dysfunction, such as heart failure, cancer, and aging. These insights may have important clinical implications for the development of novel therapeutic strategies for the treatment of IR and related metabolic disorders.',\n",
       "  'Proteomics_Fig_6_Sup_Fig_7-Gene_names',\n",
       "  'Correlation analysis between proteomics and human traits'),\n",
       " 'PMC10752590-elife-87340-data12.xlsx-Proteomics_Fig_6_Sup_Fig_7-Gene_names': ('Mitochondrial electron transport chain, ceramide, and coenzyme Q are linked in a pathway that drives insulin resistance in skeletal muscle',\n",
       "  'Insulin resistance (IR) is a complex metabolic disorder that underlies several human diseases, including type 2 diabetes and cardiovascular disease. Despite extensive research, the precise mechanisms underlying IR development remain poorly understood. Previously we showed that deficiency of coenzyme Q (CoQ) is necessary and sufficient for IR in adipocytes and skeletal muscle (Fazakerley et al., 2018). Here, we provide new insights into the mechanistic connections between cellular alterations associated with IR, including increased ceramides, CoQ deficiency, mitochondrial dysfunction, and oxidative stress. We demonstrate that elevated levels of ceramide in the mitochondria of skeletal muscle cells result in CoQ depletion and loss of mitochondrial respiratory chain components, leading to mitochondrial dysfunction and IR. Further, decreasing mitochondrial ceramide levels in vitro and in animal models (mice, C57BL/6J) (under chow and high-fat diet) increased CoQ levels and was protective against IR. CoQ supplementation also rescued ceramide-associated IR. Examination of the mitochondrial proteome from human muscle biopsies revealed a strong correlation between the respirasome system and mitochondrial ceramide as key determinants of insulin sensitivity. Our findings highlight the mitochondrial ceramide-CoQ-respiratory chain nexus as a potential foundation of an IR pathway that may also play a critical role in other conditions associated with ceramide accumulation and mitochondrial dysfunction, such as heart failure, cancer, and aging. These insights may have important clinical implications for the development of novel therapeutic strategies for the treatment of IR and related metabolic disorders.',\n",
       "  'Proteomics_Fig_6_Sup_Fig_7-Gene_names',\n",
       "  'Human proteomics analysis Rd_clamp_correlation_all_protein'),\n",
       " 'PMC9448736-41467_2022_32976_MOESM8_ESM.xlsx-MSigDB-Gene_name': ('PIM1 promotes hepatic conversion by suppressing reprogramming-induced ferroptosis and cell cycle arrest',\n",
       "  'Protein kinase-mediated phosphorylation plays a critical role in many biological processes. However, the identification of key regulatory kinases is still a great challenge. Here, we develop a trans-omics-based method, central kinase inference, to predict potentially key kinases by integrating quantitative transcriptomic and phosphoproteomic data. Using known kinases associated with anti-cancer drug resistance, the accuracy of our method denoted by the area under the curve is 5.2% to 29.5% higher than Kinase-Substrate Enrichment Analysis. We further use this method to analyze trans-omic data in hepatocyte maturation and hepatic reprogramming of human dermal fibroblasts, uncovering 5 kinases as regulators in the two processes. Further experiments reveal that a serine/threonine kinase, PIM1, promotes hepatic conversion and protects human dermal fibroblasts from reprogramming-induced ferroptosis and cell cycle arrest. This study not only reveals new regulatory kinases, but also provides a helpful method that might be extended to predict central kinases involved in other biological processes.',\n",
       "  'MSigDB-Gene_name',\n",
       "  'profiled the transcriptomes of HDFs undergoing hepatic conversion with PIM1 or GFP overexpression (Fig. **5**)  Supplementary Data 5'),\n",
       " 'PMC6961210-mmc4.xlsx-Multitissue_UP-Gene_Symbol': ('Ageing-associated changes in the expression of lncRNAs in human tissues reflect a transcriptional modulation in ageing pathways',\n",
       "  'Ageing-associated changes in the protein coding transcriptome have been extensively characterised, but less attention has been paid to the non-coding portion of the human genome, especially to long non-coding RNAs (lncRNAs). Only a minority of known lncRNAs have been functionally characterised; however, a handful of these lncRNAs have already been linked to ageing-associated processes. To gain more information on the effects of ageing on lncRNAs, we identified from GTEx data lncRNAs that show ageing-associated expression patterns (age-lncRNAs) in 29 human tissues in 20-79-year-old individuals. The age-lncRNAs identified were highly tissue-specific, but the protein coding genes co-expressed with the age-lncRNAs and the functional categories associated with the age-lncRNAs showed significant overlap across tissues. Functions associated with the age-lncRNAs, including immune system processes and transcription, were similar to what has previously been reported for protein coding genes with ageing-associated expression pattern. As the tissue-specific age-lncRNAs were associated with shared functions across tissues, they may reflect the tissue-specific fine-tuning of the common ageing-associated processes. The present study can be utilised as a resource when selecting and prioritising lncRNAs for further functional analyses.',\n",
       "  'Multitissue_UP-Gene_Symbol',\n",
       "  'None here'),\n",
       " 'PMC9533278-41380_2022_1786_MOESM11_ESM.xlsx-Gene_Signatures-Unnamed_2': ('SARS-CoV-2 promotes microglial synapse elimination in human brain organoids',\n",
       "  'Neuropsychiatric manifestations are common in both the acute and post-acute phase of SARS-CoV-2 infection, but the mechanisms of these effects are unknown. In a newly established brain organoid model with innately developing microglia, we demonstrate that SARS-CoV-2 infection initiate neuronal cell death and cause a loss of post-synaptic termini. Despite limited neurotropism and a decelerating viral replication, we observe a threefold increase in microglial engulfment of postsynaptic termini after SARS-CoV-2 exposure. We define the microglial responses to SARS-CoV-2 infection by single cell transcriptomic profiling and observe an upregulation of interferon-responsive genes as well as genes promoting migration and synapse engulfment. To a large extent, SARS-CoV-2 exposed microglia adopt a transcriptomic profile overlapping with neurodegenerative disorders that display an early synapse loss as well as an increased incident risk after a SARS-CoV-2 infection. Our results reveal that brain organoids infected with SARS-CoV-2 display disruption in circuit integrity via microglia-mediated synapse elimination and identifies a potential novel mechanism contributing to cognitive impairments in patients recovering from COVID-19.',\n",
       "  'Gene_Signatures-Unnamed_2',\n",
       "  'organoids exposed to an estimated MOI of 0.3, we then decreased the MOI to 0.**1**  Supplementary Table 1'),\n",
       " 'PMC8323949-pcbi.1009228.s007.xlsx-HNSCC_ALL_exceptLN-X2': ('Single-cell analysis reveals the pan-cancer invasiveness-associated transition of adipose-derived stromal cells into COL11A1-expressing cancer-associated fibroblasts',\n",
       "  'During the last ten years, many research results have been referring to a particular type of cancer-associated fibroblasts associated with poor prognosis, invasiveness, metastasis and resistance to therapy in multiple cancer types, characterized by a gene expression signature with prominent presence of genes COL11A1, THBS2 and INHBA. Identifying the underlying biological mechanisms responsible for their creation may facilitate the discovery of targets for potential pan-cancer therapeutics. Using a novel computational approach for single-cell gene expression data analysis identifying the dominant cell populations in a sequence of samples from patients at various stages, we conclude that these fibroblasts are produced by a pan-cancer cellular transition originating from a particular type of adipose-derived stromal cells naturally present in the stromal vascular fraction of normal adipose tissue, having a characteristic gene expression signature. Focusing on a rich pancreatic cancer dataset, we provide a detailed description of the continuous modification of the gene expression profiles of cells as they transition from APOD-expressing adipose-derived stromal cells to COL11A1-expressing cancer-associated fibroblasts, identifying the key genes that participate in this transition. These results also provide an explanation to the well-known fact that the adipose microenvironment contributes to cancer progression.',\n",
       "  'HNSCC_ALL_exceptLN-X2',\n",
       "  'MMP11, INHBA, THBS2, COL10A1, COL8A1, FN1 and ASC associated genes (APOD, C7, PTGDS), respectively (**S4 Table**)  Differentially expressed genes among different clusters of HNSCC dataset.(XLSX)  Click here for additional data file'),\n",
       " 'PMC8323949-pcbi.1009228.s007.xlsx-X2-Unnamed_0': ('Single-cell analysis reveals the pan-cancer invasiveness-associated transition of adipose-derived stromal cells into COL11A1-expressing cancer-associated fibroblasts',\n",
       "  'During the last ten years, many research results have been referring to a particular type of cancer-associated fibroblasts associated with poor prognosis, invasiveness, metastasis and resistance to therapy in multiple cancer types, characterized by a gene expression signature with prominent presence of genes COL11A1, THBS2 and INHBA. Identifying the underlying biological mechanisms responsible for their creation may facilitate the discovery of targets for potential pan-cancer therapeutics. Using a novel computational approach for single-cell gene expression data analysis identifying the dominant cell populations in a sequence of samples from patients at various stages, we conclude that these fibroblasts are produced by a pan-cancer cellular transition originating from a particular type of adipose-derived stromal cells naturally present in the stromal vascular fraction of normal adipose tissue, having a characteristic gene expression signature. Focusing on a rich pancreatic cancer dataset, we provide a detailed description of the continuous modification of the gene expression profiles of cells as they transition from APOD-expressing adipose-derived stromal cells to COL11A1-expressing cancer-associated fibroblasts, identifying the key genes that participate in this transition. These results also provide an explanation to the well-known fact that the adipose microenvironment contributes to cancer progression.',\n",
       "  'X2-Unnamed_0',\n",
       "  'MMP11, INHBA, THBS2, COL10A1, COL8A1, FN1 and ASC associated genes (APOD, C7, PTGDS), respectively (**S4 Table**)  Differentially expressed genes among different clusters of HNSCC dataset.(XLSX)  Click here for additional data file'),\n",
       " 'PMC10956522-peerj-12-17002-s010.xlsx-Supplmentray_table_3-probes': ('Integrating single-cell and bulk sequencing data to identify glycosylation-based genes in non-alcoholic fatty liver disease-associated hepatocellular carcinoma',\n",
       "  'Background: The incidence of non-alcoholic fatty liver disease (NAFLD) associated hepatocellular carcinoma (HCC) has been increasing. However, the role of glycosylation, an important modification that alters cellular differentiation and immune regulation, in the progression of NAFLD to HCC is rare. Methods: We used the NAFLD-HCC single-cell dataset to identify variation in the expression of glycosylation patterns between different cells and used the HCC bulk dataset to establish a link between these variations and the prognosis of HCC patients. Then, machine learning algorithms were used to identify those glycosylation-related signatures with prognostic significance and to construct a model for predicting the prognosis of HCC patients. Moreover, it was validated in high-fat diet-induced mice and clinical cohorts. Results: The NAFLD-HCC Glycogene Risk Model (NHGRM) signature included the following genes: SPP1, SOCS2, SAPCD2, S100A9, RAMP3, and CSAD. The higher NHGRM scores were associated with a poorer prognosis, stronger immune-related features, immune cell infiltration and immunity scores. Animal experiments, external and clinical cohorts confirmed the expression of these genes. Conclusion: The genetic signature we identified may serve as a potential indicator of survival in patients with NAFLD-HCC and provide new perspectives for elucidating the role of glycosylation-related signatures in this pathologic process.',\n",
       "  'Supplmentray_table_3-probes',\n",
       "  'None here'),\n",
       " 'PMC9533278-41380_2022_1786_MOESM11_ESM.xlsx-Gene_Signatures-Unnamed_1': ('SARS-CoV-2 promotes microglial synapse elimination in human brain organoids',\n",
       "  'Neuropsychiatric manifestations are common in both the acute and post-acute phase of SARS-CoV-2 infection, but the mechanisms of these effects are unknown. In a newly established brain organoid model with innately developing microglia, we demonstrate that SARS-CoV-2 infection initiate neuronal cell death and cause a loss of post-synaptic termini. Despite limited neurotropism and a decelerating viral replication, we observe a threefold increase in microglial engulfment of postsynaptic termini after SARS-CoV-2 exposure. We define the microglial responses to SARS-CoV-2 infection by single cell transcriptomic profiling and observe an upregulation of interferon-responsive genes as well as genes promoting migration and synapse engulfment. To a large extent, SARS-CoV-2 exposed microglia adopt a transcriptomic profile overlapping with neurodegenerative disorders that display an early synapse loss as well as an increased incident risk after a SARS-CoV-2 infection. Our results reveal that brain organoids infected with SARS-CoV-2 display disruption in circuit integrity via microglia-mediated synapse elimination and identifies a potential novel mechanism contributing to cognitive impairments in patients recovering from COVID-19.',\n",
       "  'Gene_Signatures-Unnamed_1',\n",
       "  'organoids exposed to an estimated MOI of 0.3, we then decreased the MOI to 0.**1**  Supplementary Table 1'),\n",
       " 'PMC5390115-mmc4.xlsx-DE_Lists-Unnamed_35': ('A Genome-wide Analysis of Human Pluripotent Stem Cell-Derived Endothelial Cells in 2D or 3D Culture',\n",
       "  'A defined protocol for efficiently deriving endothelial cells from human pluripotent stem cells was established and vascular morphogenesis was used as a model system to understand how synthetic hydrogels influence global biological function compared with common 2D and 3D culture platforms. RNA sequencing demonstrated that gene expression profiles were similar for endothelial cells and pericytes cocultured in polyethylene glycol (PEG) hydrogels or Matrigel, while monoculture comparisons identified distinct vascular signatures for each cell type. Endothelial cells cultured on tissue-culture polystyrene adopted a proliferative phenotype compared with cells cultured on or encapsulated in PEG hydrogels. The proliferative phenotype correlated to increased FAK-ERK activity, and knockdown or inhibition of ERK signaling reduced proliferation and expression for cell-cycle genes while increasing expression for \"3D-like\" vasculature development genes. Our results provide insight into the influence of 2D and 3D culture formats on global biological processes that regulate cell function.',\n",
       "  'DE_Lists-Unnamed_35',\n",
       "  '(on the same day) than for comparisons between PEG hydrogels with different crosslinking densities (**Table S3**)  Table S3. Differential Gene Expression Analysis for H1-EC-PC Cocultures, Related to Figure 2'),\n",
       " 'PMC10302540-mmc5.xlsx-G5_VE-Names_of_the_transcripts_with_Gi5_0_02': ('Overexpressed kinetochore genes are used by cancer cells as genome destabilizers and transformation catalysts',\n",
       "  'Cancer cells have an altered transcriptome, which contributes to their abnormal behavior. Many tumors have high levels of kinetochore genes, which play important roles in genome stability. This overexpression could be utilized to destabilize cancer cell genomes, however this has not been proven specifically. We investigated the link between kinetochore gene overexpression, chromosomal number variations (CNVs) and genomic instability. Data on RNA expression and CNV from 12 different cancer types were evaluated using information theory. In all cancer types, we looked at the relationship between RNA expression and CNVs. Kinetochore gene expression was found to be substantially linked with CNV levels. In all cancer types, with the exception of thyroid cancer, highly expressed kinetochore genes were enriched in the most dominant cancer-specific co-expression subnetworks characterizing the largest patient subgroups. Except for thyroid cancer, kinetochore inner protein CENPA was among the transcripts most strongly associated with CNV values in all cancer types studied, with significantly higher expression levels in patients with high CNVs than in patients with low CNVs. CENPA function was investigated further in cell models by transfecting genomically stable (HCT116) and unstable (MCF7 and HT29) cancer cell lines using CENPA overexpression vectors. This overexpression increased the number of abnormal cell divisions in the stable cancer cell line HCT116 and, to a lesser extent, in the unstable cell lines MCF7 and HT29. Overexpression improved anchorage-independent growth properties of all cell lines. Our findings suggest that overexpression of kinetochore genes in general, and CENPA in particular, can cause genomic instability and cancer progression.',\n",
       "  'G5_VE-Names_of_the_transcripts_with_Gi5_0_02',\n",
       "  'Table S5'),\n",
       " 'PMC9800263-mmc2.xlsx-Upregulated_in_PC3_SCM-Gene_Name': ('Transcriptomic analysis of the innate immune response to in vitro transfection of plasmid DNA',\n",
       "  'The innate immune response to cytosolic DNA is intended to protect the host from viral infections, but it can also inhibit the delivery and expression of therapeutic transgenes in gene and cell therapies. The goal of this work was to use mRNA sequencing to identify genes that may influence transfection efficiency in four different cell types (PC-3, Jurkat, HEK-293T, and primary T cells). The highest transfection efficiency was observed in HEK-293T cells, which upregulated only 142 genes with no known antiviral functions after transfection with lipofectamine. Lipofection upregulated 1,057 cytokine-stimulated genes (CSGs) in PC-3 cells, which exhibited a significantly lower transfection efficiency. However, when PC-3 cells were transfected in serum-containing media or electroporated, the observed transfection efficiencies were significantly higher while the expression levels of cytokines and CSGs decreased. In contrast, lipofection of Jurkat and primary T cells only upregulated a few genes, but several of the antiviral CSGs that were absent in HEK-293T cells and upregulated in PC-3 cells were observed to be constitutively expressed in T cells, which may explain the relatively low Lipofection efficiencies observed with T cells (8%-21% GFP + ). Indeed, overexpression of one CSG (IFI16) significantly decreased transfection efficiency in HEK-293T cells.',\n",
       "  'Upregulated_in_PC3_SCM-Gene_Name',\n",
       "  'complete lists of the differentially expressed genes (DEGs) are included in the supplemental information (**Table S8**)  Table S8. Lists of all the upregulated and downregulated genes for each type of gene delivery experiment mentioned in the manuscript, along with their corresponding padj and TPM values'),\n",
       " 'PMC7887593-ADVS-8-2003526-s004.xlsx-goblet_cell-Goblet_cell_gene_name_n_401': ('In Well-Treated Celiac Patients Low-Level Mucosal Inflammation Predicts Response to 14-day Gluten Challenge',\n",
       "  'In celiac disease (CeD), gluten activates adaptive immune cells that cause damage to the small intestinal mucosa. Histological evaluation of intestinal biopsies allows for grading of disease severity. CeD can effectively be treated with a life-long gluten-free diet. Gluten challenge of treated CeD patients is used to confirm diagnosis and to test drug efficacy in clinical trials, but patients respond with different magnitudes to the same gluten challenge. In this study of 19 well-treated CeD patients, proteome analysis of total tissue or isolated epithelial cell compartment from formalin-fixed paraffin embedded biopsies collected before and after 14-day gluten challenge demonstrates that patients with strong mucosal response to challenge have signs of ongoing tissue inflammation already before challenge. This low-level tissue inflammation at baseline is paralleled by increased gluten specific CD4+ T-cell frequencies in the gut and presence of a low-level blood inflammatory profile. Thus, apparently well-treated CeD is frequently not entirely quiescent, with presence of low-grade inflammation and antigluten immunity in the gut mucosa. Histology assessment alone appears insufficient to judge full recovery and gut mucosal healing of CeD patients. The findings raise a concern whether a seemingly proper gluten-free diet is able to curb gut inflammation in all CeD patients.',\n",
       "  'goblet_cell-Goblet_cell_gene_name_n_401',\n",
       "  'Supplemental Table 4  Click here for additional data file'),\n",
       " 'PMC8086068-13073_2021_880_MOESM3_ESM.xlsx-c_Co-expressed_genes-c_Genes_co-expressed_positively_or_negatively_with_miRNAs_of_each_cluster': ('Crosstalk between microRNA expression and DNA methylation drives the hormone-dependent phenotype of breast cancer',\n",
       "  'Background: Abnormal DNA methylation is observed as an early event in breast carcinogenesis. However, how such alterations arise is still poorly understood. microRNAs (miRNAs) regulate gene expression at the post-transcriptional level and play key roles in various biological processes. Here, we integrate miRNA expression and DNA methylation at CpGs to study how miRNAs may affect the breast cancer methylome and how DNA methylation may regulate miRNA expression. Methods: miRNA expression and DNA methylation data from two breast cancer cohorts, Oslo2 (n = 297) and The Cancer Genome Atlas (n = 439), were integrated through a correlation approach that we term miRNA-methylation Quantitative Trait Loci (mimQTL) analysis. Hierarchical clustering was used to identify clusters of miRNAs and CpGs that were further characterized through analysis of mRNA/protein expression, clinicopathological features, in silico deconvolution, chromatin state and accessibility, transcription factor binding, and long-range interaction data. Results: Clustering of the significant mimQTLs identified distinct groups of miRNAs and CpGs that reflect important biological processes associated with breast cancer pathogenesis. Notably, two major miRNA clusters were related to immune or fibroblast infiltration, hence identifying miRNAs associated with cells of the tumor microenvironment, while another large cluster was related to estrogen receptor (ER) signaling. Studying the chromatin landscape surrounding CpGs associated with the estrogen signaling cluster, we found that miRNAs from this cluster are likely to be regulated through DNA methylation of enhancers bound by FOXA1, GATA2, and ER-alpha. Further, at the hub of the estrogen cluster, we identified hsa-miR-29c-5p as negatively correlated with the mRNA and protein expression of DNA methyltransferase DNMT3A, a key enzyme regulating DNA methylation. We found deregulation of hsa-miR-29c-5p already present in pre-invasive breast lesions and postulate that hsa-miR-29c-5p may trigger early event abnormal DNA methylation in ER-positive breast cancer. Conclusions: We describe how miRNA expression and DNA methylation interact and associate with distinct breast cancer phenotypes.',\n",
       "  'c_Co-expressed_genes-c_Genes_co-expressed_positively_or_negatively_with_miRNAs_of_each_cluster',\n",
       "  '= 297 and The Cancer Genome Atlas (TCGA) Breast Invasive Carcinoma cohort (n = 4**3**)  Additional file 3. a) Overview and annotation of the 119 mimQTL miRNAs. b) Overview and annotation of the 26,746 mimQTL CpGs. c) Genes found positively or negatively correlated to miRNAs of each cluster (mRNA-miRNA Spearman correlation > 0.4 or <-0.3, respectively, in both the Oslo2 and TCGA cohorts). d) Enrichr [33] Pathway enrichment (KEGG 2019 Human database) of genes positively (top) and negatively (bottom) correlated to miRNAs of each cluster. e) miRNA expression correlation to the Nanodissect [34] lymphocyte infiltration score, the xCell [35] fibroblast infiltration score, and ESR1 mRNA expression. f) Differential expression of miRNAs between clinically relevant breast cancer groups. g) Results from generalized linear modeling (GLM) of miRNA expression as a multivariate function of lymphocyte and fibroblast infiltration and ESR1 mRNA expression. h) Pathway enrichment using Enrichr [33] of genes mapped to the CpGs of cluster 1 according to the Illumina HumanMethylation450k array. i) ChromHMM [37] enrichment of genomic regions mapped to the CpGs of cluster 1 and 2. j) Pathway enrichment using Enrichr [33] of genes mapped to the CpGs of cluster 2 according to the Illumina HumanMethylation450k array. k) Table of 273 unique mimQTL associations with 69 unique CpGs residing in miRNA super-enhancer regions. l) Long-range loops overlapping with mimQTLs residing on the same chromosome. m) Correlation between the Global Methylation Alteration (GMA) score and 119 mimQTL miRNAs in the Oslo2 and TCGA cohorts. n) TargetScan [52] in silico predicted miRNA-target interactions for Global Methylation Alteration (GMA) score-correlated miRNAs and epigenetic regulator genes (conserved and non-conserved sites considered)'),\n",
       " 'PMC9734136-41419_2022_5488_MOESM17_ESM.xlsx-Sheet8-Gene_Name': ('circPLIN2 promotes clear cell renal cell carcinoma progression by binding IGF2BP proteins and miR-199a-3p',\n",
       "  'Recent evidence has indicated that circular RNAs (circRNAs), a novel type of regulatory RNA, play important roles in the development and progression of various cancers. However, the potential regulatory roles and molecular mechanisms of circRNAs in clear cell renal cell carcinoma (ccRCC) remain largely unclear. Here, we explored circRNA expression profiles in 10 paired samples of RCC (including cancer tissues and surrounding tissues) from the Gene Expression Omnibus (GEO) datasets GSE124453 and GSE108735. We initially identified hsa_circ_0086457, designated circPLIN2, derived from exons 4 to 5 of the PLIN2 gene. We observed that circPLIN2 was preferentially located in the cytoplasm and was more stable than its linear counterpart PLIN2. circPLIN2 was significantly upregulated in ccRCC cells and tissues, and its overexpression was correlated with higher clinical stage and worse prognosis for ccRCC patients. Moreover, gain- and loss-of-function assays indicated that circPLIN2 promoted ccRCC cell proliferation, migration, and invasion in vitro and ccRCC tumor growth and metastasis in vivo. Mechanistically, circPLIN2 not only increased the stability of the c-Myc and MARCKSL1 mRNAs by binding to the KH domains of IGF2BP proteins but also competitively sponged miR-199a-3p to abolish the repressive effect of miR-199a-3p on ZEB1 expression, which ultimately resulted in ccRCC tumorigenesis and progression. Collectively, our results suggest that circPLIN2 may represent a promising diagnostic and prognostic biomarker and a potential therapeutic target for ccRCC patients.',\n",
       "  'Sheet8-Gene_Name',\n",
       "  'Supplementary LC-MS Results_MS2_Vector_2'),\n",
       " 'PMC8993817-41467_2022_29562_MOESM5_ESM.xlsx-DEG_Chip_vs_human-Increase_human': ('Mechanical control of innate immune responses against viral infection revealed in a human lung alveolus chip',\n",
       "  'Mechanical breathing motions have a fundamental function in lung development and disease, but little is known about how they contribute to host innate immunity. Here we use a human lung alveolus chip that experiences cyclic breathing-like deformations to investigate whether physical forces influence innate immune responses to viral infection. Influenza H3N2 infection of mechanically active chips induces a cascade of host responses including increased lung permeability, apoptosis, cell regeneration, cytokines production, and recruitment of circulating immune cells. Comparison with static chips reveals that breathing motions suppress viral replication by activating protective innate immune responses in epithelial and endothelial cells, which are mediated in part through activation of the mechanosensitive ion channel TRPV4 and signaling via receptor for advanced glycation end products (RAGE). RAGE inhibitors suppress cytokines induction, while TRPV4 inhibition attenuates both inflammation and viral burden, in infected chips with breathing motions. Therefore, TRPV4 and RAGE may serve as new targets for therapeutic intervention in patients infected with influenza and other potential pandemic viruses that cause life-threatening lung inflammation.',\n",
       "  'DEG_Chip_vs_human-Increase_human',\n",
       "  'also seen in human lung during the transition from fetus to birth (Supplementary Data **3**)  Supplementary Data 3'),\n",
       " 'PMC9023741-Table4.xlsx-DEG_inf_vs_con-Unnamed_0': ('Digital Spatial Profiling Reveals Functional Shift of Enterochromaffin Cell in Patients With Ulcerative Colitis',\n",
       "  \"As a major component of the enteroendocrine system, enterochromaffin (EC) cells play a key role in ulcerative colitis (UC). However, the scarcity of EC cells has limited the investigation of their function. In this study, we applied digital spatial profiling to acquire transcriptomic data for EC cells and other epithelial cells from colonoscopic biopsy samples from eight patients with UC and seven healthy controls. Differential expression analysis, gene set enrichment analysis, and weighted gene coexpression network analysis were performed to identify differentially expressed genes and pathways and coexpression networks. Results were validated using an online dataset obtained by single-cell RNA sequencing, along with immunofluorescence staining and quantitative real-time PCR. In healthy participants, 10 genes were significantly enriched in EC cells, functionally concentrated in protein and bioamine synthesis. A coexpression network containing 17 hub genes, including TPH1 , CHGA , and GCLC , was identified in EC cells. In patients with UC, EC cells gained increased capacity for protein synthesis, along with novel immunological functions such as antigen processing and presentation, whereas chemical sensation was downregulated. The specific expression of CHGB and RGS2 in EC cells was confirmed by immunofluorescence staining. Our results illuminate the transcriptional signatures of EC cells in the human colon. EC cells' newly observed functional shift from sensation to secretion and immunity indicates their pivotal role in UC.\",\n",
       "  'DEG_inf_vs_con-Unnamed_0',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC11005406-mbio.02623-23-s0006.xlsx-DEG_IFNB_vs_Media_IFNLR-Name': ('Interferon lambda restricts herpes simplex virus skin disease by suppressing neutrophil-mediated pathology',\n",
       "  'Type III interferons (IFN-λ) are antiviral and immunomodulatory cytokines that have been best characterized in respiratory and gastrointestinal infections, but the effects of IFN-λ against skin infections have not been extensively investigated. We sought to define the skin-specific effects of IFN-λ against the highly prevalent human pathogen, herpes simplex virus (HSV). We infected mice lacking the IFN-λ receptor ( Ifnlr1 -/- ), both the IFN-λ and the IFN-α/β receptors ( Ifnar1 -/- Ifnlr1 -/- ), or IFN-λ cytokines ( Ifnl2/3 -/- ) and found that IFN-λ restricts the severity of HSV-1 and HSV-2 skin lesions without affecting viral loads. We used RNAseq to define IFN-λ- and IFN-β-induced transcriptional responses in primary mouse keratinocytes. Using conditional knockout mice, we found that IFN-λ signaling in both keratinocytes and neutrophils was necessary to control HSV-1 skin lesion severity and that IFN-λ signaling in keratinocytes suppressed CXCL9-mediated neutrophil recruitment to the skin. Furthermore, depleting neutrophils or blocking CXCL9 protected against severe HSV-1 skin lesions in Ifnlr1 -/- mice. Altogether, our results suggest that IFN-λ plays an immunomodulatory role in the skin that restricts neutrophil-mediated pathology during HSV infection and suggests potential applications for IFN-λ in treating viral skin infections.IMPORTANCEType III interferons (IFN-λ) have been shown to have antiviral and immunomodulatory effects at epithelial barriers such as the respiratory and gastrointestinal tracts, but their effects on the skin have not been extensively investigated. We used mice lacking IFN-λ signaling to investigate the skin-specific effects of IFN-λ against the herpes simplex virus (HSV), which targets epithelial tissues to cause cold sores and genital herpes. We found that IFN-λ limited the severity of HSV skin lesions without affecting viral load and that this protective effect required IFN-λ signaling in both keratinocytes and neutrophils. We found that IFN-λ signaling in keratinocytes suppressed neutrophil recruitment to the skin and that depleting neutrophils protected against severe HSV skin lesions in the absence of IFN-λ. Altogether, our results suggest that IFN-λ plays an immunomodulatory role in the skin that restricts neutrophil-mediated pathology during HSV infection and suggests potential applications for IFN-λ in treating viral skin infections.',\n",
       "  '23-s0006.xlsx-DEG_IFNB_vs_Media_IFNLR-Name',\n",
       "  'None here'),\n",
       " 'PMC8563012-elife-70692-supp2.xlsx-table3B-geneR1A_uniq': ('RNA splicing programs define tissue compartments and cell types at single-cell resolution',\n",
       "  'The extent splicing is regulated at single-cell resolution has remained controversial due to both available data and methods to interpret it. We apply the SpliZ, a new statistical approach, to detect cell-type-specific splicing in >110K cells from 12 human tissues. Using 10X Chromium data for discovery, 9.1% of genes with computable SpliZ scores are cell-type-specifically spliced, including ubiquitously expressed genes MYL6 and RPS24 . These results are validated with RNA FISH, single-cell PCR, and Smart-seq2. SpliZ analysis reveals 170 genes with regulated splicing during human spermatogenesis, including examples conserved in mouse and mouse lemur. The SpliZ allows model-based identification of subpopulations indistinguishable based on gene expression, illustrated by subpopulation-specific splicing of classical monocytes involving an ultraconserved exon in SAT1 . Together, this analysis of differential splicing across multiple organs establishes that splicing is regulated cell-type-specifically.',\n",
       "  'table3B-geneR1A_uniq',\n",
       "  'of 1353 of genes with computable SpliZ scores as having consistent compartment-specific splicing effects (**Supplementary file 2**)  Differential alternative splicing per compartment.Separate table with the p-value based on the SpliZ and SpliZVD for each gene in each dataset, testing differences between compartments. The gene name, SpliZ p-value, SpliZVD p-value, and largest magnitude median for all compartments for SpliZ and SpliZVD for each gene are given by the geneR1A_uniq, perm_pval_adj_scZ, perm_pval_adj_svd_z0, max_abs_median_scZ, and max_abs_median_svd_z0 columns, respectively. (A) Human individual 1 10X; (B) human individual 2 10X; (C) human individual 1 SS2; (D) human individual 2 SS2; (E) lemur individual 1 10X; (F) lemur individual 2 10X; (G) mouse individual 1 10X; (H) mouse individual 2 10X'),\n",
       " 'PMC9062249-mmc2.xlsx-Combined-Alternate_ID': ('Spirocyclic dimer SpiD7 activates the unfolded protein response to selectively inhibit growth and induce apoptosis of cancer cells',\n",
       "  'The unfolded protein response (UPR) is an adaptation mechanism activated to resolve transient accumulation of unfolded/misfolded proteins in the endoplasmic reticulum. Failure to resolve the transient accumulation of such proteins results in UPR-mediated programmed cell death. Loss of tumor suppressor gene or oncogene addiction in cancer cells can result in sustained higher basal UPR levels; however, it is not clear if these higher basal UPR levels in cancer cells can be exploited as a therapeutic strategy. We hypothesized that covalent modification of surface-exposed cysteine (SEC) residues could simulate unfolded/misfolded proteins to activate the UPR, and that higher basal UPR levels in cancer cells would provide the necessary therapeutic window. To test this hypothesis, here we synthesized analogs that can covalently modify multiple SEC residues and evaluated them as UPR activators. We identified a spirocyclic dimer, SpiD7, and evaluated its effects on UPR activation signals, that is, XBP1 splicing, phosphorylation of eIF2α, and a decrease in ATF 6 levels, in normal and cancer cells, which were further confirmed by RNA-Seq analyses. We found that SpiD7 selectively induced caspase-mediated apoptosis in cancer cells, whereas normal cells exhibited robust XBP1 splicing, indicating adaptation to stress. Furthermore, SpiD7 inhibited the growth of high-grade serous carcinoma cell lines ~3-15-fold more potently than immortalized fallopian tube epithelial (paired normal control) cells and reduced clonogenic growth of high-grade serous carcinoma cell lines. Our results suggest that induction of the UPR by covalent modification of SEC residues represents a cancer cell vulnerability and can be exploited to discover novel therapeutics.',\n",
       "  'Combined-Alternate_ID',\n",
       "  'Supplemental Table S1'),\n",
       " 'PMC7678224-12977_2020_544_MOESM2_ESM.xlsx-Micoglia_Signature-Feature': ('Comparative analysis of human microglial models for studies of HIV replication and pathogenesis',\n",
       "  'Background: HIV associated neurocognitive disorders cause significant morbidity and mortality despite the advent of highly active antiretroviral therapy. A deeper understanding of fundamental mechanisms underlying HIV infection and pathogenesis in the central nervous system is warranted. Microglia are resident myeloid cells of the brain that are readily infected by HIV and may constitute a CNS reservoir. We evaluated two microglial model cell lines (C20, HMC3) and two sources of primary cell-derived microglia (monocyte-derived microglia [MMG] and induced pluripotent stem cell-derived microglia [iPSC-MG]) as potential model systems for studying HIV-microglia interactions. Results: All four microglial model cells expressed typical myeloid markers with the exception of low or absent CD45 and CD11b expression by C20 and HMC3, and all four expressed the microglia-specific markers P2RY12 and TMEM119. Marked differences were observed upon gene expression profiling, however, indicating that MMG and iPSC-MG cluster closely together with primary human microglial cells, while C20 and HMC3 were similar to each other but very different from primary microglia. Expression of HIV-relevant genes also revealed important differences, with iPSC-MG and MMG expressing relevant genes at levels more closely resembling primary microglia. iPSC-MG and MMG were readily infected with R5-tropic HIV, while C20 and HMC3 lack CD4 and require pseudotyping for infection. Despite many similarities, HIV replication dynamics and HIV-1 particle capture by Siglec-1 differed markedly between the MMG and iPSC-MG. Conclusions: MMG and iPSC-MG appear to be viable microglial models that are susceptible to HIV infection and bear more similarities to authentic microglia than two transformed microglia cell lines. The observed differences in HIV replication and particle capture between MMG and iPSC-MG warrant further study.',\n",
       "  'Micoglia_Signature-Feature',\n",
       "  'results from the current study as iPSC-MG1 and those from the Blurton-Jones laboratory as iPSC-MG**2**  Additional file 2: Table S1. Table of 780 microglia-enriched genes as outlined in the text, related to Fig. 4b'),\n",
       " 'PMC7688486-NIHMS1600952-supplement-8.xlsx-DOWN-gene_name': ('CARM1 inhibition reduces histone acetyltransferase activity causing synthetic lethality in CREBBP/EP300-mutated lymphomas',\n",
       "  'Somatic mutations affecting CREBBP and EP300 are a hallmark of diffuse large B-cell lymphoma (DLBCL). These mutations are frequently monoallelic, within the histone acetyltransferase (HAT) domain and usually mutually exclusive, suggesting that they might affect a common pathway, and their residual WT expression is required for cell survival. Using in vitro and in vivo models, we found that inhibition of CARM1 activity (CARM1i) slows DLBCL growth, and that the levels of sensitivity are positively correlated with the CREBBP/EP300 mutation load. Conversely, treatment of DLBCLs that do not have CREBBP/EP300 mutations with CARM1i and a CBP/p300 inhibitor revealed a strong synergistic effect. Our mechanistic data show that CARM1i further reduces the HAT activity of CBP genome wide and downregulates CBP-target genes in DLBCL cells, resulting in a synthetic lethality that leverages the mutational status of CREBBP/EP300 as a biomarker for the use of small-molecule inhibitors of CARM1 in DLBCL and other cancers.',\n",
       "  'DOWN-gene_name',\n",
       "  'collect enough live cells to proceed with downstream studies (Figure 1C). We found approximately **2**'),\n",
       " 'PMC11257952-43587_2024_640_MOESM4_ESM.xlsx-Tab3_-_Exc_upregulated_genes-EXC6': ('Single-nucleus sequencing reveals enriched expression of genetic risk factors in extratelencephalic neurons sensitive to degeneration in ALS',\n",
       "  'Amyotrophic lateral sclerosis (ALS) is a neurodegenerative disorder characterized by a progressive loss of motor function linked to degenerating extratelencephalic neurons/Betz cells (ETNs). The reasons why these neurons are selectively affected remain unclear. Here, to understand the unique molecular properties that may sensitize ETNs to ALS, we performed RNA sequencing of 79,169 single nuclei from cortices of patients and controls. In both patients and unaffected individuals, we found significantly higher expression of ALS risk genes in THY1 + ETNs, regardless of diagnosis. In patients, this was accompanied by the induction of genes involved in protein homeostasis and stress responses that were significantly induced in a wide collection of ETNs. Examination of oligodendroglial and microglial nuclei revealed patient-specific downregulation of myelinating genes in oligodendrocytes and upregulation of an endolysosomal reactive state in microglia. Our findings suggest that selective vulnerability of extratelencephalic neurons is partly connected to their intrinsic molecular properties sensitizing them to genetics and mechanisms of degeneration.',\n",
       "  'Tab3_-_Exc_upregulated_genes-EXC6',\n",
       "  'None here'),\n",
       " 'PMC3338421-pone.0035618.s005.xls-Consensus_signature_genes-Symbol': ('Exploring the transcriptome of ciliated cells using in silico dissection of human tissues',\n",
       "  \"Cilia are cell organelles that play important roles in cell motility, sensory and developmental functions and are involved in a range of human diseases, known as ciliopathies. Here, we search for novel human genes related to cilia using a strategy that exploits the previously reported tendency of cell type-specific genes to be coexpressed in the transcriptome of complex tissues. Gene coexpression networks were constructed using the noise-resistant WGCNA algorithm in 12 publicly available microarray datasets from human tissues rich in motile cilia: airways, fallopian tubes and brain. A cilia-related coexpression module was detected in 10 out of the 12 datasets. A consensus analysis of this module's gene composition recapitulated 297 known and predicted 74 novel cilia-related genes. 82% of the novel candidates were supported by tissue-specificity expression data from GEO and/or proteomic data from the Human Protein Atlas. The novel findings included a set of genes (DCDC2, DYX1C1, KIAA0319) related to a neurological disease dyslexia suggesting their potential involvement in ciliary functions. Furthermore, we searched for differences in gene composition of the ciliary module between the tissues. A multidrug-and-toxin extrusion transporter MATE2 (SLC47A2) was found as a brain-specific central gene in the ciliary module. We confirm the localization of MATE2 in cilia by immunofluorescence staining using MDCK cells as a model. While MATE2 has previously gained attention as a pharmacologically relevant transporter, its potential relation to cilia is suggested for the first time. Taken together, our large-scale analysis of gene coexpression networks identifies novel genes related to human cell cilia.\",\n",
       "  'Consensus_signature_genes-Symbol',\n",
       "  'Joint application of the two requirements produced a consensus signature composed of 371 genes (**Table S5**)  Number of datasets that support each gene as belonging to the ciliary module. The “Consensus signature genes” spread sheet provides data for genes from the consensus signature. The “Rest of the genes” spread sheet provides data for genes that belonged to the ciliary module in at least one dataset but were not included to the consensus signature. For each gene-dataset pair, “1” denotes membership of the gene in the ciliary module, “0” – absence of the gene in the ciliary module. The “Total positive datasets” column describes number of datasets that support the gene as belonging to the ciliary module (sum across the previous columns). The “Tissues” column provides number of tissues (brain, airways, reproductive tracts) in which the gene belongs to the ciliary module in at least one dataset.(XLS)  Click here for additional data file'),\n",
       " 'PMC3338421-pone.0035618.s008.xls-Tissue_specificity-Symbol': ('Exploring the transcriptome of ciliated cells using in silico dissection of human tissues',\n",
       "  \"Cilia are cell organelles that play important roles in cell motility, sensory and developmental functions and are involved in a range of human diseases, known as ciliopathies. Here, we search for novel human genes related to cilia using a strategy that exploits the previously reported tendency of cell type-specific genes to be coexpressed in the transcriptome of complex tissues. Gene coexpression networks were constructed using the noise-resistant WGCNA algorithm in 12 publicly available microarray datasets from human tissues rich in motile cilia: airways, fallopian tubes and brain. A cilia-related coexpression module was detected in 10 out of the 12 datasets. A consensus analysis of this module's gene composition recapitulated 297 known and predicted 74 novel cilia-related genes. 82% of the novel candidates were supported by tissue-specificity expression data from GEO and/or proteomic data from the Human Protein Atlas. The novel findings included a set of genes (DCDC2, DYX1C1, KIAA0319) related to a neurological disease dyslexia suggesting their potential involvement in ciliary functions. Furthermore, we searched for differences in gene composition of the ciliary module between the tissues. A multidrug-and-toxin extrusion transporter MATE2 (SLC47A2) was found as a brain-specific central gene in the ciliary module. We confirm the localization of MATE2 in cilia by immunofluorescence staining using MDCK cells as a model. While MATE2 has previously gained attention as a pharmacologically relevant transporter, its potential relation to cilia is suggested for the first time. Taken together, our large-scale analysis of gene coexpression networks identifies novel genes related to human cell cilia.\",\n",
       "  'Tissue_specificity-Symbol',\n",
       "  \"endometrium, testis, using a permutation-based test (Methods). The tissue-specificity P-values are provided in the **Table S8**  Tissue specificity of the signature genes. For each gene, a p-value is provided estimating significance of the gene's up-regulation in ciliated tissues compared to the rest of the tissues in the human body (see Methods). The “Top 15 tissue samples” column specifies the 15 samples with highest expression levels of the gene. The samples are listed in a decreasing order of the gene's expression level and were selected from the total of 309 samples in GSE7307 (see Methods). Some of the tissues are represented by more than one sample since GSE7307 contained biological replicates.(XLS)  Click here for additional data file\"),\n",
       " 'PMC3338421-pone.0035618.s007.xls-Novelty_categories-Symbol': ('Exploring the transcriptome of ciliated cells using in silico dissection of human tissues',\n",
       "  \"Cilia are cell organelles that play important roles in cell motility, sensory and developmental functions and are involved in a range of human diseases, known as ciliopathies. Here, we search for novel human genes related to cilia using a strategy that exploits the previously reported tendency of cell type-specific genes to be coexpressed in the transcriptome of complex tissues. Gene coexpression networks were constructed using the noise-resistant WGCNA algorithm in 12 publicly available microarray datasets from human tissues rich in motile cilia: airways, fallopian tubes and brain. A cilia-related coexpression module was detected in 10 out of the 12 datasets. A consensus analysis of this module's gene composition recapitulated 297 known and predicted 74 novel cilia-related genes. 82% of the novel candidates were supported by tissue-specificity expression data from GEO and/or proteomic data from the Human Protein Atlas. The novel findings included a set of genes (DCDC2, DYX1C1, KIAA0319) related to a neurological disease dyslexia suggesting their potential involvement in ciliary functions. Furthermore, we searched for differences in gene composition of the ciliary module between the tissues. A multidrug-and-toxin extrusion transporter MATE2 (SLC47A2) was found as a brain-specific central gene in the ciliary module. We confirm the localization of MATE2 in cilia by immunofluorescence staining using MDCK cells as a model. While MATE2 has previously gained attention as a pharmacologically relevant transporter, its potential relation to cilia is suggested for the first time. Taken together, our large-scale analysis of gene coexpression networks identifies novel genes related to human cell cilia.\",\n",
       "  'Novelty_categories-Symbol',\n",
       "  'evidence from the previous studies (“novel candidates”). The gene-to-category mapping is provided in the **Table S7**  Stratification of the signature genes into novelty categories. For each gene, the table specifies number of ciliome studies in which the gene was detected with a strong/medium/weak evidence (according to the CilDB evidence codes). The “MEDLINE” column specifies whether the gene is described as potentially related to cilia in MEDLINE abstracts. The “Category” column provides the resulting assignment of the gene to a novelty category: strong evidence from the previous studies, weak evidence from the previous studies, no evidence from the previous studies.(XLS)  Click here for additional data file'),\n",
       " 'PMC3338421-pone.0035618.s010.xls-Summary-Symbol': ('Exploring the transcriptome of ciliated cells using in silico dissection of human tissues',\n",
       "  \"Cilia are cell organelles that play important roles in cell motility, sensory and developmental functions and are involved in a range of human diseases, known as ciliopathies. Here, we search for novel human genes related to cilia using a strategy that exploits the previously reported tendency of cell type-specific genes to be coexpressed in the transcriptome of complex tissues. Gene coexpression networks were constructed using the noise-resistant WGCNA algorithm in 12 publicly available microarray datasets from human tissues rich in motile cilia: airways, fallopian tubes and brain. A cilia-related coexpression module was detected in 10 out of the 12 datasets. A consensus analysis of this module's gene composition recapitulated 297 known and predicted 74 novel cilia-related genes. 82% of the novel candidates were supported by tissue-specificity expression data from GEO and/or proteomic data from the Human Protein Atlas. The novel findings included a set of genes (DCDC2, DYX1C1, KIAA0319) related to a neurological disease dyslexia suggesting their potential involvement in ciliary functions. Furthermore, we searched for differences in gene composition of the ciliary module between the tissues. A multidrug-and-toxin extrusion transporter MATE2 (SLC47A2) was found as a brain-specific central gene in the ciliary module. We confirm the localization of MATE2 in cilia by immunofluorescence staining using MDCK cells as a model. While MATE2 has previously gained attention as a pharmacologically relevant transporter, its potential relation to cilia is suggested for the first time. Taken together, our large-scale analysis of gene coexpression networks identifies novel genes related to human cell cilia.\",\n",
       "  'Summary-Symbol',\n",
       "  'ciliary candidates. The complete summary (for all the signature proteins) is provided in the **Table S10**  Data summary for the signature genes. For each signature gene, this table summarizes data from the Tables S5, S7, S8 and S9.(XLS)  Click here for additional data file'),\n",
       " 'PMC3338421-pone.0035618.s006.xls-Orthology_information-Gene': ('Exploring the transcriptome of ciliated cells using in silico dissection of human tissues',\n",
       "  \"Cilia are cell organelles that play important roles in cell motility, sensory and developmental functions and are involved in a range of human diseases, known as ciliopathies. Here, we search for novel human genes related to cilia using a strategy that exploits the previously reported tendency of cell type-specific genes to be coexpressed in the transcriptome of complex tissues. Gene coexpression networks were constructed using the noise-resistant WGCNA algorithm in 12 publicly available microarray datasets from human tissues rich in motile cilia: airways, fallopian tubes and brain. A cilia-related coexpression module was detected in 10 out of the 12 datasets. A consensus analysis of this module's gene composition recapitulated 297 known and predicted 74 novel cilia-related genes. 82% of the novel candidates were supported by tissue-specificity expression data from GEO and/or proteomic data from the Human Protein Atlas. The novel findings included a set of genes (DCDC2, DYX1C1, KIAA0319) related to a neurological disease dyslexia suggesting their potential involvement in ciliary functions. Furthermore, we searched for differences in gene composition of the ciliary module between the tissues. A multidrug-and-toxin extrusion transporter MATE2 (SLC47A2) was found as a brain-specific central gene in the ciliary module. We confirm the localization of MATE2 in cilia by immunofluorescence staining using MDCK cells as a model. While MATE2 has previously gained attention as a pharmacologically relevant transporter, its potential relation to cilia is suggested for the first time. Taken together, our large-scale analysis of gene coexpression networks identifies novel genes related to human cell cilia.\",\n",
       "  'Orthology_information-Gene',\n",
       "  'each gene from the signature we determined its orthologs in the model organisms (see **Table S6**)  Orthology information and overlap of the consensus signature with the previous studies. Spreadsheet “Orthology information”: for each gene, orthologs in the model organisms are specified. The orthology information was extracted from the CilDB database. Spreadsheet “Comparison with prev. studies”: overlap of the consensus signature with ciliary genes lists from the previous studies. Columns in this table correspond to those in the Table 3 of the main text (see Results).(XLS)  Click here for additional data file'),\n",
       " 'PMC8502965-Table_3.xlsx-Table_S3_Blue_network_nodes_in-fromNode': ('An Advanced Systems Pharmacology Strategy Reveals AKR1B1, MMP2, PTGER3 as Key Genes in the Competing Endogenous RNA Network of Compound Kushen Injection Treating Gastric Carcinoma by Integrated Bioinformatics and Experimental Verification',\n",
       "  'Gastric carcinoma (GC) is a severe tumor of the digestive tract with high morbidity and mortality and poor prognosis, for which novel treatment options are urgently needed. Compound Kushen injection (CKI), a classical injection of Chinese medicine, has been widely used to treat various tumors in clinical practice for decades. In recent years, a growing number of studies have confirmed that CKI has a beneficial therapeutic effect on GC, However, there are few reports on the potential molecular mechanism of action. Here, using systems pharmacology combined with proteomics analysis as a core concept, we identified the ceRNA network, key targets and signaling pathways regulated by CKI in the treatment of GC. To further explore the role of these key targets in the development of GC, we performed a meta-analysis to compare the expression differences between GC and normal gastric mucosa tissues. Functional enrichment analysis was further used to understand the biological pathways significantly regulated by the key genes. In addition, we determined the significance of the key genes in the prognosis of GC by survival analysis and immune infiltration analysis. Finally, molecular docking simulation was performed to verify the combination of CKI components and key targets. The anti-gastric cancer effect of CKI and its key targets was verified by in vivo and in vitro experiments. The analysis of ceRNA network of CKI on GC revealed that the potential molecular mechanism of CKI can regulate PI3K/AKT and Toll-like receptor signaling pathways by interfering with hub genes such as AKR1B1, MMP2 and PTGERR3 . In conclusion, this study not only partially highlighted the molecular mechanism of CKI in GC therapy but also provided a novel and advanced systems pharmacology strategy to explore the mechanisms of traditional Chinese medicine formulations.',\n",
       "  'Table_S3_Blue_network_nodes_in-fromNode',\n",
       "  'gene linkages and the brown module consists of 232 genes and 11316 gene linkages (**Supplementary Tables 3**)  Blue network nodes in WGCNA  Click here for additional data file'),\n",
       " 'PMC6471696-12859_2019_2687_MOESM2_ESM.xlsx-Table_SM3-Unnamed_2': ('Walking pathways with positive feedback loops reveal DNA methylation biomarkers of colorectal cancer',\n",
       "  'Background: The search for molecular biomarkers of early-onset colorectal cancer (CRC) is an important but still quite challenging and unsolved task. Detection of CpG methylation in human DNA obtained from blood or stool has been proposed as a promising approach to a noninvasive early diagnosis of CRC. Thousands of abnormally methylated CpG positions in CRC genomes are often located in non-coding parts of genes. Novel bioinformatic methods are thus urgently needed for multi-omics data analysis to reveal causative biomarkers with a potential driver role in early stages of cancer. Methods: We have developed a method for finding potential causal relationships between epigenetic changes (DNA methylations) in gene regulatory regions that affect transcription factor binding sites (TFBS) and gene expression changes. This method also considers the topology of the involved signal transduction pathways and searches for positive feedback loops that may cause the carcinogenic aberrations in gene expression. We call this method \"Walking pathways\", since it searches for potential rewiring mechanisms in cancer pathways due to dynamic changes in the DNA methylation status of important gene regulatory regions (\"epigenomic walking\"). Results: In this paper, we analysed an extensive collection of full genome gene-expression data (RNA-seq) and DNA methylation data of genomic CpG islands (using Illumina methylation arrays) generated from a sample of tumor and normal gut epithelial tissues of 300 patients with colorectal cancer (at different stages of the disease) (data generated in the EU-supported SysCol project). Identification of potential epigenetic biomarkers of DNA methylation was performed using the fully automatic multi-omics analysis web service \"My Genome Enhancer\" (MGE) (my-genome-enhancer.com). MGE uses the database on gene regulation TRANSFAC®, the signal transduction pathways database TRANSPATH®, and software that employs AI (artificial intelligence) methods for the analysis of cancer-specific enhancers. Conclusions: The identified biomarkers underwent experimental testing on an independent set of blood samples from patients with colorectal cancer. As a result, using advanced methods of statistics and machine learning, a minimum set of 6 biomarkers was selected, which together achieve the best cancer detection potential. The markers include hypermethylated positions in regulatory regions of the following genes: CALCA, ENO1, MYC, PDX1, TCF7, ZNF43.',\n",
       "  'Table_SM3-Unnamed_2',\n",
       "  'Cancer samples versus Control samples. Limma calculated the LogFC (the logarithm to the base **2**)  The MS Excel file that contains Tables S3-S8. (XLSX 507 kb)'),\n",
       " 'PMC7517826-13045_2020_941_MOESM14_ESM.xlsx-N01-Unnamed_0': ('A single-cell survey of cellular hierarchy in acute myeloid leukemia',\n",
       "  'Background: Acute myeloid leukemia (AML) is a fatal hematopoietic malignancy and has a prognosis that varies with its genetic complexity. However, there has been no appropriate integrative analysis on the hierarchy of different AML subtypes. Methods: Using Microwell-seq, a high-throughput single-cell mRNA sequencing platform, we analyzed the cellular hierarchy of bone marrow samples from 40 patients and 3 healthy donors. We also used single-cell single-molecule real-time (SMRT) sequencing to investigate the clonal heterogeneity of AML cells. Results: From the integrative analysis of 191727 AML cells, we established a single-cell AML landscape and identified an AML progenitor cell cluster with novel AML markers. Patients with ribosomal protein high progenitor cells had a low remission rate. We deduced two types of AML with diverse clinical outcomes. We traced mitochondrial mutations in the AML landscape by combining Microwell-seq with SMRT sequencing. We propose the existence of a phenotypic \"cancer attractor\" that might help to define a common phenotype for AML progenitor cells. Finally, we explored the potential drug targets by making comparisons between the AML landscape and the Human Cell Landscape. Conclusions: We identified a key AML progenitor cell cluster. A high ribosomal protein gene level indicates the poor prognosis. We deduced two types of AML and explored the potential drug targets. Our results suggest the existence of a cancer attractor.',\n",
       "  'N01-Unnamed_0',\n",
       "  'BMMCs. We used Microwell-seq on three healthy donors and established the analysis pipeline (Fig. S**1**)  Additional file 14: Table S1. Marker genes of individual t-SNE map'),\n",
       " 'PMC7517826-13045_2020_941_MOESM14_ESM.xlsx-N01-gene': ('A single-cell survey of cellular hierarchy in acute myeloid leukemia',\n",
       "  'Background: Acute myeloid leukemia (AML) is a fatal hematopoietic malignancy and has a prognosis that varies with its genetic complexity. However, there has been no appropriate integrative analysis on the hierarchy of different AML subtypes. Methods: Using Microwell-seq, a high-throughput single-cell mRNA sequencing platform, we analyzed the cellular hierarchy of bone marrow samples from 40 patients and 3 healthy donors. We also used single-cell single-molecule real-time (SMRT) sequencing to investigate the clonal heterogeneity of AML cells. Results: From the integrative analysis of 191727 AML cells, we established a single-cell AML landscape and identified an AML progenitor cell cluster with novel AML markers. Patients with ribosomal protein high progenitor cells had a low remission rate. We deduced two types of AML with diverse clinical outcomes. We traced mitochondrial mutations in the AML landscape by combining Microwell-seq with SMRT sequencing. We propose the existence of a phenotypic \"cancer attractor\" that might help to define a common phenotype for AML progenitor cells. Finally, we explored the potential drug targets by making comparisons between the AML landscape and the Human Cell Landscape. Conclusions: We identified a key AML progenitor cell cluster. A high ribosomal protein gene level indicates the poor prognosis. We deduced two types of AML and explored the potential drug targets. Our results suggest the existence of a cancer attractor.',\n",
       "  'N01-gene',\n",
       "  'BMMCs. We used Microwell-seq on three healthy donors and established the analysis pipeline (Fig. S**1**)  Additional file 14: Table S1. Marker genes of individual t-SNE map'),\n",
       " 'PMC4617899-12864_2015_2012_MOESM2_ESM.xlsx-CD8a_DC_SIG_minPair_mean_2x-Unnamed_1': ('BubbleGUM: automatic extraction of phenotype molecular signatures and comprehensive visualization of multiple Gene Set Enrichment Analyses',\n",
       "  \"Background: Recent advances in the analysis of high-throughput expression data have led to the development of tools that scaled-up their focus from single-gene to gene set level. For example, the popular Gene Set Enrichment Analysis (GSEA) algorithm can detect moderate but coordinated expression changes of groups of presumably related genes between pairs of experimental conditions. This considerably improves extraction of information from high-throughput gene expression data. However, although many gene sets covering a large panel of biological fields are available in public databases, the ability to generate home-made gene sets relevant to one's biological question is crucial but remains a substantial challenge to most biologists lacking statistic or bioinformatic expertise. This is all the more the case when attempting to define a gene set specific of one condition compared to many other ones. Thus, there is a crucial need for an easy-to-use software for generation of relevant home-made gene sets from complex datasets, their use in GSEA, and the correction of the results when applied to multiple comparisons of many experimental conditions. Result: We developed BubbleGUM (GSEA Unlimited Map), a tool that allows to automatically extract molecular signatures from transcriptomic data and perform exhaustive GSEA with multiple testing correction. One original feature of BubbleGUM notably resides in its capacity to integrate and compare numerous GSEA results into an easy-to-grasp graphical representation. We applied our method to generate transcriptomic fingerprints for murine cell types and to assess their enrichments in human cell types. This analysis allowed us to confirm homologies between mouse and human immunocytes. Conclusions: BubbleGUM is an open-source software that allows to automatically generate molecular signatures out of complex expression datasets and to assess directly their enrichment by GSEA on independent datasets. Enrichments are displayed in a graphical output that helps interpreting the results. This innovative methodology has recently been used to answer important questions in functional genomics, such as the degree of similarities between microarray datasets from different laboratories or with different experimental models or clinical cohorts. BubbleGUM is executable through an intuitive interface so that both bioinformaticians and biologists can use it. It is available at http://www.ciml.univ-mrs.fr/applications/BubbleGUM/index.html .\",\n",
       "  'CD8a_DC_SIG_minPair_mean_2x-Unnamed_1',\n",
       "  'None here'),\n",
       " 'PMC9283889-mmc2.xlsx-B_cell-gene': ('Establishment of a reference single-cell RNA sequencing dataset for human pancreatic adenocarcinoma',\n",
       "  'Single-cell RNA sequencing (scRNAseq) has been used to assess the intra-tumor heterogeneity and microenvironment of pancreatic ductal adenocarcinoma (PDAC). However, previous knowledge is not fully universalized. Here, we built a single cell atlas of PDAC from six datasets containing over 70 samples and >130,000 cells, and demonstrated its application to the reanalysis of the previous bulk transcriptomic cohorts and inferring cell-cell communications. The cell decomposition of bulk transcriptomics using scRNAseq data showed the cellular heterogeneity of PDAC; moreover, high levels of tumor cells and fibroblasts were indicative of poor-prognosis. Refined tumor subtypes signature indicated the tumor cell dynamics in intra-tumor and their specific regulatory network. We further identified functionally distinct tumor clusters that had close interaction with fibroblast subtypes via different signaling pathways dependent on subtypes. Our analysis provided a reference dataset for PDAC and showed its utility in research on the microenvironment of intra-tumor heterogeneity.',\n",
       "  'B_cell-gene',\n",
       "  '2, respectively. The integrated data included 10 cell types with reasonable differentially expressed genes (**Table S1**)  Table S1. Differential expression genes in scRNAseq data of PDAC, related to Figure 1'),\n",
       " 'PMC6273696-molecules-21-00148.nxml--Symbol': ('A SILAC-Based Approach Elicits the Proteomic Responses to Vancomycin-Associated Nephrotoxicity in Human Proximal Tubule Epithelial HK-2 Cells',\n",
       "  'Vancomycin, a widely used antibiotic, often induces nephrotoxicity, however, the molecular targets and underlying mechanisms of this side effect remain unclear. The present study aimed to examine molecular interactome and analyze the signaling pathways related to the vancomycin-induced nephrotoxicity in human proximal tubule epithelial HK-2 cells using the stable isotope labeling by amino acids in cell culture (SILAC) approach. The quantitative proteomic study revealed that there were at least 492 proteins interacting with vancomycin and there were 290 signaling pathways and cellular functions potentially regulated by vancomycin in HK-2 cells. These proteins and pathways played a critical role in the regulation of cell cycle, apoptosis, autophagy, EMT, and ROS generation. These findings suggest that vancomycin-induced proteomic responses in HK-2 cells involvefunctional proteins and pathways that regulate cell cycle, apoptosis, autophagy, and redox homeostasis. This is the first systemic study revealed the networks of signaling pathways and proteomic responses to vancomycin treatment in HK-2 cells, and the data may be used to discriminate the molecular and clinical subtypes and to identify new targets and biomarkers for vancomycin-induced nephrotoxic effect. Further studies are warranted to explore the potential of quantitative proteomic analysis in the identification of new targets and biomarkers for drug-induced renal toxicity.',\n",
       "  'Symbol',\n",
       "  'to IPA pathway analysis. A total of 486 molecular proteins were mapped using IPA (**Table 2**)  486 mapped molecular proteins by IPAAARSACAT1ACAT2ACLYACTA1ACTG1ACTN1ACTN4ACTR2ACTR3AHCYAHNAKAHSA1AIPAKR1B1ALDOAALDOCALYREFANXA1ANXA2ANXA5ANXA6AP1B1AP2B1APEX1ARF1ARHGDIAARPC1BARPC2ATICATP1A1ATP5A1ATP5BATP5HB2MBASP1BSGBTF3BUB3C14orf166C1QBPCACYBPCALRCALUCAND1CANXCAP1CAPGCAPN1CAPN2CAPNS1CAPRIN1CAPZA1CAPZBCARHSP1CASTCAV1CBX3CCT2CCT3CCT4CCT5CCT6ACCT7CCT8CD44CD59CDK1CDV3CFL1CKAP4CLIC1CLIC4CLTCCNN3COPACOPB1COPB2COPECOPG1CORO1CCOTL1CPNE1CRIP2CSCSE1LCSRP1CSTBCTNNA1CTPS1CTSACTSBCTTNCUTACYB5R3CYC1CYR61DBIDBN1DDX1DDX39BDDX3XDDX5DDX6DHX9DLATDNAJA1DNAJA2DNAJC8DNM1LDPY30DPYSL2DYNC1H1DYNC1I2ECHS1EEF1A1EEF1B2EEF1DEEF1GEEF2EHD1EIF2S1EIF2S2EIF3BEIF3CEIF3IEIF3MEIF4A1EIF4G1EIF4HEIF5AEIF6ENAHENO1ENO2EPRSERP29ESDESYT1ETFAEZRFASNFDPSFKBP1AFKBP4FLNAFLNBFLNCFN1FUBP3FUSG3BP1G6PDGANABGAPDHGARSGCN1L1GDI1GDI2GLO1GLUD1GMLGMPSGNAI2GNB2L1GPIGSTP1H2AFYH3F3A/H3F3BHADHHADHAHARSHDGFHDLBPHEXBHINT1HIST1H1CHIST1H2BLHIST2H2ACHLA-AHLA-BHMGB1HN1HNRNPA1HNRNPA2B1HNRNPA3HNRNPH1HNRNPH3HNRNPKHNRNPLHNRNPMHNRNPRHNRNPUHSD17B10HSP90AA1HSP90AB1HSP90B1HSPA4HSPA5HSPA8HSPA9HSPB1HSPD1HYOU1IARSIDI1IFITM3IGF2RILF2ILF3IMMTIMPDH2IPO5IPO7IPO9IQGAP1ITGB1KHDRBS1KHSRPKIF5BKPNA2KPNA3KPNB1KRT1KRT18KRT8KRT9LAMB1LASP1LDHALDHBLGALS1LMNALMNB1LOC102724594/U2AF1LONP1LRPPRCLRRC59MAP1BMAP4MAPK1MAPRE1MARCKSMARSMATR3MCM3MCM4MCM6MCM7MDH1MDH2MSNMVPMYH10MYH9MYL12ANACANAP1L1NAP1L4NASPNCLNME1-NME2NNMTNONONPEPPSNPLOC4NPM1NSFL1CNUMA1OLA1OTUB1P4HBPA2G4PABPC1PAFAH1B2PAICSPARK7PCBP1PCBP2PCM1PCMT1PCNAPDCD6PDE6HPDIA3PDIA4PDIA6PDLIM1PDXKPEA15PEBP1PFKPPFN1PGAM1PGDPGK1PGM1PHB2PHGDHPICALMPKMPLECPLIN3PLOD2PLS3PPA1PPIAPPM1GPPP1CAPPP2CAPPP2R1APRDX1PRDX2PRDX4PRDX5PRKCSHPRKDCPRMT5PRPF19PSMA1PSMA2PSMA4PSMA5PSMA7PSMB1PSMC1PSMC4PSMD2PSME1PSME2PTBP1PTGES3PTMAPTRFPUF60PYGBPYGLQARSRAB11ARAB1BRAB7ARAD23BRANRANBP1RARSRBM3RCN1RHOARNH1RPL10RPL10ARPL12RPL18RPL21RPL22RPL23RPL27RPL3RPL4RPL5RPL6RPL7RPL7ARPL8RPLP0RPLP1RPLP2RPN1RPN2RPS10RPS11RPS12RPS15RPS2RPS24RPS27ARPS3RPS3ARPS4XRPS5RPS7RPS8RPSARRBP1RTN4S100A6SAE1SDHASEC13SEPT2SEPT7SEPT9SERBP1SERPINH1SETSF3A1SF3B1SFNSFPQSH3BGRL3SHMT2SLC25A3SLC25A5SLC25A6SLC38A2SND1SNRPD1SNRPD3SNX3SOD1SPTAN1SPTBN1SQSTM1SSRP1STAT1STIP1STMN1STOML2SUPT16HSURF4SYNCRIPTAGLNTAGLN2TALDO1TARSTBCATCP1TFRCTKTTLN1TMOD3TMPOTNCTNPO1TPI1TPM2TPM3TPM4TPP1TRIM28TRMT1TUBA1BTUBBTUBB4BTUBB6TUBB8TUFMTWF1TWF2TXLNATXNDC5U2AF2UAP1UBA1UBE2L3UBE2NUBXN4UCHL1UQCRC1VAPAVARSVAT1VCLVCPVDAC1VDAC2VIMVPS35WARSWDR1XPO1XRCC5XRCC6YBX1YBX3YWHABYWHAEYWHAGYWHAHYWHAQYWHAZZYX'),\n",
       " 'PMC9281170-12859_2022_4817_MOESM1_ESM.xlsx-Comet_B-gene_1': ('Detection of cell markers from single cell RNA-seq with sc2marker',\n",
       "  'Background: Single-cell RNA sequencing (scRNA-seq) allows the detection of rare cell types in complex tissues. The detection of markers for rare cell types is useful for further biological analysis of, for example, flow cytometry and imaging data sets for either physical isolation or spatial characterization of these cells. However, only a few computational approaches consider the problem of selecting specific marker genes from scRNA-seq data. Results: Here, we propose sc2marker, which is based on the maximum margin index and a database of proteins with antibodies, to select markers for flow cytometry or imaging. We evaluated the performances of sc2marker and competing methods in ranking known markers in scRNA-seq data of immune and stromal cells. The results showed that sc2marker performed better than the competing methods in accuracy, while having a competitive running time.',\n",
       "  'Comet_B-gene_1',\n",
       "  'performing method should have low rank scores; i.e., true marker genes are ranked as **1**  Additional file 1: Results of markers detection for MCA-Spleen dataset with sc2marker and other competing mehtods'),\n",
       " 'PMC9849137-41587_2022_1397_MOESM9_ESM.xlsx-common_SE_markers-x': ('Uncovering the mode of action of engineered T cells in patient cancer organoids',\n",
       "  \"Extending the success of cellular immunotherapies against blood cancers to the realm of solid tumors will require improved in vitro models that reveal therapeutic modes of action at the molecular level. Here we describe a system, called BEHAV3D, developed to study the dynamic interactions of immune cells and patient cancer organoids by means of imaging and transcriptomics. We apply BEHAV3D to live-track >150,000 engineered T cells cultured with patient-derived, solid-tumor organoids, identifying a 'super engager' behavioral cluster comprising T cells with potent serial killing capacity. Among other T cell concepts we also study cancer metabolome-sensing engineered T cells (TEGs) and detect behavior-specific gene signatures that include a group of 27 genes with no previously described T cell function that are expressed by super engager killer TEGs. We further show that type I interferon can prime resistant organoids for TEG-mediated killing. BEHAV3D is a promising tool for the characterization of behavioral-phenotypic heterogeneity of cellular immunotherapies and may support the optimization of personalized solid-tumor-targeting cell therapies.\",\n",
       "  'common_SE_markers-x',\n",
       "  'between 10T- and 13T-cocultured TEGs (Fig. 6b, Extended Data Fig. 8a,b and Supplementary Table **7**)  DEG analyses and common genes corresponding to Fig. 6 analyses'),\n",
       " 'PMC9973344-mbio.03231-22-st003.xlsx-M1wt-M1T108E_24h-Unnamed_16': ('Phosphorylation of Influenza A Virus Matrix Protein 1 at Threonine 108 Controls Its Multimerization State and Functional Association with the STRIPAK Complex',\n",
       "  'The influenza A virus (IAV)-encoded matrix protein 1 (M1) acts as a master regulator of virus replication and fulfills multiple structural and regulatory functions in different cell compartments. Therefore, the spatiotemporal regulation of M1 is achieved by different mechanisms, including its structural and pH-dependent flexibility, differential association with cellular factors, and posttranslational modifications. Here, we investigated the function of M1 phosphorylation at the evolutionarily conserved threonine 108 (T108) and found that its mutation to a nonphosphorylatable alanine prohibited virus replication. Absent T108, phosphorylation led to strongly increased self-association of M1 at the cell membrane and consequently prohibited its ability to enter the nucleus and to contribute to viral ribonucleoprotein nuclear export. M1 T108 phosphorylation also controls the binding affinity to the cellular STRIPAK (striatin-interacting phosphatases and kinases) complex, which contains different kinases and the phosphatase PP2A to shape phosphorylation-dependent signaling networks. IAV infection led to the redistribution of the STRIPAK scaffolding subunits STRN and STRN3 from the cell membrane to cytosolic and perinuclear clusters, where it colocalized with M1. Inactivation of the STRIPAK complex resulted in compromised M1 polymerization and IAV replication. IMPORTANCE Influenza viruses pose a major threat to human health and cause annual epidemics and occasional pandemics. Many virus-encoded proteins exert various functions in different subcellular compartments, as exemplified by the M1 protein, but the molecular mechanisms endowing the multiplicity of functions remain incompletely understood. Here, we report that phosphorylation of M1 at T108 is essential for virus replication and controls its propensity for self-association and nuclear localization. This phosphorylation also controls binding affinity of the M1 protein to the STRIPAK complex, which contributes to M1 polymerization and virus replication.',\n",
       "  '22-st003.xlsx-M1wt-M1T108E_24h-Unnamed_16',\n",
       "  'None here'),\n",
       " 'PMC9973344-mbio.03231-22-st003.xlsx-M1wt-M1T108A_12h-Unnamed_15': ('Phosphorylation of Influenza A Virus Matrix Protein 1 at Threonine 108 Controls Its Multimerization State and Functional Association with the STRIPAK Complex',\n",
       "  'The influenza A virus (IAV)-encoded matrix protein 1 (M1) acts as a master regulator of virus replication and fulfills multiple structural and regulatory functions in different cell compartments. Therefore, the spatiotemporal regulation of M1 is achieved by different mechanisms, including its structural and pH-dependent flexibility, differential association with cellular factors, and posttranslational modifications. Here, we investigated the function of M1 phosphorylation at the evolutionarily conserved threonine 108 (T108) and found that its mutation to a nonphosphorylatable alanine prohibited virus replication. Absent T108, phosphorylation led to strongly increased self-association of M1 at the cell membrane and consequently prohibited its ability to enter the nucleus and to contribute to viral ribonucleoprotein nuclear export. M1 T108 phosphorylation also controls the binding affinity to the cellular STRIPAK (striatin-interacting phosphatases and kinases) complex, which contains different kinases and the phosphatase PP2A to shape phosphorylation-dependent signaling networks. IAV infection led to the redistribution of the STRIPAK scaffolding subunits STRN and STRN3 from the cell membrane to cytosolic and perinuclear clusters, where it colocalized with M1. Inactivation of the STRIPAK complex resulted in compromised M1 polymerization and IAV replication. IMPORTANCE Influenza viruses pose a major threat to human health and cause annual epidemics and occasional pandemics. Many virus-encoded proteins exert various functions in different subcellular compartments, as exemplified by the M1 protein, but the molecular mechanisms endowing the multiplicity of functions remain incompletely understood. Here, we report that phosphorylation of M1 at T108 is essential for virus replication and controls its propensity for self-association and nuclear localization. This phosphorylation also controls binding affinity of the M1 protein to the STRIPAK complex, which contributes to M1 polymerization and virus replication.',\n",
       "  '22-st003.xlsx-M1wt-M1T108A_12h-Unnamed_15',\n",
       "  'None here'),\n",
       " 'PMC9973344-mbio.03231-22-st003.xlsx-M1wt-M1T108A_24h-Unnamed_16': ('Phosphorylation of Influenza A Virus Matrix Protein 1 at Threonine 108 Controls Its Multimerization State and Functional Association with the STRIPAK Complex',\n",
       "  'The influenza A virus (IAV)-encoded matrix protein 1 (M1) acts as a master regulator of virus replication and fulfills multiple structural and regulatory functions in different cell compartments. Therefore, the spatiotemporal regulation of M1 is achieved by different mechanisms, including its structural and pH-dependent flexibility, differential association with cellular factors, and posttranslational modifications. Here, we investigated the function of M1 phosphorylation at the evolutionarily conserved threonine 108 (T108) and found that its mutation to a nonphosphorylatable alanine prohibited virus replication. Absent T108, phosphorylation led to strongly increased self-association of M1 at the cell membrane and consequently prohibited its ability to enter the nucleus and to contribute to viral ribonucleoprotein nuclear export. M1 T108 phosphorylation also controls the binding affinity to the cellular STRIPAK (striatin-interacting phosphatases and kinases) complex, which contains different kinases and the phosphatase PP2A to shape phosphorylation-dependent signaling networks. IAV infection led to the redistribution of the STRIPAK scaffolding subunits STRN and STRN3 from the cell membrane to cytosolic and perinuclear clusters, where it colocalized with M1. Inactivation of the STRIPAK complex resulted in compromised M1 polymerization and IAV replication. IMPORTANCE Influenza viruses pose a major threat to human health and cause annual epidemics and occasional pandemics. Many virus-encoded proteins exert various functions in different subcellular compartments, as exemplified by the M1 protein, but the molecular mechanisms endowing the multiplicity of functions remain incompletely understood. Here, we report that phosphorylation of M1 at T108 is essential for virus replication and controls its propensity for self-association and nuclear localization. This phosphorylation also controls binding affinity of the M1 protein to the STRIPAK complex, which contributes to M1 polymerization and virus replication.',\n",
       "  '22-st003.xlsx-M1wt-M1T108A_24h-Unnamed_16',\n",
       "  'None here'),\n",
       " 'PMC9973344-mbio.03231-22-st003.xlsx-M1wt-IgGwt_24h-Unnamed_16': ('Phosphorylation of Influenza A Virus Matrix Protein 1 at Threonine 108 Controls Its Multimerization State and Functional Association with the STRIPAK Complex',\n",
       "  'The influenza A virus (IAV)-encoded matrix protein 1 (M1) acts as a master regulator of virus replication and fulfills multiple structural and regulatory functions in different cell compartments. Therefore, the spatiotemporal regulation of M1 is achieved by different mechanisms, including its structural and pH-dependent flexibility, differential association with cellular factors, and posttranslational modifications. Here, we investigated the function of M1 phosphorylation at the evolutionarily conserved threonine 108 (T108) and found that its mutation to a nonphosphorylatable alanine prohibited virus replication. Absent T108, phosphorylation led to strongly increased self-association of M1 at the cell membrane and consequently prohibited its ability to enter the nucleus and to contribute to viral ribonucleoprotein nuclear export. M1 T108 phosphorylation also controls the binding affinity to the cellular STRIPAK (striatin-interacting phosphatases and kinases) complex, which contains different kinases and the phosphatase PP2A to shape phosphorylation-dependent signaling networks. IAV infection led to the redistribution of the STRIPAK scaffolding subunits STRN and STRN3 from the cell membrane to cytosolic and perinuclear clusters, where it colocalized with M1. Inactivation of the STRIPAK complex resulted in compromised M1 polymerization and IAV replication. IMPORTANCE Influenza viruses pose a major threat to human health and cause annual epidemics and occasional pandemics. Many virus-encoded proteins exert various functions in different subcellular compartments, as exemplified by the M1 protein, but the molecular mechanisms endowing the multiplicity of functions remain incompletely understood. Here, we report that phosphorylation of M1 at T108 is essential for virus replication and controls its propensity for self-association and nuclear localization. This phosphorylation also controls binding affinity of the M1 protein to the STRIPAK complex, which contributes to M1 polymerization and virus replication.',\n",
       "  '22-st003.xlsx-M1wt-IgGwt_24h-Unnamed_16',\n",
       "  'None here'),\n",
       " 'PMC9973344-mbio.03231-22-st003.xlsx-filtered_data_715_proteins-Unnamed_80': ('Phosphorylation of Influenza A Virus Matrix Protein 1 at Threonine 108 Controls Its Multimerization State and Functional Association with the STRIPAK Complex',\n",
       "  'The influenza A virus (IAV)-encoded matrix protein 1 (M1) acts as a master regulator of virus replication and fulfills multiple structural and regulatory functions in different cell compartments. Therefore, the spatiotemporal regulation of M1 is achieved by different mechanisms, including its structural and pH-dependent flexibility, differential association with cellular factors, and posttranslational modifications. Here, we investigated the function of M1 phosphorylation at the evolutionarily conserved threonine 108 (T108) and found that its mutation to a nonphosphorylatable alanine prohibited virus replication. Absent T108, phosphorylation led to strongly increased self-association of M1 at the cell membrane and consequently prohibited its ability to enter the nucleus and to contribute to viral ribonucleoprotein nuclear export. M1 T108 phosphorylation also controls the binding affinity to the cellular STRIPAK (striatin-interacting phosphatases and kinases) complex, which contains different kinases and the phosphatase PP2A to shape phosphorylation-dependent signaling networks. IAV infection led to the redistribution of the STRIPAK scaffolding subunits STRN and STRN3 from the cell membrane to cytosolic and perinuclear clusters, where it colocalized with M1. Inactivation of the STRIPAK complex resulted in compromised M1 polymerization and IAV replication. IMPORTANCE Influenza viruses pose a major threat to human health and cause annual epidemics and occasional pandemics. Many virus-encoded proteins exert various functions in different subcellular compartments, as exemplified by the M1 protein, but the molecular mechanisms endowing the multiplicity of functions remain incompletely understood. Here, we report that phosphorylation of M1 at T108 is essential for virus replication and controls its propensity for self-association and nuclear localization. This phosphorylation also controls binding affinity of the M1 protein to the STRIPAK complex, which contributes to M1 polymerization and virus replication.',\n",
       "  '22-st003.xlsx-filtered_data_715_proteins-Unnamed_80',\n",
       "  'None here'),\n",
       " 'PMC9973344-mbio.03231-22-st003.xlsx-M1wt-M1T108E_12h-Unnamed_15': ('Phosphorylation of Influenza A Virus Matrix Protein 1 at Threonine 108 Controls Its Multimerization State and Functional Association with the STRIPAK Complex',\n",
       "  'The influenza A virus (IAV)-encoded matrix protein 1 (M1) acts as a master regulator of virus replication and fulfills multiple structural and regulatory functions in different cell compartments. Therefore, the spatiotemporal regulation of M1 is achieved by different mechanisms, including its structural and pH-dependent flexibility, differential association with cellular factors, and posttranslational modifications. Here, we investigated the function of M1 phosphorylation at the evolutionarily conserved threonine 108 (T108) and found that its mutation to a nonphosphorylatable alanine prohibited virus replication. Absent T108, phosphorylation led to strongly increased self-association of M1 at the cell membrane and consequently prohibited its ability to enter the nucleus and to contribute to viral ribonucleoprotein nuclear export. M1 T108 phosphorylation also controls the binding affinity to the cellular STRIPAK (striatin-interacting phosphatases and kinases) complex, which contains different kinases and the phosphatase PP2A to shape phosphorylation-dependent signaling networks. IAV infection led to the redistribution of the STRIPAK scaffolding subunits STRN and STRN3 from the cell membrane to cytosolic and perinuclear clusters, where it colocalized with M1. Inactivation of the STRIPAK complex resulted in compromised M1 polymerization and IAV replication. IMPORTANCE Influenza viruses pose a major threat to human health and cause annual epidemics and occasional pandemics. Many virus-encoded proteins exert various functions in different subcellular compartments, as exemplified by the M1 protein, but the molecular mechanisms endowing the multiplicity of functions remain incompletely understood. Here, we report that phosphorylation of M1 at T108 is essential for virus replication and controls its propensity for self-association and nuclear localization. This phosphorylation also controls binding affinity of the M1 protein to the STRIPAK complex, which contributes to M1 polymerization and virus replication.',\n",
       "  '22-st003.xlsx-M1wt-M1T108E_12h-Unnamed_15',\n",
       "  'None here'),\n",
       " 'PMC9973344-mbio.03231-22-st003.xlsx-M1wt-IgGwt_12h-Unnamed_15': ('Phosphorylation of Influenza A Virus Matrix Protein 1 at Threonine 108 Controls Its Multimerization State and Functional Association with the STRIPAK Complex',\n",
       "  'The influenza A virus (IAV)-encoded matrix protein 1 (M1) acts as a master regulator of virus replication and fulfills multiple structural and regulatory functions in different cell compartments. Therefore, the spatiotemporal regulation of M1 is achieved by different mechanisms, including its structural and pH-dependent flexibility, differential association with cellular factors, and posttranslational modifications. Here, we investigated the function of M1 phosphorylation at the evolutionarily conserved threonine 108 (T108) and found that its mutation to a nonphosphorylatable alanine prohibited virus replication. Absent T108, phosphorylation led to strongly increased self-association of M1 at the cell membrane and consequently prohibited its ability to enter the nucleus and to contribute to viral ribonucleoprotein nuclear export. M1 T108 phosphorylation also controls the binding affinity to the cellular STRIPAK (striatin-interacting phosphatases and kinases) complex, which contains different kinases and the phosphatase PP2A to shape phosphorylation-dependent signaling networks. IAV infection led to the redistribution of the STRIPAK scaffolding subunits STRN and STRN3 from the cell membrane to cytosolic and perinuclear clusters, where it colocalized with M1. Inactivation of the STRIPAK complex resulted in compromised M1 polymerization and IAV replication. IMPORTANCE Influenza viruses pose a major threat to human health and cause annual epidemics and occasional pandemics. Many virus-encoded proteins exert various functions in different subcellular compartments, as exemplified by the M1 protein, but the molecular mechanisms endowing the multiplicity of functions remain incompletely understood. Here, we report that phosphorylation of M1 at T108 is essential for virus replication and controls its propensity for self-association and nuclear localization. This phosphorylation also controls binding affinity of the M1 protein to the STRIPAK complex, which contributes to M1 polymerization and virus replication.',\n",
       "  '22-st003.xlsx-M1wt-IgGwt_12h-Unnamed_15',\n",
       "  'None here'),\n",
       " 'PMC10866903-41598_2024_53999_MOESM2_ESM.xlsx-HCI-002-gene': ('An inflamed tumor cell subpopulation promotes chemotherapy resistance in triple negative breast cancer',\n",
       "  'Individual cancers are composed of heterogeneous tumor cells with distinct phenotypes and genotypes, with triple negative breast cancers (TNBC) demonstrating the most heterogeneity among breast cancer types. Variability in transcriptional phenotypes could meaningfully limit the efficacy of monotherapies and fuel drug resistance, although to an unknown extent. To determine if transcriptional differences between tumor cells lead to differential drug responses we performed single cell RNA-seq on cell line and PDX models of breast cancer revealing cell subpopulations in states associated with resistance to standard-of-care therapies. We found that TNBC models contained a subpopulation in an inflamed cellular state, often also present in human breast cancer samples. Inflamed cells display evidence of heightened cGAS/STING signaling which we demonstrate is sufficient to cause tumor cell resistance to chemotherapy. Accordingly, inflamed cells were enriched in human tumors taken after neoadjuvant chemotherapy and associated with early recurrence, highlighting the potential for diverse tumor cell states to promote drug resistance.',\n",
       "  'HCI-002-gene',\n",
       "  'None here'),\n",
       " 'PMC2680378-blood-2008-06-162958_TableS5.xls-specific-CD66b-External_Gene_ID': ('A HaemAtlas: characterizing gene expression in differentiated human blood cells',\n",
       "  'Hematopoiesis is a carefully controlled process that is regulated by complex networks of transcription factors that are, in part, controlled by signals resulting from ligand binding to cell-surface receptors. To further understand hematopoiesis, we have compared gene expression profiles of human erythroblasts, megakaryocytes, B cells, cytotoxic and helper T cells, natural killer cells, granulocytes, and monocytes using whole genome microarrays. A bioinformatics analysis of these data was performed focusing on transcription factors, immunoglobulin superfamily members, and lineage-specific transcripts. We observed that the numbers of lineage-specific genes varies by 2 orders of magnitude, ranging from 5 for cytotoxic T cells to 878 for granulocytes. In addition, we have identified novel coexpression patterns for key transcription factors involved in hematopoiesis (eg, GATA3-GFI1 and GATA2-KLF1). This study represents the most comprehensive analysis of gene expression in hematopoietic cells to date and has identified genes that play key roles in lineage commitment and cell function. The data, which are freely accessible, will be invaluable for future studies on hematopoiesis and the role of specific genes and will also aid the understanding of the recent genome-wide association studies.',\n",
       "  'specific-CD66b-External_Gene_ID',\n",
       "  '[Supplemental Appendix, Tables, and Figure]'),\n",
       " 'PMC4952232-13075_2016_1052_MOESM2_ESM.xlsx-Additional_file_1-Unnamed_5': ('Identification of baseline gene expression signatures predicting therapeutic responses to three biologic agents in rheumatoid arthritis: a retrospective observational study',\n",
       "  'Background: According to EULAR recommendations, biologic DMARDs (bDMARDs) such as tumor necrosis factor inhibitor, tocilizumab (TCZ), and abatacept (ABT) are in parallel when prescribing to rheumatoid arthritis (RA) patients who have shown insufficient response to conventional synthetic DMARDs. However, most prediction studies of therapeutic response to bDMARDs using gene expression profiles were focused on a single bDMARD, and consideration of the results from the perspective of RA pathophysiology was insufficient. The aim of this study was to identify the specific molecular biological features predicting the therapeutic outcomes of three bDMARDs (infliximab [IFX], TCZ, and ABT) by studying blood gene expression signatures of patients before biologic treatment in a unified test platform. Methods: RA patients who responded inadequately to methotrexate and were later commenced on any one of IFX (n = 140), TCZ (n = 38), or ABT (n = 31) as their first biologic between May 2007 and November 2011 were enrolled. Whole-blood gene expression data were obtained before biologic administration. Patients were categorized into remission (REM) and nonremission (NON-REM) groups according to CDAI at 6 months of biologic therapy. We employed Gene Set Enrichment Analysis (GSEA) to identify functional gene sets differentially expressed between these two groups for each biologic. Then, we compiled \"signature scores\" for these gene sets, and the prediction performances were assessed. Results: GSEA showed that inflammasome genes were significantly upregulated with IFX in the NON-REM group compared with the REM group. With TCZ in the REM group, B-cell-specifically expressed genes were upregulated. RNA elongation, apoptosis-related, and NK-cell-specifically expressed genes were upregulated with ABT in the NON-REM group. Logistic regression analyses showed that \"signature scores\" of inflammasomes, B-cell-specifically expressed, and NK-cell-specifically expressed genes were significant, independently predictive factors for treatment outcome with IFX, TCZ, and ABT, respectively. The AUCs of ROC curves of these signature scores were 0.637, 0.796, and 0.768 for IFX, TCZ, and ABT, respectively. Conclusions: We have identified original gene expression predictive signatures uniquely underlying the therapeutic effects of IFX, TCZ, and ABT. This is, to our knowledge, the first attempt to predict therapeutic effects of three drugs concomitantly using a unified gene expression test platform.',\n",
       "  'Additional_file_1-Unnamed_5',\n",
       "  'None here'),\n",
       " 'PMC8130971-pbio.3001229.s007.xlsx-808_DEGs_in_human_atrium-gene_name': ('Integrated transcriptomics and epigenomics reveal chamber-specific and species-specific characteristics of human and mouse hearts',\n",
       "  'DNA methylation, chromatin accessibility, and gene expression represent different levels information in biological process, but a comprehensive multiomics analysis of the mammalian heart is lacking. Here, we applied nucleosome occupancy and methylome sequencing, which detected DNA methylation and chromatin accessibility simultaneously, as well as RNA-seq, for multiomics analysis of the 4 chambers of adult and fetal human hearts, and adult mouse hearts. Our results showed conserved region-specific patterns in the mammalian heart at transcriptome and DNA methylation level. Adult and fetal human hearts showed distinct features in DNA methylome, chromatin accessibility, and transcriptome. Novel long noncoding RNAs were identified in the human heart, and the gene expression profiles of major cardiovascular diseases associated genes were displayed. Furthermore, cross-species comparisons revealed human-specific and mouse-specific differentially expressed genes between the atria and ventricles. We also reported the relationship among multiomics and found there was a bell-shaped relationship between gene-body methylation and expression in the human heart. In general, our study provided comprehensive spatiotemporal and evolutionary insights into the regulation of gene expression in the heart.',\n",
       "  '808_DEGs_in_human_atrium-gene_name',\n",
       "  'substantial number of protein-coding genes were differentially expressed in the human atria and ventricles (**S2 Table**)  DEGs (protein-coding genes and lncRNAs) between the human atria and ventricles.(XLSX)  Click here for additional data file'),\n",
       " 'PMC7801680-41598_2020_79740_MOESM11_ESM.xlsx-network_hubGenes-Hub_gene_in_AD1_module': (\"Combinatorial analyses reveal cellular composition changes have different impacts on transcriptomic changes of cell type specific genes in Alzheimer's Disease\",\n",
       "  \"Alzheimer's disease (AD) brains are characterized by progressive neuron loss and gliosis. Previous studies of gene expression using bulk tissue samples often fail to consider changes in cell-type composition when comparing AD versus control, which can lead to differences in expression levels that are not due to transcriptional regulation. We mined five large transcriptomic AD datasets for conserved gene co-expression module, then analyzed differential expression and differential co-expression within the modules between AD samples and controls. We performed cell-type deconvolution analysis to determine whether the observed differential expression was due to changes in cell-type proportions in the samples or to transcriptional regulation. Our findings were validated using four additional datasets. We discovered that the increased expression of microglia modules in the AD samples can be explained by increased microglia proportions in the AD samples. In contrast, decreased expression and perturbed co-expression within neuron modules in the AD samples was likely due in part to altered regulation of neuronal pathways. Several transcription factors that are differentially expressed in AD might account for such altered gene regulation. Similarly, changes in gene expression and co-expression within astrocyte modules could be attributed to combined effects of astrogliosis and astrocyte gene activation. Gene expression in the astrocyte modules was also strongly correlated with clinicopathological biomarkers. Through this work, we demonstrated that combinatorial analysis can delineate the origins of transcriptomic changes in bulk tissue data and shed light on key genes and pathways involved in AD.\",\n",
       "  'network_hubGenes-Hub_gene_in_AD1_module',\n",
       "  'AD samples. We identified 136 genes for each case (Supplementary Fig. 6; Supplementary Table **14**)  Supplementary Table 14'),\n",
       " 'PMC8591264-Table_1.XLS-GSE6764_DOWN-Gene_symbol': ('Downregulated Ferroptosis-Related Gene STEAP3 as a Novel Diagnostic and Prognostic Target for Hepatocellular Carcinoma and Its Roles in Immune Regulation',\n",
       "  'Ferroptosis, a distinct type of regulated cell death, has been reported to be involved in the tumorigenesis of liver hepatocellular carcinoma (LIHC). However, the precise functions and potential mechanisms of ferroptosis in LIHC were still poorly understood. Herein, we investigated the biological roles of ferroptosis-related gene STEAP3 in LIHC. STEAP3 was previously proved to serve a key regulator in ferroptosis via mediating the iron metabolism. Comprehensive bioinformatics from several databases revealed that STEAP3 was significantly downregulated in LIHC tissues and exhibited the favorable prognostic significance in LIHC patients. The downregulated STEAP3 was further confirmed in two LIHC cells Huh7 and MHCC97H using real-time PCR and western blot. And STEAP3 overexpression significantly inhibited the cell proliferation in Huh7 and MHCC97H cells. In addition, clinical data identified the relationship between STEAP3 expression and several clinicopathological parameters of LIHC patients, including histologic grade, alpha fetal protein (AFP) concentration, etc. Receiver operation characteristic (ROC) curve revealed STEAP3 as a potential diagnostic biomarker for LIHC patients. Moreover, the co-expression network of STEAP3 was explored to gain a better insight into its underlying signaling pathways. Finally, aberrant STEAP3 might participate in varieties of immune-associated signatures in LIHC pathogenesis, including immunostimulators, immunoinhibitors, chemokines, and chemokine receptors. Taken together, these findings could enhance our knowledge regarding the inhibitory roles and underlying biological significance of STEAP3 in LIHC tumorigenesis.',\n",
       "  'GSE6764_DOWN-Gene_symbol',\n",
       "  'genes in GSE14520, and 343 up-regulated genes and 258 down-regulated genes in GSE14323, respectively (**Supplementary Table 1**)  Click here for additional data file'),\n",
       " 'PMC10866903-41598_2024_53999_MOESM2_ESM.xlsx-HCI-002-Unnamed_0': ('An inflamed tumor cell subpopulation promotes chemotherapy resistance in triple negative breast cancer',\n",
       "  'Individual cancers are composed of heterogeneous tumor cells with distinct phenotypes and genotypes, with triple negative breast cancers (TNBC) demonstrating the most heterogeneity among breast cancer types. Variability in transcriptional phenotypes could meaningfully limit the efficacy of monotherapies and fuel drug resistance, although to an unknown extent. To determine if transcriptional differences between tumor cells lead to differential drug responses we performed single cell RNA-seq on cell line and PDX models of breast cancer revealing cell subpopulations in states associated with resistance to standard-of-care therapies. We found that TNBC models contained a subpopulation in an inflamed cellular state, often also present in human breast cancer samples. Inflamed cells display evidence of heightened cGAS/STING signaling which we demonstrate is sufficient to cause tumor cell resistance to chemotherapy. Accordingly, inflamed cells were enriched in human tumors taken after neoadjuvant chemotherapy and associated with early recurrence, highlighting the potential for diverse tumor cell states to promote drug resistance.',\n",
       "  'HCI-002-Unnamed_0',\n",
       "  'None here'),\n",
       " 'PMC9973344-mbio.03231-22-st003.xlsx-all_data_726_proteins-Unnamed_89': ('Phosphorylation of Influenza A Virus Matrix Protein 1 at Threonine 108 Controls Its Multimerization State and Functional Association with the STRIPAK Complex',\n",
       "  'The influenza A virus (IAV)-encoded matrix protein 1 (M1) acts as a master regulator of virus replication and fulfills multiple structural and regulatory functions in different cell compartments. Therefore, the spatiotemporal regulation of M1 is achieved by different mechanisms, including its structural and pH-dependent flexibility, differential association with cellular factors, and posttranslational modifications. Here, we investigated the function of M1 phosphorylation at the evolutionarily conserved threonine 108 (T108) and found that its mutation to a nonphosphorylatable alanine prohibited virus replication. Absent T108, phosphorylation led to strongly increased self-association of M1 at the cell membrane and consequently prohibited its ability to enter the nucleus and to contribute to viral ribonucleoprotein nuclear export. M1 T108 phosphorylation also controls the binding affinity to the cellular STRIPAK (striatin-interacting phosphatases and kinases) complex, which contains different kinases and the phosphatase PP2A to shape phosphorylation-dependent signaling networks. IAV infection led to the redistribution of the STRIPAK scaffolding subunits STRN and STRN3 from the cell membrane to cytosolic and perinuclear clusters, where it colocalized with M1. Inactivation of the STRIPAK complex resulted in compromised M1 polymerization and IAV replication. IMPORTANCE Influenza viruses pose a major threat to human health and cause annual epidemics and occasional pandemics. Many virus-encoded proteins exert various functions in different subcellular compartments, as exemplified by the M1 protein, but the molecular mechanisms endowing the multiplicity of functions remain incompletely understood. Here, we report that phosphorylation of M1 at T108 is essential for virus replication and controls its propensity for self-association and nuclear localization. This phosphorylation also controls binding affinity of the M1 protein to the STRIPAK complex, which contributes to M1 polymerization and virus replication.',\n",
       "  '22-st003.xlsx-all_data_726_proteins-Unnamed_89',\n",
       "  'None here'),\n",
       " 'PMC9894840-41467_2023_36269_MOESM8_ESM.xlsx-cell_2600-GeneSymbol': ('Sample multiplexing-based targeted pathway proteomics with real-time analytics reveals the impact of genetic variation on protein expression',\n",
       "  'Targeted proteomics enables hypothesis-driven research by measuring the cellular expression of protein cohorts related by function, disease, or class after perturbation. Here, we present a pathway-centric approach and an assay builder resource for targeting entire pathways of up to 200 proteins selected from >10,000 expressed proteins to directly measure their abundances, exploiting sample multiplexing to increase throughput by 16-fold. The strategy, termed GoDig, requires only a single-shot LC-MS analysis, ~1 µg combined peptide material, a list of up to 200 proteins, and real-time analytics to trigger simultaneous quantification of up to 16 samples for hundreds of analytes. We apply GoDig to quantify the impact of genetic variation on protein expression in mice fed a high-fat diet. We create several GoDig assays to quantify the expression of multiple protein families (kinases, lipid metabolism- and lipid droplet-associated proteins) across 480 fully-genotyped Diversity Outbred mice, revealing protein quantitative trait loci and establishing potential linkages between specific proteins and lipid homeostasis.',\n",
       "  'cell_2600-GeneSymbol',\n",
       "  'generated tryptic peptides from biological quadruplicates of four human cell lines—RPE1, U2OS, HEK293T and HCT11**6**  Dataset 6'),\n",
       " 'PMC9894840-41467_2023_36269_MOESM3_ESM.xlsx-2600_peptides-GeneSymbol': ('Sample multiplexing-based targeted pathway proteomics with real-time analytics reveals the impact of genetic variation on protein expression',\n",
       "  'Targeted proteomics enables hypothesis-driven research by measuring the cellular expression of protein cohorts related by function, disease, or class after perturbation. Here, we present a pathway-centric approach and an assay builder resource for targeting entire pathways of up to 200 proteins selected from >10,000 expressed proteins to directly measure their abundances, exploiting sample multiplexing to increase throughput by 16-fold. The strategy, termed GoDig, requires only a single-shot LC-MS analysis, ~1 µg combined peptide material, a list of up to 200 proteins, and real-time analytics to trigger simultaneous quantification of up to 16 samples for hundreds of analytes. We apply GoDig to quantify the impact of genetic variation on protein expression in mice fed a high-fat diet. We create several GoDig assays to quantify the expression of multiple protein families (kinases, lipid metabolism- and lipid droplet-associated proteins) across 480 fully-genotyped Diversity Outbred mice, revealing protein quantitative trait loci and establishing potential linkages between specific proteins and lipid homeostasis.',\n",
       "  '2600_peptides-GeneSymbol',\n",
       "  'each category were collected using either GoDig or RTS analyses of the unfractionated sample (RTS**1**)  Dataset 1'),\n",
       " 'PMC8502965-Table_3.xlsx-Table_S3_Blue_network_nodes_in-toNode': ('An Advanced Systems Pharmacology Strategy Reveals AKR1B1, MMP2, PTGER3 as Key Genes in the Competing Endogenous RNA Network of Compound Kushen Injection Treating Gastric Carcinoma by Integrated Bioinformatics and Experimental Verification',\n",
       "  'Gastric carcinoma (GC) is a severe tumor of the digestive tract with high morbidity and mortality and poor prognosis, for which novel treatment options are urgently needed. Compound Kushen injection (CKI), a classical injection of Chinese medicine, has been widely used to treat various tumors in clinical practice for decades. In recent years, a growing number of studies have confirmed that CKI has a beneficial therapeutic effect on GC, However, there are few reports on the potential molecular mechanism of action. Here, using systems pharmacology combined with proteomics analysis as a core concept, we identified the ceRNA network, key targets and signaling pathways regulated by CKI in the treatment of GC. To further explore the role of these key targets in the development of GC, we performed a meta-analysis to compare the expression differences between GC and normal gastric mucosa tissues. Functional enrichment analysis was further used to understand the biological pathways significantly regulated by the key genes. In addition, we determined the significance of the key genes in the prognosis of GC by survival analysis and immune infiltration analysis. Finally, molecular docking simulation was performed to verify the combination of CKI components and key targets. The anti-gastric cancer effect of CKI and its key targets was verified by in vivo and in vitro experiments. The analysis of ceRNA network of CKI on GC revealed that the potential molecular mechanism of CKI can regulate PI3K/AKT and Toll-like receptor signaling pathways by interfering with hub genes such as AKR1B1, MMP2 and PTGERR3 . In conclusion, this study not only partially highlighted the molecular mechanism of CKI in GC therapy but also provided a novel and advanced systems pharmacology strategy to explore the mechanisms of traditional Chinese medicine formulations.',\n",
       "  'Table_S3_Blue_network_nodes_in-toNode',\n",
       "  'gene linkages and the brown module consists of 232 genes and 11316 gene linkages (**Supplementary Tables 3**)  Blue network nodes in WGCNA  Click here for additional data file'),\n",
       " 'PMC9122025-NIHMS1780000-supplement-Data_S2.xlsx-CD8_Tem-Unnamed_0': ('CD4 + T cells contribute to neurodegeneration in Lewy body dementia',\n",
       "  'Recent studies indicate that the adaptive immune system plays a role in Lewy body dementia (LBD). However, the mechanism regulating T cell brain homing in LBD is unknown. Here, we observed T cells adjacent to Lewy bodies and dopaminergic neurons in postmortem LBD brains. Single-cell RNA sequencing of cerebrospinal fluid (CSF) identified up-regulated expression of C-X-C motif chemokine receptor 4 ( CXCR4 ) in CD4 + T cells in LBD. CSF protein levels of the CXCR4 ligand, C-X-C motif chemokine ligand 12 (CXCL12), were associated with neuroaxonal damage in LBD. Furthermore, we observed clonal expansion and up-regulated interleukin 17A expression by CD4 + T cells stimulated with a phosphorylated α-synuclein epitope. Thus, CXCR4-CXCL12 signaling may represent a mechanistic target for inhibiting pathological interleukin-17–producing T cell trafficking in LBD.',\n",
       "  'CD8_Tem-Unnamed_0',\n",
       "  'CD4+ T cells as the most transcriptionally altered immune cell subtype (Fig. 2C and **Data S2**)'),\n",
       " 'PMC9357053-NIHMS1810045-supplement-11.xlsx-Cluster_1-gene': ('Cell function and identity revealed by comparative scRNA-seq analysis in human nasal, bronchial and epididymis epithelia',\n",
       "  'The evolutionary relationship of cells within tissues having a similar function but located in different anatomical sites is of considerable biological interest. The development of single-cell RNA sequencing (scRNA-seq) protocols has greatly enhanced opportunities to address this topic. Here we focus on cells in the epithelium which lines two regions of the human respiratory tract and the male genital ducts to delineate the shared, differentiated functions of the different cell populations. Transcriptomic data were used to assess the gene expression profiles of human bronchial, nasal, and epididymal epithelium (HBE, HNE, and HEE). Bulk RNA-seq showed many shared genes expressed in cells from the nasal and bronchial epithelium and highlighted their divergence from the epididymal epithelium. ScRNA-seq in HBE and HNE cells demonstrated overlapping gene expression patterns within basal and secretory cell populations. Moreover, the distribution of cell types was altered in HNE cells derived from donors with cystic fibrosis (CF) when compared to cells from healthy donors. Next, the HBE and HNE datasets were merged and confirmed intersection of cell type gene expression profiles from the two sites. However, secretory and ciliated cells were the most abundant types in the HBE samples, while more basal cells were seen in the HNE populations. We then merged single-cell data from the epididymis to determine if overlapping functions of these cells corresponded to those in the airway. Of note, only the pulmonary ionocytes/epididymis clear cells showed a strongly conserved identity, which was confirmed by imputation in bulk RNA-seq datasets from the same cells.',\n",
       "  'Cluster_1-gene',\n",
       "  'components and a resolution of 0.1 visualized the distribution between 10 clusters (Fig. 5A, **Table S8**)'),\n",
       " 'PMC8063054-Table_2.xlsx-Sheet1-Table_S2_Microglial_signature_genes_in_this_study': ('Validation of Induced Microglia-Like Cells (iMG Cells) for Future Studies of Brain Diseases',\n",
       "  \"Microglia are the primary resident immune cells of the central nervous system that maintain physiological homeostasis in the brain and contribute to the pathogenesis of many psychiatric disorders and neurodegenerative diseases. Due to the lack of appropriate human cellular models, it is difficult to study the basic pathophysiological processes linking microglia to brain diseases. In this study, we adopted a microglia-like cellular model derived from peripheral blood monocytes with granulocyte-macrophage colony-stimulating factor (GM-CSF) and interleukin-34 (IL-34). We characterized and validated this in vitro cellular model by morphology, immunocytochemistry, gene expression profiles, and functional study. Our results indicated that the iMG cells developed typical microglial ramified morphology, expressed microglial specific surface markers (P2RY12 and TMEM119), and possessed phagocytic activity. Principal component analyses and multidimensional scaling analyses of RNA-seq data showed that iMG cells were distinct from monocytes and induced macrophages (iMacs) but clustered closer to human microglia and hiPSC-induced microglia. Heatmap analyses also found that iMG cells, but not monocytes, were closely clustered with human primary microglia. Further pathway and relative expression analysis indicated that unique genes from iMG cells were involved in the regulation of the complement system, especially in the synapse and ion transport. Overall, our data demonstrated that the iMG model mimicked many features of the brain resident microglia, highlighting its utility in the study of microglial function in many brain diseases, such as schizophrenia and Alzheimer's disease (AD).\",\n",
       "  'Sheet1-Table_S2_Microglial_signature_genes_in_this_study',\n",
       "  \"and 42 microglial genes from McQuade and Blurton-Jones's study (McQuade and Blurton-Jones, 2019) (See **Supplementary Table 2**)  Microglia signature genes in this study  Click here for additional data file\"),\n",
       " 'PMC5649134-Table_5.XLSX-Orthologous-upreg-Unnamed_1': ('Helicobacter pylori Affects the Antigen Presentation Activity of Macrophages Modulating the Expression of the Immune Receptor CD300E through miR-4270',\n",
       "  'Helicobacter pylori (Hp) is a Gram-negative bacterium that infects the human gastric mucosa, leading to chronic inflammation. If not eradicated with antibiotic treatment, the bacterium persists in the human stomach for decades increasing the risk to develop chronic gastritis, gastroduodenal ulcer, and gastric adenocarcinoma. The lifelong persistence of Hp in the human stomach suggests that the host response fails to clear the infection. It has been recently shown that during Hp infection phagocytic cells promote high Hp loads rather than contributing to bacterial clearance. Within these cells Hp survives in \"megasomes,\" large structures arising from homotypic fusion of phagosomes, but the mechanism that Hp employs to avoid phagocytic killing is not completely understood. Here, we show that Hp infection induces the downregulation of specific microRNAs involved in the regulation of transcripts codifying for inflammatory proteins. miR-4270 targets the most upregulated gene: the immune receptor CD300E , whose expression is strictly dependent on Hp infection. CD300E engagement enhances the pro-inflammatory potential of macrophages, but in parallel it affects their ability to express and expose MHC class II molecules on the plasma membrane, without altering phagocytosis. This effect compromises the possibility for effector T cells to recognize and activate the killing potential of macrophages, which, in turn would become a survival niche for the bacterium. Taken together, our data add another piece to the complicate puzzle represented by the long-life coexistence between Hp and the human host and contribute with new insights toward understanding the regulation and function of the immune receptor CD300E.',\n",
       "  'Orthologous-upreg-Unnamed_1',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC9166698-41467_2022_30861_MOESM4_ESM.xlsx-Figure_5-FeatureName_1': ('Peptide vaccine-treated, long-term surviving cancer patients harbor self-renewing tumor-specific CD8 + T cells',\n",
       "  'The behaviors and fates of immune cells in cancer patients, such as dysfunction and stem-like states leading to memory formation in T cells, are in intense focus of investigation. Here we show, by post hoc analysis of peripheral blood lymphocytes of hepatocellular carcinoma patients previously undergoing vaccination with tumour-associated antigen-derived peptides in our clinical trials (registration numbers UMIN000003511, UMIN000004540, UMIN000005677, UMIN000003514 and UMIN000005678), that induced peptide-specific T cell responses may persist beyond 10 years following vaccination. Tracking TCR clonotypes at the single cell level reveals in two patients that peptide-specific long-lasting CD8 + T cells acquire an effector memory phenotype that associates with cell cycle-related genes (CCNA2 and CDK1), and are characterized by high expression of IL7R, SELL, and NOSIP along with a later stage promotion of the AP-1 transcription factor network (5 years or more past vaccination). We conclude that effective anti-tumor immunity is governed by potentially proliferative memory T cells, specific to cancer antigens.',\n",
       "  'Figure_5-FeatureName_1',\n",
       "  'None here'),\n",
       " 'PMC8800370-NIHMS1769771-supplement-Supplemental_Table_2.xlsx-YFV_RNA_sig_gmx-YFV_T_Eff': ('CD19-CAR T cells undergo exhaustion DNA methylation programming in patients with acute lymphoblastic leukemia',\n",
       "  'CD19-CAR T cell therapy has evolved into the standard of care for relapsed/refractory B cell acute lymphoblastic leukemia (ALL); however, limited persistence of the CAR T cells enables tumor relapse for many patients. To gain a deeper understanding of the molecular characteristics associated with CAR T cell differentiation, we performed longitudinal genome-wide DNA methylation profiling of CD8 + CD19-CAR T cells post-infusion in ALL patients. We report that CAR T cells undergo a rapid and broad erasure of repressive DNA methylation reprograms at effector-associated genes. The CAR T cell post-infusion changes are further characterized by repression of genes (e.g., TCF7 and LEF1) associated with memory potential and a DNA methylation signature (e.g., demethylation at CX3CR1, BATF, and TOX) demarcating a transition toward exhaustion-progenitor T cells. Thus, CD19-CAR T cells undergo exhaustion-associated DNA methylation programming, indicating that efforts to prevent this process may be an attractive approach to improve CAR T cell efficacy.',\n",
       "  'YFV_RNA_sig_gmx-YFV_T_Eff',\n",
       "  'None here'),\n",
       " 'PMC4951018-pone.0159329.s004.xlsx-Cluster_2_670_2-5-14_txt-Unnamed_13': ('Full Spectrum of LPS Activation in Alveolar Macrophages of Healthy Volunteers by Whole Transcriptomic Profiling',\n",
       "  'Despite recent advances in understanding macrophage activation, little is known regarding how human alveolar macrophages in health calibrate its transcriptional response to canonical TLR4 activation. In this study, we examined the full spectrum of LPS activation and determined whether the transcriptomic profile of human alveolar macrophages is distinguished by a TIR-domain-containing adapter-inducing interferon-β (TRIF)-dominant type I interferon signature. Bronchoalveolar lavage macrophages were obtained from healthy volunteers, stimulated in the presence or absence of ultrapure LPS in vitro, and whole transcriptomic profiling was performed by RNA sequencing (RNA-Seq). LPS induced a robust type I interferon transcriptional response and Ingenuity Pathway Analysis predicted interferon regulatory factor (IRF)7 as the top upstream regulator of 89 known gene targets. Ubiquitin-specific peptidase (USP)-18, a negative regulator of interferon α/β responses, was among the top up-regulated genes in addition to IL10 and USP41, a novel gene with no known biological function but with high sequence homology to USP18. We determined whether IRF-7 and USP-18 can influence downstream macrophage effector cytokine production such as IL-10. We show that IRF-7 siRNA knockdown enhanced LPS-induced IL-10 production in human monocyte-derived macrophages, and USP-18 overexpression attenuated LPS-induced production of IL-10 in RAW264.7 cells. Quantitative PCR confirmed upregulation of USP18, USP41, IL10, and IRF7. An independent cohort confirmed LPS induction of USP41 and IL10 genes. These results suggest that IRF-7 and predicted downstream target USP18, both elements of a type I interferon gene signature identified by RNA-Seq, may serve to fine-tune early cytokine response by calibrating IL-10 production in human alveolar macrophages.',\n",
       "  'Cluster_2_670_2-5-14_txt-Unnamed_13',\n",
       "  'Expression values of all upregulated genes. (control vs. LPS, t-test, adjusted p-value <0.05, Benjamini-Hochberg).Values represent mean expression (log2).(XLSX)  Click here for additional data file'),\n",
       " 'PMC7801680-41598_2020_79740_MOESM11_ESM.xlsx-network_hubGenes-AD_Hub_Genes': (\"Combinatorial analyses reveal cellular composition changes have different impacts on transcriptomic changes of cell type specific genes in Alzheimer's Disease\",\n",
       "  \"Alzheimer's disease (AD) brains are characterized by progressive neuron loss and gliosis. Previous studies of gene expression using bulk tissue samples often fail to consider changes in cell-type composition when comparing AD versus control, which can lead to differences in expression levels that are not due to transcriptional regulation. We mined five large transcriptomic AD datasets for conserved gene co-expression module, then analyzed differential expression and differential co-expression within the modules between AD samples and controls. We performed cell-type deconvolution analysis to determine whether the observed differential expression was due to changes in cell-type proportions in the samples or to transcriptional regulation. Our findings were validated using four additional datasets. We discovered that the increased expression of microglia modules in the AD samples can be explained by increased microglia proportions in the AD samples. In contrast, decreased expression and perturbed co-expression within neuron modules in the AD samples was likely due in part to altered regulation of neuronal pathways. Several transcription factors that are differentially expressed in AD might account for such altered gene regulation. Similarly, changes in gene expression and co-expression within astrocyte modules could be attributed to combined effects of astrogliosis and astrocyte gene activation. Gene expression in the astrocyte modules was also strongly correlated with clinicopathological biomarkers. Through this work, we demonstrated that combinatorial analysis can delineate the origins of transcriptomic changes in bulk tissue data and shed light on key genes and pathways involved in AD.\",\n",
       "  'network_hubGenes-AD_Hub_Genes',\n",
       "  'AD samples. We identified 136 genes for each case (Supplementary Fig. 6; Supplementary Table **14**)  Supplementary Table 14'),\n",
       " 'PMC11005406-mbio.02623-23-s0006.xlsx-DEG_IFNB_vs_Media_RenLuc-Name': ('Interferon lambda restricts herpes simplex virus skin disease by suppressing neutrophil-mediated pathology',\n",
       "  'Type III interferons (IFN-λ) are antiviral and immunomodulatory cytokines that have been best characterized in respiratory and gastrointestinal infections, but the effects of IFN-λ against skin infections have not been extensively investigated. We sought to define the skin-specific effects of IFN-λ against the highly prevalent human pathogen, herpes simplex virus (HSV). We infected mice lacking the IFN-λ receptor ( Ifnlr1 -/- ), both the IFN-λ and the IFN-α/β receptors ( Ifnar1 -/- Ifnlr1 -/- ), or IFN-λ cytokines ( Ifnl2/3 -/- ) and found that IFN-λ restricts the severity of HSV-1 and HSV-2 skin lesions without affecting viral loads. We used RNAseq to define IFN-λ- and IFN-β-induced transcriptional responses in primary mouse keratinocytes. Using conditional knockout mice, we found that IFN-λ signaling in both keratinocytes and neutrophils was necessary to control HSV-1 skin lesion severity and that IFN-λ signaling in keratinocytes suppressed CXCL9-mediated neutrophil recruitment to the skin. Furthermore, depleting neutrophils or blocking CXCL9 protected against severe HSV-1 skin lesions in Ifnlr1 -/- mice. Altogether, our results suggest that IFN-λ plays an immunomodulatory role in the skin that restricts neutrophil-mediated pathology during HSV infection and suggests potential applications for IFN-λ in treating viral skin infections.IMPORTANCEType III interferons (IFN-λ) have been shown to have antiviral and immunomodulatory effects at epithelial barriers such as the respiratory and gastrointestinal tracts, but their effects on the skin have not been extensively investigated. We used mice lacking IFN-λ signaling to investigate the skin-specific effects of IFN-λ against the herpes simplex virus (HSV), which targets epithelial tissues to cause cold sores and genital herpes. We found that IFN-λ limited the severity of HSV skin lesions without affecting viral load and that this protective effect required IFN-λ signaling in both keratinocytes and neutrophils. We found that IFN-λ signaling in keratinocytes suppressed neutrophil recruitment to the skin and that depleting neutrophils protected against severe HSV skin lesions in the absence of IFN-λ. Altogether, our results suggest that IFN-λ plays an immunomodulatory role in the skin that restricts neutrophil-mediated pathology during HSV infection and suggests potential applications for IFN-λ in treating viral skin infections.',\n",
       "  '23-s0006.xlsx-DEG_IFNB_vs_Media_RenLuc-Name',\n",
       "  'None here'),\n",
       " 'PMC9579367-Table_1.xlsx-GSE6764_DOWN-Gene_symbol': ('Application of genomic selection and experimental techniques to predict cell death and immunotherapeutic efficacy of ferroptosis-related CXCL2 in hepatocellular carcinoma',\n",
       "  \"Since most hepatocellular carcinoma (HCC) patients are diagnosed at advanced stages, there is no effective treatment to improve patient survival. Ferroptosis, a regulated cell death driven by iron accumulation and lipid peroxidation, has been reported to play an important role in tumorigenesis. However, the detailed mechanism and biological function of ferroptosis are still incompletely understood in HCC patients. In this study, we analyzed genomic profiles of three HCC datasets, GSE6764, GSE14520, and GSE14323. Venn diagrams were implemented to visualize the overlapping genes between differentially expressed genes and ferroptosis-related gene set. Then, one up-regulated gene, ACSL4, and five down-regulated genes, STEAP3, MT1G, GCH1, HAMP, and CXCL2, were screened. Based on the survival analysis performed by Kaplan-Meier plotter database, ferroptosis-related gene CXCL2 was demonstrated positively-correlated with the patients' prognosis. Moreover, CXCL2 overexpression significantly inhibited cell growth and improved cellular ROS, Fe 2+ and MDA levels in HCC cells Huh7 and MHCC97H, suggesting the roles of CXCL2 in inducing ferroptotic cell death. In addition, aberrantly expressed CXCL2 was negatively associated with malignancy clinical features, such as nodal metastasis and higher grades. The ssGSEA enrichment analysis revealed that CXCL2 co-expressed molecules were mainly involved in inflammation and immune-related pathways, such as acute inflammatory response, humoral immune response, adaptive immune response. TISIDB algorithm indicated the positive correlation between CXCL2 expression and tumor-infiltrating immune cells, including neutrophils and macrophages. Additionally, we also found that CXCL2 was positively correlated with immune infiltration score, and HCC patients with higher score harbored better prognosis. Together, these findings suggested that CXCL2 may enhance ferroptosis sensitivity and regulate immune microenvironment in HCC, and serve as a promising prognosis biomarker for HCC patients.\",\n",
       "  'GSE6764_DOWN-Gene_symbol',\n",
       "  'and 583 in GSE14520 had been identified to be significantly down-regulated in HCC ( **Supplementary Table S1**)  Differentially expressed genes in GSE6764, GSE14323, and GSE14520  Click here for additional data file'),\n",
       " 'PMC8125900-mmc2.csv--Unnamed_2': ('Longitudinal analysis reveals that delayed bystander CD8+ T cell activation and early immune pathology distinguish severe COVID-19 from mild disease',\n",
       "  'The kinetics of the immune changes in COVID-19 across severity groups have not been rigorously assessed. Using immunophenotyping, RNA sequencing, and serum cytokine analysis, we analyzed serial samples from 207 SARS-CoV2-infected individuals with a range of disease severities over 12 weeks from symptom onset. An early robust bystander CD8 + T cell immune response, without systemic inflammation, characterized asymptomatic or mild disease. Hospitalized individuals had delayed bystander responses and systemic inflammation that was already evident near symptom onset, indicating that immunopathology may be inevitable in some individuals. Viral load did not correlate with this early pathological response but did correlate with subsequent disease severity. Immune recovery is complex, with profound persistent cellular abnormalities in severe disease correlating with altered inflammatory responses, with signatures associated with increased oxidative phosphorylation replacing those driven by cytokines tumor necrosis factor (TNF) and interleukin (IL)-6. These late immunometabolic and immune defects may have clinical implications.',\n",
       "  'Unnamed_2',\n",
       "  'modules correlated with both disease severity and time (Figures 3A, 3B, S4C, and S4D; **Table S3**)  Table S3. WGCNA module gene list, heme GSEA intersect genes (shared between C/D/E in the leading edge), and signature used for GSEA anlysis, related to Figures 3, 7B, and 5F, respectively'),\n",
       " 'PMC10274745-media-4.xlsx-S3A-Table_S3_Reanalysis_of_publicly_available_data_to_identify_AR_and_ER_direct_target_genes_related_to_Figure_1': ('The human Y and inactive X chromosomes similarly modulate autosomal gene expression',\n",
       "  'Somatic cells of human males and females have 45 chromosomes in common, including the \"active\" X chromosome. In males the 46 th chromosome is a Y; in females it is an \"inactive\" X (Xi). Through linear modeling of autosomal gene expression in cells from individuals with zero to three Xi and zero to four Y chromosomes, we found that Xi and Y impact autosomal expression broadly and with remarkably similar effects. Studying sex-chromosome structural anomalies, promoters of Xi- and Y-responsive genes, and CRISPR inhibition, we traced part of this shared effect to homologous transcription factors - ZFX and ZFY - encoded by Chr X and Y. This demonstrates sex-shared mechanisms by which Xi and Y modulate autosomal expression. Combined with earlier analyses of sex-linked gene expression, our studies show that 21% of all genes expressed in lymphoblastoid cells or fibroblasts change expression significantly in response to Xi or Y chromosomes.',\n",
       "  'S3A-Table_S3_Reanalysis_of_publicly_available_data_to_identify_AR_and_ER_direct_target_genes_related_to_Figure_1',\n",
       "  'genes did not overlap with direct target genes for estrogen or androgen receptors (Methods, **Table S3**)'),\n",
       " 'PMC7811787-peerj-09-10628-s004.xlsx-cox_result-stromal-Terms': ('Tumor microenvironment related novel signature predict lung adenocarcinoma survival',\n",
       "  'Background: Lung adenocarcinoma (LUAD) is the most common histological type of lung cancers, which is the primary cause of cancer-related mortality worldwide. Growing evidence has suggested that tumor microenvironment (TME) plays a pivotal role in tumorigenesis and progression. Hence, we investigate the correlation of TME related genes with LUAD prognosis. Method: The information of LUAD gene expression data was obtained from The Cancer Genome Atlas (TCGA). According to their immune/stromal scores calculated by the ESTIMATE algorithm, differentially expressed genes (DEGs) were identified. Then, we performed univariate Cox regression analysis on DEGs to obtain genes that are apparently bound up with LUAD survival (SurGenes). Functional annotation and protein-protein interaction (PPI) was also conducted on SurGenes. By validating the SurGenes with data sets of lung cancer from the Gene Expression Omnibus (GEO), 106 TME related SurGenes were generated. Further, intersection analysis was executed between the 106 TME related SurGenes and hub genes from PPI network, PTPRC and CD19 were obtained. Gene Set Enrichment Analysis and CIBERSORT analysis were performed on PTPRC and CD19. Based on the TCGA LUAD dataset, we conducted factor analysis and Step-wise multivariate Cox regression analysis for 106 TME related SurGenes to construct the prognostic model for LUAD survival prediction. The LUAD dataset in GEO (GSE68465) was used as the testing dataset to confirm the prognostic model. Multivariate Cox regression analysis was used between risk score from the prognostic model and clinical parameters. Result: A total of 106 TME related genes were collected in our research totally, which were markedly correlated with the overall survival (OS) of LUAD patient. Bioinformatics analysis suggest them mainly concentrated on immune response, cell adhesion, and extracellular matrix. More importantly, among 106 TME related SurGenes, PTPRC and CD19 were highly interconnected nodes among PPI network and correlated with immune activity, exhibiting significant prognostic potential. The prognostic model was a weighted linear combination of the 106 genes, by which the low-OS LUAD samples could be separated from the high-OS samples with success. This model was also able to rebustly predict the situation of survival (training set: p -value < 0.0001, area under the curve (AUC) = 0.649; testing set: p -value = 0.0009, AUC = 0.617). By combining with clinical parameters, the prognostic model was optimized. The AUC achieved 0.716 for 3 year and 0.699 for 5 year. Conclusion: A series of TME-related prognostic genes were acquired in this research, which could reflect immune disorders within TME, and PTPRC and CD19 show the potential to be an indicator for LUAD prognosis and tumor microenvironment modulation. The prognostic model constructed base on those prognostic genes presented a high predictive ability, and may have clinical implications in the overall survival prediction of LUAD.',\n",
       "  'cox_result-stromal-Terms',\n",
       "  'total of 542 genes (prognostic DEGs) associated with overall survival in patients with LUAD (**Table S1**)  Cox coefficients and significance of cox regression  Click here for additional data file'),\n",
       " 'PMC10098144-JEM_20230002_TableS3.xlsx-Tonsil_Tfh_cells-NOS1AP': ('Functional specialization of short-lived and long-lived macrophage subsets in human tonsils',\n",
       "  'Macrophages play a central role in tissue homeostasis and host defense. However, the properties of human macrophages in non-diseased tissues remain poorly understood. Here, we characterized human tonsil macrophages and identified three subsets with distinct phenotype, transcriptome, life cycle, and function. CD36hi macrophages were related to monocytes, while CD36lo macrophages showed features of embryonic origin and CD36int macrophages had a mixed profile. scRNA-seq on non-human primate tonsils showed that monocyte recruitment did not pre-exist an immune challenge. Functionally, CD36hi macrophages were specialized for stimulating T follicular helper cells, by producing Activin A. Combining reconstruction of ligand-receptor interactions and functional assays, we identified stromal cell-derived TNF-α as an inducer of Activin A secretion. However, only CD36hi macrophages were primed for Activin A expression, via the activity of IRF1. Our results provide insight into the heterogeneity of human lymphoid organ macrophages and show that tonsil CD36hi macrophage specialization is the result of both intrinsic features and interaction with stromal cells.',\n",
       "  'Tonsil_Tfh_cells-NOS1AP',\n",
       "  'yolk sac myeloid progenitors and human CD14+ blood monocytes, as well as cell cycle (**Table S3**)  shows gene signatures used in GSEA analysis  Click here for additional data file'),\n",
       " 'PMC3597511-pgen.1003362.s001.xlsx-Sheet1-Unnamed_21': ('Blood-informative transcripts define nine common axes of peripheral blood gene expression',\n",
       "  'We describe a novel approach to capturing the covariance structure of peripheral blood gene expression that relies on the identification of highly conserved Axes of variation. Starting with a comparison of microarray transcriptome profiles for a new dataset of 189 healthy adult participants in the Emory-Georgia Tech Center for Health Discovery and Well-Being (CHDWB) cohort, with a previously published study of 208 adult Moroccans, we identify nine Axes each with between 99 and 1,028 strongly co-regulated transcripts in common. Each axis is enriched for gene ontology categories related to sub-classes of blood and immune function, including T-cell and B-cell physiology and innate, adaptive, and anti-viral responses. Conservation of the Axes is demonstrated in each of five additional population-based gene expression profiling studies, one of which is robustly associated with Body Mass Index in the CHDWB as well as Finnish and Australian cohorts. Furthermore, ten tightly co-regulated genes can be used to define each Axis as \"Blood Informative Transcripts\" (BITs), generating scores that define an individual with respect to the represented immune activity and blood physiology. We show that environmental factors, including lifestyle differences in Morocco and infection leading to active or latent tuberculosis, significantly impact specific axes, but that there is also significant heritability for the Axis scores. In the context of personalized medicine, reanalysis of the longitudinal profile of one individual during and after infection with two respiratory viruses demonstrates that specific axes also characterize clinical incidents. This mode of analysis suggests the view that, rather than unique subsets of genes marking each class of disease, differential expression reflects movement along the major normal Axes in response to environmental and genetic stimuli.',\n",
       "  'Sheet1-Unnamed_21',\n",
       "  'included in a list of between 99 and 1028 genes that define each axis (**Dataset S1**)  List of genes associated with each of the 9 axes in both Atlanta and Morocco. The 9 sets of columns show the Illumina Probe identifier, the Gene name, and the negative logarithm of the p-value (NLP) for the association of the transcript with the axis score (PC1) in the multiple regression with all 9 axes in Atlanta (NLP_ATL) and Morocco (NLP_MOR). Only probes significant at the Bonferroni adjusted significance level (NLP>5.53) in both studies are included, and they are sorted by average NLP.(XLSX)  Click here for additional data file'),\n",
       " 'PMC10274745-media-4.xlsx-S3A-Unnamed_12': ('The human Y and inactive X chromosomes similarly modulate autosomal gene expression',\n",
       "  'Somatic cells of human males and females have 45 chromosomes in common, including the \"active\" X chromosome. In males the 46 th chromosome is a Y; in females it is an \"inactive\" X (Xi). Through linear modeling of autosomal gene expression in cells from individuals with zero to three Xi and zero to four Y chromosomes, we found that Xi and Y impact autosomal expression broadly and with remarkably similar effects. Studying sex-chromosome structural anomalies, promoters of Xi- and Y-responsive genes, and CRISPR inhibition, we traced part of this shared effect to homologous transcription factors - ZFX and ZFY - encoded by Chr X and Y. This demonstrates sex-shared mechanisms by which Xi and Y modulate autosomal expression. Combined with earlier analyses of sex-linked gene expression, our studies show that 21% of all genes expressed in lymphoblastoid cells or fibroblasts change expression significantly in response to Xi or Y chromosomes.',\n",
       "  'S3A-Unnamed_12',\n",
       "  'genes did not overlap with direct target genes for estrogen or androgen receptors (Methods, **Table S3**)'),\n",
       " 'PMC10849790-CPR-57-e13545-s005.xlsx-prolife_vs_myoblast1SJ_expre-gene_short_name': ('Bulk and single-cell alternative splicing analyses reveal roles of TRA2B in myogenic differentiation',\n",
       "  'Alternative splicing (AS) disruption has been linked to disorders of muscle development, as well as muscular atrophy. However, the precise changes in AS patterns that occur during myogenesis are not well understood. Here, we employed isoform long-reads RNA-seq (Iso-seq) and single-cell RNA-seq (scRNA-seq) to investigate the AS landscape during myogenesis. Our Iso-seq data identified 61,146 full-length isoforms representing 11,682 expressed genes, of which over 52% were novel. We identified 38,022 AS events, with most of these events altering coding sequences and exhibiting stage-specific splicing patterns. We identified AS dynamics in different types of muscle cells through scRNA-seq analysis, revealing genes essential for the contractile muscle system and cytoskeleton that undergo differential splicing across cell types. Gene-splicing analysis demonstrated that AS acts as a regulator, independent of changes in overall gene expression. Two isoforms of splicing factor TRA2B play distinct roles in myogenic differentiation by triggering AS of TGFBR2 to regulate canonical TGF-β signalling cascades differently. Our study provides a valuable transcriptome resource for myogenesis and reveals the complexity of AS and its regulation during myogenesis.',\n",
       "  'prolife_vs_myoblast1SJ_expre-gene_short_name',\n",
       "  'None here'),\n",
       " 'PMC9945212-DataSheet_4.csv--Gene': ('Deep learning-based morphological feature analysis and the prognostic association study in colon adenocarcinoma histopathological images',\n",
       "  \"Colorectal cancer (CRC) is now the third most common malignancy to cause mortality worldwide, and its prognosis is of great importance. Recent CRC prognostic prediction studies mainly focused on biomarkers, radiometric images, and end-to-end deep learning methods, while only a few works paid attention to exploring the relationship between the quantitative morphological features of patients' tissue slides and their prognosis. However, existing few works in this area suffered from the drawback of choosing the cells randomly from the whole slides, which contain the non-tumor region that lakes information about prognosis. In addition, the existing works, which tried to demonstrate their biological interpretability using patients' transcriptome data, failed to show the biological meaning closely related to cancer. In this study, we proposed and evaluated a prognostic model using morphological features of cells in the tumor region. The features were first extracted by the software CellProfiler from the tumor region selected by Eff-Unet deep learning model. Features from different regions were then averaged for each patient as their representative, and the Lasso-Cox model was used to select the prognosis-related features. The prognostic prediction model was at last constructed using the selected prognosis-related features and was evaluated through KM estimate and cross-validation. In terms of biological meaning, Gene Ontology (GO) enrichment analysis of the expressed genes that correlated with the prognostically significant features was performed to show the biological interpretability of our model.With the help of tumor segmentation, our model achieved better statistical significance and better biological interpretability compared to the results without tumor segmentation. Statistically, the Kaplan Meier (KM) estimate of our model showed that the model using features in the tumor region has a higher C-index, a lower p-value, and a better performance on cross-validation than the model without tumor segmentation. In addition, revealing the pathway of the immune escape and the spread of the tumor, the model with tumor segmentation demonstrated a biological meaning much more related to cancer immunobiology than the model without tumor segmentation. Our prognostic prediction model using quantitive morphological features from tumor regions was almost as good as the TNM tumor staging system as they had a close C-index, and our model can be combined with the TNM tumor stage system to make a better prognostic prediction. And to the best of our knowledge, the biological mechanisms in our study were the most relevant to the immune mechanism of cancer compared to the previous studies.\",\n",
       "  'Gene',\n",
       "  'of 645 unique genes that were correlated with one or more morphological features ( **Supplementary Table 4**)  The correlation between patients’ prognostically significant morphological features and their expressed genes  Click here for additional data file'),\n",
       " 'PMC8632991-41525_2021_264_MOESM2_ESM.xlsx-Table_S13_Long_gene_overlap-Unnamed_5': ('Recurrent integration of human papillomavirus genomes at transcriptional regulatory hubs',\n",
       "  'Oncogenic human papillomavirus (HPV) genomes are often integrated into host chromosomes in HPV-associated cancers. HPV genomes are integrated either as a single copy or as tandem repeats of viral DNA interspersed with, or without, host DNA. Integration occurs frequently in common fragile sites susceptible to tandem repeat formation and the flanking or interspersed host DNA often contains transcriptional enhancer elements. When co-amplified with the viral genome, these enhancers can form super-enhancer-like elements that drive high viral oncogene expression. Here we compiled highly curated datasets of HPV integration sites in cervical (CESC) and head and neck squamous cell carcinoma (HNSCC) cancers, and assessed the number of breakpoints, viral transcriptional activity, and host genome copy number at each insertion site. Tumors frequently contained multiple distinct HPV integration sites but often only one \"driver\" site that expressed viral RNA. As common fragile sites and active enhancer elements are cell-type-specific, we mapped these regions in cervical cell lines using FANCD2 and Brd4/H3K27ac ChIP-seq, respectively. Large enhancer clusters, or super-enhancers, were also defined using the Brd4/H3K27ac ChIP-seq dataset. HPV integration breakpoints were enriched at both FANCD2-associated fragile sites and enhancer-rich regions, and frequently showed adjacent focal DNA amplification in CESC samples. We identified recurrent integration \"hotspots\" that were enriched for super-enhancers, some of which function as regulatory hubs for cell-identity genes. We propose that during persistent infection, extrachromosomal HPV minichromosomes associate with these transcriptional epicenters and accidental integration could promote viral oncogene expression and carcinogenesis.',\n",
       "  'Table_S13_Long_gene_overlap-Unnamed_5',\n",
       "  'dataset of HPV integration breakpoints was assembled from various sources5,6,8,9,30–37, as outlined in Fig. **1**  Supplementary Data 1-17'),\n",
       " 'PMC6969427-10020_2019_135_MOESM3_ESM.xlsx-IPF-Unnamed_29': ('Identification of biomarkers in common chronic lung diseases by co-expression networks and drug-target interactions analysis',\n",
       "  'Background: asthma, chronic obstructive pulmonary disease (COPD), and idiopathic pulmonary fibrosis (IPF) are three serious pulmonary diseases that contain common and unique characteristics. Therefore, the identification of biomarkers that differentiate these diseases is of importance for preventing misdiagnosis. In this regard, the present study aimed to identify the disorders at the early stages, based on lung transcriptomics data and drug-target interactions. Methods: To this end, the differentially expressed genes were found in each disease. Then, WGCNA was utilized to find specific and consensus gene modules among the three diseases. Finally, the disease-disease similarity was analyzed, followed by determining candidate drug-target interactions. Results: The results confirmed that the asthma lung transcriptome was more similar to COPD than IPF. In addition, the biomarkers were found in each disease and thus were proposed for further clinical validations. These genes included RBM42, STX5, and TRIM41 in asthma, CYP27A1, GM2A, LGALS9, SPI1, and NLRC4 in COPD, ATF3, PPP1R15A, ZFP36, SOCS3, NAMPT, and GADD45B in IPF, LRRC48 and CETN2 in asthma-COPD, COL15A1, GIMAP6, and JAM2 in asthma-IPF and LMO7, TSPAN13, LAMA3, and ANXA3 in COPD-IPF. Finally, analyzing drug-target networks suggested anti-inflammatory candidate drugs for treating the above mentioned diseases. Conclusion: In general, the results revealed the unique and common biomarkers among three chronic lung diseases. Eventually, some drugs were suggested for treatment purposes.',\n",
       "  'IPF-Unnamed_29',\n",
       "  'Disease-specific DEGs (i.e., 1907, 1770, and 18**3**)  Additional file 3. The list of all gene symbols in candidate disease-specific modules'),\n",
       " 'PMC10274745-media-4.xlsx-S3A-Unnamed_10': ('The human Y and inactive X chromosomes similarly modulate autosomal gene expression',\n",
       "  'Somatic cells of human males and females have 45 chromosomes in common, including the \"active\" X chromosome. In males the 46 th chromosome is a Y; in females it is an \"inactive\" X (Xi). Through linear modeling of autosomal gene expression in cells from individuals with zero to three Xi and zero to four Y chromosomes, we found that Xi and Y impact autosomal expression broadly and with remarkably similar effects. Studying sex-chromosome structural anomalies, promoters of Xi- and Y-responsive genes, and CRISPR inhibition, we traced part of this shared effect to homologous transcription factors - ZFX and ZFY - encoded by Chr X and Y. This demonstrates sex-shared mechanisms by which Xi and Y modulate autosomal expression. Combined with earlier analyses of sex-linked gene expression, our studies show that 21% of all genes expressed in lymphoblastoid cells or fibroblasts change expression significantly in response to Xi or Y chromosomes.',\n",
       "  'S3A-Unnamed_10',\n",
       "  'genes did not overlap with direct target genes for estrogen or androgen receptors (Methods, **Table S3**)'),\n",
       " 'PMC10274745-media-4.xlsx-S3A-Unnamed_13': ('The human Y and inactive X chromosomes similarly modulate autosomal gene expression',\n",
       "  'Somatic cells of human males and females have 45 chromosomes in common, including the \"active\" X chromosome. In males the 46 th chromosome is a Y; in females it is an \"inactive\" X (Xi). Through linear modeling of autosomal gene expression in cells from individuals with zero to three Xi and zero to four Y chromosomes, we found that Xi and Y impact autosomal expression broadly and with remarkably similar effects. Studying sex-chromosome structural anomalies, promoters of Xi- and Y-responsive genes, and CRISPR inhibition, we traced part of this shared effect to homologous transcription factors - ZFX and ZFY - encoded by Chr X and Y. This demonstrates sex-shared mechanisms by which Xi and Y modulate autosomal expression. Combined with earlier analyses of sex-linked gene expression, our studies show that 21% of all genes expressed in lymphoblastoid cells or fibroblasts change expression significantly in response to Xi or Y chromosomes.',\n",
       "  'S3A-Unnamed_13',\n",
       "  'genes did not overlap with direct target genes for estrogen or androgen receptors (Methods, **Table S3**)'),\n",
       " 'PMC8558420-DataSheet_1.csv--Unnamed_0': ('Integrative Characterization of the Role of IL27 In Melanoma Using Bioinformatics Analysis',\n",
       "  'Background: IL27 has been reported to play dual roles in cancer; however, its effects on the tumor microenvironment (TME), immunotherapy, and prognosis in melanoma remain largely unclear. This study was aimed to uncover the effects of IL27 on TME, immunotherapy and prognosis in patients with melanoma. Methods: RNA-seq data, drug sensitivity data, and clinical data were obtained from TCGA, GEO, CCLE, and CTRP. Log-rank test was used to determine the survival value of IL27 . Univariate and multivariate Cox regression analyses were employed to determine the independent predictors of survival outcomes. DAVID and GSEA were used to perform gene set functional annotations. ssGSEA was used to explore the association between IL27 and immune infiltrates. ConsensusClusterPlus was used to classify melanoma tissues into hot tumors or cold tumors. Results: Clinically, IL27 was negatively correlated with Breslow depth ( P = 0.00042) and positively associated with response to radiotherapy ( P = 0.038). High IL27 expression showed an improved survival outcome ( P = 0.00016), and could serve as an independent predictor of survival outcomes (hazard ratio: 0.32 - 0.88, P = 0.015). Functionally, elevated IL27 expression could induce an enhanced immune response and pyroptosis ( R = 0.64, P = 1.2e-55), autophagy ( R = 0.37, P = 7.1e-17) and apoptosis ( R = 0.47, P = 1.1e-27) in patients with melanoma. Mechanistically, elevated IL27 expression was positively correlated with cytotoxic cytokines (including INFG and GZMB ), enhanced immune infiltrates, and elevated CD8/Treg ratio ( R = 0.14, P = 0.02), possibly driving CD8 + T cell infiltration by suppressing β-catenin signaling in the TME. Furthermore, IL27 was significantly associated with hot tumor state, multiple predictors of response to immunotherapy, and improved drug response in patients with melanoma. Conclusions: IL27 was correlated with enriched CD8 + T cells, desirable therapeutic response and improved prognosis. It thus can be utilized as a promising modulator in the development of cytokine-based immunotherapy for melanoma.',\n",
       "  'Unnamed_0',\n",
       "  'autophagy, and β-catenin signaling were obtained by retrieving previous literature and are provided in **Supplementary Table S1**  Click here for additional data file'),\n",
       " 'PMC5390115-mmc6.xlsx-DE_LISTS-Unnamed_28': ('A Genome-wide Analysis of Human Pluripotent Stem Cell-Derived Endothelial Cells in 2D or 3D Culture',\n",
       "  'A defined protocol for efficiently deriving endothelial cells from human pluripotent stem cells was established and vascular morphogenesis was used as a model system to understand how synthetic hydrogels influence global biological function compared with common 2D and 3D culture platforms. RNA sequencing demonstrated that gene expression profiles were similar for endothelial cells and pericytes cocultured in polyethylene glycol (PEG) hydrogels or Matrigel, while monoculture comparisons identified distinct vascular signatures for each cell type. Endothelial cells cultured on tissue-culture polystyrene adopted a proliferative phenotype compared with cells cultured on or encapsulated in PEG hydrogels. The proliferative phenotype correlated to increased FAK-ERK activity, and knockdown or inhibition of ERK signaling reduced proliferation and expression for cell-cycle genes while increasing expression for \"3D-like\" vasculature development genes. Our results provide insight into the influence of 2D and 3D culture formats on global biological processes that regulate cell function.',\n",
       "  'DE_LISTS-Unnamed_28',\n",
       "  'genes (higher expression on TCP) were predominantly associated with the cell cycle and mitosis (**Table S5**)  Table S5. Differential Gene Expression for Cells Cultured on TCP Surfaces, in 2D, or Encapsulated in PEG Hydrogels, in 3D, Related to Figures 4 and 5'),\n",
       " 'PMC9392949-12876_2022_2475_MOESM1_ESM.xlsx-Table_S7-Unnamed_1': ('NAPSB as a predictive marker for prognosis and therapy associated with an immuno-hot tumor microenvironment in hepatocellular carcinoma',\n",
       "  'Background: Napsin B Aspartic Peptidase, Pseudogene (NAPSB) was associated with CD4 + T cell infiltration in pancreatic ductal adenocarcinoma. However, the biological role of NAPSB in hepatocellular carcinoma (HCC) remains to be determined. Methods: The expression of NAPSB in HCC as well as its clinicopathological association were analyzed using data from several public datasets. qRT-PCR was used to verify the relative expression of NAPSB in patients with HCC using the Zhongnan cohort. Kaplan-Meier analyses, and univariate and multivariate Cox regression were conducted to determine the prognosis value of NAPSB on patients with HCC. Then enrichment analyses were performed to identify the possible biological functions of NAPSB. Subsequently, the immunological characteristics of NAPSB in the HCC tumor microenvironment (TME) were demonstrated comprehensively. The role of NAPSB in predicting hot tumors and its impact on immunotherapy and chemotherapy responses was also analyzed by bioinformatics methods. Results: NAPSB was downregulated in patients with HCC and high NAPSB expression showed an improved survival outcome. Enrichment analyses showed that NAPSB was related to immune activation. NAPSB was positively correlated with immunomodulators, tumor-infiltrating immune cells, T cell inflamed score and cancer-immunity cycle, and highly expressed in immuno-hot tumors. High expression of NAPSB was sensitive to immunotherapy and chemotherapy, possibly due to its association with pyroptosis, apoptosis and necrosis. Conclusions: NAPSB was correlated with an immuno-hot and inflamed TME, and tumor cell death. It can be utilized as a promising predictive marker for prognosis and therapy in HCC.',\n",
       "  'Table_S7-Unnamed_1',\n",
       "  '(MHC), immunomodulators, chemokines and receptors) were collected from the study of Charoentong et al. [**1**]  Additional file 1. Supplementary Tables 1-4 and 7-15. Table S1. 50 immunomodulators. Table S2. Effector genes of TIICs. Table S3. Gene sets of therapeutic signatures. Table S4. Gene sets of cell death signatures. Table S7. Co-Expressed genes of NAPSB. Table S8. Upregulated expressed genes between NAPSB subgroups. Table S9. Common genes of co-expressed genes and upregulated DEGs. Table S10. Functional annotation and pathway analysis for common genes (Gene Ontology-Biological process). Table S11. Functional annotation and pathway analysis for common genes (KEGG pathways)'),\n",
       " 'PMC6020373-13073_2018_558_MOESM2_ESM.xlsx-Table_S2-Unnamed_17': ('Disease-specific regulation of gene expression in a comparative analysis of juvenile idiopathic arthritis and inflammatory bowel disease',\n",
       "  \"Background: The genetic and immunological factors that contribute to differences in susceptibility and progression between sub-types of inflammatory and autoimmune diseases continue to be elucidated. Inflammatory bowel disease and juvenile idiopathic arthritis are both clinically heterogeneous and known to be due in part to abnormal regulation of gene activity in diverse immune cell types. Comparative genomic analysis of these conditions is expected to reveal differences in underlying genetic mechanisms of disease. Methods: We performed RNA-Seq on whole blood samples from 202 patients with oligoarticular, polyarticular, or systemic juvenile idiopathic arthritis, or with Crohn's disease or ulcerative colitis, as well as healthy controls, to characterize differences in gene expression. Gene ontology analysis combined with Blood Transcript Module and Blood Informative Transcript analysis was used to infer immunological differences. Comparative expression quantitative trait locus (eQTL) analysis was used to quantify disease-specific regulation of transcript abundance. Results: A pattern of differentially expressed genes and pathways reveals a gradient of disease spanning from healthy controls to oligoarticular, polyarticular, and systemic juvenile idiopathic arthritis (JIA); Crohn's disease; and ulcerative colitis. Transcriptional risk scores also provide good discrimination of controls, JIA, and IBD. Most eQTL are found to have similar effects across disease sub-types, but we also identify disease-specific eQTL at loci associated with disease by GWAS. Conclusion: JIA and IBD are characterized by divergent peripheral blood transcriptomes, the genetic regulation of which displays limited disease specificity, implying that disease-specific genetic influences are largely independent of, or downstream of, eQTL effects.\",\n",
       "  'Table_S2-Unnamed_17',\n",
       "  'browser since it has much higher power than the small disease samples. Thirty-nine and **2**  Table S1. List of genes included in transcriptional risk scores. Table S2. List of differentially expressed genes. Table S3. BTM across disease sub-types. Table S4. List of disease-by-eQTL interactions. Table S5. Colocalization analysis. (XLSX 329 kb)'),\n",
       " 'PMC8784737-Table1.XLSX-Table_S3-Unnamed_1': ('Comprehensive Characterization of Tumor Purity and Its Clinical Implications in Gastric Cancer',\n",
       "  'Solid tumour tissues are composed of tumour and non-tumour cells, such as stromal cells and immune cells. These non-tumour cells constitute an essential part of the tumour microenvironment (TME), which decrease the tumour purity and play an important role in carcinogenesis, malignancy progression, treatment resistance and prognostic assessment. However, the implications of various purity levels in gastric cancer (GC) remain largely unknown. In the present study, we used an in-silico approach to infer the tumour purity of 2,259 GC samples obtained from our hospital and 12 public datasets based on the transcriptomic data. We systematically evaluated the association of tumour purity with clinical outcomes, biological features, TME characteristics and treatment response in GC. We found that tumour purity might be a patient-specific intrinsic characteristic of GC. Low tumour purity was independently correlated with shorter survival time and faster recurrence and significantly associated with mesenchymal, invasive and metastatic phenotypes. Integrating GC purity into a clinical prognostic nomogram significantly improved predictive validity and reliability. In addition, low tumour purity was strongly associated with immune and stromal cell functions. Fibroblasts, endothelial cells and monocytes were markedly enriched in low-purity tumours, serving as robust indicators of a poor prognosis. Moreover, patients with low GC purity may not benefit more from adjuvant chemotherapy. Our findings highlight that tumour purity confers important clinical, biological, microenvironmental and treatment implications for patients with GC. Therefore, a comprehensive evaluation of tumour purity in individual tumours can provide more insights into the molecular mechanisms of GC, facilitate precise classification and clinical prediction and help to develop more effective individualised treatment strategies.',\n",
       "  'Table_S3-Unnamed_1',\n",
       "  'Click here for additional data file'),\n",
       " 'PMC9616830-42003_2022_4093_MOESM3_ESM.csv--Zhang2019_CD1C-CD141-_dendritic_cell': ('Multi-level cellular and functional annotation of single-cell transcriptomes using scPipeline',\n",
       "  'Single-cell RNA-sequencing (scRNA-seq) offers functional insight into complex biology, allowing for the interrogation of cellular populations and gene expression programs at single-cell resolution. Here, we introduce scPipeline, a single-cell data analysis toolbox that builds on existing methods and offers modular workflows for multi-level cellular annotation and user-friendly analysis reports. Advances to scRNA-seq annotation include: (i) co-dependency index (CDI)-based differential expression, (ii) cluster resolution optimization using a marker-specificity criterion, (iii) marker-based cell-type annotation with Miko scoring, and (iv) gene program discovery using scale-free shared nearest neighbor network (SSN) analysis. Both unsupervised and supervised procedures were validated using a diverse collection of scRNA-seq datasets and illustrative examples of cellular transcriptomic annotation of developmental and immunological scRNA-seq atlases are provided herein. Overall, scPipeline offers a flexible computational framework for in-depth scRNA-seq analysis.',\n",
       "  'Zhang2019_CD1C-CD141-_dendritic_cell',\n",
       "  'efforts, we performed DE analysis on the eight public scRNA-seq datasets presented in Table **1**  Supplementary Data 1'),\n",
       " 'PMC4468152-pone.0130028.s003.xlsx-enhanced-ensg_id': ('Defining the Human Brain Proteome Using Transcriptomics and Antibody-Based Profiling with a Focus on the Cerebral Cortex',\n",
       "  'The mammalian brain is a complex organ composed of many specialized cells, harboring sets of both common, widely distributed, as well as specialized and discretely localized proteins. Here we focus on the human brain, utilizing transcriptomics and public available Human Protein Atlas (HPA) data to analyze brain-enriched (frontal cortex) polyadenylated messenger RNA and long non-coding RNA and generate a genome-wide draft of global and cellular expression patterns of the brain. Based on transcriptomics analysis of altogether 27 tissues, we have estimated that approximately 3% (n=571) of all protein coding genes and 13% (n=87) of the long non-coding genes expressed in the human brain are enriched, having at least five times higher expression levels in brain as compared to any of the other analyzed peripheral tissues. Based on gene ontology analysis and detailed annotation using antibody-based tissue micro array analysis of the corresponding proteins, we found the majority of brain-enriched protein coding genes to be expressed in astrocytes, oligodendrocytes or in neurons with molecular properties linked to synaptic transmission and brain development. Detailed analysis of the transcripts and the genetic landscape of brain-enriched coding and non-coding genes revealed brain-enriched splice variants. Several clusters of neighboring brain-enriched genes were also identified, suggesting regulation of gene expression on the chromatin level. This multi-angle approach uncovered the brain-enriched transcriptome and linked genes to cell types and functions, providing novel insights into the molecular foundation of this highly specialized organ.',\n",
       "  'enhanced-ensg_id',\n",
       "  'expressed in many tissues. Comparing brain to global expression we identified 1,113 enhanced-genes (6%; **S1 Table**)  Brain-enriched and enhanced genes (1113), contains FPKM values of frontal cortex and 26 peripheral organ and tissue types.(XLSX)  Click here for additional data file'),\n",
       " 'PMC7615267-EMS152526-supplement-Supplementary_Tables.xlsx-Table_S33-Unnamed_40': ('Cellular and transcriptional dynamics of human neutrophils at steady state and upon stress',\n",
       "  'Traditionally viewed as poorly plastic, neutrophils are now recognized as functionally diverse; however, the extent and determinants of neutrophil heterogeneity in humans remain unclear. We performed a comprehensive immunophenotypic and transcriptome analysis, at a bulk and single-cell level, of neutrophils from healthy donors and patients undergoing stress myelopoiesis upon exposure to growth factors, transplantation of hematopoietic stem cells (HSC-T), development of pancreatic cancer and viral infection. We uncover an extreme diversity of human neutrophils in vivo, reflecting the rates of cell mobilization, differentiation and exposure to environmental signals. Integrated control of developmental and inducible transcriptional programs linked flexible granulopoietic outputs with elicitation of stimulus-specific functional responses. In this context, we detected an acute interferon (IFN) response in the blood of patients receiving HSC-T that was mirrored by marked upregulation of IFN-stimulated genes in neutrophils but not in monocytes. Systematic characterization of human neutrophil plasticity may uncover clinically relevant biomarkers and support the development of diagnostic and therapeutic tools.',\n",
       "  'Table_S33-Unnamed_40',\n",
       "  'of PB or BM samples from control subjects and age-matched individuals undergoing G-CSF treatment (**Supplementary Tables 1-6**)'),\n",
       " 'PMC9635625-Table_5.xlsx-cluster4vs1_2_3_0_5_8_12-Supplementary_Data_5_List_of_the_marker_genes_per_granulosa_cell_GC_cluster_FDR_0_05_by_comparing_each_GC_cluster_to_all_other_GC_clusters': ('Single-cell RNA-seq analysis and cell-cluster deconvolution of the human preovulatory follicular fluid cells provide insights into the pathophysiology of ovarian hyporesponse',\n",
       "  'Reduction in responsiveness to gonadotropins or hyporesponsiveness may lead to the failure of in vitro fertilization (IVF), due to a low number of retrieved oocytes. The ovarian sensitivity index (OSI) is used to reflect the ovarian responsiveness to gonadotropin stimulation before IVF. Although introduced to clinical practice already years ago, its usefulness to predict clinical outcomes requires further research. Nevertheless, pathophysiological mechanisms of ovarian hyporesponse, along with advanced maternal age and in younger women, have not been fully elucidated. Follicles consist of multiple cell types responsible for a repertoire of biological processes including responding to pituitary gonadotropins necessary for follicle growth and oocyte maturation as well as ovulation. Encouraging evidence suggests that hyporesponse could be influenced by many contributing factors, therefore, investigating the variability of ovarian follicular cell types and their gene expression in hyporesponders is highly informative for increasing their prognosis for IVF live birth. Due to advancements in single-cell analysis technologies, the role of somatic cell populations in the development of infertility of ovarian etiology can be clarified. Here, somatic cells were collected from the fluid of preovulatory ovarian follicles of patients undergoing IVF, and RNA-seq was performed to study the associations between OSI and gene expression. We identified 12 molecular pathways differentially regulated between hypo- and normoresponder patient groups (FDR<0.05) from which extracellular matrix organization, post-translational protein phosphorylation, and regulation of Insulin-like Growth Factor (IGF) transport and uptake by IGF Binding Proteins were regulated age-independently. We then generated single-cell RNA-seq data from matching follicles revealing 14 distinct cell clusters. Using cell cluster-specific deconvolution from the bulk RNA-seq data of 18 IVF patients we integrated the datasets as a novel approach and discovered that the abundance of three cell clusters significantly varied between hypo- and normoresponder groups suggesting their role in contributing to the deviations from normal ovarian response to gonadotropin stimulation. Our work uncovers new information regarding the differences in the follicular gene expression between hypo- and normoresponders. In addition, the current study fills the gap in understanding the inter-patient variability of cell types in human preovulatory follicles, as revealed by single-cell analysis of follicular fluid cells.',\n",
       "  'cluster4vs1_2_3_0_5_8_12-Supplementary_Data_5_List_of_the_marker_genes_per_granulosa_cell_GC_cluster_FDR_0_05_by_comparing_each_GC_cluster_to_all_other_GC_clusters',\n",
       "  'GC cluster was compared to the pooled dataset of the remaining GC clusters ( **Supplementary Table 5**)  Click here for additional data file'),\n",
       " 'PMC5390115-mmc4.xlsx-DE_Lists-Unnamed_25': ('A Genome-wide Analysis of Human Pluripotent Stem Cell-Derived Endothelial Cells in 2D or 3D Culture',\n",
       "  'A defined protocol for efficiently deriving endothelial cells from human pluripotent stem cells was established and vascular morphogenesis was used as a model system to understand how synthetic hydrogels influence global biological function compared with common 2D and 3D culture platforms. RNA sequencing demonstrated that gene expression profiles were similar for endothelial cells and pericytes cocultured in polyethylene glycol (PEG) hydrogels or Matrigel, while monoculture comparisons identified distinct vascular signatures for each cell type. Endothelial cells cultured on tissue-culture polystyrene adopted a proliferative phenotype compared with cells cultured on or encapsulated in PEG hydrogels. The proliferative phenotype correlated to increased FAK-ERK activity, and knockdown or inhibition of ERK signaling reduced proliferation and expression for cell-cycle genes while increasing expression for \"3D-like\" vasculature development genes. Our results provide insight into the influence of 2D and 3D culture formats on global biological processes that regulate cell function.',\n",
       "  'DE_Lists-Unnamed_25',\n",
       "  '(on the same day) than for comparisons between PEG hydrogels with different crosslinking densities (**Table S3**)  Table S3. Differential Gene Expression Analysis for H1-EC-PC Cocultures, Related to Figure 2'),\n",
       " 'PMC8297391-Table_3.XLSX-Immune_Diff-gene': ('P2RY14 Is a Potential Biomarker of Tumor Microenvironment Immunomodulation and Favorable Prognosis in Patients With Head and Neck Cancer',\n",
       "  'The tumor microenvironment (TME) has a crucial role in tumor development, progression, and treatment response. Yet, the exact interaction between cancer biology and the TME is not fully understood. The following study analyzed the correlation between immune/stromal/estimate scores and survival prognosis in head and neck squamous cell carcinoma (HNSC) using a bioinformatic method. As a result, a predictive biomarker, UDP-glucose-specific G(i) protein-coupled P2Y receptor (P2RY14), was discovered. The potential role of P2RY14-driven signaling pathways in the immune-remodeling of TME was then investigated. Briefly, low immune scores were associated with unfavorable prognosis and clinical-stage, larger tumor size, and the down-regulation of P2RY14 in HNSC patients. In addition, the survival analysis showed that HNSC patients with high expression had longer survival than patients with low expression from both TCGA databases and our own patients. We further discovered that P2RY14 is involved in the immune activity in the TME of HNSC; a downregulation of P2RY14 resulted in being an indicator for the conversion of TME status (from immune-dominant to metabolic-dominant status). The intersection analysis of genes co-expressed with P2RY14 indicated that the T-cell receptor signaling pathway and PD-L1 expression and PD-1 checkpoint pathway were candidate signaling pathways driven by the P2RY14 gene in HNSC. Further investigation of immune-associated signaling pathways regulated by P2RY14 may help HNSC patients gain higher immunotherapy benefits.',\n",
       "  'Immune_Diff-gene',\n",
       "  'Identical up and down-regulate differential genes from stromal and immune groups  Click here for additional data file'),\n",
       " 'PMC10894295-41598_2024_55189_MOESM4_ESM.xlsx-Sheet1-EVT_Up': ('Dynamic and distinct histone modifications facilitate human trophoblast lineage differentiation',\n",
       "  'The placenta serves as an essential organ for fetal growth throughout pregnancy. Histone modification is a crucial regulatory mechanism involved in numerous biological processes and development. Nevertheless, there remains a significant gap in our understanding regarding the epigenetic regulations that influence trophoblast lineage differentiation, a fundamental aspect of placental development. Here, through comprehensive mapping of H3K4me3, H3K27me3, H3K9me3, and H3K27ac loci during the differentiation of trophoblast stem cells (TSCs) into syncytiotrophoblasts (STs) and extravillous trophoblasts (EVTs), we reveal dynamic reconfiguration in H3K4me3 and H3K27ac patterns that establish an epigenetic landscape conducive to proper trophoblast lineage differentiation. We observe that broad H3K4me3 domains are associated with trophoblast lineage-specific gene expression. Unlike embryonic stem cells, TSCs lack robust bivalent domains. Notably, the repression of ST- and EVT-active genes in TSCs is primarily attributed to the weak H3K4me3 signal rather than bivalent domains. We also unveil the inactivation of TSC enhancers precedes the activation of ST enhancers during ST formation. Our results provide a comprehensive global map of diverse histone modifications, elucidating the dynamic histone modifications during trophoblast lineage differentiation.',\n",
       "  'Sheet1-EVT_Up',\n",
       "  'None here'),\n",
       " 'PMC11251028-41467_2024_50224_MOESM4_ESM.xlsx-Data-Unnamed_16': ('Efficient and reproducible generation of human iPSC-derived cardiomyocytes and cardiac organoids in stirred suspension systems',\n",
       "  'Human iPSC-derived cardiomyocytes (hiPSC-CMs) have proven invaluable for cardiac disease modeling and regeneration. Challenges with quality, inter-batch consistency, cryopreservation and scale remain, reducing experimental reproducibility and clinical translation. Here, we report a robust stirred suspension cardiac differentiation protocol, and we perform extensive morphological and functional characterization of the resulting bioreactor-differentiated iPSC-CMs (bCMs). Across multiple different iPSC lines, the protocol produces 1.2E6/mL bCMs with ~94% purity. bCMs have high viability after cryo-recovery (>90%) and predominantly ventricular identity. Compared to standard monolayer-differentiated CMs, bCMs are more reproducible across batches and have more mature functional properties. The protocol also works with magnetically stirred spinner flasks, which are more economical and scalable than bioreactors. Minor protocol modifications generate cardiac organoids fully in suspension culture. These reproducible, scalable, and resource-efficient approaches to generate iPSC-CMs and organoids will expand their applications, and our benchmark data will enable comparison to cells produced by other cardiac differentiation protocols.',\n",
       "  'Data-Unnamed_16',\n",
       "  'None here'),\n",
       " 'PMC9039753-mmc5.xlsx-Microglia-enriched_genes_in_MIG-Unnamed_2': ('Homozygous ALS-linked FUS P525L mutations cell- autonomously perturb transcriptome profile and chemoreceptor signaling in human iPSC microglia',\n",
       "  'Amyotrophic lateral sclerosis is a fatal disease pathologically typified by motor and cortical neurodegeneration as well as microgliosis. The FUS P525L mutation is highly penetrant and causes ALS cases with earlier disease onset and more aggressive progression. To date, how P525L mutations may affect microglia during ALS pathogenesis had not been explored. In this study, we engineered isogenic control and P525L mutant FUS in independent human iPSC lines and differentiated them into microglia-like cells. We report that the P525L mutation causes FUS protein to mislocalize from the nucleus to cytoplasm. Homozygous P525L mutations perturb the transcriptome profile in which many differentially expressed genes are associated with microglial functions. Specifically, the dysregulation of several chemoreceptor genes leads to altered chemoreceptor-activated calcium signaling. However, other microglial functions such as phagocytosis and cytokine release are not significantly affected. Our study underscores the cell-autonomous effects of the ALS-linked FUS P525L mutation in a human microglia model.',\n",
       "  'Microglia-enriched_genes_in_MIG-Unnamed_2',\n",
       "  'in decreasing order of expression level, followed by non-enriched genes in increasing order. See **Table S4**  Table S4. Actively expressed microglia-enriched genes in primary human microglia and MIGs'),\n",
       " 'PMC10835465-mmc6.xlsx-Cluster4-Unnamed_0': ('Association of HBsAg levels with differential gene expression in NK, CD8 T, and memory B cells in treated patients with chronic HBV',\n",
       "  'Background & aims: HBsAg secretion may impact immune responses to chronic HBV infection. Thus, therapeutic approaches to suppress HBsAg production are being investigated. Our study aims to examine the immunomodulatory effects of high and low levels of circulating HBsAg and thereby improve our understanding of anti-HBV immunity. Methods: An optimized 10x Genomics single-cell RNA sequencing workflow was applied to blood samples and liver fine-needle aspirates from 18 patients undergoing tenofovir/entecavir (NUC) treatment for chronic HBV infection. They were categorized based on their HBsAg levels: high (920-12,447 IU/ml) or low (1-100 IU/ml). Cluster frequencies, differential gene expression, and phenotypes were analyzed. Results: In the blood of HBV-infected patients on NUC, the proportion of KLRC2 + \"adaptive\" natural killer (NK) cells was significantly lower in the HBsAg-high group and, remarkably, both KLRC2 + NK and KLRG1+ CD8 T cells display enrichment of lymphocyte activation-associated gene sets in the HBsAg-low group. High levels of HBsAg were associated with mild immune activation in the liver. However, no suppression of liver-resident CXCR6+ NCAM1+ NK or CXCR6+ CD69+ CD8 T cells was detected, while memory B cells showed signs of activation in both the blood and liver. Conclusions: Among NUC-treated patients, we observed a minimal impact of HBsAg on leukocyte populations in the blood and liver. However, for the first time, we found that HBsAg has distinct effects, restricted to NK-, CD8 T-, and memory B-cell subsets, in the blood and liver. Our findings are highly relevant for current clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Impact and implications: This study provides unique insight into the impact of HBsAg on gene expression levels of immune cell subsets in the blood and liver, particularly in the context of NUC-treated chronic HBV infection. It holds significant relevance for current and future clinical studies evaluating treatment strategies aimed at suppressing HBsAg production and reinvigorating immunity to HBV. Our findings raise questions about the effectiveness of such treatment strategies and challenge the previously hypothesized immunomodulatory effects of HBsAg on immune responses against HBV.',\n",
       "  'Cluster4-Unnamed_0',\n",
       "  'None here'),\n",
       " 'PMC9362915-pone.0272017.s004.xlsx-Day0_0_0_Top_A-EPSTI1': ('Adrenergic signaling controls early transcriptional programs during CD8+ T cell responses to viral infection',\n",
       "  'Norepinephrine is a key sympathetic neurotransmitter, which acts to suppress CD8 + T cell cytokine secretion and lytic activity by signaling through the β2-adrenergic receptor (ADRB2). Although ADRB2 signaling is considered generally immunosuppressive, its role in regulating the differentiation of effector T cells in response to infection has not been investigated. Using an adoptive transfer approach, we compared the expansion and differentiation of wild type (WT) to Adrb2-/- CD8 + T cells throughout the primary response to vesicular stomatitis virus (VSV) infection in vivo. We measured the dynamic changes in transcriptome profiles of antigen-specific CD8 + T cells as they responded to VSV. Within the first 7 days of infection, WT cells out-paced the expansion of Adrb2-/- cells, which correlated with reduced expression of IL-2 and the IL-2Rα in the absence of ADRB2. RNASeq analysis identified over 300 differentially expressed genes that were both temporally regulated following infection and selectively regulated in WT vs Adrb2-/- cells. These genes contributed to major transcriptional pathways including cytokine receptor activation, signaling in cancer, immune deficiency, and neurotransmitter pathways. By parsing genes within groups that were either induced or repressed over time in response to infection, we identified three main branches of genes that were differentially regulated by the ADRB2. These gene sets were predicted to be regulated by specific transcription factors involved in effector T cell development, such as Tbx21 and Eomes. Collectively, these data demonstrate a significant role for ADRB2 signaling in regulating key transcriptional pathways during CD8 + T cells responses to infection that may dramatically impact their functional capabilities and downstream memory cell development.',\n",
       "  'Day0_0_0_Top_A-EPSTI1',\n",
       "  'them. A full list of genes for each path is provided in S3 and **S4**  Tables of genes in each path of the Adrb2-/- SMARTS model.(XLSX)  Click here for additional data file')}"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "def pmc_again(pmcids):\n",
    "    pmc_dict = {}\n",
    "    for pm in pmcids:\n",
    "        pmc_dict[pm] = (pmc_sum[pmcids[pm][1]][\"title\"], pmc_sum[pmcids[pm][1]][\"abstract\"], extract_after_extension(pm), pmcids[pm][0])\n",
    "\n",
    "    return pmc_dict\n",
    "\n",
    "\n",
    "fin_pmc = pmc_again(df_rummagene_map)\n",
    "fin_pmc\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'GSE159694-3-vs-0-human up': (['Retinoic Acid Promotes Endothelial Cell Cycle Early G1 State to Enable Human Hemogenic Endothelial Cell Specification'],\n",
       "  'Endothelial cells derived from hESCs were separated into 4 different cell cycle phases via FACS and sequenced (i.e., early G1, late G1, G1/S, S/G2/M)',\n",
       "  'endothelial cells late g1 phase rep hesc derived untreated',\n",
       "  'endothelial cells g1/ phase rep hesc derived untreated',\n",
       "  'human'),\n",
       " 'GSE184536-7-vs-11-human dn': (['The NF-κB transcriptional footprint is essential for SARS-CoV-2 replication'],\n",
       "  'SARS-CoV-2, the etiological agent of COVID-19, is characterized by a delay in Type I interferon (IFN-I)-mediated antiviral defenses alongside robust cytokine production. Here we investigate the underlying molecular basis for this imbalance and implicate virus-mediated activation of NF-κB in the absence of other canonical IFN-I-related transcription factors. Epigenetic and single cell transcriptomic analyses show a selective NF-κB signature that was most prominent in infected cells. Disruption of NF-κB signaling through the silencing of the NF-κB transcription factors p65 or p50 resulted in loss of virus replication that was rescued upon reconstitution. These findings could be further corroborated with the use of NF-κB inhibitors, which reduced SARS-CoV-2 replication in vitro. These data suggest that the robust cytokine production in response to SARS-CoV-2, despite a diminished IFN-I response, is the product of a dependency on NF-κB for viral replication.',\n",
       "  'mrnaseq mock 12h lung adenocarcinoma virus 12 h a549 ace2 cells',\n",
       "  'mrnaseq mock 4h lung adenocarcinoma virus 4 h a549 ace2 cells',\n",
       "  'human'),\n",
       " 'GSE159694-3-vs-2-human up': (['Retinoic Acid Promotes Endothelial Cell Cycle Early G1 State to Enable Human Hemogenic Endothelial Cell Specification'],\n",
       "  'Endothelial cells derived from hESCs were separated into 4 different cell cycle phases via FACS and sequenced (i.e., early G1, late G1, G1/S, S/G2/M)',\n",
       "  'endothelial cells late g1 phase rep hesc derived untreated',\n",
       "  'endothelial cells /g2/ phase rep hesc derived untreated',\n",
       "  'human'),\n",
       " 'GSE141756-2-vs-1-human up': (['Gene expression profiles of active and restricted R/G-HIV+ primary human fetal astrocytes'],\n",
       "  'The gene expression patterns of astrocyte harboring active vs restricted LTRs were similar, and that the active population demonstrated more widespread and robust changes.',\n",
       "  'rgh wt plus fetal brain passage() 6 sex gestational age (days) condition infection primary astrocytes donor',\n",
       "  'uninfected fetal brain passage() sex gestational age (days) condition infection primary astrocytes donor',\n",
       "  'human'),\n",
       " 'GSE139542-0-vs-1-mouse up': (['RNA-seq of lipid droplet-low and lipid droplet-rich microglia isolated from the hippocampi of aged mice'],\n",
       "  'Sequencing data related to the manuscript entitled, \"Lipid droplet accumulating microglia represent a dysfunctional and pro-inflammatory state in the aging brain\".',\n",
       "  'lo strain c57bl/6 age 18 month old sex male sample type bodipy low microglia (lipid droplet low) hippocampus',\n",
       "  'hi strain c57bl/6 age 18 month old sex male sample type bodipy high microglia (lipid droplet rich) hippocampus',\n",
       "  'mouse'),\n",
       " 'GSE215771-4-vs-2-human up': (['IFN-γ induced transcriptional profile of A549 cells in absence of key transcriptional regulators IRF1 and NF2'],\n",
       "  'Type-II interferon or IFN-gamma is a critical cytokine in innate immunity that control multitude of intracellular infections. IFN-gamma binds to interferon gamma receptors to phosphorylate and activate STAT1 mediated transcription of interferon stimulated genes (ISGs). We executed a genome-wide CRISPR-Cas9 screen against ~20,000 human genes to identify IRF-1 and NF2 as significant regulators of Toxoplasma infection in A549 lung epithelial cells.  IRF-1 is a key transcription factor immediately downstream of STAT1 induced transcription that amplifies ISG expression in different cell types. NF2 or MERLIN is a tumor suppressor that is known to regulate expression of genes involved with cellular growth and metabolism. Transcriptomic signature of A549 cells lacking IRF-1 and NF2 were measured upon activation with IFN-gamma to find set of genes that may be regulating Toxoplasma infection. A total of 1,046,709,676 reads from 3 independent biological replicates were mapped to human hg38 reference genome. Expression of a ISGs that failed to induced in cells lacking IRF1 were found to be dysregulated in absence of NF2 as well.',\n",
       "  'nf2ko control cell line a549 epithelial (lung) replicate',\n",
       "  'nf2ko ifng cell line a549 epithelial (lung) ifn î³ replicate',\n",
       "  'human'),\n",
       " 'GSE111437-0-vs-2-human dn': (['X-Ray induced DNA-Hydroxymethylation changes'],\n",
       "  'In this study we addressed the question whether changes of DNA hydroxymethylation after X-ray irradiation occur in a random fashion or if they show preferences for specific regions or features. We did not only focus on gene loci and promotor sights but also on gene body hydroxymethylation alterations and associated changes in gene expression. A more profound insight into hydroxymethylation characteristics triggered by radiation could provide clues to mechanisms and consequences of cellular response to irradiation including the immediate consequence of DNA strand breaks associated with active DNA demethylation as well as the long term risk of tumorigenesis or other associated diseases.',\n",
       "  'imr90 0gy 24h r1 cell line (h) dose x ray (gy) 0 human caucasian fetal lung fibroblast',\n",
       "  'imr90 2gy 24h r1 cell line (h) dose x ray (gy) 2 human caucasian fetal lung fibroblast',\n",
       "  'human'),\n",
       " 'GSE234585,GSE234587-9-vs-5-human dn': (['Malaria-specific Type 1 Regulatory T cells are more abundant in first pregnancies and associated with placental malaria [Whole Blood RNA-seq]'],\n",
       "  'To investigate gravidity dependent immunity to malaria we sequenced whole blood RNA from women with and without parasitemia and both pregnant and non-pregnant controls.',\n",
       "  'positive pregnant whole blood parasitemia pregnancy status',\n",
       "  'negative nonpregnant whole blood parasitemia pregnancy status',\n",
       "  'human'),\n",
       " 'GSE213488-8-vs-10-human dn': (['Transcriptome analyses of the cortex and white matter of focal cortical dysplasia type II human samples: novel insights into disease mechanisms and contributions to tissue characterization'],\n",
       "  'Focal cortical dysplasia (FCD) is a common cause of pharmacoresistant epilepsy. According to the 2011 International League Against Epilepsy classification, FCD type II is characterized by dysmorphic neurons (IIa and IIb) and may be associated with balloon cells (IIb). We present a multicentric study to evaluate the transcriptomes of the gay and white matters of surgical FCD type II specimens. Our aim was to contribute to pathophysiology and tissue characterization. We investigated FCD II (a and b) and control samples by performing RNA-sequencing followed by immunohistochemical validation by means of digital analyses. We found 342 and 399 transcripts differentially expressed in the gray matter of IIa and IIb lesions compared to controls, respectively. The top enriched cellular pathway in IIa and IIb gray matter was cholesterol biosynthesis, their genes HMGCS1, HMGCR and SQLE being upregulated in both type II groups. We also found 12 differentially expressed genes when comparing transcriptomes of IIa and IIb lesions. Only 1 transcript (MTRNR2L12) was significantly upregulated in FCD IIa. The white matter in IIa and IIb lesions showed 2 and 24 transcripts differentially expressed respectively compared to controls. No enriched cellular pathways were detected. GPNMB, not previously described in FCD samples, was upregulated in IIb compared to IIa and control groups. Upregulations of cholesterol biosynthesis enzymes and GPNMB genes in FCD groups were immunohistochemically validated. Such enzymes were mainly detected in both dysmorphic and normal neurons, whereas GPNMB was observed only in balloon cells. Overall, our study contributed to the identification of cortical enrichment of cholesterol biosynthesis in FCD type II, which may correspond to neuroprotective response to seizures. Moreover, specific analyses in either the gay or the white matter revealed upregulations of MTRNR2L12 and GPNMB, which might be potential neuropathological biomarkers of a cortex chronically exposed to seizures and of balloon cells, respectively.',\n",
       "  'fcd iib sample (white matter) frontal lobe disease state',\n",
       "  'fcd iib sample (gray matter) frontal lobe disease state',\n",
       "  'human'),\n",
       " 'GSE224063-1-vs-2-human dn': (['COVD-19 infection during pregnancy induces a hematopoietic stem cell signature in cord blood fetal cells'],\n",
       "  'Pregnant women and their fetuses are particularly susceptible to respiratory pathogens. How they respond to SARS-CoV-2 infection is still under investigation. Here, we studied the transcriptome and phenotype of umbilical cord blood cells in pregnant women infected or not with SARS-CoV-2. We found that symptomatic maternal COVID-19 was associated with a transcriptional erythroid cell signature as compared with asymptomatic and uninfected mothers. We observed an expansion of fetal hematopoietic progenitors skewed towards erythroid differentiation and displaying increased clonogenicity. We found no difference in inflammatory cytokines in the cord blood upon SARS-CoV-2 infection. Interestingly, we showed an activation of hypoxia pathway in cord blood cells from symptomatic COVID-19 mothers, suggesting that maternal hypoxia may be triggering this fetal stress erythropoiesis. Overall, these results show a fetal hematopoietic response to symptomatic COVID-19 in pregnant mothers in the absence of vertically transmitted SARS-CoV-2 infection, which is likely to be a mechanism of fetal adaptation to maternal infection and reduced oxygen supply.',\n",
       "  'cord blood mononuclear cells (cbpmcs) symptomatic covid+ human disease state',\n",
       "  'cord blood mononuclear cells (cbpmcs) asymptomatic covid+ human disease state',\n",
       "  'human'),\n",
       " 'GSE213488-5-vs-8-human up': (['Transcriptome analyses of the cortex and white matter of focal cortical dysplasia type II human samples: novel insights into disease mechanisms and contributions to tissue characterization'],\n",
       "  'Focal cortical dysplasia (FCD) is a common cause of pharmacoresistant epilepsy. According to the 2011 International League Against Epilepsy classification, FCD type II is characterized by dysmorphic neurons (IIa and IIb) and may be associated with balloon cells (IIb). We present a multicentric study to evaluate the transcriptomes of the gay and white matters of surgical FCD type II specimens. Our aim was to contribute to pathophysiology and tissue characterization. We investigated FCD II (a and b) and control samples by performing RNA-sequencing followed by immunohistochemical validation by means of digital analyses. We found 342 and 399 transcripts differentially expressed in the gray matter of IIa and IIb lesions compared to controls, respectively. The top enriched cellular pathway in IIa and IIb gray matter was cholesterol biosynthesis, their genes HMGCS1, HMGCR and SQLE being upregulated in both type II groups. We also found 12 differentially expressed genes when comparing transcriptomes of IIa and IIb lesions. Only 1 transcript (MTRNR2L12) was significantly upregulated in FCD IIa. The white matter in IIa and IIb lesions showed 2 and 24 transcripts differentially expressed respectively compared to controls. No enriched cellular pathways were detected. GPNMB, not previously described in FCD samples, was upregulated in IIb compared to IIa and control groups. Upregulations of cholesterol biosynthesis enzymes and GPNMB genes in FCD groups were immunohistochemically validated. Such enzymes were mainly detected in both dysmorphic and normal neurons, whereas GPNMB was observed only in balloon cells. Overall, our study contributed to the identification of cortical enrichment of cholesterol biosynthesis in FCD type II, which may correspond to neuroprotective response to seizures. Moreover, specific analyses in either the gay or the white matter revealed upregulations of MTRNR2L12 and GPNMB, which might be potential neuropathological biomarkers of a cortex chronically exposed to seizures and of balloon cells, respectively.',\n",
       "  'control sample (gray matter) frontal lobe disease state (autopsy)',\n",
       "  'fcd iib sample (white matter) frontal lobe disease state',\n",
       "  'human'),\n",
       " 'GSE86104,GSE86105-0-vs-1-human dn': (['RNA-Seq analysis RPE1 cells following exposure to Nutlin-3 to identify target genes of p53 [tpo12]'],\n",
       "  'Tetraploidization, or genome doubling, is a prominent event in tumorigenesis, primarily because cell division in polyploid cells is error-prone and produces aneuploid cells.  This study investigates changes in gene expression evoked in acute and adapted tetraploid cells and their impact on cell-cycle progression.  Acute polyploidy was generated by knockdown of essential regulator of cytokinesis Anillin, which resulted in cytokinesis failure and formation of binucleate cells, or by chemical inhibition of Aurora kinases, causing abnormal mitotic exit with formation of single cells with aberrant nuclear morphology.  Transcriptome analysis of these acute tetraploid cells revealed common signatures of activation of the tumor-suppressor protein p53.  Suppression of proliferation in these cells was dependent on p53 and its transcriptional target - Cdk inhibitor p21.  Rare proliferating tetraploid cells can emerge from acute polyploid populations.  Gene expression analysis of single-cell derived, adapted tetraploid clones showed upregulation of several p53 target genes and cyclin D2, the activator of Cdk4/6/2.  Overexpression of cyclin D2 in diploid cells strongly potentiated the ability to proliferate with increased DNA content despite the presence of functional p53.  These results point out that p53-mediated suppression of proliferation of polyploid cells can be averted by increased levels of oncogenes such as Cyclin D2, elucidating a possible route for tetraploidy-mediated genomic instability in carcinogenesis.',\n",
       "  'rpe1 dmso cell line replicate drug',\n",
       "  'rpe1 nutlin cell line replicate drug 3',\n",
       "  'human'),\n",
       " 'GSE155447,GSE155448-1-vs-0-human dn': (['RNA-seq and Ribo-seq analysis of the effects of METTL3/14 depletion on gene expression during the interferon response in Huh7 cells'],\n",
       "  \"The RNA modification N6-methyladenosine (m6A) regulates gene expression through various transcript-specific effects. The overall goal of these experiments was to determine the effects of the m6A methyltransferase complex proteins METTL3 and METTL14 on the expression of interferon stimulated genes. First, we depleted METTL3 and METTL14 in Huh7 cells using siRNAs and treated with Mock or IFN-β to analyze the regulatory effects of METTL3/14 on interferon-stimulated genes' transcript abundance. We then used Ribo-seq following METTL3/14 siRNA depletion and IFN treatment to quantify its effect on the translation of ISGs and other genes. Overall, we found that METTL3/14 exerts transcript-specific regulatory effects on a subset of interferon-stimulated genes. Overall, METTL3/14 appeared to enhance the translation of a subset of interferon-stimulated genes. We characterized the molecular effects of METTL3/14 and m6A on certain ISGs and determined that m6A augments the antiviral effects of the interferon response.\",\n",
       "  'ifn sictrl rna interferon sirna huh7 cells',\n",
       "  'mock simettl3/14 sirna huh7 cells',\n",
       "  'human'),\n",
       " 'GSE98274-0-vs-2-human dn': (['RNA-Seq of SHEP TET21N cells upon Doxorubicin treatment'],\n",
       "  'MYCN-high and MYCN-low neuroblastoma cells differ in their responses to Doxorubicin treatment. To explain this difference we compared the global trancriptomes of MYCN-high and MYCN-low cells before, during and after treatment.',\n",
       "  '72 plus time 72h mycn level mycnhigh doxorubicin washout cell line',\n",
       "  '72h doxo time mycn level doxorubicin cell line',\n",
       "  'human'),\n",
       " 'GSE214657-3-vs-5-human up': (['Generation and Integrated Analysis of Advanced Patient- derived Orthoxenograft Models (PDOX) for the Rational Assessment of Targeted Therapies in Endometrial Cancer'],\n",
       "  'Clinical management of endometrial cancer (EC) is handicapped by the limited availability of second line treatments and bona fide molecular biomarkers to predict recurrence. These limitations have hampered the treatment of these patients, whose survival rates have not improved over the last four decades. The advent of coordinated studies such as the TCGA_UCEC have partially solved this issue, but the lack of proper experimental systems still represents a bottleneck that precludes translational studies from successful clinical testing in EC patients. Within this context, we present the first study reporting the generation of a collection of endometrioid EC patient-derived orthoxenograft (PDOX) mouse models that we believe can overcome these experimental constraints and pave the way towards state-of-the-art precision medicine in EC. We have characterized our collection of primary tumors and derived PDOXs through an integrative approach based on transcriptomics, mutational profiles and morphological analysis; and demonstrate that EC tumors engrafted in the mouse uterus retain the main molecular and morphological features from analogous tumor donors. Finally, we have harnessed the molecular properties of these tumors to assess the therapeutic potential of trastuzumab, a HER2 inhibitor with growing interest in EC, using patient-derived organotypic multicellular tumor spheroids and in vivo experiments.',\n",
       "  'derived orthoxenograft (pdox) xenograft cell line tumor',\n",
       "  'primary tumor endometrioid endometrial cancer cell line',\n",
       "  'human'),\n",
       " 'GSE151165-0-vs-2-human dn': (['Neutrophils expressing lysyl oxidase-like 4 protein are present in colorectal cancer liver metastases resistant to anti-angiogenic therapy'],\n",
       "  'Colorectal cancer liver metastases (CRCLM) that present with a replacement histopathological growth pattern (HGP) are\\xa0resistant to neoadjuvant anti-angiogenic therapy. Surrogate biomarkers are not available to preoperatively identifypatients with these tumors. Here we identify differentially expressed genes between CRCLM with a replacement HGP\\xa0and those with a desmoplastic HGP using RNA sequencing.We demonstrate that LOXL4 is transcriptionally upregulated in replacement HGP CRCLM compared with desmoplastic HGP CRCLM and the adjacent normal liver. Interestingly, lysyl\\xa0oxidase-like 4 (LOXL4) protein was expressed by neutrophils present in the tumor microenvironment in replacement HGP CRCLM. We further demonstrate that LOXL4 expression is higher in circulating neutrophils of cancer patients compared\\xa0with healthy control patients and its expression can be induced by stimulation with lipopolysaccharide and TNF-alpha. Our study is the first to show the expression of LOXL4 in neutrophils and reveals the potential for LOXL4-expressing neutrophils\\xa0to support the replacement HGP phenotype and to serve as a surrogate biomarker for this subtype of CRCLM.',\n",
       "  'kr dn age sex normal disease colorectal cancer liver metastases (crclm) sample type naive rna',\n",
       "  'kr rt age sex tumor disease colorectal cancer liver metastases (crclm) sample type naive rna',\n",
       "  'human'),\n",
       " 'GSE248853,GSE248854-0-vs-3-human up': (['RFX6 maintains gene expression and function of adult human islet ⍺ cells [RNAseq]'],\n",
       "  'Transcriptomic profiling by high-throughput sequencing of primary human alpha and beta cells following RFX6 loss of function',\n",
       "  'alpha rfx6kd cadaveric pancreatic cell line primary islet cells rfx6 shrna',\n",
       "  'beta rfx6kd cadaveric pancreatic cell line primary islet cells rfx6 shrna',\n",
       "  'human'),\n",
       " 'GSE215915-0-vs-2-human up': (['Multiomic chromatin accessibility and gene expression in peripheral blood predicts future decompensation in hospitalized adults with COVID-19'],\n",
       "  'To elucidate host response elements that define impending decompensation during SARS-CoV-2 infection, we enrolled subjects hospitalized with COVID-19 who were matched for disease severity and comorbidities at the time of admission and initial sampling. We then performed combined single-cell RNA sequencing (scRNA-seq) and single cell Assay for Transposase-Accessible Chromatin using sequencing (scATAC-seq) on peripheral blood mononuclear cells (PBMCs) from the time of admission and compared subjects who improved from their initial low-level oxygen requirement (no change from initial WHO Ordinal category 4/5) to those who later clinically decompensated and required invasive mechanical ventilation or died (WHO Ordinal score 7/8). Chromatin accessibility and transcriptomic immune profiles were markedly altered at admission in patients who will go on to develop critical illness. The greatest immunologic signals were seen in CD4+ T cells, inflammatory T cells, dendritic cells, and NK cell subsets, where an aggregate multiomic signature score calculated at admission offered strong prediction of future clinical deterioration (area under the receiver operating characteristic curve (auROC) 0.91). Epigenetic and transcriptional changes in PBMCs reveal unique, early, and conserved aspects of the immune response before typical clinical signals of decompensation are apparent and thus provide novel biomarkers that can predict future disease severity.',\n",
       "  'atac peripheral blood mononuclear cells time age sex male cohort worsening',\n",
       "  'bee0d0 peripheral blood mononuclear cells time age 54 sex female cohort worsening',\n",
       "  'human'),\n",
       " 'GSE117614-1-vs-8-human up': (['RNA-seq analysis of canonical and adaptive human NK cell and CD8+ T cell subsets from HCMV seropositive donors'],\n",
       "  'We report our results of RNA-seq analysis on freshly isolated, sorted subsets of cytotoxic lymphocytes',\n",
       "  'tag facs sorted late mature adaptive nk cells cd3 cd56dimcd57+nkg2c+ peripheral blood mononuclear',\n",
       "  'tag facs sorted early mature canonical nk cells cd3 cd56dimcd57 nkg2c peripheral blood mononuclear',\n",
       "  'human'),\n",
       " 'GSE153970-0-vs-1-human up': (['Primary Human Airway Epithelial Cultures infected with SARS-CoV-2'],\n",
       "  'We performed RNAseq analysis on primary human airway epithelial cultures either mock infected (PBS) or infected with SARS-CoV-2. Transcriptional profiling studies found that infected pHAE cells had a molecular signature dominated by pro-inflammatory cytokines and chemokine induction, including IL-6, TNFα, CXCL8, and identified NF-κB and ATF4 as key drivers of this pro-inflammatory cytokine response. Surprisingly, we observed a lack of a type I or III interferon (IFN) response to SARS-CoV-2 infection',\n",
       "  'hae mock primary human airway epithelial (phae) culture agent infection 48 hours',\n",
       "  'hae sarscov2 primary human airway epithelial (phae) culture agent sars cov 2 48 hours',\n",
       "  'human'),\n",
       " 'GSE220806-2-vs-0-human dn': (['Characteristic impairment of progesterone response in the cultured cervical fibroblast obtained from patients with refractory cervical insufficiency'],\n",
       "  'We compared the gene expression in response to progesterone (1μM) in human cervical fibroblasts.',\n",
       "  'ci notx condition untreated cells derived cervical insufficiency human uterine fibroblast passage 4 8',\n",
       "  'control notx condition untreated cells derived unripend cervix human uterine cervical fibroblast passage 4 8',\n",
       "  'human'),\n",
       " 'GSE178640-1-vs-3-human up': (['Transcriptomics changes after IFN-alpha2 treatment following dBRD9-A or DMSO pretreatment in A549 lung epithelial cells'],\n",
       "  'These experiments aim at defining first the entirety of Interferon-stimulated gene expression  in A549 in response to treatment with 100 IU/mL IFN-alpha2. The second second aim was to characterize changes to the transcriptome caused by the prior degradation of bromodomain-containing protein9 (BRD9) by the use of a specific BRD9-degrader (dBRD9-A) both in the uninduced and IFN-alpha2-induced conditions.',\n",
       "  'dmso cell line a549 lung epithelial cells pretreated ifn',\n",
       "  'dmso ifn cell line a549 lung epithelial cells pretreated plus alpha2',\n",
       "  'human'),\n",
       " 'GSE126999-1-vs-2-human dn': (['The effect on p53-induced gene transactivation by ARVCF knockdown.'],\n",
       "  'To explore the effect of ARVCF on gene transactivation induced by p53, we evaluated the changes in gene expression by RNA-seq in ARVCF knockdown or control U2OS cells.',\n",
       "  'u2os treat /variation cells stably expressing untreated cell line osteosarcoma',\n",
       "  'u2os sharvcf /variation cells stably expressing targeting arvcf treated 10um nutlin 3a cell line osteosarcoma',\n",
       "  'human'),\n",
       " 'GSE166761-1-vs-4-human dn': (['STK4 deficiency underlies impaired interferon signaling and T cell immunity'],\n",
       "  'We investigated transcriptional responses following in vitro stimulation of peripheral blood mononuclear cells (PBMCs) obtained from a patient with inherited STK4 deficiency. Samples from the heterozygous parents and unrelated healthy donors with a wild-type genotype were analyzed for comparison.',\n",
       "  'ifna ifnb pbmc disease state healthy control timepoint 2h',\n",
       "  'rpmi pbmc disease state healthy control timepoint 2h',\n",
       "  'human'),\n",
       " 'GSE129338-2-vs-1-mouse dn': (['Cdk8 is required for establishment of chromatin marks and gene repression by Xist and mouse embryonic lineage development'],\n",
       "  'We previously reported the cyclin dependent kinase Cdk8 as a putative silencing factor for Xist. Here we investigate the role of Cdk8 in X inactivation and embryo development. We engineered a Cdk8 mutation in mouse embryonic stem cells carrying an inducible system for studying Xist function. We find that Xist repressed X-linked genes to half the expression level of the active X chromosome in the absence of Cdk8, whereas near complete silencing was observed in cells with intact Cdk8. The reduced ability of Xist to repress genes is paralleled with reduced Ezh2 recruitment and establishment of histone H3 lysine 27 tri-methylation. Introduction of wild-type but not catalytically inactive Cdk8 transgenes restores efficient gene repression and PRC2 recruitment. Notably, mutation of the highly homologous kinase Cdk19 does not affect Xist function. Combined mutations of Cdk8 and Cdk19 resemble the Cdk8 single mutation. Analysis of a conditional Cdk8 mutation in mice reveals a post-implantation lethal phenotype with variable onset between day 8.5 and 10.5. A clear female specific phenotype is masked by requirements of Cdk8 for processes in addition to X inactivation.',\n",
       "  'nt transgenic system dox inducible xist expression wt 48h mouse embryonic stem cells',\n",
       "  'dox transgenic system inducible xist expression wt 48h yes mouse embryonic stem cells',\n",
       "  'mouse'),\n",
       " 'GSE115168-2-vs-3-human dn': (['ER proteostasis and temperature differentially impact the mutational tolerance of influenza hemagglutinin'],\n",
       "  'Here, we present a systematic and quantitative test of the hypothesis that the composition and activities of the endoplasmic reticulum (ER) proteostasis network impact mutational tolerance of secretory pathway client proteins. We focus on influenza hemagluttinin (HA), a viral coat protein that folds in the host’s ER via a complex but well-characterized pathway. By integrating chemical methods to modulate the unfolded protein response with deep mutational scanning to assess mutational tolerance, we discover that upregulation of ER chaperones broadly enhances HA mutational tolerance across numerous sites and secondary/tertiary structure elements, including sites targeted by host antibodies. Remarkably, this host chaperone-enhanced mutational tolerance is observed at the same HA sites where mutational tolerance is most reduced by propagation at a fever-like temperature. Thus, host ER proteostasis mechanisms and temperature modulate HA mutational tolerance in opposite directions. This finding has important implications for influenza evolution, because influenza immune escape is contingent on HA possessing sufficient mutational tolerance to acquire antibody resistance while still maintaining the capacity to fold and function. More broadly, this work provides the first experimental evidence that the composition and activities of the ER proteostasis network critically define the mutational tolerance and, therefore, the evolution of secretory pathway client proteins.',\n",
       "  'd17 atf6f/xbp1s 37c adult derived epithelial cell line human embryonic kidney (hek) 293t rex clonal stably expressing dhfr.atf6 tet xbp1s tetracycline repressor condition 24h 0.1 ug/ml doxycycline + 1 um trimethoprim cells',\n",
       "  'd17 basal 37c adult derived epithelial cell line human embryonic kidney (hek) 293t rex clonal stably expressing dhfr.atf6 tet xbp1s tetracycline repressor condition 24h 0.01% dmso cells',\n",
       "  'human'),\n",
       " 'GSE151165-0-vs-1-human dn': (['Neutrophils expressing lysyl oxidase-like 4 protein are present in colorectal cancer liver metastases resistant to anti-angiogenic therapy'],\n",
       "  'Colorectal cancer liver metastases (CRCLM) that present with a replacement histopathological growth pattern (HGP) are\\xa0resistant to neoadjuvant anti-angiogenic therapy. Surrogate biomarkers are not available to preoperatively identifypatients with these tumors. Here we identify differentially expressed genes between CRCLM with a replacement HGP\\xa0and those with a desmoplastic HGP using RNA sequencing.We demonstrate that LOXL4 is transcriptionally upregulated in replacement HGP CRCLM compared with desmoplastic HGP CRCLM and the adjacent normal liver. Interestingly, lysyl\\xa0oxidase-like 4 (LOXL4) protein was expressed by neutrophils present in the tumor microenvironment in replacement HGP CRCLM. We further demonstrate that LOXL4 expression is higher in circulating neutrophils of cancer patients compared\\xa0with healthy control patients and its expression can be induced by stimulation with lipopolysaccharide and TNF-alpha. Our study is the first to show the expression of LOXL4 in neutrophils and reveals the potential for LOXL4-expressing neutrophils\\xa0to support the replacement HGP phenotype and to serve as a surrogate biomarker for this subtype of CRCLM.',\n",
       "  'kr dn age sex normal disease colorectal cancer liver metastases (crclm) sample type naive rna',\n",
       "  'kr dt age sex tumor disease colorectal cancer liver metastases (crclm) sample type naive rna',\n",
       "  'human'),\n",
       " 'GSE155447,GSE155448-0-vs-3-human up': (['RNA-seq and Ribo-seq analysis of the effects of METTL3/14 depletion on gene expression during the interferon response in Huh7 cells'],\n",
       "  \"The RNA modification N6-methyladenosine (m6A) regulates gene expression through various transcript-specific effects. The overall goal of these experiments was to determine the effects of the m6A methyltransferase complex proteins METTL3 and METTL14 on the expression of interferon stimulated genes. First, we depleted METTL3 and METTL14 in Huh7 cells using siRNAs and treated with Mock or IFN-β to analyze the regulatory effects of METTL3/14 on interferon-stimulated genes' transcript abundance. We then used Ribo-seq following METTL3/14 siRNA depletion and IFN treatment to quantify its effect on the translation of ISGs and other genes. Overall, we found that METTL3/14 exerts transcript-specific regulatory effects on a subset of interferon-stimulated genes. Overall, METTL3/14 appeared to enhance the translation of a subset of interferon-stimulated genes. We characterized the molecular effects of METTL3/14 and m6A on certain ISGs and determined that m6A augments the antiviral effects of the interferon response.\",\n",
       "  'mock simettl3/14 sirna huh7 cells',\n",
       "  'ifn simettl3/14 rna interferon sirna huh7 cells',\n",
       "  'human'),\n",
       " 'GSE152461-4-vs-8-human up': (['Transcriptomic analysis of brian tissues identifies a role for CCAAT enhancer binding protein β in HIV- associated neurocognitive disorder'],\n",
       "  'In this study, using 33 postmortem frontal cortex (FC) tissues, neuropathological, molecular, and biochemical analyses were used to determine cellular localization and validate expression levels of the prolific transcription factor (TF), CCAAT enhancer binding protein (C/EBP) β, in brain tissues from HIV+ cognitively normal and HAND cases. RNA sequencing (seq) and transcriptomic analyses were performed on FC tissues including 24 specimens from well-characterized people with HIV that had undergone neurocognitive assessments. In vitro models for brain cells were used to investigate the role of C/EBPβ in mediating gene expression.',\n",
       "  'frontal cortex age sex male neurocognitive status mnd arv haart',\n",
       "  'frontal cortex age neurocognitive status arv',\n",
       "  'human'),\n",
       " 'GSE221240-4-vs-10-human dn': (['Characterization of cell-type specific circular RNAs associated with colorectal cancer metastasis'],\n",
       "  'Colorectal cancer (CRC) is the most common gastrointestinal malignancy in the U.S.A. and approximately 50% of CRC patients develop metastatic disease (mCRC) with an average 5-year survival rate of 13%. Despite our understanding of circular RNAs (circRNAs) in primary cancer, their role throughout mCRC remains poorly characterized. Further, despite the availability of bulk RNA-Seq data for monitoring circRNA expression little is known about their cell-type specific expression profiles to fully understand their functions between tumor cells and the microenvironment. To evaluate circRNAs in mCRC progression, total RNA-Seq was performed on 30 matched normal, primary, and distant metastatic samples from 14 mCRC patients. Additionally, the transcriptome of five CRC cell lines were sequenced to construct a circRNA catalog in CRC. Differential expression (DE) analysis identified circRNAs dysregulated between normal, primary, and metastatic tissues. Differential backsplicing (DS) analysis was used to highlight circRNAs with an altered tendency in backsplicing as compared to their linear counterparts. Cell-type deconvolution was performed using published CRC single-cell RNA-Seq datasets as signature matrices. The cell-type specific expression of circRNAs was then estimated through a non-negative least squares statistical model.',\n",
       "  'normal liver subject sample id',\n",
       "  'primary colon cancer subject sample id',\n",
       "  'human'),\n",
       " 'GSE201767-6-vs-11-mouse up': (['Transcriptome response of mouse mammary gland from age 12 to 30 months of age in Esr1 and CYP19A1 genetically engineered mouse models of breast cancer risk in the presence and absence of transgene induction'],\n",
       "  'Breast cancer is an age-related cancer in women with two peaks, one at 50 and one at 70 years of age. Here we used two conditional genetically engineered mouse models of breast cancer risk to study mammary gland transcriptional changes that occur as female mice age from 12 to 30 months of age, paralleling aging from 58 to 85 years of age in women. The two models express either mammary epithelial cell-targeted Estrogen Receptor (ER) alpha (Esr1) or Aromatase (CYP19A1A) over-expression beginning at age 12 or 18 months of age. Both of these risk factors increase estrogen pathway signaling, either directly though the receptor or by increasing local estrogen production. The goals of the study are to determine how quickly significant transcriptional changes occur in the mammary gland following transgene induction, to determine how the transcriptome becomes modified as the mice age past reproductive senescence through old and very old life stages, and to identify similarities and differences in the transcriptomes between the two risk conditions at different ages and conditions. The first specific objective of this transcriptome-based study is to identify significantly differentially expressed genes within each model between different ages and times of transgene induction. This includes transcriptional changes induced by 1 week, 6, 12 and 18 months of transgene expression with transgene expression initiated at age 12 months as well as transcriptional changes induced by 1 week and 6 months of transgene expression initiated at age 18 months. The second specific objective is to compare the two models at different ages (12, 18, 24 and 30 months of age) and different transgene induction times (6, 12 and 18 months) to identify differentially expressed genes between the models. Comparative control mice from each model include age 18- and 24-month-old mice without transgene induction.',\n",
       "  'mammary gland age 12m cyp19a1 transgene induction biol rep [arom012m arom mmtv rtta/tet op',\n",
       "  'mammary gland age 24m 6 months cyp19a1 transgene induction biol rep s6] mmtv rtta/tet op',\n",
       "  'mouse'),\n",
       " 'GSE168526-1-vs-3-human up': (['Novel human-derived RET fusion NSCLC cell lines have heterogeneous responses to RET inhibitors and differential regulation of downstream signaling'],\n",
       "  'The goal of this study was to understand RET signaling dynamics and regulation of response to RET inhibitors',\n",
       "  'lc 2 ad rxdx 105 cell line 2/ad',\n",
       "  'lc 2 ad dmso cell line 2/ad',\n",
       "  'human'),\n",
       " 'GSE67543-2-vs-4-human up': (['Novel Observations from Next Generation RNA Sequencing of Highly Purified Human Adult and Fetal Islet Cell Subsets'],\n",
       "  'Understanding distinct gene expression patterns of normal adult and developing fetal human pancreatic a and b cells is crucial for developing stem cell therapies, islet regeneration strategies, and therapies designed to increase b cell function in patients with diabetes (type 1 or 2). Toward that end, we have developed methods to highly purify a, b, and d cells from human fetal and adult pancreata by intracellular staining for the cell-specific hormone content, sorting the sub-populations by flow cytometry and, using next generation RNA sequencing, we report on the detailed transcriptomes of fetal and adult a and b cells. We observed that human islet composition was not influenced by age, gender, or body mass index and transcripts for inflammatory gene products were noted in fetal b cells. In addition, within highly purified adult glucagon-expressing a cells, we observed surprisingly high insulin mRNA expression, but not insulin protein expression. This transcriptome analysis from highly purified islet a and b cell subsets from fetal and adult pancreata offers clear implications for strategies that seek to increase insulin expression in type 1 and type 2 diabetes.',\n",
       "  'adult alpha developmental stage cells',\n",
       "  'adult beta developmental stage cells',\n",
       "  'human'),\n",
       " 'GSE202454-0-vs-1-mouse up': (['Orthotopic CRC organoid transplantation in mice: A novel model for the study and treatment of peritoneal carcinomatosis [Bulk RNA-Seq]'],\n",
       "  'Peritoneal carcinomatosis (PC) originating from colorectal cancer (CRC) is associated with a poor prognosis and rapid disease progression. Metastasis is the major cause of death in patients with CRC whereas primary locoregionally circumscribed tumors are mostly curable. Due to the lack of appropriate and realistic mouse models for metastasis of CRC until recently, the research about metastasis of these tumors and possible therapeutic options is still in its infancy. With the establishment of murine CRC tumor organoids, mouse models were established which realistically mimic the process of tumor cell metastasis to the liver and the lung. However, to date no appropriate model for PC is described. Here, we establish a realistic model for peritoneal metastasis of colorectal cancer upon surgical transplantation of tumor organoids into the cecum wall. This mouse model for PC recapitulates human PC which makes it an ideal model for research of the biology of metastasis of colorectal cancer to the peritoneal cavity and for its usage for preclinical drug screening as well as for the evaluation of new therapeutic approaches which are highly needed in clinical reality.',\n",
       "  'disease state colorectal cancer (crc) organoid derived peritoneal carcinomatosis',\n",
       "  'disease state colorectal cancer (crc) organoid derived crc liver metastasis',\n",
       "  'mouse'),\n",
       " 'GSE73721-1-vs-3-human up': (['RNA-Seq of human astrocytes'],\n",
       "  \"Astrocytes were purified from fetal and adult human brain tissue using an immunopanning method with the HepaCAM antibody. Samples were taken from otherwise 'healthy' pieces of tissue, unless otherwise specified.\",\n",
       "  'ctx astro age years old cell subtype hepacam positive cells astrocyte adult temporal lobe',\n",
       "  'ctx oligo age years old cell subtype galc positive cells oligodendrocyte adult temporal lobe',\n",
       "  'human'),\n",
       " 'GSE85443-2-vs-1-human up': (['Transcriptome analysis identifies the dysregulation of ultraviolet target genes in human skin cancers'],\n",
       "  'In this study, we performed RNA-Seq studies to generate a transcriptomic cohort containing UV-responsive genes in human skin cells exposed to different UVR conditions. We then performed rigorous bioinformatics analysis to define a UV gene expression signature that is conserved among cells from different donors. We further demonstrated that the UV signature genes are significantly enriched in genes dysregulated in human skin squamous cell carcinomas.',\n",
       "  'n1 uv (time/dose) vs. control neonatal foreskin cultured primary keratinocytes uvr',\n",
       "  'control neonatal foreskin cultured primary keratinocytes uvr',\n",
       "  'human'),\n",
       " 'GSE149311-1-vs-3-mouse up': (['RNAseq analysis of distinct small intestinal cell populations isolated from Lgr5-EGFP C57BL/6 mice treated with NRG1'],\n",
       "  'Purpose: The goal of the study was to investigate the effect of Neuregulin 1 (NRG1) on intestinal crypt-based columnar stem cells, their progenitors and differentiated cells.',\n",
       "  'low ngr1 strain c57bl/6 small intestine age week old nrg1 subsequent progenitors (lgr5low cd24low)',\n",
       "  'neg ngr1 strain c57bl/6 small intestine age week old nrg1 differentiated cells (lgr5neg cd24neg)',\n",
       "  'mouse'),\n",
       " 'GSE57973,GSE79469-1-vs-5-human dn': (['Age-dependent gene expression changes in human islets'],\n",
       "  'We report the transcriptomic profiling of islet cell subtypes obtained from young and old human donors.',\n",
       "  'beta pancreas age group old',\n",
       "  'alpha pancreas age group old',\n",
       "  'human'),\n",
       " 'GSE103940-3-vs-1-human up': (['Integration of metabolomics and transcriptomic expression profiling discloses pivotal role of arachidonic acid metabolism pathway in response to acute hypoxia exposure'],\n",
       "  'Background: Arachidonic acid (AA) metabolism pathway is dominant in metabolic programming after hypoxia exposure, but its biological function is disputed and require in-depth studies. In this research, we aimed at integrating plasma metabolomics and transcriptomics approaches to systematically explore its roles in response to acute hypoxia based on model of acute high-altitude exposure.',\n",
       "  'plain stress control blood',\n",
       "  'highaltitude stress high altitude exposure blood',\n",
       "  'human'),\n",
       " 'GSE187413,GSE187414-0-vs-1-human up': (['Potent stimulation of the androgen receptor instigates a viral mimicry response in prostate cancer [RNA-seq]'],\n",
       "  'Inhibiting the androgen receptor (AR), a ligand-activated transcription factor, with androgen deprivation therapy is a standard-of-care treatment for metastatic prostate cancer (PCa). Paradoxically, activation of AR can also inhibit the growth of PCa in some patients and experimental systems, but the mechanisms underlying this phenomenon are poorly understood. This study exploited a potent synthetic androgen, methyltestosterone (MeT), to investigate AR agonist-induced growth inhibition. MeT strongly inhibited growth of PCa cells expressing AR, but not AR-negative models. Genes and pathways regulated by MeT were highly analogous to those regulated by DHT, although MeT induced a quantitatively greater androgenic response in PCa cells. MeT potently down-regulated DNA methyltransferases, leading to global DNA hypomethylation. These epigenomic changes were associated with dysregulation of transposable element expression, with long-term MeT treatment resulting in upregulation of endogenous retrovirus (ERV) transcripts. Increased ERV expression led to accumulation of double-stranded RNA and a “viral mimicry” response characterised by activation of interferon signalling, upregulation of MHC Class I molecules and enhanced recognition of murine PCa cells by CD8+ T cells. Positive associations between AR activity and ERVs/anti-viral pathways were evident in patient transcriptomic data, supporting the clinical relevance of our findings. Collectively, our study reveals that the potent androgen MeT can increase the immunogenicity of PCa cells via a viral mimicry response, a finding that has potential implications for the development of strategies to sensitize this cancer type to immunotherapies.',\n",
       "  'dht lncap 1nm',\n",
       "  'vehicle lncap ethanol 0.1% (vehicle)',\n",
       "  'human'),\n",
       " 'GSE125101-0-vs-5-mouse dn': (['Environmental Arginine Controls Multinuclear Giant Cell Metabolism and Formation'],\n",
       "  'Multinucleated giant cells (MGCs) are implicated in many diseases including schistosomiasis, sarcoidosis and arthritis. Formation of MGCs is energy intensive to enforce membrane fusion and cytoplasmic expansion. Here we used receptor activator of nuclear factor kappa-Β ligand (RANKL) induced osteoclastogenesis to model MGC formation. We found amino acid (AA) scarcity controls MGC formation and reveal specific requirements for extracellular arginine in RANKL cellular programming. Systemic arginine restriction improved outcome in multiple murine arthritis models and its removal induced preosteoclast metabolic quiescence, associated with impaired tricarboxylic acid (TCA) cycle function and metabolite induction. Effects of arginine deprivation on osteoclastogenesis were independent of mTORC1 activity or global transcriptional and translational inhibition. Arginine scarcity also dampened generation of IL-4 induced polykaryons, another form of MGCs. Strikingly, in the absence of extracellular arginine, both cell types displayed flexibility as their formation could be restored with select arginine precursors. These data establish how environmental amino acids control the metabolic fate of polykaryons and suggest metabolic ways to manipulate MGC-associated pathologies and bone remodelling.',\n",
       "  'condition mcsf+rankl+depl mouse preosteoclasts',\n",
       "  'condition mcsf+rankl+recarg1 mouse preosteoclasts',\n",
       "  'mouse'),\n",
       " 'GSE82291,GSE82292-4-vs-0-human dn': (['Quantitative profiling of the UGT transcriptome in human drug metabolizing tissues [Total RNA]'],\n",
       "  'Purpose: Maintenance of cellular homeostasis and xenobiotics detoxification relies on the glucuronidation pathway mediated by 19 human UDP-glucuronosyltransferase enzymes (UGTs) encoded by 10 highly homologous genes. Recent evidence suggests that alternative splicing largely expands the human UGT transcriptome.',\n",
       "  'liver normal (n=3 donors) total rna sex pooled male female',\n",
       "  'intestine colon tumor (n=3 donors) total rna sex pooled male female',\n",
       "  'human'),\n",
       " 'GSE115168-0-vs-1-human dn': (['ER proteostasis and temperature differentially impact the mutational tolerance of influenza hemagglutinin'],\n",
       "  'Here, we present a systematic and quantitative test of the hypothesis that the composition and activities of the endoplasmic reticulum (ER) proteostasis network impact mutational tolerance of secretory pathway client proteins. We focus on influenza hemagluttinin (HA), a viral coat protein that folds in the host’s ER via a complex but well-characterized pathway. By integrating chemical methods to modulate the unfolded protein response with deep mutational scanning to assess mutational tolerance, we discover that upregulation of ER chaperones broadly enhances HA mutational tolerance across numerous sites and secondary/tertiary structure elements, including sites targeted by host antibodies. Remarkably, this host chaperone-enhanced mutational tolerance is observed at the same HA sites where mutational tolerance is most reduced by propagation at a fever-like temperature. Thus, host ER proteostasis mechanisms and temperature modulate HA mutational tolerance in opposite directions. This finding has important implications for influenza evolution, because influenza immune escape is contingent on HA possessing sufficient mutational tolerance to acquire antibody resistance while still maintaining the capacity to fold and function. More broadly, this work provides the first experimental evidence that the composition and activities of the ER proteostasis network critically define the mutational tolerance and, therefore, the evolution of secretory pathway client proteins.',\n",
       "  'd17 xbp1s 39c adult derived epithelial cell line human embryonic kidney (hek) 293t rex clonal stably expressing dhfr.atf6 tet tetracycline repressor condition 24h 0.1 ug/ml doxycycline cells',\n",
       "  'd17 basal 39c adult derived epithelial cell line human embryonic kidney (hek) 293t rex clonal stably expressing dhfr.atf6 tet xbp1s tetracycline repressor condition 24h 0.01% dmso cells',\n",
       "  'human'),\n",
       " 'GSE246481,GSE246482-0-vs-1-mouse up': (['Alveolar differentiation drives resistance to KRAS inhibition in lung adenocarcinoma [scRNA-seq]'],\n",
       "  'Lung adenocarcinoma (LUAD), commonly driven by KRAS mutations, is responsible for 7% of all cancer mortality. The first allele-specific KRAS inhibitors were recently approved in LUAD, but clinical benefit is limited by intrinsic and acquired resistance. LUAD predominantly arises from alveolar type 2 (AT2) cells, which function as facultative alveolar stem cells by self-renewing and replacing alveolar type 1 (AT1) cells. Using genetically engineered mouse models, patient-derived xenografts, and patient samples we found inhibition of KRAS promotes transition to a quiescent AT1-like cancer cell state in LUAD tumors. Similarly, suppressing Kras induced AT1 differentiation of wild-type AT2 cells upon lung injury. The AT1-like LUAD cells exhibited high growth and differentiation potential upon treatment cessation, whereas ablation of the AT1-like cells robustly improved treatment response to KRAS inhibitors. Our results uncover an unexpected role for KRAS in promoting intra-tumoral heterogeneity and suggest targeting alveolar differentiation may augment KRAS-targeted therapies in LUAD.',\n",
       "  'kp mrtx hto lung primary tumor cells',\n",
       "  'shkras dox hto lung primary cells',\n",
       "  'mouse'),\n",
       " 'GSE146795-0-vs-2-human up': (['RSV-induced Cytoskeletal Inflammation'],\n",
       "  'We studied RSV infection in an appropriate in vitro model of respiratory epithelium, a pseudostratified and fully differentiated mucociliary epithelium of normal human bronchial epithelial (NHBE) cells. RSV infection increased actin cytoskeleton without compromising adherent-, tight-, and tricellular-junctions as well as ciliary functions and epithelial tissue barrier integrity. This increased cytoskeleton depends on actin polymerization and the induction of proinflammatory cytokines and chemokines. Thus, we observed a novel signature “increased cytoskeleton” termed “cytoskeletal inflammation” in RSV-infected respiratory epithelium that presumably lacks classical antigen presenting cells, such as resident dendritic cells and macrophages. Our results suggest that RSV-induced cytoskeletal inflammation is a noncanonical earliest host response to the pathogen and contributes to airway inflammation.',\n",
       "  'rsv nhbe c16 p4',\n",
       "  'mock nhbe c16 p4',\n",
       "  'human'),\n",
       " 'GSE247401-2-vs-6-human dn': (['Blood transcriptome profiling reveals distinct gene networks induced by mRNA vaccination against COVID-19'],\n",
       "  'Messenger RNA (mRNA) vaccines represent a new class of vaccines that has been shown to be highly effective during the COVID-19 pandemic and that holds great potential for other preventative and therapeutic applications. Understanding the underlying mechanisms of the immune responses induced by this novel vaccine type and their relation to vaccine responses might help to further refine and optimize future vaccine design. In this study, we conducted an in-depth analysis of the blood transcriptome before and 24h after second and third vaccination with licensed mRNA vaccines against COVID-19 in humans, following a prime vaccination with either mRNA or ChAdOx vaccines. Utilizing an unsupervised weighted gene correlation network analysis, we identified distinct gene networks of co-varying genes characterized by either an expressional up- or down-regulation in response to vaccination. Down-regulated networks were associated with cell metabolic processes and regulation of transcription factors, while up-regulated networks were associated with myeloid differentiation, antigen presentation, and antiviral, interferon-driven pathways. Within this interferon-associated network, we identified highly connected hub genes such as STAT2 and RIGI, potentially playing important regulatory roles in the vaccine-induced immune response. The expression profile of this network significantly correlated with S1-specific IgG levels at the follow-up visit in vaccinated individuals. Those findings could be corroborated in an independent cohort of mRNA vaccine recipients. Collectively, insights from this study might contribute to current research endeavors aimed at further enhancing/ or optimizing vaccine-induced immunity.',\n",
       "  'chad whole blood participant id vaccination v2 24h v1 vaccine type chadox bnt162b v3 sex female 2',\n",
       "  'mrna whole blood participant id vaccination v2 pre v1 vaccine type bnt162b v3 sex female 2',\n",
       "  'human'),\n",
       " 'GSE122252,GSE136864-1-vs-3-human up': (['The ETS transcription factor ELF1 regulates a critical, broadly antiviral program distinct from the type I interferon response'],\n",
       "  'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'a549 cells empty mock cell line lung epithelial adenocarcinoma passages 25 elf1 construct transduction replicate ifn transduced pscrpsy',\n",
       "  'a549 cells empty cell line lung epithelial adenocarcinoma passages 25 elf1 construct transduction replicate ifn transduced pscrpsy treated hours',\n",
       "  'human'),\n",
       " 'GSE232817,GSE232824-2-vs-4-human up': (['Modular Pooled Discovery of Synthetic Knockin Sequences to Program Durable Cell Therapies [RNA-1]'],\n",
       "  'Chronic stimulation can cause T cell dysfunction and limit efficacy of cellular immunotherapies. CRISPR screens have nominated gene targets for engineered T cells, but improved methods are required to compare large numbers of synthetic knockin sequences to reprogram cell functions. Here, we developed Modular Pooled Knockin Screening (ModPoKI), an adaptable platform for modular construction of DNA knockin libraries using barcoded multicistronic adaptors. We built two ModPoKI libraries of 100 transcription factors (TFs) and 129 natural and synthetic surface receptors. Over 30 ModPoKI screens across human TCR and CAR T cells in diverse conditions identified a transcription factor AP4 (TFAP4) construct that enhanced fitness of chronically-stimulated CAR T cells and anti-cancer function in vitro and in vivo. ModPoKI’s modularity allowed us to generate a ~10,000-member library of TF combinations. Non-viral knockin of a combined BATF-TFAP4 polycistronic construct further enhanced function. ModPoKI facilitates discovery of complex gene constructs to program cellular functions.',\n",
       "  'cells tngfr cd19 bbz car knockin day 15 donor individual blood cell line primary none time',\n",
       "  'cells tngfr cd19 28z car knockin day 6 donor individual blood cell line primary none time',\n",
       "  'human'),\n",
       " 'GSE235908,GSE235909-7-vs-3-human dn': (['Characterization of a drug-resilient cancer cell phenotype [RNA-Seq 1]'],\n",
       "  'Resistance to treatment and post-therapy recurrence due to small populations of resistant cells remain core challenges in cancer therapy. The mechanisms underlying therapy resistance are undefined for many patients. Here, we demonstrate that some cancer cells survive therapeutic stress via a transient state of whole genome doubling. At the onset of the polyploidization program, we identified upregulation of key transcriptional regulators, including the early stress-response protein AP-1 and normoxic stabilization of HIF-2α. Moreover, chromatin accessibility is altered and expression of pRB is ablated, and we identified enrichment of AP-1 motif accessibility along with an upregulation of replication-dependent histone variants. Here we demonstrate that AP-1and HIF-2α regulate a therapy resilient and survivor phenotype in cancer cells. Consistent with this, genetic or pharmacologic targeting of AP-1 and HIF-2α reduced the frequency of surviving cells following treatment with chemotherapy. These results suggest a novel avenue for tackling chemotherapy-induced resistance in cancer.',\n",
       "  'hct116 cells control colon cell line cancer wt vehicle',\n",
       "  'hct116 cells platinum treated colon cell line cancer wt cisplatin',\n",
       "  'human'),\n",
       " 'GSE96055-2-vs-3-mouse up': (['Time-course analysis of microglia-specific RNA-seq in two severities of spinal cord injury'],\n",
       "  'RNA-seq analysis of microglia following spinal cord injury demonstrated that microglia response is driven by time after injury but not lesion severity.',\n",
       "  'mgni /variation cx3cr1+/egfp injury non injured time start point microglia',\n",
       "  '/variation cx3cr1+/egfp injury hemi section time microglia',\n",
       "  'mouse'),\n",
       " 'GSE185670-0-vs-1-human up': (['Transcriptome profiling of hESC-derived endoderm progenitors by RNA-seq'],\n",
       "  'Transcriptome profiling of hESC-derived endoderm progenitors by RNA-seq. RNA-seq analysis during the stepwise progression of hESC toward the pancreatic progenitors, at five defined stages of differentiation and expansion: hESC, ADE, VFG.p3, VFG.p6. PE.',\n",
       "  'rna seq hues4 170 3 vfg derived pe cell line pdx1 egfp reporter facs sorted population positive cells pancreatic endoderm progenitor',\n",
       "  'rna seq hues4 170 3 hesc derived vfg passage cell line pdx1 egfp reporter facs sorted population cd184 pecy7 cd117 apc double positive cells ventral foregut progenitor',\n",
       "  'human'),\n",
       " 'GSE239662,GSE239663-4-vs-0-human up': (['Novel method of isolating nuclei of human oligodendrocyte precursor cells reveals substantial developmental changes in gene expression and H3K27ac histone modification. [RNA-Seq]'],\n",
       "  'Oligodendrocyte precursor cells (OPCs) generate differentiated mature oligodendrocytes (MOs) during development. In adult brain, OPCs replenish MOs in adaptive plasticity, neurodegenerative disorders, and after trauma. The ability of OPCs to differentiate to MOs decreases with age and is compromised in disease. Here we explored the cell specific and age-dependent differences in gene expression and H3K27ac histone mark in these two cell types. H3K27ac is indicative of active promoters and enhancers. We developed a novel flow-cytometry-based approach to isolate OPC and MO nuclei from human postmortem brain and profiled gene expression and H3K27ac in adult and infant OPCs and MOs genome-wide.In adult brain, we detected extensive H3K27ac differences between the two cell types with high concordance between gene expression and epigenetic changes. Notably, the expression of genes that distinguish MOs from OPCs appears to be under a strong regulatory control by the H3K27ac modification in MOs but not in OPCs. Comparison of gene expression and H3K27ac between infants and adults uncovered numerous developmental changes in each cell type, which were linked to several biological processes, including cell proliferation and glutamate signaling. A striking example was a subset of histone genes that were highly active in infant samples but fully lost activity in adult brain. Our findings demonstrate a considerable rearrangement of the H3K27ac landscape that occurs during the differentiation of OPCs to MOs and during postnatal development of these cell types, which aligned with changes in gene expression. The uncovered regulatory changes justify further in-depth epigenetic studies of OPCs and MOs in development and disease.',\n",
       "  'subject oligodendrocyte precursor cells rna seq dorsolateral prefrontal cortex donor id disease control sex male',\n",
       "  'subject mature oligodendrocytes rna seq dorsolateral prefrontal cortex donor id disease control sex female',\n",
       "  'human'),\n",
       " 'GSE188458-2-vs-0-human dn': (['RNA-seq analysis of differentially expressed genes in response to Neocarzinostatin and Hydrogen Peroxide'],\n",
       "  'In this study we identified differentially expressed genes in hTERT-HPNE cells in response to two distinct cellular stressors over time.  1) Neocarzinostatin induced DNA double strand breaks and 2) Oxidative stress induced through hydrogen peroxide treatment.  Using these data sets we identified specific genes involved in differential cell fate determination in response to these stresses.',\n",
       "  'h2o2 0 rep untreated cell line htert hpne pancreatic',\n",
       "  'ncs 24 rep 400 ng/ml neocarzinostatin cell line htert hpne pancreatic',\n",
       "  'human'),\n",
       " 'GSE73721-0-vs-1-human dn': (['RNA-Seq of human astrocytes'],\n",
       "  \"Astrocytes were purified from fetal and adult human brain tissue using an immunopanning method with the HepaCAM antibody. Samples were taken from otherwise 'healthy' pieces of tissue, unless otherwise specified.\",\n",
       "  'ctx myeloid age years old cell subtype cd45 positive cells adult temporal lobe',\n",
       "  'ctx astro age years old cell subtype hepacam positive cells astrocyte adult temporal lobe',\n",
       "  'human'),\n",
       " 'GSE111595,GSE111605-2-vs-3-mouse dn': (['Chemical Screening Identifies Enhancers of Mutant Oligodendrocyte Survival and Unmasks a Distinct Pathological Phase in Pelizaeus-Merzbacher Disease [RNA-seq]'],\n",
       "  'Pelizaeus-Merzbacher disease (PMD) is a fatal X-linked disorder caused by loss of myelinating oligodendrocytes and consequent hypomyelination. The underlying cellular and molecular dysfunctions are not fully defined, but therapeutic enhancement of oligodendrocyte survival could restore functional myelination in patients. Here we generated pure, scalable quantities of iPSC-derived oligodendrocyte progenitor cells (OPCs) from a severe mouse model of PMD, Plp1jimpy. Temporal phenotypic and transcriptomic studies defined an early pathological window characterized by endoplasmic-reticulum (ER) stress and cell death as OPCs exit their progenitor state. High-throughput phenotypic screening identified a compound, Ro 25-6981, which modulates the ER stress response and rescues mutant oligodendrocyte survival in jimpy, in vitro and in vivo, and in human PMD oligocortical spheroids. Surprisingly, increasing oligodendrocyte survival did not restore subsequent myelination, revealing a second pathological phase. Collectively, our work shows that PMD oligodendrocyte loss can be rescued pharmacologically and defines a need for multifactorial intervention to restore myelination.',\n",
       "  'wild type opcs (.wt strain b6cbaca/j aw j/ source (ipsc line) .wt oligodendrocyte progenitor cells',\n",
       "  'jimpy 1d oligodendrocytes (.jp strain b6cbaca/j aw j/ source (ipsc line) .jp differentiating',\n",
       "  'mouse'),\n",
       " 'GSE178640-2-vs-3-human up': (['Transcriptomics changes after IFN-alpha2 treatment following dBRD9-A or DMSO pretreatment in A549 lung epithelial cells'],\n",
       "  'These experiments aim at defining first the entirety of Interferon-stimulated gene expression  in A549 in response to treatment with 100 IU/mL IFN-alpha2. The second second aim was to characterize changes to the transcriptome caused by the prior degradation of bromodomain-containing protein9 (BRD9) by the use of a specific BRD9-degrader (dBRD9-A) both in the uninduced and IFN-alpha2-induced conditions.',\n",
       "  'dbrd9 cell line a549 lung epithelial cells pretreated ifn',\n",
       "  'dmso ifn cell line a549 lung epithelial cells pretreated plus alpha2',\n",
       "  'human'),\n",
       " 'GSE198321-0-vs-4-human dn': (['Cell type-specific gene expression profiles in CNS of patients with DM1'],\n",
       "  'Myotonic dystrophy type 1 (DM1) is a multisystem genetic disorder involving the muscle, heart, and central nervous system (CNS). The pathogenesis of CNS symptoms prevalent in patients with DM1 remains unelucidated. To elucidate the CNS pathogenesis in DM1, we investigated cell type-specific abnormalities in cortical neurons, white matter glial cells, and spinal motor neurons of patients with DM1 via laser-capture microdissection(LCM).',\n",
       "  'ctrl white matter glial cells disease control brain',\n",
       "  'dm1 cortical neurons disease brain',\n",
       "  'human'),\n",
       " 'GSE246229,GSE246230-4-vs-3-human up': (['HNF1αmaintains pancreaticα and β cell functionsin primary human islets [RNA-seq]'],\n",
       "  'Transcriptomic profiling by high-throughput sequencing of primary human alpha and beta cells following HNF1A loss of function.',\n",
       "  'beta control cadaveric pancreatic cell line primary islet cells developmental stage adult /variation',\n",
       "  'alpha control cadaveric pancreatic cell line primary islet cells developmental stage adult /variation',\n",
       "  'human'),\n",
       " 'GSE223008-7-vs-6-mouse up': (['CALHM6 is an immunological synapse ion channel required for macrophage-natural killer cell communication upon bacterial infection'],\n",
       "  'Membrane ion channels of the calcium homeostasis modulator (CALHM) family promote cell-cell crosstalk at synapses via ATP release, where ATP acts as a neurotransmitter. CALHM6, the only CALHM highly expressed in immune cells, has been linked to induction of natural killer (NK) cell anti-tumour activity. Its mechanism of action and broader functions in the immune system are, however, unclear.  Here we generated Calhm6-/- mice and report that CALHM6 is important for regulating the early innate control of Listeria monocytogenes infection in vivo. We find that CALHM6 is upregulated in macrophages by pathogen-derived signals, and that it relocates from intracellular compartment to the macrophage-NK cell synapse, facilitating ATP release and controlling the kinetics of NK cell activation. Anti-inflammatory cytokines terminate CALHM6 expression. CALHM6 forms an ion channel when expressed in the plasma membrane of Xenopus oocytes, where channel opening is controlled by a conserved acidic residue, E119, but CALHM6 is localized to intracellular compartments in mammalian cells. Our results contribute to understanding of neurotransmitter-like signal exchange between immune cells that fine-tunes the timing of innate immune responses.',\n",
       "  'bmdm wt polyic bone marrow derived macrophages wild type poly( c) (50 ug/ml',\n",
       "  'bmdm ko un bone marrow derived macrophages calhm6 / untreated',\n",
       "  'mouse'),\n",
       " 'GSE216273,GSE216292-14-vs-8-human dn': (['Palbociclib releases the latent differentiation capacity of neuroblastoma cells [RNA-seq PB]'],\n",
       "  'We have used RNA-seq to assess the global impact of palbociclib on the transcriptome at early and late time points in three neuroblastoma cell lines.',\n",
       "  'shsy5y control neuroblastoma cell line sh sy5y agent',\n",
       "  'shsy5y 24hpb neuroblastoma cell line sh sy5y 24h agent palbociclib',\n",
       "  'human'),\n",
       " 'GSE86064-10-vs-4-human dn': (['Mandatory Role of HMGA1 in Human Airway Epithelial Normal Differentiation and Post-injury Regeneration'],\n",
       "  'Background: High mobility group AT-hook1 (HMGA1) is essential for airway basal cell mucociliary differentiation, barrier integrity and wound repair.  HMGA1 expression suppresses the abnormal basal cell differentiation to squamous, inflammatory and epithelial-mesenchymal transition phenotype commonly observed in association with cigarette smoking and chronic obstructive pulmonary disease (COPD).',\n",
       "  'dgm sm smoking status nonsmoker small airway epithelial cells',\n",
       "  'dgm lae bc smoking status copd smoker large airway basal cells cultured epithelium',\n",
       "  'human'),\n",
       " 'GSE67543-2-vs-4-human dn': (['Novel Observations from Next Generation RNA Sequencing of Highly Purified Human Adult and Fetal Islet Cell Subsets'],\n",
       "  'Understanding distinct gene expression patterns of normal adult and developing fetal human pancreatic a and b cells is crucial for developing stem cell therapies, islet regeneration strategies, and therapies designed to increase b cell function in patients with diabetes (type 1 or 2). Toward that end, we have developed methods to highly purify a, b, and d cells from human fetal and adult pancreata by intracellular staining for the cell-specific hormone content, sorting the sub-populations by flow cytometry and, using next generation RNA sequencing, we report on the detailed transcriptomes of fetal and adult a and b cells. We observed that human islet composition was not influenced by age, gender, or body mass index and transcripts for inflammatory gene products were noted in fetal b cells. In addition, within highly purified adult glucagon-expressing a cells, we observed surprisingly high insulin mRNA expression, but not insulin protein expression. This transcriptome analysis from highly purified islet a and b cell subsets from fetal and adult pancreata offers clear implications for strategies that seek to increase insulin expression in type 1 and type 2 diabetes.',\n",
       "  'adult alpha developmental stage cells',\n",
       "  'adult beta developmental stage cells',\n",
       "  'human'),\n",
       " 'GSE146782-4-vs-5-mouse up': (['The role of IL-33 on red pulp macrophage (RPM) development'],\n",
       "  'This study investigated the effects of IL1RL1 deficiency on the devlopment of red pulp macrophages (RPM)',\n",
       "  'splenocytes kom monocyte strain balb/c il1rl1 / sorting ly6glo cd11bhi f4/80 cells splenic monocytes mouse',\n",
       "  'splenocytes kor red pulp macrophage strain balb/c il1rl1 / sorting ly6glo cd11blo/ f4/80hi cells rpm mouse',\n",
       "  'mouse'),\n",
       " 'GSE211802-2-vs-1-human dn': (['Effect of depletion of MDMX (aka MDM4) on gene expression in uveal melanoma cell line MEL202'],\n",
       "  'To investigate the effect of MDMX depletion on the transcriptome of MEL202 cells we established 5 derivatives either containing a doxycyline- inducible Contrl shRNA or with 4 distinct MDMX -targeting shRNAs',\n",
       "  'mel202/ shctrl doxycycline biol rep cell line uveal melanoma ctrl knockdown vehicle 48 hrs',\n",
       "  'mel202/ shmdmx + doxycycline biol rep cell line uveal melanoma mdmx knockdown 48 hrs',\n",
       "  'human'),\n",
       " 'GSE122252,GSE136864-0-vs-3-human up': (['The ETS transcription factor ELF1 regulates a critical, broadly antiviral program distinct from the type I interferon response'],\n",
       "  'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'a549 cells mock cell line lung epithelial adenocarcinoma passages 25 elf1 construct transduction none replicate ifn',\n",
       "  'a549 cells empty cell line lung epithelial adenocarcinoma passages 25 elf1 construct transduction replicate ifn transduced pscrpsy treated hours',\n",
       "  'human'),\n",
       " 'GSE73721-0-vs-4-human dn': (['RNA-Seq of human astrocytes'],\n",
       "  \"Astrocytes were purified from fetal and adult human brain tissue using an immunopanning method with the HepaCAM antibody. Samples were taken from otherwise 'healthy' pieces of tissue, unless otherwise specified.\",\n",
       "  'ctx myeloid age years old cell subtype cd45 positive cells adult temporal lobe',\n",
       "  'tumor core astro age years old cell subtype hepacam positive cells astrocyte adult',\n",
       "  'human'),\n",
       " 'GSE102014-2-vs-6-mouse up': (['Gene co-expression networks identify Trem2 and Tyrobp as major hubs in human APOE expressing mice following traumatic brain injury.'],\n",
       "  'To determine if there is an APOE isoform-specific response to TBI we performed controlled cortical impact on 3-month-old mice expressing human APOE3 or APOE4 isoforms. Following injury, we used several behavior paradigms to test for anxiety and learning and found that APOE3 and APOE4 targeted replacement mice demonstrate cognitive impairments following moderate TBI. Transcriptional profiling 14\\xa0days following injury revealed a significant effect of TBI, which was similar in both genotypes.',\n",
       "  'f e3 strain background c57bl/6 targeted replacement apoe3+/+ injury sham brain cortices hippocampi site',\n",
       "  'f e3 strain background c57bl/6 targeted replacement apoe3+/+ injury tbi brain cortices hippocampi site',\n",
       "  'mouse'),\n",
       " 'GSE200485-0-vs-1-mouse up': (['Isolevuglandins disrupt PU.1 mediated C1q expression and promote autoimmunity and hypertension in systemic lupus erythematosus'],\n",
       "  'To determine the gene expression of profile of dendritic cells (DCs) from lupus prone mice we sorted CD11c+ cells from 4 7 week old female C57Bl/6 mice and 4 7 week old B6.SLE123 mice.',\n",
       "  'cell line primary dendritic cells age 7 weeks old',\n",
       "  'cell line primary dendritic cells age 7 weeks old',\n",
       "  'mouse'),\n",
       " 'GSE118880-1-vs-2-mouse up': (['Decomposing cell identity for transfer learning across cellular measurements, platforms, tissues, and species.'],\n",
       "  'Bulk ATAC-seq and RNA-seq on sorted retinal cells from Chx10:GFP reporter mice',\n",
       "  'gfp rna seq post mitotic retinal cells developmental embryonic day strain cd1',\n",
       "  'gfp+ rna seq retinal progenitor cells (rpcs) developmental embryonic day strain cd1',\n",
       "  'mouse'),\n",
       " 'GSE119336,GSE119337-1-vs-2-human dn': (['RNA over-editing leads to aggressiveness of intrahepatic cholangiocarcinoma [RNA-Seq]'],\n",
       "  'Alteration in RNA editing has been connected to tumor progression and many other important human diseases. However, the role of RNA editing in intrahepatic cholangiocarcinoma (ICC) remains unclear. Here we conducted a transcriptome-wide investigation in ICCs, and revealed ADAR-associated over-editing occurred uniformly in ICCs. Collectively, these results reveal a previously unrecognized role of RNA editing in malignant transformation into ICC, and ADAR1 inhibition may obviate progression and relapse of this malignancy.',\n",
       "  'gender male age adjacent non tumor liver',\n",
       "  'gender male age intrahepatic cholangiocarcinoma tumor',\n",
       "  'human'),\n",
       " 'GSE204721,GSE204722-1-vs-2-human dn': (['Global Epigenome and Transcriptome Maps of Human Trophoblast Cell Lineage Development [RNA-seq I]'],\n",
       "  'The extravillous trophoblast (EVT) cell lineage is a key feature of placentation and critical for spiral\\xa0artery remodeling and successful pregnancy. Our knowledge of transcriptional regulation driving EVT cell development is limited. Here, we mapped the transcriptome and epigenome landscape as well as chromatin interactions of human trophoblast stem (TS) cells and their transition into the differentiated EVT cell lineage. We identified that chromatin accessibility in intergenic regions was more extensive in EVT cells than in TS cells in the stem state, which is indicative of increased enhancer-driven gene regulation. Using reference epigenome annotation, we noted that 18% of the chromatin landscape in EVT cells was uncharted.\\xa0 We linked regulatory regions to their cognate target genes and characterized the three-dimensional organization of the TS cell functional genome by Hi-C. Integrational analysis of chromatin accessibility, long-range chromatin interactions, transcriptomic, and transcription factor binding motif enrichment enabled identification of transcription factors and regulatory mechanisms associated with EVT cell lineage development. Subsequent analyses elucidated functional roles for\\xa0TFAP2C,\\xa0EPAS1,\\xa0SNAI1, and\\xa0DLX6\\xa0in the regulation of EVT cell lineage development.\\xa0EPAS1\\xa0was identified as an upstream regulator of EVT cell transcription factors, including\\xa0SNAI1\\xa0and\\xa0DLX6, and was found to be upregulated in idiopathic recurrent pregnancy loss. Collectively, we have revealed activation of a dynamic regulatory network that provides a framework for understanding EVT cell specification in trophoblast cell lineage development and human placentation.',\n",
       "  'ct27 evt source human trophoblast stem cells cell line extravillous',\n",
       "  'ct27 stem source human trophoblast cells cell line state',\n",
       "  'human'),\n",
       " 'GSE110146-4-vs-9-human up': (['Transcriptomic-based functional characterization of adverse events following lymphatic filariasis treatment'],\n",
       "  'Intro: Lymphatic filariasis (LF) is a neglected tropical disease caused by the nematode parasite Wuchereria bancrofti. The primary tool used by the Global Program to Eliminate LF is mass drug administration (MDA), and some 500 million people take the medications each year. Mild to moderate adverse events (AEs) are common after LF treatment, and these pose a challenge for the LF elimination program. To better understand the pathogenesis of AEs, we studied patients from a LF treatment trial in Côte d’Ivoire.',\n",
       "  'post buffy coat adverse event none pre age sex male village microfilaria/ml arm ivm/alb',\n",
       "  'pre buffy coat adverse event moderate post age sex male village microfilaria/ml arm',\n",
       "  'human'),\n",
       " 'GSE212873-0-vs-1-human dn': (['Fibroblast Growth Factor-2 bound to specific dermal fibroblasts-derived extracellular vesicles is protected from degradation: relevance to wound healing'],\n",
       "  'Fibroblast growth factor-2 (FGF2) has multiples roles in cutaneous wound healing but its natural low stability prevents the development of its use in therapeutic for impaired skin repair. Here we show that FGF2 binds the outer surface of dermal fibroblast (DF)-derived extracellular vesicles (EVs) and this association protects FGF2 from fast degradation. We performed RNA-seq analysis after EV treatment to examine modulated genes in DFs.',\n",
       "  'igg skin samples primary human dermal fibroblasts (df) donor fgf treated cells antibody',\n",
       "  'afgf skin samples primary human dermal fibroblasts (df) donor exosome treated cells fgf antibody',\n",
       "  'human'),\n",
       " 'GSE120299-2-vs-1-human dn': (['Human Pancreatic Islets Expressing HNF1A Variant Have Defective β cell Transcriptional Regulatory Networks'],\n",
       "  'Using an integrated approach to characterize the pancreatic tissue and isolated islets from a 33-year-old with 17 years of type 1 diabetes (T1D), we found donor islets contained β cells without insulitis and lacked glucose-stimulated insulin secretion despite a normal insulin response to cAMP-evoked stimulation. With these unexpected findings for T1D, we sequenced the donor DNA and found a pathogenic heterozygous variant in hepatocyte nuclear factor 1 alpha (HNF1A). In one of the first studies of human pancreatic islets with a disease-causing HNF1A variant associated with the most common form of monogenic diabetes, we found that HNF1A dysfunction leads to insulin-insufficient diabetes reminiscent of T1D by impacting the regulatory processes critical for glucose-stimulated insulin secretion and suggest a rationale for a therapeutic alternative to current treatment.',\n",
       "  'control [beta cells] pancreatic beta cells age gender disease state non diabetic cadaveric islet',\n",
       "  'control [alpha cells] facs purified alpha cells human pancreatic islets age donor years old gender disease state non diabetic',\n",
       "  'human'),\n",
       " 'GSE73721-0-vs-5-human dn': (['RNA-Seq of human astrocytes'],\n",
       "  \"Astrocytes were purified from fetal and adult human brain tissue using an immunopanning method with the HepaCAM antibody. Samples were taken from otherwise 'healthy' pieces of tissue, unless otherwise specified.\",\n",
       "  'ctx myeloid age years old cell subtype cd45 positive cells adult temporal lobe',\n",
       "  'hippocampus astro age years old cell subtype hepacam positive cells astrocyte',\n",
       "  'human'),\n",
       " 'GSE204665-4-vs-11-human up': (['The unexpected role of HBx in UC'],\n",
       "  'Ulcerative colitis is a chronic inflammatory disorder for which a definitive cure is still missing. This is characterized by an overwhelming inflammatory milieu in the colonic tract where a composite set of immune and non-immune cells orchestrate its pathogenesis. Over the last years, a growing body of evidence has been pinpointing gut virome dysbiosis as underlying its progression. Nonetheless, its role during the early phases of chronic inflammation is far from being fully defined. Here we show the gut virome-associated Hepatitis B virus protein X, most likely acquired after an event of zoonotic spillover, to be associated with the early stages of ulcerative colitis and to induce colonic inflammation in mice. It acts as a transcriptional regulator in epithelial cells, provoking barrier leakage and altering mucosal immunity at the level of both innate and adaptive immunity. This study paves the way to the comprehension of the aetiopathogenesis of intestinal inflammation and encourages further investigations of the virome as a trigger also in other scenarios. Moreover, it provides a brand-new standpoint that looks at the virome as a target for tailored treatments, blocking the early phases of chronic inflammation and possibly leading to better disease management.',\n",
       "  'post facs lpmc colon dendritic cells',\n",
       "  'post facs en lpmc colon endothelium',\n",
       "  'human'),\n",
       " 'GSE202109-1-vs-2-human up': (['Single-cell profiling of healthy human kidney reveals features of sex-based transcriptional programs and tissue-specific immunity'],\n",
       "  'We present a single-cell perspective of healthy human kidney from 19 living donors, with equal contribution from males and females, profiling the transcriptome of 27677 high-quality cells to map healthy kidney at high resolution. Our sex-balanced dataset revealed sex-based differences in gene expression within proximal tubular cells, specifically, increased anti-oxidant metallothionein genes in females and the predominance of aerobic metabolism-related genes in males. Using CD45+ enrichment in 10 samples, we profile the immune niche of the kidney, identifying kidney-specific lymphocyte populations with unique transcriptional profiles indicative of kidney-adapted functions. We observed significant heterogeneity in resident myeloid populations and identified an MRC1+ LYVE1+ FOLR2+ C1QC+ population as the predominant myeloid population in healthy kidney. This study provides a broad cellular map of healthy human kidney, revealing novel insights into the complexity of renal parenchymal cells and kidney-resident immune populations.',\n",
       "  'group total homogenate kidney core biopsy sex age',\n",
       "  'cd45 group enriched kidney core biopsy sex female age',\n",
       "  'human'),\n",
       " 'GSE146782-2-vs-0-mouse dn': (['The role of IL-33 on red pulp macrophage (RPM) development'],\n",
       "  'This study investigated the effects of IL1RL1 deficiency on the devlopment of red pulp macrophages (RPM)',\n",
       "  'splenocytes wtr red pulp macrophage strain balb/c wild type sorting ly6glo cd11blo/ f4/80hi cells rpm mouse',\n",
       "  'splenocytes wtm monocyte strain balb/c wild type sorting ly6glo cd11bhi f4/80 cells splenic monocytes mouse',\n",
       "  'mouse'),\n",
       " 'GSE103001-4-vs-6-human dn': (['Stranded RNASeq of human mammary primary tumors ER+ and paired adjacent healthy tissues'],\n",
       "  'Non-coding RNAs (ncRNA) represent at least 1/5 of the mammalian transcript amount, and about 90% of the genome length is actively transcribed. Many ncRNAs have been demonstrated to play a role in cancer. Among them, natural antisense transcripts (NAT) are RNA sequences which are complementary and overlapping to those of protein-coding transcripts (PCT). NATs were punctually described as regulating gene expression, and are expected to act more frequently in cis than other ncRNAs that commonly function in trans. In this work, 22 breast cancers expressing estrogen receptors and their paired healthy tissues were analyzed by strand-specific RNA sequencing to highlight the potential role of NATs in gene regulations occurring in breast cancer.',\n",
       "  'pat 13 normal nature tumor receptors / bloom er subtype tnm mammary',\n",
       "  'pat 14 tumor nature primary invasive breast carcinoma receptors er+/her2 bloom er subtype tnm mammary',\n",
       "  'human'),\n",
       " 'GSE256003,GSE256012-0-vs-1-human up': (['An endogenous retrovirus regulates tumor-specific expression of the immune transcriptional regulator SP140 [RNA-seq]'],\n",
       "  'Speckled Protein 140 (SP140) is a chromatin reader with critical roles regulating immune cell transcriptional programs, and SP140 splice variants are associated with immune diseases including Crohn’s disease, multiple sclerosis, and chronic lymphocytic leukemia. SP140 expression is currently thought to be restricted to immune cells. However, by analyzing human transcriptomic datasets from a wide range of normal and cancer cell types, we found recurrent cancer-specific expression of SP140, driven by an alternative intronic promoter derived from an intronic endogenous retrovirus (ERV). The ERV belongs to the primate-specific LTR8B family. The ERV drives expression of multiple cancer-specific isoforms, including a nearly full-length isoform that retains all the functional domains of the full-length canonical isoform and is also localized within the nucleus, consistent with a role in chromatin regulation. In a fibrosarcoma cell line, silencing the cancer-specific ERV promoter of SP140 resulted in increased sensitivity to interferon-mediated cytotoxicity and dysregulation of multiple genes. Our findings implicate aberrant ERV-mediated SP140 expression as a novel mechanism contributing to immune gene dysregulation in a wide range of cancer cells.',\n",
       "  'sp140 kd untreated cell line ht1080 dcas9 crispri ltr8b.sp140',\n",
       "  'sp140 kd ifnb cell line ht1080 dcas9 crispri ltr8b.sp140',\n",
       "  'human'),\n",
       " 'GSE57973,GSE79469-1-vs-5-human up': (['Age-dependent gene expression changes in human islets'],\n",
       "  'We report the transcriptomic profiling of islet cell subtypes obtained from young and old human donors.',\n",
       "  'beta pancreas age group old',\n",
       "  'alpha pancreas age group old',\n",
       "  'human'),\n",
       " 'GSE118373-1-vs-3-human up': (['High-throughput sequencing of the human hepatic progenitor cell niche in PSC and HCV.'],\n",
       "  'This study identified hepatic progenitor cell (HPC) niche-associated signalling pathways relevant in different chronic liver diseases using a high-throughput sequencing approach. The HPC niche was isolated using laser microdissection from patient samples diagnosed with hepatitis C virus (HCV) or primary sclerosing cholangitis (PSC), as models for hepatocellular or biliary regeneration.  Differentially expressed genes in the HPC niche of PSC and HCV correlated to pathways involved in immune signalling, fibrogenesis and angiogenesis.',\n",
       "  'psc hpc disease state primary sclerosing cholangitis hepatic progenitor cells',\n",
       "  'psc hep disease state primary sclerosing cholangitis hepatocytes',\n",
       "  'human'),\n",
       " 'GSE125007,GSE125008-6-vs-4-mouse up': (['Genome-wide expression profiling of Uhrf1hepKO mouse livers at baseline and following PHx'],\n",
       "  'This study was carried out to investigate the mechanism behind the enhanced liver regenerative response observed in Uhrf1hepKO mice. Livers was collected at baseline and at 24hrs, 30hrs, 40hrs, 48hrs, 96hrs, 7days and 4 weeks following 2/3 partial hepatectomy surgery (PHx). These timepoints correspond to key cell cycle events that occur following PHx as established in literature.',\n",
       "  'wt phx 4wks rep strain c57bl/6 liver time mouse',\n",
       "  'wt phx 40hrs rep strain c57bl/6 liver time mouse',\n",
       "  'mouse'),\n",
       " 'GSE200130,GSE200133-9-vs-5-human up': (['Revealing and harnessing CD39 for the treatment of colorectal cancer and liver metastases by engineered T cells [CRC]'],\n",
       "  'Colorectal tumors are often densely infiltrated by immune cells that have a role in surveillance and modulation of tumor progression but are burdened by exhaustion mechanisms fueled by the tumor microenvironment, which must be counteracted to achieve control. Here, we deployed a multidimensional approach to unravel the T cell functional landscape in tumor and peritumoral tissues from primary colorectal cancers and liver metastases. We found that T-cells are mainly localized at the front edge and that tumor-infiltrating T cells co-express multiple inhibitory receptors and highlighted CD39 as the major driver of exhaustion in both primary and metastatic colorectal tumors. By CRISPR/Cas9 genome editing tools, we simultaneously redirected T-cell specificity employing a novel T-cell receptor targeting the HER-2 antigen, and disrupted CD39, thus generating triple-knockout engineered lymphocytes. We showed that the absence of CD39 confers HER2-specific T cells a functional advantage in eliminating HER2+ patient-derived organoids, starring the relevance of the CD39 axis for further exploitation in adoptive T-cell therapy strategies to treat primary and metastatic colorectal cancer.',\n",
       "  'primary colon type peritumor colorectal cancer',\n",
       "  'primary colon type tumor colorectal cancer',\n",
       "  'human'),\n",
       " 'GSE86064-10-vs-7-human dn': (['Mandatory Role of HMGA1 in Human Airway Epithelial Normal Differentiation and Post-injury Regeneration'],\n",
       "  'Background: High mobility group AT-hook1 (HMGA1) is essential for airway basal cell mucociliary differentiation, barrier integrity and wound repair.  HMGA1 expression suppresses the abnormal basal cell differentiation to squamous, inflammatory and epithelial-mesenchymal transition phenotype commonly observed in association with cigarette smoking and chronic obstructive pulmonary disease (COPD).',\n",
       "  'dgm sm smoking status nonsmoker small airway epithelial cells',\n",
       "  'dgm lae bc smoking status nonsmoker large airway basal cells cultured epithelium',\n",
       "  'human'),\n",
       " 'GSE82043-0-vs-5-mouse up': (['Itaconate links inhibition of succinate dehydrogenase with macrophage metabolic remodeling and regulation of inflammation'],\n",
       "  'Remodeling of the tricarboxylic acid (TCA) cycle is a metabolic adaptation mechanism accompanying inflammatory macrophage activation. During this process, endogenous metabolites can adopt regulatory roles that govern specific aspects of inflammatory response, as recently shown for succinate, which regulates the downstream pro-inflammatory IL-1β-HIF1a axis. Itaconate is one of the most highly induced metabolites in activated macrophages, yet its functional significance remains unknown. Here, we show that itaconate modulates macrophage metabolism and effector functions via its effect on succinate dehydrogenase, by inhibiting conversion of succinate to fumarate. Through this action, itaconate exerts anti-inflammatory effects when administered in vitro and in vivo during macrophage activation and ischemia-reperfusion injury. Using newly generated Irg1-/- mice, which lack the ability to produce itaconate, we show that endogenous itaconate regulates succinate levels and function, changes in mitochondrial respiration, and inflammatory cytokine production during macrophage activation. These studies highlight itaconate as a major physiological regulator of the global metabolic rewiring and effector functions of inflammatory macrophages.',\n",
       "  'exp1 unst strain background c57bl/6 /variation wt bone marrow derived macrophages treated unstimulated',\n",
       "  'exp2 irg1ko lps ifng strain background c57bl/6 /variation irg1 / bone marrow derived macrophages treated 24h',\n",
       "  'mouse'),\n",
       " 'GSE161916,GSE161917-5-vs-6-human up': (['Interferon-regulated genetic programs and JAK/STAT pathway activate the intronic promoter of the short ACE2 isoform in renal proximal tubules [RNA-seq]'],\n",
       "  'Angiotensin-Converting Enzyme 2 (ACE2) binds SARS-CoV-2 spike protein and facilitates viral entry into the cell. ACE2 is expressed in the kidney and renal injury strongly impacts COVID-19 prognosis. In addition, a short, interferon-inducible isoform of ACE2, dACE2 was recently identified. Change in ACE2 expression has already been linked to several human nephropathies. However, these changes were not analyzed in context of dACE2 and regulation of ACE2 expression in the kidney remains unclear. Human primary Renal Proximal Tubule cells were used for all experiments. ChIP-seq analysis of histone modifications (H3K27ac, H3K4me3, K3K4me1), RNA polymerase loading and DNAse hypersensitive sites (DHS) were used to identify gene regulatory elements in the ACE2 locus. qRT-PCR was used to assess mRNA expression of ACE2, dACE2, TMPRSS2 and STAT1. RNA-seq was used to identify changes in global gene expression after cytokine treatment. Putative regulatory elements controlling dACE2 expression were identified using ChIP-seq profiles and RNA-seq mapping. qRT-PCR differentiating between ACE2 and dACE2 revealed that only dACE2 is interferon inducible. dACE2 mRNA was upregulated 300- and 600-fold by IFNα and IFNβ, respectively, while full length ACE2 was unchanged. Interferon stimulation also resulted in elevated STAT1 expression. JAK inhibitor ruxolitinib ablated both STAT1 and dACE2 expression after interferon treatment. RNA-seq analysis revealed approximately 1200 genes induced by IFNβ, largely innate immune response-related. We showed that human primary proximal tubules express ACE2 and its interferon-inducible short isoform, dACE2, from an intronic promoter. We also deliver new datasets identifying cytokine response genes in proximal tubule cells.',\n",
       "  'rna seq hppt untreated human primary proximal tubule cells',\n",
       "  'rna seq hppt ifnb human primary proximal tubule cells',\n",
       "  'human'),\n",
       " 'GSE185245-1-vs-10-human dn': (['TEFM variants impair mitochondrial transcription elongation causing childhood-onset neurological disease'],\n",
       "  \"We report that TEFM variants are associated with mitochondrial respiratory chain deficiency. We show that patient's cells have reduced levels of promoter distal mitochondrial RNA transcripts.\",\n",
       "  'muscle control subject status/id / molecule subtype rrna depleted rna',\n",
       "  'rep1dep subject status/id / fibroblasts molecule subtype rrna depleted rna',\n",
       "  'human'),\n",
       " 'GSE239662,GSE239663-2-vs-1-human dn': (['Novel method of isolating nuclei of human oligodendrocyte precursor cells reveals substantial developmental changes in gene expression and H3K27ac histone modification. [RNA-Seq]'],\n",
       "  'Oligodendrocyte precursor cells (OPCs) generate differentiated mature oligodendrocytes (MOs) during development. In adult brain, OPCs replenish MOs in adaptive plasticity, neurodegenerative disorders, and after trauma. The ability of OPCs to differentiate to MOs decreases with age and is compromised in disease. Here we explored the cell specific and age-dependent differences in gene expression and H3K27ac histone mark in these two cell types. H3K27ac is indicative of active promoters and enhancers. We developed a novel flow-cytometry-based approach to isolate OPC and MO nuclei from human postmortem brain and profiled gene expression and H3K27ac in adult and infant OPCs and MOs genome-wide.In adult brain, we detected extensive H3K27ac differences between the two cell types with high concordance between gene expression and epigenetic changes. Notably, the expression of genes that distinguish MOs from OPCs appears to be under a strong regulatory control by the H3K27ac modification in MOs but not in OPCs. Comparison of gene expression and H3K27ac between infants and adults uncovered numerous developmental changes in each cell type, which were linked to several biological processes, including cell proliferation and glutamate signaling. A striking example was a subset of histone genes that were highly active in infant samples but fully lost activity in adult brain. Our findings demonstrate a considerable rearrangement of the H3K27ac landscape that occurs during the differentiation of OPCs to MOs and during postnatal development of these cell types, which aligned with changes in gene expression. The uncovered regulatory changes justify further in-depth epigenetic studies of OPCs and MOs in development and disease.',\n",
       "  'subject mature oligodendrocytes rna seq dorsolateral prefrontal cortex donor id disease control sex female',\n",
       "  'subject oligodendrocyte precursor cells rna seq dorsolateral prefrontal cortex donor id disease control sex female',\n",
       "  'human'),\n",
       " 'GSE239662,GSE239663-1-vs-2-human up': (['Novel method of isolating nuclei of human oligodendrocyte precursor cells reveals substantial developmental changes in gene expression and H3K27ac histone modification. [RNA-Seq]'],\n",
       "  'Oligodendrocyte precursor cells (OPCs) generate differentiated mature oligodendrocytes (MOs) during development. In adult brain, OPCs replenish MOs in adaptive plasticity, neurodegenerative disorders, and after trauma. The ability of OPCs to differentiate to MOs decreases with age and is compromised in disease. Here we explored the cell specific and age-dependent differences in gene expression and H3K27ac histone mark in these two cell types. H3K27ac is indicative of active promoters and enhancers. We developed a novel flow-cytometry-based approach to isolate OPC and MO nuclei from human postmortem brain and profiled gene expression and H3K27ac in adult and infant OPCs and MOs genome-wide.In adult brain, we detected extensive H3K27ac differences between the two cell types with high concordance between gene expression and epigenetic changes. Notably, the expression of genes that distinguish MOs from OPCs appears to be under a strong regulatory control by the H3K27ac modification in MOs but not in OPCs. Comparison of gene expression and H3K27ac between infants and adults uncovered numerous developmental changes in each cell type, which were linked to several biological processes, including cell proliferation and glutamate signaling. A striking example was a subset of histone genes that were highly active in infant samples but fully lost activity in adult brain. Our findings demonstrate a considerable rearrangement of the H3K27ac landscape that occurs during the differentiation of OPCs to MOs and during postnatal development of these cell types, which aligned with changes in gene expression. The uncovered regulatory changes justify further in-depth epigenetic studies of OPCs and MOs in development and disease.',\n",
       "  'subject oligodendrocyte precursor cells rna seq dorsolateral prefrontal cortex donor id disease control sex female',\n",
       "  'subject mature oligodendrocytes rna seq dorsolateral prefrontal cortex donor id disease control sex female',\n",
       "  'human'),\n",
       " 'GSE220806-1-vs-3-human up': (['Characteristic impairment of progesterone response in the cultured cervical fibroblast obtained from patients with refractory cervical insufficiency'],\n",
       "  'We compared the gene expression in response to progesterone (1μM) in human cervical fibroblasts.',\n",
       "  'control p4 2 condition progesterone treated cells derived unripend cervix human uterine cervical fibroblast passage 4 8',\n",
       "  'ci p4 condition progesterone treated cells derived cervical insufficiency human uterine fibroblast passage 4 8',\n",
       "  'human'),\n",
       " 'GSE73721-0-vs-3-human dn': (['RNA-Seq of human astrocytes'],\n",
       "  \"Astrocytes were purified from fetal and adult human brain tissue using an immunopanning method with the HepaCAM antibody. Samples were taken from otherwise 'healthy' pieces of tissue, unless otherwise specified.\",\n",
       "  'ctx myeloid age years old cell subtype cd45 positive cells adult temporal lobe',\n",
       "  'ctx oligo age years old cell subtype galc positive cells oligodendrocyte adult temporal lobe',\n",
       "  'human'),\n",
       " 'GSE154919-5-vs-7-human up': (['Aneuploid cells activate NF-κB to promote their immune clearance by NK cells'],\n",
       "  'The immune system plays a major role in the protection against cancer. Identifying and characterizing the pathways mediating this immune surveillance is thus critical for understanding how cancer cells are recognized and eliminated. Aneuploidy is a hallmark of cancer and we previously found that untransformed cells that had undergone senescence due to highly abnormal karyotypes are eliminated by Natural Killer (NK) cells in vitro. However, the mechanisms underlying this process remained elusive. Here we show that NK-mediated immune clearance of aneuploid cells is predominantly mediated by non-cell autonomous mechanisms. Our data indicate that NF-κB signaling in aneuploid cells is central to elicit this immune response. Inactivating NF-κB abolishes NK-cell mediated clearance of untransformed aneuploid cells in vitro. In cancer cell lines, NF-κB activation correlates with degree of aneuploidy, raising the possibility that aneuploidy-induced immune recognition is partially retained in cancer.',\n",
       "  'rpe1 htert nutlin3 treated cells population bulk culture',\n",
       "  'cycling stat1 ko single cell clone rpe1 htert population cells',\n",
       "  'human'),\n",
       " 'GSE148822-2-vs-14-human dn': ([\"Distinct amyloid-b and tau associated microglia profiles in Alzheimer's disease\"],\n",
       "  'Alzheimer’s disease (AD) is the most prevalent form of dementia and is characterized by abnormal extracellular aggregates of amyloid-b and intraneuronal hyperphosphorylated, tau tangles and neuropil threads. Microglia, the tissue-resident macrophages of the central nervous system (CNS), are important for CNS homeostasis and implicated in AD pathology. In amyloid mouse models, a phagocytic/activated microglia phenotype has been identified. How increasing levels of amyloid-b and tau pathology affect human microglia transcriptional profiles is unknown. Here, we performed snRNAseq on 482,472 nuclei from non-demented control brains and AD brains containing only amyloid-b plaques or both amyloid-b plaques and tau pathology. Within the microglia population, distinct expression profiles were identified of which two were AD pathology-associated. The phagocytic/activated AD1-microglia population abundance strongly correlated with tissue amyloid-b load and localized to amyloid-b plaques. The AD2-microglia abundance strongly correlated with tissue phospho-tau load and these microglia were more abundant in samples with over tau pathology. This full characterization of human disease associated microglia phenotypes provides new insights in the pathophysiological role of microglia in AD and offers new targets for microglia-state-specific therapeutic strategies.',\n",
       "  'bulk olig brain library type rna sequencing nuclei population olig2pos region occipitotemporal cortex age gender f clinical diagnosis ad sample group rin braak/abc score',\n",
       "  'bulk neun brain library type rna sequencing nuclei population neunpos region occipitotemporal cortex age gender clinical diagnosis ad sample group rin braak/abc score',\n",
       "  'human'),\n",
       " 'GSE123813,GSE123814-0-vs-1-human up': (['Clonal replacement of tumor-specific T cells following PD-1 blockade [single cells]'],\n",
       "  'Immunotherapies that block inhibitory checkpoint receptors on T cells have transformed the clinical care of cancer patients. However, the clonal origin of tumor-specific T cells following checkpoint blockade in patients remains unclear. Here, we performed paired single-cell RNA- and T cell receptor (TCR)- sequencing on site-matched tumors from patients with basal cell carcinoma (BCC) pre- and post-anti-PD-1 therapy. Tracking TCR clonotypes and transcriptome phenotypes revealed a coupling of tumor-recognition, clonal expansion, and T cell dysfunction: the response to treatment was accompanied by a clonal expansion of CD8+CD39+ T cells, which co-expressed markers of chronic T cell activation and exhaustion. However, this response was not accompanied by an expansion of pre-existing tumor-specific T cell clonotypes; rather, expanded T cell clones post-therapy comprised novel clonotypes, which were not previously observed in the same tumor. Clonal replacement of T cells was preferentially observed in tumor-specific exhausted CD8+ T cells, compared to other distinct T cell phenotypes, and was more evident in patients who exhibited a clinical response to treatment. These results, enabled by single-cell multi-omic profiling of clinical samples, demonstrate that the chronic activation of pre-existing tumor-infiltrating T cells may limit their re-invigoration following checkpoint blockade, and that a successful response relies on the expansion of a distinct repertoire of tumor-specific T cell clones.',\n",
       "  'tumor cd45 rna donor pre cd3 basal cell carinoma',\n",
       "  'tcell rna donor cd45+ cd3+ squamous cell carcinoma tumor',\n",
       "  'human'),\n",
       " 'GSE146782-0-vs-5-mouse up': (['The role of IL-33 on red pulp macrophage (RPM) development'],\n",
       "  'This study investigated the effects of IL1RL1 deficiency on the devlopment of red pulp macrophages (RPM)',\n",
       "  'splenocytes wtm monocyte strain balb/c wild type sorting ly6glo cd11bhi f4/80 cells splenic monocytes mouse',\n",
       "  'splenocytes kor red pulp macrophage strain balb/c il1rl1 / sorting ly6glo cd11blo/ f4/80hi cells rpm mouse',\n",
       "  'mouse'),\n",
       " 'GSE212873-0-vs-3-human dn': (['Fibroblast Growth Factor-2 bound to specific dermal fibroblasts-derived extracellular vesicles is protected from degradation: relevance to wound healing'],\n",
       "  'Fibroblast growth factor-2 (FGF2) has multiples roles in cutaneous wound healing but its natural low stability prevents the development of its use in therapeutic for impaired skin repair. Here we show that FGF2 binds the outer surface of dermal fibroblast (DF)-derived extracellular vesicles (EVs) and this association protects FGF2 from fast degradation. We performed RNA-seq analysis after EV treatment to examine modulated genes in DFs.',\n",
       "  'igg skin samples primary human dermal fibroblasts (df) donor fgf treated cells antibody',\n",
       "  'afgf skin samples primary human dermal fibroblasts (df) donor cells fgf antibody',\n",
       "  'human'),\n",
       " 'GSE117221-9-vs-10-human dn': (['Gender-specific transcriptional profiles identified in β-thalassemia patients'],\n",
       "  'Lifelong treatment and major complications characterize β-thalassemia patients, whether they suffer from the milder type of the disease (β-thalassemia intermedia, TI) or the severe type (β-thalassemia major, TM). This study aimed to ascertain the differences in the transcriptional profiles of TI and TM patients to identify targets for stratification and the development of personalized therapeutic strategies. RNA-seq of 49 participants (16 TI, 16 TM, 17 healthy) identified a higher number of differentially expressed genes in TM than in TI patients when compared to healthy participants portraying accurately the clinical observations. When TI patients were compared to TM, the absence of significantly differentially expressed genes highlighted the increased variability seen in the TI expression profiles suggesting that a continuous spectrum describes the disease and not distinct conditions. Although no confounding effect was seen due to different centers, age groups and various clinical characteristics, when male and female patients were compared to healthy participants of the same gender, it was evident that males and females are affected in distinctive ways by β-thalassemia. TI triggers a pronounced and consistent transcriptional signature in males, whereas a small effect is seen in females reflecting increased biological variability. TM induces transcriptional changes of similar magnitude in both genders with limited overlap between them emphasizing that different pathways are affected. Our study provides a framework for further stratification studies in β-thalassemia and highlights the need to consider gender as an important variable for developing differential diagnostic and therapeutic strategies.',\n",
       "  'sample subject status thalassemia gender male peripheral blood erythroid precursor cells (erpcs) erpcs',\n",
       "  'sample subject status thalassemia intermedia gender female peripheral blood erythroid precursor cells (erpcs) erpcs',\n",
       "  'human'),\n",
       " 'GSE151113-1-vs-2-human dn': (['Effects on imidazoacridinone C-1311 on the transcriptomic profile in prostate cancer cell depending on their AR status.'],\n",
       "  'Human prostate cancer cells, LNCaP (AR+) and DU-145 (AR-), were treated with C-1311 to identify the transcriptomic profile and functionally relevant androgen-dependent and independent genes using RNA sequencing.',\n",
       "  'lncap control rep disease carcinoma morphology epithelial untreated cell line prostate cancer',\n",
       "  'lncap c 1311 treated rep disease carcinoma morphology epithelial 24 hours cell line prostate cancer',\n",
       "  'human'),\n",
       " 'GSE86064-10-vs-3-human dn': (['Mandatory Role of HMGA1 in Human Airway Epithelial Normal Differentiation and Post-injury Regeneration'],\n",
       "  'Background: High mobility group AT-hook1 (HMGA1) is essential for airway basal cell mucociliary differentiation, barrier integrity and wound repair.  HMGA1 expression suppresses the abnormal basal cell differentiation to squamous, inflammatory and epithelial-mesenchymal transition phenotype commonly observed in association with cigarette smoking and chronic obstructive pulmonary disease (COPD).',\n",
       "  'dgm sm smoking status nonsmoker small airway epithelial cells',\n",
       "  'dgm lae bc smoking status copd smoker large airway basal cells cultured epithelium',\n",
       "  'human'),\n",
       " 'GSE223100,GSE223842-6-vs-10-mouse up': (['pDC-like cells are pre-DC2 and require Klf4 to control homeostatic CD4 T cells'],\n",
       "  'Plasmacytoid dendritic cells (pDCs) have been shown to play an important role during immune responses, ranging from initial viral control through the production of type I interferons (IFNs) to antigen presentation. However, recent evidence uncovered unexpected heterogeneity among pDCs. Notably we identified a new subset, referred to as pDC-like cells, that resembles pDCs but shares cDC features. Here we show that this subset is a circulating progenitor distinct from common DC progenitors (CDP), with prominent cDC2 precursor potential. This precursor subset responds to homeostatic cytokines, such as macrophage colony stimulating factor (M-CSF) by expanding and differentiating into cDC2 that efficiently prime Th17 cell. Development of pDC-like cells into CX3CR1+ESAM- cDC2b but not CX3CR1- ESAM+ cDC2a requires the transcription factor KLF4. Finally, we show that under homeostatic conditions this developmental pathway regulates the immune threshold at barrier sites, by controlling the pool of Th17 cells within the skin draining lymph nodes.',\n",
       "  'spl pdc like spleen (spl) replicate',\n",
       "  'bm pdc like bone marrow (bm) replicate',\n",
       "  'mouse'),\n",
       " 'GSE188477-0-vs-1-human dn': (['Variant specific global loss of cellular m6A RNA methylation during SARS-CoV-2 infection'],\n",
       "  'COVID-19 pandemic caused by SARS-CoV-2 is far from getting over because of the emergence of different SARS-CoV-2 variants even though vaccination against SARS-CoV-2 is ongoing. Variant specific mechanistic understanding of host-viral interaction will be required to design novel anti-viral therapeutics. N6-methyladenosine modification (m6A) which is one of the abundant cellular RNA modifications regulates key processes in RNA metabolism during stress response.  We observed different SARS-CoV-2 variants during infection causes global loss of m6A in cellular RNA whereas m6A modification was detected abundantly in viral RNA. The loss of m6A in cellular RNAs was more drastic in B.1 and  B1.1.7 strains and the main m6A methyltransferase METTL3 shows cytoplasmic localization post-infection. In B1.351 variant effect on METTL3 localization and loss of m6A was less pronounced compared to B.1 and  B1.1.7  variants. Gene expression profile suggested changes in RNA catabolism-related pathways including key genes related to m6A readers and erasers during SARS-CoV-2 infection. Genes with m6A peaks were more susceptible to down-regulation during viral infection. Inhibition of export protein XPO1 during infection results in restoration of METTL3 nuclear localization and reduction in viral infection in cell culture suggesting XPO1 inhibition could be an effective therapeutic option for SARS-CoV-2.',\n",
       "  'noninfected human bronchial epithelium antibody none non infected cells dpi) spike organism escherichia coli k 12',\n",
       "  'sars cov 2 infected human epithelium antibody none cells spike organism escherichia coli k 12',\n",
       "  'human'),\n",
       " 'GSE129770-3-vs-4-mouse up': (['PS2APP sorted cell types'],\n",
       "  'Mice of indicated genotypes were perfused and their brains dissected and dissociated. Cells were fixed, immunolabeled and FACS sorted. RNA was extracted from neuron, astrocyte, and microglial cell populations. Typical RIN=4-5 for neurons, 6-8 for astrocytes, and 5-7 for microglia. Typical RNA yields 200-300ng for neurons, ~20ng for microglia, and ~5ng for astrocytes.',\n",
       "  'non tg cd11b sample id transgenic cd11b+ myeloid cells age 11.5 month old',\n",
       "  'non tg neun sample id transgenic neun+ neurons age 11.5 month old',\n",
       "  'mouse'),\n",
       " 'GSE163548-8-vs-10-human up': (['Functional genomics atlas of synovial fibroblasts defining rheumatoid arthritis heritability'],\n",
       "  'Genome-wide association studies have reported more than 100 risk loci for rheumatoid arthritis, and there loci have been shown to be enriched in immune cell-specific enhancers, we add Synovial fibroblasts (FLS), the local stromal cells of joints to this analysis.  We integrated ChIP-seq, Hi-C, Capture Hi-C, ATAC-seq and RNA-seq datasets from FLS , with genetic fine-mapping of RA loci. From this we identified putative casual variants, enhancers and genes for 30-60% of RA loci and demonstrated that FLS regulatory elements accounts for up to 24% of RA heritability.  We also examined the effects of TNF stimulation on the chromatin landscape of FLS, and we found there are significant alterations in the organization of topologically associated domains, chromatin interactions and the expression of several putative causal genes.',\n",
       "  'rna seq primary fls unstimulated sample id stimulation none joint disease state rheumatoid arthritis',\n",
       "  'tnf rna seq primary fls stimulated sample id stimulation 10 ng/ml 24 hours joint shoulder disease state rheumatoid arthritis',\n",
       "  'human'),\n",
       " 'GSE157785-1-vs-6-human up': (['RNA-seq analysis of cells derived from breast milk'],\n",
       "  'We have developed a method to grow breast epithelial cells released into breast milk. RNA from cells of left and right breasts milk were subjected to RNA sequencing in triplicate',\n",
       "  'breast milk derived epithelial cells location left',\n",
       "  'm4r breast milk derived epithelial cells location right',\n",
       "  'human'),\n",
       " 'GSE221240-4-vs-2-human dn': (['Characterization of cell-type specific circular RNAs associated with colorectal cancer metastasis'],\n",
       "  'Colorectal cancer (CRC) is the most common gastrointestinal malignancy in the U.S.A. and approximately 50% of CRC patients develop metastatic disease (mCRC) with an average 5-year survival rate of 13%. Despite our understanding of circular RNAs (circRNAs) in primary cancer, their role throughout mCRC remains poorly characterized. Further, despite the availability of bulk RNA-Seq data for monitoring circRNA expression little is known about their cell-type specific expression profiles to fully understand their functions between tumor cells and the microenvironment. To evaluate circRNAs in mCRC progression, total RNA-Seq was performed on 30 matched normal, primary, and distant metastatic samples from 14 mCRC patients. Additionally, the transcriptome of five CRC cell lines were sequenced to construct a circRNA catalog in CRC. Differential expression (DE) analysis identified circRNAs dysregulated between normal, primary, and metastatic tissues. Differential backsplicing (DS) analysis was used to highlight circRNAs with an altered tendency in backsplicing as compared to their linear counterparts. Cell-type deconvolution was performed using published CRC single-cell RNA-Seq datasets as signature matrices. The cell-type specific expression of circRNAs was then estimated through a non-negative least squares statistical model.',\n",
       "  'normal liver subject sample id',\n",
       "  'normal colon mucosa subject sample id',\n",
       "  'human'),\n",
       " 'GSE227592-5-vs-0-human up': (['ADAR1 biology can hinder effective antiviral RNA interference'],\n",
       "  'Viruses constantly evolve and adapt to the antiviral defenses of their hosts. The biology of viral circumvention of these selective pressures can often be attributed to the acquisition of novel antagonistic gene products or by rapid genome change that prevents host recognition. To study viral evasion of RNA interference (RNAi)-based defenses, we established a robust antiviral system in mammalian cells using recombinant Sendai virus designed to be targeted by endogenous host microRNAs (miRNAs) with perfect complementarity. Using this system, we previously demonstrated the intrinsic ability of positive-strand RNA viruses to escape this selective pressure via homologous recombination, which was not observed in negative-strand RNA viruses. Here we show that given extensive time, escape of miRNA-targeted Sendai virus was enabled by host adenosine deaminase acting on RNA 1 (ADAR1). Independent of the viral transcript targeted, ADAR1 editing resulted in disruption of the miRNA-silencing motif, suggesting an intolerance for extensive RNA:RNA interactions necessary for antiviral RNAi. This was further supported in N. benthamiana, where exogenous expression of ADAR1 interfered with endogenous RNAi. Together, these results suggest that ADAR1 diminishes the effectiveness of RNAi and may explain why it is absent in species that utilize this antiviral defense system.',\n",
       "  'wt a549 amplicon seq rsev n5t cell line lung carcinoma epithelial cells (moi=5)',\n",
       "  'wt a549 rsev n5t cell line lung carcinoma epithelial cells (moi=5)',\n",
       "  'human'),\n",
       " 'GSE86064-3-vs-13-human up': (['Mandatory Role of HMGA1 in Human Airway Epithelial Normal Differentiation and Post-injury Regeneration'],\n",
       "  'Background: High mobility group AT-hook1 (HMGA1) is essential for airway basal cell mucociliary differentiation, barrier integrity and wound repair.  HMGA1 expression suppresses the abnormal basal cell differentiation to squamous, inflammatory and epithelial-mesenchymal transition phenotype commonly observed in association with cigarette smoking and chronic obstructive pulmonary disease (COPD).',\n",
       "  'dgm lae bc smoking status copd smoker large airway basal cells cultured epithelium',\n",
       "  'dgm sm smoking status nonsmoker small airway epithelial cells',\n",
       "  'human'),\n",
       " 'GSE86064-13-vs-3-human dn': (['Mandatory Role of HMGA1 in Human Airway Epithelial Normal Differentiation and Post-injury Regeneration'],\n",
       "  'Background: High mobility group AT-hook1 (HMGA1) is essential for airway basal cell mucociliary differentiation, barrier integrity and wound repair.  HMGA1 expression suppresses the abnormal basal cell differentiation to squamous, inflammatory and epithelial-mesenchymal transition phenotype commonly observed in association with cigarette smoking and chronic obstructive pulmonary disease (COPD).',\n",
       "  'dgm sm smoking status nonsmoker small airway epithelial cells',\n",
       "  'dgm lae bc smoking status copd smoker large airway basal cells cultured epithelium',\n",
       "  'human'),\n",
       " 'GSE159094-0-vs-3-human dn': (['Development of a glucocorticoid exposure signature for SLE'],\n",
       "  'We performed RNA-seq on peripheral blood mononuclear cells (PBMC) from healthy human donors treated with interferon alpha 2a (1000IU/mL) +/- dexamethasone 10^-7M, with the aims of studying interactions between IFN and glucocorticoid induced gene expression, as well as identifying potential transcripts that may be used as a glucocorticoid exposure signature in clinical practice in patients with SLE. Transcripts identified from RNA-seq data were analysed in SLE patient data sets to validate clinical utility. We found that dexamethasone minimally impacts on interferon stimulated gene expression however IFN altered gene transcription of many previously reported glucocorticoid induced genes.',\n",
       "  'pbmc ifn donor individual peripheral blood mononuclear cells sex',\n",
       "  'pbmc dex donor individual peripheral blood mononuclear cells sex',\n",
       "  'human'),\n",
       " 'GSE137824-1-vs-3-human up': (['Systematic identification of tumor-intrinsic resistance mechanisms and combinatorial therapy targets for immunotherapy'],\n",
       "  'A number of reports have demonstrated that tumor-intrinsic mechanisms of resistance, such as the loss of genes critical for antigen presentation and inflammatory responses, along with the activation of various cellular signaling cascades, can limit the efficacy of immunotherapy. Strategies to sensitize tumor cells to immunotherapy may overcome some resistance mechanisms, but identifying therapeutic targets has remained challenging. Here, we integrate a two-cell type (2CT) whole-genome CRISPR-Cas9 screen with dynamic transcriptional profiling of the tumor/T cell interaction to comprehensively identify tumor genes that are induced to promote tumor survival. We assessed the therapeutic potential of pharmacological inhibition of these and other top CRISPR identified targets as combinatorial targets to improve the efficacy of tumor destruction by T cells.',\n",
       "  'mel624 star.genome.sorted cell line melanoma cells interaction',\n",
       "  'hko veh cell line a375 melanoma cells transduced cas9 non targeting sgrna',\n",
       "  'human'),\n",
       " 'GSE220806-2-vs-1-human dn': (['Characteristic impairment of progesterone response in the cultured cervical fibroblast obtained from patients with refractory cervical insufficiency'],\n",
       "  'We compared the gene expression in response to progesterone (1μM) in human cervical fibroblasts.',\n",
       "  'ci notx condition untreated cells derived cervical insufficiency human uterine fibroblast passage 4 8',\n",
       "  'control p4 2 condition progesterone treated cells derived unripend cervix human uterine cervical fibroblast passage 4 8',\n",
       "  'human'),\n",
       " 'GSE86202-0-vs-1-human dn': (['Analyzing the interactions of mRNAs, miRNAs, lncRNAs and circRNAs to predict competing endogenous RNA networks in glioblastoma'],\n",
       "  'Cross-talk between competitive endogenous RNAs (ceRNAs) may play a critical role in revealing potential mechanisms of tumor development and physiology. Glioblastoma is the most common type of malignant primary brain tumor, and the mechanisms of tumor genesis and development in glioblastoma are unclear. Here, to investigate the role of non-coding RNAs and the ceRNA network in glioblastoma, we performed paired-end RNA sequencing and microarray analyses to obtain the expression profiles of mRNAs, lncRNAs circRNAs and miRNAs. We identified that the expression of 501 lncRNAs, 1789 mRNAs, 2038 circRNAs and 143 miRNAs were often altered between glioblastoma and matched normal brain tissue. Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analyses were performed on these differentially expressed mRNAs and miRNA-mediated target genes of lncRNAs and circRNAs. Furthermore, we used a multi-step computational framework and several bioinformatics methods to construct a ceRNA network combining mRNAs, miRNAs, lncRNAs and circRNA, based on co-expression analysis between the differentially expressed RNAs. We identified that plenty of lncRNAs, CircRNAs and their downstream target genes in the ceRNA network are related to glutamatergic synapse, suggesting that glutamate metabolism is involved in glioma biological functions. Our results will accelerate the understanding of tumorigenesis, cancer progression and even therapeutic targeting in glioblastoma. We hope to inspire researchers to study the role of non-coding RNAs in glioblastoma.',\n",
       "  'glioblastoma sex male',\n",
       "  'brain sex male brian',\n",
       "  'human'),\n",
       " 'GSE164687,GSE175880-8-vs-9-human up': (['CD66b-CD64dimCD115- cells in the human bone marrow represent neutrophil-committed progenitors'],\n",
       "  'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'neutrophil committed progenitors bone marrow donor id',\n",
       "  'mm metamyelocytes bone marrow donor id',\n",
       "  'human'),\n",
       " 'GSE101353,GSE128709-0-vs-1-human dn': (['Exaggerated BMP4 signalling alters human airway basal progenitor cell differentiation to cigarette smoking-related phenotypes [RNA-Seq]'],\n",
       "  'Airway remodelling in chronic obstructive pulmonary disease (COPD) originates, in part, from smoking-induced changes in airway basal stem/progenitor cells (BCs). Based on the knowledge that bone morphogenetic protein 4 (BMP4) influences epithelial progenitor function in the developing and adult mouse lung, we hypothesised that BMP4 signalling may regulate the biology of adult human airway BCs relevant to COPD.',\n",
       "  'dgm m0 lg (bmp4 study) smoking status large airway epithelial cells time serial bronchoscopy (month) epithelium',\n",
       "  'dgm la bc (bmp4 study) smoking status nonsmoker basal cells cultured airway epithelial large',\n",
       "  'human'),\n",
       " 'GSE107685-0-vs-1-human up': (['Gene expression changes in human iPSC-derived cardiomyocytes after X-ray irradiation'],\n",
       "  'Within the present study we evaluated hiPSC-CMs as a possible methodological extension for the assessment of organ specific responses to ionizing radiation regarding gene expression changes. We aimed to review the feasibility of hiPSC-CMs as a valid testing model for gene expression changes and to determine alterations within the gene expression pattern of hiPSC-CMs after exposure to X-ray radiation, subsequently.',\n",
       "  'cor4u 5gy ipsc derived cardiomyocytes 48h incubation irradiation hipsc cms (cor4u)',\n",
       "  'cor4u 0gy ipsc derived cardiomyocytes 48h incubation sham irradiation hipsc cms (cor4u)',\n",
       "  'human'),\n",
       " 'GSE253540-1-vs-2-human dn': (['Treg cells from human blood differentiate into non-lymphoid tissue-resident effector cells upon TNFR2 costimulation'],\n",
       "  'Regulatory T (Treg) cells can facilitate transplant tolerance and attenuate autoimmune- and inflammatory diseases. Therefore, it is clinically relevant to stimulate Treg cell expansion and function in vivo and to create therapeutic Treg cell products in vitro. We report that TNF receptor 2 (TNFR2) is a unique costimulus for naïve, thymus-derived (t)Treg cells from human blood that promotes their differentiation into non-lymphoid tissue (NLT)-resident effector Treg cells, without Th-like polarization. In contrast, CD28 costimulation maintains a lymphoid tissue (LT)-resident Treg cell phenotype. We base this conclusion on transcriptome and proteome analysis of TNFR2- and CD28-costimulated CD4+ tTreg cells and conventional T (Tconv) cells, followed by bioinformatic comparison with published transcriptomic Treg cell signatures from NLT and LT in health and disease, including autoimmunity and cancer. These analyses illuminated that TNFR2 costimulation promotes tTreg cell capacity for survival, migration, immunosuppression and tissue regeneration. Functional studies confirmed improved migratory ability of TNFR2-costimulated tTreg cells. Flow cytometry validated the presence of the TNFR2-driven tTreg cell signature in effector/memory Treg cells from the human placenta as opposed to blood. Thus, TNFR2 can be exploited as driver of NLT-resident tTreg cell differentiation for adoptive cell therapy or antibody-based immunomodulation in human disease.',\n",
       "  'ttreg tnfr2 peripheral blood regulatory cd4 cell il 2 anti cd3/tnfr2 agonistic mabs',\n",
       "  'ttreg cd28 peripheral blood regulatory cd4 cell il 2 anti cd3/cd28 agonistic mabs',\n",
       "  'human'),\n",
       " 'GSE154258,GSE154260-10-vs-2-mouse dn': (['Tuft cells restrain pancreatic tumorigenesis through paracrine eicosanoid signaling [bulk RNA-seq]'],\n",
       "  'We performed bulk RNA-seq on pancreatic and intestinal tuft cells, bone marrow derived macrophages with various treatments, and mouse pancreatic stellate cells with various treatments.',\n",
       "  'wt intestine tuft strain mixed age 12 month mouse',\n",
       "  'wt intestine non tuft strain mixed age 12 month mouse',\n",
       "  'mouse'),\n",
       " 'GSE116511,GSE116536-5-vs-0-mouse dn': (['Tuft cells play a suppressive role in pancreatic injury and tumorigenesis [RNA-seq]'],\n",
       "  'We perfromed RNA-seq on pancreatic and intestinal tuft cells and bone marrow derived macrophages',\n",
       "  'wt intestine tuft strain background mixed /variation wild type',\n",
       "  'wt intestine non tuft strain background mixed /variation wild type',\n",
       "  'mouse'),\n",
       " 'GSE120299-2-vs-1-human up': (['Human Pancreatic Islets Expressing HNF1A Variant Have Defective β cell Transcriptional Regulatory Networks'],\n",
       "  'Using an integrated approach to characterize the pancreatic tissue and isolated islets from a 33-year-old with 17 years of type 1 diabetes (T1D), we found donor islets contained β cells without insulitis and lacked glucose-stimulated insulin secretion despite a normal insulin response to cAMP-evoked stimulation. With these unexpected findings for T1D, we sequenced the donor DNA and found a pathogenic heterozygous variant in hepatocyte nuclear factor 1 alpha (HNF1A). In one of the first studies of human pancreatic islets with a disease-causing HNF1A variant associated with the most common form of monogenic diabetes, we found that HNF1A dysfunction leads to insulin-insufficient diabetes reminiscent of T1D by impacting the regulatory processes critical for glucose-stimulated insulin secretion and suggest a rationale for a therapeutic alternative to current treatment.',\n",
       "  'control [beta cells] pancreatic beta cells age gender disease state non diabetic cadaveric islet',\n",
       "  'control [alpha cells] facs purified alpha cells human pancreatic islets age donor years old gender disease state non diabetic',\n",
       "  'human'),\n",
       " 'GSE82043-4-vs-5-mouse up': (['Itaconate links inhibition of succinate dehydrogenase with macrophage metabolic remodeling and regulation of inflammation'],\n",
       "  'Remodeling of the tricarboxylic acid (TCA) cycle is a metabolic adaptation mechanism accompanying inflammatory macrophage activation. During this process, endogenous metabolites can adopt regulatory roles that govern specific aspects of inflammatory response, as recently shown for succinate, which regulates the downstream pro-inflammatory IL-1β-HIF1a axis. Itaconate is one of the most highly induced metabolites in activated macrophages, yet its functional significance remains unknown. Here, we show that itaconate modulates macrophage metabolism and effector functions via its effect on succinate dehydrogenase, by inhibiting conversion of succinate to fumarate. Through this action, itaconate exerts anti-inflammatory effects when administered in vitro and in vivo during macrophage activation and ischemia-reperfusion injury. Using newly generated Irg1-/- mice, which lack the ability to produce itaconate, we show that endogenous itaconate regulates succinate levels and function, changes in mitochondrial respiration, and inflammatory cytokine production during macrophage activation. These studies highlight itaconate as a major physiological regulator of the global metabolic rewiring and effector functions of inflammatory macrophages.',\n",
       "  'exp1 di strain background c57bl/6 /variation wt bone marrow derived macrophages treated dimethyl itaconate',\n",
       "  'exp2 irg1ko lps ifng strain background c57bl/6 /variation irg1 / bone marrow derived macrophages treated 24h',\n",
       "  'mouse'),\n",
       " 'GSE214558,GSE214559-3-vs-6-human dn': (['RNA Polymerase II pausing temporally coordinates cell cycle progression and erythroid differentiation [RNA-seq]'],\n",
       "  'The controlled release of promoter-proximal RNA polymerase II (Pol II) pausing into productive elongation is a major step in gene regulation. Yet, functionally analyzing Pol II pausing is difficult because the factors that regulate pause release are generally essential. Here, we identified  heterozygous loss-of-function mutations in SUPT5H, encoding SPT5, in individuals with β-thalassemia trait unlinked to the β-globin locus. Recapitulating the pathogenic mutations in human erythroid cells or acute treatment with transcription inhibitors replicates the patient phenotype of altered globin gene expression. Furthermore, in SUPT5H-edited cells, Pol II pause release was globally disrupted. We observed dynamic patterns of pausing at cell cycle and erythroid differentiation genes at the transition from progenitors to precursors during erythropoiesis. At this same transition, in SUPT5H-edited cells, we observed delayed differentiation and altered cell cycle kinetics, as well as a delay in the onset of gene expression programs. Therefore, hindering pause release perturbs cell cycle and delays differentiation at key stages of erythropoiesis, revealing a role for Pol II pausing in the temporal coordination between the cell cycle and differentiation.',\n",
       "  'aavs1 bone marrow hematopoietic stem progenitor cells (hspcs) de identified donor r003783 developmental stage progenitors grna',\n",
       "  'supt5h bone marrow hematopoietic stem progenitor cells (hspcs) de identified donor developmental stage differntiation culture day grna',\n",
       "  'human'),\n",
       " 'GSE82289,GSE82297-2-vs-5-human dn': (['BRD4 localization to lineage-specific enhancers is associated with a distinct transcription factor repertoire [mRNA-Seq]'],\n",
       "  'We examined context specific function of BRD4 in promoting lineage specific gene expression and show that BRD4 is essential for osteoblast differentiation.',\n",
       "  'mrna und sicntr immortalization type sv40 large antigen immortalized human fetal osteoblasts (hfob) protocol undifferentiated',\n",
       "  'mrna und dmso immortalization type sv40 large antigen immortalized human fetal osteoblasts (hfob) protocol undifferentiated',\n",
       "  'human'),\n",
       " 'GSE128163-1-vs-3-human dn': (['The human fetal thymus generates effector invariant γδ T cells in a Lin28b-dependent manner'],\n",
       "  'In the mouse thymus, invariant γδ T cells are generated at well-defined times during development and acquire effector functions before exiting the thymus. However, whether such thymic programming can occur in human is not known. Here we analyzed human fetal and post-natal γδ thymocytes and investigated the role of hematopoietic-stem-and-precursor-cells (HSPC) in the generation of human γδ T cells. Unlike post-natal γδ thymocytes, fetal γδ thymocytes were functionally programmed (e.g. IFNγ, granzymes), expressed low levels of terminal-deoxynucleotidyl-transferase (TdT) and were highly enriched for invariant/public germline-encoded CDR3 sequences (TRGV8-TRJP1-CATWDTTGWFKIF, TRDV2-TRDD3-CACDTGGY and TRDV1-TRDD3-CALGELGD, previously shown to be expanded in cytomegalovirus infection) composed of short-homology-repeat-containing gene segments. Furthermore, these unique characteristics were due to an intrinsic property of fetal HSPC caused by high expression of the RNA-binding protein Lin28b. In conclusion, our data indicate that the human fetal thymus generates, in a HSPC/Lin28b-dependent manner, invariant γδ T cells with programmed effector functions.',\n",
       "  'sample op9dl1 ab age >18years / source hematopoietic stem progenitor cells (hspc) gamma delta (hspc derived)',\n",
       "  'sample exvivo ft gd age weeks / source thymus gamma delta thymocytes cell subset nonvg9vd2',\n",
       "  'human'),\n",
       " 'GSE99081-1-vs-4-human dn': (['External RNA Control Consortium (ERCC) spikes in RNA sequencing (RNA-Seq) data normalization: transcriptome analysis of yellow fever virus-infected cells'],\n",
       "  'We explored the ability of External RNA Control Consortium (ERCC) standards to detect technical biases between batches of sequencing experiments and to set detection thresholds. We compared three normalization methods (FPKM, DESeq2 and RUV) to identify the optimal analysis approach.',\n",
       "  'hep/kup yfv 17d hepatocytes + kupffer cells model pluricellular infection vaccine time 24h post',\n",
       "  'hep yfv 17d hepatocytes model monocellular infection vaccine time 24h post',\n",
       "  'human'),\n",
       " 'GSE196117-5-vs-7-human up': (['Transcriptional profiling of whole blood leukocytes obtained from critically ill patients with sepsis treated with clarithromycin, or not, and healthy particpants'],\n",
       "  'To evaluate whether clarithromycin improves 28-day mortality among patients with sepsis, respiratory and multiple-organ dysfunction syndrome. The INntravenous CLArithromycin in Sepsis and multiple organ dysfunction Syndrome trial was a phase 3, randomized, double blind, placebo-controlled clinical study, conducted in 11 intensive care units and 2 Internal Medicine wards in 2 countries. Patients with sepsis, respiratory failure and total sequential organ failure assessment score of ≥7 were enrolled between December 2017 and September 2019. Follow-up lasted 90 days. Patients were randomized to receive 1 gr of intravenous clarithromycin or placebo once daily for 4 consecutive days.',\n",
       "  'inc whole blood individual condition sepsis class clarithromycin time day age sex male',\n",
       "  'inc whole blood individual condition sepsis class clarithromycin time day age sex female',\n",
       "  'human'),\n",
       " 'GSE159094-4-vs-0-human up': (['Development of a glucocorticoid exposure signature for SLE'],\n",
       "  'We performed RNA-seq on peripheral blood mononuclear cells (PBMC) from healthy human donors treated with interferon alpha 2a (1000IU/mL) +/- dexamethasone 10^-7M, with the aims of studying interactions between IFN and glucocorticoid induced gene expression, as well as identifying potential transcripts that may be used as a glucocorticoid exposure signature in clinical practice in patients with SLE. Transcripts identified from RNA-seq data were analysed in SLE patient data sets to validate clinical utility. We found that dexamethasone minimally impacts on interferon stimulated gene expression however IFN altered gene transcription of many previously reported glucocorticoid induced genes.',\n",
       "  'pbmc untreated donor individual peripheral blood mononuclear cells ut sex',\n",
       "  'pbmc ifn donor individual peripheral blood mononuclear cells sex',\n",
       "  'human'),\n",
       " 'GSE73114-0-vs-3-human dn': (['RNA-sequencing analysis of human fibrolamellar hepatocellular carcinoma and 4 maturational stages of the human hepatobiliary system'],\n",
       "  'The etiology of fibrolamellar hepatocellular carcinoma (FL-HCC), a liver cancer occurring increasingly in children to young adults, is poorly understood. We performed high-throughput sequencing on mRNA isolated from a newly developed model of FL-HCC and 4 different maturational lineages of the hepatobiliary system. Our results indicate that the transcriptome of FL-HCC is most similar to that of biliary tree stem cells.',\n",
       "  'ahep adult hepatocytes',\n",
       "  'btsc biliary tree stem cells',\n",
       "  'human'),\n",
       " 'GSE184536-3-vs-4-human up': (['The NF-κB transcriptional footprint is essential for SARS-CoV-2 replication'],\n",
       "  'SARS-CoV-2, the etiological agent of COVID-19, is characterized by a delay in Type I interferon (IFN-I)-mediated antiviral defenses alongside robust cytokine production. Here we investigate the underlying molecular basis for this imbalance and implicate virus-mediated activation of NF-κB in the absence of other canonical IFN-I-related transcription factors. Epigenetic and single cell transcriptomic analyses show a selective NF-κB signature that was most prominent in infected cells. Disruption of NF-κB signaling through the silencing of the NF-κB transcription factors p65 or p50 resulted in loss of virus replication that was rescued upon reconstitution. These findings could be further corroborated with the use of NF-κB inhibitors, which reduced SARS-CoV-2 replication in vitro. These data suggest that the robust cytokine production in response to SARS-CoV-2, despite a diminished IFN-I response, is the product of a dependency on NF-κB for viral replication.',\n",
       "  'mrnaseq mock 24h lung adenocarcinoma virus 24 h a549 ace2 cells',\n",
       "  'mrnaseq ifnb 12h lung adenocarcinoma virus mock 12 h ifn î² (100 u/ml) a549 ace2 cells',\n",
       "  'human'),\n",
       " 'GSE174539-7-vs-13-human up': (['Emergent mechanical control of vascular morphogenesis'],\n",
       "  'mRNA profiles of endothelial cells and fibroblasts undergoing vasculogenesis under juxtacrine co-culture vs. monoculture.',\n",
       "  'culture huvec fb coculture time day human normal lung fibroblasts',\n",
       "  'culture huvec fb coculture time day human umbilical vein endothelial cells',\n",
       "  'human'),\n",
       " 'GSE86064-7-vs-2-human up': (['Mandatory Role of HMGA1 in Human Airway Epithelial Normal Differentiation and Post-injury Regeneration'],\n",
       "  'Background: High mobility group AT-hook1 (HMGA1) is essential for airway basal cell mucociliary differentiation, barrier integrity and wound repair.  HMGA1 expression suppresses the abnormal basal cell differentiation to squamous, inflammatory and epithelial-mesenchymal transition phenotype commonly observed in association with cigarette smoking and chronic obstructive pulmonary disease (COPD).',\n",
       "  'dgm lae bc smoking status nonsmoker large airway basal cells cultured epithelium',\n",
       "  'dgm lg smoking status nonsmoker large airway epithelial cells',\n",
       "  'human'),\n",
       " 'GSE119336,GSE119337-0-vs-3-human up': (['RNA over-editing leads to aggressiveness of intrahepatic cholangiocarcinoma [RNA-Seq]'],\n",
       "  'Alteration in RNA editing has been connected to tumor progression and many other important human diseases. However, the role of RNA editing in intrahepatic cholangiocarcinoma (ICC) remains unclear. Here we conducted a transcriptome-wide investigation in ICCs, and revealed ADAR-associated over-editing occurred uniformly in ICCs. Collectively, these results reveal a previously unrecognized role of RNA editing in malignant transformation into ICC, and ADAR1 inhibition may obviate progression and relapse of this malignancy.',\n",
       "  'gender age intrahepatic cholangiocarcinoma tumor',\n",
       "  'gender age adjacent non tumor liver',\n",
       "  'human'),\n",
       " 'GSE248853,GSE248854-0-vs-3-human dn': (['RFX6 maintains gene expression and function of adult human islet ⍺ cells [RNAseq]'],\n",
       "  'Transcriptomic profiling by high-throughput sequencing of primary human alpha and beta cells following RFX6 loss of function',\n",
       "  'alpha rfx6kd cadaveric pancreatic cell line primary islet cells rfx6 shrna',\n",
       "  'beta rfx6kd cadaveric pancreatic cell line primary islet cells rfx6 shrna',\n",
       "  'human'),\n",
       " 'GSE154919-2-vs-0-human dn': (['Aneuploid cells activate NF-κB to promote their immune clearance by NK cells'],\n",
       "  'The immune system plays a major role in the protection against cancer. Identifying and characterizing the pathways mediating this immune surveillance is thus critical for understanding how cancer cells are recognized and eliminated. Aneuploidy is a hallmark of cancer and we previously found that untransformed cells that had undergone senescence due to highly abnormal karyotypes are eliminated by Natural Killer (NK) cells in vitro. However, the mechanisms underlying this process remained elusive. Here we show that NK-mediated immune clearance of aneuploid cells is predominantly mediated by non-cell autonomous mechanisms. Our data indicate that NF-κB signaling in aneuploid cells is central to elicit this immune response. Inactivating NF-κB abolishes NK-cell mediated clearance of untransformed aneuploid cells in vitro. In cancer cell lines, NF-κB activation correlates with degree of aneuploidy, raising the possibility that aneuploidy-induced immune recognition is partially retained in cancer.',\n",
       "  'rpe1 htert control cells population bulk culture',\n",
       "  'rpe1 htert doxorubicin treated cells population bulk culture',\n",
       "  'human'),\n",
       " 'GSE138053-0-vs-2-human dn': (['AR isoform 2 is widely expressed in human tissues and acts as a decoy receptor to regulate AR transcriptional activity'],\n",
       "  'Together with the Estrogen, Progesterone, and Glucocorticoid Receptors, the Androgen Receptor (AR) is a member of the type I nuclear receptor superfamily, a class of transcription factors that are activated by steroid hormones1. Nuclear receptors share a common three-domain structure, comprising a highly variable amino-terminal domain (NTD), with potent transcriptional activator capability, a central two zinc finger motifs DNA-binding domain (DBD), and a well-conserved carboxy-terminal ligand-binding domain (LBD) responsible for interacting with the transcriptional machinery1. Given their modular structure, expression of isoforms resulting from alternative splicing or starting and termination events have profound effects on nuclear receptor activity and function. Several AR variants (AR-Vs) have been identified from prostate cancer clinical specimens, arising from multiple mechanisms, including genomic rearrangements2 and upregulation of splicing factors3. This seminal discovery has provided the molecular basis for a deeper understanding of AR-related diseases, with potential to be translated in identification of novel therapeutic targets and promising prognostic markers4–6. By deeply sequencing the transcriptome of human brains, here we identified AR-2 as the most expressed AR isoform. Furthermore, we found that AR-2 is widely expressed in other human tissues, forms heterodimers with the full length AR and acts as a repressor of its transcriptional activity. Delivery of this isoform via adeno-associated virus (AAV9) resulted in significant amelioration of the disease phenotype in a mouse model of spinal and bulbar muscular atrophy (SBMA), a neuromuscular disease with unmet clinical need caused by a toxic gain of function of polyglutamine-expanded (polyQ) AR7. Collectively, these results establish a physiological role of AR-2 in fine-tuning full length AR activity and demonstrate the feasibility and therapeutic benefit of modulating this variant for treating SBMA and other AR-related diseases.',\n",
       "  'control lateral motor cortex bioreplicate culture condition na biological replicate sample group donor age gender race author id reference cgnd hra',\n",
       "  'lumbar spinal cord bioreplicate 1 culture condition na biological replicate sample group donor age gender race author id reference cgnd hra',\n",
       "  'human'),\n",
       " 'GSE182877-2-vs-1-human dn': (['In vitro expansion of cirrhosis derived liver epithelial cells reveals progenitor-like properties'],\n",
       "  'Our goal is to in vitro expand mouse hepatocytes and human cirrhotic liver hepatocytes. Under the extreme culture condition, mouse hepatocytes proliferated and de-diffrentiated into progenitor cell like cells, moreover, the progenitor like cells could re-differentiate into hepatocytes in vivo. On the other hand, huamn cirrhotic liver cells could also expand and retain the probability of differentiate in vivo.',\n",
       "  'huhc normal human hepatocytes unclutured passage number diagnosis age sex',\n",
       "  'cirrho lim cirrhotic liver cells cultured passage number diagnosis alcoholic steatohepatitis age year sex male',\n",
       "  'human'),\n",
       " 'GSE93526-6-vs-4-human up': (['Gene signature profiles in respiratory epithelium infected with nontuberculous mycobacteria'],\n",
       "  'The incidence of pulmonary nontuberculous mycobacterial (PNTM) disease is increasing, but host susceptibility factors are not fully understood. We infected air-liquid interface (ALI) primary respiratory epithelial cell cultures with Mycobacterium avium complex (MAC) or Mycobacterium abscessus (MAB) and performed transcriptome sequencing (RNA-Seq) to identify relevant gene expression differences. We used cells from 4 different donors in order to try to obtain generalizable data. The differentiated respiratory epithelial cells in ALI were infected with MAC or MAB at MOI of 100:1 or 1000:1, and RNA-seq was performed at 1 and 3 days after infection. We found downregulation of ciliary genes, including several identified with polymorphisms in previous PNTM cohorts. The cytokine IL-32, the superpathway of cholesterol biosynthesis and downstream targets within the IL-17 signaling pathway were all elevated. The integrin signaling pathway was more upregulated by MAB than MAC infection. Working with primary respiratory epithelial cells infected with nontuberculous mycobacteria at ALI, we identified ciliary function, cholesterol biosynthesis, chemokine production and the IL-17 pathway as major targets of host responses to infection. Some of these pathways may be amenable to therapeutic manipulation.',\n",
       "  '42f mab moi 1 infection differentiated respiratory epithelial cells age 42 years old gender female normal primary human bronchial (nhbe)',\n",
       "  '57f mac moi 1 infection differentiated respiratory epithelial cells age 57 years old gender female normal primary human bronchial (nhbe)',\n",
       "  'human'),\n",
       " 'GSE202182-2-vs-7-human dn': (['A specific molecular signature in SARS-CoV-2 infected kidney biopsies'],\n",
       "  'To characterize the pathophysiology of kidney injury in a large kidney biopsy series of living SARS-CoV-2–infected patients',\n",
       "  'healthy kidney (sample disease state virus none',\n",
       "  'fish positive kidney (sample disease state covid 19 virus sars cov 2',\n",
       "  'human'),\n",
       " 'GSE113369-2-vs-3-human dn': (['CDK4 inhibition diminishes p53 activation by Mdm2-antagonists'],\n",
       "  'CDK4 inhibitors have reached clinical approval for cancer therapy. In parallel, the p53 antagonist Mdm2 remains an attractive target for anti-cancer therapy, including numerous clinical studies. The genes encoding Mdm2 and CDK4 are frequently co-amplified in human malignancies, most notably in liposarcomas, suggesting their combined targeting for therapy. Here we show, however, that small compounds that inhibit Mdm2 and CDK4 antagonize each other rather than synergize in their cytotoxicity towards sarcoma cells. CDK4 inhibition attenuates the induction of p53-responsive genes upon Mdm2 inhibition, and similar results were obtained when depleting Mdm2 and/or CDK4 with siRNA. CDK4 inhibitors also interfered with p53 activity in response to DNA damage. CDK4 inhibition did not reduce p53 binding or histone acetylation to promoters, but rather attenuated the subsequent recruitment of RNA polymerase II. The complexes of p53 and Mdm2, as well as CDK4 and Cyclin D1, physically associated with each other. Upon combined inhibition of Mdm2 and CDK4/6, the interaction of this complex was impaired. Thus, the CDK4-Cyclin D1 complex plays a key role in enabling the transcription of p53 target genes. Taken together, our results raise caution regarding the combination of CDK4 inhibitors with Mdm2 antagonists or conventional DNA-damaging chemotherapeutics in the clinics. Moreover, they suggest a hitherto unknown role for CDK4-cyclin D1 complex in sustaining p53 activity, possibly focusing p53-mediated transcription on actively proliferating cells.',\n",
       "  'dmso 1 cell line sjsa human osteosarcoma',\n",
       "  'n+pd30 1 cell line sjsa human osteosarcoma',\n",
       "  'human'),\n",
       " 'GSE93526-4-vs-0-human dn': (['Gene signature profiles in respiratory epithelium infected with nontuberculous mycobacteria'],\n",
       "  'The incidence of pulmonary nontuberculous mycobacterial (PNTM) disease is increasing, but host susceptibility factors are not fully understood. We infected air-liquid interface (ALI) primary respiratory epithelial cell cultures with Mycobacterium avium complex (MAC) or Mycobacterium abscessus (MAB) and performed transcriptome sequencing (RNA-Seq) to identify relevant gene expression differences. We used cells from 4 different donors in order to try to obtain generalizable data. The differentiated respiratory epithelial cells in ALI were infected with MAC or MAB at MOI of 100:1 or 1000:1, and RNA-seq was performed at 1 and 3 days after infection. We found downregulation of ciliary genes, including several identified with polymorphisms in previous PNTM cohorts. The cytokine IL-32, the superpathway of cholesterol biosynthesis and downstream targets within the IL-17 signaling pathway were all elevated. The integrin signaling pathway was more upregulated by MAB than MAC infection. Working with primary respiratory epithelial cells infected with nontuberculous mycobacteria at ALI, we identified ciliary function, cholesterol biosynthesis, chemokine production and the IL-17 pathway as major targets of host responses to infection. Some of these pathways may be amenable to therapeutic manipulation.',\n",
       "  '57f mac moi 1 infection differentiated respiratory epithelial cells age 57 years old gender female normal primary human bronchial (nhbe)',\n",
       "  '69f mac moi 1 infection differentiated respiratory epithelial cells age 69 years old gender female normal primary human bronchial (nhbe)',\n",
       "  'human'),\n",
       " 'GSE84073-0-vs-5-human dn': (['Patient-derived organoid cultures model human primary liver cancer in vitro'],\n",
       "  'The study of human liver cancer has been hampered by the lack of reliable models that faithfully recapitulate the physiopathology of the patient’s original tumour ex vivo. Here, we describe the first culture system to allow the establishment and long-term expansion of human primary liver cancer (PLC) organoids (called tumoroids) from the three most common PLC subtypes: Hepatocellular carcinoma (HCC), Cholangiocarcinoma (CC) and mixed HCC/CC tumours (CHC). We evaluated whether tumoroid lines maintain the expression profile and the genomic landscape of the original tumor they derived from by performing genome-wide (RNAseq/WES) analyses before and after culture. These analyses reveal that the cultures closely recapitulate the gene expression and the mutational landscape of the original tumor, allowing discrimination between different tumor subtypes (HCC, CHC, CC).',\n",
       "  '[rna seq] human liver biopsy passages n/ gender',\n",
       "  '[rna seq] human liver biopsy passages n/ gender',\n",
       "  'human'),\n",
       " 'GSE154767-4-vs-2-mouse up': (['ADAM10 shapes myeloid antigen-presenting cell homeostasis in the splenic marginal zone'],\n",
       "  'The spleen contains phenotypically and functionally distinct cDC1 and cDC2 subpopulations, which each can be divided into several smaller and less-well characterized subsets. Despite advances in understanding the complexity of DC ontogeny and function by transcriptional programming, the significance of post-translational modifications in controlling tissue-specific cDC (subset) immunobiology remains elusive. Here, we identified the cell surface-expressed A-disintegrin-and-metalloproteinase 10 (ADAM10) as an essential regulator of cDC1 and cDC2 homeostasis in the splenic marginal zone (MZ). Mice with a CD11c-specific deletion of ADAM10 (ADAM10ΔCD11c) exhibited a complete loss of splenic ESAMhi cDC2A, because ADAM10 controlled their commitment, differentiation, survival, and EBI2-mediated localization within the MZ. Moreover, we discovered that ADAM10 is a molecular switch regulating cDC2 subset heterogeneity in the spleen, as this unbalanced cDC2A homeostasis in the absence of ADAM10 was compensated for by the emergence of a novel Clec12a+ cDC2B subset, closely resembling cDC2 generally found in peripheral lymph nodes. Moreover, in ADAM10ΔCD11c mice terminal differentiation of cDC1 was abrogated, resulting in reduced numbers of Langerin+ cDC1.',\n",
       "  'wt cd4 dc sample strain c57bl/6 spleen dendritic cell',\n",
       "  'wt cd8+ dc sample strain c57bl/6 spleen dendritic cell',\n",
       "  'mouse'),\n",
       " 'GSE218723-3-vs-4-mouse up': (['The rDNA antisense transcript PAPAS promotes lactogenic differentiation of mammary epithelial cells and suppresses breast cancer'],\n",
       "  'PAPAS is an antisense transcript of rRNA genes and mediates silencing of the locus in non-dividing cells. Here, we have analyzed PAPAS function in lactogenic differentiation of mammary epithelial cells (HC11 cells) and in mammary cancer cells (4T1). Using doxycycline (Dox) inducible shRNA against PAPAS, we found that the knockdown perturbs differentiation and inhibits expression of lactogenic genes. In contrast, Dox-inducible overexpression of PAPAS in 4T1 cells suppresses genes required for ribosome biogenesis and slows down the neoplastic growth.',\n",
       "  'cell line hc11 mammary epithalial cells untransformed dox + dip',\n",
       "  'cell line hc11 mammary epithalial cells untransformed dox',\n",
       "  'mouse'),\n",
       " 'GSE248853,GSE248854-1-vs-3-human dn': (['RFX6 maintains gene expression and function of adult human islet ⍺ cells [RNAseq]'],\n",
       "  'Transcriptomic profiling by high-throughput sequencing of primary human alpha and beta cells following RFX6 loss of function',\n",
       "  'alpha control cadaveric pancreatic cell line primary islet cells non targeting shrna',\n",
       "  'beta rfx6kd cadaveric pancreatic cell line primary islet cells rfx6 shrna',\n",
       "  'human'),\n",
       " 'GSE136598,GSE136600-0-vs-1-human up': (['Direct Induction of Trophoblast Stem Cells from Human Fibroblasts (RNA-seq)'],\n",
       "  'Transcription profile analysis revealed similarity/dissimilarity between human blastocyst derived trophoblast stem cells (hbdTSCs), human induced trophoblast stem cells (hiTSCs), human foreskin fibroblasts (HFFs) and pluripotent stem cells (hESCs, hiPSCs)',\n",
       "  'human stem cell line',\n",
       "  'human induced trophoblast stem cells line',\n",
       "  'human'),\n",
       " 'GSE61566-0-vs-2-human dn': (['Isolation and Transcriptome Analyses of Human Erythroid Progenitors: BFU-E and CFU-E'],\n",
       "  'Burst-forming unit-erythroid (BFU-E) and colony-forming unit-erythroid (CFU-E) cells are erythroid progenitors traditionally defined by colony assays. We developed a flow cytometry-based strategy for isolating human BFU-E and CFU-E cells based on the changes in expression of cell surface markers during in vitro erythroid cell culture. BFU-E and CFU-E are characterized by CD45+GPA-IL-3R-CD34+CD36-CD71low and CD45+GPA-IL-3R-CD34-CD36+CD71high phenotypes, respectively. Colony assays validated phenotypic assignment giving rise to BFU-E and CFU-E colonies, both at a purity ~90%. The BFU-E colony forming ability of CD45+GPA-IL-3R-CD34+CD36-CD71low cells required SCF and erythropoietin, while the CFU-E colony forming ability of CD45+GPA-IL-3R-CD34-CD36+CD71high cells required only erythropoietin. Bioinformatic analysis of the RNA-seq data revealed unique transcriptomes in each differentiation stage. The sorting strategy was validated in uncultured primary cells isolated from bone marrow and peripheral blood, indicating that marker expression is not an artifact of in vitro cell culture, but represents an in vivo characteristic of erythroid progenitor populations. The ability to isolate highly pure human BFU-E and CFU-E progenitors will enable detailed cellular and molecular characterization of these distinct progenitor populations and define their contribution to disordered erythropoiesis in inherited and acquired hematological disease. Our data provide important resource for future studies.',\n",
       "  'hs bfu burst forming unit erythroid cells (bfu e) cord blood e',\n",
       "  'hs cfu colony forming unit erythroid cells (cfu e) cord blood e',\n",
       "  'human'),\n",
       " 'GSE99081-5-vs-2-human up': (['External RNA Control Consortium (ERCC) spikes in RNA sequencing (RNA-Seq) data normalization: transcriptome analysis of yellow fever virus-infected cells'],\n",
       "  'We explored the ability of External RNA Control Consortium (ERCC) standards to detect technical biases between batches of sequencing experiments and to set detection thresholds. We compared three normalization methods (FPKM, DESeq2 and RUV) to identify the optimal analysis approach.',\n",
       "  'hep yfv asibi hepatocytes model monocellular infection wild type time 24h post',\n",
       "  'hep/kup yfv asibi hepatocytes + kupffer cells model pluricellular infection wild type time 24h post',\n",
       "  'human'),\n",
       " 'GSE232817,GSE232824-0-vs-4-human up': (['Modular Pooled Discovery of Synthetic Knockin Sequences to Program Durable Cell Therapies [RNA-1]'],\n",
       "  'Chronic stimulation can cause T cell dysfunction and limit efficacy of cellular immunotherapies. CRISPR screens have nominated gene targets for engineered T cells, but improved methods are required to compare large numbers of synthetic knockin sequences to reprogram cell functions. Here, we developed Modular Pooled Knockin Screening (ModPoKI), an adaptable platform for modular construction of DNA knockin libraries using barcoded multicistronic adaptors. We built two ModPoKI libraries of 100 transcription factors (TFs) and 129 natural and synthetic surface receptors. Over 30 ModPoKI screens across human TCR and CAR T cells in diverse conditions identified a transcription factor AP4 (TFAP4) construct that enhanced fitness of chronically-stimulated CAR T cells and anti-cancer function in vitro and in vivo. ModPoKI’s modularity allowed us to generate a ~10,000-member library of TF combinations. Non-viral knockin of a combined BATF-TFAP4 polycistronic construct further enhanced function. ModPoKI facilitates discovery of complex gene constructs to program cellular functions.',\n",
       "  'cells tngfr cd19 28z car knockin day 15 donor individual blood cell line primary none time',\n",
       "  'cells tngfr cd19 28z car knockin day 6 donor individual blood cell line primary none time',\n",
       "  'human'),\n",
       " 'GSE53545-1-vs-2-mouse up': (['Region-specific identity of small intestinal stem cells from the ileum and jejunum'],\n",
       "  'We report the expression profiles of three stem-cell varieties derived from the ileum and jejunum, focusing on gene signatures that are consistent in all stem cells of an intestinal location.  We find that intestinal stem cells establish and maintain a regional identity that is discernable through gene expression patterns consistent with intestinal function particular to that region.',\n",
       "  'distal ileal stem cells strain c57bl6 cell selection location ileum age 6 12 weeks gender male',\n",
       "  'proximal crypt jejenum stem cells strain c57bl6 cell selection location age 6 12 weeks gender male',\n",
       "  'mouse'),\n",
       " 'GSE226334-1-vs-4-mouse dn': (['TTP overexpression inhibits reconstitution potential of HSC but generates a dominant mitigating effect in collagen antibody-induced arthritis mice models'],\n",
       "  'Tristetraprolin (TTP), encoded by Zfp36 in mice, is one of the best characterized tandem zinc-finger mRNA binding protein involved in mRNA deadenylation and decay. TTPΔARE mice lack an AU-rich motif in the 3’ untranslated regions of TTP mRNA leading to increased TTP mRNA stability and more TTP protein, resulting in elevated mRNA decay rates of TTP targets.  We examined the effect of TTP overexpression on the hematopoietic system and found alterations in red blood cell and white blood cell frequencies, with loss of platelets and B220 cells and gains of eosinophils and T-cells. TTPΔARE mice also have skewed primitive populations in the bone marrow with increases in myeloid-biased HSCs, but loss of granulocyte/macrophage biased MPP3 in both young and mid-aged mice. These populations changes were associated with cell-specific transcriptional alterations, but surprisingly consistent upregulation of TNF and TGFB signaling pathways in both progenitor populations. HSCs with overexpression of TTP had decreased reconstitution potential but generate hematopoietic environments that mitigate the inflammatory response induced by collagen antibody-induced arthritis (CAIA) model- even when TTP overexpressing cells are present at low frequencies. We present an overall analysis of elevated TTP expression in the early hematopoietic compartments which drives reduced overall reconstitution potential of the HSCs but leads to a dominant repression of inflammation induced in a rheumatoid arthritis model.',\n",
       "  'mrnaseq ywt mpp3 born marrow multipotent progenitor 3 wt none age young hematopoetic stem cells',\n",
       "  'mrnaseq yttpdeltaare hsc born marrow hematopoetic stem cells tppdeltaare none age young',\n",
       "  'mouse'),\n",
       " 'GSE61566-0-vs-1-human up': (['Isolation and Transcriptome Analyses of Human Erythroid Progenitors: BFU-E and CFU-E'],\n",
       "  'Burst-forming unit-erythroid (BFU-E) and colony-forming unit-erythroid (CFU-E) cells are erythroid progenitors traditionally defined by colony assays. We developed a flow cytometry-based strategy for isolating human BFU-E and CFU-E cells based on the changes in expression of cell surface markers during in vitro erythroid cell culture. BFU-E and CFU-E are characterized by CD45+GPA-IL-3R-CD34+CD36-CD71low and CD45+GPA-IL-3R-CD34-CD36+CD71high phenotypes, respectively. Colony assays validated phenotypic assignment giving rise to BFU-E and CFU-E colonies, both at a purity ~90%. The BFU-E colony forming ability of CD45+GPA-IL-3R-CD34+CD36-CD71low cells required SCF and erythropoietin, while the CFU-E colony forming ability of CD45+GPA-IL-3R-CD34-CD36+CD71high cells required only erythropoietin. Bioinformatic analysis of the RNA-seq data revealed unique transcriptomes in each differentiation stage. The sorting strategy was validated in uncultured primary cells isolated from bone marrow and peripheral blood, indicating that marker expression is not an artifact of in vitro cell culture, but represents an in vivo characteristic of erythroid progenitor populations. The ability to isolate highly pure human BFU-E and CFU-E progenitors will enable detailed cellular and molecular characterization of these distinct progenitor populations and define their contribution to disordered erythropoiesis in inherited and acquired hematological disease. Our data provide important resource for future studies.',\n",
       "  'hs bfu burst forming unit erythroid cells (bfu e) cord blood e',\n",
       "  'hs cd34 positive cells (cd34) cord blood',\n",
       "  'human'),\n",
       " 'GSE178615-5-vs-2-human up': (['Coordinated metabolic transitions and gene expression by NAD+ during adipogenesis.'],\n",
       "  'Adipocytes are the main cell type in adipose tissue, a critical regulator of metabolism, highly specialized in storing energy as fat. Adipocytes differentiate from multipotent mesenchymal stromal cells through adipogenesis. Using RNA-seq we explored the effect of manipulating NAD+ bioavailability during adipogenic differentiation from human mesenchymal stem cells. Cells were harvested either undifferentiated (hMSC) or after days 8 and 16 of adipogenic induction. During adipogenesis, cells were treated with either NAD+ (5 mM), or FK866 (1 nM) or EX527 (50 μM).',\n",
       "  'adipo day16 cell line mesenchymal differentiation stage adipogenic induction days 16 untreated human',\n",
       "  'adipo+ex527 day16 cell line mesenchymal differentiation stage adipogenic induction days 16 ex527 human',\n",
       "  'human'),\n",
       " 'GSE153434-0-vs-2-human up': (['Exaggerated autophagy as molecular characterization of Stanford type A aortic dissection: A transcriptome analysis of human ascending aortic tissues'],\n",
       "  'Purpose: The aim of this study is to have a fullscape of  molecular pathology of Stanford type A aortic dissection',\n",
       "  'normal ascending aorta disease smooth muscle',\n",
       "  'taad ascending aorta disease smooth muscle',\n",
       "  'human'),\n",
       " 'GSE108621-0-vs-1-human up': (['Transcriptional profiling of HT-29 cells with different treatments'],\n",
       "  'Through RNA-seq, we investigated the transcriptional changes of necroptotic cell induced by TNF-α, SM-164 and zVAD. We report here that necroptosis leads to transcriptional activation of a large set of genes including pro-inflammatory cytokines, which is much stronger than that triggered by TNF-α alone. Furthermore, the robust induction of cytokine expression could be blocked by necroptotic inhibitor Nec-1s. Thus, we show that necroptosis exerts a profound effect on transcription, and activate the expression of pro-inflammatory genes, providing insights into the mechanism by which necroptosis promotes inflammation in human diseases.',\n",
       "  'dmso mrna seq colon cell line adenocarcinoma ht29 ht 29 cells',\n",
       "  'tsz mrna seq colon cell line adenocarcinoma ht29 tnf î± sm 164 zvad ht 29 cells',\n",
       "  'human'),\n",
       " 'GSE239662,GSE239663-3-vs-1-human dn': (['Novel method of isolating nuclei of human oligodendrocyte precursor cells reveals substantial developmental changes in gene expression and H3K27ac histone modification. [RNA-Seq]'],\n",
       "  'Oligodendrocyte precursor cells (OPCs) generate differentiated mature oligodendrocytes (MOs) during development. In adult brain, OPCs replenish MOs in adaptive plasticity, neurodegenerative disorders, and after trauma. The ability of OPCs to differentiate to MOs decreases with age and is compromised in disease. Here we explored the cell specific and age-dependent differences in gene expression and H3K27ac histone mark in these two cell types. H3K27ac is indicative of active promoters and enhancers. We developed a novel flow-cytometry-based approach to isolate OPC and MO nuclei from human postmortem brain and profiled gene expression and H3K27ac in adult and infant OPCs and MOs genome-wide.In adult brain, we detected extensive H3K27ac differences between the two cell types with high concordance between gene expression and epigenetic changes. Notably, the expression of genes that distinguish MOs from OPCs appears to be under a strong regulatory control by the H3K27ac modification in MOs but not in OPCs. Comparison of gene expression and H3K27ac between infants and adults uncovered numerous developmental changes in each cell type, which were linked to several biological processes, including cell proliferation and glutamate signaling. A striking example was a subset of histone genes that were highly active in infant samples but fully lost activity in adult brain. Our findings demonstrate a considerable rearrangement of the H3K27ac landscape that occurs during the differentiation of OPCs to MOs and during postnatal development of these cell types, which aligned with changes in gene expression. The uncovered regulatory changes justify further in-depth epigenetic studies of OPCs and MOs in development and disease.',\n",
       "  'subject mature oligodendrocytes rna seq dorsolateral prefrontal cortex donor id disease control sex male',\n",
       "  'subject oligodendrocyte precursor cells rna seq dorsolateral prefrontal cortex donor id disease control sex female',\n",
       "  'human'),\n",
       " 'GSE85121-10-vs-12-human up': (['Dysregulation of Lung Biology of Healthy Never Smokers Following Acute Inhalation of E-Cigarette Vapors'],\n",
       "  'Little is known about the health risks to humans associated with smoking or “vaping” electronic cigarettes (EC). Based on the knowledge that the adverse effects of cigarette smoking are observed in the dysregulation of lung biology long before clinical abnormalities manifest, we assessed the hypothesis that acute exposure to EC vapors, with or without nicotine, modifies the biology of the lung of healthy never smokers. No significant changes in clinical parameters were observed following this brief exposure to EC vapors. However, marked changes were observed in the transcriptome of small airway epithelium and alveolar macrophage compared to baseline for both exposure groups with and without nicotine, and the group inhaling the nicotine containing vapors had elevated plasma endothelial microparticle levels. While additional studies are needed to ascertain long-term health risks of EC, this study provides preliminary in vivo data suggesting that EC vapors likely dysregulates normal human lung homeostasis, at least at the biologic level.',\n",
       "  'dgm d7 individual smoking status time alveolar macrophage',\n",
       "  'dgm d7 sm individual smoking status time small airway epithelium brushing',\n",
       "  'human'),\n",
       " 'GSE178901-1-vs-2-human up': (['STING-driven interferon signaling triggers metabolic alterations in pancreas cancer cells visualized by [18F]FLT PET imaging'],\n",
       "  'Type I interferons (IFNs) are critical effectors of emerging cancer immunotherapies designed to activate pattern recognition receptors (PRRs). A challenge in the clinical translation of these new immuno-modulatory agents is the lack of pharmacodynamic biomarkers indicative of increased intratumoral IFN signaling following PRR activation. Positron emission tomography (PET) imaging enables the visualization of tissue metabolic activity but whether IFN signaling-induced alterations in tumor cell metabolism can be detected using PET has not been investigated. We found that IFN signaling augments pancreatic ductal adenocarcinoma (PDAC) cell nucleotide metabolism via transcriptional induction of metabolism-associated genes including thymidine phosphorylase (TYMP). TYMP catalyzes the first step in the catabolism of thymidine which competitively inhibits intratumoral accumulation of the nucleoside analog PET probe [18F]-fluorothymidine ([18F]FLT). Accordingly, IFN treatment stimulates cancer cell [18F]FLT uptake in the presence of thymidine and this effect is dependent on TYMP expression. In vivo, genetic activation of stimulator of interferon genes (STING), a PRR widely expressed in PDAC, enhances the [18F]FLT avidity of xenograft tumors. Treatment with a small molecule STING agonist triggers IFN signaling-dependent TYMP expression in PDAC cells and enhances both subcutaneous and orthotopic tumor [18F]FLT uptake in vivo following systemic treatment. These findings indicate that [18F]FLT accumulation in tumors is sensitive to IFN signaling and that [18F]FLT PET may serve as a novel pharmacodynamic biomarker for PRR agonist-based immunotherapies in PDAC and possibly other malignancies characterized by elevated STING expression.',\n",
       "  'suit2 control treated cell line pancreas cancer cells',\n",
       "  'suit2 100 u/ml ifnî² 6 hour cell line pancreas cancer cells human hours',\n",
       "  'human'),\n",
       " 'GSE100745-1-vs-3-human up': (['RNA-seq analysis of YFV-17D specific and total naive CD8 T cells in humans'],\n",
       "  'These 14 samples originate from peripheral blood mononuclear cells (PBMC) from donors at different times after they were vaccinated with the YF-17D vaccine.',\n",
       "  'd14 donor day 14 post yfv 17d sorted a2 ns4b214 tetramer+ cd8 cells pbmc',\n",
       "  'donor day post yfv 17d sorted a2 ns4b214 tetramer+ cd8 cells pbmc',\n",
       "  'human'),\n",
       " 'GSE156692-2-vs-4-mouse up': (['Identification of cellular context sensitive regulatory variation in mouse genomes'],\n",
       "  'RNA-seq and DHS mapping in hybrid mouse kidney, liver, lung, and B cells',\n",
       "  'rna seq lung strain',\n",
       "  'rna seq b cell strain',\n",
       "  'mouse'),\n",
       " 'GSE82043-4-vs-1-mouse up': (['Itaconate links inhibition of succinate dehydrogenase with macrophage metabolic remodeling and regulation of inflammation'],\n",
       "  'Remodeling of the tricarboxylic acid (TCA) cycle is a metabolic adaptation mechanism accompanying inflammatory macrophage activation. During this process, endogenous metabolites can adopt regulatory roles that govern specific aspects of inflammatory response, as recently shown for succinate, which regulates the downstream pro-inflammatory IL-1β-HIF1a axis. Itaconate is one of the most highly induced metabolites in activated macrophages, yet its functional significance remains unknown. Here, we show that itaconate modulates macrophage metabolism and effector functions via its effect on succinate dehydrogenase, by inhibiting conversion of succinate to fumarate. Through this action, itaconate exerts anti-inflammatory effects when administered in vitro and in vivo during macrophage activation and ischemia-reperfusion injury. Using newly generated Irg1-/- mice, which lack the ability to produce itaconate, we show that endogenous itaconate regulates succinate levels and function, changes in mitochondrial respiration, and inflammatory cytokine production during macrophage activation. These studies highlight itaconate as a major physiological regulator of the global metabolic rewiring and effector functions of inflammatory macrophages.',\n",
       "  'exp1 di strain background c57bl/6 /variation wt bone marrow derived macrophages treated dimethyl itaconate',\n",
       "  'exp2 wt lps ifng strain background c57bl/6 /variation bone marrow derived macrophages treated 24h',\n",
       "  'mouse'),\n",
       " 'GSE93526-7-vs-4-human up': (['Gene signature profiles in respiratory epithelium infected with nontuberculous mycobacteria'],\n",
       "  'The incidence of pulmonary nontuberculous mycobacterial (PNTM) disease is increasing, but host susceptibility factors are not fully understood. We infected air-liquid interface (ALI) primary respiratory epithelial cell cultures with Mycobacterium avium complex (MAC) or Mycobacterium abscessus (MAB) and performed transcriptome sequencing (RNA-Seq) to identify relevant gene expression differences. We used cells from 4 different donors in order to try to obtain generalizable data. The differentiated respiratory epithelial cells in ALI were infected with MAC or MAB at MOI of 100:1 or 1000:1, and RNA-seq was performed at 1 and 3 days after infection. We found downregulation of ciliary genes, including several identified with polymorphisms in previous PNTM cohorts. The cytokine IL-32, the superpathway of cholesterol biosynthesis and downstream targets within the IL-17 signaling pathway were all elevated. The integrin signaling pathway was more upregulated by MAB than MAC infection. Working with primary respiratory epithelial cells infected with nontuberculous mycobacteria at ALI, we identified ciliary function, cholesterol biosynthesis, chemokine production and the IL-17 pathway as major targets of host responses to infection. Some of these pathways may be amenable to therapeutic manipulation.',\n",
       "  '69f mab moi 1 infection differentiated respiratory epithelial cells age 69 years old gender female normal primary human bronchial (nhbe)',\n",
       "  '57f mac moi 1 infection differentiated respiratory epithelial cells age 57 years old gender female normal primary human bronchial (nhbe)',\n",
       "  'human'),\n",
       " 'GSE202852,GSE203385-1-vs-0-human up': (['Transcription and splicing regulation by NLRC5 shape the interferon response in human pancreatic β-cells [RNA-seq]'],\n",
       "  'IFNa is a key regulator of the dialogue between pancreatic β-cells and the immune system in early type 1 diabetes (T1D). IFNa up-regulates HLA class I expression in human β-cells fostering autoantigen presentation to the immune system. We observed by bulk and single cell RNA sequencing that exposure of human induced pluripotent-derived islet-like cells to IFNa induces expression of HLA class I and of other genes involved in antigen presentation, including the transcriptional activator NLRC5. We next evaluated the global role of NLRC5 in human insulin-producing EndoC-βH1 and human islets cells by RNA sequencing and targeted gene/protein determination. NLRC5 regulates expression of HLA class I expression and related genes and of chemokines. NLRC5 also mediates the effects of IFNa on alternative splicing, a generator of β-cell neoantigens, suggesting that it is a central mediator of the effects of IFNa on β-cells that contribute to trigger and amplify autoimmunity in T1D.',\n",
       "  'endoc î²h1 nt 24h sict cell line beta h1 untreated knock (kd) time',\n",
       "  'endoc î²h1 ifna 24h sinlrc5 cell line beta h1 (interferon alpha) knock (kd) time',\n",
       "  'human'),\n",
       " 'GSE189847-5-vs-0-human up': (['A transcriptomic analysis of the effects of macrophage polarization and endotoxin tolerance on the response to Salmonella'],\n",
       "  'Salmonella is an intracellular pathogen causing significant morbidity and mortality. Its ability to grow inside macrophages is important to virulence, and is dependent on the activation state of the macrophages. Classically activated M1 macrophages are non-permissive for Salmonella growth, while alternatively activated M2 macrophages are permissive for Salmonella growth. Here we showed that endotoxin-primed macrophages (MEP), such as those associated with sepsis, showed similar levels of Salmonella resistance to M1 macrophages after 2 hr of intracellular infection, but at the 4 hr and 24 hr time points were susceptible like M2 macrophages. To understand this mechanistically, transcriptome sequencing, RNA-Seq, was performed. This showed that M1 and MEP macrophages that had not been exposed to Salmonella, demonstrated a process termed here as primed activation, in expressing relatively higher levels of particular anti-infective genes and pathways, including the JAK-STAT pathway. In contrast, in M2 macrophages these genes and pathways were largely expressed only in response to infection. Conversely, in response to infection, M1 macrophages, but not MEP macrophages, modulated additional genes known to be associated with susceptibility to Salmonella infection, possibly contributing to the differences in resistance at later time points. Application of the JAK inhibitor Ruxolitinib before infection reduced resistance in M1 macrophages, supporting the importance of early JAK-STAT signalling in M1 resistance to Salmonella.',\n",
       "  'm2 un polarization il4 untreated donor monocyte derived macrophages',\n",
       "  'm2 inf polarization il4 typh donor monocyte derived macrophages',\n",
       "  'human'),\n",
       " 'GSE82043-1-vs-0-mouse dn': (['Itaconate links inhibition of succinate dehydrogenase with macrophage metabolic remodeling and regulation of inflammation'],\n",
       "  'Remodeling of the tricarboxylic acid (TCA) cycle is a metabolic adaptation mechanism accompanying inflammatory macrophage activation. During this process, endogenous metabolites can adopt regulatory roles that govern specific aspects of inflammatory response, as recently shown for succinate, which regulates the downstream pro-inflammatory IL-1β-HIF1a axis. Itaconate is one of the most highly induced metabolites in activated macrophages, yet its functional significance remains unknown. Here, we show that itaconate modulates macrophage metabolism and effector functions via its effect on succinate dehydrogenase, by inhibiting conversion of succinate to fumarate. Through this action, itaconate exerts anti-inflammatory effects when administered in vitro and in vivo during macrophage activation and ischemia-reperfusion injury. Using newly generated Irg1-/- mice, which lack the ability to produce itaconate, we show that endogenous itaconate regulates succinate levels and function, changes in mitochondrial respiration, and inflammatory cytokine production during macrophage activation. These studies highlight itaconate as a major physiological regulator of the global metabolic rewiring and effector functions of inflammatory macrophages.',\n",
       "  'exp2 wt lps ifng strain background c57bl/6 /variation bone marrow derived macrophages treated 24h',\n",
       "  'exp1 unst strain background c57bl/6 /variation wt bone marrow derived macrophages treated unstimulated',\n",
       "  'mouse'),\n",
       " 'GSE216638-4-vs-2-human up': (['Molecular signatures of anthroponotic cutaneous leishmaniasis in the lesions of patients infected with Leishmania tropica'],\n",
       "  'In this study employed a systems analysis approach to study molecular signatures of cutaneous leishmaniasis (CL)  caused by Leishmania tropica (L. tropica) in the skin lesions of ulcerativeCL (UCL) and non-ulcerative  CL( NUCL) patients.  Results from RNA-seq analysis determined shared and unique functional transcriptional pathways in the lesions of the UCL and Nucl patients. Several transcriptional pathways involved in inflammatory response were positively enriched in the CL lesions. These results enhance our understanding of human skin response to CL caused by L. tropica.',\n",
       "  'healthy skin disease state',\n",
       "  'acute stage /nucl skin disease state prior anti leishmanial',\n",
       "  'human'),\n",
       " 'GSE198321-5-vs-0-human up': (['Cell type-specific gene expression profiles in CNS of patients with DM1'],\n",
       "  'Myotonic dystrophy type 1 (DM1) is a multisystem genetic disorder involving the muscle, heart, and central nervous system (CNS). The pathogenesis of CNS symptoms prevalent in patients with DM1 remains unelucidated. To elucidate the CNS pathogenesis in DM1, we investigated cell type-specific abnormalities in cortical neurons, white matter glial cells, and spinal motor neurons of patients with DM1 via laser-capture microdissection(LCM).',\n",
       "  'ctrl cortical neurons disease control brain',\n",
       "  'ctrl white matter glial cells disease control brain',\n",
       "  'human'),\n",
       " 'GSE239662,GSE239663-1-vs-0-human up': (['Novel method of isolating nuclei of human oligodendrocyte precursor cells reveals substantial developmental changes in gene expression and H3K27ac histone modification. [RNA-Seq]'],\n",
       "  'Oligodendrocyte precursor cells (OPCs) generate differentiated mature oligodendrocytes (MOs) during development. In adult brain, OPCs replenish MOs in adaptive plasticity, neurodegenerative disorders, and after trauma. The ability of OPCs to differentiate to MOs decreases with age and is compromised in disease. Here we explored the cell specific and age-dependent differences in gene expression and H3K27ac histone mark in these two cell types. H3K27ac is indicative of active promoters and enhancers. We developed a novel flow-cytometry-based approach to isolate OPC and MO nuclei from human postmortem brain and profiled gene expression and H3K27ac in adult and infant OPCs and MOs genome-wide.In adult brain, we detected extensive H3K27ac differences between the two cell types with high concordance between gene expression and epigenetic changes. Notably, the expression of genes that distinguish MOs from OPCs appears to be under a strong regulatory control by the H3K27ac modification in MOs but not in OPCs. Comparison of gene expression and H3K27ac between infants and adults uncovered numerous developmental changes in each cell type, which were linked to several biological processes, including cell proliferation and glutamate signaling. A striking example was a subset of histone genes that were highly active in infant samples but fully lost activity in adult brain. Our findings demonstrate a considerable rearrangement of the H3K27ac landscape that occurs during the differentiation of OPCs to MOs and during postnatal development of these cell types, which aligned with changes in gene expression. The uncovered regulatory changes justify further in-depth epigenetic studies of OPCs and MOs in development and disease.',\n",
       "  'subject oligodendrocyte precursor cells rna seq dorsolateral prefrontal cortex donor id disease control sex female',\n",
       "  'subject mature oligodendrocytes rna seq dorsolateral prefrontal cortex donor id disease control sex female',\n",
       "  'human'),\n",
       " 'GSE197346-3-vs-7-human dn': (['Characterization of a new genetic signature associated with copy number variation that determines outcome in lung adenocarcinoma patients'],\n",
       "  'Background: The occurrence of lung adenocarcinoma (LUAD) is a complicated process, involving the genetic and epigenetic changes of proto-oncogenes and oncogenes. The objective of this study was to establish new predictive signatures of lung adenocarcinoma based on copy number variation (CNVs) and gene expression data.  Methods: Next-generation sequencing was implemented to obtain gene expression and CNV information. According to univariate, multivariate survival Cox regression analysis and Lasso analysis, the expression profiles of lung adenocarcinoma patients were screened and a risk score formula was established and experimentally validated in a local cohort. The model was evaluated by three independent cohorts, and then validated by clinical samples from LUAD patients.  Results: A total of 844 CNV-related differential expressed genes (CNV related DEGs) were identified. An CNV associated-six gene signature was dramatically linked to overall survival in lung adenocarcinoma samples from both training and validation groups. Functional enrichment analysis further revealed involvement of genes in p53 signaling pathway and cell cycle, as well as the mismatch repair pathway. Risk score is an independent marker considering clinical parameters and had better prediction in clinical sub-population. \\xa0The same signature also classified tumors tissues of clinical patients those with CNV detected from their corresponding non-tumorous tissues with an accuracy of 0.92.  Conclusions: In conclusion, we identified a new class of 6 CNV-related gene markers that may act as efficient prognostic predictors of lung adenocarcinoma, thus contributing to individualized treatment decisions in patients.',\n",
       "  'p2 41 age gender lung disease state adenocarcinoma adjacent normal',\n",
       "  'p2 41 age gender male lung stage n tumor smoking 1 disease state adenocarcinoma',\n",
       "  'human'),\n",
       " 'GSE131784,GSE184245-6-vs-5-mouse dn': (['Spen regulates X chromosome inactivation'],\n",
       "  'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'bglg gfp spoc control rnaseq cell line tx1072 mesc untreated strain bl/6j x cast eij spen aid',\n",
       "  'bglg gfp doxauxin rnaseq cell line tx1072 mesc dox+auxin24h strain bl/6j x cast eij spen aid',\n",
       "  'mouse'),\n",
       " 'GSE85121-2-vs-10-human dn': (['Dysregulation of Lung Biology of Healthy Never Smokers Following Acute Inhalation of E-Cigarette Vapors'],\n",
       "  'Little is known about the health risks to humans associated with smoking or “vaping” electronic cigarettes (EC). Based on the knowledge that the adverse effects of cigarette smoking are observed in the dysregulation of lung biology long before clinical abnormalities manifest, we assessed the hypothesis that acute exposure to EC vapors, with or without nicotine, modifies the biology of the lung of healthy never smokers. No significant changes in clinical parameters were observed following this brief exposure to EC vapors. However, marked changes were observed in the transcriptome of small airway epithelium and alveolar macrophage compared to baseline for both exposure groups with and without nicotine, and the group inhaling the nicotine containing vapors had elevated plasma endothelial microparticle levels. While additional studies are needed to ascertain long-term health risks of EC, this study provides preliminary in vivo data suggesting that EC vapors likely dysregulates normal human lung homeostasis, at least at the biologic level.',\n",
       "  'dgm d1 sm individual smoking status nonsmoker time small airway epithelium brushing',\n",
       "  'dgm d7 individual smoking status time alveolar macrophage',\n",
       "  'human'),\n",
       " 'GSE201767-0-vs-6-mouse dn': (['Transcriptome response of mouse mammary gland from age 12 to 30 months of age in Esr1 and CYP19A1 genetically engineered mouse models of breast cancer risk in the presence and absence of transgene induction'],\n",
       "  'Breast cancer is an age-related cancer in women with two peaks, one at 50 and one at 70 years of age. Here we used two conditional genetically engineered mouse models of breast cancer risk to study mammary gland transcriptional changes that occur as female mice age from 12 to 30 months of age, paralleling aging from 58 to 85 years of age in women. The two models express either mammary epithelial cell-targeted Estrogen Receptor (ER) alpha (Esr1) or Aromatase (CYP19A1A) over-expression beginning at age 12 or 18 months of age. Both of these risk factors increase estrogen pathway signaling, either directly though the receptor or by increasing local estrogen production. The goals of the study are to determine how quickly significant transcriptional changes occur in the mammary gland following transgene induction, to determine how the transcriptome becomes modified as the mice age past reproductive senescence through old and very old life stages, and to identify similarities and differences in the transcriptomes between the two risk conditions at different ages and conditions. The first specific objective of this transcriptome-based study is to identify significantly differentially expressed genes within each model between different ages and times of transgene induction. This includes transcriptional changes induced by 1 week, 6, 12 and 18 months of transgene expression with transgene expression initiated at age 12 months as well as transcriptional changes induced by 1 week and 6 months of transgene expression initiated at age 18 months. The second specific objective is to compare the two models at different ages (12, 18, 24 and 30 months of age) and different transgene induction times (6, 12 and 18 months) to identify differentially expressed genes between the models. Comparative control mice from each model include age 18- and 24-month-old mice without transgene induction.',\n",
       "  'mammary gland age 30m 18 months esr1 transgene induction biol rep mmtv rtta/tet op',\n",
       "  'mammary gland age 12m cyp19a1 transgene induction biol rep [arom012m arom mmtv rtta/tet op',\n",
       "  'mouse'),\n",
       " 'GSE204721,GSE204722-0-vs-1-human up': (['Global Epigenome and Transcriptome Maps of Human Trophoblast Cell Lineage Development [RNA-seq I]'],\n",
       "  'The extravillous trophoblast (EVT) cell lineage is a key feature of placentation and critical for spiral\\xa0artery remodeling and successful pregnancy. Our knowledge of transcriptional regulation driving EVT cell development is limited. Here, we mapped the transcriptome and epigenome landscape as well as chromatin interactions of human trophoblast stem (TS) cells and their transition into the differentiated EVT cell lineage. We identified that chromatin accessibility in intergenic regions was more extensive in EVT cells than in TS cells in the stem state, which is indicative of increased enhancer-driven gene regulation. Using reference epigenome annotation, we noted that 18% of the chromatin landscape in EVT cells was uncharted.\\xa0 We linked regulatory regions to their cognate target genes and characterized the three-dimensional organization of the TS cell functional genome by Hi-C. Integrational analysis of chromatin accessibility, long-range chromatin interactions, transcriptomic, and transcription factor binding motif enrichment enabled identification of transcription factors and regulatory mechanisms associated with EVT cell lineage development. Subsequent analyses elucidated functional roles for\\xa0TFAP2C,\\xa0EPAS1,\\xa0SNAI1, and\\xa0DLX6\\xa0in the regulation of EVT cell lineage development.\\xa0EPAS1\\xa0was identified as an upstream regulator of EVT cell transcription factors, including\\xa0SNAI1\\xa0and\\xa0DLX6, and was found to be upregulated in idiopathic recurrent pregnancy loss. Collectively, we have revealed activation of a dynamic regulatory network that provides a framework for understanding EVT cell specification in trophoblast cell lineage development and human placentation.',\n",
       "  'ct29 stem source human trophoblast cells cell line state',\n",
       "  'ct27 evt source human trophoblast stem cells cell line extravillous',\n",
       "  'human'),\n",
       " 'GSE178615-5-vs-7-human dn': (['Coordinated metabolic transitions and gene expression by NAD+ during adipogenesis.'],\n",
       "  'Adipocytes are the main cell type in adipose tissue, a critical regulator of metabolism, highly specialized in storing energy as fat. Adipocytes differentiate from multipotent mesenchymal stromal cells through adipogenesis. Using RNA-seq we explored the effect of manipulating NAD+ bioavailability during adipogenic differentiation from human mesenchymal stem cells. Cells were harvested either undifferentiated (hMSC) or after days 8 and 16 of adipogenic induction. During adipogenesis, cells were treated with either NAD+ (5 mM), or FK866 (1 nM) or EX527 (50 μM).',\n",
       "  'adipo day16 cell line mesenchymal differentiation stage adipogenic induction days 16 untreated human',\n",
       "  'adipo+nad+ day16 cell line mesenchymal differentiation stage adipogenic induction days 16 nad+ human',\n",
       "  'human'),\n",
       " 'GSE161752-1-vs-2-human up': (['Profiling the blood compartment of Hematopoietic Stem Cell Transplant patients during Human Cytomegalovirus reactivation'],\n",
       "  'Human cytomegalovirus (HCMV) reactivation is a leading infectious cause of morbidity and mortality in hematopoietic stem cell transplantation (HSCT) patients. HCMV reactivation is characterized by detection of viral DNA in the blood (DNAemia), however, the role of the blood compartment during HCMV reactivation is not well characterized. Using powerful molecular tools, we characterized HCMV infection in Peripheral Blood Mononuclear Cells (PBMCs) during DNAemia in HCMV reactivating HSCT patients. Surprisingly we found that all PBMC populations harbored extremely low levels of viral DNA and extremely low levels of viral transcripts. Remarkably, the viral transcripts detected in these samples were mainly immediate early genes indicating that these cells do not go through a full productive cycle. Analysis of the cellular transcriptome revealed characteristic transcriptional changes in high DNAemic samples implying the development of HCMV-specific T cells and inhibition of regulatory T cell function. Overall, our data indicate that DNAemia in HCMV reactivating HSCT patients is associated with distinct immune signatures but is not necessarily accompanied by substantial infection of PBMCs, and that PBMCs are not productively infected. These findings are important for understanding the role of the blood compartment in progression and control of HCMV during reactivation.',\n",
       "  'fibroblasts hpi fibroblast infected hcmv tb40',\n",
       "  'pbmc low viremia hsct',\n",
       "  'human'),\n",
       " 'GSE154428-1-vs-3-mouse up': (['Different effects of constitutive and induced microbiota modulation on microglia in a mouse model of Alzheimer’s disease'],\n",
       "  'It was recently revealed that gut microbiota promote amyloid-beta (Aβ) burden in mouse models of Alzheimer’s disease (AD). However, the underlying mechanisms when using either germ-free (GF) housing conditions or treatments with antibiotics (ABX) remained unknown. In this study, we show that GF and ABX-treated 5x familial AD (5xFAD) mice developed attenuated hippocampal Aβ pathology and associated neuronal loss, and thereby delayed disease-related memory deficits. While Ab production remained unaffected in both GF and ABX-treated 5xFAD mice, we noticed in GF 5xFAD mice enhanced microglial Aβ uptake at early stages of the disease compared to ABX-treated 5xFAD mice. Furthermore, RNA-sequencing of hippocampal microglia from SPF, GF and ABX-treated 5xFAD mice revealed distinct microbiota-dependent gene expression profiles associated with phagocytosis and altered microglial activation states. Taken together, we observed that constitutive or induced microbiota modulation in 5xFAD mice differentially controls microglial Aβ clearance mechanisms preventing neurodegeneration and cognitive deficits.',\n",
       "  'wt spf housing microglia age 16 weeks brain',\n",
       "  'fad abx housing microglia age 16 weeks brain',\n",
       "  'mouse'),\n",
       " 'GSE243329-0-vs-2-human up': (['The effect of MEL-6 on gene expression in LNCaP cells'],\n",
       "  'Prostate cancer patients benefit from treatment with androgen receptor signaling inhibitors such as AR antagonists. The emergence of resistance to the clinically applied AR antagonists opens up the need for development of alternative AR antagonists. We show here the development of a novel AR antagonists that is structurally different from enzalutamide. Moreover, MEL-6 remains active in the presence of several AR mutations (T877A, W741C and F876L) that are found in patients resistant to hydroxyflutamide, bicalutamide and enzalutamide.  To validate the androgen receptor (AR) as the target of our experimental AR antagonist, MEL-6, we investigated the effect of MEL-6 treatment on the prostate cancer cell line, LNCaP.',\n",
       "  'lncap vehicle biol cell line prostate cancer',\n",
       "  'lncap r1881 biol cell line prostate cancer 1nm',\n",
       "  'human'),\n",
       " 'GSE246229,GSE246230-0-vs-2-human up': (['HNF1αmaintains pancreaticα and β cell functionsin primary human islets [RNA-seq]'],\n",
       "  'Transcriptomic profiling by high-throughput sequencing of primary human alpha and beta cells following HNF1A loss of function.',\n",
       "  'beta hnf1akd cadaveric pancreatic cell line primary islet cells developmental stage adult /variation hnf1a knockdown',\n",
       "  'alpha hnf1akd cadaveric pancreatic cell line primary islet cells developmental stage adult /variation hnf1a knockdown',\n",
       "  'human'),\n",
       " 'GSE197346-4-vs-7-human dn': (['Characterization of a new genetic signature associated with copy number variation that determines outcome in lung adenocarcinoma patients'],\n",
       "  'Background: The occurrence of lung adenocarcinoma (LUAD) is a complicated process, involving the genetic and epigenetic changes of proto-oncogenes and oncogenes. The objective of this study was to establish new predictive signatures of lung adenocarcinoma based on copy number variation (CNVs) and gene expression data.  Methods: Next-generation sequencing was implemented to obtain gene expression and CNV information. According to univariate, multivariate survival Cox regression analysis and Lasso analysis, the expression profiles of lung adenocarcinoma patients were screened and a risk score formula was established and experimentally validated in a local cohort. The model was evaluated by three independent cohorts, and then validated by clinical samples from LUAD patients.  Results: A total of 844 CNV-related differential expressed genes (CNV related DEGs) were identified. An CNV associated-six gene signature was dramatically linked to overall survival in lung adenocarcinoma samples from both training and validation groups. Functional enrichment analysis further revealed involvement of genes in p53 signaling pathway and cell cycle, as well as the mismatch repair pathway. Risk score is an independent marker considering clinical parameters and had better prediction in clinical sub-population. \\xa0The same signature also classified tumors tissues of clinical patients those with CNV detected from their corresponding non-tumorous tissues with an accuracy of 0.92.  Conclusions: In conclusion, we identified a new class of 6 CNV-related gene markers that may act as efficient prognostic predictors of lung adenocarcinoma, thus contributing to individualized treatment decisions in patients.',\n",
       "  'p2 age gender lung disease state adenocarcinoma adjacent normal',\n",
       "  'p2 41 age gender male lung stage n tumor smoking 1 disease state adenocarcinoma',\n",
       "  'human'),\n",
       " 'GSE178615-0-vs-7-human dn': (['Coordinated metabolic transitions and gene expression by NAD+ during adipogenesis.'],\n",
       "  'Adipocytes are the main cell type in adipose tissue, a critical regulator of metabolism, highly specialized in storing energy as fat. Adipocytes differentiate from multipotent mesenchymal stromal cells through adipogenesis. Using RNA-seq we explored the effect of manipulating NAD+ bioavailability during adipogenic differentiation from human mesenchymal stem cells. Cells were harvested either undifferentiated (hMSC) or after days 8 and 16 of adipogenic induction. During adipogenesis, cells were treated with either NAD+ (5 mM), or FK866 (1 nM) or EX527 (50 μM).',\n",
       "  'adipo day8 cell line mesenchymal differentiation stage adipogenic induction days 8 untreated human',\n",
       "  'adipo+nad+ day16 cell line mesenchymal differentiation stage adipogenic induction days 16 nad+ human',\n",
       "  'human'),\n",
       " 'GSE136598,GSE136600-0-vs-3-human up': (['Direct Induction of Trophoblast Stem Cells from Human Fibroblasts (RNA-seq)'],\n",
       "  'Transcription profile analysis revealed similarity/dissimilarity between human blastocyst derived trophoblast stem cells (hbdTSCs), human induced trophoblast stem cells (hiTSCs), human foreskin fibroblasts (HFFs) and pluripotent stem cells (hESCs, hiPSCs)',\n",
       "  'human stem cell line',\n",
       "  'bdhtsc human bastocyst derived trophoblast stem cells line',\n",
       "  'human'),\n",
       " 'GSE168430,GSE168431-9-vs-6-mouse up': (['Identification and drug-induced reversion of molecular signatures of Alzheimer´s disease onset and progression in AppNL-G-F, AppNL-F and 3xTg-AD mouse models (RNA-seq: WT/NLF/NLGF)'],\n",
       "  \"Alzheimer's disease (AD) is the most common form of dementia. Over fifty years of intense research have revealed many key elements of the biology of this neurodegenerative disorder. However, our understanding of the molecular bases of the disease is still incomplete, and the available medical treatments for AD are mainly symptomatic and hardly effective. Indeed, the robustness of biological systems have revealed that the modulation of a single target is unlikely to yield the desired outcome, and we should move from gene-centric to   systemic therapeutic strategies. Here, we present the complete characterization of three murine models of AD at different stages of the disease (i.e. onset, progression and advanced). To identify genotype-to-phenotype relationships, we combine the cognitive assessment of these mice with histological analyses and a full transcriptional and protein quantification profiling from hippocampus. Comparison between the gene and protein expression trends observed in AD progression and physiological ageing exposed certain commonalities, such as the upregulation of microglial and inflammation markers. However, although there is an accelerated ageing in AD models, there are other factors specifically associated with A_ pathology. Despite the clear correlation between mRNA and protein levels of the dysregulated genes, we discovered a few proteins whose abundance increases with AD progression, while the corresponding transcript levels remain stable. Indeed, we show that at least two of these proteins, namely lfit3 and Syt11, co-localize with A_ plaques in the brain. Finally, we derive specific A_-related molecular AD signatures, and we look for drugs able to globally revert them. We found two NSAIDs (dexketoprofen and etodolac) and two anti-hypertensives (penbutolol and bendroflumethiazide) that overturn the cognitive impairment in AD mice   while reducing A_ plaques in the hippocampus and partially restoring the physiological levels of AD signature   genes to wild-type levels.\",\n",
       "  'wt 3m hippocampal sample mouse id processing age condition hippocampus',\n",
       "  'ki 9m hippocampal sample app nl g f mouse id processing age condition hippocampus',\n",
       "  'mouse'),\n",
       " 'GSE120084-0-vs-1-human up': (['Macrophage Ad5f35 PhenoMap'],\n",
       "  'Human monocyte derived macrophages (GM-CSF differentiated) were stimulated with control media, IFNy/LPS (M1), IL4 (M2), empty vector Ad5f35 (MOI 1000), or anti-HER2 CAR Ad5f35 (MOI 1000). RNA was harvested 48 hours post transduction with Ambion RiboPure RNA Purification Kit (Thermo Fisher) and RNA-Seq libraries were generated using TruSeq RNA Library Prep Kit (Illumina). Libraries were sequenced on 75bp single-end reads using a NextSeq (Illumina).',\n",
       "  'control media rna seq sample monocyte derived macrophage untreated',\n",
       "  'anti her2 car rna seq sample monocyte derived macrophage treated ad5f35',\n",
       "  'human'),\n",
       " 'GSE131840-2-vs-1-human up': (['Transcriptomic analysis of effect of oncolytic measles virus (MV) on transformed and non-transformed bone marrow-derived mesenchymal stem cells (MSCs)'],\n",
       "  'The mechanisms by which oncolytic vaccine strains of measles virus exert their tumor selectivity are not well elucidated.',\n",
       "  '5h uninfected cell line mv susceptible msc (5h) infection bone marrow derived mesenchymal stromal cells',\n",
       "  '5h mv cell line susceptible msc (5h) infection infected bone marrow derived mesenchymal stromal cells',\n",
       "  'human'),\n",
       " 'GSE223100,GSE223842-4-vs-11-mouse dn': (['pDC-like cells are pre-DC2 and require Klf4 to control homeostatic CD4 T cells'],\n",
       "  'Plasmacytoid dendritic cells (pDCs) have been shown to play an important role during immune responses, ranging from initial viral control through the production of type I interferons (IFNs) to antigen presentation. However, recent evidence uncovered unexpected heterogeneity among pDCs. Notably we identified a new subset, referred to as pDC-like cells, that resembles pDCs but shares cDC features. Here we show that this subset is a circulating progenitor distinct from common DC progenitors (CDP), with prominent cDC2 precursor potential. This precursor subset responds to homeostatic cytokines, such as macrophage colony stimulating factor (M-CSF) by expanding and differentiating into cDC2 that efficiently prime Th17 cell. Development of pDC-like cells into CX3CR1+ESAM- cDC2b but not CX3CR1- ESAM+ cDC2a requires the transcription factor KLF4. Finally, we show that under homeostatic conditions this developmental pathway regulates the immune threshold at barrier sites, by controlling the pool of Th17 cells within the skin draining lymph nodes.',\n",
       "  'mln pdcs mesenteric lymph nodes (mln) replicate',\n",
       "  'spl spleen (spl) replicate',\n",
       "  'mouse'),\n",
       " 'GSE152782-0-vs-5-human up': (['Cell-intrinsic Innate Immune Responses against Chikungunya Virus in a Human Ex Vivo Synovial Fibroblast Model'],\n",
       "  'Purpose: Human primary synovial fibroblasts extracted from the knee were established as an ex vivo infection model for Chikungunya Virus. The cell-intrinsic immune response was determined using RNA-seq of infected and uninfected fibroblasts from healthy donors and donors with an osteoarthritic background.',\n",
       "  'c9 state osteoarthritic donor oasf knee synovial fibroblast',\n",
       "  'chikv state osteoarthritic donor oasf knee synovial fibroblast',\n",
       "  'human'),\n",
       " 'GSE73721-0-vs-8-human dn': (['RNA-Seq of human astrocytes'],\n",
       "  \"Astrocytes were purified from fetal and adult human brain tissue using an immunopanning method with the HepaCAM antibody. Samples were taken from otherwise 'healthy' pieces of tissue, unless otherwise specified.\",\n",
       "  'ctx myeloid age years old cell subtype cd45 positive cells adult temporal lobe',\n",
       "  'ctx astro age years old cell subtype hepacam positive cells astrocyte adult temporal lobe',\n",
       "  'human'),\n",
       " 'GSE96055-2-vs-5-mouse up': (['Time-course analysis of microglia-specific RNA-seq in two severities of spinal cord injury'],\n",
       "  'RNA-seq analysis of microglia following spinal cord injury demonstrated that microglia response is driven by time after injury but not lesion severity.',\n",
       "  'mgni /variation cx3cr1+/egfp injury non injured time start point microglia',\n",
       "  'mgft2w /variation cx3cr1+/egfp injury full transection time 2 weeks microglia 2wk',\n",
       "  'mouse'),\n",
       " 'GSE201767-6-vs-13-mouse up': (['Transcriptome response of mouse mammary gland from age 12 to 30 months of age in Esr1 and CYP19A1 genetically engineered mouse models of breast cancer risk in the presence and absence of transgene induction'],\n",
       "  'Breast cancer is an age-related cancer in women with two peaks, one at 50 and one at 70 years of age. Here we used two conditional genetically engineered mouse models of breast cancer risk to study mammary gland transcriptional changes that occur as female mice age from 12 to 30 months of age, paralleling aging from 58 to 85 years of age in women. The two models express either mammary epithelial cell-targeted Estrogen Receptor (ER) alpha (Esr1) or Aromatase (CYP19A1A) over-expression beginning at age 12 or 18 months of age. Both of these risk factors increase estrogen pathway signaling, either directly though the receptor or by increasing local estrogen production. The goals of the study are to determine how quickly significant transcriptional changes occur in the mammary gland following transgene induction, to determine how the transcriptome becomes modified as the mice age past reproductive senescence through old and very old life stages, and to identify similarities and differences in the transcriptomes between the two risk conditions at different ages and conditions. The first specific objective of this transcriptome-based study is to identify significantly differentially expressed genes within each model between different ages and times of transgene induction. This includes transcriptional changes induced by 1 week, 6, 12 and 18 months of transgene expression with transgene expression initiated at age 12 months as well as transcriptional changes induced by 1 week and 6 months of transgene expression initiated at age 18 months. The second specific objective is to compare the two models at different ages (12, 18, 24 and 30 months of age) and different transgene induction times (6, 12 and 18 months) to identify differentially expressed genes between the models. Comparative control mice from each model include age 18- and 24-month-old mice without transgene induction.',\n",
       "  'mammary gland age 12m cyp19a1 transgene induction biol rep [arom012m arom mmtv rtta/tet op',\n",
       "  'mammary gland age 30m 18 months cyp19a1 transgene induction biol rep s8] mmtv rtta/tet op',\n",
       "  'mouse'),\n",
       " 'GSE185245-9-vs-1-human up': (['TEFM variants impair mitochondrial transcription elongation causing childhood-onset neurological disease'],\n",
       "  \"We report that TEFM variants are associated with mitochondrial respiratory chain deficiency. We show that patient's cells have reduced levels of promoter distal mitochondrial RNA transcripts.\",\n",
       "  'control subject status/id / fibroblasts molecule subtype rrna depleted rna',\n",
       "  'muscle control subject status/id / molecule subtype rrna depleted rna',\n",
       "  'human'),\n",
       " 'GSE130117-0-vs-1-human up': (['Simultaneous measurement of DNA repair activities and mRNA abundance in single cells'],\n",
       "  'New methods that measure the static abundance of DNA, RNA, and proteins in single cells are rapidly transforming biology. But these methods do not provide insight into dynamic cellular activities, precluding a complete understanding of cellular heterogeneity. We developed a method to measure enzymatic activities in single cells and used it to simultaneously measure DNA repair enzyme activities and mRNA expression. By including polyadenylated DNA hairpin substrates with defined lesions in a single-cell mRNA sequencing experiment, we measured repair activities by capturing repair intermediates and products catalyzed by activities in single cells.',\n",
       "  'mrna 1 mix ungko hap1 cell line rnaseh2cko',\n",
       "  'mrna human pbmcs',\n",
       "  'human'),\n",
       " 'GSE184536-4-vs-7-human dn': (['The NF-κB transcriptional footprint is essential for SARS-CoV-2 replication'],\n",
       "  'SARS-CoV-2, the etiological agent of COVID-19, is characterized by a delay in Type I interferon (IFN-I)-mediated antiviral defenses alongside robust cytokine production. Here we investigate the underlying molecular basis for this imbalance and implicate virus-mediated activation of NF-κB in the absence of other canonical IFN-I-related transcription factors. Epigenetic and single cell transcriptomic analyses show a selective NF-κB signature that was most prominent in infected cells. Disruption of NF-κB signaling through the silencing of the NF-κB transcription factors p65 or p50 resulted in loss of virus replication that was rescued upon reconstitution. These findings could be further corroborated with the use of NF-κB inhibitors, which reduced SARS-CoV-2 replication in vitro. These data suggest that the robust cytokine production in response to SARS-CoV-2, despite a diminished IFN-I response, is the product of a dependency on NF-κB for viral replication.',\n",
       "  'mrnaseq ifnb 12h lung adenocarcinoma virus mock 12 h ifn î² (100 u/ml) a549 ace2 cells',\n",
       "  'mrnaseq mock 12h lung adenocarcinoma virus 12 h a549 ace2 cells',\n",
       "  'human'),\n",
       " 'GSE124975-0-vs-2-human dn': (['Salinomycin induces expression of ER and Golgi apparatus related genes in EMT cell model'],\n",
       "  'Epithelial-to-mesenchymal transition (EMT) gives rise to cells with properties similar to cancer stem cells (CSCs) that drive tumor metastasis. Recently, a screening of a large compound library on a breast EMT model has identified salinomycin, a K+/H+ ionophore, as a highly selective drug towards CSCs. We used the same EMT model to show that salinomycin targets Golgi apparatus. We have performed RNA-seq analysis on HMLE-Twist and HMLE-pBp cells (EMT and non-EMT) that were either mock treated or treated for 24h with micro molar concentration (0.2uM) of salinomycin. Salinomycin induced expression of genes enriched by known ER and Golgi stressors.',\n",
       "  'cell line hmle pbp breast phenotype nonemt mock',\n",
       "  'cell line hmle twist breast phenotype emt mock',\n",
       "  'human'),\n",
       " 'GSE158152,GSE158156-1-vs-7-mouse dn': ([\"The microglial transcriptome of a novel murine model of Alzheimer's Disease: FACS-isolated microglia from heterozygous and homozygous App-SAA mice and WT littermate controls.\"],\n",
       "  \"We created a novel murine model of Alzheimer's Disease using a knock-in strategy to humanize the sequence of the murine App gene and introduced three familial AD (FAD) mutations, Swedish (Swedish  (KM670/671NL), Arctic (E693G)) and Austrian (T712I).  We characterized the effects of these genetic modifications on the transcriptome of FACS-isolated microglia from 8-month-old App-SAA mice. Numerous genes were differentially expressed between cells from homozygous App-SAA animals compared to those from WT littermates. For example, we observed up-regulation of Disease-associated microglia (DAM) genes. In contrast, the transcriptome of microglia from heterozygous App-SAA animals broadly resembles that of their WT counterparts.\",\n",
       "  'drn id group microglia hom lib size norm factor age 8 unit months sex library replicate brain (cortex & hippocampus)',\n",
       "  'drn id group microglia het lib size norm factor age 8 unit months day3 sex female library replicate brain (cortex & hippocampus)',\n",
       "  'mouse'),\n",
       " 'GSE201767-0-vs-4-mouse dn': (['Transcriptome response of mouse mammary gland from age 12 to 30 months of age in Esr1 and CYP19A1 genetically engineered mouse models of breast cancer risk in the presence and absence of transgene induction'],\n",
       "  'Breast cancer is an age-related cancer in women with two peaks, one at 50 and one at 70 years of age. Here we used two conditional genetically engineered mouse models of breast cancer risk to study mammary gland transcriptional changes that occur as female mice age from 12 to 30 months of age, paralleling aging from 58 to 85 years of age in women. The two models express either mammary epithelial cell-targeted Estrogen Receptor (ER) alpha (Esr1) or Aromatase (CYP19A1A) over-expression beginning at age 12 or 18 months of age. Both of these risk factors increase estrogen pathway signaling, either directly though the receptor or by increasing local estrogen production. The goals of the study are to determine how quickly significant transcriptional changes occur in the mammary gland following transgene induction, to determine how the transcriptome becomes modified as the mice age past reproductive senescence through old and very old life stages, and to identify similarities and differences in the transcriptomes between the two risk conditions at different ages and conditions. The first specific objective of this transcriptome-based study is to identify significantly differentially expressed genes within each model between different ages and times of transgene induction. This includes transcriptional changes induced by 1 week, 6, 12 and 18 months of transgene expression with transgene expression initiated at age 12 months as well as transcriptional changes induced by 1 week and 6 months of transgene expression initiated at age 18 months. The second specific objective is to compare the two models at different ages (12, 18, 24 and 30 months of age) and different transgene induction times (6, 12 and 18 months) to identify differentially expressed genes between the models. Comparative control mice from each model include age 18- and 24-month-old mice without transgene induction.',\n",
       "  'mammary gland age 30m 18 months esr1 transgene induction biol rep mmtv rtta/tet op',\n",
       "  'mammary gland age 12m 1 week esr1 transgene induction biol rep [cerm1wdoxy12m cerm mmtv rtta/tet op',\n",
       "  'mouse'),\n",
       " 'GSE111930-6-vs-11-human dn': (['An endogenous retrovirus-derived long non-coding RNA promotes fetal cardiomyocyte migration in primates'],\n",
       "  'The protein-coding gene networks regulating heart development are well known. Less understood are the roles of long non-coding RNAs (lncRNAs), many of which are poorly conserved due to their origins from transposable elements (TEs) such as endogenous retroviruses (ERVs). Here, we report hundreds of ERV elements from the primate-specific MER41 family are expressed in pluripotent cell-derived cardiomyocytes and fetal heart, some of which are regulated by the cardiogenic transcription factor TBX5. The most significant of these are located within BANCR, an lncRNA exclusively expressed in primate fetal cardiomyocytes in normal physiology. Surprisingly, BANCR promotes cellular movement in fetal cardiomyocytes, which is due in part to Rho GTPase and ephrin receptor signaling. We postulate that ERV-derived BANCR is a recent evolutionary mechanism for enabling larger heart sizes in primates, and underscores the remarkable retroviral origins of humans.',\n",
       "  'human fetal left ventricle fetus gender',\n",
       "  'human fetal fetus gender',\n",
       "  'human'),\n",
       " 'GSE119336,GSE119337-2-vs-3-human up': (['RNA over-editing leads to aggressiveness of intrahepatic cholangiocarcinoma [RNA-Seq]'],\n",
       "  'Alteration in RNA editing has been connected to tumor progression and many other important human diseases. However, the role of RNA editing in intrahepatic cholangiocarcinoma (ICC) remains unclear. Here we conducted a transcriptome-wide investigation in ICCs, and revealed ADAR-associated over-editing occurred uniformly in ICCs. Collectively, these results reveal a previously unrecognized role of RNA editing in malignant transformation into ICC, and ADAR1 inhibition may obviate progression and relapse of this malignancy.',\n",
       "  'gender male age intrahepatic cholangiocarcinoma tumor',\n",
       "  'gender age adjacent non tumor liver',\n",
       "  'human'),\n",
       " 'GSE105152,GSE105154-0-vs-2-human up': (['Trnascriptome analysis of HeLa cells infected with rTHOV-wt, -dML, -SW mutant or mock-treated'],\n",
       "  'The goal of the study was to compare transcriptome changes in HeLa cells after infection with recombinant Thogoto virus (wild-type, ML deletioin mutant or ML SW mutant not able to interact wiith TFIIB. While wild-type virus is able to inhibit inflammatory genes, ML deletion mutant and TFIIB-non-interacting mutant lose this effect on gene transcription.',\n",
       "  'mock infected control cell line hela s3 passage 5 15 (control)',\n",
       "  'rthov delta ml infection cell line hela s3 passage 5 15 infected',\n",
       "  'human'),\n",
       " 'GSE149631-10-vs-13-human dn': (['Targeted determination of pri-miRNA processing'],\n",
       "  'Targeted sequencing at Microprocessor cleavage sites in pri-miRNAs to determine processing efficiency of several pri-miRNAs in vivo in one experiment',\n",
       "  'rna normal liver d4 disease state healthy',\n",
       "  'rna hcc a1 liver (hepatocellular carcinoma) disease state cancer',\n",
       "  'human'),\n",
       " 'GSE176271,GSE176289-4-vs-3-human dn': (['Integrative analysis of DNA methylation and microRNA expression reveals mechanisms of disparity in hepatocellular carcinoma [RNA-Seq]'],\n",
       "  'We evaluate the epigenetic mechanisms of racial disparity in HCC through an integrated analysis of DNA methylation, miRNA, and combined regulation of gene expression. Specifically, we acquired DNA methylation, mRNA-seq, and miRNA-seq data through the analysis of tumor and adjacent normal liver tissues from African Americans (AA) and European Americans (EA) with HCC.',\n",
       "  'disease status adjacent normal race aa liver',\n",
       "  'disease status hcc race aa liver',\n",
       "  'human'),\n",
       " 'GSE165238-1-vs-2-human dn': (['Tumor-associated hematopoietic stem and progenitor cells positively linked to glioblastoma progression'],\n",
       "  'Brain tumors are typically immunosuppressive and refractory to immunotherapies for reasons that remain poorly understood. The unbiased profiling of immune cell types in the tumor microenvironment may reveal immunologic networks affecting therapy and course of disease. Here we identify and validate the presence of hematopoietic stem and progenitor cells (HSPCs) within tumor tissues that are positively linked to glioblastoma malignancy. In contrast to the medullary hematopoietic compartment, tumor-associated HSPCs are comprised of a majority of CD38- immature cells, such as hematopoietic stem cells and multipotent progenitors, express genes related to glioblastoma progression and display patterns of accelerated cell cycle. When cultured ex vivo, tumor-associated HSPCs form prevalently myeloid colonies, suggesting potential in situ myelopoiesis. In experimental models, HSPCs promote tumor cell proliferation, expression of the immune checkpoint PD-L1 and secretion of tumor promoting cytokines such as IL-6, IL-8 and CCL2, indicating concomitant support of both malignancy and immunosuppression. Additionally, the frequency of tumor-associated HSPCs is prognostic for patient survival and correlate with immunosuppressive phenotypes. These findings identify a new element in the complex landscape of glioblastoma that may serve as an independent prognostic factor or target for brain tumor immunotherapies.',\n",
       "  'tumor free sample',\n",
       "  'blood',\n",
       "  'human'),\n",
       " 'GSE115219-1-vs-3-mouse dn': (['TRAP-seq following Pdx1 depletion'],\n",
       "  'TRAP performed in GFP-RPL10A Min6 cells transfected with siRNA targeting Pdx1 or a non-targeting (NT) control. IP RNA was isolated by TRAP protocol and input lysate was used to isolate Total RNA. RNA-seq was performed on both IP and Total RNA.',\n",
       "  'control ip min6 cells passage p29 sirna nt mouse insulinoma cell line',\n",
       "  'pdx1 ip min6 cells passage p29 sirna mouse insulinoma cell line',\n",
       "  'mouse'),\n",
       " 'GSE110146-3-vs-8-human dn': (['Transcriptomic-based functional characterization of adverse events following lymphatic filariasis treatment'],\n",
       "  'Intro: Lymphatic filariasis (LF) is a neglected tropical disease caused by the nematode parasite Wuchereria bancrofti. The primary tool used by the Global Program to Eliminate LF is mass drug administration (MDA), and some 500 million people take the medications each year. Mild to moderate adverse events (AEs) are common after LF treatment, and these pose a challenge for the LF elimination program. To better understand the pathogenesis of AEs, we studied patients from a LF treatment trial in Côte d’Ivoire.',\n",
       "  'post buffy coat adverse event moderate pre age sex male village microfilaria/ml arm',\n",
       "  'pre buffy coat adverse event post age sex male village microfilaria/ml arm ivm/alb',\n",
       "  'human'),\n",
       " 'GSE152782-2-vs-5-human up': (['Cell-intrinsic Innate Immune Responses against Chikungunya Virus in a Human Ex Vivo Synovial Fibroblast Model'],\n",
       "  'Purpose: Human primary synovial fibroblasts extracted from the knee were established as an ex vivo infection model for Chikungunya Virus. The cell-intrinsic immune response was determined using RNA-seq of infected and uninfected fibroblasts from healthy donors and donors with an osteoarthritic background.',\n",
       "  'mock state osteoarthritic donor oasf knee synovial fibroblast',\n",
       "  'chikv state osteoarthritic donor oasf knee synovial fibroblast',\n",
       "  'human'),\n",
       " 'GSE185670-0-vs-1-human dn': (['Transcriptome profiling of hESC-derived endoderm progenitors by RNA-seq'],\n",
       "  'Transcriptome profiling of hESC-derived endoderm progenitors by RNA-seq. RNA-seq analysis during the stepwise progression of hESC toward the pancreatic progenitors, at five defined stages of differentiation and expansion: hESC, ADE, VFG.p3, VFG.p6. PE.',\n",
       "  'rna seq hues4 170 3 vfg derived pe cell line pdx1 egfp reporter facs sorted population positive cells pancreatic endoderm progenitor',\n",
       "  'rna seq hues4 170 3 hesc derived vfg passage cell line pdx1 egfp reporter facs sorted population cd184 pecy7 cd117 apc double positive cells ventral foregut progenitor',\n",
       "  'human'),\n",
       " 'GSE188992-1-vs-2-mouse dn': (['Bulk RNAseq of Mouse Splenic Dendritic Cell Subsets'],\n",
       "  'Dendritic cells (DCs) are comprised of a heterogenous population. Here we characterize the transcriptomic profiles of mouse splenic DC subsets at steady state to highlight key differences between subsets.',\n",
       "  'b6 pdc strain c57bl/6j state steady spleen dc cell subset',\n",
       "  'b6 cd8neg cdc2 strain c57bl/6j state steady spleen dc cell subset',\n",
       "  'mouse'),\n",
       " 'GSE208420,GSE208424-1-vs-0-human dn': (['RNA-sequencing analysis to identify genes regulated by MYCN, WDR5 and G9a in neuroblastoma cell line IMR32'],\n",
       "  'In this study we identified genes regulated by MYCN, WDR5 and G9a in IMR32 cells',\n",
       "  'imr32 sictrl 48h cell line neuroblastoma cells',\n",
       "  'imr32 siwdr5 48h cell line neuroblastoma cells',\n",
       "  'human'),\n",
       " 'GSE208394-0-vs-11-mouse up': (['Protein Kinase C Delta Regulates Mononuclear Phagocytes by Repressing Type I and II Interferon Signaling and Hinders Response to Immunotherapy in Cancer'],\n",
       "  'Mononuclear phagocytes (MPs) comprise monocytes, macrophages and dendritic cells and play a crucial role in tissue homeostasis. However, accumulating evidence suggests that mononuclear phagocytes contribute to tumor progression and resistance to checkpoint blockade. Protein kinase C delta (PKCδ) is a serine/threonine kinase that plays a crucial role in regulating the immune response by preventing autoimmunity. However, the role of PKCδ in antitumor immunity is unknown. In this study, we found that PKCδ is abundantly expressed by MPs in several human and mouse tumors. PKCδ-/- mice were more resistant to growth of various cancers compared to wild-type mice and were more responsive to anti-PD-1 immunotherapy. Furthermore, we found that tumors from PKCδ-/- mice harbor a Th-1-skewed immune response including increased antigen cross-presentation and T cell activation. Depletion of MPs in vivo altered tumor growth in wild-type mice, but not in PKCδ-/- mice. In addition, co-injection of PKCδ-/- M2-like macrophages with cancer cells into wild-type mice markedly delayed tumor growth and significantly increased intratumoral T cell activation compared to wild-type M2-like macrophages co-injected with cancer cells. Finally, intrinsic loss of PKCδ functionally reprogrammed macrophages and dendritic cells by promoting their antigen presenting and cross-presenting capacity and triggered Type I and II interferon signaling. Our results suggest that PKCδ can be targeted to reprogram mononuclear phagocytes and augment checkpoint blockade efficacy.',\n",
       "  'bmdm primary mouse cells tumor bone marrow macrophages strain c57bl/6j pkc delta knockout',\n",
       "  'm1 primary mouse cells tumor bone marrow macrophages polarized lps+ifnî³ strain c57bl/6j pkc delta knockout',\n",
       "  'mouse'),\n",
       " 'GSE80463,GSE82292-0-vs-2-human dn': (['Divergent expression and metabolic functions of human glucuronosyltransferases through alternative splicing'],\n",
       "  'Purpose: Maintenance of cellular homeostasis and xenobiotics detoxification relies on the glucuronidation pathway mediated by 19 human UDP-glucuronosyltransferase enzymes (UGTs) encoded by 10 highly homologous genes. Recent evidence suggests that alternative splicing largely expands the human UGT transcriptome.',\n",
       "  'liver normal (n=3 donors) sex pooled male female type biopsy',\n",
       "  'kidney tumor (n=3 donors) sex pooled male female type tumoral biopsy',\n",
       "  'human'),\n",
       " 'GSE155109-0-vs-1-human dn': (['Single cell atlas identifies lipid-processing and immunomodulatory endothelial cells in healthy and malignant breast'],\n",
       "  'Since a detailed inventory of endothelial cell (EC) heterogeneity in breast cancer (BC) is lacking, we perform single cell RNA-sequencing of 26,515 cells (including 8,433 ECs) from 9 BC patients and compare them to published EC taxonomies from lung tumors. Angiogenic ECs are phenotypically similar, while other EC subtypes are different. Predictive interactome analysis reveals known but also previously unreported receptor-ligand interactions between ECs and immune cells, suggesting an involvement of breast EC subtypes in immune responses. We also identify a capillary EC subtype (LIPEC (Lipid Processing EC)), which expresses genes involved in lipid processing that are regulated by PPAR-gamma and is more abundant in peri-tumoral breast tissue. Retrospective analysis of 4,648 BC patients reveals that treatment with metformin (an indirect PPAR-gamma agonist) provides long-lasting clinical benefit and is positively associated with LIPEC abundance. Our findings warrant further exploration of this LIPEC/PPAR-gamma link for BC treatment.',\n",
       "  'breast tumor tec sample type cancer resection specimen condition id',\n",
       "  'breast stroma sample type cancer resection specimen condition id',\n",
       "  'human'),\n",
       " 'GSE168430,GSE168431-8-vs-6-mouse up': (['Identification and drug-induced reversion of molecular signatures of Alzheimer´s disease onset and progression in AppNL-G-F, AppNL-F and 3xTg-AD mouse models (RNA-seq: WT/NLF/NLGF)'],\n",
       "  \"Alzheimer's disease (AD) is the most common form of dementia. Over fifty years of intense research have revealed many key elements of the biology of this neurodegenerative disorder. However, our understanding of the molecular bases of the disease is still incomplete, and the available medical treatments for AD are mainly symptomatic and hardly effective. Indeed, the robustness of biological systems have revealed that the modulation of a single target is unlikely to yield the desired outcome, and we should move from gene-centric to   systemic therapeutic strategies. Here, we present the complete characterization of three murine models of AD at different stages of the disease (i.e. onset, progression and advanced). To identify genotype-to-phenotype relationships, we combine the cognitive assessment of these mice with histological analyses and a full transcriptional and protein quantification profiling from hippocampus. Comparison between the gene and protein expression trends observed in AD progression and physiological ageing exposed certain commonalities, such as the upregulation of microglial and inflammation markers. However, although there is an accelerated ageing in AD models, there are other factors specifically associated with A_ pathology. Despite the clear correlation between mRNA and protein levels of the dysregulated genes, we discovered a few proteins whose abundance increases with AD progression, while the corresponding transcript levels remain stable. Indeed, we show that at least two of these proteins, namely lfit3 and Syt11, co-localize with A_ plaques in the brain. Finally, we derive specific A_-related molecular AD signatures, and we look for drugs able to globally revert them. We found two NSAIDs (dexketoprofen and etodolac) and two anti-hypertensives (penbutolol and bendroflumethiazide) that overturn the cognitive impairment in AD mice   while reducing A_ plaques in the hippocampus and partially restoring the physiological levels of AD signature   genes to wild-type levels.\",\n",
       "  'wt 9m hippocampal sample mouse id processing age condition hippocampus',\n",
       "  'ki 9m hippocampal sample app nl g f mouse id processing age condition hippocampus',\n",
       "  'mouse'),\n",
       " 'GSE180705-3-vs-4-human dn': (['Deciphering the Molecular Signature of Human Hyalocytes in Relation to Other Innate Immune Cell Populations'],\n",
       "  'Purpose: The present study compares the transcriptional profiles of human hyalo-cytes, retinal microglia as well as classical, intermediate and non-classical mono-cytes in a pairwise analysis.   Methods: Using fluorescence-activated cell sorting, hyalocytes and retinal microglia were isolated from the vitreous or retinal tissue of enucleated eyes due to melano-ma of the iris, ciliary body or choroid. In addition, classical, intermediate and non-classical monocytes were isolated from whole blood. RNA sequencing was per-formed and the transcriptional profiles of all five immune cell populations were ana-lyzed using bioinformatic analysis.  Results: Pairwise analysis indicated distinct differences between hyalocytes and all three subsets of monocytes, whereas a high degree of similarity was apparent when compared to retinal microglia, with comparable expression levels of established MG markers such asTREM2, P2RY12 and TMEM119. Cluster analysis based on signa-ture genes of yolk sac-derived cells as well as hematopoietic stem and progenitor cells indicated that in contrast to monocytes, human hyalocytes and retinal microglia share a common origin in the yolk sac. Among the top expressed genes in hyalo-cytes, SPP1, CD74 and C3 were significantly upregulated when compared to mon-ocytes. Despite the high level of similarity, genes such as TNF, CCL3 and CXCL8 were significantly upregulated in hyalocytes when compared to retinal microglia. In addition, ten hyalocyte-specific genes were identified, among them FOS, DUSP1 and EGR2.   Conclusions: This study provides detailed insights into the molecular profile of iso-lated human hyalocytes and, for the first time, human retinal microglia. The expres-sion profile of hyalocytes is similar to that of retinal microglia and indicates that hy-alocytes are, contrary to the prevailing notion, yolk sac-derived cells. The hyalocyte-specific markers identified in this study will contribute in the future to deciphering the different roles of this fascinating cell population in health and diseases of the vitre-oretinal interface.',\n",
       "  'intermmo intermediate monocytes human',\n",
       "  'nonclassmo non classical monocytes human',\n",
       "  'human'),\n",
       " 'GSE148822-7-vs-13-human dn': ([\"Distinct amyloid-b and tau associated microglia profiles in Alzheimer's disease\"],\n",
       "  'Alzheimer’s disease (AD) is the most prevalent form of dementia and is characterized by abnormal extracellular aggregates of amyloid-b and intraneuronal hyperphosphorylated, tau tangles and neuropil threads. Microglia, the tissue-resident macrophages of the central nervous system (CNS), are important for CNS homeostasis and implicated in AD pathology. In amyloid mouse models, a phagocytic/activated microglia phenotype has been identified. How increasing levels of amyloid-b and tau pathology affect human microglia transcriptional profiles is unknown. Here, we performed snRNAseq on 482,472 nuclei from non-demented control brains and AD brains containing only amyloid-b plaques or both amyloid-b plaques and tau pathology. Within the microglia population, distinct expression profiles were identified of which two were AD pathology-associated. The phagocytic/activated AD1-microglia population abundance strongly correlated with tissue amyloid-b load and localized to amyloid-b plaques. The AD2-microglia abundance strongly correlated with tissue phospho-tau load and these microglia were more abundant in samples with over tau pathology. This full characterization of human disease associated microglia phenotypes provides new insights in the pathophysiological role of microglia in AD and offers new targets for microglia-state-specific therapeutic strategies.',\n",
       "  'bulk neun brain library type rna sequencing nuclei population neunpos region occipital cortex age gender clinical diagnosis ctr sample group rin braak/abc score',\n",
       "  'bulk olig brain library type rna sequencing nuclei population olig2pos region occipital cortex age gender f clinical diagnosis ad sample group rin braak/abc score',\n",
       "  'human'),\n",
       " 'GSE158152,GSE158156-10-vs-1-mouse up': ([\"The microglial transcriptome of a novel murine model of Alzheimer's Disease: FACS-isolated microglia from heterozygous and homozygous App-SAA mice and WT littermate controls.\"],\n",
       "  \"We created a novel murine model of Alzheimer's Disease using a knock-in strategy to humanize the sequence of the murine App gene and introduced three familial AD (FAD) mutations, Swedish (Swedish  (KM670/671NL), Arctic (E693G)) and Austrian (T712I).  We characterized the effects of these genetic modifications on the transcriptome of FACS-isolated microglia from 8-month-old App-SAA mice. Numerous genes were differentially expressed between cells from homozygous App-SAA animals compared to those from WT littermates. For example, we observed up-regulation of Disease-associated microglia (DAM) genes. In contrast, the transcriptome of microglia from heterozygous App-SAA animals broadly resembles that of their WT counterparts.\",\n",
       "  'drn id group microglia wt lib size norm factor age 8 unit months sex library replicate brain (cortex & hippocampus)',\n",
       "  'drn id group microglia hom lib size norm factor age 8 unit months sex library replicate brain (cortex & hippocampus)',\n",
       "  'mouse'),\n",
       " 'GSE190544,GSE190548-5-vs-6-human dn': (['Isoform-resolved transcriptome of the human preimplantation embryo [Illumina]'],\n",
       "  'Human preimplantation development is a complex process involving extensive remodeling of gene expression. However, the preimplantation embryo transcriptome has only been annotated using short-read sequencing, which fails to capture full-length mRNAs and associated isoform diversity. We present a novel human embryo transcriptome using integrated long- and short-read RNA sequencing data. Our analysis reveals a total of 110,212 novel isoforms transcribed from known genes containing either a novel combination of known splice sites or at least one novel splice site, and 17,964 isoforms transcribed from completely novel genes located either in antisense direction of known genes or in intergenic space.',\n",
       "  'human 1c b3 developmental stage 3 embryo quality score high',\n",
       "  'human 2c developmental stage embryo quality score high',\n",
       "  'human'),\n",
       " 'GSE124453-0-vs-1-human dn': (['Genome-wide identification of cancer-specific alternative splicing in circRNA'],\n",
       "  'Circular RNA (circRNA) is a group of RNA family generated by RNA circularization, which was discovered ubiquitously across different cancers. However, the internal structure of circRNA is hard to be determined since of the alternative splicing happened in exons and introns of circRNA and the cancer-specific alternative splicing of circRNA is less to be identified. Here we proposed a custom tool CircSplice, which could identify internal alternative splicing of circRNA and compare the differential splicing between samples. By applying CircSplice in ccRCC, we identified cancer-specific alternative splicing (AS) events belong to circRNAs in ccRCC. We further confirmed the existence of alternative splicing events in circRNAs by RT-PCR. This study is the first exploration of cancer-specific alternative splicing in circRNAs, which helps researchers to better understand potential functions of splicing of circRNAs in cancers.',\n",
       "  'sample subject status clear cell renal carcinoma type adjacent normal',\n",
       "  'sample subject status clear cell renal carcinoma type',\n",
       "  'human'),\n",
       " 'GSE168008-0-vs-1-human up': (['RNA sequencing of mesenchymal stem cells reveals a blocking of differentiation and immunomodulatory activities under inflammatory conditions in rheumatoid arthritis patients'],\n",
       "  'We identified the differentially expressed genes in bone marrow mesenchymal stem cells after exposure to an inflammatory environment. The transcriptome profiling was evaluated by means of the precise measurement of transcripts provided by the RNA-Seq technology. Our results evidenced the existence of blocking of both regenerative (differentiation) and immunomodulatory phenotypes under inflammatory conditions characterized by an upregulation of genes involved in immune processes and a simultaneous downregulation of genes mainly involved in regenerative or cell differentiation functions.',\n",
       "  'bm act mesenchymal stem cell resting pbmcs bone marrow cells',\n",
       "  'bm mesenchymal stem cell activated pbmcs bone marrow cells',\n",
       "  'human'),\n",
       " 'GSE161916,GSE161917-2-vs-0-human up': (['Interferon-regulated genetic programs and JAK/STAT pathway activate the intronic promoter of the short ACE2 isoform in renal proximal tubules [RNA-seq]'],\n",
       "  'Angiotensin-Converting Enzyme 2 (ACE2) binds SARS-CoV-2 spike protein and facilitates viral entry into the cell. ACE2 is expressed in the kidney and renal injury strongly impacts COVID-19 prognosis. In addition, a short, interferon-inducible isoform of ACE2, dACE2 was recently identified. Change in ACE2 expression has already been linked to several human nephropathies. However, these changes were not analyzed in context of dACE2 and regulation of ACE2 expression in the kidney remains unclear. Human primary Renal Proximal Tubule cells were used for all experiments. ChIP-seq analysis of histone modifications (H3K27ac, H3K4me3, K3K4me1), RNA polymerase loading and DNAse hypersensitive sites (DHS) were used to identify gene regulatory elements in the ACE2 locus. qRT-PCR was used to assess mRNA expression of ACE2, dACE2, TMPRSS2 and STAT1. RNA-seq was used to identify changes in global gene expression after cytokine treatment. Putative regulatory elements controlling dACE2 expression were identified using ChIP-seq profiles and RNA-seq mapping. qRT-PCR differentiating between ACE2 and dACE2 revealed that only dACE2 is interferon inducible. dACE2 mRNA was upregulated 300- and 600-fold by IFNα and IFNβ, respectively, while full length ACE2 was unchanged. Interferon stimulation also resulted in elevated STAT1 expression. JAK inhibitor ruxolitinib ablated both STAT1 and dACE2 expression after interferon treatment. RNA-seq analysis revealed approximately 1200 genes induced by IFNβ, largely innate immune response-related. We showed that human primary proximal tubules express ACE2 and its interferon-inducible short isoform, dACE2, from an intronic promoter. We also deliver new datasets identifying cytokine response genes in proximal tubule cells.',\n",
       "  'rna seq hppt untreated primary proximal tubule cells human',\n",
       "  'rna seq hppt ifna primary proximal tubule cells human',\n",
       "  'human'),\n",
       " 'GSE99675-6-vs-2-human dn': (['CDK4/6 inhibitor resistance in prostate cancer'],\n",
       "  'CDK4/6 kinase inhibitors have shown great promise in clinical trials in various cancer types and have recently entered clinical trial for advanced prostate cancer. Although patients are expected to respond well to this class of drugs, development of resistance in some patients is anticipated. To pre-empt this and study how prostate cancer may evade CDK4/6 inhibition, new resistance models were generated from LNCaP and LAPC4 prostate cancer cells cells by prolonged culturing in presence of 0.5uM palbociclib. RNA sequencing data was integrated with phospho-proteomics to unravel the molecular underpinnings of acquired resistance to palbociclib and resultant broad CDK4/6 inhibitor resistance.',\n",
       "  'lapc4 parental ctrl treated replicate immortalized cell line developmental stage derived lymph node metastasis concentration vehicle control human prostate cancer cells',\n",
       "  'lapc4 parental pd treated replicate immortalized cell line developmental stage derived lymph node metastasis concentration 0.5um human prostate cancer cells',\n",
       "  'human'),\n",
       " 'GSE136587-7-vs-12-human up': (['RNA-seq analysis of human asthma bronchoalveolar lavage (BAL) cells'],\n",
       "  'Next-generation sequencing of human BAL cells obtained from human patients with differing degrees of asthma in a search for variations in gene expressions.',\n",
       "  'bronchoalveolar lavage cells asthma degree mild/moderate gender age steroid clustering group 2 sequencing (bal)',\n",
       "  'bronchoalveolar lavage cells asthma degree gender age steroid clustering group sequencing (bal)',\n",
       "  'human'),\n",
       " 'GSE178615-1-vs-8-human up': (['Coordinated metabolic transitions and gene expression by NAD+ during adipogenesis.'],\n",
       "  'Adipocytes are the main cell type in adipose tissue, a critical regulator of metabolism, highly specialized in storing energy as fat. Adipocytes differentiate from multipotent mesenchymal stromal cells through adipogenesis. Using RNA-seq we explored the effect of manipulating NAD+ bioavailability during adipogenic differentiation from human mesenchymal stem cells. Cells were harvested either undifferentiated (hMSC) or after days 8 and 16 of adipogenic induction. During adipogenesis, cells were treated with either NAD+ (5 mM), or FK866 (1 nM) or EX527 (50 μM).',\n",
       "  'adipo+nad+ day8 cell line mesenchymal differentiation stage adipogenic induction days 8 nad+ human',\n",
       "  'adipo+ex527 day8 cell line mesenchymal differentiation stage adipogenic induction days 8 ex527 human',\n",
       "  'human'),\n",
       " 'GSE191088-6-vs-11-human up': (['Comprehensive comparison of adaptive immune response by inactivated SARS-CoV-2 vaccine between young and old [RNA-seq]'],\n",
       "  'Purpose:Comprehensively compared the adaptive immune response of SARS-CoV-2 inactivated vaccines in young and elderly.',\n",
       "  'a2y cd8 vaccine 2 dose cell age young',\n",
       "  'a2y cd4 vaccine 2 dose cell age young',\n",
       "  'human'),\n",
       " 'GSE140035,GSE140044-5-vs-0-mouse up': (['Deconvoluting global cytokine signaling networks in natural killer cells [RNA-Seq]'],\n",
       "  'Cytokine signaling via signal transducer and activator of transcription (STAT) proteins is crucial for optimal antiviral responses of natural killer (NK) cells. However, the pleiotropic effects of both cytokine and STAT signaling preclude the ability to precisely attribute molecular changes to either source. In this study, we employ a multi-“omics” approach to deconstruct and rebuild the complex interaction of three major STAT signaling pathways in NK cells. We uncover a global STAT4-STAT5 cooperative axis that is distinct and sometimes antagonistic to the STAT1 axis. We generate a network of STAT targets that highlights an integrated negative feedback loop, and demonstrate distinct STAT modes of epigenetic regulation. Finally, we identify shared signatures between mouse in vitro profiles and profiles from viral infection and humans, highlighting clinically translatable targets. Overall, we begin to unravel the intricate crosstalk between cytokine signaling pathways, and offer a valuable resource for improving cellular immunotherapy.',\n",
       "  'rna mouse wt ifna ex vivo nk cells condition replicate splenic',\n",
       "  'rna mouse wt il12il18ifna ex vivo nk cells condition replicate splenic',\n",
       "  'mouse'),\n",
       " 'GSE131728,GSE131731-0-vs-2-human dn': (['Paxillin regulates genomic networks in prostate cancer [C4-2]'],\n",
       "  'We report that the adaptor protein, paxillin, regulates some androgen responsive genes in the castration resistant prostate cancer cell line,C4-2.',\n",
       "  'c4 2 nspsirna transfection non specific sirna 2.5nm dihydrotestosterone(dht) prostate cancer cell line',\n",
       "  'c4 2 nspsirna transfection non specific sirna vehicle prostate cancer cell line',\n",
       "  'human'),\n",
       " 'GSE196049-0-vs-2-human dn': (['RNA sequencing of human monocytes stimulated with murine platelets from wild-type or Clec1b-/- mice'],\n",
       "  'C-type lectin receptors (CLRs) recognize glycans present on the surface of pathogens or self and mediate host defense as well as homeostasis. Dectin-1 is the most well-characterized CLR which recognizes β-glucan on various pathogens and expressed on human myeloid cells. Recently, we screened various tissue-derived primary cells for Dectin-1 binding and found that human Dectin-1 selectively binds to platelets. By establishing anti-platelet monoclonal antibodies that inhibit human Dectin-1 binding, we identified that CLEC-2 is responsible for the interaction with human Dectin-1. CLEC-2 is critical for lymphangiogenesis in early embryonic stage. In this study, we evaluated the influence on CLEC-2 binding in human monocytes which highly expresses human Dectin-1 by RNA sequencing.',\n",
       "  'hmono non stim human monocytes untreated',\n",
       "  'hmono ko plts human monocytes treated murine platelets clec1b / mice',\n",
       "  'human'),\n",
       " 'GSE130688-7-vs-4-human dn': (['Annotation and functional characterization of long noncoding RNAs deregulated in pancreatic adenocarcinoma'],\n",
       "  'Purpose: Transcriptome analysis of pancreatic ductal adenocarcinoma (PDAC) has been useful to identify gene expression changes that sustain malignant phenotypes. Yet, most studies examined only tumor tissues and focused on protein-coding genes, leaving long non-coding RNAs (lncRNAs) largely underexplored.',\n",
       "  'non tumor subject status pancreatic ductal adenocarcinoma (pdac) sex age adjacent pancreas',\n",
       "  'tumor subject status pancreatic ductal adenocarcinoma (pdac) sex age pdac pancreas',\n",
       "  'human'),\n",
       " 'GSE138748-2-vs-3-human dn': (['Transcriptome data from human endocrine cells recovered from mouse grafts with pre-engraftment controls'],\n",
       "  'We examined the transcriptomes of alpha and beta cells recovered from mouse sub-renal capsule grafts of human islets as compared to alpha and beta cells from the same islet preps pre-engraftment. We found that after three to four weeks in the graft the human islets were still glucose responsive, secreting insulin following glucose challenge. In addition, gene expression was largely unchanged from the pre-engraftment control samples.',\n",
       "  'beta rec pancreatic recovered cells',\n",
       "  'alpha rec pancreatic recovered cells',\n",
       "  'human'),\n",
       " 'GSE92592-4-vs-7-human dn': (['mRNA Sequencing of Ideopathic Pulmonary Fibrosis (IPF) and Control Samples from the Lung Tissue Research Consortium (LTRC)'],\n",
       "  'IPF (n=20) and control (n=19) samples were obtained through the LTRC and were sequenced on an Illumina HiSeq 2000 following TruSeq RNA Sample Prep Kit v2 library preparation.',\n",
       "  'lung disease state control',\n",
       "  'lung disease state idiopathic pulmonary fibrosis',\n",
       "  'human'),\n",
       " 'GSE136823-1-vs-3-human dn': (['Multi-treatment models for breast cancer'],\n",
       "  'RNA seq analysis of multiple treatments and their effects on in vitro breast cancer models',\n",
       "  '4 oht cell line mcf 7 agent mammary',\n",
       "  'fulv+sel cell line mcf 7 agent mammary',\n",
       "  'human'),\n",
       " 'GSE190544,GSE190548-2-vs-5-human up': (['Isoform-resolved transcriptome of the human preimplantation embryo [Illumina]'],\n",
       "  'Human preimplantation development is a complex process involving extensive remodeling of gene expression. However, the preimplantation embryo transcriptome has only been annotated using short-read sequencing, which fails to capture full-length mRNAs and associated isoform diversity. We present a novel human embryo transcriptome using integrated long- and short-read RNA sequencing data. Our analysis reveals a total of 110,212 novel isoforms transcribed from known genes containing either a novel combination of known splice sites or at least one novel splice site, and 17,964 isoforms transcribed from completely novel genes located either in antisense direction of known genes or in intergenic space.',\n",
       "  'human 1c developmental stage embryo quality score high',\n",
       "  'human 1c b3 developmental stage 3 embryo quality score high',\n",
       "  'human'),\n",
       " 'GSE185245-1-vs-4-human dn': (['TEFM variants impair mitochondrial transcription elongation causing childhood-onset neurological disease'],\n",
       "  \"We report that TEFM variants are associated with mitochondrial respiratory chain deficiency. We show that patient's cells have reduced levels of promoter distal mitochondrial RNA transcripts.\",\n",
       "  'muscle control subject status/id / molecule subtype rrna depleted rna',\n",
       "  'rep2dep subject status/id / fibroblasts molecule subtype rrna depleted rna',\n",
       "  'human'),\n",
       " 'GSE184536-4-vs-5-human dn': (['The NF-κB transcriptional footprint is essential for SARS-CoV-2 replication'],\n",
       "  'SARS-CoV-2, the etiological agent of COVID-19, is characterized by a delay in Type I interferon (IFN-I)-mediated antiviral defenses alongside robust cytokine production. Here we investigate the underlying molecular basis for this imbalance and implicate virus-mediated activation of NF-κB in the absence of other canonical IFN-I-related transcription factors. Epigenetic and single cell transcriptomic analyses show a selective NF-κB signature that was most prominent in infected cells. Disruption of NF-κB signaling through the silencing of the NF-κB transcription factors p65 or p50 resulted in loss of virus replication that was rescued upon reconstitution. These findings could be further corroborated with the use of NF-κB inhibitors, which reduced SARS-CoV-2 replication in vitro. These data suggest that the robust cytokine production in response to SARS-CoV-2, despite a diminished IFN-I response, is the product of a dependency on NF-κB for viral replication.',\n",
       "  'mrnaseq ifnb 12h lung adenocarcinoma virus mock 12 h ifn î² (100 u/ml) a549 ace2 cells',\n",
       "  'mrnaseq mock 9h lung adenocarcinoma virus 9 h a549 ace2 cells',\n",
       "  'human'),\n",
       " 'GSE118373-2-vs-3-human up': (['High-throughput sequencing of the human hepatic progenitor cell niche in PSC and HCV.'],\n",
       "  'This study identified hepatic progenitor cell (HPC) niche-associated signalling pathways relevant in different chronic liver diseases using a high-throughput sequencing approach. The HPC niche was isolated using laser microdissection from patient samples diagnosed with hepatitis C virus (HCV) or primary sclerosing cholangitis (PSC), as models for hepatocellular or biliary regeneration.  Differentially expressed genes in the HPC niche of PSC and HCV correlated to pathways involved in immune signalling, fibrogenesis and angiogenesis.',\n",
       "  'hcv hpc disease state hepatitis c hepatic progenitor cells',\n",
       "  'psc hep disease state primary sclerosing cholangitis hepatocytes',\n",
       "  'human'),\n",
       " 'GSE172116-4-vs-7-human up': (['Epigenomic analysis reveals a dynamic and context-specific macrophage enhancer landscape associated with innate immune activation and tolerance'],\n",
       "  'Background',\n",
       "  'mdms rnaseq ut monocyte derived macrophage',\n",
       "  'mdms rnaseq hd monocyte derived macrophage',\n",
       "  'human'),\n",
       " 'GSE85121-8-vs-12-human up': (['Dysregulation of Lung Biology of Healthy Never Smokers Following Acute Inhalation of E-Cigarette Vapors'],\n",
       "  'Little is known about the health risks to humans associated with smoking or “vaping” electronic cigarettes (EC). Based on the knowledge that the adverse effects of cigarette smoking are observed in the dysregulation of lung biology long before clinical abnormalities manifest, we assessed the hypothesis that acute exposure to EC vapors, with or without nicotine, modifies the biology of the lung of healthy never smokers. No significant changes in clinical parameters were observed following this brief exposure to EC vapors. However, marked changes were observed in the transcriptome of small airway epithelium and alveolar macrophage compared to baseline for both exposure groups with and without nicotine, and the group inhaling the nicotine containing vapors had elevated plasma endothelial microparticle levels. While additional studies are needed to ascertain long-term health risks of EC, this study provides preliminary in vivo data suggesting that EC vapors likely dysregulates normal human lung homeostasis, at least at the biologic level.',\n",
       "  'dgm d1 individual smoking status nonsmoker time alveolar macrophage',\n",
       "  'dgm d7 sm individual smoking status time small airway epithelium brushing',\n",
       "  'human'),\n",
       " 'GSE211190-3-vs-1-human up': (['Vascular Surgery Patients with Elevated Neutrophil Lymphocyte Ratios Have Downregulated Neutrophil Complement RNA Expression'],\n",
       "  'Elevated NLR (neutrophil-lymphocyte ratio) in elective vascular surgery (EVS) patients is associated with increased mortality independent of perioperative surgical outcome. To understand why high NLR is associated with higher mortality, we investigated neutrophil and lymphocyte transcriptome expression in patients undergoing EVS.',\n",
       "  'lymphocyte control [nlr l blood nl ratio',\n",
       "  'neutrophil control [nlr n blood nl ratio',\n",
       "  'human'),\n",
       " 'GSE209587-1-vs-2-mouse dn': (['Immune cells and their inflammatory mediators modify beta cells and cause checkpoint inhibitor-induced diabetes'],\n",
       "  'Checkpoint inhibitors (CPIs) targeting PD-1/PD-L1 and CTLA-4 have revolutionized cancer treatment but can trigger autoimmune complications including CPI-induced diabetes (CPI-DM), which occurs preferentially with PD-1 blockade. We found evidence of pancreatic inflammation in patients with CPI-DM with shrinkage of pancreases, increased pancreatic enzymes, and in a case from a patient who died with CPI-DM, peri-islet lymphocytic infiltration. In the NOD mouse model, anti-PD-L1 but not anti-CTLA-4 induces DM rapidly. RNA sequencing revealed that cytolytic IFNγ+ CD8+ T cells infiltrated islets with anti-PD-L1.  Changes in β cells were predominantly driven by IFNγ and TNFα and included induction of a novel β cell population with transcriptional changes suggesting dedifferentiation. IFNγ increased checkpoint ligand expression and activated apoptosis pathways in human β cells in vitro. Treatment with anti-IFNγ and anti-TNFα prevented CPI-DM in anti-PD-L1 treated NOD mice. CPIs targeting the PD-1/PD-L1 pathway result in transcriptional changes in β cells and immune infiltrates that may lead to the development of diabetes. Inhibition of inflammatory cytokines can prevent CPI-DM, suggesting a strategy for clinical application to prevent this complication.',\n",
       "  'cd45 cells ctla4 pancreatic islet strain nod age 7 week',\n",
       "  'islet cells ctla4 pancreatic strain nod age 7 week',\n",
       "  'mouse'),\n",
       " 'GSE147876,GSE147877-10-vs-11-human dn': (['Chromatin Occupancy of AR, BRD4, and H3K27Ac in MR42D Cells in response to Enzalutamide [RNA-seq]'],\n",
       "  'To identify mechanisms that contribute to lineage plasticity and t-NEPC, we focused on enza-resistant cell lines with persistent AR expression that have features of lineage plasticity, including high expression of NEPC markers: the MR42D and MR42F cell lines. We cultured these enzalutamide resistant lines, which are usually maintained in medium with enzalutamide, without enzalutamide for 72 hors to \"washout\" the effects of enzalutamide, and then either enzalutamide or vehicle was added back to the growing media for 24 hours. Enzalutamide sensitive lines LNCaP and V16D were treated with either enzalutamide or vehicle for 24 hours. Enzalutamide reversibly increased expression neuronal markers in MR42D and MR42F cells, but not enza-sentitive LNCaP and V16D cells, suggesting that the AR may negatively regulate neuronal gene expression in these t-NEPC cells. Moreover, bromodomain inhibitors (BETi) ARV-771 and JQ1 reduced expression of NEPC target genes in MR42D cells.',\n",
       "  'mr42d washout mdv cell line prostate carcinoma enzalutamide resistant',\n",
       "  'mr42d washout veh cell line prostate carcinoma enzalutamide resistant vehicle',\n",
       "  'human'),\n",
       " 'GSE191088-6-vs-9-human up': (['Comprehensive comparison of adaptive immune response by inactivated SARS-CoV-2 vaccine between young and old [RNA-seq]'],\n",
       "  'Purpose:Comprehensively compared the adaptive immune response of SARS-CoV-2 inactivated vaccines in young and elderly.',\n",
       "  'a2y cd8 vaccine 2 dose cell age young',\n",
       "  'bo cd4 vaccine cell age old',\n",
       "  'human'),\n",
       " 'GSE242259-2-vs-9-human dn': (['Effects of mir-483 and mir-663a knockout in human cancer cell lines'],\n",
       "  'CRISPR/Cas9 knockout screens were performed to identify core fitness miRNA gene in human cancer cell lines.',\n",
       "  'mfc7 empty rep tumor breast cell line mcf7 adenocarcinoma control lenticrisprv2 none',\n",
       "  'mfc7 sgmir 663a rep tumor breast cell line mcf7 adenocarcinoma lenticrisprvs none',\n",
       "  'human'),\n",
       " 'GSE86573-4-vs-5-mouse dn': (['Macrophage ontogeny underlies differences in tumor-specific education in brain malignancies'],\n",
       "  'Recent efforts have uncovered immense transcriptional and ontogenetic diversity among tissue-resident macrophages, each with their own transcriptional profile endowing the cell with its tissue-specific functions. However, it is currently unknown whether the origins of different macrophage populations may affect their roles in malignancy. Given potential artifacts associated with irradiation-based lineage tracing, it remains unclear if bone marrow-derived macrophages (BMDM) are even present in tumors of the brain, a tissue where there is no homeostatic involvement of peripherally-derived myeloid cells. Here, we employed multiple models of murine brain malignancy and genetic lineage tracing models to demonstrate that BMDM are indeed abundant in primary and metastatic brain tumors. Transcriptional profiling of tumor-associated BMDM and resident microglia showed that these cells acquire substantially different gene expression profiles. Our data suggest that transcriptional networks in each cell population are associated with tumor-mediated education, yet are also influenced by chromatin landscapes established before tumor initiation. Furthermore, we demonstrate that microglia specifically repress Itga4 (CD49D), enabling its utility as a discriminatory marker between brain-resident microglia and peripherally-derived macrophages in both primary and metastatic disease in mouse and human.',\n",
       "  'gemm mg tam microglia strain c57bl/6 tumor associated',\n",
       "  'gemm bmdm tam bone marrow derived macrophage strain c57bl/6 tumor associated',\n",
       "  'mouse'),\n",
       " 'GSE116765-4-vs-5-human dn': (['Pleiotropic impact of DNA-PK in cancer and implications to therapeutic strategies'],\n",
       "  'DNA-dependent protein kinase catalitic subunit (DNA-PK) is associated with aggressive disease. DNA-PK plays a role in transcriptional regulation of cancer-relevent pathways but it is not well understood.  This study uses transcriptomic analyses to understand the transcriptional role of DNA-PK in castration resistent prostate cancer (CRPC) upon DNA-PK inhibition',\n",
       "  'c4 2 cc 223 cell line prostate cancer cells tork inhibitor 24hr 1um',\n",
       "  'c4 2 cc 223+ enza cell line prostate cancer cells ar antagonist 24hr 1um tork inhibitor 223',\n",
       "  'human'),\n",
       " 'GSE246642,GSE246643-6-vs-1-human up': (['XIST dampens X chromosome activity in a SPEN-dependent manner during early human development [RNA-seq]'],\n",
       "  'XIST long non-coding RNA is responsible for X chromosome inactivation (XCI) in placental mammals, yet it accumulates on both X chromosomes in human female pre-implantation embryos without triggering X chromosome silencing. The long non-coding RNA XACT co-accumulates with XIST on active Xs and may antagonize XIST function. Here we used human ES cells in a naïve state of pluripotency to assess the function of XIST and XACT in shaping the X chromosome chromatin and transcriptional landscapes during pre-implantation development. We show that XIST triggers the deposition of polycomb-mediated repressive histone modifications and attenuates transcription of most X-linked genes in a SPEN-dependent manner, while XACT deficiency does not significantly affect XIST activity or X-linked gene expression. Our study demonstrates that XIST is functional prior to XCI, confirms the existence of a transient process of X chromosome dosage compensation, and reveals that X chromosome inactivation and dampening rely on the same set of factors.',\n",
       "  'pxgl wt hesc status cell line h9 human embryonic stem',\n",
       "  'pxgl xist ko hesc status cell line h9 human embryonic stem',\n",
       "  'human'),\n",
       " 'GSE223100,GSE223842-9-vs-11-mouse dn': (['pDC-like cells are pre-DC2 and require Klf4 to control homeostatic CD4 T cells'],\n",
       "  'Plasmacytoid dendritic cells (pDCs) have been shown to play an important role during immune responses, ranging from initial viral control through the production of type I interferons (IFNs) to antigen presentation. However, recent evidence uncovered unexpected heterogeneity among pDCs. Notably we identified a new subset, referred to as pDC-like cells, that resembles pDCs but shares cDC features. Here we show that this subset is a circulating progenitor distinct from common DC progenitors (CDP), with prominent cDC2 precursor potential. This precursor subset responds to homeostatic cytokines, such as macrophage colony stimulating factor (M-CSF) by expanding and differentiating into cDC2 that efficiently prime Th17 cell. Development of pDC-like cells into CX3CR1+ESAM- cDC2b but not CX3CR1- ESAM+ cDC2a requires the transcription factor KLF4. Finally, we show that under homeostatic conditions this developmental pathway regulates the immune threshold at barrier sites, by controlling the pool of Th17 cells within the skin draining lymph nodes.',\n",
       "  'spl pdcs spleen (spl) replicate',\n",
       "  'spl spleen (spl) replicate',\n",
       "  'mouse'),\n",
       " 'GSE161916,GSE161917-2-vs-7-human up': (['Interferon-regulated genetic programs and JAK/STAT pathway activate the intronic promoter of the short ACE2 isoform in renal proximal tubules [RNA-seq]'],\n",
       "  'Angiotensin-Converting Enzyme 2 (ACE2) binds SARS-CoV-2 spike protein and facilitates viral entry into the cell. ACE2 is expressed in the kidney and renal injury strongly impacts COVID-19 prognosis. In addition, a short, interferon-inducible isoform of ACE2, dACE2 was recently identified. Change in ACE2 expression has already been linked to several human nephropathies. However, these changes were not analyzed in context of dACE2 and regulation of ACE2 expression in the kidney remains unclear. Human primary Renal Proximal Tubule cells were used for all experiments. ChIP-seq analysis of histone modifications (H3K27ac, H3K4me3, K3K4me1), RNA polymerase loading and DNAse hypersensitive sites (DHS) were used to identify gene regulatory elements in the ACE2 locus. qRT-PCR was used to assess mRNA expression of ACE2, dACE2, TMPRSS2 and STAT1. RNA-seq was used to identify changes in global gene expression after cytokine treatment. Putative regulatory elements controlling dACE2 expression were identified using ChIP-seq profiles and RNA-seq mapping. qRT-PCR differentiating between ACE2 and dACE2 revealed that only dACE2 is interferon inducible. dACE2 mRNA was upregulated 300- and 600-fold by IFNα and IFNβ, respectively, while full length ACE2 was unchanged. Interferon stimulation also resulted in elevated STAT1 expression. JAK inhibitor ruxolitinib ablated both STAT1 and dACE2 expression after interferon treatment. RNA-seq analysis revealed approximately 1200 genes induced by IFNβ, largely innate immune response-related. We showed that human primary proximal tubules express ACE2 and its interferon-inducible short isoform, dACE2, from an intronic promoter. We also deliver new datasets identifying cytokine response genes in proximal tubule cells.',\n",
       "  'rna seq hppt untreated primary proximal tubule cells human',\n",
       "  'rna seq hppt ifng primary proximal tubule cells human',\n",
       "  'human'),\n",
       " 'GSE149246-0-vs-1-human dn': (['Characterizing an HSF1-dependent transcriptional program in metastatic lung adenocarcinoma'],\n",
       "  'The purpose of this study was to identify gene targets differentially regulated by shRNA-mediated knockdown of the HSF1 transcription factor in the brain-metastatic lung adenocarcinoma cell line PC9-BrM3.',\n",
       "  'pc9 brm3 rep cell line shrna tet plko non small lung adenocarcinoma',\n",
       "  'pc9 rep cell line non small lung adenocarcinoma',\n",
       "  'human'),\n",
       " 'GSE78747-1-vs-3-mouse up': (['GRN knockout, sorted brain cells'],\n",
       "  '1 year old mice were perfused and brains were dissociated. Cells were fixed, immunolabeled and FACS sorted. RNA was extracted from neuron, astrocyte, and microglial cell populations. Typical RIN=4-5 for neurons, 6-8 for astrocytes, and 5-7 for microglia. Typical RNA yields ~100ng for neurons, ~20ng for microglia, and ~10ng for astrocytes.',\n",
       "  '5641 cd11b brain sort sex male grn.het rna sorted cells',\n",
       "  '5641 neun brain sort sex grn.het rna sorted cells',\n",
       "  'mouse'),\n",
       " 'GSE135291-2-vs-5-human up': (['Transcriptomic analysis of freshly purified human CD226- and CD226+  CD8+ T cell subsets isolated from the peripheral blood of healthy donors.'],\n",
       "  'We recently found that loss of the activating receptor CD226 (DNAM-1) was a critical mechanism affecting CD8+ T cell responsiveness to TCR stimulation. To better understand the molecular differences between CD226- and CD226+ CD8+ T cells, we decided to perform a global transcriptional analysis of purified naive (Tn, CD62L+CD45RA+), central memory (Tcm, CD62L+CD45RA-)  and terminal effector memory (Temra, CD62L-CD45RA+) CD226+ and CD226- CD8+  T cells using next generation RNA sequencing.',\n",
       "  'tn cd226 moins gender sampling date 2016 peripheral blood',\n",
       "  'temra cd226 plus gender sampling date 2016 peripheral blood cd226+',\n",
       "  'human'),\n",
       " 'GSE132228-4-vs-2-human dn': (['Social network architecture of human immune cells in ZIKV-infected patients highlights a unique role of plasmacytoid dendritic cells'],\n",
       "  'Zika virus (ZIKV)\\xa0is an emerging mosquito-borne pathogen with increasing public health significance. To characterize interconnected immune responses to ZIKV, we here examined transcriptional signatures of CD4 and CD8 T cells, B cells, NK cells, monocytes, myeloid dendritic cells (mDCs), and plasmacytoid dendritic cells (pDCs) from three individuals with naturally-acquired ZIKV infection. While gene expression patterns from most cell subsets displayed signs of impaired antiviral immune activity, pDCs showed a highly distinct transcriptional response marked by an activation of innate immune recognition and type I interferon signaling pathways, combined with a downregulation of key host factors known to support ZIKV replication steps. At the same time, pDCs exhibited a unique expression pattern of gene modules that were tightly correlated with alternative cell populations, suggesting highly collaborative interactions between pDCs and other immune cells. Together, these results point towards a discrete but integrative role of pDCs in the human immune response to ZIKV infection.',\n",
       "  'participant pdc subject id infection status healthy donor peripheral blood environment cell subtype',\n",
       "  'participant nk subject id infection status healthy donor peripheral blood environment ex vivo cell subtype',\n",
       "  'human'),\n",
       " 'GSE61966,GSE61967-1-vs-4-human up': (['BRG1 recruitment by transcription factors MITF and SOX10 defines a specific configuration of regulatory elements in the melanocyte lineage (RNA-seq)'],\n",
       "  'Microphthalmia-associated transcription factor (MITF) is the master regulator of the melanocyte lineage. By tandem affinity purification and mass spectrometry, we present a comprehensive characterisation of the MITF interactome comprising multiple novel cofactors involved in transcription, DNA replication and repair and chromatin organisation, including a BRG1 chromatin remodelling complex comprising CHD7. BRG1 is essential for melanoma cell proliferation in vitro and for normal melanocyte development in vivo. MITF and SOX10 actively recruit BRG1 to a set of MITF-associated regulatory elements (MAREs) at active enhancers. MITF, SOX10 and YY1 bind between two BRG1-occupied nucleosomes thus defining both a combinatorial signature of transcription factors essential for the melanocyte lineage and a specific chromatin organisation of MAREs. Nevertheless, BRG1 silencing enhances MITF occupancy at MAREs showing that BRG1 acts to promote dynamic MITF interactions with chromatin.',\n",
       "  'h3a shmitf cell line hermes 3a melanocyte knockdown',\n",
       "  'h3a shscr cell line hermes 3a melanocyte knockdown',\n",
       "  'human'),\n",
       " 'GSE239662,GSE239663-4-vs-2-human up': (['Novel method of isolating nuclei of human oligodendrocyte precursor cells reveals substantial developmental changes in gene expression and H3K27ac histone modification. [RNA-Seq]'],\n",
       "  'Oligodendrocyte precursor cells (OPCs) generate differentiated mature oligodendrocytes (MOs) during development. In adult brain, OPCs replenish MOs in adaptive plasticity, neurodegenerative disorders, and after trauma. The ability of OPCs to differentiate to MOs decreases with age and is compromised in disease. Here we explored the cell specific and age-dependent differences in gene expression and H3K27ac histone mark in these two cell types. H3K27ac is indicative of active promoters and enhancers. We developed a novel flow-cytometry-based approach to isolate OPC and MO nuclei from human postmortem brain and profiled gene expression and H3K27ac in adult and infant OPCs and MOs genome-wide.In adult brain, we detected extensive H3K27ac differences between the two cell types with high concordance between gene expression and epigenetic changes. Notably, the expression of genes that distinguish MOs from OPCs appears to be under a strong regulatory control by the H3K27ac modification in MOs but not in OPCs. Comparison of gene expression and H3K27ac between infants and adults uncovered numerous developmental changes in each cell type, which were linked to several biological processes, including cell proliferation and glutamate signaling. A striking example was a subset of histone genes that were highly active in infant samples but fully lost activity in adult brain. Our findings demonstrate a considerable rearrangement of the H3K27ac landscape that occurs during the differentiation of OPCs to MOs and during postnatal development of these cell types, which aligned with changes in gene expression. The uncovered regulatory changes justify further in-depth epigenetic studies of OPCs and MOs in development and disease.',\n",
       "  'subject oligodendrocyte precursor cells rna seq dorsolateral prefrontal cortex donor id disease control sex male',\n",
       "  'subject mature oligodendrocytes rna seq dorsolateral prefrontal cortex donor id disease control sex female',\n",
       "  'human'),\n",
       " 'GSE151073,GSE151095-4-vs-9-human up': (['Transcriptional comparison of in vitro and in vivo generated human dendritic cells'],\n",
       "  'Dendritic cells (DC) are antigen presenting cells controlling T cell activation. In human, the diversity, ontogeny and functional capabilities of DC subsets are not fully understood. Here, we identified circulating CD88-CD1c+CD163+ DC (termed as DC3) as an immediate precursor of inflammatory CD88-CD14+CD1c+CD163+FcεRI+ DC. DC3 develop via a specific pathway, independent from the cDC-restricted (CDP) and monocyte-restricted (cMoP) progenitors, and are activated by GM-CSF. As classical DC, but unlike monocytes, DC3 drove the activation of naïve T cells. In vitro, DC3 displayed a distinctive ability to prime CD8+ T cells expressing a tissue-homing signature and the epithelial homing alpha-E integrin (CD103) through transforming growth factor-b (TGF-β) signaling. In vivo, DC3 infiltrated luminal breast cancer primary tumors and DC3 infiltration correlated positively with CD8+CD103+CD69+ tissue-resident memory T cells. Altogether, these findings define DC3 as a lineage of inflammatory DC endowed with a strong potential to regulate tumor immunity.',\n",
       "  'naã¯ tcells peripheral blood donor cd3+ cd45ro+ cd8+ cells',\n",
       "  'pdc peripheral blood donor cd123+cd45ra+axl siglec6 cells sorted human periferal mononuclear phagocytes',\n",
       "  'human'),\n",
       " 'GSE136975-1-vs-3-human up': (['Modulating the unfolded protein response: Impacts of radiation on the response of prostate cancer cells to ONC201.'],\n",
       "  'RNA-Seq analysis of prostate cancer cells primed to irradiation for 24hours with ONC201 shows a sustained suppression of transcripts encoding cell cycle regulators as well as components of the DNA damage response pathways.',\n",
       "  '72h treatments cells treated onc201 vitro prostate cancer',\n",
       "  '24h treatments cells irradiated harvested vitro prostate cancer',\n",
       "  'human'),\n",
       " 'GSE129387-2-vs-3-mouse up': (['Gene expression of blood and cerebellum of Mecp2-null and WT male mice'],\n",
       "  'In order to find a relationship between gene expression of blood and brain in Rett Syndrome (RTT), we performed RNA sequencing on from cerebella and blood of 7 week-old male Mecp2-null mice (a model of RTT) and WT controls.',\n",
       "  'strain c57bl/6j wt cerebellum age 7 weeks',\n",
       "  'strain c57bl/6j mecp2 null blood age 7 weeks',\n",
       "  'mouse'),\n",
       " 'GSE213488-4-vs-8-human up': (['Transcriptome analyses of the cortex and white matter of focal cortical dysplasia type II human samples: novel insights into disease mechanisms and contributions to tissue characterization'],\n",
       "  'Focal cortical dysplasia (FCD) is a common cause of pharmacoresistant epilepsy. According to the 2011 International League Against Epilepsy classification, FCD type II is characterized by dysmorphic neurons (IIa and IIb) and may be associated with balloon cells (IIb). We present a multicentric study to evaluate the transcriptomes of the gay and white matters of surgical FCD type II specimens. Our aim was to contribute to pathophysiology and tissue characterization. We investigated FCD II (a and b) and control samples by performing RNA-sequencing followed by immunohistochemical validation by means of digital analyses. We found 342 and 399 transcripts differentially expressed in the gray matter of IIa and IIb lesions compared to controls, respectively. The top enriched cellular pathway in IIa and IIb gray matter was cholesterol biosynthesis, their genes HMGCS1, HMGCR and SQLE being upregulated in both type II groups. We also found 12 differentially expressed genes when comparing transcriptomes of IIa and IIb lesions. Only 1 transcript (MTRNR2L12) was significantly upregulated in FCD IIa. The white matter in IIa and IIb lesions showed 2 and 24 transcripts differentially expressed respectively compared to controls. No enriched cellular pathways were detected. GPNMB, not previously described in FCD samples, was upregulated in IIb compared to IIa and control groups. Upregulations of cholesterol biosynthesis enzymes and GPNMB genes in FCD groups were immunohistochemically validated. Such enzymes were mainly detected in both dysmorphic and normal neurons, whereas GPNMB was observed only in balloon cells. Overall, our study contributed to the identification of cortical enrichment of cholesterol biosynthesis in FCD type II, which may correspond to neuroprotective response to seizures. Moreover, specific analyses in either the gay or the white matter revealed upregulations of MTRNR2L12 and GPNMB, which might be potential neuropathological biomarkers of a cortex chronically exposed to seizures and of balloon cells, respectively.',\n",
       "  'fcd iia sample (gray matter) frontal lobe disease state',\n",
       "  'fcd iib sample (white matter) frontal lobe disease state',\n",
       "  'human'),\n",
       " 'GSE208858-9-vs-11-human dn': (['Utomilumab in Refractory Melanoma and NSCLC (NCT01307267)'],\n",
       "  'The paper reports clinical and translational results from patients with non-small-cell lung cancer (NSCLC) or melanoma dosed once every 4 weeks with either 0.24 mg/kg or 1.2 mg/kg of single-agent utomilumab, a 4-1BB/CD137 agonist mAb. The dose levels tested were well-tolerated but clinical efficacy was low. Analysis of gene expression in tumor biopsies suggested utomilumab efficacy was associated with immune-active tumor microenvironments, and pharmacodynamic activity may be reduced at higher doses. In vitro models of stimulated human T cells showed loss of pharmacodynamic activity at higher doses of utomilumab.',\n",
       "  'melanoma 0.24 mg/kg collected days relative first dose (rna disease state utomilumab biopsy collection study therapy stage response prior refractory best objective tumor subjid',\n",
       "  'melanoma 1.2 mg/kg collected days relative first dose (rna disease state utomilumab biopsy collection study therapy stage iv response prior best objective progression tumor subjid',\n",
       "  'human'),\n",
       " 'GSE64283-6-vs-3-human dn': (['Tissue-specific circular RNA induction during human fetal development'],\n",
       "  'The pervasive expression of circular RNA from protein coding loci is a recently discovered feature of many eukaryotic gene expression programs.  Computational methods to discover and quantify circular RNA are essential to the study of the mechanisms of circular RNA biogenesis and potential functional roles they may play.  In this paper, we present a new statistical algorithm that increases the sensitivity and specificity of circular RNA detection.by discovering and quantifying circular and linear RNA splicing events at both annotated exon boundaries and in un-annotated regions of the genome  Unlike previous approaches which rely on heuristics like read count and homology between exons predicted to be circularized to determine confidence in  prediction of circular RNA expression, our algorithm is a statistical approach. We have used this algorithm to discover general induction of circular RNAs in many tissues during human fetal development.  We find that some regions of the brain show marked enrichment for genes where circular RNA is the dominant isoform. Beyond this global trend, specific circular RNAs are tissue specifically induced during fetal development, including a circular isoform of NCX1 in the developing fetal heart that, by 20 weeks, is more highly expressed than the linear isoform as well as beta-actin. In addition, while the vast majority of circular RNA production occurs at canonical U2 (major spliceosome) splice sites, we find the first examples of developmentally induced circular RNAs processed by the U12 (minor) spliceosome, and an enriched propensity of U12 donors to splice into circular RNA at un-annotated, rather than annotated, exons.  Together, our algorithm and its results suggest a potentially significant role for circular RNA in human development.',\n",
       "  'heart age weeks',\n",
       "  'lung age weeks',\n",
       "  'human'),\n",
       " 'GSE202755-3-vs-1-human dn': (['RNA sequencing of LNCaP cells treated with castration and/or enzalutamide'],\n",
       "  'The goal of this study was to determine how androgen receptor inhibition alters transcriptional programs in prostate cancer cells. LNCaP prostate cancer cells were grown in 10% charcoal-stripped serum (CSS) supplemented with 0.5 nanomolar dihydrotestosterone (DHT), in CSS without DHT modeling castration, with CSS + DHT but in the presence of 10 micromolar enzalutamide, or in CSS without DHT (castrated) and in the presence of enzalutamide for 72 hours. Analysis shows that androgen receptor target genes are reduced with castration, enzalutamide or the combination of castration + enzalutamide.',\n",
       "  'lncap css + dht control cell line prostate cancer wt',\n",
       "  'lncap css castration + enzalutamide cell line prostate cancer wt',\n",
       "  'human'),\n",
       " 'GSE73721-0-vs-2-human dn': (['RNA-Seq of human astrocytes'],\n",
       "  \"Astrocytes were purified from fetal and adult human brain tissue using an immunopanning method with the HepaCAM antibody. Samples were taken from otherwise 'healthy' pieces of tissue, unless otherwise specified.\",\n",
       "  'ctx myeloid age years old cell subtype cd45 positive cells adult temporal lobe',\n",
       "  'whole cortex age years old cell subtype adult temporal lobe',\n",
       "  'human'),\n",
       " 'GSE150834-3-vs-1-human up': (['Transcriptomes of Effector Memory T cells undergoing lymphopenia induced proliferation'],\n",
       "  'Purpose: T cells undergo intense proliferation in the setting of lymphopenia and chemotherapy pre-conditioning is therefore an essential component of CAR-T therapy. Lymphopenia-induced proliferation (LIP) is thought to be mediated primarily by homeostatic cytokines such as IL-7 and IL-15 but little is known regarding the underlying mechanisms in human subjects',\n",
       "  'hd cd8 em effector memory cell disease state healthy donor peripheral blood',\n",
       "  'auto cd8 em effector memory cell disease state day 12 post autograft peripheral blood',\n",
       "  'human'),\n",
       " 'GSE239662,GSE239663-4-vs-3-human up': (['Novel method of isolating nuclei of human oligodendrocyte precursor cells reveals substantial developmental changes in gene expression and H3K27ac histone modification. [RNA-Seq]'],\n",
       "  'Oligodendrocyte precursor cells (OPCs) generate differentiated mature oligodendrocytes (MOs) during development. In adult brain, OPCs replenish MOs in adaptive plasticity, neurodegenerative disorders, and after trauma. The ability of OPCs to differentiate to MOs decreases with age and is compromised in disease. Here we explored the cell specific and age-dependent differences in gene expression and H3K27ac histone mark in these two cell types. H3K27ac is indicative of active promoters and enhancers. We developed a novel flow-cytometry-based approach to isolate OPC and MO nuclei from human postmortem brain and profiled gene expression and H3K27ac in adult and infant OPCs and MOs genome-wide.In adult brain, we detected extensive H3K27ac differences between the two cell types with high concordance between gene expression and epigenetic changes. Notably, the expression of genes that distinguish MOs from OPCs appears to be under a strong regulatory control by the H3K27ac modification in MOs but not in OPCs. Comparison of gene expression and H3K27ac between infants and adults uncovered numerous developmental changes in each cell type, which were linked to several biological processes, including cell proliferation and glutamate signaling. A striking example was a subset of histone genes that were highly active in infant samples but fully lost activity in adult brain. Our findings demonstrate a considerable rearrangement of the H3K27ac landscape that occurs during the differentiation of OPCs to MOs and during postnatal development of these cell types, which aligned with changes in gene expression. The uncovered regulatory changes justify further in-depth epigenetic studies of OPCs and MOs in development and disease.',\n",
       "  'subject oligodendrocyte precursor cells rna seq dorsolateral prefrontal cortex donor id disease control sex male',\n",
       "  'subject mature oligodendrocytes rna seq dorsolateral prefrontal cortex donor id disease control sex male',\n",
       "  'human'),\n",
       " 'GSE103001-2-vs-6-human dn': (['Stranded RNASeq of human mammary primary tumors ER+ and paired adjacent healthy tissues'],\n",
       "  'Non-coding RNAs (ncRNA) represent at least 1/5 of the mammalian transcript amount, and about 90% of the genome length is actively transcribed. Many ncRNAs have been demonstrated to play a role in cancer. Among them, natural antisense transcripts (NAT) are RNA sequences which are complementary and overlapping to those of protein-coding transcripts (PCT). NATs were punctually described as regulating gene expression, and are expected to act more frequently in cis than other ncRNAs that commonly function in trans. In this work, 22 breast cancers expressing estrogen receptors and their paired healthy tissues were analyzed by strand-specific RNA sequencing to highlight the potential role of NATs in gene regulations occurring in breast cancer.',\n",
       "  'pat normal nature tumor receptors / bloom er subtype tnm mammary',\n",
       "  'pat 14 tumor nature primary invasive breast carcinoma receptors er+/her2 bloom er subtype tnm mammary',\n",
       "  'human'),\n",
       " 'GSE178296,GSE178304-2-vs-4-mouse up': (['Transcriptome profiling of mouse brain microglia in Cnp null mice with 5xFAD background'],\n",
       "  'Alzheimer’s disease (AD) is the most common form of dementia and neurodegenerative disease with increasing prevalence due to longer lifespan in the human population. Why aging constitutes the greatest risk factor for development of AD, however, remains poorly understood. Aging markedly affects oligodendrocytes and the structural integrity of their myelin sheaths which also causes secondary tissue inflammation. We propose a mechanistic link between aging-associated myelin dysfunction and the deposition of Amyloid-ß (Aß) as primary neuropathological hallmark of early AD and hypothesized that breakdown of myelin - especially in cortical regions – is an upstream driver of amyloid deposition in AD. Here, we show that in transgenic mouse models of AD, genetically induced myelin defects by Cnp or Plp1 depletion, as well as direct demyelination are potent drivers of amyloid deposition in vivo as shown by light sheet microscopy imaging. At transcriptomic level, bulk and single-cell RNA sequencing revealed successfully induced disease-associated-microglia (DAM)-like phenotypes in Cnp-/- animals. These activated microglia, however, are primarily engaged with myelin seemingly preventing the protective reactions of the microglia pool to Aß plaques. Our work, therefore, identifies myelin aging as a previously overlooked risk factor for AD and makes the case for myelin health-directed therapies in AD.',\n",
       "  'wt bulkrna strain c57bl6/j age 6 month microglia brain mouse model alzheimerâ\\x80\\x99 disease (ad)',\n",
       "  'cnp ko bulkrna strain c57bl6/j age 6 month / microglia brain mouse model alzheimerâ\\x80\\x99 disease (ad)',\n",
       "  'mouse'),\n",
       " 'GSE75246-6-vs-7-mouse dn': (['LPS and brain inflammation'],\n",
       "  'Normal mice were injected i.p. with LPS or saline. 24h later, perfused brains were dissociated. Cells were fixed, immunolabeled and FACS sorted. RNA was extracted from neuron, astrocyte, and microglial cell populations. Typical RIN=4-5 for neurons, 6-8 for astrocytes, and 5-7 for microglia. Typical RNA yields ~100ng for neurons, ~20ng for microglia, and ~10ng for astrocytes.',\n",
       "  'sample neuron vehicle brain rna sorted cells',\n",
       "  'sample microglia vehicle brain rna sorted cells',\n",
       "  'mouse'),\n",
       " 'GSE217386-3-vs-5-human dn': (['Widespread genomic/molecular alterations of DNA helicases and their clinical/therapeutic implications across human cancer'],\n",
       "  'DNA helicases are essential to genomic stability by regulating DNA metabolisms and their loss-of-function mutations lead to genomic instability and predisposition to cancer. Paradoxically, overexpression of DNA helicases is observed in several cancers, but systemic analyses of their alterations and biological or clinical implications in human cancer have not been reported. Here we analyzed genomic and molecular alterations in 12 important DNA helicases across 33 cancer types from the TCGA dataset to provide an overview of their aberrations. Significant expression heterogeneity of 12 DNA helicases was observed. We calculated DNA helicase score (DHS) based on their expression, and categorized tumors into high, low and intermediate subtypes. High DHS subtypes were robustly associated with stemness, proliferation, hyperactivated oncogenic signaling, longer telomeres, total mutation burden, copy number alterations (CNAs) and shorter survival. Importantly, tumors with high DHSs exhibited stronger expression of alternative end-join (alt-EJ) factors, indicative of sensitivity to chemo- and radio-therapies. Several drugs are identified to inhibit DNA helicases, with the Auror A kinase inhibitor Danusertib as the strongest candidate. The aberrant expression of DNA helicases was associated with CNAs, DNA methylation and m6A regulators. Our findings thus reveal widespread dysregulation of DNA helicases and their broad connection with featured oncogenic aberrations across human cancer. The close association of DHS with the alt-EJ pathway and cellular proliferation suggests that tumors with higher DHS are sensitive to chemotherapy and radiotherapy; The Auror A kinase inhibitor Danusertib may also inhibit DNA helicases. These findings thus contribute to DNA helicase-based cancer therapy.',\n",
       "  'adjacent non neoplastic sample age sex male tumor grade stage n m0 size',\n",
       "  'clear cell renal carcinoma sample age sex tumor grade stage n m0 size',\n",
       "  'human'),\n",
       " 'GSE137824-0-vs-1-human dn': (['Systematic identification of tumor-intrinsic resistance mechanisms and combinatorial therapy targets for immunotherapy'],\n",
       "  'A number of reports have demonstrated that tumor-intrinsic mechanisms of resistance, such as the loss of genes critical for antigen presentation and inflammatory responses, along with the activation of various cellular signaling cascades, can limit the efficacy of immunotherapy. Strategies to sensitize tumor cells to immunotherapy may overcome some resistance mechanisms, but identifying therapeutic targets has remained challenging. Here, we integrate a two-cell type (2CT) whole-genome CRISPR-Cas9 screen with dynamic transcriptional profiling of the tumor/T cell interaction to comprehensively identify tumor genes that are induced to promote tumor survival. We assessed the therapeutic potential of pharmacological inhibition of these and other top CRISPR identified targets as combinatorial targets to improve the efficacy of tumor destruction by T cells.',\n",
       "  'sgb veh cell line a375 melanoma cells transduced cas9 birc2 sgrna',\n",
       "  'mel624 star.genome.sorted cell line melanoma cells interaction',\n",
       "  'human'),\n",
       " 'GSE117614-2-vs-8-human up': (['RNA-seq analysis of canonical and adaptive human NK cell and CD8+ T cell subsets from HCMV seropositive donors'],\n",
       "  'We report our results of RNA-seq analysis on freshly isolated, sorted subsets of cytotoxic lymphocytes',\n",
       "  '6 tag facs sorted effector memory cells cd3+cd56 cd4 cd8+cd45ra+cd57+ peripheral blood mononuclear',\n",
       "  'tag facs sorted early mature canonical nk cells cd3 cd56dimcd57 nkg2c peripheral blood mononuclear',\n",
       "  'human'),\n",
       " 'GSE196694-2-vs-1-human dn': (['The Transcriptomic Landscape of Pediatric Astrocytoma'],\n",
       "  'Tumors of the central nervous system are the most common solid neoplasia during human childhood, representing one of the leading causes of cancer-related mortality. Tumors that originate from astrocyte cells (astrocytoma) in the brain are the most frequently found. According to their histological and pathological features, these tumors are classified into four categories. However, recently an extra layer of molecular classification of the tumorigenesis-associated genes IDH1/2 and H3F3A has been incorporated into the classification guidelines. While mutations in H3F3A are exclusively found in a subtype of pediatric astrocytoma grade IV, mutations in IDH1/2 are very rare in children younger than 14 years old. The transcriptomic profiles of astrocytoma in adults and children have been extensively studied however focusing on the study of the transcriptomic profile of the different grades of astrocytoma (including the additional layer of molecular classification) in pediatric populations are scarce. Therefore, we have profiled the transcriptomic landscape of the four grades of pediatric astrocytoma by RNA sequencing.',\n",
       "  'b cerebrum type 07 histological grade molecular classification pathology control',\n",
       "  '.iii type 07 anaplastic astrocytoma histological grade molecular classification idh wildtype pathology iii',\n",
       "  'human'),\n",
       " 'GSE191081,GSE191083-2-vs-1-human up': (['Specific methylation marks in promoter regions are associated to the pathogenic process of Chronic Chagas disease Cardiomyopathy by modifying transcription factor binding patterns  [RNA-Seq]'],\n",
       "  'Chagas disease, caused by Trypanosoma cruzi, is an endemic parasitical disease of Latin America, affecting 7 million people. Although most patients are asymptomatic, 30% develop complications, including Chronic Chagasic Cardiomyopathy (CCC), which ranges\\u202ffrom moderate to severe stages depending on the cardiac ejection fraction. The pathogenic process remains poorly understood, although genetic and epigenetic factors have already been proposed. Based on bulk RNA-seq and EPIC methylation data, we investigated\\u202fthe genetic and epigenetic deregulations present in the moderate and severe stages of CCC. We identified 4 main biological processes associated with the pathology\\u202fdevelopment, including\\u202fimmune response, ion transport, cardiac muscle processes and nervous system. An\\u202fin-depth study of\\u202fthe transcription factors binding sites in the\\u202fdifferentially methylated regions corroborated the importance of these processes. We also conducted a methylation study on blood to identify potential biomarkers for CCC. Our data revealed 198 differentially methylated positions (DMPs) that could serve as biomarkers of the disease, of which 61 are associated with disease severity.',\n",
       "  '[rna seq] left ventricular wall group non chagasic control heart',\n",
       "  '[rna seq] left ventricular wall group chronic chagasic cardiomyopathy heart',\n",
       "  'human'),\n",
       " 'GSE121446-1-vs-2-mouse dn': (['RNA-seq profiling of plasmacytoid and conventional dendritic cells from wildtype and PU.1 knockout mice'],\n",
       "  'Dendritic cells (DCs) are can be broadly divided into conventional (cDC) and plasmacytoid (pDC) subsets. Despite the importance of this lineage diversity, its genetic basis is not fully understood. RNA-seq gene expression analysis of wild type (WT) and PU.1 transgenic mice revealed a key role for PU.1 in maintaining cDC identity.',\n",
       "  'background strain c56bl/6 spleen /variation library prepation protocol truseq plasmacytoid dentritic cells',\n",
       "  'background strain c56bl/6 spleen /variation library prepation protocol truseq conventional dendritic cells',\n",
       "  'mouse'),\n",
       " 'GSE223100,GSE223842-5-vs-11-mouse dn': (['pDC-like cells are pre-DC2 and require Klf4 to control homeostatic CD4 T cells'],\n",
       "  'Plasmacytoid dendritic cells (pDCs) have been shown to play an important role during immune responses, ranging from initial viral control through the production of type I interferons (IFNs) to antigen presentation. However, recent evidence uncovered unexpected heterogeneity among pDCs. Notably we identified a new subset, referred to as pDC-like cells, that resembles pDCs but shares cDC features. Here we show that this subset is a circulating progenitor distinct from common DC progenitors (CDP), with prominent cDC2 precursor potential. This precursor subset responds to homeostatic cytokines, such as macrophage colony stimulating factor (M-CSF) by expanding and differentiating into cDC2 that efficiently prime Th17 cell. Development of pDC-like cells into CX3CR1+ESAM- cDC2b but not CX3CR1- ESAM+ cDC2a requires the transcription factor KLF4. Finally, we show that under homeostatic conditions this developmental pathway regulates the immune threshold at barrier sites, by controlling the pool of Th17 cells within the skin draining lymph nodes.',\n",
       "  'thy pdcs thymus (thy) replicate',\n",
       "  'spl spleen (spl) replicate',\n",
       "  'mouse'),\n",
       " 'GSE159094-1-vs-3-human dn': (['Development of a glucocorticoid exposure signature for SLE'],\n",
       "  'We performed RNA-seq on peripheral blood mononuclear cells (PBMC) from healthy human donors treated with interferon alpha 2a (1000IU/mL) +/- dexamethasone 10^-7M, with the aims of studying interactions between IFN and glucocorticoid induced gene expression, as well as identifying potential transcripts that may be used as a glucocorticoid exposure signature in clinical practice in patients with SLE. Transcripts identified from RNA-seq data were analysed in SLE patient data sets to validate clinical utility. We found that dexamethasone minimally impacts on interferon stimulated gene expression however IFN altered gene transcription of many previously reported glucocorticoid induced genes.',\n",
       "  'pbmc dex + ifn donor individual peripheral blood mononuclear cells dex.ifn sex',\n",
       "  'pbmc dex donor individual peripheral blood mononuclear cells sex',\n",
       "  'human'),\n",
       " 'GSE166036,GSE166059-7-vs-3-human up': (['Examining gene expression profile from varying preparations of fibrotic lung tissue'],\n",
       "  'Gene expression of human lung tissue bronchoalveolar lavage (BAL), whole tissue biopsy, and digested single cell suspension of lung tissue from control and ILD samples.',\n",
       "  'normal bal sample id disease lung bronchoalveolar lavage',\n",
       "  'ipf bal sample id disease lung bronchoalveolar lavage',\n",
       "  'human'),\n",
       " 'GSE145641-5-vs-6-human up': (['Recapitulation of Prostate Tissue Cell Type-specific Transcriptomes by an in vivo Primary Prostate Tissue Xenograft Model'],\n",
       "  'Purpose: Validate the cell type specific (Epithelial, Endothelial, and Stromal) transcriptome profile between prostate primary tissue xenografts and of fresh prostate tissue.  Nude-Fox1nu mice were castrated and silastic tubing containing dry testosterone (T) was inserted providing “humanized” levels of circulating T.  5-8 pieces of fresh human prostate tissue was then inserted subcutaneously along the flank of the animal.  After 30 days the animal was euthanized and tissue was harvested.  Tissue from xenograft and from fresh human prostate was then enriched into endothelial, epithelial and the remaining stromal compartments using magnetic bead enrichment.  RNA-Seq was then preformed on cell types from both origins and the results compared to show the primary tissue xenograft model faithfully recapitulates transcriptomic profile of the human prostate for epithelial, endothelial and stromal cell types.',\n",
       "  'nt epi fresh individual epithelial source prostate',\n",
       "  'stroma fresh individual stromal source prostate',\n",
       "  'human'),\n",
       " 'GSE163568-1-vs-2-human dn': (['Notch signaling rapidly regulates the expression of the small GTPase RND1 and a diverse endothelial transcriptome'],\n",
       "  'Endothelial Notch signaling regulates transcription of many downstream effectors controlling sprouting, migration, proliferation, barrier formation, and other phenotypes. However, endothelial-relevant Notch targets are largely uncharacterized. Few are studied in depth, and many transient, early response Notch targets are yet to be described. We therefore determined the Notch early response transcriptional profile in several experimentally relevant in vivo and in vitro contexts: ligand-specific or EGTA-induced Notch activation in different primary endothelial cells, and gamma secretase-mediated Notch inhibition in neonatal brain endothelium in a RiboTag mouse model. The intersection of these profiles reveals a core set of early Notch-responsive targets.  Among these are effectors in GPCR signaling, not previously implicated downstream of Notch signaling. We demonstrate the Rho GTPase RND1 is a direct Notch transcriptional target and required for Notch control of sprouting angiogenesis, endothelial migration and Ras activity.',\n",
       "  'huvec tla control primary cells (human umbilical vein endothelial cells) human (huvec)',\n",
       "  'hrec tla dll4 primary cells (human microvascular retinal endothelial cells) human (hrec)',\n",
       "  'human'),\n",
       " 'GSE152820-2-vs-5-human up': (['Involvement of SIRT1-regulated transcription profiles in prostate cancer'],\n",
       "  'In order to understand the mechanism associated with SIRT1-mediated development of prostate cancer progression, we conducted RNA-sequencing analysis in SIRT1 suppressed hormone sensitive prostate cancer cells (LNCaP) under conditions of androgen sufficiency, deprivation, androgen stimulation or AR suppression.',\n",
       "  'ln ntc de 1 cell line lncap hormone sensitive prostate cancer shrna scrambled css supplemented medium nm dihydrotestosterone stimulation plus 10 âµm enzalutamide 24 hr',\n",
       "  'ln ntc 1 cell line lncap hormone sensitive prostate cancer shrna scrambled css supplemented medium nm dihydrotestosterone stimulation 24 hr',\n",
       "  'human'),\n",
       " 'GSE213323-0-vs-5-human dn': (['Endothelial Cell Cycle Responses to Laminar Shear Stress'],\n",
       "  'Quiescence is a tempoary arrest from the cell cycle that can be induced by many methods, including contact inhibition, fluid shear stress, and serum starvation. p27 is important to inhibit cell cycle progression. Here we evaluate transcriptional differences between three different methods of quiescence induction, flow, contact inhibition and serum starvation, in the presence or depletion of p27.',\n",
       "  'p27 ld biol rep cell line huvec endothelial sirna low density',\n",
       "  'p27 flow biol rep cell line huvec endothelial sirna 72h shear stress 15 dyn/cm2',\n",
       "  'human'),\n",
       " 'GSE151290-3-vs-5-human up': (['Identification of novel transcription factors that regulate prostate cancer cell metabolism'],\n",
       "  'To understand of the impact of KLF5 and NFYA on metabolism in prostate cancer cells',\n",
       "  'lncap cells neg cell line transfection control sirna 96 % ethanol',\n",
       "  'lncap cells pos cell line transfection siklf5 synthetic androgen (r1881)',\n",
       "  'human'),\n",
       " 'GSE141136-2-vs-1-human dn': (['A Stem Cell-Derived Endothelial Cell Model that Responds to Tobacco Smoke Like Primary Endothelial Cells'],\n",
       "  'To clarify how smoking leads to heart attacks and stroke, we developed an endothelial cell model (iECs) generated from human induced Pluripotent Stem Cells (iPSC) and evaluated its responses to tobacco smoke. These iECs exhibited a uniform endothelial morphology, expressed markers PECAM1/CD31, VWF/von Willebrand Factor, and CDH5/VE-Cadherin, and exhibited tube formation and Acetyl-LDL uptake comparable to primary endothelial cells (EC). RNA sequencing (RNAseq) revealed a robust correlation coefficient between iECs and EC (R = 0.76), whereas gene responses to smoke were qualitatively nearly identical between iECs and primary ECs (R=0.86). Further analysis of transcriptional responses implicated eighteen transcription factors in regulating responses to smoke treatment, and identified gene sets regulated by each transcription factor, including oxidative stress, DNA damage/repair, ER stress, apoptosis, and cell cycle arrest. Assays for 42 cytokines in HUVEC cells and iECs identified 23 cytokines that responded dynamically to cigarette smoke. These cytokines and cellular stress response pathways describe endothelial responses for lymphocyte attachment, activation of coagulation and complement, lymphocyte growth factors, and inflammation and fibrosis; EC-initiated events that collectively lead to atherosclerosis. Thus, these studies validate the iEC model and identify transcriptional response networks by which ECs respond to tobacco smoke. Our results systematically trace how ECs use these response networks to regulate genes and pathways, and finally cytokine signals to other cells, to initiate the diverse processes that lead to atherosclerosis and cardiovascular disease.',\n",
       "  'huvec control 6h cell long name human umbilical vein endothelial iec clone n/ study arm response smoke line characterization',\n",
       "  'ipsc control cell long name induced pluripotent stem iec clone n/ study arm line characterization',\n",
       "  'human'),\n",
       " 'GSE188992-0-vs-1-mouse up': (['Bulk RNAseq of Mouse Splenic Dendritic Cell Subsets'],\n",
       "  'Dendritic cells (DCs) are comprised of a heterogenous population. Here we characterize the transcriptomic profiles of mouse splenic DC subsets at steady state to highlight key differences between subsets.',\n",
       "  'b6 cd8pos cdc1 strain c57bl/6j state steady spleen dc cell subset',\n",
       "  'b6 pdc strain c57bl/6j state steady spleen dc cell subset',\n",
       "  'mouse'),\n",
       " 'GSE173509-2-vs-6-human up': (['Time-course transcriptomic profiling of cell-cycle entry in MCF10A cells'],\n",
       "  'To identify potential novel regulators of G1/S transition and S-phase progression, we performed an RNA sequencing (RNA-Seq) time-course analysis of non-transformed human breast epithelial cells (MCF10A) released into the cell cycle from serum starvation, in the presence of DMSO or a specific CDK4/6 inhibitor palbociclib.',\n",
       "  'mcf10a cdk4/6i time since release (hours) drug palbociclib breast epithelial cell line non transformed (mcf10a)',\n",
       "  'mcf10a dmso time since release (hours) drug breast epithelial cell line non transformed (mcf10a)',\n",
       "  'human'),\n",
       " 'GSE115168-1-vs-4-human up': (['ER proteostasis and temperature differentially impact the mutational tolerance of influenza hemagglutinin'],\n",
       "  'Here, we present a systematic and quantitative test of the hypothesis that the composition and activities of the endoplasmic reticulum (ER) proteostasis network impact mutational tolerance of secretory pathway client proteins. We focus on influenza hemagluttinin (HA), a viral coat protein that folds in the host’s ER via a complex but well-characterized pathway. By integrating chemical methods to modulate the unfolded protein response with deep mutational scanning to assess mutational tolerance, we discover that upregulation of ER chaperones broadly enhances HA mutational tolerance across numerous sites and secondary/tertiary structure elements, including sites targeted by host antibodies. Remarkably, this host chaperone-enhanced mutational tolerance is observed at the same HA sites where mutational tolerance is most reduced by propagation at a fever-like temperature. Thus, host ER proteostasis mechanisms and temperature modulate HA mutational tolerance in opposite directions. This finding has important implications for influenza evolution, because influenza immune escape is contingent on HA possessing sufficient mutational tolerance to acquire antibody resistance while still maintaining the capacity to fold and function. More broadly, this work provides the first experimental evidence that the composition and activities of the ER proteostasis network critically define the mutational tolerance and, therefore, the evolution of secretory pathway client proteins.',\n",
       "  'd17 basal 39c adult derived epithelial cell line human embryonic kidney (hek) 293t rex clonal stably expressing dhfr.atf6 tet xbp1s tetracycline repressor condition 24h 0.01% dmso cells',\n",
       "  'd17 xbp1s 37c adult derived epithelial cell line human embryonic kidney (hek) 293t rex clonal stably expressing dhfr.atf6 tet tetracycline repressor condition 24h 0.1 ug/ml doxycycline cells',\n",
       "  'human'),\n",
       " 'GSE166209-1-vs-5-human dn': (['Pharmacological Activation of STING blocks SARS-CoV-2 infection'],\n",
       "  'We infected Calu3 cells (derived from human lung epithelium) with SARS-CoV-2 for 24 and 48 hours. Calu3 cells were also treated with the STING agonist diABZI for 6 and 12 hours. Finally, heterozygous K18-hACE c57BL/6J mice were treated with diABZI for 6 and 12 hours and lung tissue was harvested for sequencing.',\n",
       "  'calu3 sars cov2 48h lung epithelial cells mock infection time 48hrs',\n",
       "  'calu3 noninfected 48h lung epithelial cells mock infection time 48hrs',\n",
       "  'human'),\n",
       " 'GSE127831-1-vs-3-human up': (['Variable gene expression and parasite load predict treatment outcome in cutaneous leishmaniasis'],\n",
       "  'Patients infected with Leishmania braziliensis develop chronic lesions that often fail to respond to treatment. To determine whether genes whose expression is highly variable in lesions might influence disease outcome, we obtained biopsies of lesions from patients prior to drug treatment, performed transcriptomic profiling, and identified highly variable genes whose expression correlated with treatment outcome. Amongst the most variable genes were components of the cytolytic pathway, the expression of which appeared to be driven by parasite load in the skin. We demonstrated that treatment failure can be directly linked to the cytolytic pathway activated during infection. Using this host-pathogen biomarker profile, we show that treatment outcome can be predicted before the start of treatment. These findings not only raise the possibility of point-of-care diagnostic screening to identify patients at high risk of treatment failure and provide a rationale for a precision medicine approach to drug selection in cutaneous leishmaniasis, but more broadly also demonstrate the value of identifying genes of high variability in other diseases to better understand and predict diverse clinical outcomes.',\n",
       "  'skin subject infected infection outcome na age sex lesion size (mm^3) dth number parasite transcripts (rna seq) parasites (qpcr) time cure (days) healthy',\n",
       "  'skin subject cutaneous leishmaniasis infection leishmania braziliensis outcome failure age sex lesion size (mm^3) dth number parasite transcripts (rna seq) parasites (qpcr) time cure (days)',\n",
       "  'human'),\n",
       " 'GSE143915-3-vs-0-mouse dn': (['Transcriptomic and Proteomic Signatures of Stemness and Differentiation in the Colon Crypt'],\n",
       "  'Abstract: The intestinal epithelium is replaced weekly by non-quiescent stem cells with kinetics that rely on a rapid loss of stemness and choice for secretory or absorptive lineage differentiation. To determine how the cellular transcriptome and proteome changes during these transitions, we developed a new cell sorting method to purify stem cells, secretory and absorptive progenitor cells, and mature, differentiated cells. Transcriptome analyses revealed that as stem cells transition to the progenitor stage, alternative mRNA splicing and polyadenylation dominate changes in the transcriptome. In contrast, as progenitors differentiate into mature cell types, alterations in gene expression and mRNA levels drive the changes. RNA processing targets mRNAs encoding regulators of cell cycle, RNA regulators, cell adhesion, SUMOylation, and Wnt and Notch signaling. Additionally, carrier-assisted mass spectrometry of sorted cell populations detected >2,800 proteins and revealed RNA:protein patterns of abundance and correlation. Paired together, these data highlight new potentials for autocrine and feedback regulation and provide new insights into cell state transitions in the crypt.',\n",
       "  'secpdg strain background c57bl/6n /variation wild type age 5 7 weeks sex male colon sorted cell cd31 cd45 epcam+ cd44hi cd24med ckitmed live br',\n",
       "  'stem strain background c57bl/6n /variation wild type age 5 7 weeks sex male colon sorted cells cd31 cd45 epcam+ cd44highest cd24low ckit live br',\n",
       "  'mouse'),\n",
       " 'GSE175686-1-vs-2-human dn': (['Telomeric 8-Oxoguanine Drives Rapid Premature Senescence in the Absence of Telomere Shortening'],\n",
       "  'Oxidative stress is a primary cause of cellular senescence and contributes to the pathogenesis of numerous human diseases. Oxidative damage to telomeric DNA is proposed to trigger premature senescence by accelerating telomere shortening. Here we tested this model directly using a precision tool to produce the common base lesion 8-oxoguanine (8oxoG) exclusively at telomeres in human fibroblast and epithelial cells. A single induction of telomeric 8oxoG is sufficient to trigger multiple hallmarks of p53-dependent senescence. Telomeric 8oxoG activates ATM and ATR signaling, and enriches for markers of telomere dysfunction in replicating, but not quiescent cells.  Acute 8oxoG production fails to shorten telomeres, but rather generates fragile sites and delayed mitotic DNA synthesis at telomeres, indicative of impaired replication. Based on our results we propose that oxidative stress promotes rapid senescence by producing oxidative base lesions which drive replication-dependent telomere fragility and dysfunction in the absence of shortening and shelterin loss.',\n",
       "  'bj ut cell line 5ta htert forskin fibroblasts /variation fap mcer trf1 untreated',\n",
       "  'bj dl cell line 5ta htert forskin fibroblasts /variation fap mcer trf1 treated dye+light',\n",
       "  'human'),\n",
       " 'GSE136574-0-vs-1-mouse up': (['Genome-wide RNA sequencing on myeloid derived suppressor cells from mice treated with lymphodepleting chemotherapy'],\n",
       "  'Myeloid derived suppressor cells (MDSCs) were sorted from B16 tumor-bearing mice that were untreated or given lymphodepleting cyclophosphamide and fludarbine.  We report that monocytic and polymorphonuclear MDSC subsets (M-MDSCs and PMN-MDSCs respectively) have a unique gene expression profile in comparison to MDSCs collected from untreated tumor-bearing animals.',\n",
       "  'mdsc ctrl strain c57bl/6 spleen monocytic myeloid derived suppressor cells ( mdscs) treatement untreated',\n",
       "  'pmn mdsc plus ld strain c57bl/6 spleen polymorphonuclear myeloid derived suppressor cells (pmn mdscs) treatement lymphodepleted',\n",
       "  'mouse'),\n",
       " 'GSE189847-4-vs-1-human up': (['A transcriptomic analysis of the effects of macrophage polarization and endotoxin tolerance on the response to Salmonella'],\n",
       "  'Salmonella is an intracellular pathogen causing significant morbidity and mortality. Its ability to grow inside macrophages is important to virulence, and is dependent on the activation state of the macrophages. Classically activated M1 macrophages are non-permissive for Salmonella growth, while alternatively activated M2 macrophages are permissive for Salmonella growth. Here we showed that endotoxin-primed macrophages (MEP), such as those associated with sepsis, showed similar levels of Salmonella resistance to M1 macrophages after 2 hr of intracellular infection, but at the 4 hr and 24 hr time points were susceptible like M2 macrophages. To understand this mechanistically, transcriptome sequencing, RNA-Seq, was performed. This showed that M1 and MEP macrophages that had not been exposed to Salmonella, demonstrated a process termed here as primed activation, in expressing relatively higher levels of particular anti-infective genes and pathways, including the JAK-STAT pathway. In contrast, in M2 macrophages these genes and pathways were largely expressed only in response to infection. Conversely, in response to infection, M1 macrophages, but not MEP macrophages, modulated additional genes known to be associated with susceptibility to Salmonella infection, possibly contributing to the differences in resistance at later time points. Application of the JAK inhibitor Ruxolitinib before infection reduced resistance in M1 macrophages, supporting the importance of early JAK-STAT signalling in M1 resistance to Salmonella.',\n",
       "  'm0 un polarization ctrl untreated donor monocyte derived macrophages',\n",
       "  'm0 inf polarization ctrl typh donor monocyte derived macrophages',\n",
       "  'human'),\n",
       " 'GSE175495-6-vs-2-human dn': (['Divergent immunometabolic changes in adipose tissue and skeletal muscle with ageing in healthy humans'],\n",
       "  \"RNA-sequencing from 12 young (20-35 years old) and 12 older (60-85 years old) participants' skeletal muscle and adipose tissue. Biopsies were taken at rest following an overnight fast, and following alcohol/caffeine restriction for 24 hours, and no strenuous physical activity for 48 hours. Skeletal muscle was taken by the Bergstrom technique under local anaesthesia (Lidocaine hydrochloride) from the dominant leg Vastus Lateralis, and adipose tissue was taken by needle aspiration 5cm lateral to the umbilicus.\",\n",
       "  'old skeletalmuscle age vastus lateralis skeletal muscle dominant leg sample type untreated rested healthy human whole (skeletal muscle)',\n",
       "  'young adiposetissue age subcutaneous abdominal adipose sample type untreated rested healthy human whole (adipose )',\n",
       "  'human'),\n",
       " 'GSE123541-0-vs-3-human dn': (['Genome-wide RNA-seq and ATAC-seq analysis of an in vivo human immune response to acute Zika virus infection'],\n",
       "  'A longitudinal systems immunologic investigation of acute Zika virus infection in an individual infected with ZIKV while traveling to Caracas, Venezuela during the zika virus epidemic. Includes longitudinal RNA-seq and ATAC-seq characterization of innate and adaptive immune responses during an acute ZIKV infection.',\n",
       "  'cd14dimcd16pos subject status individual naat confirmed acute zika virus infection pbmc cell subtype monocyte time post onset symptoms days',\n",
       "  'cd4tcell subject status individual naat confirmed acute zika virus infection pbmc cell subtype cd4 time post onset symptoms days',\n",
       "  'human'),\n",
       " 'GSE222553-3-vs-4-human dn': (['Investigating iRhom2-associated transcriptional changes in Tylosis with Esophageal Cancer'],\n",
       "  'BACKGROUND AND AIMS: Survival rates for esophageal squamous cell carcinoma (ESCC) are extremely low due to the late diagnosis of most cases. An understanding of the early molecular processes that lead to ESCC may facilitate opportunities for early diagnosis; however, these remain poorly defined. Tylosis with esophageal cancer (TOC) is a rare syndrome associated with a high lifetime risk of ESCC and germline mutations in RHBDF2, encoding iRhom2. Using TOC as a model of ESCC predisposition, this study aimed to identify early-stage transcriptional changes in ESCC development.',\n",
       "  'esophageal control wildtype',\n",
       "  'esophageal toc rhbdf2i186t/+',\n",
       "  'human'),\n",
       " 'GSE210497-6-vs-9-mouse dn': ([\"SARS-CoV-2 pathogenesis mediated by E protein's PBM was prevented by modulators of CFTR function\"],\n",
       "  'The three human deadly coronaviruses (CoVs) (SARS-CoV, MERS-CoV and SARS-CoV-2) have infected around eight thousand, two thousand six hundred and 262 million people so far (December 2021), causing the death of 10%, 37% and 2% of them, respectively, so their implication in health is very important. These CoVs have proteins with a PBM motif that binds to PDZ cell domains. PDZ domains are found in more than 400 cellular proteins, therefore, viruses with PBM motifs have a high potential to modify cell behavior. It was studied the implication of the E protein PBM motif of various virulent or attenuated human CoVs (hCoVs) in the pathogenesis induced by these viruses. Variants of SARS-CoV, MERS-CoV and SARS-CoV-2 that lack the E protein PBM have been generated by reverse genetics and their pathogenicity has been analyzed in mice. The PBM motifs of these three hCoVs have been shown to be virulence factors and participate in their replication. Furthermore, a collection of SARS-CoV mutants has been constructed in which the E protein PBM domain was replaced by the one derived from virulent or attenuated hCoVs, and their virulence was analyzed. A virulence gradient was observed, depending on whether the E protein PBM domain was derived from an attenuated or virulent hCoV. The gene expression patterns associated with the different PBM motifs in lungs of mice infected with SARS-CoV was analyzed by deep sequencing of the mRNAs expressed in lungs of infected mice, and it was observed that the E protein PBM motif of SARS-CoV and SARS-CoV-2 dysregulates the expression of genes related to ion transport and cell homeostasis. Specifically, a decrease of the mRNA expression of the cystic fibrosis transmembrane conductance regulator (CFTR), which is essential for edema resolution, was observed. The reduction of CFTR mRNA levels was associated with edema accumulation in lungs of mice infected with SARS-CoV and SARS-CoV-2. The effect of compounds that modulate the expression and activity of CFTR on the pathogenesis and replication of SARS-CoV-2 was studied and it was observed that these compounds drastically reduced the production of SARS-CoV-2 in cell culture and protect against SARS-CoV-2 pathogenesis. These results showed the high relevance of the PBM motif in the replication and virulence of CoVs, and have allowed the identification of cellular targets for the selection of antivirals.',\n",
       "  '3a 2dpã¬ [e82 4 lung strain balb/c time 2dpi infected sars cov carrying pbm variant \"3a\"',\n",
       "  'mock 4dpã¬ [e93 2 lung strain balb/c time 4dpi infected',\n",
       "  'mouse'),\n",
       " 'GSE178615-6-vs-7-human dn': (['Coordinated metabolic transitions and gene expression by NAD+ during adipogenesis.'],\n",
       "  'Adipocytes are the main cell type in adipose tissue, a critical regulator of metabolism, highly specialized in storing energy as fat. Adipocytes differentiate from multipotent mesenchymal stromal cells through adipogenesis. Using RNA-seq we explored the effect of manipulating NAD+ bioavailability during adipogenic differentiation from human mesenchymal stem cells. Cells were harvested either undifferentiated (hMSC) or after days 8 and 16 of adipogenic induction. During adipogenesis, cells were treated with either NAD+ (5 mM), or FK866 (1 nM) or EX527 (50 μM).',\n",
       "  'adipo+fk866 day8 cell line mesenchymal differentiation stage adipogenic induction days 8 fk866 human',\n",
       "  'adipo+nad+ day16 cell line mesenchymal differentiation stage adipogenic induction days 16 nad+ human',\n",
       "  'human'),\n",
       " 'GSE86573-4-vs-5-mouse up': (['Macrophage ontogeny underlies differences in tumor-specific education in brain malignancies'],\n",
       "  'Recent efforts have uncovered immense transcriptional and ontogenetic diversity among tissue-resident macrophages, each with their own transcriptional profile endowing the cell with its tissue-specific functions. However, it is currently unknown whether the origins of different macrophage populations may affect their roles in malignancy. Given potential artifacts associated with irradiation-based lineage tracing, it remains unclear if bone marrow-derived macrophages (BMDM) are even present in tumors of the brain, a tissue where there is no homeostatic involvement of peripherally-derived myeloid cells. Here, we employed multiple models of murine brain malignancy and genetic lineage tracing models to demonstrate that BMDM are indeed abundant in primary and metastatic brain tumors. Transcriptional profiling of tumor-associated BMDM and resident microglia showed that these cells acquire substantially different gene expression profiles. Our data suggest that transcriptional networks in each cell population are associated with tumor-mediated education, yet are also influenced by chromatin landscapes established before tumor initiation. Furthermore, we demonstrate that microglia specifically repress Itga4 (CD49D), enabling its utility as a discriminatory marker between brain-resident microglia and peripherally-derived macrophages in both primary and metastatic disease in mouse and human.',\n",
       "  'gemm mg tam microglia strain c57bl/6 tumor associated',\n",
       "  'gemm bmdm tam bone marrow derived macrophage strain c57bl/6 tumor associated',\n",
       "  'mouse'),\n",
       " 'GSE249723,GSE266408-5-vs-8-human up': (['ESR1 fusions invoke breast cancer subtype-dependent enrichment of ligand independent oncogenic signatures and phenotypes [RNA-seq]'],\n",
       "  'Breast cancer is a leading cause of female mortality and despite advancements in personalized therapeutics, metastatic disease largely remains incurable due to drug resistance. The estrogen receptor (ER, ESR1) is expressed in two-thirds of all breast cancer, and under endocrine stress, somatic ESR1 mutations arise in ~30% of cases that result in endocrine resistance. We and others reported ESR1 fusions as a mechanism of ER mediated endocrine resistance. ER fusions, which retain the AF1 and DNA binding domains, harbor ESR1 exons 1-6 fused to an in-frame gene partner resulting in loss of the ER ligand binding domain (LBD). We demonstrate that in a no-special type (invasive ductal carcinoma (IDC)- NST) and an invasive lobular carcinoma (ILC) cell line, ER fusions exhibit robust hyperactivation of canonical ER signaling pathways independent of estradiol or anti-endocrine therapies. We employ cell line models stably overexpressing ER fusions with concurrent endogenous ER knockdown to minimize endogenous ER influence. Cell lines exhibited shared transcriptomic enrichment in pathways known to be drivers of metastatic disease, notably MYC signaling. Cells expressing the 3’ fusion partners SOX9 and YAP1 consistently demonstrated enhanced growth and cell survival. ILC cells expressing the DAB2 fusion led to enhanced growth, survival, and migration; phenotypes not appreciated in the IDC- NST DAB2 model. Herein, we report that cell line activity is subtype-, fusion-, and assay-specific suggesting that LBD loss, the fusion partner, and the cellular landscape all influence fusion activities. Therefore, it will be critical to assess fusion frequency, in the context of the tumor clinicopathology.',\n",
       "  'esr1 dab2 shesr1 cell line t47d special type breast cancer veh duration 24hr',\n",
       "  'esr1 yap1 shesr1 cell line t47d special type breast cancer veh duration 24hr',\n",
       "  'human'),\n",
       " 'GSE161752-0-vs-1-human dn': (['Profiling the blood compartment of Hematopoietic Stem Cell Transplant patients during Human Cytomegalovirus reactivation'],\n",
       "  'Human cytomegalovirus (HCMV) reactivation is a leading infectious cause of morbidity and mortality in hematopoietic stem cell transplantation (HSCT) patients. HCMV reactivation is characterized by detection of viral DNA in the blood (DNAemia), however, the role of the blood compartment during HCMV reactivation is not well characterized. Using powerful molecular tools, we characterized HCMV infection in Peripheral Blood Mononuclear Cells (PBMCs) during DNAemia in HCMV reactivating HSCT patients. Surprisingly we found that all PBMC populations harbored extremely low levels of viral DNA and extremely low levels of viral transcripts. Remarkably, the viral transcripts detected in these samples were mainly immediate early genes indicating that these cells do not go through a full productive cycle. Analysis of the cellular transcriptome revealed characteristic transcriptional changes in high DNAemic samples implying the development of HCMV-specific T cells and inhibition of regulatory T cell function. Overall, our data indicate that DNAemia in HCMV reactivating HSCT patients is associated with distinct immune signatures but is not necessarily accompanied by substantial infection of PBMCs, and that PBMCs are not productively infected. These findings are important for understanding the role of the blood compartment in progression and control of HCMV during reactivation.',\n",
       "  'pbmc high viremia hsct',\n",
       "  'fibroblasts hpi fibroblast infected hcmv tb40',\n",
       "  'human'),\n",
       " 'GSE118279-11-vs-9-human dn': (['Unr is a potential post-transcriptional master regulator of protein expression'],\n",
       "  'Ribonucleoprotein immunoprecipitation followed by mass spectrometry and RNA-Seq was used to investigate the ‘Upstream of N-RAS’ (UNR) protein interactome across three cancer cell lines. A number of novel UNR-interacting proteins and transcripts were discovered, including the ubiquitin E3 ligase, HUWE1; the nucleolar protein, NARR; the multifunctional protein, SQSTM1; and the erythropoiesis-related protein, LDB1\\xa0. GO-term analysis of putative UNR interactors suggested that UNR interacts with both proteins and mRNAs encoding proteins that are involved in RNA binding',\n",
       "  'saos igg cell line 2 disease state osteosarcoma stress control rip antibody (pulldown) (invitrogen 10500c) replicate',\n",
       "  'saos unr cell line 2 disease state osteosarcoma stress control rip antibody (pulldown) (polyclonal rabbit novus biologicals nbp1 71914) replicate',\n",
       "  'human'),\n",
       " 'GSE196860-5-vs-3-human dn': (['MBD5 and MBD6 stabilize the BAP1 complex and promote BAP1-dependent cancer'],\n",
       "  'The BRCA1-associated protein 1 (BAP1) is a ubiquitin carboxy-terminal hydrolase (UCH), which forms a multi-protein complex with different epigenetic factors such as ASXL1-3, and FOXK1/2. At chromatin, BAP1 catalyzes the removal of mono-ubiquitination on histone H2AK119 in collaboration with other subunits within the complex, and therefore functions as a transcriptional activator. However, the crosstalk between different subunits and how these subunits impact BAP1 function remains unclear. Here, we report the identification of the methyl-CpG-binding domain proteins 5 and 6 (MBD5 and MBD6) that bind to the C-terminal PHD fingers of the large scaffold subunits ASXL1-3 and stabilize the BAP1 complex at chromatin. We further identified a previously uncharacterized Drosophila protein, the six-banded (SBA), as the ortholog of human MBD5/6. We demonstrated the core module of the BAP1 complex is structurally and functionally conserved during the evolution from Drosophila (Calypso/ASX/SBA) to human cells (BAP1/ASXL/MBD). Dysfunction of the BAP1 complex induced by the misregulation/mutations in each subunit is frequent in human cancer. In BAP1-dependent human cancers, MBD6 tends to be a dominant form. Depletion of MBD6 leads to a global loss of BAP1 occupancy at chromatin, resulting in a reduction of BAP1-dependent gene expression and tumor growth in vitro and in vivo. In summary, our study has uncovered MBD5/6 as important regulators of the BAP1 complex and transcription, and sheds light on the therapeutic potential of targeting MBD5/6 in human cancer.',\n",
       "  'h1963 sgbap1 small cell lung cancer passages low (6 10) line normal cells nci',\n",
       "  'h1963 sgnont small cell lung cancer passages low (6 10) line normal cells nci',\n",
       "  'human'),\n",
       " 'GSE235602,GSE235603-2-vs-5-mouse dn': (['Lung tumor-infiltrating Treg have divergent transcriptional profiles and function linked to checkpoint blockade response [mouse]'],\n",
       "  'Regulatory T cells (Treg) are conventionally viewed to suppress endogenous and therapy-induced anti-tumor immunity; however, their role in modulating responses to immune checkpoint blockade (ICB) is unclear. In this study, we integrated single-cell RNAseq/TCRseq of >73,000 tumor-infiltrating Treg (TIL-Treg) from anti-PD-1-treated and treatment naive non-small cell lung cancers (NSCLC) with single cell analysis of tumor-associated antigen (TAA)-specific Treg derived from a murine tumor model. We identified 10 subsets of human TIL-Treg, most of which have high concordance with murine TIL-Treg subsets. Notably, only one subset selectively expresses high levels of OX40 and GITR, whose engangement by cognate ligand mediated proliferative programs and NF-kB activation, as well as multiple genes involved in Treg suppression, including LAG3. Functionally, the OX40hiGITRhi subset is the most highly suppressive ex vivo and its higher representation among total TIL-Treg correlated with resistance to PD-1 blockade. Surprisingly, in the murine tumor model, we found that virtually all TIL-Treg expressing T cell receptors that are specific for TAA fully develop a distinct Th1-like signature over a two-week period after entry into the tumor, down-regulating FoxP3 and up-regulating expression of TBX21 (Tbet), IFN\\uf067 and certain pro-inflammatory granzymes. Transfer learning of a gene score from the murine TAA-specific Th1-like Treg subset to the human single-cell dataset revealed a highly analogous subcluster that was enriched in anti-PD-1 responding tumors. These findings demonstrate that TIL-Treg partition into multiple distinct transcriptionally-defined subsets with potentially opposing effects on ICB-induced anti-tumor immunity and suggest that TAA-specific TIL-Treg may positively contribute to anti-tumor responses.',\n",
       "  'scrna seq spleen cell',\n",
       "  'sctcr seq spleen cell',\n",
       "  'mouse'),\n",
       " 'GSE178296,GSE178304-2-vs-3-mouse up': (['Transcriptome profiling of mouse brain microglia in Cnp null mice with 5xFAD background'],\n",
       "  'Alzheimer’s disease (AD) is the most common form of dementia and neurodegenerative disease with increasing prevalence due to longer lifespan in the human population. Why aging constitutes the greatest risk factor for development of AD, however, remains poorly understood. Aging markedly affects oligodendrocytes and the structural integrity of their myelin sheaths which also causes secondary tissue inflammation. We propose a mechanistic link between aging-associated myelin dysfunction and the deposition of Amyloid-ß (Aß) as primary neuropathological hallmark of early AD and hypothesized that breakdown of myelin - especially in cortical regions – is an upstream driver of amyloid deposition in AD. Here, we show that in transgenic mouse models of AD, genetically induced myelin defects by Cnp or Plp1 depletion, as well as direct demyelination are potent drivers of amyloid deposition in vivo as shown by light sheet microscopy imaging. At transcriptomic level, bulk and single-cell RNA sequencing revealed successfully induced disease-associated-microglia (DAM)-like phenotypes in Cnp-/- animals. These activated microglia, however, are primarily engaged with myelin seemingly preventing the protective reactions of the microglia pool to Aß plaques. Our work, therefore, identifies myelin aging as a previously overlooked risk factor for AD and makes the case for myelin health-directed therapies in AD.',\n",
       "  'wt bulkrna strain c57bl6/j age 6 month microglia brain mouse model alzheimerâ\\x80\\x99 disease (ad)',\n",
       "  'cnp 5xfad bulkrna strain c57bl6/j age 6 month / x microglia brain mouse model alzheimerâ\\x80\\x99 disease (ad)',\n",
       "  'mouse'),\n",
       " 'GSE182754,GSE183011-1-vs-0-human dn': (['Transcriptional effects of HSV-1-ICP0 and Ad5-E4ORF3 expression in BLaER1 monocytes [Virulence factors]'],\n",
       "  'We have identified that the viral virulence factors ICP0 from HSV-1 and E4ORF3 from Adenovirus5 can activate the MORC3 pathway of innate immune sensing. To study the global transcriptional effects of virulence factor sensing, we overexpressed mCherry or these virulence factors from a doxycycline-inducible lentiviral construct in BLaER1 monocytes of various genotypes and performed RNA-seq.',\n",
       "  'wt e4orf3 cell line blaer1',\n",
       "  'ifnar1ko ifnar2ko rep3 cell line blaer1 ifnar1 / ifnar2',\n",
       "  'human'),\n",
       " 'GSE107821-3-vs-2-human up': (['Transcriptomes of human monocytes after ex vivo exposure to uric acid'],\n",
       "  \"Innate immune memory, also refered to as trained immunity (TRIM) is the phenomenon whereby innate immune cells such as monocytes or macrophages undergo functional reprogramming after exposure to microbial components.  In this study  we exposed monocytes from healthy donors to uric acid ex vivo, and analyzed the effect of this exposure on gene expression changes. After 'uric acid' or 'media only' treatments, monocytes were exposed to LPS for 4 hours to measure they response to this pyrogen.\",\n",
       "  'rna rpmi 24 hours monocyte diagnosis healthy donor adult human peripheral blood',\n",
       "  'rna rpmi 24 hours lps 4h monocyte diagnosis healthy donor adult human peripheral blood',\n",
       "  'human'),\n",
       " 'GSE125066-2-vs-1-human up': (['Effect of Toxoplasma gondii efector TgIST on global transcriptome of human foreskin fibroblasts (HFFs) upon type I IFN activation'],\n",
       "  'TgIST is an effector secreted by Toxoplasma gondii into the host cell that translocates to the nucleus and block STAT1 mediated transcription. STAT1 is involved in type I and II interferon (IFN) mediated upregulation of range of anti-parasitic molecules.  A total of 544,234,059 read sequences generated from 3 independent biological replicates were mapped to human hg19 reference genome. Type I IFN activation of HFFs infected with TgIST knockout parasites showed upregulation of genes involved in interferon signaling compared to wild type parasites.',\n",
       "  'ui ut cell line hff fibroblasts (skin) infection uninfected untreated replicate',\n",
       "  'ui ifnb cell line hff fibroblasts (skin) infection uninfected ifn î² replicate',\n",
       "  'human'),\n",
       " 'GSE174293,GSE199346-8-vs-10-human dn': (['High-Throughput Transcriptome Profiling of Single Nuclei and Single Synapses Using Single-Cell Total-RNA-Seq [Hs]'],\n",
       "  'We developed the first droplet-based single-cell total-RNA-seq method. We refer to this platform as Multiple Annealing and Tailing-based Quantitative scRNA-seq in Droplet (MATQ-Drop). With the detection of nascent RNA species, we showed that the cell atlas of human brain samples could be effectively constructed based on nascent RNA species. Furthermore, we observed that only lncRNA species are sufficient to construct the cell atlas, suggesting that MATQ-Drop allows a large-scale identification of the cell-type-specific lncRNA species. Beyond total-RNA profiling for single nuclei, we also showed that MATQ-Drop could be used to profile the transcripts in different neuronal sub-compartments. Based on transcriptome profiling, we were able to determine different types of sub-compartments—in particular, synapses. Here we have referred to the transcriptome of individual synapses as the synaptome, and we identified different subtypes of pre-synapses and post-synapses. Furthermore, between pre-synapses and post-synapses, as well as between pre-synapses and the nuclei, we were able to identify different functional enrichments. Our result unveils unprecedented new insights about gene expression in individual synapses. It also demonstrates the feasibility of using MATQ-drop to profile the transcriptome of sub-cellular compartments.',\n",
       "  'pfc synapse prefrontal cortical tissues hoechst negative subneuronal compartments frozen postmortem human',\n",
       "  'hippo nucleus hippocampal tissues nuclei frozen postmortem human',\n",
       "  'human'),\n",
       " 'GSE183316,GSE183317-0-vs-5-mouse up': (['Tet2-mediated epigenetic programing of T follicular helper cell differentiation (RNA-Seq)'],\n",
       "  'Tet2-mediated demethylation is a key component of epigenetic programing that promotes lineage specific gene expression and contributes to cellular differentiation and function. While the differentiation of CD4+ T cell subsets has been studied extensively, the epigenetic programs that regulate these processes remain unclear. We report that Tet2 acts to restrict the differentiation of T follicular helper (Tfh) cells in CD4+ T cells responding to viral infection. Tet2-deficient CD4+ T cells preferentially differentiated into highly functional germinal center (GC) Tfh cells that provided enhanced help for B cell responses. Using genome-wide expression and methylation analyses combined with Foxo1 ChIPseq analysis, we found that Tet2 coordinates with multiple transcription factors, including Foxo1, to mediate the demethylation and expression of their target genes following activation.',\n",
       "  'ko naã¯ genetics tet2 background c57bl/6 mouse smarta cd4 cells cell subset naive infection infected sex spleen',\n",
       "  'ko th1 genetics tet2 background c57bl/6 mouse smarta cd4 cells cell subset infection lcmv armstrong 7 days post sex female spleen',\n",
       "  'mouse'),\n",
       " 'GSE174159-2-vs-6-human dn': (['Identifying new pronostic markers of IBD'],\n",
       "  'Studying the variations in VNN1 expression at different stages of the disease and under therapy,',\n",
       "  \"q intestinal biopsy disease state crohn' (cd) activity quiescent (q) immunosuppressors ()\",\n",
       "  'cort intestinal biopsy disease state ulcerative colitis (uc) activity severe () corticoids (cort)',\n",
       "  'human'),\n",
       " 'GSE115168-1-vs-5-human up': (['ER proteostasis and temperature differentially impact the mutational tolerance of influenza hemagglutinin'],\n",
       "  'Here, we present a systematic and quantitative test of the hypothesis that the composition and activities of the endoplasmic reticulum (ER) proteostasis network impact mutational tolerance of secretory pathway client proteins. We focus on influenza hemagluttinin (HA), a viral coat protein that folds in the host’s ER via a complex but well-characterized pathway. By integrating chemical methods to modulate the unfolded protein response with deep mutational scanning to assess mutational tolerance, we discover that upregulation of ER chaperones broadly enhances HA mutational tolerance across numerous sites and secondary/tertiary structure elements, including sites targeted by host antibodies. Remarkably, this host chaperone-enhanced mutational tolerance is observed at the same HA sites where mutational tolerance is most reduced by propagation at a fever-like temperature. Thus, host ER proteostasis mechanisms and temperature modulate HA mutational tolerance in opposite directions. This finding has important implications for influenza evolution, because influenza immune escape is contingent on HA possessing sufficient mutational tolerance to acquire antibody resistance while still maintaining the capacity to fold and function. More broadly, this work provides the first experimental evidence that the composition and activities of the ER proteostasis network critically define the mutational tolerance and, therefore, the evolution of secretory pathway client proteins.',\n",
       "  'd17 basal 39c adult derived epithelial cell line human embryonic kidney (hek) 293t rex clonal stably expressing dhfr.atf6 tet xbp1s tetracycline repressor condition 24h 0.01% dmso cells',\n",
       "  'd17 atf6f/xbp1s 39c adult derived epithelial cell line human embryonic kidney (hek) 293t rex clonal stably expressing dhfr.atf6 tet xbp1s tetracycline repressor condition 24h 0.1 ug/ml doxycycline + 1 um trimethoprim cells',\n",
       "  'human'),\n",
       " 'GSE155697-0-vs-7-human up': (['Transcriptomic analysis of monocytes, M0, M1 and M2 macropahges from H syndrome patients and SAIDs.'],\n",
       "  'Biallelic mutations in SLC29A3 cause histiocytosis-lymphadenopathy plus syndrome, also known as H syndrome (HS). HS is a complex disorder, with ~25% of patients developing autoinflammatory complications consisting of unexplained fevers, persistently elevated inflammatory markers and unusual lymphadenopathies, with infiltrating CD68+, S100+ and CD1a– histiocytes, resembling the immunophenotype found in Rosai-Dorfman disease (RDD). We investigated the transcriptomic profiles of monocytes, non-activated (M0), classically-activated (M1) and alternatively-activated macrophages (M2) in two patients with HS, one without autoinflammatory (HS1) and one with autoinflammatory complications (HS2). RNA sequencing revealed a dysregulated transcriptomic profile in both HS patients compared to healthy controls (HC). HS2, when compared to HS1, had several differentially expressed genes, including genes associated with lymphocytic-histiocytic predominance (e.g. NINL) and immunodeficiencies (e.g. B2M). The transcriptomic and cytokine profiles of HS patients were comparable to patients with SAID with high levels of TNF. SERPINA1 gene expression was found to be upregulated in all patients. Moreover, higher levels of IFNg were found in the serum of both HS patients when compared to HC. Gene-ontology (GO) enrichment analysis of the DEGs in HS patients revealed the terms \"type I IFN\", \"IFNg signalling pathway\" and \"immune responses\" as the top 3 most significant terms for monocytes. Finally, the transcriptomic profiles of M0 and M1 macrophages, from patient HS2, were similar to the transcriptomic profile of tissue biopsies taken from patients with RDD-like lymphadenopathies. Monocytes and macrophages from both HS patients showed transcriptomic profiles similar to SAIDs and with a unique dysregulated IFNgsignalling. These findings may help to find better therapeutic options for this rare disorder.',\n",
       "  'm0 macropahges / mdms activation gm csf peripheral blood',\n",
       "  'm1 macropahges / mdms activation gm csf ifng/lps peripheral blood',\n",
       "  'human'),\n",
       " 'GSE115244-1-vs-4-mouse up': (['Gamma entrainment binds higher order brain regions and alleviates neurodegeneration'],\n",
       "  'We report the RNA-Seq data of microglia from CK-p25 mice visual cortex',\n",
       "  'rna seq cd11b+cd45 visual cortex microglia fraction total wild type control mice',\n",
       "  'rna seq cd11b+cd45 ck p25 visual cortex microglia fraction total mice stimulated 40 hz stimulation',\n",
       "  'mouse'),\n",
       " 'GSE123541-2-vs-5-human up': (['Genome-wide RNA-seq and ATAC-seq analysis of an in vivo human immune response to acute Zika virus infection'],\n",
       "  'A longitudinal systems immunologic investigation of acute Zika virus infection in an individual infected with ZIKV while traveling to Caracas, Venezuela during the zika virus epidemic. Includes longitudinal RNA-seq and ATAC-seq characterization of innate and adaptive immune responses during an acute ZIKV infection.',\n",
       "  'myeloiddc subject status individual naat confirmed acute zika virus infection pbmc cell subtype myeloid dendritic time post onset symptoms days',\n",
       "  'plasmacytoiddc subject status individual naat confirmed acute zika virus infection pbmc cell subtype plasmacytoid dendritic time post onset symptoms days',\n",
       "  'human'),\n",
       " 'GSE114191-0-vs-1-human up': (['Repurposing of promoters and enhancers during mammalian evolution'],\n",
       "  'The spatiotemporal control of gene expression exerted by promoters and enhancers is central for organismal development, physiology and behaviour. These two types of regulatory elements have long been distinguished from each other based on their function, but recent work highlighted common architectural and functional features. It also suggested that inheritable alterations in the epigenetic and sequence context of regulatory elements might underlie evolutionary changes of their principal activity, which could result in changes in the transcriptional profile of genes under their control or even facilitate the birth of new genes. Here, based on integrated cross-mammalian analyses of DNase hypersensitivity, chromatin modification and transcriptional data, we provide support for this hypothesis by detecting 445 regulatory elements with signatures of activity turnover in sister species from the primate and rodent lineages (termed \"P/E\" elements). Through the comparison with outgroup species, we defined the directionality of turnover events, which revealed that most instances represent transformations of putative ancestral enhancers into promoters, leading to the emergence of species-specific transcribed loci or 5\\' exons. Notably, P/E elements have distinct GC sequence compositions and stabilizing 5\\' splicing (U1) regulatory motif patterns, which may predispose them to functional repurposing during evolution. Moreover, we trace changes in the U1 and polyadenylation signal densities and distributions that accompanied and likely drove the evolutionary activity switches. Overall, our work highlights functional repurposing as a notable mechanism that likely facilitated regulatory innovation and the origination of new genes and exons during mammalian evolution.',\n",
       "  'human heart adult sex developmenta stage',\n",
       "  'human brain adult sex male developmenta stage',\n",
       "  'human'),\n",
       " 'GSE164963-2-vs-3-mouse dn': (['Co-infection of H9N2 influenza A virus and Escherichia coli in a BALB/c mouse model aggravates lung injury by synergistic effects'],\n",
       "  'Pathogens that cause respiratory diseases in poultry are very complicated, and co-infections with multiple pathogens are prevalent. The H9N2 strain of avian influenza virus (AIV) and Escherichia coli (E. coli) are common poultry pathogens that limit the development of the poultry industry. This study aimed to clarify the interaction between these two pathogens and their pathogenic mechanism using a mouse model. Co-infection with H9N2 AIV and E. coli significantly increased the mortality rate of mice compared to single viral or bacterial infections. It also led to the development of more severe lung lesions compared to single viral or bacterial infections. Co-infection further causes a storm of cytokines, which aggravates the host’s disease by regulating the STAT/SOCS and ERK1/2 pathways. Moreover, co-infection mutually benefited the virus and the bacteria by increasing their multiplication rates. Importantly, nitric oxide synthase 2 (NOS2) expression was also significantly enhanced by the co-infection. It played a key role in the rapid proliferation of E. coli in the presence of the coinfecting H9N2 virus. Therefore, our study underscores the role of NOS2 as a determinant for bacteria growth and illustrates its importance as an additional mechanism that enhances influenza virus-bacteria synergy. It further provides a scientific basis for investigating the synergistic infection mechanism between viruses and bacteria.',\n",
       "  'co infection strain balb/c lung age seven week old',\n",
       "  'mock strain balb/c lung age seven week old',\n",
       "  'mouse'),\n",
       " 'GSE162160,GSE162184-1-vs-3-human dn': (['RFX7-regulated transcriptome in HCT116'],\n",
       "  'Identification of the RFX7-regulated transcriptome in Nutlin-3a and DMSO control treated HCT116 cells',\n",
       "  'hct116 sicontrol dmso rnaseq cell line',\n",
       "  'hct116 sirfx7 nutlin rnaseq 3a cell line',\n",
       "  'human'),\n",
       " 'GSE125933-2-vs-4-human up': (['Compound kushen injection and YFSJ reduce gefitinib resistance in gefitinib resistant lung cancer cell lines'],\n",
       "  'Compound Kushen Injection (CKI) has been used in clinics for nearly 20 years to treat different kinds of cancers. In this study, we found that CKI inhibited gefitinib resistant cell line proliferation, reduced drug resistance and improved anti-cancer effects when associated with gefitinib in a dose-dependent fashion. We also identified candidate CKI associated molecular mechanisms that might reduce drug resistance using transcriptome analysis methods, including gene differential expression analysis and pathway over-representation analysis. Multiple pathways were perturbed and the PI3K/AKT/MTOR and autophagy pathways were identified as the potential primary target pathways of CKI in this context. We also investigated the molecular mechanisms of another TCM, Yifei-Sanjie Wan (YFSJ) on reducing gefitinib resistance using transcriptome analysis.',\n",
       "  '24h 75umgefitinib g75 cell line h1975 tumor type non small lung cancer',\n",
       "  '24h 2.5mg yfsj [y2 5 cell line h1975 tumor type non small lung cancer',\n",
       "  'human'),\n",
       " 'GSE80463,GSE82292-0-vs-1-human dn': (['Divergent expression and metabolic functions of human glucuronosyltransferases through alternative splicing'],\n",
       "  'Purpose: Maintenance of cellular homeostasis and xenobiotics detoxification relies on the glucuronidation pathway mediated by 19 human UDP-glucuronosyltransferase enzymes (UGTs) encoded by 10 highly homologous genes. Recent evidence suggests that alternative splicing largely expands the human UGT transcriptome.',\n",
       "  'liver normal (n=3 donors) sex pooled male female type biopsy',\n",
       "  'intestine colon tumor (n=3 donors) sex pooled male female type tumoral biopsy',\n",
       "  'human'),\n",
       " 'GSE239495-1-vs-0-human dn': (['Canagliflozin mediates tumor suppression alone and in combination with radiotherapy in non-small cell lung cancer (NSCLC) through inhibition of HIF-1a'],\n",
       "  'Non-small cell lung cancer (NSCLC) has a poor prognosis and effective therapeutic strategies are lacking. The diabetes drug canagliflozin inhibits NSCLC cell proliferation and the mammalian target of rapamycin (mTOR) pathway, which mediates cell growth and survival, but it is unclear whether this drug can enhance response rates when combined with cytotoxic therapy. Here, we evaluated the effects of canagliflozin on human NSCLC response to cytotoxic therapy in tissue cultures and xenografts. Ribonucleic acid sequencing (RNA-seq), real-time quantitative PCR (RT-qPCR), metabolic function, small interfering ribonucleic acid (siRNA) knockdown and protein expression assays were used in mechanistic analyses. We found that canagliflozin inhibited proliferation and clonogenic survival of NSCLC cells, and augmented the efficacy of radiotherapy to mediate these effects and inhibit NSCLC xenograft growth. Canagliflozin treatment alone moderately inhibited mitochondrial oxidative phosphorylation and exhibited greater anti-proliferative capacity than specific mitochondrial complex-I inhibitors. The treament downregulated genes mediating hypoxia-inducible factor (HIF)-1a stability, metabolism and survival, activated adenosine monophosphate-activated protein kinase (AMPK) and inhibited mTOR, a critical activator of HIF-1a signaling. HIF-1a knockdown and stabilization experiments suggested that canagliflozin mediates anti-proliferative effects, in part, through suppression of HIF-1a. Transcriptional regulatory network analysis pinpointed histone deacetylase 2 (HDAC2), a gene suppressed by canagliflozin, as a key mediator of canagliflozin’s transcriptional reprogramming. HDAC2 knockdown eliminated HIF-1a levels and enhanced the anti-proliferative effects of canagliflozin. HDAC2-regulated genes suppressed by canagliflozin are associated with poor prognosis in several clinical NSCLC datasets. In addition, we include evidence that canagliflozin also improves NSCLC response to chemotherapy. In summary, canagliflozin may be a promising therapy to develop in combination with cytotoxic therapy in NSCLC.',\n",
       "  'a549 cells canagliflozin cell line nsclc wt 10 mm',\n",
       "  'a549 cells radiotherapy cell line nsclc wt 5 gy + dmso',\n",
       "  'human'),\n",
       " 'GSE63166-0-vs-1-mouse up': (['Small RNA and transcriptome profiling of mice deficient in either germline Dgcr8 or Dicer'],\n",
       "  'We used Illumina Small RNA and RNA-Seq kits to prepare both small RNA and RNA-Seq libraries from total RNA isolated from either leptotenze/zygotene or pachytene spermatocytes purified from either Dgcr8 or Dicer germline conditional knockout mice.',\n",
       "  'pach rna testis strain c57bl/6 x fvb pachytene spermatocytes library prep method illumina truseq seq purified',\n",
       "  'l/z rna testis strain c57bl/6 x fvb leptotene/zygotene spermatocytes library prep method illumina truseq seq purified',\n",
       "  'mouse'),\n",
       " 'GSE49379-0-vs-2-human up': (['Large-scale multi-species survey of metabolome and lipidome'],\n",
       "  'This dataset was generated with the goal of comparative study of gene expression in three brain regions and two non-neural tissues of humans, chimpanzees, macaque monkeys and mice. Using this dataset, we performed studies of gene expression and gene splicing evolution across species and search of tissue-specific gene expression and splicing patterns. We also used the gene expression information of genes encoding metabolic enzymes in this dataset to support a larger comparative study of metabolome evolution in the same set of tissues and species.',\n",
       "  'strain/background c57bl/6 kidney sex age (years)',\n",
       "  'strain/background c57bl/6 muscle sex age (years)',\n",
       "  'human'),\n",
       " 'GSE71713-1-vs-0-mouse dn': (['Acquired tissue-specific promoter bivalency is a basis for PRC2 necessity in adult somatic cells'],\n",
       "  'To define H3K27me3 modified genes in intestinal stem, progenitor and epithelial cells, we performed chromatin immunoprecipitation coupled with high-throughput sequencing (ChIP-Seq).',\n",
       "  'wild type villus rna seq biological rep strain background b6 129 mixed /variation eedf/f cre adult epitheial cells proximal 1/3 intestine control mouse',\n",
       "  'wild type lgr5+ isc rna seq biological rep strain background b6 129 mixed /variation eed+/+ lgr5 egfp ires creert2 adult intestinal stem cell proximal 1/3 intestine control mouse cells',\n",
       "  'mouse'),\n",
       " 'GSE54729-0-vs-1-human dn': (['Transcriptome profiling of a mouse model of alveolar soft part sarcoma'],\n",
       "  'In order to dtermine how well a mouse genetic model of alveolar soft part sarcoma (ASPS) mimics the human disease, five human ASPS tumor samples and three normal skeletal muscle samples were profiled by RNAseq and compared to samples from five mouse tumors induced by expression of ASPSCR1-TFE3 and three normal mouse skeletal muscle samples, also profiled by RNAseq.',\n",
       "  'human skeletal muscle control group banked quadriceps sample',\n",
       "  'human alveolar soft part sarcoma group tumor banked sample',\n",
       "  'human'),\n",
       " 'GSE161916,GSE161917-2-vs-6-human up': (['Interferon-regulated genetic programs and JAK/STAT pathway activate the intronic promoter of the short ACE2 isoform in renal proximal tubules [RNA-seq]'],\n",
       "  'Angiotensin-Converting Enzyme 2 (ACE2) binds SARS-CoV-2 spike protein and facilitates viral entry into the cell. ACE2 is expressed in the kidney and renal injury strongly impacts COVID-19 prognosis. In addition, a short, interferon-inducible isoform of ACE2, dACE2 was recently identified. Change in ACE2 expression has already been linked to several human nephropathies. However, these changes were not analyzed in context of dACE2 and regulation of ACE2 expression in the kidney remains unclear. Human primary Renal Proximal Tubule cells were used for all experiments. ChIP-seq analysis of histone modifications (H3K27ac, H3K4me3, K3K4me1), RNA polymerase loading and DNAse hypersensitive sites (DHS) were used to identify gene regulatory elements in the ACE2 locus. qRT-PCR was used to assess mRNA expression of ACE2, dACE2, TMPRSS2 and STAT1. RNA-seq was used to identify changes in global gene expression after cytokine treatment. Putative regulatory elements controlling dACE2 expression were identified using ChIP-seq profiles and RNA-seq mapping. qRT-PCR differentiating between ACE2 and dACE2 revealed that only dACE2 is interferon inducible. dACE2 mRNA was upregulated 300- and 600-fold by IFNα and IFNβ, respectively, while full length ACE2 was unchanged. Interferon stimulation also resulted in elevated STAT1 expression. JAK inhibitor ruxolitinib ablated both STAT1 and dACE2 expression after interferon treatment. RNA-seq analysis revealed approximately 1200 genes induced by IFNβ, largely innate immune response-related. We showed that human primary proximal tubules express ACE2 and its interferon-inducible short isoform, dACE2, from an intronic promoter. We also deliver new datasets identifying cytokine response genes in proximal tubule cells.',\n",
       "  'rna seq hppt untreated primary proximal tubule cells human',\n",
       "  'rna seq hppt ifnb human primary proximal tubule cells',\n",
       "  'human'),\n",
       " 'GSE243360,GSE243361-1-vs-3-human dn': (['RNA-seq of human head and neck squamous cell carcinoma (HNSCC) to compare different tissue dissociation methods [bulkRNA]'],\n",
       "  'Tissue dissociation, mechanical and/or enzymatic, is a method of disaggregating 3D connective tissue to release cells for downstream single cell analysis.  Without proper controls for tissue dissociation experiments, potential artifacts can be mistaken for baseline tissue data. We hypothesized that enzymatic (collagenase/hyaluronidase) compared to mechanical dissociation leads to dissociated related artifacts not found in endogenous tumor tissue. To distinguish artifacts between enzymatic and mechanical dissociation, we compared separate enzymatic and mechanical dissociated samples with baseline samples from healthy and tumor patient whole blood, melanoma and head and neck squamous cell carcinoma (HNSCC) tumor tissues in flow cytometry, RNA sequencing and CODEX experiments. In bulk sorted tumor tissue, there is a significant difference between enzymatic and mechanical digested samples in cell stress, heat shock proteins and cell death related genes. These genes were also significantly upregulated in enzymatic over mechanical samples in the single cell RNA sequencing HNSCC tissue data. Fibroblasts and CD39+CD103+ CD8 T cells from tumor tissue, appear to require enzymatic digestion for efficient release from tissue compared to the mechanical digested samples. CD39+CD103+ CD8 tumor infiltrating lymphocytes (TILs) show different phenotypes depending on dissociation technique and tumor tissue. In tumor tissue, CD3+CD8+CD69+ T cells and gene expression were elevated in the enzymatic samples over the mechanical and baseline frozen tissue. Tissue dissociation techniques and tissue analysis from downstream applications have the potential for dissociation related artifacts that can mislead both protein and RNA sequencing data analysis in cancer research.',\n",
       "  'l head neck squamous cell carcinoma',\n",
       "  'head neck squamous cell carcinoma',\n",
       "  'human'),\n",
       " 'GSE135833-6-vs-7-human up': (['JC Polyomavirus Infection of Primary Human Renal Epithelial Cells Is Controlled by a Type I IFN-Induced Response'],\n",
       "  'JC polyomavirus (JCPyV) established a persistent infection, but BK polyomavirus (BKPyV) killed the cells in 15 days. To identify the cellular factors responsible for controlling JCPyV infection and promoting viral persistence, we profiled the transcriptomes of JCPyV- and BKPyV-infected cells at several time points postinfection. We found that  interferon-stimulated genes (ISGs) were only activated in the JCPyV and not in the BKPyV-infected cells.',\n",
       "  'uninfected control cells 9 days post infection replicate primary human renal epithelial (atcc) cell source 13 months old female none (uninfected)',\n",
       "  'jc infected cells 3 days post infection replicate primary human renal epithelial (atcc) cell source 13 months old female polyomavirus (jcpyv)',\n",
       "  'human'),\n",
       " 'GSE75246-3-vs-4-mouse dn': (['LPS and brain inflammation'],\n",
       "  'Normal mice were injected i.p. with LPS or saline. 24h later, perfused brains were dissociated. Cells were fixed, immunolabeled and FACS sorted. RNA was extracted from neuron, astrocyte, and microglial cell populations. Typical RIN=4-5 for neurons, 6-8 for astrocytes, and 5-7 for microglia. Typical RNA yields ~100ng for neurons, ~20ng for microglia, and ~10ng for astrocytes.',\n",
       "  'sample neuron lps brain rna sorted cells',\n",
       "  'sample microglia lps brain rna sorted cells',\n",
       "  'mouse'),\n",
       " 'GSE154428-2-vs-5-mouse up': (['Different effects of constitutive and induced microbiota modulation on microglia in a mouse model of Alzheimer’s disease'],\n",
       "  'It was recently revealed that gut microbiota promote amyloid-beta (Aβ) burden in mouse models of Alzheimer’s disease (AD). However, the underlying mechanisms when using either germ-free (GF) housing conditions or treatments with antibiotics (ABX) remained unknown. In this study, we show that GF and ABX-treated 5x familial AD (5xFAD) mice developed attenuated hippocampal Aβ pathology and associated neuronal loss, and thereby delayed disease-related memory deficits. While Ab production remained unaffected in both GF and ABX-treated 5xFAD mice, we noticed in GF 5xFAD mice enhanced microglial Aβ uptake at early stages of the disease compared to ABX-treated 5xFAD mice. Furthermore, RNA-sequencing of hippocampal microglia from SPF, GF and ABX-treated 5xFAD mice revealed distinct microbiota-dependent gene expression profiles associated with phagocytosis and altered microglial activation states. Taken together, we observed that constitutive or induced microbiota modulation in 5xFAD mice differentially controls microglial Aβ clearance mechanisms preventing neurodegeneration and cognitive deficits.',\n",
       "  'wt abx housing microglia age 16 weeks brain',\n",
       "  'fad spf housing microglia age 16 weeks brain',\n",
       "  'mouse'),\n",
       " 'GSE175495-8-vs-6-human up': (['Divergent immunometabolic changes in adipose tissue and skeletal muscle with ageing in healthy humans'],\n",
       "  \"RNA-sequencing from 12 young (20-35 years old) and 12 older (60-85 years old) participants' skeletal muscle and adipose tissue. Biopsies were taken at rest following an overnight fast, and following alcohol/caffeine restriction for 24 hours, and no strenuous physical activity for 48 hours. Skeletal muscle was taken by the Bergstrom technique under local anaesthesia (Lidocaine hydrochloride) from the dominant leg Vastus Lateralis, and adipose tissue was taken by needle aspiration 5cm lateral to the umbilicus.\",\n",
       "  'young adiposetissue age subcutaneous abdominal adipose sample type untreated rested healthy human whole (adipose )',\n",
       "  'old skeletalmuscle age vastus lateralis skeletal muscle dominant leg sample type untreated rested healthy human whole (skeletal muscle)',\n",
       "  'human'),\n",
       " 'GSE182754,GSE183011-4-vs-1-human up': (['Transcriptional effects of HSV-1-ICP0 and Ad5-E4ORF3 expression in BLaER1 monocytes [Virulence factors]'],\n",
       "  'We have identified that the viral virulence factors ICP0 from HSV-1 and E4ORF3 from Adenovirus5 can activate the MORC3 pathway of innate immune sensing. To study the global transcriptional effects of virulence factor sensing, we overexpressed mCherry or these virulence factors from a doxycycline-inducible lentiviral construct in BLaER1 monocytes of various genotypes and performed RNA-seq.',\n",
       "  'wt mcherry cell line blaer1',\n",
       "  'wt e4orf3 cell line blaer1',\n",
       "  'human'),\n",
       " 'GSE222181,GSE234342-2-vs-0-human dn': (['CRISPR/dCas9 DNA methylation editing is heritable during human haematopoiesis and can impair differentiation in vivo.'],\n",
       "  'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'granulocyte control [g wt humanised murine bone marrow non targeting guide rna',\n",
       "  'monocyte p15 methylated [ humanised murine bone marrow guide rna',\n",
       "  'human'),\n",
       " 'GSE139292-2-vs-7-human up': (['Comparison of different shear stress conditions on gene proflile modulation in HCASMC and HCAEC, in the presence or absence of BVS+Everolimus'],\n",
       "  'We have analyzed the global gene expression modulation induced by mechanical stress (shear stress) on vascular wall cells, seeded in the Ibidi system, with and without BVS+Drug. By the comparison of the different flow conditions we obtained a small panel of genes, mainly involved  in inflammation, angiogenesis, leukocyte migration, smooth muscle cell growth, thrombosis (BMP4, SELE, HMOX1, MEOX2, SEMA3E): all these genes will be integrated in an in-silico model for the evaluation of restenosis and thrombosis development after BVS deployment.',\n",
       "  'hcasmc dyne/cm2 human coronary artery smooth muscle cells passages 2 6 bvs+drug presence n primary',\n",
       "  'hcaec dyne/cm2 bvs+drug human coronary artery endothelial cells passages 2 6 presence primary',\n",
       "  'human'),\n",
       " 'GSE211190-2-vs-10-human dn': (['Vascular Surgery Patients with Elevated Neutrophil Lymphocyte Ratios Have Downregulated Neutrophil Complement RNA Expression'],\n",
       "  'Elevated NLR (neutrophil-lymphocyte ratio) in elective vascular surgery (EVS) patients is associated with increased mortality independent of perioperative surgical outcome. To understand why high NLR is associated with higher mortality, we investigated neutrophil and lymphocyte transcriptome expression in patients undergoing EVS.',\n",
       "  'neutrophil nlr low [nlr n blood nl ratio',\n",
       "  'lymphocyte nlr low [nlr l blood nl ratio',\n",
       "  'human'),\n",
       " 'GSE115168-3-vs-4-human up': (['ER proteostasis and temperature differentially impact the mutational tolerance of influenza hemagglutinin'],\n",
       "  'Here, we present a systematic and quantitative test of the hypothesis that the composition and activities of the endoplasmic reticulum (ER) proteostasis network impact mutational tolerance of secretory pathway client proteins. We focus on influenza hemagluttinin (HA), a viral coat protein that folds in the host’s ER via a complex but well-characterized pathway. By integrating chemical methods to modulate the unfolded protein response with deep mutational scanning to assess mutational tolerance, we discover that upregulation of ER chaperones broadly enhances HA mutational tolerance across numerous sites and secondary/tertiary structure elements, including sites targeted by host antibodies. Remarkably, this host chaperone-enhanced mutational tolerance is observed at the same HA sites where mutational tolerance is most reduced by propagation at a fever-like temperature. Thus, host ER proteostasis mechanisms and temperature modulate HA mutational tolerance in opposite directions. This finding has important implications for influenza evolution, because influenza immune escape is contingent on HA possessing sufficient mutational tolerance to acquire antibody resistance while still maintaining the capacity to fold and function. More broadly, this work provides the first experimental evidence that the composition and activities of the ER proteostasis network critically define the mutational tolerance and, therefore, the evolution of secretory pathway client proteins.',\n",
       "  'd17 basal 37c adult derived epithelial cell line human embryonic kidney (hek) 293t rex clonal stably expressing dhfr.atf6 tet xbp1s tetracycline repressor condition 24h 0.01% dmso cells',\n",
       "  'd17 xbp1s 37c adult derived epithelial cell line human embryonic kidney (hek) 293t rex clonal stably expressing dhfr.atf6 tet tetracycline repressor condition 24h 0.1 ug/ml doxycycline cells',\n",
       "  'human'),\n",
       " 'GSE127530-6-vs-10-human up': (['NCBI GEO Submission of human whole blood transcriptomes in response to a high-fat meal'],\n",
       "  'Modern humans spend most of their time having eaten recently. The purpose of the current project is to understand how the blood, which contains immune cells, responds in the hours after eating a meal that is moderately high in fat. We used a sequencing method to observe the expression of all the genes in blood cells in five participants who were each fed a high fat meal on three separate days. The results are reported in the manuscript, “Temporal changes in postprandial blood transcriptomes reveal subject-specific pattern of expression of innate immunity genes after a high-fat meal.\"',\n",
       "  's2 fast study subject id # 2 sex male test day time blood draw fasting coded/blinded arm unblinded whole',\n",
       "  's17 d1 study subject id # 17 sex female test day 1 time blood draw coded/blinded arm unblinded placebo sunfloweroil whole',\n",
       "  'human'),\n",
       " 'GSE103001-2-vs-7-human dn': (['Stranded RNASeq of human mammary primary tumors ER+ and paired adjacent healthy tissues'],\n",
       "  'Non-coding RNAs (ncRNA) represent at least 1/5 of the mammalian transcript amount, and about 90% of the genome length is actively transcribed. Many ncRNAs have been demonstrated to play a role in cancer. Among them, natural antisense transcripts (NAT) are RNA sequences which are complementary and overlapping to those of protein-coding transcripts (PCT). NATs were punctually described as regulating gene expression, and are expected to act more frequently in cis than other ncRNAs that commonly function in trans. In this work, 22 breast cancers expressing estrogen receptors and their paired healthy tissues were analyzed by strand-specific RNA sequencing to highlight the potential role of NATs in gene regulations occurring in breast cancer.',\n",
       "  'pat normal nature tumor receptors / bloom er subtype tnm mammary',\n",
       "  'pat tumor nature primary invasive breast carcinoma receptors er+/her2 bloom er subtype tnm mammary',\n",
       "  'human'),\n",
       " 'GSE237229-3-vs-5-human dn': (['Transcriptional profiling of primary human aortic smooth muscle cells (hAoSMC) isolated from non-anerysmal vs. hAoSMCs isolated from anerysmal abdominal aorta (differential expression analysis) [bulk RNA-seq]'],\n",
       "  'An abdominal aortic aneurysm (AAA) is a pathological widening of the aortic wall characterized by loss of AoSMCs, extracellular matrix degradation and local inflammation. This condition is often asymptomatic until rupture occurs, leading to high morbidity and mortality rates. We aimed at characterizing the transcriptional landscape AoSMCs  and to highlight aortic disease-driven changes in gene expression, in terms of contractility, synthetic, inflammation, proliferation and apoptosis markers.',\n",
       "  'aaa smcs v1 (techn rep abdominal aorta cell line primary cells smooth muscle subject id status',\n",
       "  'aaa smcs 86 (techn rep abdominal aorta cell line primary cells smooth muscle subject id status',\n",
       "  'human'),\n",
       " 'GSE211386-3-vs-1-mouse up': (['The transcriptome alteration in tumoral NK cells in response to  dual or monotherapy of STING agonist and IL-35 blocking antibodies'],\n",
       "  'To investigate the role of STING agonist and IL-35 blocking antibodies in the regulation of NK cell function, tumor-infiltrated NK cells were flow cytometrically separated from control (IgG), cGAMP, anti-IL-35, or combination treated groups',\n",
       "  'ctrl1 cell line nk cells wt igg time day 21',\n",
       "  'aeb3 cell line nk cells wt il 35 blocking antibodies time day 21',\n",
       "  'mouse'),\n",
       " 'GSE162160,GSE162184-1-vs-0-human dn': (['RFX7-regulated transcriptome in HCT116'],\n",
       "  'Identification of the RFX7-regulated transcriptome in Nutlin-3a and DMSO control treated HCT116 cells',\n",
       "  'hct116 sicontrol dmso rnaseq cell line',\n",
       "  'hct116 sicontrol nutlin rnaseq 3a cell line',\n",
       "  'human'),\n",
       " 'GSE174539-2-vs-8-human up': (['Emergent mechanical control of vascular morphogenesis'],\n",
       "  'mRNA profiles of endothelial cells and fibroblasts undergoing vasculogenesis under juxtacrine co-culture vs. monoculture.',\n",
       "  'culture fb monoculture time day 1 human normal lung fibroblasts',\n",
       "  'culture huvec fb coculture time day human umbilical vein endothelial cells',\n",
       "  'human'),\n",
       " 'GSE77661-5-vs-7-human up': (['RNA Sequencing Facilitates Quantitative Analysis of Transcriptomes in Human Normal and Cancerous Tissues'],\n",
       "  'Circular RNAs (circRNAs) represent a novel class of widespread and diverse endogenous RNAs that may regulate gene expression in eukaryotes. However, the regulation and function of human circRNAs remain largely unknown. Here we generate ribosomal-depleted RNA sequencing data from six normal tissues and seven cancers, and detect at least 27,000 circRNA candidates.',\n",
       "  'cancer',\n",
       "  '',\n",
       "  'human'),\n",
       " 'GSE178388-1-vs-0-human up': (['Downregulation of exhausted cytotoxic T cells in gene expression networks of multisystem inflammatory syndrome in children'],\n",
       "  'An unbiased, transcriptome-wide survey implicates downregulation of NK cells and cytotoxic T cell exhaustion in the pathogenesis of MIS-C.',\n",
       "  'subject id diagnosis healthy blood age range (years) gender female ethnicity comorbidities none immnosuppressive timepoint (t0 = #>0= sample taken ) t1',\n",
       "  'c3 subject id diagnosis covid blood age range (years) 0 5 gender male ethnicity hispanic comorbidities autoimmune anemia immnosuppressive steroids timepoint (t0 = #>0= sample taken )',\n",
       "  'human'),\n",
       " 'GSE99081-5-vs-1-human up': (['External RNA Control Consortium (ERCC) spikes in RNA sequencing (RNA-Seq) data normalization: transcriptome analysis of yellow fever virus-infected cells'],\n",
       "  'We explored the ability of External RNA Control Consortium (ERCC) standards to detect technical biases between batches of sequencing experiments and to set detection thresholds. We compared three normalization methods (FPKM, DESeq2 and RUV) to identify the optimal analysis approach.',\n",
       "  'hep yfv asibi hepatocytes model monocellular infection wild type time 24h post',\n",
       "  'hep/kup yfv 17d hepatocytes + kupffer cells model pluricellular infection vaccine time 24h post',\n",
       "  'human'),\n",
       " 'GSE98527-4-vs-10-mouse up': (['Mouse lung transcriptome response to infection with H1N1, H3N2, and H5N1 influenza virus.'],\n",
       "  'The cellular transcriptome of C57BL/6 mouse lungs was profiled by mRNA-Seq analysis at multiple time points in response to infection with influenza A/California/04/09 (H1N1), A/Wyoming/03/03 (H3N2), and A/Vietnam/1203/04 (H5N1) HALo virus. The Influenza A/Vietnam/1203/04 (H5N1) HALo mutant virus is an attenuated H5N1 virus generated from wild-type Influenza A/Vietnam/1203/04 (H5N1) virus as described in Steel, J., et al. J Virol. 2009 Feb; 83(4):1742-53. A/Wyoming/03/03 (H3N2) influenza virus replicates poorly in mice and lung tissue collected from mice inoculated with this virus did not carry viral loads detectable by plaque assay.',\n",
       "  'c57bl/6 h3n2 pfu1e4 02d lung strain infection /wyoming/03/03 (h3n2) biological replicate mouse',\n",
       "  'c57bl/6 h1n1 pfu1e4 04d lung strain infection /california/04/09 (h1n1) biological replicate mouse',\n",
       "  'mouse'),\n",
       " 'GSE68220-4-vs-3-mouse up': (['Genome-wide maps of CD14-dependent and CD14-independent bladder innate immune responses to UPEC'],\n",
       "  'We report a transcriptional response in the mouse bladder that encompasses multiple innate immune networks not previously associated with CD14. Major CD14-dependent pathways included immune cell trafficking, differential cytokine production in macrophages, and IL-17 signaling',\n",
       "  'cd14 pbs /variation deficient b6.129s cd14tm1frm/j infected (control) whole bladder mice control',\n",
       "  'wt upec strain c57bl/6 /variation wild type infected cft073 whole bladder mice',\n",
       "  'mouse'),\n",
       " 'GSE85520-5-vs-0-mouse up': (['IRF8 transcription factor controls survival and function of terminally differentiated conventional and plasmacytoid dendritic cells respectively'],\n",
       "  'Interferon regulatory factor-8 (IRF8) has been proposed to be essential for development of monocytes, plasmacytoid dendritic cells (pDCs) and type 1 conventional dendritic cells (cDC1s) and remains highly expressed in differentiated DCs. Transcription factors that are required to maintain the identity of terminally differentiated cells are designated “terminal selectors”. Using BM chimeras, conditional Irf8fl/fl mice and various promotors to target Cre recombinase to different stages of monocyte and DC development, we have identified IRF8 as a terminal selector of the cDC1 lineage controlling survival. In monocytes, IRF8 was necessary during early but not late development. Complete or late deletion of IRF8 had no effect on pDC development or survival but altered their phenotype and gene expression profile leading to increased T cell stimulatory function but decreased type 1 interferon production. Thus, IRF8 differentially controls the survival and function of terminally differentiated monocytes, cDC1s and pDCs.',\n",
       "  'cdc2 wt strain c57bl/6 spleen state irf8fl/fl steady cd172a+ conventional dendritic cells (cdc2s) cdc2s mice',\n",
       "  'pdc wt strain c57bl/6 spleen state irf8fl/fl steady plasmacytoid dendritic cells pdcs mice',\n",
       "  'mouse'),\n",
       " 'GSE163568-2-vs-5-human up': (['Notch signaling rapidly regulates the expression of the small GTPase RND1 and a diverse endothelial transcriptome'],\n",
       "  'Endothelial Notch signaling regulates transcription of many downstream effectors controlling sprouting, migration, proliferation, barrier formation, and other phenotypes. However, endothelial-relevant Notch targets are largely uncharacterized. Few are studied in depth, and many transient, early response Notch targets are yet to be described. We therefore determined the Notch early response transcriptional profile in several experimentally relevant in vivo and in vitro contexts: ligand-specific or EGTA-induced Notch activation in different primary endothelial cells, and gamma secretase-mediated Notch inhibition in neonatal brain endothelium in a RiboTag mouse model. The intersection of these profiles reveals a core set of early Notch-responsive targets.  Among these are effectors in GPCR signaling, not previously implicated downstream of Notch signaling. We demonstrate the Rho GTPase RND1 is a direct Notch transcriptional target and required for Notch control of sprouting angiogenesis, endothelial migration and Ras activity.',\n",
       "  'hrec tla dll4 primary cells (human microvascular retinal endothelial cells) human (hrec)',\n",
       "  'huvec tla dll4 primary cells (human umbilical vein endothelial cells) human (huvec)',\n",
       "  'human'),\n",
       " 'GSE130534-1-vs-8-human up': (['Identification of drivers and therapeutic vulnerabilities in anti-androgen resistant prostate cancer cells'],\n",
       "  'Inhibition of the androgen receptor (AR) by anti-androgens is the standard treatment for castration resistant prostate cancer (CRPC), but it inevitably leads to the development of resistance. We generated two multi anti-androgen resistant cell-line models by treatment of LNCaP cells with enzalutamide (ResA) and RD-162 (ResB). Both cell-lines have an AR independent, non-neuroendocrine phenotype. To identify the resistance mechanisms we performed RNASeq analysis. To this end, the three cell-lines were plated in medium with 5% dextran-coated charcoal (DCC) treated FCS at a density of 1 million cells/well in 6-well plates. After 24 hours of steroid starvation the cells were treated with 10 nM of the androgen dihydrotestosterone (DHT), 10 µM of the anti-androgen enzalutamide, a combination of the two, or vehicle (DMSO) for 18 hours.',\n",
       "  'lncap dmso b1 cell line batch1',\n",
       "  'lncap 10 nm dht b1 cell line batch1',\n",
       "  'human'),\n",
       " 'GSE118279-11-vs-5-human dn': (['Unr is a potential post-transcriptional master regulator of protein expression'],\n",
       "  'Ribonucleoprotein immunoprecipitation followed by mass spectrometry and RNA-Seq was used to investigate the ‘Upstream of N-RAS’ (UNR) protein interactome across three cancer cell lines. A number of novel UNR-interacting proteins and transcripts were discovered, including the ubiquitin E3 ligase, HUWE1; the nucleolar protein, NARR; the multifunctional protein, SQSTM1; and the erythropoiesis-related protein, LDB1\\xa0. GO-term analysis of putative UNR interactors suggested that UNR interacts with both proteins and mRNAs encoding proteins that are involved in RNA binding',\n",
       "  'saos igg cell line 2 disease state osteosarcoma stress control rip antibody (pulldown) (invitrogen 10500c) replicate',\n",
       "  'saos+ unr cell line saos 2 disease state osteosarcoma stress oxidative (0.5mm sodium arsenite) rip antibody (pulldown) (polyclonal rabbit novus biologicals nbp1 71914) replicate',\n",
       "  'human'),\n",
       " 'GSE153129-0-vs-1-human dn': (['SPT6 Promotes Epidermal Differentiation and Blockade of an Intestinal Phenotype through Control of Transcriptional Elongation'],\n",
       "  'In adult tissue, stem and progenitor cells must tightly regulate the balance between proliferation and differentiation to sustain homeostasis. It is assumed that the rate-limiting step during the induction of somatic tissue differentiation is the recruitment of RNA polymerase II (Pol II) to promoters. Here we show that ~30% of induced differentiation genes already contain stalled Pol II at the promoters in epidermal stem and progenitor cells which is then released into productive transcription elongation upon differentiation. Central to this process is the histone chaperone, SPT6 which is necessary for the elongation of these differentiation genes. Upon SPT6 depletion there is a loss of human skin differentiation and stratification. Unexpectedly, loss of SPT6 also caused the spontaneous transdifferentiation of epidermal cells into an intestinal phenotype due to the stalled transcription of the master regulator of epidermal fate P63. Our findings suggest that control of transcription elongation through SPT6 plays a prominent role in adult somatic tissue differentiation and the inhibition of alternative cell fate choices.',\n",
       "  'ctli rnaseq keratinocyte transfect spt6i undergo organotypic culture 6 days human epidermal',\n",
       "  'spt6i rnaseq keratinocyte transfect undergo organotypic culture 6 days human epidermal',\n",
       "  'human'),\n",
       " 'GSE152714-5-vs-6-human dn': (['Seminal Plasma-Derived Extracellular-Vesicle Fractions from HIV-Infected Men Exhibit Unique MicroRNA Signatures and Induce a Proinflammatory Response in Cells Isolated from the Female Reproductive Tract'],\n",
       "  'The continuing spread of HIV/AIDS is predominantly fueled by sexual exposure to HIV-contaminated semen. Seminal plasma (SP), the liquid portion of semen, harbors a variety of factors that may favor HIV transmission by facilitating viral entry into host cells, eliciting the production of pro-inflammatory cytokines, and enhancing the translocation of HIV across the genital epithelium. One important and abundant class of factors in SP is extracellular vesicles (EVs), which in general are important intercellular signal transducers. Although numerous studies have characterized blood plasma-derived EVs from both uninfected and HIV-infected individuals, little is known about the properties of EVs from semen of HIV-infected individuals. We report here that fractionated SP enriched for EVs from HIV-infected men induce potent transcriptional responses in epithelial and stromal cells that interface with luminal contents of the female reproductive tract. Compared to semen EV fractions from uninfected individuals, those from acutely-infected individuals induced a more pro-inflammatory signature. This was not associated with any observable differences in the surface phenotypes of the vesicles. However, miRNA expression profiling analysis revealed that EV fractions from infected individuals exhibit a broader and more diverse profile. Taken together, our data suggest that SP EVs from HIV-infected individuals exhibit unique miRNA signatures and exert potent pro-inflammatory transcriptional changes in cells of the female reproductive tract, which may facilitate HIV transmission.',\n",
       "  'eec sp acut cell tyoe endometrial epithelial cells seminal plasma acutely treated hiv infected individuals endometrium',\n",
       "  'esf mock (seminal plasma derived extracellular vesicle control) cell tyoe endometrial stromal fibroblasts treated endometrium',\n",
       "  'human'),\n",
       " 'GSE211802-2-vs-3-human dn': (['Effect of depletion of MDMX (aka MDM4) on gene expression in uveal melanoma cell line MEL202'],\n",
       "  'To investigate the effect of MDMX depletion on the transcriptome of MEL202 cells we established 5 derivatives either containing a doxycyline- inducible Contrl shRNA or with 4 distinct MDMX -targeting shRNAs',\n",
       "  'mel202/ shctrl doxycycline biol rep cell line uveal melanoma ctrl knockdown vehicle 48 hrs',\n",
       "  'mel202/ shmdmx #1 + doxycycline biol rep cell line shmdmx#1 uveal melanoma mdmx knockdown 48 hrs',\n",
       "  'human'),\n",
       " 'GSE125933-7-vs-2-human dn': (['Compound kushen injection and YFSJ reduce gefitinib resistance in gefitinib resistant lung cancer cell lines'],\n",
       "  'Compound Kushen Injection (CKI) has been used in clinics for nearly 20 years to treat different kinds of cancers. In this study, we found that CKI inhibited gefitinib resistant cell line proliferation, reduced drug resistance and improved anti-cancer effects when associated with gefitinib in a dose-dependent fashion. We also identified candidate CKI associated molecular mechanisms that might reduce drug resistance using transcriptome analysis methods, including gene differential expression analysis and pathway over-representation analysis. Multiple pathways were perturbed and the PI3K/AKT/MTOR and autophagy pathways were identified as the potential primary target pathways of CKI in this context. We also investigated the molecular mechanisms of another TCM, Yifei-Sanjie Wan (YFSJ) on reducing gefitinib resistance using transcriptome analysis.',\n",
       "  '24h untreated [ut24 cell line h1975 tumor type non small lung cancer',\n",
       "  '24h 75umgefitinib g75 cell line h1975 tumor type non small lung cancer',\n",
       "  'human'),\n",
       " 'GSE129083-0-vs-4-mouse up': (['Age effects on CL-induced beige fat remodeling'],\n",
       "  'Analysis of iWAT tissues from young (2 month) and aged (12 month) C57bl/6 male mice with vehicle or CL treatments. CL-induced tissue remodeling is severely impaired in aged mice. Results provide insight into molecular mechanisms underlying age-related changes in iWAT.',\n",
       "  'iwat young ctrl strain c57bl/6 age 2 months old vehicle control inguinal adipose',\n",
       "  'iwat young cl strain c57bl/6 age 2 months old cardiolipin (cl) inguinal adipose',\n",
       "  'mouse'),\n",
       " 'GSE196049-1-vs-0-human up': (['RNA sequencing of human monocytes stimulated with murine platelets from wild-type or Clec1b-/- mice'],\n",
       "  'C-type lectin receptors (CLRs) recognize glycans present on the surface of pathogens or self and mediate host defense as well as homeostasis. Dectin-1 is the most well-characterized CLR which recognizes β-glucan on various pathogens and expressed on human myeloid cells. Recently, we screened various tissue-derived primary cells for Dectin-1 binding and found that human Dectin-1 selectively binds to platelets. By establishing anti-platelet monoclonal antibodies that inhibit human Dectin-1 binding, we identified that CLEC-2 is responsible for the interaction with human Dectin-1. CLEC-2 is critical for lymphangiogenesis in early embryonic stage. In this study, we evaluated the influence on CLEC-2 binding in human monocytes which highly expresses human Dectin-1 by RNA sequencing.',\n",
       "  'hmono wt plts human monocytes treated murine platelets wild type mice',\n",
       "  'hmono non stim human monocytes untreated',\n",
       "  'human'),\n",
       " 'GSE225713,GSE225717-4-vs-0-mouse dn': (['Single-cell RNA-seq for identification of B-cell specific checkpoint molecules regulating anti-tumor immunity'],\n",
       "  'The role of B cells in anti-tumor immunity is still debated and accordingly, most therapies have focused on targeting T and NK cells to inhibit tumor growth1,2. Here, using high-throughput flow cytometry, bulk and single-cell RNA- and BCR-sequencing of B cells temporally during B16F10 melanoma growth, we identified a subset of B cells that expands specifically in the draining lymph node over time in tumor bearing-mice. The expanding B cell subset expresses the cell surface molecule T cell immunoglobulin and mucin domain 1 (TIM-1) and a unique transcriptional signature, including multiple co-inhibitory molecules such as PD-1, TIM-3, TIGIT and LAG-3. While conditional deletion of these co-inhibitory molecules on B cells had little or no effect on tumor burden, selective deletion of Havcr1 (the gene encoding TIM-1) in B cells both dramatically inhibited tumor growth and enhanced effector T cell responses. Loss of TIM-1 enhanced the type 1 interferon response in B cells, which augmented B cell activation and increased antigen presentation and co-stimulation, resulting in increased expansion of tumor-specific effector T cells. Our results demonstrate that manipulation of TIM-1-expressing B cells enables engagement of the second arm of adaptive immunity to promote anti-tumor immunity and inhibit tumor growth.',\n",
       "  'bko bcr tumor lymph nodes immune cells wt tim 1bko hased time n/',\n",
       "  'timecourse tcr tumor lymph nodes immune cells wt time 3 points hashed',\n",
       "  'mouse'),\n",
       " 'GSE165518-1-vs-0-human up': (['A longitudinal sampling study of transcriptomic and epigenetic profiles in patients with thrombocytopenia syndrome'],\n",
       "  'Severe fever with thrombocytopenia syndrome (SFTS) is a new tick-borne infectious disease caused by a new SFTS virus (SFTSV). Due to its 12%-50% high fatality rate and the possibility of pandemic transmission, as well as no specific antiviral drugs for the treatment so far, SFTSV has been listed as one of the top 10 priority infectious diseases by the World Health Organization. Currently, there are rare studies of transcriptomic analysis for the patients infected with SFTSV, which makes it difficult to deeply understand the life cyle and pathogenicity of this virus. To explore the differences of transcripts after SFTSV infection, we performed a longitudinal sampling study to systematically investigate the chronological changes of viral load and transcriptomic and epigenetic characterization using white blood cells from SFTSV patients. The results showed significant changes in the expression of some genes from onset to recovery of SFTSV infection. Moreover, these differentially expressed genes showed good consistency in the three patients at different stages of treatment, which may contribute to the pathophysiology of SFTSV and thus lead to a breakthrough in SFTSV therapy. Moreover, by m6A-seq, we found some genes that might be regulated by m6A. This study in transcript changes and RNA modification may open a brand new direction to our understanding of SFTSV and play an important role in the drugs discovery for effective treatment.',\n",
       "  'white blood cell disease state healthy donors',\n",
       "  'white blood cell disease state sftsv infection',\n",
       "  'human'),\n",
       " 'GSE129244-1-vs-5-mouse up': (['Ldb1 is required for Lmo2 oncogene-induced thymocyte self-renewal and T-cell Acute Lymphoblastic Leukemia'],\n",
       "  'Prolonged or enhanced expression of the proto-oncogene Lmo2 is associated with a severe form of T-cell Acute Lymphoblastic Leukemia (T-ALL), designated Early T-progenitor ALL (ETP-ALL), that is characterized by the aberrant self-renewal and subsequent oncogenic transformation of immature thymocytes. Recent data suggest that Lmo2 may exert these effects by functioning as component of a multi-subunit transcription complex that includes the ubiquitous adapter Ldb1 along with b-HLH and/or GATA family transcription factors. In this study, we investigated the importance of Ldb1 for Lmo2-induced T-ALL by conditional deletion of Ldb1 in thymocytes in a Lmo2 transgenic mouse model of T-ALL. Our results identify a critical requirement for Ldb1 in the induction of thymocyte self-renewal, thymocyte radio-resistance and transition to T-ALL in Lmo2 transgenic mice. Ldb1 was also required for acquisition of the pre-leukemic ETP gene expression signature in immature Lmo2 transgenic thymocytes. Together, these results support a model where Lmo2-induced T-ALL results from failure to down-regulate Ldb/Lmo2 nucleated transcription complexes that normally function to enforce self-renewal in bone marrow hematopoietic progenitors',\n",
       "  'lmo2tg ldb1 ko dn2/3 replicate cd4 cells cell subtype',\n",
       "  'lmo2tg ldb1 ko dn4 replicate cd4 cells cell subtype',\n",
       "  'mouse'),\n",
       " 'GSE242170,GSE242171-4-vs-5-human up': (['IFN-λ signaling protects mice from severe HSV-1 skin disease II'],\n",
       "  'Type III interferons (IFN-λ) are antiviral and immunomodulatory cytokines that have been best characterized in respiratory and gastrointestinal infections, but the effects of IFN-λ against skin infections have not been extensively investigated. We sought to define the skin-specific effects of IFN-λ against the highly prevalent human pathogen herpes simplex virus (HSV). We infected mice lacking the IFN-λ receptor (Ifnlr1-/-), both the IFN-λ and the IFN-αβ receptor (Ifnar1-/- Ifnlr1-/-), or IFN-λ cytokines (Ifnl2/3-/-) and found that IFN-λ restricts the severity of HSV-1 and HSV-2 skin lesions, independent of a direct effect on viral load. Using conditional knockout mice, we found that IFN-λ signaling in both keratinocytes and neutrophils was necessary to control HSV-1 skin lesion severity, and that IFN-λ signaling in keratinocytes suppressed CXCL9-mediated neutrophil recruitment to the skin. Furthermore, depleting neutrophils prevented the development of severe HSV-1 skin lesions in Ifnlr1-/- mice. Altogether, our results suggest that IFN-λ plays an immunomodulatory role in the skin that restricts neutrophil-mediated pathology during HSV infection, and suggest potential applications for IFN-λ in treating viral skin infections.',\n",
       "  'a549 ifnlrko media lung adenocarcinoma cell line epithelial ifnlr1 ko',\n",
       "  'a549 ifnlrko ifnb lung adenocarcinoma cell line epithelial ifnlr1 ko ifn î²',\n",
       "  'human'),\n",
       " 'GSE99794-0-vs-1-human up': (['Activation of the p53 transcriptional program sensitizes cancer cells to Cdk7 inhibitors'],\n",
       "  'Cdk7, the CDK-activating kinase and transcription factor IIH component, is a target of inhibitors that kill cancer cells by exploiting tumor-specific transcriptional dependencies. However, whereas selective inhibition of analog-sensitive (AS) Cdk7 in colon cancer derived cells arrests division and disrupts transcription, it does not by itself trigger apoptosis efficiently. Here we show that p53 activation by 5-fluorouracil or nutlin-3 synergizes with a reversible Cdk7as inhibitor to induce cell death. Synthetic lethality was recapitulated with covalent inhibitors of wild-type Cdk7, THZ1 or the more selective YKL-1-116. The effects were allele-specific; a CDK7as mutation conferred both sensitivity to bulky adenine analogs and resistance to covalent inhibitors. Non-transformed colon epithelial cells were resistant to these combinations, as were cancer-derived cells with p53-inactivating mutations. Apoptosis was dependent on death receptor DR5, a p53 transcriptional target whose expression was refractory to Cdk7 inhibition. Therefore, p53 activation induces transcriptional dependency to sensitize cancer cells to Cdk7 inhibition.',\n",
       "  '5fu 12hr 4hr cell line hct 116 colorectal cancer treated 5 fluorouracil 12 hours followed 4',\n",
       "  'dmso 12hr 4hr cell line hct 116 colorectal cancer treated 12 hours followed 4',\n",
       "  'human'),\n",
       " 'GSE191054-1-vs-2-human dn': (['Transcriptomic data on the response of macropohages to monosodium urate crystals (MSUc)'],\n",
       "  'We dissect the mechanism of how MSUc activate a distinct inflammatory and metabolic program in macrophages by regulating JUN-JNK.',\n",
       "  'lps mdm agent 5h',\n",
       "  'msu mdm agent 5h',\n",
       "  'human'),\n",
       " 'GSE243360,GSE243361-1-vs-9-human dn': (['RNA-seq of human head and neck squamous cell carcinoma (HNSCC) to compare different tissue dissociation methods [bulkRNA]'],\n",
       "  'Tissue dissociation, mechanical and/or enzymatic, is a method of disaggregating 3D connective tissue to release cells for downstream single cell analysis.  Without proper controls for tissue dissociation experiments, potential artifacts can be mistaken for baseline tissue data. We hypothesized that enzymatic (collagenase/hyaluronidase) compared to mechanical dissociation leads to dissociated related artifacts not found in endogenous tumor tissue. To distinguish artifacts between enzymatic and mechanical dissociation, we compared separate enzymatic and mechanical dissociated samples with baseline samples from healthy and tumor patient whole blood, melanoma and head and neck squamous cell carcinoma (HNSCC) tumor tissues in flow cytometry, RNA sequencing and CODEX experiments. In bulk sorted tumor tissue, there is a significant difference between enzymatic and mechanical digested samples in cell stress, heat shock proteins and cell death related genes. These genes were also significantly upregulated in enzymatic over mechanical samples in the single cell RNA sequencing HNSCC tissue data. Fibroblasts and CD39+CD103+ CD8 T cells from tumor tissue, appear to require enzymatic digestion for efficient release from tissue compared to the mechanical digested samples. CD39+CD103+ CD8 tumor infiltrating lymphocytes (TILs) show different phenotypes depending on dissociation technique and tumor tissue. In tumor tissue, CD3+CD8+CD69+ T cells and gene expression were elevated in the enzymatic samples over the mechanical and baseline frozen tissue. Tissue dissociation techniques and tissue analysis from downstream applications have the potential for dissociation related artifacts that can mislead both protein and RNA sequencing data analysis in cancer research.',\n",
       "  'l head neck squamous cell carcinoma',\n",
       "  'e head neck squamous cell carcinoma',\n",
       "  'human'),\n",
       " 'GSE233400-0-vs-2-mouse dn': (['Microglial senescence contributes to female-biased neuroinflammation in the aging mouse hippocampus: implications for Alzheimer’s disease'],\n",
       "  'Microglia, the brain’s principal immune cells, have been implicated in the pathogenesis of Alzheimer’s disease (AD), a condition shown to affect more females than males. Although sex differences in microglial function and transcriptomic programming have been described across development and in disease models of AD, no studies have comprehensively identified the sex divergences that emerge in the aging mouse hippocampus. Further, existing models of AD generally develop pathology (amyloid plaques and tau tangles) early in life and fail to recapitulate the aged brain environment associated with late-onset AD. Here, we examined and compared the transcriptomic and translatomic sex effects in young and old murine hippocampal microglia. Hippocampal tissue from C57BL6/N and microglial NuTRAP mice of both sexes were collected at young (5-6 month-old [mo]) and old (22-25 mo) ages. Cell sorting and affinity purification techniques were used to isolate the microglial transcriptome and translatome for RNA-sequencing and differential expression analyses. Flow cytometry, qPCR, and imaging approaches were used to confirm the transcriptomic findings.  There were marginal sex differences identified in the young hippocampal microglia, with most differentially expressed genes (DEGs) restricted to the sex chromosomes. Both sex chromosomally- and autosomally-encoded sex differences emerged with aging. These sex DEGs identified at old age were primarily female-biased and enriched in senescent and disease-associated microglial signatures. Pathway analyses identified upstream regulators induced to a greater extent in females than in males, including inflammatory mediators IFNG, TNF, and IL1B, as well as AD-risk genes TREM2 and APP. These data suggest that female microglia adopt disease-associated and senescent phenotypes in the aging mouse hippocampus, even in the absence of disease pathology, to a greater extent than males. This sexually divergent microglial phenotype may explain the difference in susceptibility and disease progression in the case of AD pathology. Future studies will need to explore sex differences in microglial heterogeneity in response to AD pathology and determine how sex-specific regulators (i.e., sex chromosomal or hormonal) elicit these sex effects.',\n",
       "  'hippocampus sex female microglia c57bl6/n age 22 25 months old isolation technique cd11b macs',\n",
       "  'hippocampus sex male microglia c57bl6/n age 5 6 months old isolation technique cd11b macs',\n",
       "  'mouse'),\n",
       " 'GSE233386-5-vs-2-human up': (['sFANS RNAseq profiling of cortical cell types from human control samples'],\n",
       "  'To study the properties of the major cell types of human cortex we have employed serial fluorescence activated nuclear sorting (sFANS) and deep molecular profiling.',\n",
       "  'oligodendrocyte ctrl cortex donor age sex pmi (hrs) category',\n",
       "  'astrocyte ctrl cortex donor age sex pmi (hrs) category',\n",
       "  'human'),\n",
       " 'GSE255484-1-vs-0-mouse up': (['Comparative analysis of gene expression in tumors responding and non-responding to anti-CTLA-4 in a preclinical study of melanoma'],\n",
       "  'To investigate the diverse responses of the M4 mouse melanoma to immune checkpoint inhibitors, the tumors responding (R) and nonresponding (NR) to anti-CTLA-4 from a preclinical study of M4 melanoma were subjected to mRNA sequencing. The differentially expressed genes were analyzed to identify the signaling pathways that asscoiated with the therapeutic responses of the tumors.',\n",
       "  'tumor igg2b control id cell line b2905',\n",
       "  'tumor anti ctla 4 stable id cell line b2905',\n",
       "  'mouse'),\n",
       " 'GSE204665-8-vs-11-human up': (['The unexpected role of HBx in UC'],\n",
       "  'Ulcerative colitis is a chronic inflammatory disorder for which a definitive cure is still missing. This is characterized by an overwhelming inflammatory milieu in the colonic tract where a composite set of immune and non-immune cells orchestrate its pathogenesis. Over the last years, a growing body of evidence has been pinpointing gut virome dysbiosis as underlying its progression. Nonetheless, its role during the early phases of chronic inflammation is far from being fully defined. Here we show the gut virome-associated Hepatitis B virus protein X, most likely acquired after an event of zoonotic spillover, to be associated with the early stages of ulcerative colitis and to induce colonic inflammation in mice. It acts as a transcriptional regulator in epithelial cells, provoking barrier leakage and altering mucosal immunity at the level of both innate and adaptive immunity. This study paves the way to the comprehension of the aetiopathogenesis of intestinal inflammation and encourages further investigations of the virome as a trigger also in other scenarios. Moreover, it provides a brand-new standpoint that looks at the virome as a target for tailored treatments, blocking the early phases of chronic inflammation and possibly leading to better disease management.',\n",
       "  'post facs lpmc colon macrophages',\n",
       "  'post facs en lpmc colon endothelium',\n",
       "  'human'),\n",
       " 'GSE141136-1-vs-4-human dn': (['A Stem Cell-Derived Endothelial Cell Model that Responds to Tobacco Smoke Like Primary Endothelial Cells'],\n",
       "  'To clarify how smoking leads to heart attacks and stroke, we developed an endothelial cell model (iECs) generated from human induced Pluripotent Stem Cells (iPSC) and evaluated its responses to tobacco smoke. These iECs exhibited a uniform endothelial morphology, expressed markers PECAM1/CD31, VWF/von Willebrand Factor, and CDH5/VE-Cadherin, and exhibited tube formation and Acetyl-LDL uptake comparable to primary endothelial cells (EC). RNA sequencing (RNAseq) revealed a robust correlation coefficient between iECs and EC (R = 0.76), whereas gene responses to smoke were qualitatively nearly identical between iECs and primary ECs (R=0.86). Further analysis of transcriptional responses implicated eighteen transcription factors in regulating responses to smoke treatment, and identified gene sets regulated by each transcription factor, including oxidative stress, DNA damage/repair, ER stress, apoptosis, and cell cycle arrest. Assays for 42 cytokines in HUVEC cells and iECs identified 23 cytokines that responded dynamically to cigarette smoke. These cytokines and cellular stress response pathways describe endothelial responses for lymphocyte attachment, activation of coagulation and complement, lymphocyte growth factors, and inflammation and fibrosis; EC-initiated events that collectively lead to atherosclerosis. Thus, these studies validate the iEC model and identify transcriptional response networks by which ECs respond to tobacco smoke. Our results systematically trace how ECs use these response networks to regulate genes and pathways, and finally cytokine signals to other cells, to initiate the diverse processes that lead to atherosclerosis and cardiovascular disease.',\n",
       "  'ipsc control cell long name induced pluripotent stem iec clone n/ study arm line characterization',\n",
       "  'hcaec smoke 6h cell long name human coronary artery endothelial iec clone n/ study arm response',\n",
       "  'human'),\n",
       " 'GSE104917-3-vs-0-human up': (['RNASeq analysis of NB1691 neuroblastoma cells treated with MDM2-p53 inhibitor RG7388 alone and in combination with temozolomide'],\n",
       "  'To further identify potential mechanisms involved in combining RG7388 with temozolomide, GSEA was performed and revealed that RG7388 and temozolomide act cooperatively to upregulate the p53 pathway and downregulate MYC targets, the former resulting in significantly greater induction of several apoptotic genes.',\n",
       "  'nb1691 r+ 72hr cell line mycn status amplified tp53 wt rg7388 + temozolomide 72h neuroblastoma',\n",
       "  'nb1691 tmz 24hr cell line mycn status amplified tp53 wt temozolomide 24h neuroblastoma',\n",
       "  'human'),\n",
       " 'GSE131768-0-vs-1-human dn': (['Generation of mature lung alveolar epithelial cells from human pluripotent stem cells'],\n",
       "  'We perform time-series global transcriptomic profiling of a directed differentiation protocol for generating alveolar epithelial type II cells (AEC2s) from pluripotent stem cells (PSCs). We analyzed 3 different timepoints in the RUES2 differentiation: 1) Day 0 undifferentiated cells, 2) Day 15 lung progenitors highly enriched in NKX2-1+ cells by CD47hi/CD26lo sorting, and 3) the outgrowth of these purified progenitors in 3D culture sorted again on Day 35 based on SFTPCtdTomato+ and SFTPCtdTomato- gating. For comparison to primary cells, we simultaneously sequenced RNA from purified primary fetal (21 week gestation) distal alveolar epithelial progenitors and purified adult human (HTII-280 sorted) AEC2s. In order to evaluate the effect of cell culture on primary fetal alveolar cells, parallel aliquots of the fetal cells were also exposed to 4 days of culture in DCI media. We find that AEC2 maturation involves downregulation of Wnt signaling activity, and that the highest differentially expressed transcripts in iPSC-derived AEC2s encode genes associated with lamellar body and surfactant biogenesis.',\n",
       "  'adult aec2 analyzed lung developmental stage years culture media none isolation method sorted ht2 280 antibody positive cells',\n",
       "  \"week fetal lung explant developmental stage gestational culture media overnight waymouth' isolation method enriched epithelial cells\",\n",
       "  'human'),\n",
       " 'GSE194396,GSE194429-1-vs-0-human dn': (['Impaired gene expression in ATRX-depleted cells following interferon signaling'],\n",
       "  'We investigated the role of the chromatin remodeling protein ATRX on ISG expression upon stimulation with IFN-β utilizing doxycycline-inducible ATRX knockdown cells.',\n",
       "  'hffs wt ifn control stimulated î² human foreskin fibroblasts (hffs)',\n",
       "  'hffs dox induced atrx knockdown treated human foreskin fibroblasts (hffs)',\n",
       "  'human'),\n",
       " 'GSE131751-0-vs-2-mouse dn': (['RNAseq of renal dendritic cells from adult Clec9a-Cre Rosa-YFP mice'],\n",
       "  'We use bulk RNA sequencing of sorted cells to characterize the gene expression profiles of renal dendritic cell (DC) subsets, cDC1 and cDC2, as well as MHCII+CD64+ F4/80hi and MHCII+CD64+ CD11bhi cells. Splenic DCs and red pulp macrophages serve as reference populations for a macrophage-like or DC-like phenotype. By sorting YFP+/YFP- F4/80hi or YFP+/YFP- CD11bhi cells from Clec9a-Cre Rosa-YFP mice we aim to reveal transcriptional differences between YFP-labelled and unlabelled cells. We showed that F4/80hi cells resemble macrophages on a transcriptional level, despite their DC origin, and that renal CD11bhi cells are a transcriptionally unique subset. However, we were not able to reveal differences between YFP+ and YFP- populations.',\n",
       "  'rpm strain c57bl/6 spleen age 10 weeks clec9a cre/cre rosa yfp/yfp red pulp macrophages sorting strategy autofluorescence+ f4/80+ yfp',\n",
       "  'cd11bhi strain c57bl/6 kidney age 10 weeks clec9a cre/cre rosa yfp/yfp cd11bhif4/80low sorting strategy cd45+ ly6c mhcii+ cd64+ f4/80low',\n",
       "  'mouse'),\n",
       " 'GSE163568-4-vs-5-human up': (['Notch signaling rapidly regulates the expression of the small GTPase RND1 and a diverse endothelial transcriptome'],\n",
       "  'Endothelial Notch signaling regulates transcription of many downstream effectors controlling sprouting, migration, proliferation, barrier formation, and other phenotypes. However, endothelial-relevant Notch targets are largely uncharacterized. Few are studied in depth, and many transient, early response Notch targets are yet to be described. We therefore determined the Notch early response transcriptional profile in several experimentally relevant in vivo and in vitro contexts: ligand-specific or EGTA-induced Notch activation in different primary endothelial cells, and gamma secretase-mediated Notch inhibition in neonatal brain endothelium in a RiboTag mouse model. The intersection of these profiles reveals a core set of early Notch-responsive targets.  Among these are effectors in GPCR signaling, not previously implicated downstream of Notch signaling. We demonstrate the Rho GTPase RND1 is a direct Notch transcriptional target and required for Notch control of sprouting angiogenesis, endothelial migration and Ras activity.',\n",
       "  'hrec tla control primary cells (human microvascular retinal endothelial cells) human (hrec)',\n",
       "  'huvec tla dll4 primary cells (human umbilical vein endothelial cells) human (huvec)',\n",
       "  'human'),\n",
       " 'GSE129770-3-vs-7-mouse up': (['PS2APP sorted cell types'],\n",
       "  'Mice of indicated genotypes were perfused and their brains dissected and dissociated. Cells were fixed, immunolabeled and FACS sorted. RNA was extracted from neuron, astrocyte, and microglial cell populations. Typical RIN=4-5 for neurons, 6-8 for astrocytes, and 5-7 for microglia. Typical RNA yields 200-300ng for neurons, ~20ng for microglia, and ~5ng for astrocytes.',\n",
       "  'non tg cd11b sample id transgenic cd11b+ myeloid cells age 11.5 month old',\n",
       "  'ps2app neun sample id transgenic mice neun+ neurons age 11.5 month old',\n",
       "  'mouse'),\n",
       " 'GSE141438,GSE141439-2-vs-0-human dn': (['RNA-seq dataset for MPNST patient tissue samples'],\n",
       "  'Comprehensive transcriptomic profiling of PRC2 mutant MPNST patient tissues with adjacent normal tissues and neurofibroma patient tissues was performed to investigate gain of specific transcriptional signature associated with PRC2 loss during transformation to MPNST.',\n",
       "  'sample rna seq muscle prc2 mutant diagnosis mpnst adjacent normal',\n",
       "  'sample rna seq mpnst prc2 mutant diagnosis tumor',\n",
       "  'human'),\n",
       " 'GSE49379-2-vs-4-human dn': (['Large-scale multi-species survey of metabolome and lipidome'],\n",
       "  'This dataset was generated with the goal of comparative study of gene expression in three brain regions and two non-neural tissues of humans, chimpanzees, macaque monkeys and mice. Using this dataset, we performed studies of gene expression and gene splicing evolution across species and search of tissue-specific gene expression and splicing patterns. We also used the gene expression information of genes encoding metabolic enzymes in this dataset to support a larger comparative study of metabolome evolution in the same set of tissues and species.',\n",
       "  'strain/background c57bl/6 muscle sex age (years)',\n",
       "  'strain/background c57bl/6 prefrontal cortex sex age (years) pfc',\n",
       "  'human'),\n",
       " 'GSE232817,GSE232824-0-vs-5-human up': (['Modular Pooled Discovery of Synthetic Knockin Sequences to Program Durable Cell Therapies [RNA-1]'],\n",
       "  'Chronic stimulation can cause T cell dysfunction and limit efficacy of cellular immunotherapies. CRISPR screens have nominated gene targets for engineered T cells, but improved methods are required to compare large numbers of synthetic knockin sequences to reprogram cell functions. Here, we developed Modular Pooled Knockin Screening (ModPoKI), an adaptable platform for modular construction of DNA knockin libraries using barcoded multicistronic adaptors. We built two ModPoKI libraries of 100 transcription factors (TFs) and 129 natural and synthetic surface receptors. Over 30 ModPoKI screens across human TCR and CAR T cells in diverse conditions identified a transcription factor AP4 (TFAP4) construct that enhanced fitness of chronically-stimulated CAR T cells and anti-cancer function in vitro and in vivo. ModPoKI’s modularity allowed us to generate a ~10,000-member library of TF combinations. Non-viral knockin of a combined BATF-TFAP4 polycistronic construct further enhanced function. ModPoKI facilitates discovery of complex gene constructs to program cellular functions.',\n",
       "  'cells tngfr cd19 28z car knockin day 15 donor individual blood cell line primary none time',\n",
       "  'cells tngfr cd19 bbz car knockin day 6 donor individual blood cell line primary none time',\n",
       "  'human'),\n",
       " 'GSE129244-3-vs-5-mouse up': (['Ldb1 is required for Lmo2 oncogene-induced thymocyte self-renewal and T-cell Acute Lymphoblastic Leukemia'],\n",
       "  'Prolonged or enhanced expression of the proto-oncogene Lmo2 is associated with a severe form of T-cell Acute Lymphoblastic Leukemia (T-ALL), designated Early T-progenitor ALL (ETP-ALL), that is characterized by the aberrant self-renewal and subsequent oncogenic transformation of immature thymocytes. Recent data suggest that Lmo2 may exert these effects by functioning as component of a multi-subunit transcription complex that includes the ubiquitous adapter Ldb1 along with b-HLH and/or GATA family transcription factors. In this study, we investigated the importance of Ldb1 for Lmo2-induced T-ALL by conditional deletion of Ldb1 in thymocytes in a Lmo2 transgenic mouse model of T-ALL. Our results identify a critical requirement for Ldb1 in the induction of thymocyte self-renewal, thymocyte radio-resistance and transition to T-ALL in Lmo2 transgenic mice. Ldb1 was also required for acquisition of the pre-leukemic ETP gene expression signature in immature Lmo2 transgenic thymocytes. Together, these results support a model where Lmo2-induced T-ALL results from failure to down-regulate Ldb/Lmo2 nucleated transcription complexes that normally function to enforce self-renewal in bone marrow hematopoietic progenitors',\n",
       "  'wild type dn2/3 replicate cd4 cells cell subtype',\n",
       "  'lmo2tg ldb1 ko dn4 replicate cd4 cells cell subtype',\n",
       "  'mouse'),\n",
       " 'GSE115168-1-vs-2-human up': (['ER proteostasis and temperature differentially impact the mutational tolerance of influenza hemagglutinin'],\n",
       "  'Here, we present a systematic and quantitative test of the hypothesis that the composition and activities of the endoplasmic reticulum (ER) proteostasis network impact mutational tolerance of secretory pathway client proteins. We focus on influenza hemagluttinin (HA), a viral coat protein that folds in the host’s ER via a complex but well-characterized pathway. By integrating chemical methods to modulate the unfolded protein response with deep mutational scanning to assess mutational tolerance, we discover that upregulation of ER chaperones broadly enhances HA mutational tolerance across numerous sites and secondary/tertiary structure elements, including sites targeted by host antibodies. Remarkably, this host chaperone-enhanced mutational tolerance is observed at the same HA sites where mutational tolerance is most reduced by propagation at a fever-like temperature. Thus, host ER proteostasis mechanisms and temperature modulate HA mutational tolerance in opposite directions. This finding has important implications for influenza evolution, because influenza immune escape is contingent on HA possessing sufficient mutational tolerance to acquire antibody resistance while still maintaining the capacity to fold and function. More broadly, this work provides the first experimental evidence that the composition and activities of the ER proteostasis network critically define the mutational tolerance and, therefore, the evolution of secretory pathway client proteins.',\n",
       "  'd17 basal 39c adult derived epithelial cell line human embryonic kidney (hek) 293t rex clonal stably expressing dhfr.atf6 tet xbp1s tetracycline repressor condition 24h 0.01% dmso cells',\n",
       "  'd17 atf6f/xbp1s 37c adult derived epithelial cell line human embryonic kidney (hek) 293t rex clonal stably expressing dhfr.atf6 tet xbp1s tetracycline repressor condition 24h 0.1 ug/ml doxycycline + 1 um trimethoprim cells',\n",
       "  'human'),\n",
       " 'GSE175495-8-vs-0-human up': (['Divergent immunometabolic changes in adipose tissue and skeletal muscle with ageing in healthy humans'],\n",
       "  \"RNA-sequencing from 12 young (20-35 years old) and 12 older (60-85 years old) participants' skeletal muscle and adipose tissue. Biopsies were taken at rest following an overnight fast, and following alcohol/caffeine restriction for 24 hours, and no strenuous physical activity for 48 hours. Skeletal muscle was taken by the Bergstrom technique under local anaesthesia (Lidocaine hydrochloride) from the dominant leg Vastus Lateralis, and adipose tissue was taken by needle aspiration 5cm lateral to the umbilicus.\",\n",
       "  'young adiposetissue age subcutaneous abdominal adipose sample type untreated rested healthy human whole (adipose )',\n",
       "  'old skeletalmuscle age vastus lateralis skeletal muscle dominant leg sample type untreated rested healthy human whole (skeletal muscle)',\n",
       "  'human'),\n",
       " 'GSE178296,GSE178304-2-vs-0-mouse up': (['Transcriptome profiling of mouse brain microglia in Cnp null mice with 5xFAD background'],\n",
       "  'Alzheimer’s disease (AD) is the most common form of dementia and neurodegenerative disease with increasing prevalence due to longer lifespan in the human population. Why aging constitutes the greatest risk factor for development of AD, however, remains poorly understood. Aging markedly affects oligodendrocytes and the structural integrity of their myelin sheaths which also causes secondary tissue inflammation. We propose a mechanistic link between aging-associated myelin dysfunction and the deposition of Amyloid-ß (Aß) as primary neuropathological hallmark of early AD and hypothesized that breakdown of myelin - especially in cortical regions – is an upstream driver of amyloid deposition in AD. Here, we show that in transgenic mouse models of AD, genetically induced myelin defects by Cnp or Plp1 depletion, as well as direct demyelination are potent drivers of amyloid deposition in vivo as shown by light sheet microscopy imaging. At transcriptomic level, bulk and single-cell RNA sequencing revealed successfully induced disease-associated-microglia (DAM)-like phenotypes in Cnp-/- animals. These activated microglia, however, are primarily engaged with myelin seemingly preventing the protective reactions of the microglia pool to Aß plaques. Our work, therefore, identifies myelin aging as a previously overlooked risk factor for AD and makes the case for myelin health-directed therapies in AD.',\n",
       "  'wt bulkrna strain c57bl6/j age 6 month microglia brain mouse model alzheimerâ\\x80\\x99 disease (ad)',\n",
       "  '5xfad bulkrna strain c57bl6/j age 6 month microglia brain mouse model alzheimerâ\\x80\\x99 disease (ad)',\n",
       "  'mouse'),\n",
       " 'GSE225336-6-vs-0-human dn': (['Genetics of Myocardial Interstitial Fibrosis in the Human Heart and Association with Disease'],\n",
       "  'Myocardial interstitial fibrosis is a common thread in multiple cardiovascular diseases including heart failure, atrial fibrillation, conduction disease and sudden cardiac death. To investigate the biologic pathways that underlie interstitial fibrosis in the human heart, we developed a machine learning model to measure myocardial T1 time, a marker of myocardial interstitial fibrosis, in 41,505 UK Biobank participants. Greater T1 time was associated with diabetes mellitus, renal disease, aortic stenosis, cardiomyopathy, heart failure, atrial fibrillation, conduction disease and rheumatoid arthritis. Mendelian randomization analysis supported a potential causal role for diabetes mellitus type 1 in myocardial interstitial fibrosis. In genome-wide association analysis, we identified 11 independent loci associated with native myocardial T1 time. The identified loci implicated genes involved in glucose transport (SLC2A12), iron homeostasis (HFE, TMPRSS6), tissue repair (ADAMTSL1, VEGFC), oxidative stress (SOD2), cardiac hypertrophy (MYH7B) and calcium signaling (CAMK2D). Transcriptome-wide association studies highlighted the role of expression of ADAMTSL1 and SLC2A12 in human cardiac tissue in modulating myocardial interstitial fibrosis. Using a TGFβ1-mediated cardiac fibroblast activation assay, we found that 9 out of the 11 genome-wide significant loci comprised genes that exhibited temporal changes in expression and/or open chromatin conformation supporting the biological relevance of these loci to myocardial fibrosis and myofibroblast cell state acquisition. Harnessing machine learning to perform large-scale phenotyping of interstitial fibrosis in the human heart, our results yield novel insights into biologically relevant pathways to myocardial fibrosis and prioritize a number of pathways for further investigation.',\n",
       "  'cf rna 24hr cell line primary cardiac fibroblasts wildtype ctrl 24hrs',\n",
       "  'cf rna 72hr cell line primary cardiac fibroblasts wildtype ctrl 72hrs',\n",
       "  'human'),\n",
       " 'GSE128623-1-vs-4-mouse dn': (['The spinal transcriptome after cortical stroke – In search of molecular factors regulating spontaneous recovery in the spinal cord'],\n",
       "  'In response to cortical stroke and unilateral corticospinal tract degeneration, compensatory sprouting of spared corticospinal fibers is associated with recovery of skilled movement in rodents.  To date, little is known about the molecular mechanisms orchestrating this spontaneous rewiring. In this study, we provide insights into the molecular changes in the spinal cord tissue after large ischemic cortical injury in adult female mice, with a focus on factors that might influence the re-innervation process by contralesional corticospinal neurons. We mapped the area of cervical grey matter re-innervation by sprouting contralesional corticospinal axons after unilateral photothrombotic stroke of the motor cortex in mice using anterograde tracing. The mRNA profile of this re-innervation area was analyzed using whole-genome sequencing to identify differentially expressed genes at selected time points during the recovery process. Bioinformatic analysis revealed two phases of processes: Early after stroke (4-7 days post injury), the spinal transcriptome is characterized by inflammatory processes, including phagocytic processes as well as complement cascade activation. Microglia are specifically activated in the denervated corticospinal projection fields in this early phase. In a later phase (28-42 days post injury), biological processes include tissue repair pathways with up-regulated genes related to neurite outgrowth. Thus, the stroke-denervated spinal grey matter, in particular its intermediate laminae, represents a growth-promoting environment for sprouting corticospinal fibers originating from the contralesional motor cortex. This data set provides a solid starting point for future studies addressing key elements of the post-stroke recovery process, with the goal to improve neuroregenerative treatment options for stroke patients.',\n",
       "  '14d a5 e1 spinal cord strain c57bl/6 developmental stage adult',\n",
       "  '28d a4 e1 spinal cord strain c57bl/6 developmental stage adult',\n",
       "  'mouse'),\n",
       " 'GSE140035,GSE140044-6-vs-1-mouse dn': (['Deconvoluting global cytokine signaling networks in natural killer cells [RNA-Seq]'],\n",
       "  'Cytokine signaling via signal transducer and activator of transcription (STAT) proteins is crucial for optimal antiviral responses of natural killer (NK) cells. However, the pleiotropic effects of both cytokine and STAT signaling preclude the ability to precisely attribute molecular changes to either source. In this study, we employ a multi-“omics” approach to deconstruct and rebuild the complex interaction of three major STAT signaling pathways in NK cells. We uncover a global STAT4-STAT5 cooperative axis that is distinct and sometimes antagonistic to the STAT1 axis. We generate a network of STAT targets that highlights an integrated negative feedback loop, and demonstrate distinct STAT modes of epigenetic regulation. Finally, we identify shared signatures between mouse in vitro profiles and profiles from viral infection and humans, highlighting clinically translatable targets. Overall, we begin to unravel the intricate crosstalk between cytokine signaling pathways, and offer a valuable resource for improving cellular immunotherapy.',\n",
       "  'rna mouse wt il12il18 ex vivo nk cells condition replicate splenic',\n",
       "  'rna mouse wt unstim ex vivo nk cells condition replicate splenic',\n",
       "  'mouse'),\n",
       " 'GSE182959-2-vs-5-human up': (['AFF1-HEXIM1 cooperation suppresses rapid-response differentiation initiators to enforce epidermal progenitor maintenance'],\n",
       "  'In this study we identified an essential role of the SEC, a crucial regulator of Pol II dynamics, in repressing differentiation for epidermal progenitor maintenance. This repression of differentiation is specifically mediated by the scaffold protein AFF1, but not AFF4. Mechanistically, we find that AFF1-SEC associates with the HEXIM1-containing inactive-PTEFb to directly repress a group of rapid-response genes, which feature robust Pol II pausing near their promoters in the progenitor state. De-repression of these genes occurs within 3 hours of SEC-PTEFb disruption by the peptidomimetic inhibitors. These rapid-response genes include the transcription factor ATF3. Increased ATF3 level is sufficient to promote the expression of key differentiation activators, such as PRDM1, OVOL1, GRHL3 and ZNF750. Furthermore, we find that the dissociation of inactivate P-TEFb from SEC mediates the earliest events of PKC signaling, in triggering epidermal differentiation.',\n",
       "  'rna undifferentiated keratinocytes shrna knockdown primary human neonatal',\n",
       "  'rna undifferentiated keratinocytes hour dmso primary human neonatal',\n",
       "  'human'),\n",
       " 'GSE157057-0-vs-3-human dn': (['Human lung organoid for modeling infection and disease conditions (Lung II)'],\n",
       "  'We sought to define the host immune response, a.k.a, the “cytokine storm” that has been implicated in fatal COVID-19 using an AI-based approach. Over 45,000 publicly available transcriptomic datasets of viral pandemics were analyzed to extract a 166-gene signature using ACE2 as a ‘seed’ gene; ACE2 was rationalized because the receptor it encodes enables the virus that causes Covid-19, SARS-CoV-2, to enter host cells. The signature was surprisingly conserved in all viral pandemics, including COVID-19, inspiring the nomenclature ViP-signature. A subset of 20-genes classified disease severity in respiratory pandemics. The ViP signatures pinpointed airway epithelial and myeloid cells as the major contributors of an IL-15 cytokine storm, and epithelial and NK cell destruction as determinants of severity/fatality. They also helped formulate precise therapeutic goals to reduce disease symptoms and severity. Thus, the ViP signatures provide a quantitative and qualitative framework for titrating the immune response in viral pandemics and may serve as a powerful unbiased tool in our armamentarium to rapidly assess disease severity and vet candidate drugs.',\n",
       "  'ja1 ali l004 condition air liquid interface infection lung',\n",
       "  'ja1 monolayer l004 condition infection lung',\n",
       "  'human'),\n",
       " 'GSE196006-6-vs-13-human up': (['Transcriptome analysis implicates MYC as a driver of early onset colorectal cancer'],\n",
       "  'Despite a global decrease in colorectal cancer (CRC) incidence, the prevalence of early onset colorectal cancer (EOCRC), or those occurring in individuals before the age of 50, has been steadily increasing over the past several decades. When compared to later onset colorectal cancers (LOCRCs) in individuals over 50, our understanding of the genetic and molecular underpinnings of EOCRCs is limited. Here, we conducted transcriptomic analysis of patient-matched normal colonic segments and tumors to identify gene expression programs involved in carcinogenesis. Amongst differentially expressed genes, we found increased expression of the c-MYC (MYC) proto-oncogene and its downstream targets in tumor samples. We identify tumors with high and low differential MYC expression, and patients with high-MYC tumors were older and overweight or obese. We also detect elevated expression of the PVT1 long-non-coding RNA (lncRNA) in most tumors and find gains in copy number for both MYC and PVT1 gene loci in 35% of tumors evaluated. Our transcriptome analysis indicates that EOCRC can be sub-classified into groups based on differential MYC expression and suggests that MYC is a critical driver of most CRCs that develop in younger patients.',\n",
       "  'normal id diagnosis early onset colorectal cancer adjacent tumor stage age surgery sigmoid colon',\n",
       "  'tumor id diagnosis early onset colorectal cancer stage age surgery sigmoid colon',\n",
       "  'human'),\n",
       " 'GSE188992-0-vs-2-mouse dn': (['Bulk RNAseq of Mouse Splenic Dendritic Cell Subsets'],\n",
       "  'Dendritic cells (DCs) are comprised of a heterogenous population. Here we characterize the transcriptomic profiles of mouse splenic DC subsets at steady state to highlight key differences between subsets.',\n",
       "  'b6 cd8pos cdc1 strain c57bl/6j state steady spleen dc cell subset',\n",
       "  'b6 cd8neg cdc2 strain c57bl/6j state steady spleen dc cell subset',\n",
       "  'mouse'),\n",
       " 'GSE175495-2-vs-0-human up': (['Divergent immunometabolic changes in adipose tissue and skeletal muscle with ageing in healthy humans'],\n",
       "  \"RNA-sequencing from 12 young (20-35 years old) and 12 older (60-85 years old) participants' skeletal muscle and adipose tissue. Biopsies were taken at rest following an overnight fast, and following alcohol/caffeine restriction for 24 hours, and no strenuous physical activity for 48 hours. Skeletal muscle was taken by the Bergstrom technique under local anaesthesia (Lidocaine hydrochloride) from the dominant leg Vastus Lateralis, and adipose tissue was taken by needle aspiration 5cm lateral to the umbilicus.\",\n",
       "  'young adiposetissue age subcutaneous abdominal adipose sample type untreated rested healthy human whole (adipose )',\n",
       "  'old skeletalmuscle age vastus lateralis skeletal muscle dominant leg sample type untreated rested healthy human whole (skeletal muscle)',\n",
       "  'human'),\n",
       " 'GSE204722,GSE235157-3-vs-2-human dn': (['Global Epigenome and Transcriptome Maps of Human Trophoblast Cell Lineage Development'],\n",
       "  'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'epas1 control source human trophoblast stem cells cell line ct27 extravillous shrna lentiviral transduction puromycin selection',\n",
       "  'tfap2c control source human trophoblast stem cells cell line ct27 state shrna lentiviral transduction puromycin selection',\n",
       "  'human'),\n",
       " 'GSE140035,GSE140044-7-vs-4-mouse dn': (['Deconvoluting global cytokine signaling networks in natural killer cells [RNA-Seq]'],\n",
       "  'Cytokine signaling via signal transducer and activator of transcription (STAT) proteins is crucial for optimal antiviral responses of natural killer (NK) cells. However, the pleiotropic effects of both cytokine and STAT signaling preclude the ability to precisely attribute molecular changes to either source. In this study, we employ a multi-“omics” approach to deconstruct and rebuild the complex interaction of three major STAT signaling pathways in NK cells. We uncover a global STAT4-STAT5 cooperative axis that is distinct and sometimes antagonistic to the STAT1 axis. We generate a network of STAT targets that highlights an integrated negative feedback loop, and demonstrate distinct STAT modes of epigenetic regulation. Finally, we identify shared signatures between mouse in vitro profiles and profiles from viral infection and humans, highlighting clinically translatable targets. Overall, we begin to unravel the intricate crosstalk between cytokine signaling pathways, and offer a valuable resource for improving cellular immunotherapy.',\n",
       "  'rna mouse wt il2il15il12il18 ex vivo nk cells condition replicate splenic',\n",
       "  'rna mouse wt il2il15 ex vivo nk cells condition replicate splenic',\n",
       "  'mouse'),\n",
       " 'GSE204665-6-vs-11-human dn': (['The unexpected role of HBx in UC'],\n",
       "  'Ulcerative colitis is a chronic inflammatory disorder for which a definitive cure is still missing. This is characterized by an overwhelming inflammatory milieu in the colonic tract where a composite set of immune and non-immune cells orchestrate its pathogenesis. Over the last years, a growing body of evidence has been pinpointing gut virome dysbiosis as underlying its progression. Nonetheless, its role during the early phases of chronic inflammation is far from being fully defined. Here we show the gut virome-associated Hepatitis B virus protein X, most likely acquired after an event of zoonotic spillover, to be associated with the early stages of ulcerative colitis and to induce colonic inflammation in mice. It acts as a transcriptional regulator in epithelial cells, provoking barrier leakage and altering mucosal immunity at the level of both innate and adaptive immunity. This study paves the way to the comprehension of the aetiopathogenesis of intestinal inflammation and encourages further investigations of the virome as a trigger also in other scenarios. Moreover, it provides a brand-new standpoint that looks at the virome as a target for tailored treatments, blocking the early phases of chronic inflammation and possibly leading to better disease management.',\n",
       "  'post facs b lpmc colon cells',\n",
       "  'post facs en lpmc colon endothelium',\n",
       "  'human'),\n",
       " 'GSE197854-1-vs-2-mouse dn': (['The immunocytokine PD1-IL2v overcomes resistance to tumor immunity, and synergizes with anti-PD-L1'],\n",
       "  'Immunotherapies have shown remarkable, albeit tumor-selective therapeutic benefits in the clinic. Here we evaluated the effects of the immunocytokine PD1-IL2v in a mouse model of de novo pancreatic neuroendocrine cancer resistant to checkpoint and other immunotherapies. PD1-IL2v is a bi-specific molecule based on a bivalent PD-1 antibody, serving as a targeting moiety, fused to an immuno-stimulatory IL-2 variant (IL2v) to precisely deliver IL2v to PD-1+ T-cells in the tumor microenvironment. PD1-IL2v elicited dramatic infiltration by CD8 T cells, notably stem-like and effector memory T cells, resulting in tumor regression and enhanced survival in mice. Furthermore, combining anti-PD-L1 with PD1-IL2v sustained the response phase, improving therapeutic efficacy both by reprogramming immunosuppressive tumor-associated macrophages and tumor endothelial cells\\xa0, and by enhancing TCR immune repertoire diversity. The data motivate consideration of clinical trials to evaluate the combination therapy of PD1-IL2v and anti-PD-L1, especially in immunotherapy-resistant tumors infiltrated with PD-1+ T stem-like CD8 T cells.',\n",
       "  'model rip tag5 pannet mouse spleen group duration 10 days mice id replicate 2',\n",
       "  'tcr model rip tag5 pannet mouse tumor group duration 10 days mice id replicate 2',\n",
       "  'mouse'),\n",
       " 'GSE118106-1-vs-2-human up': (['Transcriptome analysis of human embryonic stem cell differentiation to endothelial cells'],\n",
       "  'Deep RNA sequencing was performed during human embryonic stem cell (hESC) differentiation to endothelial cells (ECs), via mesodermal precursors. Our work focused on long non-coding RNAs (lncRNAs) regulated during EC differentiation as lncRNA are known regulators of cell differentiation, physiology and disease. EC-enriched lncRNAs were screened for binding sites for the transcription factor ERG, important for endothelial cell identity and function. The LINC00961 gene, first annotated as a lncRNA but now reassigned as a protein coding gene for the SPAAR micropeptide, was potently activated during differentiation. Using primary ECs, we studied the role of LINC00961 locus after depletion and gain of function experiments and showed its effect on EC adhesion, tube formation, migration and proliferation. We showed that LINC00961 functions as a bona fide non-coding RNA in ECs, at least in part, through binding to TMSB4X.',\n",
       "  'hesc human embryonic stem cells replicate endothelial cell (ec) differentiation (hesc)',\n",
       "  'hsvec human saphenous vein endothelial cells replicate',\n",
       "  'human'),\n",
       " 'GSE146441-4-vs-3-human dn': (['Stimulation of gluten-specific CD4+ T cell lines with aCD3/aCD28'],\n",
       "  'Gluten specific CD4 positive T cell upon anti CD3 and anti CD28 stimulation',\n",
       "  '3 tcc 1 t180 polyclonal gluten specific cell line timepoint (minutes) 180 small intestine derived lines',\n",
       "  '3 tcc t0 polyclonal gluten specific cell line timepoint (minutes) 0 small intestine derived lines',\n",
       "  'human'),\n",
       " 'GSE112339-4-vs-8-human dn': (['Regional differences in mRNA and lncRNA expression patterns in the non-failing human heart'],\n",
       "  'Background: Although chamber specialization is critical for proper cardiac function, a comprehensive, genome-wide analysis of the cardiac transcriptome, including identification of regional differences in mRNA and lncRNA expression patterns for the four chambers and interventricular septum of the non-failing human heart, has not been performed.   Methods and Results: mRNA and long noncoding RNA (lncRNA) transcriptional profiling of the left (LA) and right (RA) atria, left (LV) and right (RV) ventricles, and the interventricular septum (IVS) of non-failing human hearts (N=8) was performed by deep sequencing. Analysis of the mRNA and lncRNA expression profiles revealed that the different regions of the heart are distinct. Differential expression analysis of paired tissue samples identified 5,747 mRNAs and 2,794 lncRNAs with chamber-enriched expression patterns. The largest differences in mRNA and lncRNA expression were evident between atria and ventricular samples, including regional differences in ~20% of all cardiac expressed mRNA and lncRNA transcripts. Regional differences in mRNA and lncRNA expression were also evident, although to a lesser extent, between the LA and RA, and between the LV, RV and IVS. Gene ontology classification of differentially expressed gene sets revealed regional differences in chamber specialization, including differences in signaling, metabolism, and muscle contraction. Sex differences in mRNA and lncRNA gene expression profiles were also identified between male and female LA and RA samples.    Conclusions: There are marked regional differences in the mRNA and lncRNA expression profiles in non-failing adult human heart, and are associated with chamber specialization.',\n",
       "  'ra [heart heart region right atrium id sex age years',\n",
       "  'rv [heart heart region right ventricle id sex age years',\n",
       "  'human'),\n",
       " 'GSE152461-4-vs-8-human dn': (['Transcriptomic analysis of brian tissues identifies a role for CCAAT enhancer binding protein β in HIV- associated neurocognitive disorder'],\n",
       "  'In this study, using 33 postmortem frontal cortex (FC) tissues, neuropathological, molecular, and biochemical analyses were used to determine cellular localization and validate expression levels of the prolific transcription factor (TF), CCAAT enhancer binding protein (C/EBP) β, in brain tissues from HIV+ cognitively normal and HAND cases. RNA sequencing (seq) and transcriptomic analyses were performed on FC tissues including 24 specimens from well-characterized people with HIV that had undergone neurocognitive assessments. In vitro models for brain cells were used to investigate the role of C/EBPβ in mediating gene expression.',\n",
       "  'frontal cortex age sex male neurocognitive status mnd arv haart',\n",
       "  'frontal cortex age neurocognitive status arv',\n",
       "  'human'),\n",
       " 'GSE176271,GSE176289-4-vs-5-human dn': (['Integrative analysis of DNA methylation and microRNA expression reveals mechanisms of disparity in hepatocellular carcinoma [RNA-Seq]'],\n",
       "  'We evaluate the epigenetic mechanisms of racial disparity in HCC through an integrated analysis of DNA methylation, miRNA, and combined regulation of gene expression. Specifically, we acquired DNA methylation, mRNA-seq, and miRNA-seq data through the analysis of tumor and adjacent normal liver tissues from African Americans (AA) and European Americans (EA) with HCC.',\n",
       "  'disease status adjacent normal race aa liver',\n",
       "  'disease status hcc race liver',\n",
       "  'human'),\n",
       " 'GSE204665-10-vs-11-human up': (['The unexpected role of HBx in UC'],\n",
       "  'Ulcerative colitis is a chronic inflammatory disorder for which a definitive cure is still missing. This is characterized by an overwhelming inflammatory milieu in the colonic tract where a composite set of immune and non-immune cells orchestrate its pathogenesis. Over the last years, a growing body of evidence has been pinpointing gut virome dysbiosis as underlying its progression. Nonetheless, its role during the early phases of chronic inflammation is far from being fully defined. Here we show the gut virome-associated Hepatitis B virus protein X, most likely acquired after an event of zoonotic spillover, to be associated with the early stages of ulcerative colitis and to induce colonic inflammation in mice. It acts as a transcriptional regulator in epithelial cells, provoking barrier leakage and altering mucosal immunity at the level of both innate and adaptive immunity. This study paves the way to the comprehension of the aetiopathogenesis of intestinal inflammation and encourages further investigations of the virome as a trigger also in other scenarios. Moreover, it provides a brand-new standpoint that looks at the virome as a target for tailored treatments, blocking the early phases of chronic inflammation and possibly leading to better disease management.',\n",
       "  'post facs t4 lpmc colon cd4 cells',\n",
       "  'post facs en lpmc colon endothelium',\n",
       "  'human'),\n",
       " 'GSE211386-3-vs-2-mouse up': (['The transcriptome alteration in tumoral NK cells in response to  dual or monotherapy of STING agonist and IL-35 blocking antibodies'],\n",
       "  'To investigate the role of STING agonist and IL-35 blocking antibodies in the regulation of NK cell function, tumor-infiltrated NK cells were flow cytometrically separated from control (IgG), cGAMP, anti-IL-35, or combination treated groups',\n",
       "  'ctrl1 cell line nk cells wt igg time day 21',\n",
       "  'combine4 cell line nk cells wt cgamp il 35 blocking antibodies time day 21',\n",
       "  'mouse'),\n",
       " 'GSE85520-2-vs-0-mouse up': (['IRF8 transcription factor controls survival and function of terminally differentiated conventional and plasmacytoid dendritic cells respectively'],\n",
       "  'Interferon regulatory factor-8 (IRF8) has been proposed to be essential for development of monocytes, plasmacytoid dendritic cells (pDCs) and type 1 conventional dendritic cells (cDC1s) and remains highly expressed in differentiated DCs. Transcription factors that are required to maintain the identity of terminally differentiated cells are designated “terminal selectors”. Using BM chimeras, conditional Irf8fl/fl mice and various promotors to target Cre recombinase to different stages of monocyte and DC development, we have identified IRF8 as a terminal selector of the cDC1 lineage controlling survival. In monocytes, IRF8 was necessary during early but not late development. Complete or late deletion of IRF8 had no effect on pDC development or survival but altered their phenotype and gene expression profile leading to increased T cell stimulatory function but decreased type 1 interferon production. Thus, IRF8 differentially controls the survival and function of terminally differentiated monocytes, cDC1s and pDCs.',\n",
       "  'cdc1 wt strain c57bl/6 spleen state irf8fl/fl steady cd24+ conventional dendritic cells (cdc1s) cdc1s mice',\n",
       "  'pdc wt strain c57bl/6 spleen state irf8fl/fl steady plasmacytoid dendritic cells pdcs mice',\n",
       "  'mouse'),\n",
       " 'GSE114668-1-vs-2-mouse up': (['Intestinal and thymic tuft cell RNA-seq'],\n",
       "  'The goal of the study was to sequence mRNA from tuft cells (identified as CD45-;EpCAM+;IL-25+ using Flare25 reporter mice) in the epithelia of thymus and small intestine. As a control, non-tuft epithelial cells (CD45-;EpCAM+;IL-25-) were also isolated. The data were used to compare tuft cell markers between the small intestine and thymus.',\n",
       "  'epi si 1 strain c57bl6.il25(flare25/flare25) age 12 weeks sex mouse status naã¯ number mice small intestine total epithelium sorting gate dapi cd45 epcam+ flare25 cells sorted',\n",
       "  'tuft si strain c57bl6.il25(flare25/flare25) age 12 weeks sex mouse status naã¯ number mice 1 small intestine cells sorting gate dapi cd45 epcam+ flare25+ sorted epithelium',\n",
       "  'mouse'),\n",
       " 'GSE114067-4-vs-6-mouse up': (['Tissue tuft cell mRNA sequencing'],\n",
       "  'The goal of the study was to sequence mRNA from tuft cells (identified as CD45-;EpCAM+;IL-25+ using Flare25 reporter mice) in the epithelia of 5 different murine tissues: trachea, thymus, colon, gall baldder, small intestine. As a control, non-tuft epithelial cells (CD45-;EpCAM+;IL-25-) were also isolated. The data were used to identify differentially expressed genes across tuft cells from different tissues and to define a transcriptional tuft cell \"signature\" that is common to all tissues analyzed but absent in the non-tuft epithelial cells.',\n",
       "  'e si 1 strain c57bl6.il25(flare25/flare25) age 12 weeks sex mouse status naive small intestine number mice total epithelium ( tuft cells) sorting gate dapi cd45 epcam+ flare25 cells sorted',\n",
       "  'si strain c57bl6.il25(flare25/flare25) age 12 weeks sex mouse status naive small intestine number mice 1 tuft cells sorting gate dapi cd45 epcam+ flare25+ sorted epithelium',\n",
       "  'mouse'),\n",
       " 'GSE185841-1-vs-3-human up': (['SUMOylation inhibitor TAK-981 potentiates rituximab activity by Type I IFN-dependent macrophage and NK cell stimulation'],\n",
       "  'SUMOylation is a reversible post-translational modification that has been implicated in the regulation of various cellular processes including inflammatory responses and expression of Type I interferons (IFN1). In this report, we have explored the activity of the selective small molecule SUMOylation inhibitor TAK-981 in promoting antitumor innate immune responses. We demonstrate that treatment with TAK-981 results in IFN1-dependent macrophage and NK cell activation, promoting macrophage phagocytosis and NK cell cytotoxicity in ex vivo assays. Furthermore, pre-treatment with TAK-981 enhanced macrophage phagocytosis or NK cell cytotoxicity against CD20-positive target cells in combination with the anti-CD20 antibody rituximab. In vivo studies demonstrated synergistic antitumor activity of TAK-981 and rituximab in CD20-positive lymphoma xenograft models. TAK-981 is currently being studied in phase 1 clinical trials (NCT03648372, NCT04074330, NCT04776018, and NCT04381650) for the treatment of patients with lymphomas and solid tumors.',\n",
       "  'donor4 dmso human monocyte derived macrophages time 24 hours',\n",
       "  'donor4 tak981 human monocyte derived macrophages 1um time 24 hours',\n",
       "  'human'),\n",
       " 'GSE148837-2-vs-1-human up': (['Single-cell profiling reveals Colonic CD8 Topography in IBD'],\n",
       "  'Aims: Single-cell transcriptomic analysis of CD8 T cells in colonic tissue in Health and Ulcerative Colitis',\n",
       "  'sample feature barcoding disease mixed uc control pool colon cd8 cells',\n",
       "  'sample gene expression disease mixed uc control pool colon cd8 cells',\n",
       "  'human'),\n",
       " 'GSE126879-6-vs-8-mouse up': (['Suppressive function of MDSC is under bidirectional control of intrinsic and immunotherapy-induced type I IFN in pancreatic cancer'],\n",
       "  'Tumors entertain a highly immunosuppressive network by merging features of regulatory cytokines and cell populations to evade immune control. Myeloid-derived suppressor cells (MDSC) comprise populations of immature myeloid cells in tumor-bearing hosts with a highly immunosuppressive capacity. We could previously identify RIG-I-like helicases (RLH) as targets for the immunotherapy of pancreatic cancer inducing immunogenic tumor cell death and type I interferons (IFN), as key mediators linking innate with adaptive immunity. Here, we show that autochthonous type I IFN signaling is critical for MDSC to gain suppressive function. In contrast, RNA sequencing of MDSC from RLH-ligand-treated mice showed an IFN-driven gene signature and a switch from a M2/G2- towards a M1/G1-polarized phenotype. Functional assays confirmed the loss of the T cell suppressive capacity. Our study identifies a dual role of type I IFN signaling in MDSC´s suppressive function, and RLH-mediated tumor control.',\n",
       "  'spl cell mdsc organ spleen untreated splenic',\n",
       "  'spl pmn cell mdsc organ spleen poly( c)c splenic',\n",
       "  'mouse'),\n",
       " 'GSE146158-4-vs-3-mouse up': (['High-throughput sequencing Transcriptomes Analysis of WT and Kcnj2 KO mouse BMDMs treated with LPS or kir2.1 inhibitor'],\n",
       "  'Purpose:Mouse BMDM is the universal cell type of studying innate immunity.This study was to analyze LPS induced innate immune response and the relationship between the inward rectifier potassium channel Kir2.1 and  LPS induced innate immune respone',\n",
       "  'bmdms strain c57bl/6 wt control group age post natal week 8 bmdm',\n",
       "  'bmdms strain c57bl/6 wt lps group age post natal week 8 bmdm',\n",
       "  'mouse'),\n",
       " 'GSE166489-5-vs-2-human up': (['Immune dysregulation and autoreactivity correlate with disease severity in SARS-CoV-2-associated multisystem inflammatory syndrome in children'],\n",
       "  'Multisystem inflammatory syndrome in children (MIS-C) is a life-threatening post-infectious complication occurring unpredictably weeks after mild or asymptomatic SARS-CoV2 infection in otherwise healthy children. Here, we define immune abnormalities in MIS-C compared to adult COVID-19 and pediatric/adult healthy controls using single-cell RNA sequencing, antigen receptor repertoire analysis, unbiased serum proteomics, and in vitro assays. Despite no evidence of active infection, we uncover elevated S100A-family alarmins in myeloid cells and marked enrichment of serum proteins that map to myeloid cells and pathways including cytokines, complement/coagulation, and fluid shear stress in MIS-C patients. Moreover, NK and CD8 T cell cytotoxicity genes are elevated, and plasmablasts harboring IgG1 and IgG3 are expanded. Consistently, we detect elevated binding of serum IgG from severe MIS-C patients to activated human cardiac microvascular endothelial cells in culture. Thus, we define immunopathology features of MIS-C with implications for predicting and managing this SARS-CoV2-induced critical illness in children.',\n",
       "  'pbmc adult healthy donor gex adt disease state subtype periperal blood mononuclear cells',\n",
       "  'pbmc mis c severe bcr disease state multisystem inflammatory syndrome children (mis c) subtype periperal blood mononuclear cells',\n",
       "  'human'),\n",
       " 'GSE100745-1-vs-3-human dn': (['RNA-seq analysis of YFV-17D specific and total naive CD8 T cells in humans'],\n",
       "  'These 14 samples originate from peripheral blood mononuclear cells (PBMC) from donors at different times after they were vaccinated with the YF-17D vaccine.',\n",
       "  'd14 donor day 14 post yfv 17d sorted a2 ns4b214 tetramer+ cd8 cells pbmc',\n",
       "  'donor day post yfv 17d sorted a2 ns4b214 tetramer+ cd8 cells pbmc',\n",
       "  'human'),\n",
       " 'GSE166238-5-vs-6-human dn': (['Transcriptomes of human monocytes exposed to oxLDL ex vivo'],\n",
       "  'Innate immune memory is the phenomenon whereby innate immune cells such as monocytes or macrophages undergo functional reprogramming after exposure to microbial components such as Beta glucan or the BCG vaccine. Here we characterize the transcriptional events following oxidized low-density lipoprotein (oxLDL) duced trained innate immunity using RNA-seq data.',\n",
       "  'd6 rpmi lps monocyte protocol macrophage exposed media 6 days 4 hours donor adult human peripheral blood',\n",
       "  'd6 oxldl monocyte protocol trained macrophage exposed 24 hours rpmi media 5 days donor adult human peripheral blood',\n",
       "  'human'),\n",
       " 'GSE115395-2-vs-5-human up': (['RNA-Seq analysis of prostate cancer cell line LNCaP treated with vehicle, androgen, androgen and IMTPPE, androgen and JJ-(+)-450, androgen and JJ-(-)450, androgen and enzalutamide'],\n",
       "  'RNA-Seq analysis of prostate cancer cell line LNCaP treated with vehicle (C), androgen (R), androgen and IMTPPE (R + IMTPPE), androgen and JJ-(+)-450 (androgen + (-)450), androgen and JJ-(-)450 (androgen + (-)450), androgen and enzalutamide (androgen +Enz).',\n",
       "  'control illumina truseq stranded mrna c] prostate epithelial passages 10 agent vehicle cell line lncap derived metastatic site left supraclavicular lymph node',\n",
       "  'r (androgen) illumina truseq stranded mrna r] prostate epithelial passages 10 agent androgen cell line lncap derived metastatic site left supraclavicular lymph node',\n",
       "  'human'),\n",
       " 'GSE118373-0-vs-1-human dn': (['High-throughput sequencing of the human hepatic progenitor cell niche in PSC and HCV.'],\n",
       "  'This study identified hepatic progenitor cell (HPC) niche-associated signalling pathways relevant in different chronic liver diseases using a high-throughput sequencing approach. The HPC niche was isolated using laser microdissection from patient samples diagnosed with hepatitis C virus (HCV) or primary sclerosing cholangitis (PSC), as models for hepatocellular or biliary regeneration.  Differentially expressed genes in the HPC niche of PSC and HCV correlated to pathways involved in immune signalling, fibrogenesis and angiogenesis.',\n",
       "  'hcv hep disease state hepatitis c hepatocytes',\n",
       "  'psc hpc disease state primary sclerosing cholangitis hepatic progenitor cells',\n",
       "  'human'),\n",
       " 'GSE210497-13-vs-9-mouse dn': ([\"SARS-CoV-2 pathogenesis mediated by E protein's PBM was prevented by modulators of CFTR function\"],\n",
       "  'The three human deadly coronaviruses (CoVs) (SARS-CoV, MERS-CoV and SARS-CoV-2) have infected around eight thousand, two thousand six hundred and 262 million people so far (December 2021), causing the death of 10%, 37% and 2% of them, respectively, so their implication in health is very important. These CoVs have proteins with a PBM motif that binds to PDZ cell domains. PDZ domains are found in more than 400 cellular proteins, therefore, viruses with PBM motifs have a high potential to modify cell behavior. It was studied the implication of the E protein PBM motif of various virulent or attenuated human CoVs (hCoVs) in the pathogenesis induced by these viruses. Variants of SARS-CoV, MERS-CoV and SARS-CoV-2 that lack the E protein PBM have been generated by reverse genetics and their pathogenicity has been analyzed in mice. The PBM motifs of these three hCoVs have been shown to be virulence factors and participate in their replication. Furthermore, a collection of SARS-CoV mutants has been constructed in which the E protein PBM domain was replaced by the one derived from virulent or attenuated hCoVs, and their virulence was analyzed. A virulence gradient was observed, depending on whether the E protein PBM domain was derived from an attenuated or virulent hCoV. The gene expression patterns associated with the different PBM motifs in lungs of mice infected with SARS-CoV was analyzed by deep sequencing of the mRNAs expressed in lungs of infected mice, and it was observed that the E protein PBM motif of SARS-CoV and SARS-CoV-2 dysregulates the expression of genes related to ion transport and cell homeostasis. Specifically, a decrease of the mRNA expression of the cystic fibrosis transmembrane conductance regulator (CFTR), which is essential for edema resolution, was observed. The reduction of CFTR mRNA levels was associated with edema accumulation in lungs of mice infected with SARS-CoV and SARS-CoV-2. The effect of compounds that modulate the expression and activity of CFTR on the pathogenesis and replication of SARS-CoV-2 was studied and it was observed that these compounds drastically reduced the production of SARS-CoV-2 in cell culture and protect against SARS-CoV-2 pathogenesis. These results showed the high relevance of the PBM motif in the replication and virulence of CoVs, and have allowed the identification of cellular targets for the selection of antivirals.',\n",
       "  'wt 2dpã¬ [e93 4 lung strain balb/c time 2dpi infected wild type sars cov',\n",
       "  'mock 4dpã¬ [e93 2 lung strain balb/c time 4dpi infected',\n",
       "  'mouse'),\n",
       " 'GSE58596,GSE58597-0-vs-2-human up': (['Global Transcriptome Analysis and Enhancer Landscape of Human Primary T Follicular Helper and T Effector Lymphocytes (RNA-Seq)'],\n",
       "  'T follicular helper (Tfh) cells are a subset of CD4+ T helper (Th) cells that migrate into germinal centers and promote B cell maturation into memory B and plasma cells. Tfh cells are necessary for promotion of protective humoral immunity following pathogen challenge, but when aberrantly regulated, drive pathogenic antibody formation in autoimmunity and undergo neoplastic transformation in angioimmunoblastic T-cell lymphoma and other primary cutaneous T-cell lymphomas. Limited information is available on the expression and regulation of genes in human Tfh cells. Using a fluorescence activated cell sorting-based strategy, we obtained primary Tfh and non-Tfh T effector (Teff) cells from tonsils and prepared genome-wide maps of active, intermediate, and poised enhancers determined by ChIP-seq, with parallel transcriptome analyses determined by RNA-seq. Tfh cell enhancers were enriched near genes highly expressed in lymphoid cells or involved in lymphoid cell function, with many mapping to sites previously associated with autoimmune disease in genome-wide association studies. A group of active enhancers unique to Tfh cells associated with differentially expressed genes was identified. Fragments from these regions directed expression in reporter gene assays. These data provide a significant resource for studies of T lymphocyte development and differentiation and normal and perturbed Tfh cell function.',\n",
       "  'naive naã¯ human tonsil cell surface markers cd4+cd45ra+tcrb+psgl 1hi',\n",
       "  'tfh human tonsil cell surface markers cd4+cd45ra tcrb+pd 1hicxcr5hipsgl 1lo',\n",
       "  'human'),\n",
       " 'GSE110146-3-vs-7-human dn': (['Transcriptomic-based functional characterization of adverse events following lymphatic filariasis treatment'],\n",
       "  'Intro: Lymphatic filariasis (LF) is a neglected tropical disease caused by the nematode parasite Wuchereria bancrofti. The primary tool used by the Global Program to Eliminate LF is mass drug administration (MDA), and some 500 million people take the medications each year. Mild to moderate adverse events (AEs) are common after LF treatment, and these pose a challenge for the LF elimination program. To better understand the pathogenesis of AEs, we studied patients from a LF treatment trial in Côte d’Ivoire.',\n",
       "  'post buffy coat adverse event moderate pre age sex male village microfilaria/ml arm',\n",
       "  'post buffy coat adverse event none pre age sex male village microfilaria/ml arm alb',\n",
       "  'human'),\n",
       " 'GSE180862-5-vs-6-mouse up': (['Systems spatiotemporal dynamics of traumatic brain injury at single cell resolution'],\n",
       "  'The etiology of mild traumatic brain injury (mTBI) remains elusive due to the tissue and cellular heterogeneity of the affected brain regions that underlie cognitive impairments and subsequent neurological disorders. This complexity is further exacerbated by disrupted circuits within and between cell populations across brain regions and the periphery, which occur at different timescales and in spatial domains. We profiled three tissues (hippocampus, frontal cortex, and blood leukocytes) at the acute (24hr) and chronic (7days) phases of mTBI at single cell resolution and demonstrated that the coordinated gene expression patterns across cell types were disrupted and re-organized by TBI at different timescales with distinct regional and cellular patterns. Gene expression-based network modeling identified astrocytes as a key regulator of the cell-cell coordination following mTBI in both hippocampus and frontal cortex across timepoints, and mt-Rnr2, which encodes the mitochondrial peptide humanin, as a potential target for intervention based on its broad regional and dynamic dysregulation following mTBI. Treatment of a murine mTBI model with humanin reversed cognitive impairment caused by mTBI through the restoration of metabolic pathways within astrocytes. Our results offer a systems-level understanding of the dynamic and spatial regulation of gene programs by mTBI and pinpoint key target genes, pathways, and cell circuits that are amenable to therapeutics.',\n",
       "  'strain/backgorund c57bl/6 tbi time cortex age 10 weeks',\n",
       "  'strain/backgorund c57bl/6 sham time 24 hours blood age 10 weeks',\n",
       "  'mouse'),\n",
       " 'GSE211190-3-vs-2-human up': (['Vascular Surgery Patients with Elevated Neutrophil Lymphocyte Ratios Have Downregulated Neutrophil Complement RNA Expression'],\n",
       "  'Elevated NLR (neutrophil-lymphocyte ratio) in elective vascular surgery (EVS) patients is associated with increased mortality independent of perioperative surgical outcome. To understand why high NLR is associated with higher mortality, we investigated neutrophil and lymphocyte transcriptome expression in patients undergoing EVS.',\n",
       "  'lymphocyte control [nlr l blood nl ratio',\n",
       "  'neutrophil nlr low [nlr n blood nl ratio',\n",
       "  'human'),\n",
       " 'GSE220656-2-vs-4-mouse dn': (['Single cell RNA sequencing reveals the suppressive effect of PPP1R15A inhibitor Sephin1 in antitumor immunity'],\n",
       "  'Sephin1 is the selective inhibitor of Protein Phosphatase 1 Regulatory Subunit 15A (PPP1R15A), which is an important factor in the integrated stress response (ISR) of mammalian cells. Sephin1 can promote ISR by inhibiting the dephosphorylation of eIF2α, and preventing neurodegenerative disease occurrence in mice. However, we found that Sephin1 plays a protumorigenic role in mouse tumor models by suppressing the antitumor immune activities, which was highly relevant to the function of PPP1R15A. To fully understand the mechanism of Sephin1 to the antitumor immunity, we designed this experiment to explore the function of PPP1R15A and Sephin1 in the single-cell expression level. To understand effect of injection of Sephin1 to the immune cells in blood and tumor microenvironment in mice, we injected Sephin1 to mice intraperitoneally, and then subcutaneously inoculated with B16F1 cells. Blood before/after the tumor development and tumor tissues were collected and immune cells were isolated for single-cell sequencing and bioinformatics analysis afterwards. We found that in C57BL/6 mice, Sephin1 treatment could lead to higher levels of ISR activity and lower levels of antitumor immune activities. Specifically, Sephin1 treatment caused reductions in several tumor-killing immune cell populations, including CD8+ T cells, NK cells and NKT cells. The expression levels of cytotoxicity-related genes in lymphocytes were altered upon Sephin1 treatment. Additionally, TCR repertoire analysis demonstrated that the clonal expansion of tumor-specific T cells was inhibited by Sephin1, resulting in a compromised antitumor immune response. A special TCR+ macrophage subtype was identified to be significantly depleted upon Sephin1 treatment, implying that this type of macrophage plays a key antitumor role in the tumor microenvironment. The effects of Sephin1 were also verified in in vitro experiments and studies of other tumor cell lines. Taken together, these data suggest that inhibiting PPP1R15A promotes tumorigenesis by inducing adverse immune suppression. PPP1R15A has the potential to be an effective target for tumor therapy. Besides, a macrophage subtype, the TCR+ macrophages, played an important role in the anti-tumor immunity and can also be significantly inhibited by Sephin1.',\n",
       "  'blood sephin1 day replicate scrnaseq immune cells peripheral strain c57bl/6slac age 10 weeks old',\n",
       "  'blood sephin1 day replicate scrnaseq tcr enriched immune cells peripheral strain c57bl/6slac age 10 weeks old',\n",
       "  'mouse'),\n",
       " 'GSE246690-6-vs-0-human dn': (['Transcriptomic analysis of therapy-induced senescent and diapause-like cancer cells'],\n",
       "  'RNA-sequencing of proliferating, doxorubicin-induced senescent, palbociclib (CDK4/6i)-induced senescent and INK128-induced diapause-like human melanoma SK-Mel-103 cells and human NSCLC A549',\n",
       "  'a549 ctr cell line replicate',\n",
       "  'a549 pb cell line replicate',\n",
       "  'human'),\n",
       " 'GSE79951-2-vs-0-mouse up': (['Topoisomerase 1 Regulates Gene Expression in Neurons through Cleavage Complex-Dependent and -Independent Mechanisms'],\n",
       "  'Topoisomerase 1 (TOP1) inhibitors, including camptothecin and topotecan, covalently trap TOP1 on DNA, creating cleavage complexes (cc’s) that must be resolved before gene transcription and DNA replication can proceed. We previously found that topotecan reduces the expression of long (>100 kb) genes and unsilences the paternal allele of Ube3a in neurons. Here, we sought to evaluate overlap between TOP1cc-dependent and -independent gene regulation in neurons. To do this, we utilized Top1 conditional knockout mice, Top1 knockdown, the CRISPR-Cas9 system to delete Top1, TOP1 catalytic inhibitors that do not generate TOP1cc’s, and a TOP1 mutation (T718A) that stabilizes TOP1cc’s. We found that topotecan treatment significantly alters the expression of many more genes, including long neuronal genes, immediate early genes, and paternal Ube3a, when compared to Top1 deletion. Our data show that topotecan has a stronger effect on neuronal transcription than Top1 deletion, and identifies TOP1cc-dependent and -independent contributions to gene expression.',\n",
       "  'wt topot strain c57bl6 wild type cortical neurons topotecan',\n",
       "  'wt veh strain c57bl6 wild type cortical neurons vehicle',\n",
       "  'mouse'),\n",
       " 'GSE226238-2-vs-13-human dn': (['Profiling immunological phenotypes in individuals during the first year after traumatic spinal cord injury (SCI): a longitudinal analysis'],\n",
       "  'Individuals with SCI are severely affected by immune system changes, resulting in a\\xa0SCI-induced immune deficiency syndrome (SCI-IDS).\\xa0While recent data support that acute SCI-IDS symptoms differ from those present in the chronic phase, only limited data in humans is available. To characterize dynamic molecular and cellular immune phenotypes over the first year, we assess RNA (bulk-RNA sequencing), protein, and flow cytometry (FACS) profiles of blood samples from 12 individuals with SCI at 0-3 days and at 3, 6, and 12 months post injury (MPI) compared to 23 uninjured individuals (controls). We identified 967 differentially expressed (DE) genes in individuals with SCI (FDR<0.001) compared to controls. Within the first 6 MPI we detected a reduced expression of NK cell genes, consistent with reduced frequencies of CD56bright, CD56dim\\xa0NK cells present at 12 MPI. Over 6MPI, we observed increased and prolonged expression of genes associated with inflammation (e.g. HMGB1, Toll-like receptor signaling) and expanded frequencies of monocytes acutely. Canonical T-cell related DE genes (e.g. FOXP3, TCF7, CD4)\\xa0were upregulated during the first 6 MPI\\xa0and increased frequencies of activated T cells at 3-12 MPI were observed. Neurological injury severity was reflected in distinct whole blood gene expression profiles at any time after SCI, indicating a persistent ‘neurogenic’ imprint. Overall, 2876 DE genes emerge when comparing profiles of motor complete to motor incomplete SCI (ANOVA, FDR<0.05), including\\xa0genes related to neutrophils, inflammation, and infection.\\xa0In summary, we identify dynamic SCI-IDS determinants in humans, including molecular and cellular profiles, which may provide potential targets to reduce inflammation, improve immunity, or serve as candidate biomarkers of injury severity.',\n",
       "  'sci acute whole blood group',\n",
       "  'sci 12mpi whole blood group',\n",
       "  'human'),\n",
       " 'GSE124975-2-vs-3-human up': (['Salinomycin induces expression of ER and Golgi apparatus related genes in EMT cell model'],\n",
       "  'Epithelial-to-mesenchymal transition (EMT) gives rise to cells with properties similar to cancer stem cells (CSCs) that drive tumor metastasis. Recently, a screening of a large compound library on a breast EMT model has identified salinomycin, a K+/H+ ionophore, as a highly selective drug towards CSCs. We used the same EMT model to show that salinomycin targets Golgi apparatus. We have performed RNA-seq analysis on HMLE-Twist and HMLE-pBp cells (EMT and non-EMT) that were either mock treated or treated for 24h with micro molar concentration (0.2uM) of salinomycin. Salinomycin induced expression of genes enriched by known ER and Golgi stressors.',\n",
       "  'cell line hmle twist breast phenotype emt mock',\n",
       "  'cell line hmle pbp breast phenotype nonemt salinomycin',\n",
       "  'human'),\n",
       " 'GSE123541-0-vs-5-human dn': (['Genome-wide RNA-seq and ATAC-seq analysis of an in vivo human immune response to acute Zika virus infection'],\n",
       "  'A longitudinal systems immunologic investigation of acute Zika virus infection in an individual infected with ZIKV while traveling to Caracas, Venezuela during the zika virus epidemic. Includes longitudinal RNA-seq and ATAC-seq characterization of innate and adaptive immune responses during an acute ZIKV infection.',\n",
       "  'cd14dimcd16pos subject status individual naat confirmed acute zika virus infection pbmc cell subtype monocyte time post onset symptoms days',\n",
       "  'plasmacytoiddc subject status individual naat confirmed acute zika virus infection pbmc cell subtype plasmacytoid dendritic time post onset symptoms days',\n",
       "  'human'),\n",
       " 'GSE85223-2-vs-3-human up': (['Hepatic differentiation of liver organoids'],\n",
       "  'We analyzed gene expression profiles of self-organizing, multi-cellular, 3D liver organoids derived by co-culture of induced Pluripotent Stem Cell and stromal progenitors. We report the RNA-seq results of liver organoid at day0, day2, day4, day6 of co-culture. We also report RNA-seq results of constituent of the liver organoid, which are human iPSC at hepatic specification stage, human Mesenchymal stem cells derived from bone marrow, human umbilical vein endothelial cell. As controls, we also report RNS-seq results of un-differentiated human iPSC, human iPSC at definitive endoderm stage, human liver tissue, and primary cultured human hepatocytes isolated from unused donor livers.',\n",
       "  'cell population huvec time na human umbilical vein endothelial cells (huvec)',\n",
       "  'cell population baseline ipsc time na baseline/undifferentiated',\n",
       "  'human'),\n",
       " 'GSE123541-3-vs-10-human up': (['Genome-wide RNA-seq and ATAC-seq analysis of an in vivo human immune response to acute Zika virus infection'],\n",
       "  'A longitudinal systems immunologic investigation of acute Zika virus infection in an individual infected with ZIKV while traveling to Caracas, Venezuela during the zika virus epidemic. Includes longitudinal RNA-seq and ATAC-seq characterization of innate and adaptive immune responses during an acute ZIKV infection.',\n",
       "  'cd4tcell subject status individual naat confirmed acute zika virus infection pbmc cell subtype cd4 time post onset symptoms days',\n",
       "  'subject status individual naat confirmed acute zika virus infection pbmc cell subtype monocyte time post onset symptoms days',\n",
       "  'human'),\n",
       " 'GSE49379-0-vs-1-human up': (['Large-scale multi-species survey of metabolome and lipidome'],\n",
       "  'This dataset was generated with the goal of comparative study of gene expression in three brain regions and two non-neural tissues of humans, chimpanzees, macaque monkeys and mice. Using this dataset, we performed studies of gene expression and gene splicing evolution across species and search of tissue-specific gene expression and splicing patterns. We also used the gene expression information of genes encoding metabolic enzymes in this dataset to support a larger comparative study of metabolome evolution in the same set of tissues and species.',\n",
       "  'strain/background c57bl/6 kidney sex age (years)',\n",
       "  'strain/background c57bl/6 primary visual cortex sex age (years) vc',\n",
       "  'human'),\n",
       " 'GSE164687,GSE175880-8-vs-3-human up': (['CD66b-CD64dimCD115- cells in the human bone marrow represent neutrophil-committed progenitors'],\n",
       "  'This SuperSeries is composed of the SubSeries listed below.',\n",
       "  'neutrophil committed progenitors bone marrow donor id',\n",
       "  'sn segmented neutrophils bone marrow donor id',\n",
       "  'human'),\n",
       " 'GSE139549,GSE139552-2-vs-3-human dn': (['Fully defined human PSC-derived microglia and tri-culture system reveals cell type specific potentiation of complement C3 production in a model of Alzheimer’s disease [smarter-seq]'],\n",
       "  'Aberrant inflammation in the central nervous system (CNS) has been implicated as a major player in the pathogenesis of human neurodegenerative disease. However, the specific contribution of each cell type to the neuroinflammatory axis in vivo remains unclear with species-specific differences in key signaling pathways further complicating the challenge. To assemble a fully human platform to study neuroinflammation, we first developed a novel approach to derive microglia from human pluripotent stem cells (hPSCs) that faithfully recapitulates microglial ontogeny as validated by single cell RNA-sequencing and stage-specific mapping onto datasets of developing mouse microglia. Using these cells, we build the first defined and completely hPSC-derived tri-culture system containing pure populations of hPSC-derived microglia, astrocytes, and neurons to dissect cellular crosstalk along the neuroinflammatory axis in vitro. We next used the tri-culture system to model neuroinflammation in Alzheimer’s Disease using hPSCs harboring the APPSWE+/+ mutation and their isogenic control. Our data reveal that production of complement C3, a protein that is increased under inflammatory conditions and implicated in synaptic loss, is potentiated under tri-culture conditions, and is further enhanced in APPSWE+/+ tri-cultures. Using cell type-specific ablation studies, we report that C3 potentiation is due to the presence of a neuroinflammatory loop in which microglia are the key initiators that activate reciprocal signaling with astrocytes to produce excess C3. Our study defines the major cellular players contributing to increased C3 in AD and presents a broadly applicable platform to study neuroinflammation in human disease.',\n",
       "  'human primary cell line/ source postmortem brain microglia replicate biological',\n",
       "  'neg cell line/ source human es line hpsc neuron protocol derived neurons sorted co cultures day 40 vitro culture. replicate biological cultured',\n",
       "  'human'),\n",
       " 'GSE99068-1-vs-2-human dn': (['IL-6/Stat3-Dependent Induction of Distinct, Obesity-Associated Natural Killer Cells Deteriorates Energy and Glucose Homeostasis'],\n",
       "  'Natural killer (NK) cells contribute to the development of obesity-associated insulin resistance. We demonstrate that in mice obesity promotes the expansion of interleukin-6 receptor (IL6Ra)-expressing NK cells, which also express a number of other myeloid lineage genes such as the colony-stimulating-factor 1 receptor (Csf1r). Selective ablation of Csf1r- expressing NK cells prevents obesity and insulin resistance. Moreover, conditional inactivation of IL6Ra or Stat3 in NK cells limits obesity-associated formation of myeloid signature NK cells, protects from obesity, insulin resistance and obesity-associated inflammation. Also in humans IL6Ra+ NK cells increase in obesity, correlate with markers of systemic low-grade inflammation and their gene expression profile overlaps with characteristic gene sets of NK cells in obese mice. Collectively, we demonstrate that obesity-associated inflammation and metabolic disturbances depend on IL-6/Stat3-dependent formation of distinct NK cells, which may provide a novel target for the treatment of obesity, metaflammation-associated pathologies and diabetes.',\n",
       "  'hu lean nk il6ra+ replicate il6ra expression positive organ pbmc bmi sorted cells',\n",
       "  'hu lean nk il6ra replicate expression negative organ pbmc bmi sorted cells',\n",
       "  'human'),\n",
       " 'GSE166238-2-vs-6-human dn': (['Transcriptomes of human monocytes exposed to oxLDL ex vivo'],\n",
       "  'Innate immune memory is the phenomenon whereby innate immune cells such as monocytes or macrophages undergo functional reprogramming after exposure to microbial components such as Beta glucan or the BCG vaccine. Here we characterize the transcriptional events following oxidized low-density lipoprotein (oxLDL) duced trained innate immunity using RNA-seq data.',\n",
       "  'd6 monocyte protocol trained macrophage exposed oxldl 24 hours rpmi media 5 days lps 4 donor adult human peripheral blood',\n",
       "  'd6 oxldl monocyte protocol trained macrophage exposed 24 hours rpmi media 5 days donor adult human peripheral blood',\n",
       "  'human'),\n",
       " 'GSE210352-5-vs-6-mouse dn': (['Identification of a glycosphingolipid+ immune regulatory innate-like CD8 T-cell network in old mice'],\n",
       "  'The reciprocal decline in naive (TN) and increase in virtual (TVM) and true memory (TTM) CD8 T-cell frequencies upon aging is incompletely understood. We established the glycosphingolipid asialo-GM1 (AsGM1) marker present on >70% of TVM/TTM and 5-15% TN to target their immune-regulatory and homeostatic potential. We identified a novel AsGM1+ old CD8 TN-subset that was related to antigen-experienced AsGM1+ CD8 TTM and showed an initial immune-regulatory (Pdcd1, Il10) gene expression signature. The program for AsGM1+ CD8 T-subsets was imprinted genetically in young and old hematopoietic stem cells that reconstitute young and old donor-type immune systems in lymphophenic RAG1-/- mice within 16 weeks, respectively. However, antibody-mediated depletion of AsGM1+ T cells in old mice revealed a disturbed recovery of CD8 TVM resulting in a rejuvenated T-subset composition and T-cell priming capacity upon DNA-vaccination. Old CD8 TVM and IL10+ PD-1+ TTM exerted immune-regulatory functions and suppressed CD3/CD28-mediated activation of TN in vitro. Glycosphingolipids thus are attractive to identify and target novel T-subsets.',\n",
       "  'old dimer pos strain c57bl/6j age cd8+ cell subpopulation dimer+ spleen',\n",
       "  'old naive neg strain c57bl/6j age cd8+ cell subpopulation asgm1 spleen',\n",
       "  'mouse'),\n",
       " 'GSE75246-4-vs-6-mouse up': (['LPS and brain inflammation'],\n",
       "  'Normal mice were injected i.p. with LPS or saline. 24h later, perfused brains were dissociated. Cells were fixed, immunolabeled and FACS sorted. RNA was extracted from neuron, astrocyte, and microglial cell populations. Typical RIN=4-5 for neurons, 6-8 for astrocytes, and 5-7 for microglia. Typical RNA yields ~100ng for neurons, ~20ng for microglia, and ~10ng for astrocytes.',\n",
       "  'sample microglia lps brain rna sorted cells',\n",
       "  'sample neuron vehicle brain rna sorted cells',\n",
       "  'mouse')}"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "def gse_again(gseids):\n",
    "    gse_dict = {}\n",
    "    for gse in gseids:\n",
    "        if gseids[gse][2] == \"human\":\n",
    "            dt = gse_human_info\n",
    "        else:\n",
    "            dt = gse_mouse_info\n",
    "        title = gse_info[gseids[gse][0]][\"title\"] \n",
    "        summary = gse_info[gseids[gse][0]][\"summary\"][0]\n",
    "        cond1= dt[gseids[gse][1]][\"titles\"][gse.split(\"-\")[1]] \n",
    "        cond2 = dt[gseids[gse][1]][\"titles\"][gse.split(\"-\")[3]]\n",
    "        dir = gse.split(\"-\")[4].split(\" \")[0]\n",
    "        gse_dict[gse] = (title, summary, cond1,cond2, dir)\n",
    "    return gse_dict\n",
    "\n",
    "fin_gse = gse_again(df_rummageo_map)\n",
    "fin_gse\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "import time\n",
    "\n",
    "def get_enrichr_terms(genes):\n",
    "    ENRICHR_URL = 'https://maayanlab.cloud/Enrichr/'\n",
    "    endpoint = 'addList'\n",
    "    user_list_id = None\n",
    "    enrichr_libraries = ['WikiPathway_2023_Human', 'GWAS_Catalog_2023', \n",
    "                         'GO_Biological_Process_2023', 'MGI_Mammalian_Phenotype_Level_4_2021']\n",
    "    \n",
    "    \n",
    "    try:\n",
    "        gen = genes.split(\";\")\n",
    "        genes_string = '\\n'.join(gen).replace(\"'\", '')\n",
    "        response = requests.post(\n",
    "            ENRICHR_URL + endpoint,\n",
    "            files={'list': (None, genes_string), 'description': (None, '')}\n",
    "            # headers={'Content-Type': 'multipart/form-data'}\n",
    "        )\n",
    "        response.raise_for_status()\n",
    "        data = response.json()\n",
    "        user_list_id = data['userListId']\n",
    "    except requests.exceptions.RequestException as error:\n",
    "        print(f\"Error: {error}\")\n",
    "        return\n",
    "\n",
    "    enriched_terms = {}\n",
    "    enrichr_stats = {}\n",
    "\n",
    "    for enrichr_library in enrichr_libraries:\n",
    "        query_string = f'enrich?userListId={user_list_id}&backgroundType={enrichr_library}'\n",
    "        try:\n",
    "            response = requests.get(ENRICHR_URL + query_string, headers={'Accept': 'application/json'})\n",
    "            response.raise_for_status()\n",
    "            data = response.json()\n",
    "\n",
    "            enriched_terms[enrichr_library] = []\n",
    "            for term in data[enrichr_library][:3]:  # Limit to top 3 results\n",
    "                term_name = term[1]\n",
    "                enriched_terms[enrichr_library].append(term_name)\n",
    "                enrichr_stats[term_name] = term\n",
    "                enrichr_stats[term_name].append(enrichr_library)\n",
    "\n",
    "            time.sleep(0.5)  # Delay between requests\n",
    "\n",
    "        except requests.exceptions.RequestException as error:\n",
    "            print(f\"Error: {error}\")\n",
    "            return\n",
    "\n",
    "    return enriched_terms, enrichr_stats\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# import requests\n",
    "# import os\n",
    "# import json\n",
    "\n",
    "# def fetch_hypothesis(pm_title, user_desc: str, desct, desc, gse_title, gse_summary: str, cond1, cond2, species, term1: str,term2: str, enriched_terms: dict, enriched_stats: dict) -> str:\n",
    "#     # Define the system prompt\n",
    "#     system_prompt = (\n",
    "#         \"You are an AI hypothesis generator for RummagenexRummaGEO (gene sets from crossing Rummagene sets with RummaGEO sets). \"\n",
    "#         \"You should act as a biologist in hypothesizing why a high overlap may exist between a Rummagene set (which are sets from PubMed papers) and a RummaGEO set (automatically generated signatures from Gene Expression Omnibus).\"\n",
    "#     )\n",
    "\n",
    "#     # Build the enriched terms string\n",
    "#     enriched_terms_string = \"\"\n",
    "#     for library, terms in enriched_terms.items():\n",
    "#         enriched_terms_string += f\"{library}: {', '.join(terms)}\\n\"\n",
    "\n",
    "#     # Define the main prompt to be sent to the AI\n",
    "#     prompt = (\n",
    "#         f\"Here are two gene sets that highly overlap. The first is from a Rummagene set. \"\n",
    "#         f\"The second is a gene set automatically computed between two conditions in a study from the Gene Expression Omnibus (GEO). \"\n",
    "#         f\"Based upon the term name (formatted as condition 1 vs. condition 2) and the abstract of the GEO gene set, \"\n",
    "#         f\"and the abstract of the Rummagene gene set, please hypothesize about why these two gene sets have a significant high overlap. \"\n",
    "#         f\"You should mention both the summary of the RummaGEO gene set and the description of the Rummagene gene set in your hypothesis. \"\n",
    "#         f\"You will also be provided with enrichment results from the Enrichr database to help you generate your hypothesis, which shows \"\n",
    "#         f\"significantly overlapping functional terms from the overlapping genes of the two sets. \"\n",
    "#         f\"For each enrichment term that appears in your response, the term should appear in the exact form it was given to you \"\n",
    "#         f\"(do not exclude any words or characters from a term. For example, Complement And Coagulation Cascades WP558 should appear as \"\n",
    "#         f\"Complement And Coagulation Cascades WP558, not Complement And Coagulation Cascades). Also, please don't use quotes around the enriched term names. \"\n",
    "#         f\"Gene set term 1 from RummaGEO: {term2}\\n\"\n",
    "#         f'\"up\" or \"dn\" in this{term2} name indicates if the genes were upregulated or downregulated in {cond1} vs {cond2} conditions in the signature for species {species}.Please make sure to include this detail in your hypothesis.\\n'\n",
    "#         f\"title of study for gene set term 1: {gse_title}\\n\"\n",
    "#         f\"summary of study for gene set term 1: {gse_summary}\\n\"\n",
    "#         f\"Gene set term 2 from Rummagene: {term1}\\n\"\n",
    "#         f\"title of paper for gene set term 2: {pm_title}\\n\"\n",
    "#         f\"abstract of paper for gene set term 2: {user_desc}\\n\"\n",
    "#         f\"{desct}is the name of the table from the paper which the gene set comes from and {desc} is its description. Please include this detail if relevant\\n\"\n",
    "#         f\"Enriched Terms from overlapping genes of the two sets:\\n\"\n",
    "#         f\"{enriched_terms_string}\"\n",
    "#     )\n",
    "\n",
    "#     # Send the request to the OpenAI API\n",
    "#     response = requests.post(\n",
    "#         \"https://api.openai.com/v1/chat/completions\",\n",
    "#         headers={\n",
    "#             \"Content-Type\": \"application/json\",\n",
    "#             \"Authorization\": f\"Bearer {os.getenv('OPENAI_API_KEY')}\",\n",
    "#         },\n",
    "#         json={\n",
    "#             \"model\": \"gpt-4o\",\n",
    "#             \"messages\": [\n",
    "#                 {\"role\": \"system\", \"content\": system_prompt},\n",
    "#                 {\"role\": \"user\", \"content\": prompt}\n",
    "#             ],\n",
    "#             \"max_tokens\": 1000,\n",
    "#             \"temperature\": 0\n",
    "#         }\n",
    "#     )\n",
    "\n",
    "#     # Parse the response from the API\n",
    "#     response_json = response.json()\n",
    "#     hypothesis = response_json['choices'][0]['message']['content']\n",
    "\n",
    "#     # Replace enriched terms in the hypothesis with detailed text\n",
    "#     for term, stats in enriched_stats.items():\n",
    "#         if term in hypothesis:\n",
    "#             details = (\n",
    "#                 f\"Term: {term}\\tLibrary: {stats[9]}\\tRank: {stats[0]}\\tP-value: {float(stats[2]):.2e}\\tOdds Ratio: {float(stats[3]):.4f}\\n\"\n",
    "#             )\n",
    "#             hypothesis = hypothesis.replace(term, f\"{term} ({details})\")\n",
    "\n",
    "#     return hypothesis\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import requests\n",
    "import json\n",
    "from google.auth import jwt\n",
    "from google.auth.transport.requests import Request\n",
    "\n",
    "def fetch_hypothesis(pm_title, user_desc: str, desct, desc, gse_title, gse_summary: str, cond1, cond2, species, term1: str, term2: str, enriched_terms: dict, enriched_stats: dict) -> str:\n",
    "    # Define the system prompt\n",
    "    system_prompt = (\n",
    "        \"You are an AI hypothesis generator for RummagenexRummaGEO (gene sets from crossing Rummagene sets with RummaGEO sets). \"\n",
    "        \"You should act as a biologist in hypothesizing why a high overlap may exist between a Rummagene set (which are sets from PubMed papers) and a RummaGEO set (automatically generated signatures from Gene Expression Omnibus).\"\n",
    "    )\n",
    "\n",
    "    # Build the enriched terms string\n",
    "    enriched_terms_string = \"\"\n",
    "    for library, terms in enriched_terms.items():\n",
    "        enriched_terms_string += f\"{library}: {', '.join(terms)}\\n\"\n",
    "\n",
    "    # Define the main prompt to be sent to the AI\n",
    "    prompt = (\n",
    "        f\"Here are two gene sets that highly overlap. The first is from a Rummagene set. \"\n",
    "        f\"The second is a gene set automatically computed between two conditions in a study from the Gene Expression Omnibus (GEO). \"\n",
    "        f\"Based upon the term name (formatted as condition 1 vs. condition 2) and the abstract of the GEO gene set, \"\n",
    "        f\"and the abstract of the Rummagene gene set, please hypothesize about why these two gene sets have a significant high overlap. \"\n",
    "        f\"You should mention both the summary of the RummaGEO gene set and the description of the Rummagene gene set in your hypothesis. \"\n",
    "        f\"You will also be provided with enrichment results from the Enrichr database to help you generate your hypothesis, which shows \"\n",
    "        f\"significantly overlapping functional terms from the overlapping genes of the two sets. \"\n",
    "        f\"For each enrichment term that appears in your response, the term should appear in the exact form it was given to you \"\n",
    "        f\"(do not exclude any words or characters from a term. For example, Complement And Coagulation Cascades WP558 should appear as \"\n",
    "        f\"Complement And Coagulation Cascades WP558, not Complement And Coagulation Cascades). Also, please don't use quotes around the enriched term names. \"\n",
    "        f\"Gene set term 1 from RummaGEO: {term2}\\n\"\n",
    "        f'\"up\" or \"dn\" in this{term2} name indicates if the genes were upregulated or downregulated in {cond1} vs {cond2} conditions in the signature for species {species}.Please make sure to include this detail in your hypothesis.\\n'\n",
    "        f\"title of study for gene set term 1: {gse_title}\\n\"\n",
    "        f\"summary of study for gene set term 1: {gse_summary}\\n\"\n",
    "        f\"Gene set term 2 from Rummagene: {term1}\\n\"\n",
    "        f\"title of paper for gene set term 2: {pm_title}\\n\"\n",
    "        f\"abstract of paper for gene set term 2: {user_desc}\\n\"\n",
    "        f\"{desct} is the name of the table from the paper which the gene set comes from and {desc} is its description. Please include this detail if relevant\\n\"\n",
    "        f\"Enriched Terms from overlapping genes of the two sets:\\n\"\n",
    "        f\"{enriched_terms_string}\"\n",
    "    )\n",
    "\n",
    "    # Authenticate with the Gemini API\n",
    "    credentials = jwt.Credentials.from_service_account_file(\n",
    "        \"path/to/service_account.json\",\n",
    "        audience=\"https://gemini.googleapis.com\",\n",
    "    )\n",
    "    auth_request = Request()\n",
    "    credentials.refresh(auth_request)\n",
    "    \n",
    "    # Send the request to the Gemini API\n",
    "    response = requests.post(\n",
    "        \"https://gemini.googleapis.com/v1/models/gemini-model/complete\",\n",
    "        headers={\n",
    "            \"Content-Type\": \"application/json\",\n",
    "            \"Authorization\": f\"Bearer {credentials.token}\",\n",
    "        },\n",
    "        json={\n",
    "            \"model\": \"gemini-model\",\n",
    "            \"messages\": [\n",
    "                {\"role\": \"system\", \"content\": system_prompt},\n",
    "                {\"role\": \"user\", \"content\": prompt}\n",
    "            ],\n",
    "            \"max_tokens\": 1000,\n",
    "            \"temperature\": 0\n",
    "        }\n",
    "    )\n",
    "\n",
    "    # Parse the response from the API\n",
    "    response_json = response.json()\n",
    "    hypothesis = response_json.get('choices', [{}])[0].get('message', {}).get('content', '')\n",
    "\n",
    "    # Replace enriched terms in the hypothesis with detailed text\n",
    "    for term, stats in enriched_stats.items():\n",
    "        if term in hypothesis:\n",
    "            details = (\n",
    "                f\"Term: {term}\\tLibrary: {stats[9]}\\tRank: {stats[0]}\\tP-value: {float(stats[2]):.2e}\\tOdds Ratio: {float(stats[3]):.4f}\\n\"\n",
    "            )\n",
    "            hypothesis = hypothesis.replace(term, f\"{term} ({details})\")\n",
    "\n",
    "    return hypothesis\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "tuples_list = list(zip(df[\"rummagene\"], df['rummageo'], df[\"overlaps\"]))\n",
    "hypotheses = {}\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "for i, (pmcid, geo_id, ov) in enumerate(tuples_list):\n",
    "        term = f\"{pmcid};{geo_id}\"\n",
    "        if term not in hypotheses: \n",
    "            try:\n",
    "                terms, stats = get_enrichr_terms(ov)\n",
    "                hypothesis = fetch_hypothesis(pm_title=fin_pmc[pmcid][0], user_desc=fin_pmc[pmcid][1], desct=fin_pmc[pmcid][2], desc=fin_pmc[pmcid][3], gse_title=fin_gse[geo_id][0],gse_summary=fin_gse[geo_id][1], cond1=fin_gse[geo_id][2], cond2=fin_gse[geo_id][3],species=fin_gse[geo_id][4], term1=pmcid, term2=geo_id, enriched_terms=terms, enriched_stats=stats)\n",
    "                hypotheses[term] = hypothesis\n",
    "                with open('data_hyp.json', 'w') as json_file:\n",
    "                    json.dump(hypotheses, json_file, indent=4)\n",
    "            except:\n",
    "                print(term)\n",
    "                print(i)\n",
    "\n",
    "   \n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.15"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
